<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Overview of BiocPkgTools • BiocPkgTools</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Overview of BiocPkgTools">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-93043521-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-93043521-1');
</script>
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">BiocPkgTools</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.1.8</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/BiocPkgTools.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/seandavi/BiocPkgTools">
    <span class="fa fa-github fa-lg"></span>
     
    github
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="BiocPkgTools_files/htmlwidgets-1.3/htmlwidgets.js"></script><link href="BiocPkgTools_files/bioc_explore-1.0.0/bioc_explore.css" rel="stylesheet">
<link href="BiocPkgTools_files/bioc_explore-1.0.0/bootstrap.min.css" rel="stylesheet">
<script src="BiocPkgTools_files/bioc_explore-1.0.0/jquery-2.2.4.min.js"></script><script src="BiocPkgTools_files/bioc_explore-1.0.0/underscore-min.js"></script><script src="BiocPkgTools_files/bioc_explore-1.0.0/bootstrap.min.js"></script><script src="BiocPkgTools_files/bioc_explore-1.0.0/d3.v3.min.js"></script><script src="BiocPkgTools_files/bioc_explore-1.0.0/bubblechart.js"></script><script src="BiocPkgTools_files/bioc_explore-binding-1.1.5/bioc_explore.js"></script><link href="BiocPkgTools_files/vis-4.20.1/vis.css" rel="stylesheet">
<script src="BiocPkgTools_files/vis-4.20.1/vis.min.js"></script><script src="BiocPkgTools_files/visNetwork-binding-2.0.5/visNetwork.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Overview of BiocPkgTools</h1>
                        <h4 class="author">Shian Su</h4>
            <address class="author_afil">
      Walter and Eliza Hall Institute, Melbourne, Australia<br><h4 class="author">Vince Carey</h4>
            <address class="author_afil">
      Channing Lab, Brigham and Womens Hospital, Harvard University, Boston, MA USA<br><h4 class="author">Lori Shepherd</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY USA<br><h4 class="author">Martin Morgan</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY USA<br><h4 class="author">Sean Davis</h4>
            <address class="author_afil">
      National Cancer Institute, National Institutes of Health, Bethesda, MD USA<br><a class="author_email" href="mailto:#"></a><a href="mailto:seandavi@gmail.com">seandavi@gmail.com</a>
      </address>
                  
      
      <small class="dont-index">Source: <a href="https://github.com/seandavi/BiocPkgTools/blob/master/vignettes/BiocPkgTools.Rmd"><code>vignettes/BiocPkgTools.Rmd</code></a></small>
      <div class="hidden name"><code>BiocPkgTools.Rmd</code></div>

    </address>
</address>
</address>
</address>
</div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      Bioconductor has a rich ecosystem of metadata around packages, usage, and build status. This package is a simple collection of functions to access that metadata from R in a tidy data format. The goal is to expose metadata for data mining and value-added functionality such as package searching, text mining, and analytics on packages.
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>Bioconductor has a rich ecosystem of metadata around packages, usage, and build status. This package is a simple collection of functions to access that metadata from R in a tidy data format. The goal is to expose metadata for data mining and value-added functionality such as package searching, text mining, and analytics on packages.</p>
<p>Functionality includes access to :</p>
<ul>
<li>Download statistics</li>
<li>General package listing</li>
<li>Build reports</li>
<li>Package dependency graphs</li>
<li>Vignettes</li>
</ul>
<script type="text/javascript">
document.addEventListener("DOMContentLoaded", function() {
  document.querySelector("h1").className = "title";
});
</script><script type="text/javascript">
document.addEventListener("DOMContentLoaded", function() {
  var links = document.links;  
  for (var i = 0, linksLength = links.length; i < linksLength; i++)
    if (links[i].hostname != window.location.hostname)
      links[i].target = '_blank';
});
</script>
</div>
<div id="build-reports" class="section level1">
<h1 class="hasAnchor">
<a href="#build-reports" class="anchor"></a>Build reports</h1>
<p>The Bioconductor build reports are available online as HTML pages. However, they are not very computable. The <code>biocBuildReport</code> function does some heroic parsing of the HTML to produce a <em>tidy</em> data.frame for further processing in R.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(BiocPkgTools)</a>
<a class="sourceLine" id="cb1-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="../reference/biocBuildReport.html">biocBuildReport</a></span>())</a></code></pre></div>
<pre><code>## # A tibble: 6 x 9
##   pkg   version author commit last_changed_date   node  stage result
##   &lt;chr&gt; &lt;chr&gt;   &lt;chr&gt;  &lt;chr&gt;  &lt;dttm&gt;              &lt;chr&gt; &lt;chr&gt; &lt;chr&gt; 
## 1 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 malb… inst… OK    
## 2 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 malb… buil… OK    
## 3 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 malb… chec… OK    
## 4 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 toka… inst… OK    
## 5 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 toka… buil… OK    
## 6 a4    1.31.0  Tobia…  a53c… 2018-10-30 11:54:29 toka… chec… OK    
## # … with 1 more variable: bioc_version &lt;chr&gt;</code></pre>
<div id="personal-build-report" class="section level2">
<h2 class="hasAnchor">
<a href="#personal-build-report" class="anchor"></a>Personal build report</h2>
<p>Because developers may be interested in a quick view of their own packages, there is a simple function, <code>problemPage</code>, to produce an HTML report of the build status of packages matching a given author <em>regex</em>. The default is to report only “problem” build statuses (ERROR, WARNING).</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1"><span class="kw"><a href="../reference/problemPage.html">problemPage</a></span>()</a></code></pre></div>
<p>When run in an interactive environment, the <code>problemPage</code> function will open a browser window for user interaction. Note that if you want to include all your package results, not just the broken ones, simply specify <code>includeOK = TRUE</code>.</p>
</div>
</div>
<div id="download-statistics" class="section level1">
<h1 class="hasAnchor">
<a href="#download-statistics" class="anchor"></a>Download statistics</h1>
<p>Bioconductor supplies download stats for all packages. The <code>biocDownloadStats</code> function grabs all available download stats for all packages in all Experiment Data, Annotation Data, and Software packages. The results are returned as a tidy data.frame for further analysis.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="../reference/biocDownloadStats.html">biocDownloadStats</a></span>())</a></code></pre></div>
<pre><code>## # A tibble: 6 x 6
##   Package  Year Month Nb_of_distinct_IPs Nb_of_downloads repo    
##   &lt;fct&gt;   &lt;int&gt; &lt;fct&gt;              &lt;int&gt;           &lt;int&gt; &lt;chr&gt;   
## 1 ABarray  2019 Jan                  104             210 Software
## 2 ABarray  2019 Feb                   80             164 Software
## 3 ABarray  2019 Mar                   96             122 Software
## 4 ABarray  2019 Apr                    0               0 Software
## 5 ABarray  2019 May                    0               0 Software
## 6 ABarray  2019 Jun                    0               0 Software</code></pre>
<p>The download statistics reported are for <strong><em>all available versions</em></strong> of a package. There are no separate, publicly available statistics broken down by version.</p>
</div>
<div id="package-details" class="section level1">
<h1 class="hasAnchor">
<a href="#package-details" class="anchor"></a>Package details</h1>
<p>The R <code>DESCRIPTION</code> file contains a plethora of information regarding package authors, dependencies, versions, etc. In a repository such as Bioconductor, these details are available in bulk for all inclucded packages. The <code>biocPkgList</code> returns a data.frame with a row for each package. Tons of information are avaiable, as evidenced by the column names of the results.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">bpi =<span class="st"> </span><span class="kw"><a href="../reference/biocPkgList.html">biocPkgList</a></span>()</a>
<a class="sourceLine" id="cb6-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/colnames">colnames</a></span>(bpi)</a></code></pre></div>
<pre><code>##  [1] "Package"                   "Version"                  
##  [3] "Depends"                   "Suggests"                 
##  [5] "License"                   "MD5sum"                   
##  [7] "NeedsCompilation"          "Title"                    
##  [9] "Description"               "biocViews"                
## [11] "Author"                    "Maintainer"               
## [13] "git_url"                   "git_branch"               
## [15] "git_last_commit"           "git_last_commit_date"     
## [17] "Date/Publication"          "source.ver"               
## [19] "win.binary.ver"            "mac.binary.el-capitan.ver"
## [21] "vignettes"                 "vignetteTitles"           
## [23] "hasREADME"                 "hasNEWS"                  
## [25] "hasINSTALL"                "hasLICENSE"               
## [27] "Rfiles"                    "Enhances"                 
## [29] "dependsOnMe"               "Imports"                  
## [31] "importsMe"                 "suggestsMe"               
## [33] "LinkingTo"                 "Archs"                    
## [35] "VignetteBuilder"           "URL"                      
## [37] "SystemRequirements"        "BugReports"               
## [39] "Video"                     "linksToMe"                
## [41] "OS_type"                   "License_restricts_use"    
## [43] "PackageStatus"             "License_is_FOSS"          
## [45] "organism"</code></pre>
<p>Some of the variables are parsed to produce <code>list</code> columns.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(bpi)</a></code></pre></div>
<pre><code>## # A tibble: 6 x 45
##   Package Version Depends Suggests License MD5sum NeedsCompilation Title
##   &lt;chr&gt;   &lt;chr&gt;   &lt;list&gt;  &lt;list&gt;   &lt;chr&gt;   &lt;chr&gt;  &lt;chr&gt;            &lt;chr&gt;
## 1 a4      1.31.0  &lt;chr [… &lt;chr [4… GPL-3   31072… no               Auto…
## 2 a4Base  1.31.0  &lt;chr [… &lt;chr [2… GPL-3   2dec7… no               Auto…
## 3 a4Clas… 1.31.0  &lt;chr [… &lt;chr [1… GPL-3   4bbcd… no               Auto…
## 4 a4Core  1.31.0  &lt;chr [… &lt;chr [1… GPL-3   a2c0c… no               Auto…
## 5 a4Prep… 1.31.0  &lt;chr [… &lt;chr [2… GPL-3   087b7… no               Auto…
## 6 a4Repo… 1.31.0  &lt;chr [… &lt;chr [1… GPL-3   1635a… no               Auto…
## # … with 37 more variables: Description &lt;chr&gt;, biocViews &lt;list&gt;,
## #   Author &lt;list&gt;, Maintainer &lt;list&gt;, git_url &lt;chr&gt;, git_branch &lt;chr&gt;,
## #   git_last_commit &lt;chr&gt;, git_last_commit_date &lt;chr&gt;,
## #   `Date/Publication` &lt;chr&gt;, source.ver &lt;chr&gt;, win.binary.ver &lt;chr&gt;,
## #   `mac.binary.el-capitan.ver` &lt;chr&gt;, vignettes &lt;list&gt;,
## #   vignetteTitles &lt;list&gt;, hasREADME &lt;chr&gt;, hasNEWS &lt;chr&gt;,
## #   hasINSTALL &lt;chr&gt;, hasLICENSE &lt;chr&gt;, Rfiles &lt;chr&gt;, Enhances &lt;list&gt;,
## #   dependsOnMe &lt;list&gt;, Imports &lt;list&gt;, importsMe &lt;list&gt;,
## #   suggestsMe &lt;list&gt;, LinkingTo &lt;chr&gt;, Archs &lt;chr&gt;,
## #   VignetteBuilder &lt;chr&gt;, URL &lt;chr&gt;, SystemRequirements &lt;chr&gt;,
## #   BugReports &lt;chr&gt;, Video &lt;chr&gt;, linksToMe &lt;chr&gt;, OS_type &lt;chr&gt;,
## #   License_restricts_use &lt;chr&gt;, PackageStatus &lt;chr&gt;,
## #   License_is_FOSS &lt;chr&gt;, organism &lt;chr&gt;</code></pre>
<p>As a simple example of how these columns can be used, extracting the <code>importsMe</code> column to find the packages that import the <em><a href="https://bioconductor.org/packages/3.9/GEOquery">GEOquery</a></em> package.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">require</a></span>(dplyr)</a>
<a class="sourceLine" id="cb10-2" data-line-number="2">bpi =<span class="st"> </span><span class="kw"><a href="../reference/biocPkgList.html">biocPkgList</a></span>()</a>
<a class="sourceLine" id="cb10-3" data-line-number="3">bpi <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb10-4" data-line-number="4"><span class="st">    </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/filter.html">filter</a></span>(Package<span class="op">==</span><span class="st">"GEOquery"</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb10-5" data-line-number="5"><span class="st">    </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/pull.html">pull</a></span>(importsMe) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb10-6" data-line-number="6"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unlist">unlist</a></span>()</a></code></pre></div>
<pre><code>##  [1] "bigmelon"        "ChIPXpress"      "coexnet"        
##  [4] "crossmeta"       "EGAD"            "GAPGOM"         
##  [7] "GSEABenchmarkeR" "MACPET"          "minfi"          
## [10] "MoonlightR"      "phantasus"       "recount"        
## [13] "SRAdb"</code></pre>
</div>
<div id="package-explorer" class="section level1">
<h1 class="hasAnchor">
<a href="#package-explorer" class="anchor"></a>Package Explorer</h1>
<p>For the end user of Bioconductor, an analysis often starts with finding a package or set of packages that perform required tasks or are tailored to a specific operation or data type. The <code>bioc_explorer()</code> function implements an interactive bubble visualization with filtering based on biocViews terms. Bubbles are sized based on download statistics. Tooltip and detail-on-click capabilities are included. To start a local session:</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1"><span class="kw"><a href="../reference/biocExplore.html">biocExplore</a></span>()</a></code></pre></div>
<div id="htmlwidget-0e0a606e2d0f1826eb92" style="width:700px;height:432.632880098888px;" class="bioc_explore html-widget"></div>
<script type="application/json" data-for="htmlwidget-0e0a606e2d0f1826eb92">{"x":{"data":{"data":[{"authors":"Willem Talloen and Tobias Verbeke","name":"a4","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis Umbrella Package","downloads_month":191,"downloads_total":58452,"page":"http://bioconductor.org/packages/release/bioc/html/a4.html"},{"authors":"Willem Talloen, Tobias Verbeke, Tine Casneuf, An De Bondt, Steven Osselaer, Hinrich Goehlmann and Willem Ligtenberg","name":"a4Base","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis","downloads_month":242,"downloads_total":53768,"page":"http://bioconductor.org/packages/release/bioc/html/a4Base.html"},{"authors":"Willem Talloen and Tobias Verbeke","name":"a4Classif","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis Classification Package","downloads_month":189,"downloads_total":55114,"page":"http://bioconductor.org/packages/release/bioc/html/a4Classif.html"},{"authors":"Willem Talloen and Tobias Verbeke","name":"a4Core","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis Core Package","downloads_month":281,"downloads_total":55328,"page":"http://bioconductor.org/packages/release/bioc/html/a4Core.html"},{"authors":"Willem Talloen and Tobias Verbeke","name":"a4Preproc","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis Preprocessing Package","downloads_month":251,"downloads_total":54480,"page":"http://bioconductor.org/packages/release/bioc/html/a4Preproc.html"},{"authors":"Tobias Verbeke","name":"a4Reporting","license":"GPL-3","tags":["Microarray"],"description":"Automated Affymetrix Array Analysis Reporting Package","downloads_month":191,"downloads_total":49014,"page":"http://bioconductor.org/packages/release/bioc/html/a4Reporting.html"},{"authors":"Steffi Grote","name":"ABAEnrichment","license":"GPL (>= 2)","tags":["GeneSetEnrichment","GeneExpression"],"description":"The package ABAEnrichment is designed to test for\nenrichment of user defined candidate genes in the set of\nexpressed genes in different human brain regions. The core\nfunction 'aba_enrich' integrates the expression of the\ncandidate gene set (averaged across donors) and the structural\ninformation of the brain using an ontology, both provided by\nthe Allen Brain Atlas project. 'aba_enrich' interfaces the\nontology enrichment software FUNC to perform the statistical\nanalyses. Additional functions provided in this package like\n'get_expression' and 'plot_expression' facilitate exploring the\nexpression data, and besides the standard candidate vs.\nbackground gene set enrichment, also three additional tests are\nimplemented, e.g. for cases when genes are ranked instead of\ndivided into candidate and background.","downloads_month":254,"downloads_total":14252,"page":"http://bioconductor.org/packages/release/bioc/html/ABAEnrichment.html"},{"authors":"Yongming Andrew Sun","name":"ABarray","license":"GPL","tags":["Microarray","OneChannel","Preprocessing"],"description":"Automated pipline to perform gene expression analysis for\nApplied Biosystems Genome Survey Microarray (AB1700) data\nformat. Functions include data preprocessing, filtering,\ncontrol probe analysis, statistical analysis in one single\nfunction. A GUI interface is also provided. The raw data,\nprocessed data, graphics output and statistical results are\norganized into folders according to the analysis settings used.","downloads_month":210,"downloads_total":68840,"page":"http://bioconductor.org/packages/release/bioc/html/ABarray.html"},{"authors":"JiaHong Fong [cre, aut] and Monther Alhamdoosh [aut]","name":"abseqR","license":"GPL-3 | file LICENSE","tags":["Sequencing","Visualization","ReportWriting","QualityControl","MultipleComparison"],"description":"AbSeq is a comprehensive bioinformatic pipeline for the\nanalysis of sequencing datasets generated from antibody\nlibraries and abseqR is one of its packages. abseqR empowers\nthe users of abseqPy (https://github.com/malhamdoosh/abseqPy)\nwith plotting and reporting capabilities and allows them to\ngenerate interactive HTML reports for the convenience of\nviewing and sharing with other researchers. Additionally,\nabseqR extends abseqPy to compare multiple repertoire analyses\nand perform further downstream analysis on its output.","downloads_month":107,"downloads_total":788,"page":"http://bioconductor.org/packages/release/bioc/html/abseqR.html"},{"authors":"Wentao Yang","name":"ABSSeq","license":"GPL (>= 3)","tags":["DifferentialExpression"],"description":"Inferring differential expression genes by absolute counts\ndifference between two groups, utilizing Negative binomial\ndistribution and moderating fold-change according to\nheterogeneity of dispersion across expression level.","downloads_month":207,"downloads_total":25698,"page":"http://bioconductor.org/packages/release/bioc/html/ABSSeq.html"},{"authors":"Juan Pablo Acosta and Liliana Lopez-Kleine","name":"acde","license":"GPL-3","tags":["DifferentialExpression","TimeCourse","PrincipalComponent","GeneExpression","Microarray","mRNAMicroarray"],"description":"This package provides a multivariate inferential analysis\nmethod for detecting differentially expressed genes in gene\nexpression data. It uses artificial components, close to the\ndata's principal components but with an exact interpretation in\nterms of differential genetic expression, to identify\ndifferentially expressed genes while controlling the false\ndiscovery rate (FDR). The methods on this package are described\nin the vignette or in the article 'Multivariate Method for\nInferential Identification of Differentially Expressed Genes in\nGene Expression Experiments' by J. P. Acosta, L. Lopez-Kleine\nand S. Restrepo (2015, pending publication).","downloads_month":169,"downloads_total":12860,"page":"http://bioconductor.org/packages/release/bioc/html/acde.html"},{"authors":"Jos B Poell","name":"ACE","license":"GPL-2","tags":["CopyNumberVariation","DNASeq","Coverage","WholeGenome","Visualization","Sequencing"],"description":"Uses segmented copy number data to estimate tumor cell\npercentage and produce copy number plots displaying absolute\ncopy numbers.","downloads_month":127,"downloads_total":932,"page":"http://bioconductor.org/packages/release/bioc/html/ACE.html"},{"authors":"Jane Fridlyand <jfridlyand@cc.ucsf.edu> and Peter Dimitrov\n<dimitrov@stat.Berkeley.EDU>","name":"aCGH","license":"GPL-2","tags":["CopyNumberVariation","DataImport","Genetics"],"description":"Functions for reading aCGH data from image analysis output\nfiles and clone information files, creation of aCGH S3 objects\nfor storing these data. Basic methods for accessing/replacing,\nsubsetting, printing and plotting aCGH objects.","downloads_month":268,"downloads_total":94244,"page":"http://bioconductor.org/packages/release/bioc/html/aCGH.html"},{"authors":"Sean Davis <sdavis2@mail.nih.gov>","name":"ACME","license":"GPL (>= 2)","tags":["Technology","Microarray","Normalization"],"description":"ACME (Algorithms for Calculating Microarray Enrichment) is\na set of tools for analysing tiling array ChIP/chip, DNAse\nhypersensitivity, or other experiments that result in regions\nof the genome showing \"enrichment\".  It does not rely on a\nspecific array technology (although the array should be a\n\"tiling\" array), is very general (can be applied in experiments\nresulting in regions of enrichment), and is very insensitive to\narray noise or normalization methods.  It is also very fast and\ncan be applied on whole-genome tiling array experiments quite\neasily with enough memory.","downloads_month":239,"downloads_total":63488,"page":"http://bioconductor.org/packages/release/bioc/html/ACME.html"},{"authors":"Ramon Diaz-Uriarte <rdiaz02@gmail.com> and Oscar M. Rueda\n<rueda.om@gmail.com>. Wavelet-based aCGH smoothing code from Li\nHsu <lih@fhcrc.org> and Douglas Grove <dgrove@fhcrc.org>.\nImagemap code from Barry Rowlingson\n<B.Rowlingson@lancaster.ac.uk>. HaarSeg code from Erez\nBen-Yaacov; downloaded from\n<http://www.ee.technion.ac.il/people/YoninaEldar/Info/software/HaarSeg.htm>.","name":"ADaCGH2","license":"GPL (>= 3)","tags":["Microarray","CopyNumberVariants"],"description":"Analysis and plotting of array CGH data. Allows usage of\nCircular Binary Segementation, wavelet-based smoothing (both as\nin Liu et al., and HaarSeg as in Ben-Yaacov and Eldar), HMM,\nBioHMM, GLAD, CGHseg. Most computations are parallelized\n(either via forking or with clusters, including MPI and sockets\nclusters) and use ff for storing data.","downloads_month":163,"downloads_total":43888,"page":"http://bioconductor.org/packages/release/bioc/html/ADaCGH2.html"},{"authors":"André Luiz Molan <andre.molan@unesp.br>, Giordano Bruno Sanches\nSeco <giordano.bruno@unesp.br>, Agnes Alessandra Sekijima\nTakeda <agnes.takeda@unesp.br> and Jose Luiz Rybarczyk Filho\n<jose.luiz@unesp.br>","name":"ADAM","license":"GPL (>= 2)","tags":["GeneSetEnrichment","Pathways","KEGG"],"description":"ADAM is a GSEA R package created to group a set of genes\nfrom comparative samples (control versus experiment) belonging\nto different species according to their respective functions\n(Gene Ontology and KEGG pathways as default) and show their\nsignificance by calculating p-values referring togene diversity\nand activity. Each group of genes is called GFAG (Group of\nFunctionally Associated Genes).","downloads_month":24,"downloads_total":186,"page":"http://bioconductor.org/packages/release/bioc/html/ADAM.html"},{"authors":"Weixin Cai [aut, cre, cph], Nima Hejazi [aut], Alan Hubbard\n[ctb and ths]","name":"adaptest","license":"GPL-2","tags":["Genetics","GeneExpression","DifferentialExpression","Sequencing","Microarray","Regression","DimensionReduction","MultipleComparison"],"description":"Data-adaptive test statistics represent a general\nmethodology for performing multiple hypothesis testing on\neffects sizes while maintaining honest statistical inference\nwhen operating in high-dimensional settings (<doi here>). The\nutilities provided here extend the use of this general\nmethodology to many common data analytic challenges that arise\nin modern computational and genomic biology.","downloads_month":141,"downloads_total":2240,"page":"http://bioconductor.org/packages/release/bioc/html/adaptest.html"},{"authors":"Josie Hayes <jlhayes1982@gmail.com>","name":"adductomicsR","license":"Artistic-2.0","tags":["MassSpectrometry","Metabolomics","Software","ThirdPartyClient","DataImport","GUI"],"description":"Processes MS2 data to identify potentially adducted\npeptides from spectra that has been corrected for mass drift\nand retention time drift and quantifies MS1 level mass spectral\npeaks.","downloads_month":0,"downloads_total":56,"page":"http://bioconductor.org/packages/release/bioc/html/adductomicsR.html"},{"authors":"Claudio Lottaz and Joern Toedling","name":"adSplit","license":"GPL (>= 2)","tags":["Microarray","Clustering"],"description":"This package implements clustering of microarray gene\nexpression profiles according to functional annotations. For\neach term genes are annotated to, splits into two subclasses\nare computed and a significance of the supporting gene set is\ndetermined.","downloads_month":167,"downloads_total":56478,"page":"http://bioconductor.org/packages/release/bioc/html/adSplit.html"},{"authors":"Alessandro Lussana [aut and cre]","name":"AffiXcan","license":"GPL-3","tags":["GeneExpression","Transcription","GeneRegulation","DimensionReduction","Regression","PrincipalComponent"],"description":"Impute a GReX (Genetically Regulated Expression) for a set\nof genes in a sample of individuals, using a method based on\nthe Total Binding Affinity (TBA). Statistical models to impute\nGReX can be trained with a training dataset where the real\ntotal expression values are known.","downloads_month":90,"downloads_total":714,"page":"http://bioconductor.org/packages/release/bioc/html/AffiXcan.html"},{"authors":"Henrik Bengtsson [aut], James Bullard [aut], Robert Gentleman\n[ctb], Kasper Daniel Hansen [aut, cre], Jim Hester [ctb] and Martin Morgan [ctb]","name":"affxparser","license":"LGPL (>= 2)","tags":["Infrastructure","DataImport","Microarray","ProprietaryPlatforms","OneChannel"],"description":"Package for parsing Affymetrix files (CDF, CEL, CHP,\nBPMAP, BAR).  It provides methods for fast and memory efficient\nparsing of Affymetrix files using the Affymetrix' Fusion SDK.\nBoth ASCII- and binary-based files are supported.  Currently,\nthere are methods for reading chip definition file (CDF) and a\ncell intensity file (CEL).  These files can be read either in\nfull or in part.  For example, probe signals from a few\nprobesets can be extracted very quickly from a set of CEL files\ninto a convenient list structure.","downloads_month":2526,"downloads_total":377792,"page":"http://bioconductor.org/packages/release/bioc/html/affxparser.html"},{"authors":"Rafael A. Irizarry <rafa@jhu.edu>, Laurent Gautier\n<lgautier@gmail.com>, Benjamin Milo Bolstad\n<bmb@bmbolstad.com>,, Crispin Miller\n<cmiller@picr.man.ac.uk> with contributions from Magnus Astrand\n<Magnus.Astrand@astrazeneca.com>, Leslie M. Cope\n<cope@mts.jhu.edu>, Robert Gentleman, Jeff Gentry, Conrad\nHalling <challing@agilixcorp.com>, Wolfgang Huber, James\nMacDonald <jmacdon@u.washington.edu>, Benjamin I. P.\nRubinstein, Christopher Workman <workman@cbs.dtu.dk> and John\nZhang","name":"affy","license":"LGPL (>= 2.0)","tags":["Microarray","OneChannel","Preprocessing"],"description":"The package contains functions for exploratory\noligonucleotide array analysis. The dependence on tkWidgets\nonly concerns few convenience functions. 'affy' is fully\nfunctional without it.","downloads_month":11892,"downloads_total":2183674,"page":"http://bioconductor.org/packages/release/bioc/html/affy.html"},{"authors":"Rafael A. Irizarry <rafa@jhu.edu> and Zhijin Wu\n<zwu@stat.brown.edu> with contributions from Simon Cawley\n<simon_cawley@affymetrix.com>","name":"affycomp","license":"GPL (>= 2)","tags":["OneChannel","Microarray","Preprocessing"],"description":"The package contains functions that can be used to compare\nexpression measures for Affymetrix Oligonucleotide Arrays.","downloads_month":242,"downloads_total":95786,"page":"http://bioconductor.org/packages/release/bioc/html/affycomp.html"},{"authors":"Martin Morgan and Robert Gentleman","name":"AffyCompatible","license":"Artistic-2.0","tags":["Infrastructure","Microarray","OneChannel"],"description":"This package provides an interface to Affymetrix chip\nannotation and sample attribute files. The package allows an\neasy way for users to download and manage local data bases of\nAffynmetrix NetAffx annotation files. The package also provides\naccess to GeneChip Operating System (GCOS) and GeneChip Command\nConsole (AGCC)-compatible sample annotation files.","downloads_month":176,"downloads_total":66656,"page":"http://bioconductor.org/packages/release/bioc/html/AffyCompatible.html"},{"authors":"V. Carey","name":"affyContam","license":"Artistic-2.0","tags":["Infrastructure"],"description":"structured corruption of cel file data to demonstrate QA\neffectiveness","downloads_month":157,"downloads_total":56310,"page":"http://bioconductor.org/packages/release/bioc/html/affyContam.html"},{"authors":"James W. MacDonald","name":"affycoretools","license":"Artistic-2.0","tags":["ReportWriting","Microarray","OneChannel","GeneExpression"],"description":"Various wrapper functions that have been written to\nstreamline the more common analyses that a core Biostatistician\nmight see.","downloads_month":657,"downloads_total":167318,"page":"http://bioconductor.org/packages/release/bioc/html/affycoretools.html"},{"authors":"Xiwei Wu <xwu@coh.org> and Xuejun Arthur Li <xueli@coh.org>","name":"AffyExpress","license":"LGPL","tags":["Microarray","OneChannel","QualityControl","Preprocessing","DifferentialExpression","Annotation","ReportWriting","Visualization"],"description":"The purpose of this package is to provide a comprehensive\nand easy-to-use tool for quality assessment and to identify\ndifferentially expressed genes in the Affymetrix gene\nexpression data.","downloads_month":178,"downloads_total":63676,"page":"http://bioconductor.org/packages/release/bioc/html/AffyExpress.html"},{"authors":"K. Myriam Kroll, Fabrice Berger, Gerard Barkema and Enrico Carlon","name":"affyILM","license":"GPL-3","tags":["Microarray","OneChannel","Preprocessing"],"description":"affyILM is a preprocessing tool which estimates gene\nexpression levels for Affymetrix Gene Chips. Input from\nphysical chemistry is employed to first background subtract\nintensities before calculating concentrations on behalf of the\nLangmuir model.","downloads_month":163,"downloads_total":44648,"page":"http://bioconductor.org/packages/release/bioc/html/affyILM.html"},{"authors":"Ben Bolstad <bmb@bmbolstad.com>","name":"affyio","license":"LGPL (>= 2)","tags":["Microarray","DataImport","Infrastructure"],"description":"Routines for parsing Affymetrix data files based upon file\nformat information. Primary focus is on accessing the CEL and\nCDF file formats.","downloads_month":11003,"downloads_total":1565486,"page":"http://bioconductor.org/packages/release/bioc/html/affyio.html"},{"authors":"James Wettenhall [cre, aut], Ken Simpson [aut], Gordon Smyth\n[aut] and Keith Satterley [ctb]","name":"affylmGUI","license":"GPL (>=2)","tags":["GUI","GeneExpression","Transcription","DifferentialExpression","DataImport","Bayesian","Regression","TimeCourse","Microarray","mRNAMicroarray","OneChannel","ProprietaryPlatforms","BatchEffect","MultipleComparison","Normalization","Preprocessing","QualityControl"],"description":"A Graphical User Interface (GUI) for analysis of\nAffymetrix microarray gene expression data using the affy and\nlimma packages.","downloads_month":235,"downloads_total":110078,"page":"http://bioconductor.org/packages/release/bioc/html/affylmGUI.html"},{"authors":"Markus Schmidberger <schmidb@ibe.med.uni-muenchen.de>, Esmeralda Vicedo <e.vicedo@gmx.net> and Ulrich Mansmann\n<mansmann@ibe.med.uni-muenchen.de>","name":"affyPara","license":"GPL-3","tags":["Microarray","Preprocessing"],"description":"The package contains parallelized functions for\nexploratory oligonucleotide array analysis. The package is\ndesigned for large numbers of microarray data.","downloads_month":179,"downloads_total":56784,"page":"http://bioconductor.org/packages/release/bioc/html/affyPara.html"},{"authors":"H. Bjorn Nielsen and Laurent Gautier (Many thanks to Li Zhang\nearly communications about the existence of the PDNN program\nand related publications).","name":"affypdnn","license":"LGPL","tags":["OneChannel","Microarray","Preprocessing"],"description":"The package contains functions to perform the PDNN method\ndescribed by Li Zhang et al.","downloads_month":214,"downloads_total":75076,"page":"http://bioconductor.org/packages/release/bioc/html/affypdnn.html"},{"authors":"Ben Bolstad <bmb@bmbolstad.com>","name":"affyPLM","license":"GPL (>= 2)","tags":["Microarray","OneChannel","Preprocessing","QualityControl"],"description":"A package that extends and improves the functionality of\nthe base affy package. Routines that make heavy use of compiled\ncode for speed. Central focus is on implementation of methods\nfor fitting probe-level models and tools using these models.\nPLM based quality assessment tools.","downloads_month":2060,"downloads_total":855352,"page":"http://bioconductor.org/packages/release/bioc/html/affyPLM.html"},{"authors":"Craig Parman <craig.parman@bifx.org>, Conrad Halling\n<conrad.halling@bifx.org> and Robert Gentleman","name":"affyQCReport","license":"LGPL (>= 2)","tags":["Microarray","OneChannel","QualityControl"],"description":"This package creates a QC report for an AffyBatch object.\nThe report is intended to allow the user to quickly assess the\nquality of a set of arrays in an AffyBatch object.","downloads_month":500,"downloads_total":548506,"page":"http://bioconductor.org/packages/release/bioc/html/affyQCReport.html"},{"authors":"Mario Fasold","name":"AffyRNADegradation","license":"GPL-2","tags":["GeneExpression","Microarray","OneChannel","Preprocessing","QualityControl"],"description":"The package helps with the assessment and correction of\nRNA degradation effects in Affymetrix 3' expression arrays. The\nparameter d gives a robust and accurate measure of RNA\nintegrity. The correction removes the probe positional bias,\nand thus improves comparability of samples that are affected by\nRNA degradation.","downloads_month":155,"downloads_total":36906,"page":"http://bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html"},{"authors":"Stan Pounds <stanley.pounds@stjude.org>; Cuilan Lani Gao\n<cuilan.gao@stjude.org>","name":"AGDEX","license":"GPL Version 2 or later","tags":["Microarray","Genetics","GeneExpression"],"description":"A tool to evaluate agreement of differential expression\nfor cross-species genomics","downloads_month":196,"downloads_total":37806,"page":"http://bioconductor.org/packages/release/bioc/html/AGDEX.html"},{"authors":"Pedro Lopez-Romero <plopez@cnic.es>","name":"AgiMicroRna","license":"GPL-3","tags":["Microarray","AgilentChip","OneChannel","Preprocessing","DifferentialExpression"],"description":"Processing and Analysis of Agilent microRNA data","downloads_month":233,"downloads_total":65190,"page":"http://bioconductor.org/packages/release/bioc/html/AgiMicroRna.html"},{"authors":"Eric R. Paquet and Michael T. Hallett","name":"AIMS","license":"Artistic-2.0","tags":["ImmunoOncology","Classification","RNASeq","Microarray","Software","GeneExpression"],"description":"This package contains the AIMS implementation. It contains\nnecessary functions to assign the five intrinsic molecular\nsubtypes (Luminal A, Luminal B, Her2-enriched, Basal-like,\nNormal-like). Assignments could be done on individual samples\nas well as on dataset of gene expression data.","downloads_month":561,"downloads_total":29304,"page":"http://bioconductor.org/packages/release/bioc/html/AIMS.html"},{"authors":"Greg Gloor, Andrew Fernandes, Jean Macklaim, Arianne Albert, Matt Links, Thomas Quinn, Jia Rong Wu, Ruth Grace Wong and ","name":"ALDEx2","license":"file LICENSE","tags":["DifferentialExpression","RNASeq","Transcriptomics","GeneExpression","DNASeq","ChIPSeq","Bayesian","Sequencing","Software","Microbiome","Metagenomics","ImmunoOncology"],"description":"A differential abundance analysis for the comparison of\ntwo or more conditions. Useful for analyzing data from standard\nRNA-seq or meta-RNA-seq assays as well as selected and\nunselected values from in-vitro sequence selections. Uses a\nDirichlet-multinomial model to infer abundance from counts,\noptimized for three or more experimental replicates. The method\ninfers biological and sampling variation to calculate the\nexpected false discovery rate, given the variation, based on a\nWilcoxon Rank Sum test and Welch's t-test (via aldex.ttest), a\nKruskal-Wallis test (via aldex.kw), a generalized linear model\n(via aldex.glm), or a correlation test (via aldex.corr). All\ntests report p-values and Benjamini-Hochberg corrected\np-values.","downloads_month":532,"downloads_total":29068,"page":"http://bioconductor.org/packages/release/bioc/html/ALDEx2.html"},{"authors":"Charlotte Soneson [aut, cre] and Avi Srivastava [aut]","name":"alevinQC","license":"MIT + file LICENSE","tags":["QualityControl","SingleCell"],"description":"Generate QC reports summarizing the output from an alevin\nrun. Reports can be generated as html or pdf files, or as shiny\napplications.","downloads_month":0,"downloads_total":18,"page":"http://bioconductor.org/packages/release/bioc/html/alevinQC.html"},{"authors":"Jesper R Gadin and Lasse Folkersen","name":"AllelicImbalance","license":"GPL-3","tags":["Genetics","Infrastructure","Sequencing"],"description":"Provides a framework for allelic specific expression\ninvestigation using RNA-seq data.","downloads_month":170,"downloads_total":30002,"page":"http://bioconductor.org/packages/release/bioc/html/AllelicImbalance.html"},{"authors":"Michael Love and Rafael Irizarry","name":"alpine","license":"GPL (>=2)","tags":["Sequencing","RNASeq","AlternativeSplicing","DifferentialSplicing","GeneExpression","Transcription","Coverage","BatchEffect","Normalization","Visualization","QualityControl"],"description":"Fragment sequence bias modeling and correction for RNA-seq\ntranscript abundance estimation.","downloads_month":178,"downloads_total":9346,"page":"http://bioconductor.org/packages/release/bioc/html/alpine.html"},{"authors":"Laurent Gautier <lgautier@gmail.com>","name":"altcdfenvs","license":"GPL (>= 2)","tags":["Microarray","OneChannel","QualityControl","Preprocessing","Annotation","ProprietaryPlatforms","Transcription"],"description":"Convenience data structures and functions to handle\ncdfenvs","downloads_month":174,"downloads_total":60222,"page":"http://bioconductor.org/packages/release/bioc/html/altcdfenvs.html"},{"authors":"Dong Li, Shan He, Zhisong Pan and Guyu Hu","name":"AMOUNTAIN","license":"GPL (>= 2)","tags":["GeneExpression","Microarray","DifferentialExpression","Network"],"description":"A pure data-driven gene network, weighted gene\nco-expression network (WGCN) could be constructed only from\nexpression profile. Different layers in such networks may\nrepresent different time points, multiple conditions or various\nspecies. AMOUNTAIN aims to search active modules in multi-layer\nWGCN using a continuous optimization approach.","downloads_month":171,"downloads_total":8574,"page":"http://bioconductor.org/packages/release/bioc/html/AMOUNTAIN.html"},{"authors":"Kornel Labun [aut], Eivind Valen [cph and cre]","name":"amplican","license":"GPL-3","tags":["ImmunoOncology","Technology","Alignment","qPCR","CRISPR"],"description":"`amplican` performs alignment of the amplicon reads,\nnormalizes gathered data, calculates multiple statistics (e.g.\ncut rates, frameshifts) and presents results in form of\naggregated reports. Data and statistics can be broken down by\nexperiments, barcodes, user defined groups, guides and\namplicons allowing for quick identification of potential\nproblems.","downloads_month":237,"downloads_total":4926,"page":"http://bioconductor.org/packages/release/bioc/html/amplican.html"},{"authors":"Brad Friedman <friedman.brad@gene.com>, Adrian Nowicki, Hunter\nWhitney <hunter@hunterwhitney.com>, Matthew Brauer\n<brauer.matthew@gene.com>, Sara Moore and Konrad Debski","name":"AnalysisPageServer","license":"Artistic-2.0","tags":["GUI","Visualization","DataRepresentation"],"description":"AnalysisPageServer is a modular system that enables\nsharing of customizable R analyses via the web.","downloads_month":253,"downloads_total":17398,"page":"http://bioconductor.org/packages/release/bioc/html/AnalysisPageServer.html"},{"authors":"Ti-Tai Wang [aut, cre], Tzu-Pin Lu [aut], Chien-Yueh Lee[ctb and ]\nEric Y. Chuang [aut]","name":"anamiR","license":"GPL-2","tags":["Software","AssayDomain","GeneExpression","BiologicalQuestion","GeneSetEnrichment","GeneTarget","Normalization","Pathways","DifferentialExpression","GeneRegulation","ResearchField","Genetics","Technology","Microarray","Sequencing","miRNA","WorkflowStep"],"description":"This package is intended to identify potential\ninteractions of miRNA-target gene interactions from miRNA and\nmRNA expression data. It contains functions for statistical\ntest, databases of miRNA-target gene interaction and functional\nanalysis.","downloads_month":175,"downloads_total":8558,"page":"http://bioconductor.org/packages/release/bioc/html/anamiR.html"},{"authors":"Ted Wong","name":"Anaquin","license":"BSD_3_clause + file LICENSE","tags":["ImmunoOncology","DifferentialExpression","Preprocessing","RNASeq","GeneExpression","Software"],"description":"The project is intended to support the use of sequins\n(synthetic sequencing spike-in controls) owned and made\navailable by the Garvan Institute of Medical Research. The goal\nis to provide a standard open source library for quantitative\nanalysis, modelling and visualization of spike-in controls.","downloads_month":217,"downloads_total":8196,"page":"http://bioconductor.org/packages/release/bioc/html/Anaquin.html"},{"authors":"Aaron Taudt, Bjorn Bakker and David Porubsky","name":"AneuFinder","license":"Artistic-2.0","tags":["ImmunoOncology","Software","Sequencing","SingleCell","CopyNumberVariation","GenomicVariation","HiddenMarkovModel","WholeGenome"],"description":"AneuFinder implements functions for copy-number detection,\nbreakpoint detection, and karyotype and heterogeneity analysis\nin single-cell whole genome sequencing and strand-seq data.","downloads_month":316,"downloads_total":12554,"page":"http://bioconductor.org/packages/release/bioc/html/AneuFinder.html"},{"authors":"Tianle Ma and Aidong Zhang","name":"ANF","license":"GPL-3","tags":["Clustering","GraphAndNetwork","Network"],"description":"This package is used for complex patient clustering by\nintegrating multi-omic data through affinity network fusion.","downloads_month":162,"downloads_total":3702,"page":"http://bioconductor.org/packages/release/bioc/html/ANF.html"},{"authors":"Colin A. Smith <colin@colinsmith.org>","name":"annaffy","license":"LGPL","tags":["OneChannel","Microarray","Annotation","GO","Pathways","ReportWriting"],"description":"Functions for handling data from Bioconductor Affymetrix\nannotation data packages. Produces compact HTML and text\nreports including experimental data and URL links to many\nonline databases. Allows searching biological metadata using\nvarious criteria.","downloads_month":663,"downloads_total":687600,"page":"http://bioconductor.org/packages/release/bioc/html/annaffy.html"},{"authors":"Tim Yates <Tim.Yates@cruk.manchester.ac.uk>","name":"annmap","license":"GPL-2","tags":["Annotation","Microarray","OneChannel","ReportWriting","Transcription","Visualization"],"description":"annmap provides annotation mappings for Affymetrix exon\narrays and coordinate based queries to support deep sequencing\ndata analysis. Database access is hidden behind the API which\nprovides a set of functions such as genesInRange(),\ngeneToExon(), exonDetails(), etc. Functions to plot gene\narchitecture and BAM file data are also provided. Underlying\ndata are from Ensembl.","downloads_month":162,"downloads_total":22496,"page":"http://bioconductor.org/packages/release/bioc/html/annmap.html"},{"authors":"R. Gentleman","name":"annotate","license":"Artistic-2.0","tags":["Annotation","Pathways","GO"],"description":"Using R enviroments for annotation.","downloads_month":31382,"downloads_total":3070650,"page":"http://bioconductor.org/packages/release/bioc/html/annotate.html"},{"authors":"Hervé Pagès, Marc Carlson, Seth Falcon and Nianhua Li","name":"AnnotationDbi","license":"Artistic-2.0","tags":["Annotation","Microarray","Sequencing","GenomeAnnotation"],"description":"Provides user interface and database connection code for\nannotation data packages using SQLite data storage.","downloads_month":38594,"downloads_total":4843756,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html"},{"authors":"Martin Morgan [aut], Johannes Rainer [aut], Joachim Bargsten\n[ctb], Daniel Van Twisk [ctb] and Bioconductor Package Maintainer\n[cre]","name":"AnnotationFilter","license":"Artistic-2.0","tags":["Annotation","Infrastructure","Software"],"description":"This package provides class and other infrastructure to\nimplement filters for manipulating Bioconductor annotation\nresources. The filters will be used by ensembldb,\nOrganism.dplyr, and other packages.","downloads_month":6430,"downloads_total":223716,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationFilter.html"},{"authors":"Marc Carlson and Hervé Pagès","name":"AnnotationForge","license":"Artistic-2.0","tags":["Annotation","Infrastructure"],"description":"Provides code for generating Annotation packages and their\ndatabases.  Packages produced are intended to be used with\nAnnotationDbi.","downloads_month":3709,"downloads_total":445606,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationForge.html"},{"authors":"Stefan McKinnon Edwards <stefan.hoj-edwards@agrsci.dk>","name":"AnnotationFuncs","license":"GPL-2","tags":["AnnotationData","Software"],"description":"Functions for handling translating between different\nidentifieres using the Biocore Data Team data-packages (e.g.\norg.Bt.eg.db).","downloads_month":205,"downloads_total":45084,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationFuncs.html"},{"authors":"Martin Morgan [cre], Marc Carlson [ctb], Dan Tenenbaum [ctb], Sonali Arora [ctb] and Lori Shepherd [ctb]","name":"AnnotationHub","license":"Artistic-2.0","tags":["Infrastructure","DataImport","GUI","ThirdPartyClient"],"description":"This package provides a client for the Bioconductor\nAnnotationHub web resource. The AnnotationHub web resource\nprovides a central location where genomic files (e.g., VCF,\nbed, wig) and other resources from standard locations (e.g.,\nUCSC, Ensembl) can be discovered. The resource includes\nmetadata about each resource, e.g., a textual description,\ntags, and date of modification. The client creates and manages\na local cache of files retrieved by the user, helping with\nquick and reproducible access.","downloads_month":4976,"downloads_total":479376,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationHub.html"},{"authors":"Martin Morgan [ctb], Marc Carlson [ctb], Dan Tenenbaum [ctb], Sonali Arora [ctb], Paul Shannon [ctb], Lori Shepherd [ctb] and Bioconductor Package Maintainer [cre]","name":"AnnotationHubData","license":"Artistic-2.0","tags":["DataImport"],"description":"These recipes convert a wide variety and a growing number\nof public bioinformatic data sets into easily-used standard\nBioconductor data structures.","downloads_month":219,"downloads_total":16526,"page":"http://bioconductor.org/packages/release/bioc/html/AnnotationHubData.html"},{"authors":"Alexandre Kuhn <alexandre.m.kuhn@gmail.com>","name":"annotationTools","license":"GPL","tags":["Microarray","Annotation"],"description":"Functions to annotate microarrays, find orthologs, and\nintegrate heterogeneous gene expression profiles using\nannotation and other molecular biology information available as\nflat file database (plain text files).","downloads_month":235,"downloads_total":70324,"page":"http://bioconductor.org/packages/release/bioc/html/annotationTools.html"},{"authors":"Raymond G. Cavalcante [aut, cre] and Maureen A. Sartor [ths]","name":"annotatr","license":"GPL-3","tags":["Software","Annotation","GenomeAnnotation","FunctionalGenomics","Visualization"],"description":"Given a set of genomic sites/regions (e.g. ChIP-seq peaks,\nCpGs, differentially methylated CpGs or regions, SNPs, etc.) it\nis often of interest to investigate the intersecting genomic\nannotations. Such annotations include those relating to gene\nmodels (promoters, 5'UTRs, exons, introns, and 3'UTRs), CpGs\n(CpG islands, CpG shores, CpG shelves), or regulatory sequences\nsuch as enhancers. The annotatr package provides an easy way to\nsummarize and visualize the intersection of genomic\nsites/regions with genomic annotations.","downloads_month":504,"downloads_total":17510,"page":"http://bioconductor.org/packages/release/bioc/html/annotatr.html"},{"authors":"Ola Larsson <ola.larsson@ki.se>, Nahum Sonenberg\n<nahum.sonenberg@mcgill.ca> and Robert Nadon\n<robert.nadon@mcgill.ca>","name":"anota","license":"GPL-3","tags":["GeneExpression","DifferentialExpression","Microarray","Sequencing"],"description":"Genome wide studies of translational control is emerging\nas a tool to study verious biological conditions. The output\nfrom such analysis is both the mRNA level (e.g. cytosolic mRNA\nlevel) and the levl of mRNA actively involved in translation\n(the actively translating mRNA level) for each mRNA. The\nstandard analysis of such data strives towards identifying\ndifferential translational between two or more sample classes -\ni.e. differences in actively translated mRNA levels that are\nindependent of underlying differences in cytosolic mRNA levels.\nThis package allows for such analysis using partial variances\nand the random variance model. As 10s of thousands of mRNAs are\nanalyzed in parallell the library performs a number of tests to\nassure that the data set is suitable for such analysis.","downloads_month":165,"downloads_total":41478,"page":"http://bioconductor.org/packages/release/bioc/html/anota.html"},{"authors":"Christian Oertlin <christian.oertlin@ki.se>, Julie Lorent\n<julie.lorent@ki.se> and Ola Larsson <ola.larsson@ki.se>","name":"anota2seq","license":"GPL-3","tags":["ImmunoOncology","GeneExpression","DifferentialExpression","Microarray","GenomeWideAssociation","BatchEffect","Normalization","RNASeq","Sequencing","GeneRegulation","Regression"],"description":"anota2seq provides analysis of translational efficiency\nand differential expression analysis for polysome-profiling and\nribosome-profiling studies (two or more sample classes)\nquantified by RNA sequencing or DNA-microarray.\nPolysome-profiling and ribosome-profiling typically generate\ndata for two RNA sources; translated mRNA and total mRNA.\nAnalysis of differential expression is used to estimate changes\nwithin each RNA source (i.e. translated mRNA or total mRNA).\nAnalysis of translational efficiency aims to identify changes\nin translation efficiency leading to altered protein levels\nthat are independent of total mRNA levels (i.e. changes in\ntranslated mRNA that are independent of levels of total mRNA)\nor buffering, a mechanism regulating translational efficiency\nso that protein levels remain constant despite fluctuating\ntotal mRNA levels (i.e. changes in total mRNA that are\nindependent of levels of translated mRNA). anota2seq applies\nanalysis of partial variance and the random variance model to\nfulfill these tasks.","downloads_month":210,"downloads_total":4508,"page":"http://bioconductor.org/packages/release/bioc/html/anota2seq.html"},{"authors":"Hector Corrada Bravo, Rafael A. Irizarry and Jeffrey T. Leek","name":"antiProfiles","license":"Artistic-2.0","tags":["GeneExpression","Classification"],"description":"Implements gene expression anti-profiles as described in\nCorrada Bravo et al., BMC Bioinformatics 2012, 13:272\ndoi:10.1186/1471-2105-13-272.","downloads_month":156,"downloads_total":28356,"page":"http://bioconductor.org/packages/release/bioc/html/antiProfiles.html"},{"authors":"Denise Scholtens <dscholtens@northwestern.edu>","name":"apComplex","license":"LGPL","tags":["ImmunoOncology","NetworkInference","MassSpectrometry","GraphAndNetwork"],"description":"Functions to estimate a bipartite graph of protein complex\nmembership using AP-MS data.","downloads_month":273,"downloads_total":64240,"page":"http://bioconductor.org/packages/release/bioc/html/apComplex.html"},{"authors":"Anqi Zhu, Joseph G. Ibrahim and Michael I. Love","name":"apeglm","license":"GPL-2","tags":["ImmunoOncology","Sequencing","RNASeq","DifferentialExpression","GeneExpression","Bayesian"],"description":"apeglm provides Bayesian shrinkage estimators for effect\nsizes for a variety of GLM models, using approximation of the\nposterior for individual coefficients.","downloads_month":2294,"downloads_total":37498,"page":"http://bioconductor.org/packages/release/bioc/html/apeglm.html"},{"authors":"Sarah Sandmann","name":"appreci8R","license":"LGPL-3","tags":["VariantDetection","GeneticVariability","SNP","VariantAnnotation","Sequencing",""],"description":"The appreci8R is an R version of our appreci8-algorithm -\nA Pipeline for PREcise variant Calling Integrating 8 tools.\nVariant calling results of our standard appreci8-tools (GATK,\nPlatypus, VarScan, FreeBayes, LoFreq, SNVer, samtools and\nVarDict), as well as up to 5 additional tools is combined,\nevaluated and filtered.","downloads_month":101,"downloads_total":844,"page":"http://bioconductor.org/packages/release/bioc/html/appreci8R.html"},{"authors":"Henrik Bengtsson [aut, cre, cph], Pierre Neuvial [ctb] and Aaron\nLun [ctb]","name":"aroma.light","license":"GPL (>= 2)","tags":["Infrastructure","Microarray","OneChannel","TwoChannel","MultiChannel","Visualization","Preprocessing"],"description":"Methods for microarray analysis that take basic data types\nsuch as matrices and lists of vectors.  These methods can be\nused standalone, be utilized in other packages, or be wrapped\nup in higher-level classes.","downloads_month":3307,"downloads_total":288962,"page":"http://bioconductor.org/packages/release/bioc/html/aroma.light.html"},{"authors":"Audrey Kauffmann, Ibrahim Emam and Michael Schubert","name":"ArrayExpress","license":"Artistic-2.0","tags":["Microarray","DataImport","OneChannel","TwoChannel"],"description":"Access the ArrayExpress Repository at EBI and build\nBioconductor data structures: ExpressionSet, AffyBatch,\nNChannelSet","downloads_month":856,"downloads_total":156200,"page":"http://bioconductor.org/packages/release/bioc/html/ArrayExpress.html"},{"authors":"Angela Goncalves and Andrew Tikhonov","name":"ArrayExpressHTS","license":"Artistic License 2.0","tags":["ImmunoOncology","RNASeq","Sequencing"],"description":"RNA-Seq processing pipeline for public ArrayExpress\nexperiments or local datasets","downloads_month":131,"downloads_total":25540,"page":"http://bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html"},{"authors":"Z. Gao, A. Asare, R. Wang and V. Carey","name":"arrayMvout","license":"Artistic-2.0","tags":["Infrastructure","Microarray","QualityControl"],"description":"This package supports the application of diverse quality\nmetrics to AffyBatch instances, summarizing these metrics via\nPCA, and then performing parametric outlier detection on the\nPCs to identify aberrant arrays with a fixed Type I error rate","downloads_month":154,"downloads_total":51632,"page":"http://bioconductor.org/packages/release/bioc/html/arrayMvout.html"},{"authors":"Agnes Paquet and Jean Yee Hwa Yang <yeehwa@stat.berkeley.edu>","name":"arrayQuality","license":"LGPL","tags":["Microarray","TwoChannel","QualityControl","Visualization"],"description":"Functions for performing print-run and array level quality\nassessment.","downloads_month":197,"downloads_total":101684,"page":"http://bioconductor.org/packages/release/bioc/html/arrayQuality.html"},{"authors":"Audrey Kauffmann and Wolfgang Huber","name":"arrayQualityMetrics","license":"LGPL (>= 2)","tags":["Microarray","QualityControl","OneChannel","TwoChannel","ReportWriting"],"description":"This package generates microarray quality metrics reports\nfor data in Bioconductor microarray data containers\n(ExpressionSet, NChannelSet, AffyBatch). One and two color\narray platforms are supported.","downloads_month":977,"downloads_total":192450,"page":"http://bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html"},{"authors":"Xiwei Wu and Arthur Li","name":"ArrayTools","license":"LGPL (>= 2.0)","tags":["Microarray","OneChannel","QualityControl","Preprocessing","StatisticalMethod","DifferentialExpression","Annotation","ReportWriting","Visualization"],"description":"This package is designed to provide solutions for quality\nassessment and to detect differentially expressed genes for the\nAffymetrix GeneChips, including both 3' -arrays and gene 1.0-ST\narrays. The package generates comprehensive analysis reports in\nHTML format. Hyperlinks on the report page will lead to a\nseries of QC plots, processed data, and differentially\nexpressed gene lists. Differentially expressed genes are\nreported in tabular format with annotations hyperlinked to\nonline biological databases.","downloads_month":228,"downloads_total":67726,"page":"http://bioconductor.org/packages/release/bioc/html/ArrayTools.html"},{"authors":"Eitan Halper-Stromberg","name":"ArrayTV","license":"GPL (>= 2)","tags":["CopyNumberVariation"],"description":"Wave correction for genotyping and copy number arrays","downloads_month":164,"downloads_total":25606,"page":"http://bioconductor.org/packages/release/bioc/html/ArrayTV.html"},{"authors":"Jean-Philippe Fortin, Celia M.T. Greenwood and Aurelie Labbe.","name":"ARRmNormalization","license":"Artistic-2.0","tags":["DNAMethylation","TwoChannel","Preprocessing","Microarray"],"description":"Perform the Adaptive Robust Regression method (ARRm) for\nthe normalization of methylation data from the Illumina\nInfinium HumanMethylation 450k assay.","downloads_month":151,"downloads_total":28672,"page":"http://bioconductor.org/packages/release/bioc/html/ARRmNormalization.html"},{"authors":"David Jimenez-Morales, Alexandre Rosa Campos and John Von Dollen.","name":"artMS","license":"GPL (>= 3) + file LICENSE","tags":["Proteomics","DifferentialExpression","BiomedicalInformatics","SystemsBiology","MassSpectrometry","Annotation","QualityControl","GeneSetEnrichment","Clustering","Normalization","ImmunoOncology","MultipleComparison"],"description":"artMS provides a set of tools for the analysis of\nproteomics label-free datasets. It takes as input the MaxQuant\nsearch result output (evidence.txt file) and performs quality\ncontrol, relative quantification using MSstats, downstream\nanalysis and integration. artMS also provides a set of\nfunctions to re-format and make it compatible with other\nanalytical tools, including, SAINTq, SAINTexpress, Phosfate,\nand PHOTON.","downloads_month":153,"downloads_total":1180,"page":"http://bioconductor.org/packages/release/bioc/html/artMS.html"},{"authors":"Qian Zhang <qszhang@uw.edu>","name":"ASAFE","license":"Artistic-2.0","tags":["SNP","GenomeWideAssociation","LinkageDisequilibrium","BiomedicalInformatics","Genetics","ExperimentalDesign"],"description":"Given admixed individuals' bi-allelic SNP genotypes and\nancestry pairs (where each ancestry can take one of three\nvalues) for multiple SNPs, perform an EM algorithm to deal with\nthe fact that SNP genotypes are unphased with respect to\nancestry pairs, in order to estimate ancestry-specific allele\nfrequencies for all SNPs.","downloads_month":151,"downloads_total":7162,"page":"http://bioconductor.org/packages/release/bioc/html/ASAFE.html"},{"authors":"Likun Wang <wanglk@hsc.pku.edu.cn> and Tingting Li\n<litt@hsc.pku.edu.cn>.","name":"ASEB","license":"GPL (>= 3)","tags":["Proteomics"],"description":"ASEB is an R package to predict lysine sites that can be\nacetylated by a specific KAT-family.","downloads_month":162,"downloads_total":34348,"page":"http://bioconductor.org/packages/release/bioc/html/ASEB.html"},{"authors":"Hela Romdhani, Stepan Grinek, Heungsun Hwang and Aurelie\nLabbe.","name":"ASGSCA","license":"GPL-3","tags":["StructuralEquationModels"],"description":"The package provides tools to model and test the\nassociation between multiple genotypes and multiple traits,\ntaking into account the prior biological knowledge. Genes, and\nclinical pathways are incorporated in the model as latent\nvariables. The method is based on Generalized Structured\nComponent Analysis (GSCA).","downloads_month":187,"downloads_total":18724,"page":"http://bioconductor.org/packages/release/bioc/html/ASGSCA.html"},{"authors":"Gaëlle Lefort [aut, cre], Rémi Servien [aut], Patrick Tardivel\n[aut] and Nathalie Vialaneix [aut]","name":"ASICS","license":"GPL (>= 2)","tags":["Software","DataImport","Cheminformatics","Metabolomics"],"description":"With a set of pure metabolite reference spectra, ASICS\nquantifies concentration of metabolites in a complex spectrum.\nThe identification of metabolites is performed by fitting a\nmixture model to the spectra of the library with a sparse\npenalty. The method and its statistical properties are\ndescribed in Tardivel et al. (2017)\n<doi:10.1007/s11306-017-1244-5>.","downloads_month":176,"downloads_total":2634,"page":"http://bioconductor.org/packages/release/bioc/html/ASICS.html"},{"authors":"Estefania Mancini, Javier Iserte, Marcelo Yanovsky and Ariel\nChernomoretz","name":"ASpli","license":"GPL","tags":["ImmunoOncology","GeneExpression","Transcription","AlternativeSplicing","Coverage","DifferentialExpression","DifferentialSplicing","TimeCourse","RNASeq","GenomeAnnotation","Sequencing","Alignment"],"description":"Integrative pipeline for the analysis of alternative\nsplicing using RNAseq.","downloads_month":272,"downloads_total":11424,"page":"http://bioconductor.org/packages/release/bioc/html/ASpli.html"},{"authors":"Deepank Korandla [aut, cre] and Erik Wright [aut]","name":"AssessORF","license":"GPL-3","tags":["ComparativeGenomics","GenePrediction","GenomeAnnotation","Genetics","Proteomics","QualityControl","Visualization"],"description":"In order to assess the quality of a set of predicted genes\nfor a genome, evidence must first be mapped to that genome.\nNext, each gene must be categorized based on how strong the\nevidence is for or against that gene. The AssessORF package\nprovides the functions and class structures necessary for\naccomplishing those tasks, using proteomic hits and\nevolutionarily conserved start codons as the forms of evidence.","downloads_month":93,"downloads_total":680,"page":"http://bioconductor.org/packages/release/bioc/html/AssessORF.html"},{"authors":"Samsiddhi Bhattacharjee, Nilanjan Chatterjee and William\nWheeler","name":"ASSET","license":"GPL-2 + file LICENSE","tags":["Software"],"description":"An R package for subset-based analysis of heterogeneous\ntraits and subtypes","downloads_month":169,"downloads_total":25494,"page":"http://bioconductor.org/packages/release/bioc/html/ASSET.html"},{"authors":"Ying Shen, Andrea H. Bild, W. Evan Johnson, and Mumtehena\nRahman","name":"ASSIGN","license":"MIT + file LICENSE","tags":["Software","GeneExpression","Pathways","Bayesian"],"description":"ASSIGN is a computational tool to evaluate the pathway\nderegulation/activation status in individual patient samples.\nASSIGN employs a flexible Bayesian factor analysis approach\nthat adapts predetermined pathway signatures derived either\nfrom knowledge-based literature or from perturbation\nexperiments to the cell-/tissue-specific pathway signatures.\nThe deregulation/activation level of each context-specific\npathway is quantified to a score, which represents the extent\nto which a patient sample encompasses the pathway\nderegulation/activation signature.","downloads_month":170,"downloads_total":21584,"page":"http://bioconductor.org/packages/release/bioc/html/ASSIGN.html"},{"authors":"Jianhong Ou, Haibo Liu, Feng Yan, Jun Yu, Michelle Kelliher, Lucio Castilla, Nathan Lawson and Lihua Julie Zhu","name":"ATACseqQC","license":"GPL (>= 2)","tags":["Sequencing","DNASeq","ATACSeq","GeneRegulation","QualityControl","Coverage","NucleosomePositioning","ImmunoOncology"],"description":"ATAC-seq, an assay for Transposase-Accessible Chromatin\nusing sequencing, is a rapid and sensitive method for chromatin\naccessibility analysis. It was developed as an alternative\nmethod to MNase-seq, FAIRE-seq and DNAse-seq. Comparing to the\nother methods, ATAC-seq requires less amount of the biological\nsamples and time to process. In the process of analyzing\nseveral ATAC-seq dataset produced in our labs, we learned some\nof the unique aspects of the quality assessment for ATAC-seq\ndata.To help users to quickly assess whether their ATAC-seq\nexperiment is successful, we developed ATACseqQC package\npartially following the guideline published in Nature Method\n2013 (Greenleaf et al.), including diagnostic plot of fragment\nsize distribution, proportion of mitochondria reads, nucleosome\npositioning pattern, and CTCF or other Transcript Factor\nfootprints.","downloads_month":496,"downloads_total":17962,"page":"http://bioconductor.org/packages/release/bioc/html/ATACseqQC.html"},{"authors":"Chandler Kuo [aut], Sunyoung Shin [aut, cre] and Sunduz Keles\n[aut]","name":"atSNP","license":"GPL-2","tags":["Software","ChIPSeq","GenomeAnnotation","MotifAnnotation","Visualization"],"description":"atSNP performs affinity tests of motif matches with the\nSNP or the reference genomes and SNP-led changes in motif\nmatches.","downloads_month":0,"downloads_total":28,"page":"http://bioconductor.org/packages/release/bioc/html/atSNP.html"},{"authors":"Jessica Mar","name":"attract","license":"LGPL (>= 2.0)","tags":["ImmunoOncology","KEGG","Reactome","GeneExpression","Pathways","GeneSetEnrichment","Microarray","RNASeq"],"description":"This package contains the functions to find the gene\nexpression modules that represent the drivers of Kauffman's\nattractor landscape. The modules are the core attractor\npathways that discriminate between different cell types of\ngroups of interest. Each pathway has a set of synexpression\ngroups, which show transcriptionally-coordinated changes in\ngene expression.","downloads_month":210,"downloads_total":43600,"page":"http://bioconductor.org/packages/release/bioc/html/attract.html"},{"authors":"Sara Aibar, Stein Aerts. Laboratory of Computational Biology.\nVIB-KU Leuven Center for Brain & Disease Research. Leuven and Belgium.","name":"AUCell","license":"GPL-3","tags":["SingleCell","GeneSetEnrichment","Transcriptomics","Transcription","GeneExpression","WorkflowStep","Normalization"],"description":"AUCell allows to identify cells with active gene sets\n(e.g. signatures, gene modules...) in single-cell RNA-seq data.\nAUCell uses the \"Area Under the Curve\" (AUC) to calculate\nwhether a critical subset of the input gene set is enriched\nwithin the expressed genes for each cell. The distribution of\nAUC scores across all the cells allows exploring the relative\nexpression of the signature. Since the scoring method is\nranking-based, AUCell is independent of the gene expression\nunits and the normalization procedure. In addition, since the\ncells are evaluated individually, it can easily be applied to\nbigger datasets, subsetting the expression matrix if needed.","downloads_month":856,"downloads_total":16362,"page":"http://bioconductor.org/packages/release/bioc/html/AUCell.html"},{"authors":"Ines de Santiago, Wei Liu, Ke Yuan, Martin O'Reilly, Chandra SR\nChilamakuri, Bruce Ponder, Kerstin Meyer and Florian Markowetz","name":"BaalChIP","license":"Artistic-2.0","tags":["Software","ChIPSeq","Bayesian","Sequencing"],"description":"The package offers functions to process multiple ChIP-seq\nBAM files and detect allele-specific events. Computes allele\ncounts at individual variants (SNPs/SNVs), implements extensive\nQC steps to remove problematic variants, and utilizes a\nbayesian framework to identify statistically significant\nallele- specific events. BaalChIP is able to account for copy\nnumber differences between the two alleles, a known\nphenotypical feature of cancer samples.","downloads_month":166,"downloads_total":7726,"page":"http://bioconductor.org/packages/release/bioc/html/BaalChIP.html"},{"authors":"Raphael Gottardo","name":"BAC","license":"Artistic-2.0","tags":["Microarray","Transcription"],"description":"This package uses a Bayesian hierarchical model to detect\nenriched regions from ChIP-chip experiments","downloads_month":154,"downloads_total":50444,"page":"http://bioconductor.org/packages/release/bioc/html/BAC.html"},{"authors":"Maarten van Iterson [aut, cre] and Erik van Zwet [ctb]","name":"bacon","license":"GPL (>= 2)","tags":["ImmunoOncology","StatisticalMethod","Bayesian","Regression","GenomeWideAssociation","Transcriptomics","RNASeq","MethylationArray","BatchEffect","MultipleComparison"],"description":"Bacon can be used to remove inflation and bias often\nobserved in epigenome- and transcriptome-wide association\nstudies. To this end bacon constructs an empirical null\ndistribution using a Gibbs Sampling algorithm by fitting a\nthree-component normal mixture on z-scores.","downloads_month":270,"downloads_total":11650,"page":"http://bioconductor.org/packages/release/bioc/html/bacon.html"},{"authors":"Andreas Neudecker and Matthias Katzfuss","name":"BADER","license":"GPL-2","tags":["ImmunoOncology","Sequencing","RNASeq","DifferentialExpression","Software","SAGE"],"description":"For RNA sequencing count data, BADER fits a Bayesian\nhierarchical model. The algorithm returns the posterior\nprobability of differential expression for each gene between\ntwo groups A and B. The joint posterior distribution of the\nvariables in the model can be returned in the form of posterior\nsamples, which can be used for further down-stream analyses\nsuch as gene set enrichment.","downloads_month":169,"downloads_total":25336,"page":"http://bioconductor.org/packages/release/bioc/html/BADER.html"},{"authors":"Sarah Sandmann","name":"BadRegionFinder","license":"LGPL-3","tags":["Coverage","Sequencing","Alignment","WholeGenome","Classification"],"description":"BadRegionFinder is a package for identifying regions with\na bad, acceptable and good coverage in sequence alignment data\navailable as bam files. The whole genome may be considered as\nwell as a set of target regions. Various visual and textual\ntypes of output are available.","downloads_month":158,"downloads_total":8844,"page":"http://bioconductor.org/packages/release/bioc/html/BadRegionFinder.html"},{"authors":"Alejandro Quiroz-Zarate","name":"BAGS","license":"Artistic-2.0","tags":["Bayesian"],"description":"R package providing functions to perform geneset\nsignificance analysis over simple cross-sectional data between\n2 and 5 phenotypes of interest.","downloads_month":152,"downloads_total":24400,"page":"http://bioconductor.org/packages/release/bioc/html/BAGS.html"},{"authors":"Jack Fu [aut], Alyssa C. Frazee [aut, cre], Leonardo\nCollado-Torres [aut], Andrew E. Jaffe [aut], Jeffrey T. Leek\n[aut and ths]","name":"ballgown","license":"Artistic-2.0","tags":["ImmunoOncology","RNASeq","StatisticalMethod","Preprocessing","DifferentialExpression"],"description":"Tools for statistical analysis of assembled\ntranscriptomes, including flexible differential expression\nanalysis, visualization of transcript structures, and matching\nof assembled transcripts to annotation.","downloads_month":2329,"downloads_total":122522,"page":"http://bioconductor.org/packages/release/bioc/html/ballgown.html"},{"authors":"Alessandro Mammana [aut, cre] and Johannes Helmuth [aut]","name":"bamsignals","license":"GPL-2","tags":["DataImport","Sequencing","Coverage","Alignment"],"description":"This package allows to efficiently obtain count vectors\nfrom indexed bam files. It counts the number of reads in given\ngenomic ranges and it computes reads profiles and coverage\nprofiles. It also handles paired-end data.","downloads_month":602,"downloads_total":29464,"page":"http://bioconductor.org/packages/release/bioc/html/bamsignals.html"},{"authors":"Emma Schwager [aut, cre] and Curtis Huttenhower [aut]","name":"banocc","license":"MIT + file LICENSE","tags":["ImmunoOncology","Metagenomics","Software","Bayesian"],"description":"BAnOCC is a package designed for compositional data, where\neach sample sums to one. It infers the approximate covariance\nof the unconstrained data using a Bayesian model coded with\n`rstan`. It provides as output the `stanfit` object as well as\nposterior median and credible interval estimates for each\ncorrelation element.","downloads_month":168,"downloads_total":5086,"page":"http://bioconductor.org/packages/release/bioc/html/banocc.html"},{"authors":"Thomas Carroll and Marian Dore","name":"basecallQC","license":"GPL (>= 3)","tags":["Sequencing","Infrastructure","DataImport","QualityControl"],"description":"The basecallQC package provides tools to work with\nIllumina bcl2Fastq (versions >= 2.1.7) software.Prior to\nbasecalling and demultiplexing using the bcl2Fastq software,\nbasecallQC functions allow the user to update Illumina sample\nsheets from versions <= 1.8.9 to >= 2.1.7 standards, clean\nsample sheets of common problems such as invalid sample names\nand IDs, create read and index basemasks and the bcl2Fastq\ncommand. Following the generation of basecalled and\ndemultiplexed data, the basecallQC packages allows the user to\ngenerate HTML tables, plots and a self contained report of\nsummary metrics from Illumina XML output files.","downloads_month":170,"downloads_total":5812,"page":"http://bioconductor.org/packages/release/bioc/html/basecallQC.html"},{"authors":"Adrian Alexa","name":"BaseSpaceR","license":"Apache License 2.0","tags":["Infrastructure","DataRepresentation","ConnectTools","Software","DataImport","HighThroughputSequencing","Sequencing","Genetics"],"description":"A rich R interface to Illumina's BaseSpace cloud computing\nenvironment, enabling the fast development of data analysis and\nvisualisation tools.","downloads_month":227,"downloads_total":31390,"page":"http://bioconductor.org/packages/release/bioc/html/BaseSpaceR.html"},{"authors":"Carolin Walter","name":"Basic4Cseq","license":"LGPL-3","tags":["ImmunoOncology","Visualization","QualityControl","Sequencing","Coverage","Alignment","RNASeq","SequenceMatching","DataImport"],"description":"Basic4Cseq is an R/Bioconductor package for basic\nfiltering, analysis and subsequent visualization of 4C-seq\ndata. Virtual fragment libraries can be created for any\nBSGenome package, and filter functions for both reads and\nfragments and basic quality controls are included. Fragment\ndata in the vicinity of the experiment's viewpoint can be\nvisualized as a coverage plot based on a running median\napproach and a multi-scale contact profile.","downloads_month":229,"downloads_total":25096,"page":"http://bioconductor.org/packages/release/bioc/html/Basic4Cseq.html"},{"authors":"Catalina Vallejos [aut, cre], Nils Eling [aut], Alan\nO'Callaghan [ctb], Sylvia Richardson [ctb] and John Marioni [ctb]","name":"BASiCS","license":"GPL (>= 2)","tags":["Normalization","Sequencing","RNASeq","Software","GeneExpression","Transcriptomics","SingleCell","DifferentialExpression","Bayesian","CellBiology","ImmunoOncology"],"description":"Single-cell mRNA sequencing can uncover novel cell-to-cell\nheterogeneity in gene expression levels in seemingly\nhomogeneous populations of cells. However, these experiments\nare prone to high levels of technical noise, creating new\nchallenges for identifying genes that show genuine\nheterogeneous expression within the population of cells under\nstudy. BASiCS (Bayesian Analysis of Single-Cell Sequencing\ndata) is an integrated Bayesian hierarchical model to perform\nstatistical analyses of single-cell RNA sequencing datasets in\nthe context of supervised experiments (where the groups of\ncells of interest are known a priori, e.g. experimental\nconditions or cell types). BASiCS performs built-in data\nnormalisation (global scaling) and technical noise\nquantification (based on spike-in genes). BASiCS provides an\nintuitive detection criterion for highly (or lowly) variable\ngenes within a single group of cells. Additionally, BASiCS can\ncompare gene expression patterns between two or more\npre-specified groups of cells. Unlike traditional differential\nexpression tools, BASiCS quantifies changes in expression that\nlie beyond comparisons of means, also allowing the study of\nchanges in cell-to-cell heterogeneity. The latter can be\nquantified via a biological over-dispersion parameter that\nmeasures the excess of variability that is observed with\nrespect to Poisson sampling noise, after normalisation and\ntechnical noise removal. Due to the strong mean/over-dispersion\nconfounding that is typically observed for scRNA-seq datasets,\nBASiCS also tests for changes in residual over-dispersion,\ndefined by residual values with respect to a global\nmean/over-dispersion trend.","downloads_month":292,"downloads_total":7064,"page":"http://bioconductor.org/packages/release/bioc/html/BASiCS.html"},{"authors":"Annika Buerger","name":"BasicSTARRseq","license":"LGPL-3","tags":["PeakDetection","GeneRegulation","FunctionalPrediction","FunctionalGenomics","Coverage"],"description":"Basic peak calling on STARR-seq data based on a method\nintroduced in \"Genome-Wide Quantitative Enhancer Activity Maps\nIdentified by STARR-seq\" Arnold et al. Science. 2013 Mar\n1;339(6123):1074-7. doi: 10.1126/science. 1232542. Epub 2013\nJan 17.","downloads_month":164,"downloads_total":9476,"page":"http://bioconductor.org/packages/release/bioc/html/BasicSTARRseq.html"},{"authors":"Solaiappan Manimaran <manimaran_1975@hotmail.com>, W. Evan\nJohnson <wej@bu.edu>, Heather Selby <selbyh@bu.edu>, Claire\nRuberman <claireruberman@gmail.com>, Kwame Okrah\n<kwame.okrah@gmail.com> and Hector Corrada Bravo\n<hcorrada@gmail.com>","name":"BatchQC","license":"GPL (>= 2)","tags":["BatchEffect","GraphAndNetwork","Microarray","PrincipalComponent","Sequencing","Software","Visualization","QualityControl","RNASeq","Preprocessing","DifferentialExpression","ImmunoOncology"],"description":"Sequencing and microarray samples often are collected or\nprocessed in multiple batches or at different times. This often\nproduces technical biases that can lead to incorrect results in\nthe downstream analysis. BatchQC is a software tool that\nstreamlines batch preprocessing and evaluation by providing\ninteractive diagnostics, visualizations, and statistical\nanalyses to explore the extent to which batch variation impacts\nthe data. BatchQC diagnostics help determine whether batch\nadjustment needs to be done, and how correction should be\napplied before proceeding with a downstream analysis. Moreover,\nBatchQC interactively applies multiple common batch effect\napproaches to the data, and the user can quickly see the\nbenefits of each method. BatchQC is developed as a Shiny App.\nThe output is organized into multiple tabs, and each tab\nfeatures an important part of the batch effect analysis and\nvisualization of the data. The BatchQC interface has the\nfollowing analysis groups: Summary, Differential Expression,\nMedian Correlations, Heatmaps, Circular Dendrogram, PCA\nAnalysis, Shape, ComBat and SVA.","downloads_month":273,"downloads_total":15302,"page":"http://bioconductor.org/packages/release/bioc/html/BatchQC.html"},{"authors":"William Chad Young","name":"BayesKnockdown","license":"GPL-3","tags":["NetworkInference","GeneExpression","GeneTarget","Network","Bayesian"],"description":"A simple, fast Bayesian method for computing posterior\nprobabilities for relationships between a single predictor\nvariable and multiple potential outcome variables,\nincorporating prior probabilities of relationships. In the\ncontext of knockdown experiments, the predictor variable is the\nknocked-down gene, while the other genes are potential targets.\nCan also be used for differential expression/2-class data.","downloads_month":147,"downloads_total":6872,"page":"http://bioconductor.org/packages/release/bioc/html/BayesKnockdown.html"},{"authors":"Christiana Spyrou, Jonathan Cairns, Rory Stark, Andy Lynch, Simon Tavar\\\\'{e} and ","name":"BayesPeak","license":"GPL (>= 2)","tags":["ChIPSeq"],"description":"This package is an implementation of the BayesPeak\nalgorithm for peak-calling in ChIP-seq data.","downloads_month":203,"downloads_total":63948,"page":"http://bioconductor.org/packages/release/bioc/html/BayesPeak.html"},{"authors":"Wenhao Tang [aut, cre], Fran<U+00E7>ois Bertaux [aut], Philipp\nThomas [aut], Claire Stefanelli [aut], Malika Saint [aut], Samuel Marguerat [aut] and Vahid Shahrezaei [aut]","name":"bayNorm","license":"GPL (>= 2)","tags":["ImmunoOncology","Normalization","RNASeq","SingleCell","Sequencing"],"description":"bayNorm is used for normalizing single-cell RNA-seq data.","downloads_month":147,"downloads_total":1096,"page":"http://bioconductor.org/packages/release/bioc/html/bayNorm.html"},{"authors":"Thomas J. Hardcastle","name":"baySeq","license":"GPL-3","tags":["Sequencing","DifferentialExpression","MultipleComparison","SAGE"],"description":"This package identifies differential expression in\nhigh-throughput 'count' data, such as that derived from\nnext-generation sequencing machines, calculating estimated\nposterior likelihoods of differential expression (or more\ncomplex hypotheses) via empirical Bayesian methods.","downloads_month":838,"downloads_total":147842,"page":"http://bioconductor.org/packages/release/bioc/html/baySeq.html"},{"authors":"Sarah Sandmann","name":"BBCAnalyzer","license":"LGPL-3","tags":["Sequencing","Alignment","Coverage","GeneticVariability","SNP"],"description":"BBCAnalyzer is a package for visualizing the relative or\nabsolute number of bases, deletions and insertions at defined\npositions in sequence alignment data available as bam files in\ncomparison to the reference bases. Markers for the relative\nbase frequencies, the mean quality of the detected bases, known\nmutations or polymorphisms and variants called in the data may\nadditionally be included in the plots.","downloads_month":168,"downloads_total":11218,"page":"http://bioconductor.org/packages/release/bioc/html/BBCAnalyzer.html"},{"authors":"Adam Ameur <Adam.Ameur@genpat.uu.se>","name":"BCRANK","license":"GPL-2","tags":["MotifDiscovery","GeneRegulation"],"description":"Functions and classes for de novo prediction of\ntranscription factor binding consensus by heuristic search","downloads_month":187,"downloads_total":56352,"page":"http://bioconductor.org/packages/release/bioc/html/BCRANK.html"},{"authors":"Jiaxing Lin [aut, cre], Jeremy Gresham [aut], Jichun Xie [aut], Kouros Owzar [aut], Tongrong Wang [ctb], So Young Kim [ctb], James Alvarez [ctb], Jeffrey S. Damrauer [ctb], Scott Floyd\n[ctb], Joshua Granek [ctb], Andrew Allen [ctb] and Cliburn Chan\n[ctb]","name":"bcSeq","license":"GPL (>= 2)","tags":["ImmunoOncology","Alignment","CRISPR","Sequencing","SequenceMatching","MultipleSequenceAlignment","Software"],"description":"This Rcpp-based package implements a highly efficient data\nstructure and algorithm for performing alignment of short reads\nfrom CRISPR or shRNA screens to reference barcode library.\nSequencing error are considered and matching qualities are\nevaluated based on Phred scores. A Bayes' classifier is\nemployed to predict the originating barcode of a read. The\npackage supports provision of user-defined probability models\nfor evaluating matching qualities. The package also supports\nmulti-threading.","downloads_month":196,"downloads_total":2800,"page":"http://bioconductor.org/packages/release/bioc/html/bcSeq.html"},{"authors":"ZHENWEI DAI <daizwhao@gmail.com>","name":"BDMMAcorrect","license":"GPL (>= 2)","tags":["ImmunoOncology","BatchEffect","Microbiome","Bayesian"],"description":"Metagenomic sequencing techniques enable quantitative\nanalyses of the microbiome. However, combining the microbial\ndata from these experiments is challenging due to the\nvariations between experiments. The existing methods for\ncorrecting batch effects do not consider the interactions\nbetween variables—microbial taxa in microbial studies—and the\noverdispersion of the microbiome data. Therefore, they are not\napplicable to microbiome data. We develop a new method,\nBayesian Dirichlet-multinomial regression meta-analysis\n(BDMMA), to simultaneously model the batch effects and detect\nthe microbial taxa associated with phenotypes. BDMMA\nautomatically models the dependence among microbial taxa and is\nrobust to the high dimensionality of the microbiome and their\nassociation sparsity.","downloads_month":126,"downloads_total":1036,"page":"http://bioconductor.org/packages/release/bioc/html/BDMMAcorrect.html"},{"authors":"Aaron Lun [aut, cre], Hervé Pagès [aut] and Mike Smith [aut]","name":"beachmat","license":"GPL-3","tags":["DataRepresentation","DataImport","Infrastructure"],"description":"Provides a consistent C++ class interface for a variety of\ncommonly used matrix types, including sparse and HDF5-backed\nmatrices.","downloads_month":3229,"downloads_total":75752,"page":"http://bioconductor.org/packages/release/bioc/html/beachmat.html"},{"authors":"Mark Dunning, Mike Smith, Jonathan Cairns, Andy Lynch and Matt\nRitchie","name":"beadarray","license":"MIT + file LICENSE","tags":["Microarray","OneChannel","QualityControl","Preprocessing"],"description":"The package is able to read bead-level data (raw TIFFs and\ntext files) output by BeadScan as well as bead-summary data\nfrom BeadStudio. Methods for quality assessment and low-level\nanalysis are provided.","downloads_month":1166,"downloads_total":254660,"page":"http://bioconductor.org/packages/release/bioc/html/beadarray.html"},{"authors":"Jan Oosting","name":"beadarraySNP","license":"GPL-2","tags":["CopyNumberVariation","SNP","GeneticVariability","TwoChannel","Preprocessing","DataImport"],"description":"Importing data from Illumina SNP experiments and\nperforming copy number calculations and reports.","downloads_month":187,"downloads_total":59088,"page":"http://bioconductor.org/packages/release/bioc/html/beadarraySNP.html"},{"authors":"Mike Smith and Andy Lynch","name":"BeadDataPackR","license":"GPL-2","tags":["Microarray"],"description":"Provides functionality for the compression and\ndecompression of raw bead-level data from the Illumina\nBeadArray platform.","downloads_month":1086,"downloads_total":174008,"page":"http://bioconductor.org/packages/release/bioc/html/BeadDataPackR.html"},{"authors":"David T. Severson <david_severson@hms.harvard.edu>","name":"BEARscc","license":"GPL-3","tags":["ImmunoOncology","SingleCell","Clustering","Transcriptomics"],"description":"BEARscc is a noise estimation and injection tool that is\ndesigned to assess putative single-cell RNA-seq clusters in the\ncontext of experimental noise estimated by ERCC spike-in\ncontrols.","downloads_month":185,"downloads_total":2620,"page":"http://bioconductor.org/packages/release/bioc/html/BEARscc.html"},{"authors":"Kemal Akman <akman@mpipz.mpg.de>","name":"BEAT","license":"LGPL (>= 3.0)","tags":["ImmunoOncology","Genetics","MethylSeq","Software","DNAMethylation","Epigenetics"],"description":"Model-based analysis of single-cell methylation data","downloads_month":201,"downloads_total":21362,"page":"http://bioconductor.org/packages/release/bioc/html/BEAT.html"},{"authors":"David Rasp, Markus Merl and Ruslan Akulenko","name":"BEclear","license":"GPL-3","tags":["BatchEffect","DNAMethylation","Software","Preprocessing","StatisticalMethod"],"description":"Provides functions to detect and correct for batch effects\nin DNA methylation data. The core function is based on latent\nfactor models and can also be used to predict missing values in\nany other matrix containing real numbers.","downloads_month":160,"downloads_total":13680,"page":"http://bioconductor.org/packages/release/bioc/html/BEclear.html"},{"authors":"Iain Wallace","name":"bgafun","license":"Artistic-2.0","tags":["Classification"],"description":"A method to identify specifity determining residues in\nprotein families using Between Group Analysis","downloads_month":146,"downloads_total":49086,"page":"http://bioconductor.org/packages/release/bioc/html/bgafun.html"},{"authors":"Andrea Komljenovic [aut, cre], Julien Roux [aut and cre]","name":"BgeeDB","license":"GPL-3","tags":["Software","DataImport","Sequencing","GeneExpression","Microarray","GO","GeneSetEnrichment"],"description":"A package for the annotation and gene expression data\ndownload from Bgee database, and TopAnat analysis: GO-like\nenrichment of anatomical terms, mapped to genes by expression\npatterns.","downloads_month":207,"downloads_total":13088,"page":"http://bioconductor.org/packages/release/bioc/html/BgeeDB.html"},{"authors":"Alex Lewin and Natalia Bochkina","name":"BGmix","license":"GPL-2","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"Fully Bayesian mixture models for differential gene\nexpression","downloads_month":129,"downloads_total":33120,"page":"http://bioconductor.org/packages/release/bioc/html/BGmix.html"},{"authors":"Rich Savage, Emma Cooke, Robert Darkins and Yang Xu","name":"BHC","license":"GPL-3","tags":["Microarray","Clustering"],"description":"The method performs bottom-up hierarchical clustering,\nusing a Dirichlet Process (infinite mixture) to model\nuncertainty in the data and Bayesian model selection to decide\nat each step which clusters to merge.  This avoids several\nlimitations of traditional methods, for example how many\nclusters there should be and how to choose a principled\ndistance metric.  This implementation accepts multinomial (i.e.\ndiscrete, with 2+ categories) or time-series data. This version\nalso includes a randomised algorithm which is more efficient\nfor larger data sets.","downloads_month":271,"downloads_total":55726,"page":"http://bioconductor.org/packages/release/bioc/html/BHC.html"},{"authors":"Pierre Gestraud","name":"BicARE","license":"GPL-2","tags":["Microarray","Transcription","Clustering"],"description":"Biclustering Analysis and Results Exploration","downloads_month":252,"downloads_total":57470,"page":"http://bioconductor.org/packages/release/bioc/html/BicARE.html"},{"authors":"Ahrim Youn [aut, cre], Eladio Marquez [aut], Nathan Lawlor\n[aut], Michael Stitzel [aut] and Duygu Ucar [aut]","name":"BiFET","license":"GPL-3","tags":["ImmunoOncology","Genetics","Epigenetics","Transcription","GeneRegulation","ATACSeq","DNaseSeq","RIPSeq","Software"],"description":"BiFET identifies TFs whose footprints are over-represented\nin target regions compared to background regions after\ncorrecting for the bias arising from the imbalance in read\ncounts and GC contents between the target and background\nregions. For a given TF k, BiFET tests the null hypothesis that\nthe target regions have the same probability of having\nfootprints for the TF k as the background regions while\ncorrecting for the read count and GC content bias. For this, we\nuse the number of target regions with footprints for TF k, t_k\nas a test statistic and calculate the p-value as the\nprobability of observing t_k or more target regions with\nfootprints under the null hypothesis.","downloads_month":178,"downloads_total":2526,"page":"http://bioconductor.org/packages/release/bioc/html/BiFET.html"},{"authors":"Anand K. Gavai and Hannes Hettling","name":"BiGGR","license":"file LICENSE","tags":["Systems Biology","Pathway","Network","GraphAndNetwork","Visualization","Metabolomics"],"description":"This package provides an interface to simulate metabolic\nreconstruction from the BiGG database(http://bigg.ucsd.edu/)\nand other metabolic reconstruction databases. The package\nfacilitates flux balance analysis (FBA) and the sampling of\nfeasible flux distributions. Metabolic networks and estimated\nfluxes can be visualized with hypergraphs.","downloads_month":195,"downloads_total":34336,"page":"http://bioconductor.org/packages/release/bioc/html/BiGGR.html"},{"authors":"Tyler J. Gorrie-Stone [cre, aut], Ayden Saffari [aut], Karim\nMalki [aut] and Leonard C. Schalkwyk [aut]","name":"bigmelon","license":"GPL-3","tags":["DNAMethylation","Microarray","TwoChannel","Preprocessing","QualityControl","MethylationArray","DataImport","CpGIsland"],"description":"Methods for working with Illumina arrays using gdsfmt.","downloads_month":161,"downloads_total":7956,"page":"http://bioconductor.org/packages/release/bioc/html/bigmelon.html"},{"authors":"Peter M. Haverty","name":"bigmemoryExtras","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"This package defines a \"BigMatrix\" ReferenceClass which\nadds safety and convenience features to the\nfilebacked.big.matrix class from the bigmemory package.\nBigMatrix protects against segfaults by monitoring and\ngracefully restoring the connection to on-disk data and it also\nprotects against accidental data modification with a\nfilesystem-based permissions system. We provide utilities for\nusing BigMatrix-derived classes as assayData matrices within\nthe Biobase package's eSet family of classes. BigMatrix\nprovides some optimizations related to attaching to, and\nindexing into, file-backed matrices with dimnames.\nAdditionally, the package provides a \"BigMatrixFactor\" class, a\nfile-backed matrix with factor properties.","downloads_month":225,"downloads_total":21798,"page":"http://bioconductor.org/packages/release/bioc/html/bigmemoryExtras.html"},{"authors":"Lindsay Rutter [aut, cre] and Dianne Cook [aut]","name":"bigPint","license":"GPL-3","tags":["Clustering","DataImport","DifferentialExpression","GeneExpression","MultipleComparison","Normalization","Preprocessing","QualityControl","RNASeq","Sequencing","Software","Transcription","Visualization"],"description":"Methods for visualizing large multivariate datasets using\nstatic and interactive scatterplot matrices, parallel\ncoordinate plots, volcano plots, and litre plots. Includes\nexamples for visualizing RNA-sequencing datasets and\ndifferentially expressed genes.","downloads_month":0,"downloads_total":26,"page":"http://bioconductor.org/packages/release/bioc/html/bigPint.html"},{"authors":"Tyler Backman, Ronly Schlenk and Thomas Girke","name":"bioassayR","license":"Artistic-2.0","tags":["ImmunoOncology","MicrotitrePlateAssay","CellBasedAssays","Visualization","Infrastructure","DataImport","Bioinformatics","Proteomics","Metabolomics"],"description":"bioassayR is a computational tool that enables\nsimultaneous analysis of thousands of bioassay experiments\nperformed over a diverse set of compounds and biological\ntargets. Unique features include support for large-scale\ncross-target analyses of both public and custom bioassays,\ngeneration of high throughput screening fingerprints (HTSFPs),\nand an optional preloaded database that provides access to a\nsubstantial portion of publicly available bioactivity data.","downloads_month":245,"downloads_total":31888,"page":"http://bioconductor.org/packages/release/bioc/html/bioassayR.html"},{"authors":"R. Gentleman, V. Carey, M. Morgan and S. Falcon","name":"Biobase","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Functions that are needed by many other packages or which\nreplace R functions.","downloads_month":48499,"downloads_total":5705876,"page":"http://bioconductor.org/packages/release/bioc/html/Biobase.html"},{"authors":"Andrew J. Bass, David G. Robinson, Steve Lianoglou, Emily\nNelson, John D. Storey and with contributions from Laurent Gatto","name":"biobroom","license":"LGPL","tags":["MultipleComparison","DifferentialExpression","Regression","GeneExpression","Proteomics","DataImport"],"description":"This package contains methods for converting standard\nobjects constructed by bioinformatics packages, especially\nthose in Bioconductor, and converting them to tidy data. It\nthus serves as a complement to the broom package, and follows\nthe same the tidy, augment, glance division of tidying methods.\nTidying data makes it easy to recombine, reshape and visualize\nbioinformatics analyses.","downloads_month":305,"downloads_total":20166,"page":"http://bioconductor.org/packages/release/bioc/html/biobroom.html"},{"authors":"Karim Mezhoud [aut and cre]","name":"bioCancer","license":"AGPL-3 | file LICENSE","tags":["GUI","DataRepresentation","Network","MultipleComparison","Pathways","Reactome","Visualization","GeneExpression","GeneTarget"],"description":"bioCancer is a Shiny App to visualize and analyse\ninteractively Multi-Assays of Cancer Genomic Data.","downloads_month":147,"downloads_total":9000,"page":"http://bioconductor.org/packages/release/bioc/html/bioCancer.html"},{"authors":"R. Gentleman, W. Huber, F. Hahne, M. Morgan and S. Falcon","name":"BiocCaseStudies","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Software and data to support the case studies.","downloads_month":189,"downloads_total":58306,"page":"http://bioconductor.org/packages/release/bioc/html/BiocCaseStudies.html"},{"authors":"Bioconductor Package Maintainer [aut, cre], Lori Shepherd\n[ctb], Daniel von Twisk [ctb] and Kevin Rue [ctb]","name":"BiocCheck","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Executes Bioconductor-specific package checks.","downloads_month":1429,"downloads_total":114606,"page":"http://bioconductor.org/packages/release/bioc/html/BiocCheck.html"},{"authors":"Lori Shepherd [aut, cre] and Martin Morgan [aut]","name":"BiocFileCache","license":"Artistic-2.0","tags":["DataImport"],"description":"This package creates a persistent on-disk cache of files\nthat the user can add, update, and retrieve. It is useful for\nmanaging resources (such as custom Txdb objects) that are\ncostly or difficult to create, web resources, and data files\nused across sessions.","downloads_month":1231,"downloads_total":28160,"page":"http://bioconductor.org/packages/release/bioc/html/BiocFileCache.html"},{"authors":"The Bioconductor Dev Team","name":"BiocGenerics","license":"Artistic-2.0","tags":["Infrastructure"],"description":"S4 generic functions needed by many Bioconductor packages.","downloads_month":51820,"downloads_total":4608958,"page":"http://bioconductor.org/packages/release/bioc/html/BiocGenerics.html"},{"authors":"Li Long <li.long@isb-sib.ch>, Robert Gentleman\n<rgentlem@fhcrc.org> and Seth Falcon <sethf@fhcrc.org> Florian\nHahne <fhahne@fhcrc.org>","name":"biocGraph","license":"Artistic-2.0","tags":["Visualization","GraphAndNetwork"],"description":"This package provides examples and code that make use of\nthe different graph related packages produced by Bioconductor.","downloads_month":406,"downloads_total":68144,"page":"http://bioconductor.org/packages/release/bioc/html/biocGraph.html"},{"authors":"Aaron Lun [aut, cre and cph]","name":"BiocNeighbors","license":"GPL-3","tags":["Clustering","Classification"],"description":"Implements exact and approximate methods for nearest\nneighbor detection, in a framework that allows them to be\neasily switched within Bioconductor packages or workflows.\nExact searches can be performed using the k-means for k-nearest\nneighbors algorithm or with vantage point trees. Approximate\nsearches can be performed using the Annoy or HNSW libraries.\nSearching on either Euclidean or Manhattan distances is\nsupported. Parallelization is achieved for all methods by using\nBiocParallel. Functions are also provided to search for all\nneighbors within a given distance.","downloads_month":1190,"downloads_total":12290,"page":"http://bioconductor.org/packages/release/bioc/html/BiocNeighbors.html"},{"authors":"Vince Carey","name":"BiocOncoTK","license":"Artistic-2.0","tags":["CopyNumberVariation","CpGIsland","DNAMethylation","GeneExpression","GeneticVariability","SNP","Transcription","ImmunoOncology"],"description":"Provide a central interface to various tools for\ngenome-scale analysis of cancer studies.","downloads_month":202,"downloads_total":2888,"page":"http://bioconductor.org/packages/release/bioc/html/BiocOncoTK.html"},{"authors":"Lluís Revilla Sancho [aut, cre]\n(<https://orcid.org/0000-0001-9747-2570>), Pau Sancho-Bru [ths]\n(<https://orcid.org/0000-0001-5569-9259>) and Juan José Salvatella\nLozano [ths] (<https://orcid.org/0000-0001-7613-3908>)","name":"BioCor","license":"MIT + file LICENSE","tags":["StatisticalMethod","Clustering","GeneExpression","Network","Pathways","NetworkEnrichment","SystemsBiology"],"description":"Calculates functional similarities based on the pathways\ndescribed on KEGG and REACTOME or in gene sets. These\nsimilarities can be calculated for pathways or gene sets,\ngenes, or clusters and combined with other similarities. They\ncan be used to improve networks, gene selection, testing\nrelationships...","downloads_month":186,"downloads_total":6266,"page":"http://bioconductor.org/packages/release/bioc/html/BioCor.html"},{"authors":"Bioconductor Package Maintainer [cre], Martin Morgan [aut], Valerie Obenchain [aut], Michel Lang [aut], Ryan Thompson\n[aut] and Nitesh Turaga [aut]","name":"BiocParallel","license":"GPL-2 | GPL-3","tags":["Infrastructure"],"description":"This package provides modified versions and novel\nimplementation of functions for parallel evaluation, tailored\nto use with Bioconductor objects.","downloads_month":44610,"downloads_total":2257746,"page":"http://bioconductor.org/packages/release/bioc/html/BiocParallel.html"},{"authors":"Sean Davis [aut, cre], Shian Su [ctb], Lori Shepherd [ctb], Martin Morgan [ctb] and Vince Carey [ctb]","name":"BiocPkgTools","license":"MIT + file LICENSE","tags":["Software","Infrastructure"],"description":"Bioconductor has a rich ecosystem of metadata around\npackages, usage, and build status. This package is a simple\ncollection of functions to access that metadata from R. The\ngoal is to expose metadata for data mining and value-added\nfunctionality such as package searching, text mining, and\nanalytics on packages.","downloads_month":140,"downloads_total":1198,"page":"http://bioconductor.org/packages/release/bioc/html/BiocPkgTools.html"},{"authors":"Aaron Lun [aut, cre and cph]","name":"BiocSingular","license":"GPL-3","tags":["Software","DimensionReduction","PrincipalComponent"],"description":"Implements exact and approximate methods for singular\nvalue decomposition and principal components analysis, in a\nframework that allows them to be easily switched within\nBioconductor packages or workflows. Where possible,\nparallelization is achieved using the BiocParallel framework.","downloads_month":0,"downloads_total":210,"page":"http://bioconductor.org/packages/release/bioc/html/BiocSingular.html"},{"authors":"Vince Carey","name":"BiocSklearn","license":"Artistic-2.0","tags":["StatisticalMethod","DimensionReduction","Infrastructure"],"description":"This package provides interfaces to selected sklearn\nelements, and demonstrates fault tolerant use of python modules\nrequiring extensive iteration.","downloads_month":152,"downloads_total":3960,"page":"http://bioconductor.org/packages/release/bioc/html/BiocSklearn.html"},{"authors":"Andrzej Oleś, Martin Morgan and Wolfgang Huber","name":"BiocStyle","license":"Artistic-2.0","tags":["Software"],"description":"Provides standard formatting styles for Bioconductor PDF\nand HTML documents. Package vignettes illustrate use and\nfunctionality.","downloads_month":3957,"downloads_total":316298,"page":"http://bioconductor.org/packages/release/bioc/html/BiocStyle.html"},{"authors":"Martin Morgan [aut], Marcel Ramos [ctb], Bioconductor Package\nMaintainer [ctb and cre]","name":"BiocVersion","license":"Artistic-2.0","tags":["Infrastructure"],"description":"This package provides repository information for the\nappropriate version of Bioconductor.","downloads_month":33059,"downloads_total":321138,"page":"http://bioconductor.org/packages/release/bioc/html/BiocVersion.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>, BJ Harshfield\n<rebjh@channing.harvard.edu>, S Falcon <sfalcon@fhcrc.org> and Sonali Arora","name":"biocViews","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Infrastructure to support Bioconductor 'views' used to\nclassify software packages. 'biocViews' are directed acyclic\ngraphs of terms from a controlled vocabulary. There are three\nmajor classifications, corresponding to 'software',\n'annotation', and 'experiment data' packages.","downloads_month":3384,"downloads_total":201494,"page":"http://bioconductor.org/packages/release/bioc/html/biocViews.html"},{"authors":"Mike Smith [aut, cre] and Andrzej Oleś [aut]","name":"BiocWorkflowTools","license":"MIT + file LICENSE","tags":["Software","ReportWriting"],"description":"Provides functions to ease the transition between\nRmarkdown and LaTeX documents when authoring a Bioconductor\nWorkflow.","downloads_month":194,"downloads_total":8684,"page":"http://bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html"},{"authors":"B. Ding, R. Gentleman and Vincent Carey","name":"bioDist","license":"Artistic-2.0","tags":["Clustering","Classification"],"description":"A collection of software tools for calculating distance\nmeasures.","downloads_month":499,"downloads_total":131764,"page":"http://bioconductor.org/packages/release/bioc/html/bioDist.html"},{"authors":"Steffen Durinck [aut], Wolfgang Huber [aut], Sean Davis [ctb], Francois Pepin [ctb], Vince S Buffalo [ctb], Mike Smith [ctb and cre]","name":"biomaRt","license":"Artistic-2.0","tags":["Annotation"],"description":"In recent years a wealth of biological data has become\navailable in public data repositories. Easy access to these\nvaluable data resources and firm integration with data analysis\nis needed for comprehensive bioinformatics data analysis.\nbiomaRt provides an interface to a growing collection of\ndatabases implementing the BioMart software suite\n(http://www.biomart.org). The package enables retrieval of\nlarge amounts of data in a uniform way without the need to know\nthe underlying database schemas or write complex SQL queries.\nThe most prominent examples of BioMart databases are maintain\nby Ensembl, which provides biomaRt users direct access to a\ndiverse set of data and enables a wide range of powerful online\nqueries from gene annotation to database mining.","downloads_month":23711,"downloads_total":3399104,"page":"http://bioconductor.org/packages/release/bioc/html/biomaRt.html"},{"authors":"Paul J. McMurdie <mcmurdie@stanford.edu> and Joseph N Paulson\n<jpaulson@jimmy.harvard.edu>","name":"biomformat","license":"GPL-2","tags":["ImmunoOncology","DataImport","Metagenomics","Microbiome"],"description":"This is an R package for interfacing with the BIOM format.\nThis package includes basic tools for reading biom-format\nfiles, accessing and subsetting data tables from a biom object\n(which is more complex than a single table), as well as limited\nsupport for writing a biom-object back to a biom-format file.\nThe design of this API is intended to match the python API and\nother tools included with the biom-format project, but with a\ndecidedly \"R flavor\" that should be familiar to R users. This\nincludes S4 classes and methods, as well as extensions of\ncommon core functions/methods.","downloads_month":5098,"downloads_total":166452,"page":"http://bioconductor.org/packages/release/bioc/html/biomformat.html"},{"authors":"Junfang Chen and Emanuel Schwarz","name":"BioMM","license":"GPL-3","tags":["Genetics","Classification","Regression","Pathways","GO","Software"],"description":"The identification of reproducible biological patterns\nfrom high-dimensional omics data is a key factor in\nunderstanding the biology of complex disease or traits.\nIncorporating prior biological knowledge into machine learning\nis an important step in advancing such research. We have\nproposed a biologically informed multi-stage machine learing\nframework termed BioMM specifically for phenotype prediction\nbased on omics-scale data where we can evaluate different\nmachine learning models with various prior biological meta\ninformation.","downloads_month":0,"downloads_total":30,"page":"http://bioconductor.org/packages/release/bioc/html/BioMM.html"},{"authors":"Elizabeth Whalen","name":"BioMVCClass","license":"LGPL","tags":["Visualization","Infrastructure","GraphAndNetwork"],"description":"Creates classes used in model-view-controller (MVC) design","downloads_month":163,"downloads_total":52158,"page":"http://bioconductor.org/packages/release/bioc/html/BioMVCClass.html"},{"authors":"Yang Du","name":"biomvRCNS","license":"GPL (>= 2)","tags":["aCGH","CopyNumberVariation","Microarray","Sequencing","Sequencing","Visualization","Genetics"],"description":"In this package, a Hidden Semi Markov Model (HSMM) and one\nhomogeneous segmentation model are designed and implemented for\nsegmentation genomic data, with the aim of assisting in\ntranscripts detection using high throughput technology like\nRNA-seq or tiling array, and copy number analysis using aCGH or\nsequencing.","downloads_month":156,"downloads_total":28668,"page":"http://bioconductor.org/packages/release/bioc/html/biomvRCNS.html"},{"authors":"Marcus Dittrich and Daniela Beisser","name":"BioNet","license":"GPL (>= 2)","tags":["Microarray","DataImport","GraphAndNetwork","Network","NetworkEnrichment","GeneExpression","DifferentialExpression"],"description":"This package provides functions for the integrated\nanalysis of protein-protein interaction networks and the\ndetection of functional modules. Different datasets can be\nintegrated into the network by assigning p-values of\nstatistical tests to the nodes of the network. E.g. p-values\nobtained from the differential expression of the genes from an\nAffymetrix array are assigned to the nodes of the network. By\nfitting a beta-uniform mixture model and calculating scores\nfrom the p-values, overall scores of network regions can be\ncalculated and an integer linear programming algorithm\nidentifies the maximum scoring subnetwork.","downloads_month":479,"downloads_total":82970,"page":"http://bioconductor.org/packages/release/bioc/html/BioNet.html"},{"authors":"Vinícius Jardim, Suzana Santos, André Fujita, and Marcos\nBuckeridge","name":"BioNetStat","license":"GPL (>= 3)","tags":["Network","NetworkInference","Pathways","GraphAndNetwork","Sequencing","Microarray","Metabolomics","Proteomics","GeneExpression","RNASeq","SystemsBiology","DifferentialExpression","GeneSetEnrichment","ImmunoOncology"],"description":"A package to perform differential network analysis,\ndifferential node analysis (differential coexpression\nanalysis), network and metabolic pathways view.","downloads_month":163,"downloads_total":2312,"page":"http://bioconductor.org/packages/release/bioc/html/BioNetStat.html"},{"authors":"Jitao David Zhang <jitao_david.zhang@roche.com>, Laura Badi, Gregor Sturm and Roland Ambs","name":"BioQC","license":"GPL (>=3)","tags":["GeneExpression","QualityControl","StatisticalMethod"],"description":"BioQC performs quality control of high-throughput\nexpression data based on tissue gene signatures. It can detect\ntissue heterogeneity in gene expression data. The core\nalgorithm is a Wilcoxon-Mann-Whitney test that is optimised for\nhigh performance.","downloads_month":164,"downloads_total":9502,"page":"http://bioconductor.org/packages/release/bioc/html/BioQC.html"},{"authors":"Li Hong sysptm@gmail.com","name":"BioSeqClass","license":"LGPL (>= 2.0)","tags":["Classification"],"description":"Extracting Features from Biological Sequences and Building\nClassification Model","downloads_month":228,"downloads_total":49092,"page":"http://bioconductor.org/packages/release/bioc/html/BioSeqClass.html"},{"authors":"Philippe Rinaudo <phd.rinaudo@gmail.com> and Etienne Thevenot\n<etienne.thevenot@cea.fr>","name":"biosigner","license":"CeCILL","tags":["Classification","FeatureExtraction","Transcriptomics","Proteomics","Metabolomics","Lipidomics"],"description":"Feature selection is critical in omics data analysis to\nextract restricted and meaningful molecular signatures from\ncomplex and high-dimension data, and to build robust\nclassifiers. This package implements a new method to assess the\nrelevance of the variables for the prediction performances of\nthe classifier. The approach can be run in parallel with the\nPLS-DA, Random Forest, and SVM binary classifiers. The\nsignatures and the corresponding 'restricted' models are\nreturned, enabling future predictions on new datasets. A Galaxy\nimplementation of the package is available within the\nWorkflow4metabolomics.org online infrastructure for\ncomputational metabolomics.","downloads_month":188,"downloads_total":10914,"page":"http://bioconductor.org/packages/release/bioc/html/biosigner.html"},{"authors":"H. Pagès, P. Aboyoun, R. Gentleman, and S. DebRoy","name":"Biostrings","license":"Artistic-2.0","tags":["SequenceMatching","Alignment","Sequencing","Genetics","DataImport","DataRepresentation","Infrastructure"],"description":"Memory efficient string containers, string matching\nalgorithms, and other utilities, for fast manipulation of large\nbiological sequences or sets of sequences.","downloads_month":39753,"downloads_total":3809590,"page":"http://bioconductor.org/packages/release/bioc/html/Biostrings.html"},{"authors":"Anneleen Daemen <daemena@gene.com> and Matthew Brauer\n<matthejb@gene.com>","name":"biosvd","license":"Artistic-2.0","tags":["TimeCourse","Visualization"],"description":"The biosvd package contains functions to reduce the input\ndata set from the feature x assay space to the reduced\ndiagonalized eigenfeature x eigenassay space, with the\neigenfeatures and eigenassays unique orthonormal superpositions\nof the features and assays, respectively. Results of SVD\napplied to the data can subsequently be inspected based on\ngenerated graphs, such as a heatmap of the eigenfeature x assay\nmatrix and a bar plot with the eigenexpression fractions of all\neigenfeatures. These graphs aid in deciding which eigenfeatures\nand eigenassays to filter out (i.e., eigenfeatures representing\nsteady state, noise, or experimental artifacts; or when applied\nto the variance in the data, eigenfeatures representing\nsteady-scale variance). After possible removal of steady state\nexpression, steady-scale variance, noise and experimental\nartifacts, and after re-applying SVD to the normalized data, a\nsummary html report of the eigensystem is generated, containing\namong others polar plots of the assays and features, a table\nwith the list of features sortable according to their\ncoordinates, radius and phase in the polar plot, and a\nvisualization of the data sorted according to the two selected\neigenfeatures and eigenassays with colored feature/assay\nannotation information when provided. This gives a global\npicture of the dynamics of expression/intensity levels, in\nwhich individual features and assays are classified in groups\nof similar regulation and function or similar cellular state\nand biological phenotype.","downloads_month":175,"downloads_total":20770,"page":"http://bioconductor.org/packages/release/bioc/html/biosvd.html"},{"authors":"Nima Hejazi [aut, cre, cph], Alan Hubbard [aut, ths] and Weixin\nCai [ctb]","name":"biotmle","license":"file LICENSE","tags":["GeneExpression","DifferentialExpression","Sequencing","Microarray","RNASeq"],"description":"This package facilitates the discovery of biomarkers from\nbiological sequencing data (e.g., microarrays, RNA-seq) based\non the associations of potential biomarkers with exposure and\noutcome variables by implementing an estimation procedure that\ncombines a generalization of moderated statistics with targeted\nminimum loss-based estimates (TMLE) of parameters defined via\ncausal inference (e.g., Average Treatment Effect) whose\nestimators admit asymptotically linear representations.","downloads_month":161,"downloads_total":5798,"page":"http://bioconductor.org/packages/release/bioc/html/biotmle.html"},{"authors":"Tengfei Yin [aut], Michael Lawrence [aut, ths, cre], Dianne\nCook [aut, ths] and Johannes Rainer [ctb]","name":"biovizBase","license":"Artistic-2.0","tags":["Infrastructure","Visualization","Preprocessing"],"description":"The biovizBase package is designed to provide a set of\nutilities, color schemes and conventions for genomic data. It\nserves as the base for various high-level packages for\nbiological data visualization. This saves development effort\nand encourages consistency.","downloads_month":6794,"downloads_total":984986,"page":"http://bioconductor.org/packages/release/bioc/html/biovizBase.html"},{"authors":"Andrea Gobbi [aut], Francesco Iorio [aut], Giuseppe Jurman\n[cbt], Davide Albanese [cbt] and Julio Saez-Rodriguez [cbt].","name":"BiRewire","license":"GPL-3","tags":["Network"],"description":"Fast functions for bipartite network rewiring through N\nconsecutive switching steps (See References) and for the\ncomputation of the minimal number of switching steps to be\nperformed in order to maximise the dissimilarity with respect\nto the original network. Includes functions for the analysis of\nthe introduced randomness across the switching steps and\nseveral other routines to analyse the resulting networks and\ntheir natural projections. Extension to undirected networks and\ndirected signed networks is also provided. Starting from\nversion 1.9.7 a more precise bound (especially for small\nnetwork) has been implemented. Starting from version 2.2.0 the\nanalysis routine is more complete and a visual montioring of\nthe underlying Markov Chain has been implemented. Starting from\n3.6.0 the library can handle also matrices with NA (not for the\ndirected signed graphs).","downloads_month":186,"downloads_total":31230,"page":"http://bioconductor.org/packages/release/bioc/html/BiRewire.html"},{"authors":"Benedikt Zacher, Khalid Abnaof, Stephan Gade, Erfan Younesi, Achim Tresch and Holger Froehlich","name":"birta","license":"GPL (>= 2)","tags":["Microarray","Sequencing","GeneExpression","Transcription","GraphAndNetwork"],"description":"Expression levels of mRNA molecules are regulated by\ndifferent processes, comprising inhibition or activation by\ntranscription factors and post-transcriptional degradation by\nmicroRNAs. birta (Bayesian Inference of Regulation of\nTranscriptional Activity) uses the regulatory networks of TFs\nand miRNAs together with mRNA and miRNA expression data to\npredict switches in regulatory activity between two conditions.\nA Bayesian network is used to model the regulatory structure\nand Markov-Chain-Monte-Carlo is applied to sample the activity\nstates.","downloads_month":159,"downloads_total":34834,"page":"http://bioconductor.org/packages/release/bioc/html/birta.html"},{"authors":"Katja Hebestreit and Hans-Ulrich Klein","name":"BiSeq","license":"LGPL-3","tags":["Genetics","Sequencing","MethylSeq","DNAMethylation"],"description":"The BiSeq package provides useful classes and functions to\nhandle and analyze targeted bisulfite sequencing (BS) data such\nas reduced-representation bisulfite sequencing (RRBS) data. In\nparticular, it implements an algorithm to detect differentially\nmethylated regions (DMRs). The package takes already aligned BS\ndata from one or multiple samples.","downloads_month":260,"downloads_total":48980,"page":"http://bioconductor.org/packages/release/bioc/html/BiSeq.html"},{"authors":"Peter Glaus, Antti Honkela and Magnus Rattray","name":"BitSeq","license":"Artistic-2.0 + file LICENSE","tags":["ImmunoOncology","GeneExpression","DifferentialExpression","Sequencing","RNASeq","Bayesian","AlternativeSplicing","DifferentialSplicing","Transcription"],"description":"The BitSeq package is targeted for transcript expression\nanalysis and differential expression analysis of RNA-seq data\nin two stage process. In the first stage it uses Bayesian\ninference methodology to infer expression of individual\ntranscripts from individual RNA-seq experiments. The second\nstage of BitSeq embraces the differential expression analysis\nof transcript expression. Providing expression estimates from\nreplicates of multiple conditions, Log-Normal model of the\nestimates is used for inferring the condition mean transcript\nexpression and ranking the transcripts based on the likelihood\nof differential expression.","downloads_month":342,"downloads_total":69692,"page":"http://bioconductor.org/packages/release/bioc/html/BitSeq.html"},{"authors":"Vojtěch Kulvait","name":"blima","license":"GPL-3","tags":["Microarray","Preprocessing","Normalization","DifferentialExpression","GeneRegulation","GeneExpression"],"description":"Package blima includes several algorithms for the\npreprocessing of Illumina microarray data. It focuses to the\nbead level analysis and provides novel approach to the quantile\nnormalization of the vectors of unequal lengths. It provides\nvariety of the methods for background correction including\nbackground subtraction, RMA like convolution and background\noutlier removal. It also implements variance stabilizing\ntransformation on the bead level. There are also implemented\nmethods for data summarization. It also provides the methods\nfor performing T-tests on the detector (bead) level and on the\nprobe level for differential expression testing.","downloads_month":155,"downloads_total":18036,"page":"http://bioconductor.org/packages/release/bioc/html/blima.html"},{"authors":"Tin Nguyen <tin@wayne.edu> and Sorin Draghici <sorin@wayne.edu>","name":"BLMA","license":"GPL (>=2)","tags":["GeneSetEnrichment","Pathways","DifferentialExpression","Microarray"],"description":"Suit of tools for bi-level meta-analysis. The package can\nbe used in a wide range of applications, including general\nhypothesis testings, differential expression analysis,\nfunctional analysis, and pathway analysis.","downloads_month":171,"downloads_total":5722,"page":"http://bioconductor.org/packages/release/bioc/html/BLMA.html"},{"authors":"Kipper Fletez-Brant [cre, aut] and Kasper Daniel Hansen [aut]","name":"bnbc","license":"Artistic-2.0","tags":["HiC","Preprocessing","Normalization","Software"],"description":"Tools to normalize (several) Hi-C data from replicates.","downloads_month":166,"downloads_total":3732,"page":"http://bioconductor.org/packages/release/bioc/html/bnbc.html"},{"authors":"Chantriolnt-Andreas Kapourani [aut and cre]","name":"BPRMeth","license":"GPL-3","tags":["ImmunoOncology","DNAMethylation","GeneExpression","GeneRegulation","Epigenetics","Genetics","Clustering","FeatureExtraction","Regression","RNASeq","Bayesian","KEGG","Sequencing","Coverage","SingleCell"],"description":"The BPRMeth package is a probabilistic method to quantify\nexplicit features of methylation profiles, in a way that would\nmake it easier to formally use such profiles in downstream\nmodelling efforts, such as predicting gene expression levels or\nclustering genomic regions or cells according to their\nmethylation profiles.","downloads_month":187,"downloads_total":7788,"page":"http://bioconductor.org/packages/release/bioc/html/BPRMeth.html"},{"authors":"Piotr Dittwald and with contributions of Dirk Valkenborg and\nJurgen Claesen","name":"BRAIN","license":"GPL-2","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"Package for calculating aggregated isotopic distribution\nand exact center-masses for chemical substances (in this\nversion composed of C, H, N, O and S). This is an\nimplementation of the BRAIN algorithm described in the paper by\nJ. Claesen, P. Dittwald, T. Burzykowski and D. Valkenborg.","downloads_month":472,"downloads_total":45260,"page":"http://bioconductor.org/packages/release/bioc/html/BRAIN.html"},{"authors":"Sara Linker [aut and cre]","name":"brainImageR","license":"CC BY-SA 4.0","tags":["Software","Transcription","GeneSetEnrichment","GeneExpression","GenePrediction"],"description":"BrainImageR is a package that provides the user with\ninformation of where in the human brain their gene set\ncorresponds to. This is provided both as a continuous variable\nand as a easily-interpretable image. BrainImageR has additional\nfunctionality of identifying approximately when in\ndevelopmental time that a gene expression dataset corresponds\nto. Both the spatial gene set enrichment and the developmental\ntime point prediction are assessed in comparison to the Allen\nBrain Atlas reference data.","downloads_month":110,"downloads_total":922,"page":"http://bioconductor.org/packages/release/bioc/html/brainImageR.html"},{"authors":"Itoshi NIKAIDO <dritoshi@gmail.com>","name":"BrainStars","license":"Artistic-2.0","tags":["Microarray","OneChannel","DataImport"],"description":"This package can search and get gene expression data and\nplots from BrainStars (B*). BrainStars is a quantitative\nexpression database of the adult mouse brain. The database has\ngenome-wide expression profile at 51 adult mouse CNS regions.","downloads_month":164,"downloads_total":34116,"page":"http://bioconductor.org/packages/release/bioc/html/BrainStars.html"},{"authors":"Beth Signal","name":"branchpointer","license":"BSD_3_clause + file LICENSE","tags":["Software","GenomeAnnotation","GenomicVariation","MotifAnnotation"],"description":"Predicts branchpoint probability for sites in intronic\nbranchpoint windows. Queries can be supplied as intronic\nregions; or to evaluate the effects of mutations, SNPs.","downloads_month":149,"downloads_total":5384,"page":"http://bioconductor.org/packages/release/bioc/html/branchpointer.html"},{"authors":"David Porubsky, Ashley Sanders and Aaron Taudt","name":"breakpointR","license":"file LICENSE","tags":["Software","Sequencing","DNASeq","SingleCell","Coverage"],"description":"This package implements functions for finding breakpoints,\nplotting and export of Strand-seq data.","downloads_month":135,"downloads_total":866,"page":"http://bioconductor.org/packages/release/bioc/html/breakpointR.html"},{"authors":"Raphael Gottardo","name":"bridge","license":"GPL (>= 2)","tags":["Microarray","OneChannel","TwoChannel","DifferentialExpression"],"description":"Test for differentially expressed genes with microarray\ndata. This package can be used with both cDNA microarrays or\nAffymetrix chip. The packge fits a robust Bayesian hierarchical\nmodel for testing for differential expression. Outliers are\nmodeled explicitly using a $t$-distribution. The model includes\nan exchangeable prior for the variances which allow different\nvariances for the genes but still shrink extreme empirical\nvariances. Our model can be used for testing for differentially\nexpressed genes among multiple samples, and can distinguish\nbetween the different possible patterns of differential\nexpression when there are three or more samples. Parameter\nestimation is carried out using a novel version of Markov Chain\nMonte Carlo that is appropriate when the model puts mass on\nsubspaces of the full parameter space.","downloads_month":161,"downloads_total":51154,"page":"http://bioconductor.org/packages/release/bioc/html/bridge.html"},{"authors":"Christ Leemans <christleemans@gmail.com>, Egon Willighagen\n<egon.willighagen@gmail.com>, Anwesha Bohler\n<anweshabohler@gmail.com> and Lars Eijssen\n<l.eijssen@maastrichtuniversity.nl>","name":"BridgeDbR","license":"AGPL-3","tags":["Software","Annotation","Metabolomics","Cheminformatics"],"description":"Use BridgeDb functions and load identifier mapping\ndatabases in R.","downloads_month":182,"downloads_total":18706,"page":"http://bioconductor.org/packages/release/bioc/html/BridgeDbR.html"},{"authors":"Paul Shannon","name":"BrowserViz","license":"GPL-2","tags":["Visualization","ThirdPartyClient"],"description":"Interactvive graphics in a web browser from R, using\nwebsockets and JSON.","downloads_month":193,"downloads_total":13418,"page":"http://bioconductor.org/packages/release/bioc/html/BrowserViz.html"},{"authors":"Hervé Pagès","name":"BSgenome","license":"Artistic-2.0","tags":["Genetics","Infrastructure","DataRepresentation","SequenceMatching","Annotation","SNP"],"description":"Infrastructure shared by all the Biostrings-based genome\ndata packages.","downloads_month":12205,"downloads_total":1490852,"page":"http://bioconductor.org/packages/release/bioc/html/BSgenome.html"},{"authors":"Kasper Daniel Hansen [aut, cre] and Peter Hickey [aut]","name":"bsseq","license":"Artistic-2.0","tags":["DNAMethylation"],"description":"A collection of tools for analyzing and visualizing\nbisulfite sequencing data.","downloads_month":1395,"downloads_total":112842,"page":"http://bioconductor.org/packages/release/bioc/html/bsseq.html"},{"authors":"Wei Zhu <zhuw@medimmune.com>, Michael Kuziora\n<kuzioram@medimmune.com>, Todd Creasy <creasyt@medimmune.com> and Brandon Higgs <higgsb@medimmune.com>","name":"BubbleTree","license":"LGPL (>= 3)","tags":["CopyNumberVariation","Software","Sequencing","Coverage"],"description":"CNV analysis in groups of tumor samples.","downloads_month":152,"downloads_total":17194,"page":"http://bioconductor.org/packages/release/bioc/html/BubbleTree.html"},{"authors":"Ben Bolstad <bmb@bmbolstad.com>","name":"BufferedMatrix","license":"LGPL (>= 2)","tags":["Infrastructure"],"description":"A tabular style data object where most data is stored\noutside main memory. A buffer is used to speed up access to\ndata.","downloads_month":167,"downloads_total":50622,"page":"http://bioconductor.org/packages/release/bioc/html/BufferedMatrix.html"},{"authors":"Ben Bolstad <bmb@bmbolstad.com>","name":"BufferedMatrixMethods","license":"GPL (>= 2)","tags":["Infrastructure"],"description":"Microarray analysis methods that use BufferedMatrix\nobjects","downloads_month":168,"downloads_total":48854,"page":"http://bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html"},{"authors":"Alina Selega (alina.selega@gmail.com), Sander Granneman and Guido\nSanguinetti","name":"BUMHMM","license":"GPL-3","tags":["ImmunoOncology","GeneticVariability","Transcription","GeneExpression","GeneRegulation","Coverage","Genetics","StructuralPrediction","Transcriptomics","Bayesian","Classification","FeatureExtraction","HiddenMarkovModel","Regression","RNASeq","Sequencing"],"description":"This is a probabilistic modelling pipeline for computing\nper- nucleotide posterior probabilities of modification from\nthe data collected in structure probing experiments. The model\nsupports multiple experimental replicates and empirically\ncorrects coverage- and sequence-dependent biases. The model\nutilises the measure of a \"drop-off rate\" for each nucleotide,\nwhich is compared between replicates through a log-ratio (LDR).\nThe LDRs between control replicates define a null distribution\nof variability in drop-off rate observed by chance and LDRs\nbetween treatment and control replicates gets compared to this\ndistribution. Resulting empirical p-values (probability of\nbeing \"drawn\" from the null distribution) are used as\nobservations in a Hidden Markov Model with a Beta-Uniform\nMixture model used as an emission model. The resulting\nposterior probabilities indicate the probability of a\nnucleotide of having being modified in a structure probing\nexperiment.","downloads_month":196,"downloads_total":5766,"page":"http://bioconductor.org/packages/release/bioc/html/BUMHMM.html"},{"authors":"Rafael A. Irizarry [cre, aut], Martin Aryee [aut], Kasper\nDaniel Hansen [aut], Hector Corrada Bravo [aut], Shan Andrews\n[ctb], Andrew E. Jaffe [ctb], Harris Jaffee [ctb] and Leonardo\nCollado-Torres [ctb]","name":"bumphunter","license":"Artistic-2.0","tags":["DNAMethylation","Epigenetics","Infrastructure","MultipleComparison","ImmunoOncology"],"description":"Tools for finding bumps in genomic data","downloads_month":3651,"downloads_total":311894,"page":"http://bioconductor.org/packages/release/bioc/html/bumphunter.html"},{"authors":"Yin Jin, Hesen Peng, Lei Wang, Raffaele Fronza and Yuanhua Liu and\nChristine Nardini","name":"BUS","license":"GPL-3","tags":["Preprocessing"],"description":"This package can be used to compute associations among\ngenes (gene-networks) or between genes and some external traits\n(i.e. clinical).","downloads_month":159,"downloads_total":41770,"page":"http://bioconductor.org/packages/release/bioc/html/BUS.html"},{"authors":"Xiangyu Luo <xyluo1991@gmail.com> and Yingying Wei\n<yweicuhk@gmail.com>","name":"BUScorrect","license":"GPL (>= 2)","tags":["GeneExpression","StatisticalMethod","Bayesian","Clustering","FeatureExtraction","BatchEffect"],"description":"High-throughput experimental data are accumulating\nexponentially in public databases. However, mining valid\nscientific discoveries from these abundant resources is\nhampered by technical artifacts and inherent biological\nheterogeneity. The former are usually termed \"batch effects,\"\nand the latter is often modelled by \"subtypes.\" The R package\nBUScorrect fits a Bayesian hierarchical model, the\nBatch-effects-correction-with-Unknown-Subtypes model (BUS), to\ncorrect batch effects in the presence of unknown subtypes. BUS\nis capable of (a) correcting batch effects explicitly, (b)\ngrouping samples that share similar characteristics into\nsubtypes, (c) identifying features that distinguish subtypes,\nand (d) enjoying a linear-order computation complexity.","downloads_month":86,"downloads_total":832,"page":"http://bioconductor.org/packages/release/bioc/html/BUScorrect.html"},{"authors":"Sander Bollen","name":"CAFE","license":"GPL-3","tags":["GeneExpression","Microarray","OneChannel","GeneSetEnrichment"],"description":"Detection and visualizations of gross chromosomal\naberrations using Affymetrix expression microarrays as input","downloads_month":167,"downloads_total":21134,"page":"http://bioconductor.org/packages/release/bioc/html/CAFE.html"},{"authors":"Malte Thodberg","name":"CAGEfightR","license":"GPL-3 + file LICENSE","tags":["Software","Transcription","Coverage","GeneExpression","GeneRegulation","PeakDetection","DataImport","DataRepresentation","Transcriptomics","Sequencing","Annotation","GenomeBrowsers","Normalization","Preprocessing","Visualization"],"description":"CAGE is a widely used high throughput assay for measuring\ntranscription start site (TSS) activity. CAGEfightR is an\nR/Bioconductor package for performing a wide range of common\ndata analysis tasks for CAGE and 5'-end data in general. Core\nfunctionality includes: import of CAGE TSSs (CTSSs), tag (or\nunidirectional) clustering for TSS identification,\nbidirectional clustering for enhancer identification,\nannotation with transcript and gene models, correlation of TSS\nand enhancer expression, calculation of TSS shapes,\nquantification of CAGE expression as expression matrices and\ngenome brower visualization.","downloads_month":148,"downloads_total":2368,"page":"http://bioconductor.org/packages/release/bioc/html/CAGEfightR.html"},{"authors":"Vanja Haberle <vanja.haberle@gmail.com>","name":"CAGEr","license":"GPL-3","tags":["Preprocessing","Sequencing","Normalization","FunctionalGenomics","Transcription","GeneExpression","Clustering","Visualization"],"description":"Preprocessing of CAGE sequencing data, identification and\nnormalization of transcription start sites and downstream\nanalysis of transcription start sites clusters (promoters).","downloads_month":276,"downloads_total":38600,"page":"http://bioconductor.org/packages/release/bioc/html/CAGEr.html"},{"authors":"Hui Zhao, Kristof Engelen, Bart De Moor and Kathleen Marchal","name":"CALIB","license":"LGPL","tags":["Microarray","TwoChannel","Preprocessing"],"description":"This package contains functions for normalizing spotted\nmicroarray data, based on a physically motivated calibration\nmodel. The model parameters and error distributions are\nestimated from external control spikes.","downloads_month":169,"downloads_total":48892,"page":"http://bioconductor.org/packages/release/bioc/html/CALIB.html"},{"authors":"Carsten Kuhl, Ralf Tautenhahn, Hendrik Treutler, Steffen\nNeumann {ckuhl|htreutle|sneumann}@ipb-halle.de and rtautenh@scripps.edu","name":"CAMERA","license":"GPL (>= 2)","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"Annotation of peaklists generated by xcms, rule based\nannotation of isotopes and adducts, isotope validation, EIC\ncorrelation based tagging of unknown adducts and fragments","downloads_month":1083,"downloads_total":119578,"page":"http://bioconductor.org/packages/release/bioc/html/CAMERA.html"},{"authors":"Lulu Chen <luluchen@vt.edu>","name":"CAMTHC","license":"GPL-2","tags":["Software","CellBiology","GeneExpression"],"description":"An R package for tissue heterogeneity characterization by\nconvex analysis of mixtures (CAM). It provides basic functions\nto perform unsupervised deconvolution on mixture expression\nprofiles by CAM and some auxiliary functions to help understand\nthe subpopulation-specific results. It also implements\nfunctions to perform supervised deconvolution based on prior\nknowledge of molecular markers, S matrix or A matrix. Combining\nmolecular markers from CAM and from prior knowledge can achieve\nsemi-supervised deconvolution of mixtures.","downloads_month":74,"downloads_total":986,"page":"http://bioconductor.org/packages/release/bioc/html/CAMTHC.html"},{"authors":"Karim Mezhoud. Nuclear Safety & Security Department. Nuclear\nScience Center of Tunisia.","name":"canceR","license":"GPL-2","tags":["GUI","GeneExpression","Software"],"description":"The package is user friendly interface based on the cgdsr\nand other modeling packages to explore, compare, and analyse\nall available Cancer Data (Clinical data, Gene Mutation, Gene\nMethylation, Gene Expression, Protein Phosphorylation, Copy\nNumber Alteration) hosted by the Computational Biology Center\nat Memorial-Sloan-Kettering Cancer Center (MSKCC).","downloads_month":195,"downloads_total":16670,"page":"http://bioconductor.org/packages/release/bioc/html/canceR.html"},{"authors":"Jan Budczies and Daniel Kosztyla","name":"cancerclass","license":"GPL 3","tags":["Cancer","Microarray","Classification","Visualization"],"description":"The classification protocol starts with a feature\nselection step and continues with nearest-centroid\nclassification. The accurarcy of the predictor can be evaluated\nusing training and test set validation, leave-one-out\ncross-validation or in a multiple random validation protocol.\nMethods for calculation and visualization of continuous\nprediction scores allow to balance sensitivity and specificity\nand define a cutoff value according to clinical requirements.","downloads_month":162,"downloads_total":30254,"page":"http://bioconductor.org/packages/release/bioc/html/cancerclass.html"},{"authors":"Thomas D. Sherman, Raymond Cheng and Elana J. Fertig","name":"CancerInSilico","license":"GPL-2","tags":["ImmunoOncology","MathematicalBiology","SystemsBiology","CellBiology","BiomedicalInformatics","GeneExpression","RNASeq","SingleCell"],"description":"The CancerInSilico package provides an R interface for\nrunning mathematical models of tumor progresson and generating\ngene expression data from the results. This package has the\nunderlying models implemented in C++ and the output and\nanalysis features implemented in R.","downloads_month":160,"downloads_total":7776,"page":"http://bioconductor.org/packages/release/bioc/html/CancerInSilico.html"},{"authors":"Giovanni Parmigiani and Simina M. Boca","name":"CancerMutationAnalysis","license":"GPL (>= 2) + file LICENSE","tags":["Genetics","Software"],"description":"This package implements gene and gene-set level analysis\nmethods for somatic mutation studies of cancer.  The gene-level\nmethods distinguish between driver genes (which play an active\nrole in tumorigenesis) and passenger genes (which are mutated\nin tumor samples, but have no role in tumorigenesis) and\nincorporate a two-stage study design.  The gene-set methods\nimplement a patient-oriented approach, which calculates\ngene-set scores for each sample, then combines them across\nsamples; a gene-oriented approach which uses the Wilcoxon test\nis also provided for comparison.","downloads_month":170,"downloads_total":34064,"page":"http://bioconductor.org/packages/release/bioc/html/CancerMutationAnalysis.html"},{"authors":"Taosheng Xu<taosheng.x@gmail.com> and Thuc\nLe<Thuc.Le@unisa.edu.au>","name":"CancerSubtypes","license":"GPL (>= 2)","tags":["Clustering","Software","Visualization","GeneExpression"],"description":"CancerSubtypes integrates the current common computational\nbiology methods for cancer subtypes identification and provides\na standardized framework for cancer subtype analysis based\nmulti-omics data, such as gene expression, miRNA expression,\nDNA methylation and others.","downloads_month":219,"downloads_total":11202,"page":"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html"},{"authors":"Caitlin McHugh and Timothy Thornton","name":"CAnD","license":"Artistic-2.0","tags":["Genetics","StatisticalMethod","GeneticVariability","SNP"],"description":"Functions to perform the CAnD test on a set of ancestry\nproportions. For a particular ancestral subpopulation, a user\nwill supply the estimated ancestry proportion for each sample,\nand each chromosome or chromosomal segment of interest. A\np-value for each chromosome as well as an overall CAnD p-value\nwill be returned for each test. Plotting functions are also\navailable.","downloads_month":151,"downloads_total":13262,"page":"http://bioconductor.org/packages/release/bioc/html/CAnD.html"},{"authors":"Henry Zhang","name":"caOmicsV","license":"GPL (>=2.0)","tags":["ImmunoOncology","Visualization","Network","RNASeq"],"description":"caOmicsV package provides methods to visualize\nmulti-dimentional cancer genomics data including of patient\ninformation, gene expressions, DNA methylations, DNA copy\nnumber variations, and SNP/mutations in matrix layout or\nnetwork layout.","downloads_month":150,"downloads_total":10856,"page":"http://bioconductor.org/packages/release/bioc/html/caOmicsV.html"},{"authors":"Kylie A. Bemis <k.bemis@northeastern.edu>","name":"Cardinal","license":"Artistic-2.0","tags":["ImmunoOncology","Software","Infrastructure","Proteomics","Lipidomics","Normalization","MassSpectrometry","ImagingMassSpectrometry Clustering","Classification"],"description":"Implements statistical & computational tools for analyzing\nmass spectrometry imaging datasets, including methods for\nefficient pre-processing, spatial segmentation, and\nclassification.","downloads_month":314,"downloads_total":20226,"page":"http://bioconductor.org/packages/release/bioc/html/Cardinal.html"},{"authors":"David Rossell, Camille Stephan-Otto, Manuel Kroiss, Miranda\nStobbe and Victor Pena","name":"casper","license":"GPL (>=2)","tags":["ImmunoOncology","GeneExpression","DifferentialExpression","Transcription","RNASeq","Sequencing"],"description":"Infer alternative splicing from paired-end RNA-seq data.\nThe model is based on counting paths across exons, rather than\npairwise exon connections, and estimates the fragment size and\nstart distributions non-parametrically, which improves\nestimation precision.","downloads_month":210,"downloads_total":29678,"page":"http://bioconductor.org/packages/release/bioc/html/casper.html"},{"authors":"Helena L. Crowell [aut, cre], Vito R.T. Zanotelli [aut], Stéphane Chevrier [aut, dtc], Mark D. Robinson [aut, fnd] and Bernd Bodenmiller [fnd]","name":"CATALYST","license":"GPL (>=2)","tags":["DifferentialExpression","FlowCytometry","ImmunoOncology","MassSpectrometry","Normalization","Preprocessing","SingleCell","Software","StatisticalMethod","Visualization"],"description":"Mass cytometry (CyTOF) uses heavy metal isotopes rather\nthan fluorescent tags as reporters to label antibodies, thereby\nsubstantially decreasing spectral overlap and allowing for\nexamination of over 50 parameters at the single cell level.\nWhile spectral overlap is significantly less pronounced in\nCyTOF than flow cytometry, spillover due to detection\nsensitivity, isotopic impurities, and oxide formation can\nimpede data interpretability. We designed CATALYST (Cytometry\ndATa anALYSis Tools) to provide a pipeline for preprocessing of\ncytometry data, including i) normalization using bead\nstandards, ii) single-cell deconvolution, and iii) bead-based\ncompensation.","downloads_month":381,"downloads_total":8754,"page":"http://bioconductor.org/packages/release/bioc/html/CATALYST.html"},{"authors":"Robert Gentleman [aut], Seth Falcon [ctb], Deepayan Sarkar\n[ctb], Robert Castelo [ctb] and Bioconductor Package Maintainer\n[cre]","name":"Category","license":"Artistic-2.0","tags":["Annotation","GO","Pathways","GeneSetEnrichment"],"description":"A collection of tools for performing category (gene set\nenrichment) analysis.","downloads_month":3824,"downloads_total":551592,"page":"http://bioconductor.org/packages/release/bioc/html/Category.html"},{"authors":"Robert M. Flight <rflight79@gmail.com>","name":"categoryCompare","license":"GPL-2","tags":["Annotation","GO","MultipleComparison","Pathways","GeneExpression"],"description":"Calculates significant annotations (categories) in each of\ntwo (or more) feature (i.e. gene) lists, determines the overlap\nbetween the annotations, and returns graphical and tabular data\nabout the significant annotations and which combinations of\nfeature lists the annotations were found to be significant.\nInteractive exploration is facilitated through the use of\nRCytoscape (heavily suggested).","downloads_month":158,"downloads_total":33078,"page":"http://bioconductor.org/packages/release/bioc/html/categoryCompare.html"},{"authors":"Glyn Bradley, Steven Barrett, Chirag Mistry, Mark Pipe, David\nWille, David Riley, Bhushan Bonde and Peter Woollard","name":"CausalR","license":"GPL (>= 2)","tags":["ImmunoOncology","SystemsBiology","Network","GraphAndNetwork","Network Inference","Transcriptomics","Proteomics","DifferentialExpression","RNASeq","Microarray"],"description":"Causal network analysis methods for regulator prediction\nand network reconstruction from genome scale data.","downloads_month":197,"downloads_total":12370,"page":"http://bioconductor.org/packages/release/bioc/html/CausalR.html"},{"authors":"Arman Shahrisa [aut, cre, cph] and Maryam Tahmasebi Birgani [aut]","name":"cbaf","license":"Artistic-2.0","tags":["Technology"],"description":"This package contains functions that allow analysing and\ncomparing various gene groups from different cancers/cancer\nsubgroups easily. So far, it is compatible with RNA-seq,\nmicroRNA-seq, microarray and methylation datasets that are\nstored on cbioportal.org.","downloads_month":125,"downloads_total":3628,"page":"http://bioconductor.org/packages/release/bioc/html/cbaf.html"},{"authors":"Jun Woo [aut, cre] and Jinhua Wang [aut]","name":"ccfindR","license":"GPL (>= 2)","tags":["Transcriptomics","SingleCell","Bayesian","Clustering"],"description":"A collection of tools for cancer genomic data clustering\nanalyses, including those for single cell RNA-seq. Cell\nclustering and feature gene selection analysis employ Bayesian\n(and maximum likelihood) non-negative matrix factorization\n(NMF) algorithm. Input data set consists of RNA count matrix,\ngene, and cell bar code annotations.  Analysis outputs are\nfactor matrices for multiple ranks and marginal likelihood\nvalues for each rank. The package includes utilities for\ndownstream analyses, including meta-gene identification,\nvisualization, and construction of rank-based trees for\nclusters.","downloads_month":208,"downloads_total":2690,"page":"http://bioconductor.org/packages/release/bioc/html/ccfindR.html"},{"authors":"Alex Pickering","name":"ccmap","license":"MIT + file LICENSE","tags":["GeneExpression","Transcription","Microarray","DifferentialExpression"],"description":"Finds drugs and drug combinations that are predicted to\nreverse or mimic gene expression signatures. These drugs might\nreverse diseases or mimic healthy lifestyles.","downloads_month":175,"downloads_total":8358,"page":"http://bioconductor.org/packages/release/bioc/html/ccmap.html"},{"authors":"Xueyuan Cao <xueyuan.cao@stjude.org> and Stanley.pounds\n<stanley.pounds@stjude.org>","name":"CCPROMISE","license":"GPL (>= 2)","tags":["Microarray","GeneExpression"],"description":"Perform Canonical correlation between two forms of high\ndemensional genetic data, and associate the first compoent of\neach form of data with a specific biologically interesting\npattern of associations with multiple endpoints. A probe level\nanalysis is also implemented.","downloads_month":159,"downloads_total":6858,"page":"http://bioconductor.org/packages/release/bioc/html/CCPROMISE.html"},{"authors":"Emma Schwager <emh146@mail.harvard.edu>, Craig\nBielski<craig.bielski@gmail.com> and George\nWeingart<george.weingart@gmail.com>","name":"ccrepe","license":"MIT + file LICENSE","tags":["ImmunoOncology","Statistics","Metagenomics","Bioinformatics","Software"],"description":"The CCREPE (Compositionality Corrected by REnormalizaion\nand PErmutation) package is designed to assess the significance\nof general similarity measures in compositional datasets.  In\nmicrobial abundance data, for example, the total abundances of\nall microbes sum to one; CCREPE is designed to take this\nconstraint into account when assigning p-values to similarity\nmeasures between the microbes.  The package has two functions:\nccrepe: Calculates similarity measures, p-values and q-values\nfor relative abundances of bugs in one or two body sites using\nbootstrap and permutation matrices of the data. nc.score:\nCalculates species-level co-variation and co-exclusion patterns\nbased on an extension of the checkerboard score to ordinal\ndata.","downloads_month":185,"downloads_total":22038,"page":"http://bioconductor.org/packages/release/bioc/html/ccrepe.html"},{"authors":"Sarah Williams [aut and cre]","name":"celaref","license":"GPL-3","tags":["ImmunoOncology","SingleCell"],"description":"After the clustering step of a single-cell RNAseq\nexperiment, this package aims to suggest labels/cell types for\nthe clusters, on the basis of similarity to a reference\ndataset. It requires a table of read counts per cell per gene,\nand a list of the cells belonging to each of the clusters, (for\nboth test and reference data).","downloads_month":144,"downloads_total":1132,"page":"http://bioconductor.org/packages/release/bioc/html/celaref.html"},{"authors":"Mohammed OE Abdallah","name":"cellbaseR","license":"Apache License (== 2.0)","tags":["Annotation","VariantAnnotation"],"description":"This R package makes use of the exhaustive RESTful Web\nservice API that has been implemented for the Cellabase\ndatabase. It enable researchers to query and obtain a wealth of\nbiological information from a single database saving a lot of\ntime. Another benefit is that researchers can easily make\nqueries about different biological topics and link all this\ninformation together as all information is integrated.","downloads_month":158,"downloads_total":5306,"page":"http://bioconductor.org/packages/release/bioc/html/cellbaseR.html"},{"authors":"Julien Gagneur <gagneur@genzentrum.lmu.de> and Andreas Neudecker\n<a.neudecker@arcor.de>","name":"cellGrowth","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","MicrotitrePlateAssay","DataImport","Visualization","TimeCourse"],"description":"This package provides functionalities for the fitting of\ncell population growth models on experimental OD curves.","downloads_month":164,"downloads_total":32532,"page":"http://bioconductor.org/packages/release/bioc/html/cellGrowth.html"},{"authors":"Ligia Bras, Wolfgang Huber <whuber@embl.de>, Michael Boutros\n<m.boutros@dkfz.de>, Gregoire Pau <gpau@embl.de> and Florian Hahne\n<florian.hahne@novartis.com>","name":"cellHTS2","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","Preprocessing","Visualization"],"description":"This package provides tools for the analysis of\nhigh-throughput assays that were performed in microtitre plate\nformats (including but not limited to 384-well plates). The\nfunctionality includes data import and management,\nnormalisation, quality assessment, replicate summarisation and\nstatistical scoring. A webpage that provides a detailed\ngraphical overview over the data and analysis results is\nproduced. In our work, we have applied the package to RNAi\nscreens on fly and human cells, and for screens of yeast\nlibraries. See ?cellHTS2 for a brief introduction.","downloads_month":466,"downloads_total":85428,"page":"http://bioconductor.org/packages/release/bioc/html/cellHTS2.html"},{"authors":"Tomislav Illicic and Davis McCarthy","name":"cellity","license":"GPL (>= 2)","tags":["ImmunoOncology","RNASeq","QualityControl","Preprocessing","Normalization","Visualization","DimensionReduction","Transcriptomics","GeneExpression","Sequencing","Software","SupportVectorMachine"],"description":"A support vector machine approach to identifying and\nfiltering low quality cells from single-cell RNA-seq datasets.","downloads_month":248,"downloads_total":11446,"page":"http://bioconductor.org/packages/release/bioc/html/cellity.html"},{"authors":"Brad Nelms","name":"CellMapper","license":"Artistic-2.0","tags":["Microarray","Software","GeneExpression"],"description":"Infers cell type-specific expression based on\nco-expression similarity with known cell type marker genes. Can\nmake accurate predictions using publicly available expression\ndata, even when a cell type has not been isolated before.","downloads_month":162,"downloads_total":7396,"page":"http://bioconductor.org/packages/release/bioc/html/CellMapper.html"},{"authors":"T.Cokelaer, F.Eduati, A.MacNamara, S.Schrier and C.Terfve","name":"CellNOptR","license":"GPL-3","tags":["CellBasedAssays","CellBiology","Proteomics","Bioinformatics","TimeCourse"],"description":"This package does optimisation of boolean logic networks\nof signalling pathways based on a previous knowledge network\nand a set of data upon perturbation of the nodes in the\nnetwork.","downloads_month":339,"downloads_total":48104,"page":"http://bioconductor.org/packages/release/bioc/html/CellNOptR.html"},{"authors":"Maia Smith [aut and cre]","name":"cellscape","license":"GPL-3","tags":["Visualization"],"description":"CellScape facilitates interactive browsing of single cell\nclonal evolution datasets. The tool requires two main inputs:\n(i) the genomic content of each single cell in the form of\neither copy number segments or targeted mutation values, and\n(ii) a single cell phylogeny. Phylogenetic formats can vary\nfrom dendrogram-like phylogenies with leaf nodes to\nevolutionary model-derived phylogenies with observed or latent\ninternal nodes. The CellScape phylogeny is flexibly input as a\ntable of source-target edges to support arbitrary\nrepresentations, where each node may or may not have associated\ngenomic data. The output of CellScape is an interactive\ninterface displaying a single cell phylogeny and a\ncell-by-locus genomic heatmap representing the mutation status\nin each cell for each locus.","downloads_month":364,"downloads_total":14740,"page":"http://bioconductor.org/packages/release/bioc/html/cellscape.html"},{"authors":"Nancy Mah and Katerina Taskova","name":"CellScore","license":"GPL-3","tags":["GeneExpression","Transcription","Microarray","MultipleComparison","ReportWriting","DataImport","Visualization"],"description":"The CellScore package contains functions to evaluate the\ncell identity of a test sample, given a cell transition defined\nwith a starting (donor) cell type and a desired target cell\ntype. The evaluation is based upon a scoring system, which uses\na set of standard samples of known cell types, as the reference\nset. The functions have been carried out on a large set of\nmicroarray data from one platform (Affymetrix Human Genome U133\nPlus 2.0). In principle, the method could be applied to any\nexpression dataset, provided that there are a sufficient number\nof standard samples and that the data are normalized.","downloads_month":136,"downloads_total":2010,"page":"http://bioconductor.org/packages/release/bioc/html/CellScore.html"},{"authors":"Daniel Ellwanger [aut, cre and cph]","name":"CellTrails","license":"Artistic-2.0","tags":["ImmunoOncology","Clustering","DataRepresentation","DifferentialExpression","DimensionReduction","GeneExpression","Sequencing","SingleCell","Software","TimeCourse"],"description":"CellTrails is an unsupervised algorithm for the de novo\nchronological ordering, visualization and analysis of\nsingle-cell expression data. CellTrails makes use of a\ngeometrically motivated concept of lower-dimensional manifold\nlearning, which exhibits a multitude of virtues that counteract\nintrinsic noise of single cell data caused by drop-outs,\ntechnical variance, and redundancy of predictive variables.\nCellTrails enables the reconstruction of branching trajectories\nand provides an intuitive graphical representation of\nexpression patterns along all branches simultaneously. It\nallows the user to define and infer the expression dynamics of\nindividual and multiple pathways towards distinct phenotypes.","downloads_month":139,"downloads_total":1248,"page":"http://bioconductor.org/packages/release/bioc/html/CellTrails.html"},{"authors":"David duVerle [aut, cre] and Koji Tsuda [aut]","name":"cellTree","license":"Artistic-2.0","tags":["ImmunoOncology","Sequencing","RNASeq","Clustering","GraphAndNetwork","Visualization","GeneExpression","GeneSetEnrichment","BiomedicalInformatics","CellBiology","FunctionalGenomics","SystemsBiology","GO","TimeCourse","Microarray"],"description":"This packages computes a Latent Dirichlet Allocation (LDA)\nmodel of single-cell RNA-seq data and builds a compact tree\nmodelling the relationship between individual cells over time\nor space.","downloads_month":270,"downloads_total":12156,"page":"http://bioconductor.org/packages/release/bioc/html/cellTree.html"},{"authors":"Pedro Russo [aut], Gustavo Ferreira [aut], Matheus Bürger\n[aut], Lucas Cardozo [aut], Diogenes Lima [aut], Thiago Hirata\n[aut], Melissa Lever [aut], Helder Nakaya [aut and cre]","name":"CEMiTool","license":"GPL-3","tags":["GeneExpression","Transcriptomics","GraphAndNetwork","mRNAMicroarray","RNASeq","Network","NetworkEnrichment","Pathways","ImmunoOncology"],"description":"The CEMiTool package unifies the discovery and the\nanalysis of coexpression gene modules in a fully automatic\nmanner, while providing a user-friendly html report with high\nquality graphs. Our tool evaluates if modules contain genes\nthat are over-represented by specific pathways or that are\naltered in a specific sample group. Additionally, CEMiTool is\nable to integrate transcriptomic data with interactome\ninformation, identifying the potential hubs on each network.","downloads_month":586,"downloads_total":9994,"page":"http://bioconductor.org/packages/release/bioc/html/CEMiTool.html"},{"authors":"Pedro Madrigal <bioinformatics.engineer@gmail.com>","name":"CexoR","license":"Artistic-2.0 | GPL-2 + file LICENSE","tags":["Transcription","Genetics","Sequencing"],"description":"Strand specific peak-pair calling in ChIP-exo replicates.\nThe cumulative Skellam distribution function (package\n'skellam') is used to detect significant normalised count\ndifferences of opposed sign at each DNA strand (peak-pairs).\nIrreproducible discovery rate for overlapping peak-pairs across\nbiological replicates is estimated using the package 'idr'.","downloads_month":153,"downloads_total":24562,"page":"http://bioconductor.org/packages/release/bioc/html/CexoR.html"},{"authors":"Herbert Braselmann","name":"CFAssay","license":"LGPL","tags":["CellBasedAssays","CellBiology","ImmunoOncology","Regression","Survival"],"description":"The package provides functions for calculation of\nlinear-quadratic cell survival curves and for ANOVA of\nexperimental 2-way designs along with the colony formation\nassay.","downloads_month":206,"downloads_total":16732,"page":"http://bioconductor.org/packages/release/bioc/html/CFAssay.html"},{"authors":"Samsiddhi Bhattacharjee, Nilanjan Chatterjee, Summer Han, Minsun Song and William Wheeler","name":"CGEN","license":"GPL-2 + file LICENSE","tags":["SNP","MultipleComparisons","Clustering"],"description":"An R package for analysis of case-control studies in\ngenetic epidemiology.","downloads_month":180,"downloads_total":60558,"page":"http://bioconductor.org/packages/release/bioc/html/CGEN.html"},{"authors":"Sjoerd Vosse and Mark van de Wiel","name":"CGHbase","license":"GPL","tags":["Infrastructure","Microarray","CopyNumberVariation"],"description":"Contains functions and classes that are needed by arrayCGH\npackages.","downloads_month":421,"downloads_total":80778,"page":"http://bioconductor.org/packages/release/bioc/html/CGHbase.html"},{"authors":"Mark van de Wiel and Sjoerd Vosse","name":"CGHcall","license":"GPL (http://www.gnu.org/copyleft/gpl.html)","tags":["Microarray","Preprocessing","Visualization"],"description":"Calls aberrations for array CGH data using a six state\nmixture model as well as several biological concepts that are\nignored by existing algorithms. Visualization of profiles is\nalso provided.","downloads_month":392,"downloads_total":74762,"page":"http://bioconductor.org/packages/release/bioc/html/CGHcall.html"},{"authors":"J. Zhang and B. Feng","name":"cghMCR","license":"LGPL","tags":["Microarray","CopyNumberVariation"],"description":"This package provides functions to identify genomic\nregions of interests based on segmented copy number data from\nmultiple samples.","downloads_month":184,"downloads_total":84022,"page":"http://bioconductor.org/packages/release/bioc/html/cghMCR.html"},{"authors":"Bart P.P. van Houte, Thomas W. Binsl and Hannes Hettling","name":"CGHnormaliter","license":"GPL (>= 3)","tags":["Microarray","Preprocessing"],"description":"Normalization and centralization of array comparative\ngenomic hybridization (aCGH) data. The algorithm uses an\niterative procedure that effectively eliminates the influence\nof imbalanced copy numbers. This leads to a more reliable\nassessment of copy number alterations (CNAs).","downloads_month":153,"downloads_total":41938,"page":"http://bioconductor.org/packages/release/bioc/html/CGHnormaliter.html"},{"authors":"Sjoerd Vosse & Mark van de Wiel","name":"CGHregions","license":"GPL (http://www.gnu.org/copyleft/gpl.html)","tags":["Microarray","CopyNumberVariation","Visualization"],"description":"Dimension Reduction for Array CGH Data with Minimal\nInformation Loss","downloads_month":173,"downloads_total":52416,"page":"http://bioconductor.org/packages/release/bioc/html/CGHregions.html"},{"authors":"Yuan Tian [cre, aut], Tiffany Morris [ctb], Lee Stirling [ctb], Andrew Feber [ctb], Andrew Teschendorff [ctb] and Ankur\nChakravarthy [ctb]","name":"ChAMP","license":"GPL-3","tags":["Microarray","MethylationArray","Normalization","TwoChannel","CopyNumber","DNAMethylation"],"description":"The package includes quality control metrics, a selection\nof normalization methods and novel methods to identify\ndifferentially methylated regions and to highlight copy number\nalterations.","downloads_month":1008,"downloads_total":91720,"page":"http://bioconductor.org/packages/release/bioc/html/ChAMP.html"},{"authors":"Benjamin Mayne","name":"CHARGE","license":"GPL-2","tags":["GeneExpression","Clustering"],"description":"Identifies genomic duplications or deletions from gene\nexpression data.","downloads_month":120,"downloads_total":2048,"page":"http://bioconductor.org/packages/release/bioc/html/CHARGE.html"},{"authors":"Martin Aryee, Peter Murakami, Harris Jaffee and Rafael Irizarry","name":"charm","license":"LGPL (>= 2)","tags":["Microarray","DNAMethylation"],"description":"This package implements analysis tools for DNA methylation\ndata generated using Nimblegen microarrays and the McrBC\nprotocol. It finds differentially methylated regions between\nsamples, calculates percentage methylation estimates and\nincludes array quality assessment tools.","downloads_month":186,"downloads_total":50784,"page":"http://bioconductor.org/packages/release/bioc/html/charm.html"},{"authors":"Kevin Horan and Thomas Girke","name":"ChemmineOB","license":"file LICENSE","tags":["Cheminformatics","BiomedicalInformatics","Pharmacogenetics","Pharmacogenomics","MicrotitrePlateAssay","CellBasedAssays","Visualization","Infrastructure","DataImport","Clustering","Proteomics","Metabolomics"],"description":"ChemmineOB provides an R interface to a subset of\ncheminformatics functionalities implemented by the OpelBabel\nC++ project. OpenBabel is an open source cheminformatics\ntoolbox that includes utilities for structure format\ninterconversions, descriptor calculations, compound similarity\nsearching and more. ChemineOB aims to make a subset of these\nutilities available from within R. For non-developers,\nChemineOB is primarily intended to be used from ChemmineR as an\nadd-on package rather than used directly.","downloads_month":6371,"downloads_total":86138,"page":"http://bioconductor.org/packages/release/bioc/html/ChemmineOB.html"},{"authors":"Y. Eddie Cao, Kevin Horan, Tyler Backman and Thomas Girke","name":"ChemmineR","license":"Artistic-2.0","tags":["Cheminformatics","BiomedicalInformatics","Pharmacogenetics","Pharmacogenomics","MicrotitrePlateAssay","CellBasedAssays","Visualization","Infrastructure","DataImport","Clustering","Proteomics","Metabolomics"],"description":"ChemmineR is a cheminformatics package for analyzing\ndrug-like small molecule data in R. Its latest version contains\nfunctions for efficient processing of large numbers of\nmolecules, physicochemical/structural property predictions,\nstructural similarity searching, classification and clustering\nof compound libraries with a wide spectrum of algorithms. In\naddition, it offers visualization functions for compound\nclustering results and chemical structures.","downloads_month":12662,"downloads_total":213914,"page":"http://bioconductor.org/packages/release/bioc/html/ChemmineR.html"},{"authors":"Carmen Maria Livi","name":"ChIC","license":"GPL-2","tags":["ChIPSeq","QualityControl"],"description":"Quality control (QC) pipeline for ChIP-seq data using a\ncomprehensive set of QC metrics, including previously proposed\nmetrics as well as novel ones, based on local characteristics\nof the enrichment profile. The package provides functions to\ncalculate a set of QC metrics, a compendium with reference\nvalues and machine learning models to score sample quality.","downloads_month":128,"downloads_total":1992,"page":"http://bioconductor.org/packages/release/bioc/html/ChIC.html"},{"authors":"Jonathan Cairns, Paula Freire Pritchett, Steven Wingett and Mikhail Spivakov","name":"Chicago","license":"Artistic-2.0","tags":["ImmunoOncology","Epigenetics","HiC","Sequencing","Software"],"description":"A pipeline for analysing Capture Hi-C data.","downloads_month":246,"downloads_total":11948,"page":"http://bioconductor.org/packages/release/bioc/html/Chicago.html"},{"authors":"Raffaele A Calogero, Matteo Carrara, Marco Beccuti and Francesca\nCordero","name":"chimera","license":"Artistic-2.0","tags":["Infrastructure"],"description":"This package facilitates the characterisation of fusion\nproducts events. It allows to import fusion data results from\nthe following fusion finders: chimeraScan, bellerophontes,\ndeFuse, FusionFinder, FusionHunter, mapSplice, tophat-fusion,\nFusionMap, STAR, Rsubread, fusionCatcher.","downloads_month":171,"downloads_total":45862,"page":"http://bioconductor.org/packages/release/bioc/html/chimera.html"},{"authors":"Stian Lågstad [aut, cre], Sen Zhao [ctb], Andreas M. Hoff\n[ctb], Bjarne Johannessen [ctb], Ole Christian Lingjærde [ctb] and Rolf Skotheim [ctb]","name":"chimeraviz","license":"Artistic-2.0","tags":["Infrastructure","Alignment"],"description":"chimeraviz manages data from fusion gene finders and\nprovides useful visualization tools.","downloads_month":254,"downloads_total":8972,"page":"http://bioconductor.org/packages/release/bioc/html/chimeraviz.html"},{"authors":"Patrick C.N.Martin & Nicolae Radu Zabet","name":"ChIPanalyser","license":"GPL-3","tags":["Software","BiologicalQuestion","WorkflowStep","Transcription","Sequencing","ChipOnChip","Coverage","Alignment","ChIPSeq","SequenceMatching","DataImport","PeakDetection"],"description":"Based on a statistical thermodynamic framework,\nChIPanalyser tries to produce ChIP-seq like profile. The model\nrelies on four consideration: TF binding sites can be scored\nusing a Position weight Matrix, DNA accessibility plays a role\nin Transcription Factor binding, binding profiles are dependant\non the number of transcription factors bound to DNA and finally\nbinding energy (another way of describing PWM's) or binding\nspecificity should be modulated (hence the introduction of a\nbinding specificity modulator). The end result of ChIPanalyser\nis to produce profiles simulating real ChIP-seq profile and\nprovide accuracy measurements of these predicted profiles after\nbeing compared to real ChIP-seq data. The ultimate goal is to\nproduce ChIP-seq like profiles predicting ChIP-seq like profile\nto circumvent the need to produce costly ChIP-seq experiments.","downloads_month":139,"downloads_total":3476,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPanalyser.html"},{"authors":"Hao Wu, Li Chen, Zhaohui S.Qin and Chi Wang","name":"ChIPComp","license":"GPL","tags":["ChIPSeq","Sequencing","Transcription","Genetics","Coverage","MultipleComparison","DataImport"],"description":"ChIPComp detects differentially bound sharp binding sites\nacross multiple conditions considering matching control.","downloads_month":145,"downloads_total":13024,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPComp.html"},{"authors":"Ryan P. Welch [aut, cph], Chee Lee [aut], Raymond G. Cavalcante\n[aut, cre], Chris Lee [aut], Laura J. Scott [ths] and Maureen A.\nSartor [ths]","name":"chipenrich","license":"GPL-3","tags":["ImmunoOncology","ChIPSeq","Epigenetics","FunctionalGenomics","GeneSetEnrichment","HistoneModification","Regression"],"description":"ChIP-Enrich performs gene set enrichment testing using\npeaks called from a ChIP-seq experiment. The method empirically\ncorrects for confounding factors such as the length of genes,\nand the mappability of the sequence surrounding genes.","downloads_month":251,"downloads_total":28846,"page":"http://bioconductor.org/packages/release/bioc/html/chipenrich.html"},{"authors":"Rene Welch, Dongjun Chung and Sunduz Keles","name":"ChIPexoQual","license":"GPL (>=2)","tags":["ChIPSeq","Sequencing","Transcription","Visualization","QualityControl","Coverage","Alignment"],"description":"Package with a quality control pipeline for ChIP-exo/nexus\ndata.","downloads_month":147,"downloads_total":5636,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPexoQual.html"},{"authors":"Lihua Julie Zhu, Jianhong Ou, Jun Yu, Hervé Pagès, Claude\nGazin, Nathan Lawson, Ryan Thompson, Simon Lin and David Lapointe\nand Michael Green","name":"ChIPpeakAnno","license":"GPL (>= 2)","tags":["Annotation","ChIPSeq","ChIPchip"],"description":"The package includes functions to retrieve the sequences\naround the peak, obtain enriched Gene Ontology (GO) terms, find\nthe nearest gene, exon, miRNA or custom features such as most\nconserved elements and other transcription factor binding sites\nsupplied by users. Starting 2.0.5, new functions have been\nadded for finding the peaks with bi-directional promoters with\nsummary statistics (peaksNearBDP), for summarizing the\noccurrence of motifs in peaks (summarizePatternInPeaks) and for\nadding other IDs to annotated peaks or enrichedGO (addGeneIDs).\nThis package leverages the biomaRt, IRanges, Biostrings,\nBSgenome, GO.db, multtest and stat packages.","downloads_month":1590,"downloads_total":355524,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html"},{"authors":"Tom Carroll, Wei Liu, Ines de Santiago and Rory Stark","name":"ChIPQC","license":"GPL (>= 3)","tags":["Sequencing","ChIPSeq","QualityControl","ReportWriting"],"description":"Quality metrics for ChIPseq data.","downloads_month":436,"downloads_total":44546,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPQC.html"},{"authors":"Guangchuang Yu [aut, cre]\n(<https://orcid.org/0000-0002-6485-8781>), Yun Yan [ctb], Hervé\nPagès [ctb], Michael Kluge [ctb] and Thomas Schwarzl [ctb]","name":"ChIPseeker","license":"Artistic-2.0","tags":["Annotation","ChIPSeq","Software","Visualization","MultipleComparison"],"description":"This package implements functions to retrieve the nearest\ngenes around the peak, annotate genomic region of the peak,\nstatstical methods for estimate the significance of overlap\namong ChIP peak data sets, and incorporate GEO database for\nuser to compare the own dataset with those deposited in\ndatabase. The comparison can be used to infer cooperative\nregulation and thus can be used to generate hypotheses. Several\nvisualization functions are implemented to summarize the\ncoverage of the peak experiment, average profile and heatmap of\npeaks binding to TSS regions, genomic annotation, distance to\nTSS, and overlap of peaks or genes.","downloads_month":1946,"downloads_total":132218,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPseeker.html"},{"authors":"Deepayan Sarkar, Robert Gentleman, Michael Lawrence and Zizhen Yao","name":"chipseq","license":"Artistic-2.0","tags":["ChIPSeq","Sequencing","Coverage","QualityControl","DataImport"],"description":"Tools for helping process short read data for chipseq\nexperiments","downloads_month":776,"downloads_total":130196,"page":"http://bioconductor.org/packages/release/bioc/html/chipseq.html"},{"authors":"Peter Humburg","name":"ChIPseqR","license":"GPL (>= 2)","tags":["ChIPSeq","Infrastructure"],"description":"ChIPseqR identifies protein binding sites from ChIP-seq\nand nucleosome positioning experiments. The model used to\ndescribe binding events was developed to locate nucleosomes but\nshould flexible enough to handle other types of experiments as\nwell.","downloads_month":233,"downloads_total":56222,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPseqR.html"},{"authors":"Nicolas Descostes","name":"ChIPSeqSpike","license":"Artistic-2.0","tags":["ImmunoOncology","ChIPSeq","Sequencing","Normalization","Transcription","Coverage","DifferentialMethylation","Epigenetics","DataImport","HistoneModification"],"description":"Chromatin Immuno-Precipitation followed by Sequencing\n(ChIP-Seq) is used to determine the binding sites of any\nprotein of interest, such as transcription factors or histones\nwith or without a specific modification, at a genome scale. The\nmany steps of the protocol can introduce biases that make\nChIP-Seq more qualitative than quantitative. For instance, it\nwas shown that global histone modification differences are not\ncaught by traditional downstream data normalization techniques.\nA case study reported no differences in histone H3 lysine-27\ntrimethyl (H3K27me3) upon Ezh2 inhibitor treatment. To tackle\nthis problem, external spike-in control were used to keep track\nof technical biases between conditions. Exogenous DNA from a\ndifferent non-closely related species was inserted during the\nprotocol to infer scaling factors that enabled an accurate\nnormalization, thus revealing the inhibitor effect.\nChIPSeqSpike offers tools for ChIP-Seq spike-in normalization.\nReady to use scaled bigwig files and scaling factors values are\nobtained as output. ChIPSeqSpike also provides tools for\nChIP-Seq spike-in assessment and analysis through a versatile\ncollection of graphical functions.","downloads_month":204,"downloads_total":2880,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPSeqSpike.html"},{"authors":"Peter Humburg","name":"ChIPsim","license":"GPL (>= 2)","tags":["Infrastructure","ChIPSeq"],"description":"A general framework for the simulation of ChIP-seq data.\nAlthough currently focused on nucleosome positioning the\npackage is designed to support different types of experiments.","downloads_month":250,"downloads_total":53594,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPsim.html"},{"authors":"George Wu","name":"ChIPXpress","license":"GPL(>=2)","tags":["ChIPchip","ChIPSeq"],"description":"ChIPXpress takes as input predicted TF bound genes from\nChIPx data and uses a corresponding database of gene expression\nprofiles downloaded from NCBI GEO to rank the TF bound targets\nin order of which gene is most likely to be functional TF\ntarget.","downloads_month":141,"downloads_total":19496,"page":"http://bioconductor.org/packages/release/bioc/html/ChIPXpress.html"},{"authors":"Hin-Tak Leung <htl10@users.sourceforge.net>","name":"chopsticks","license":"GPL-3","tags":["Microarray","SNPsAndGeneticVariability","SNP","GeneticVariability"],"description":"Implements classes and methods for large-scale SNP\nassociation studies","downloads_month":272,"downloads_total":58964,"page":"http://bioconductor.org/packages/release/bioc/html/chopsticks.html"},{"authors":"Oscar Reina and David Rossell","name":"chroGPS","license":"GPL (>=2.14)","tags":["Epigenetics","Clustering","ChIPchip","ChIPSeq","HistoneModification","Visualization","DataRepresentation","ImmunoOncology"],"description":"We provide intuitive maps to visualize, analyze and\ncompare the association between genetic elements based on their\nepigenetic profiles. The approach is based on Multi-Dimensional\nScaling, and includes a parallelized implementation for\nhandling high dimensional datasets. We provide several sensible\ndistance metrics, and adjustment procedures to remove\nsystematic biases typically observed when merging data obtained\nunder different technologies or genetic backgrounds. We also\nprovide functions and methods to perform differential analysis\nof epigenome maps at factor and gene level.","downloads_month":158,"downloads_total":29118,"page":"http://bioconductor.org/packages/release/bioc/html/chroGPS.html"},{"authors":"Jan Janecka, Ing. and Mgr. CEITEC Masaryk University","name":"chromDraw","license":"GPL-3","tags":["Software"],"description":"ChromDraw is a R package for drawing the schemes of\nkaryotype(s) in the linear and circular fashion. It is possible\nto visualized cytogenetic marsk on the chromosomes. This tool\nhas own input data format. Input data can be imported from the\nGenomicRanges data structure. This package can visualized the\ndata in the BED file format. Here is requirement on to the\nfirst nine fields of the BED format. Output files format are\n*.eps and *.svg.","downloads_month":159,"downloads_total":13350,"page":"http://bioconductor.org/packages/release/bioc/html/chromDraw.html"},{"authors":"Tim F. Rayner","name":"ChromHeatMap","license":"Artistic-2.0","tags":["Visualization"],"description":"The ChromHeatMap package can be used to plot genome-wide\ndata (e.g. expression, CGH, SNP) along each strand of a given\nchromosome as a heat map. The generated heat map can be used to\ninteractively identify probes and genes of interest.","downloads_month":170,"downloads_total":45382,"page":"http://bioconductor.org/packages/release/bioc/html/ChromHeatMap.html"},{"authors":"Ricardo A. Verdugo and Karen Y. Orostica","name":"chromPlot","license":"GPL (>= 2)","tags":["DataRepresentation","FunctionalGenomics","Genetics","Sequencing","Annotation","Visualization"],"description":"Package designed to visualize genomic data along the\nchromosomes, where the vertical chromosomes are sorted by\nnumber, with sex chromosomes at the end.","downloads_month":200,"downloads_total":11014,"page":"http://bioconductor.org/packages/release/bioc/html/chromPlot.html"},{"authors":"Aaron Taudt, Maria Colome Tatche, Matthias Heinig and Minh Anh\nNguyen","name":"chromstaR","license":"Artistic-2.0","tags":["ImmunoOncology","Software","DifferentialPeakCalling","HiddenMarkovModel","ChIPSeq","HistoneModification","MultipleComparison","Sequencing","PeakDetection","ATACSeq"],"description":"This package implements functions for combinatorial and\ndifferential analysis of ChIP-seq data. It includes uni- and\nmultivariate peak-calling, export to genome browser viewable\nfiles, and functions for enrichment analyses.","downloads_month":222,"downloads_total":8824,"page":"http://bioconductor.org/packages/release/bioc/html/chromstaR.html"},{"authors":"Selin Jessa [aut, cre] and Claudia L. Kleinman [aut]","name":"chromswitch","license":"MIT + file LICENSE","tags":["ImmunoOncology","MultipleComparison","Transcription","GeneExpression","DifferentialPeakCalling","HistoneModification","Epigenetics","FunctionalGenomics","Clustering"],"description":"Chromswitch implements a flexible method to detect\nchromatin state switches between samples in two biological\nconditions in a specific genomic region of interest given peaks\nor chromatin state calls from ChIP-seq data.","downloads_month":191,"downloads_total":4006,"page":"http://bioconductor.org/packages/release/bioc/html/chromswitch.html"},{"authors":"Alicia Schep [aut, cre], Jason Buenrostro [ctb], Caleb Lareau\n[ctb], William Greenleaf [ths] and Stanford University [cph]","name":"chromVAR","license":"MIT + file LICENSE","tags":["SingleCell","Sequencing","GeneRegulation"],"description":"Determine variation in chromatin accessibility across sets\nof annotations or peaks. Designed primarily for single-cell or\nsparse chromatin accessibility data, e.g. from scATAC-seq or\nsparse bulk ATAC or DNAse-seq experiments.","downloads_month":392,"downloads_total":7140,"page":"http://bioconductor.org/packages/release/bioc/html/chromVAR.html"},{"authors":"Aristidis G. Vrahatis, Konstantina Dimitrakopoulou and Panos\nBalomenos","name":"CHRONOS","license":"GPL-2","tags":["SystemsBiology","GraphAndNetwork","Pathways","KEGG"],"description":"A package used for efficient unraveling of the inherent\ndynamic properties of pathways. MicroRNA-mediated subpathway\ntopologies are extracted and evaluated by exploiting the\ntemporal transition and the fold change activity of the linked\ngenes/microRNAs.","downloads_month":174,"downloads_total":9574,"page":"http://bioconductor.org/packages/release/bioc/html/CHRONOS.html"},{"authors":"Hannah Pliner [aut, cre] and Cole Trapnell [aut]","name":"cicero","license":"MIT + file LICENSE","tags":["Sequencing","Clustering","CellBasedAssays","ImmunoOncology\nGeneRegulation","GeneTarget","Epigenetics","ATACSeq","SingleCell"],"description":"Cicero computes putative cis-regulatory maps from\nsingle-cell chromatin accessibility data. It also extends\nmonocle 2 for use in chromatin accessibility data.","downloads_month":207,"downloads_total":1772,"page":"http://bioconductor.org/packages/release/bioc/html/cicero.html"},{"authors":"Lei Song, Krithika Bhuvaneshwar, Yue Wang, Yuanjian Feng, Ie-Ming Shih, Subha Madhavan and Yuriy Gusev","name":"CINdex","license":"GPL (>= 2)","tags":["Software","CopyNumberVariation","GenomicVariation","aCGH","Microarray","Genetics","Sequencing"],"description":"The CINdex package addresses important area of\nhigh-throughput genomic analysis. It allows the automated\nprocessing and analysis of the experimental DNA copy number\ndata generated by Affymetrix SNP 6.0 arrays or similar high\nthroughput technologies. It calculates the chromosome\ninstability (CIN) index that allows to quantitatively\ncharacterize genome-wide DNA copy number alterations as a\nmeasure of chromosomal instability. This package calculates not\nonly overall genomic instability, but also instability in terms\nof copy number gains and losses separately at the chromosome\nand cytoband level.","downloads_month":133,"downloads_total":8484,"page":"http://bioconductor.org/packages/release/bioc/html/CINdex.html"},{"authors":"Likun Wang <wanglk@hsc.pku.edu.cn>","name":"cisPath","license":"GPL (>= 3)","tags":["Proteomics"],"description":"cisPath is an R package that uses web browsers to\nvisualize and manage protein-protein interaction networks.","downloads_month":152,"downloads_total":29008,"page":"http://bioconductor.org/packages/release/bioc/html/cisPath.html"},{"authors":"Dario Strbenac, John Ormerod, Graham Mann and Jean Yang","name":"ClassifyR","license":"GPL-3","tags":["Classification","Survival"],"description":"The software formalises a framework for classification in\nR. There are four stages; Data transformation, feature\nselection, classifier training, and prediction. The\nrequirements of variable types and names are fixed, but\nspecialised variables for functions can also be provided. The\nclassification framework is wrapped in a driver loop, that\nreproducibly carries out a number of cross-validation schemes.\nFunctions for differential expression, differential\nvariability, and differential distribution are included.\nAdditional functions may be developed by the user, by creating\nan interface to the framework.","downloads_month":209,"downloads_total":20362,"page":"http://bioconductor.org/packages/release/bioc/html/ClassifyR.html"},{"authors":"Sarah Sheppard, Jianhong Ou, Nathan Lawson and Lihua Julie Zhu","name":"cleanUpdTSeq","license":"GPL-2","tags":["Sequencing","SequenceMatching","Genetics","GeneRegulation"],"description":"This package uses the Naive Bayes classifier (from e1071)\nto assign probability values to putative polyadenylation sites\n(pA sites) based on training data from zebrafish. This will\nallow the user to separate true, biologically relevant pA sites\nfrom false, oligodT primed pA sites.","downloads_month":143,"downloads_total":24700,"page":"http://bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html"},{"authors":"Sebastian Gibb [aut and cre]\n(<https://orcid.org/0000-0001-7406-4443>)","name":"cleaver","license":"GPL (>= 3)","tags":["Proteomics"],"description":"In-silico cleavage of polypeptide sequences. The cleavage\nrules are taken from:\nhttp://web.expasy.org/peptide_cutter/peptidecutter_enzymes.html","downloads_month":363,"downloads_total":36968,"page":"http://bioconductor.org/packages/release/bioc/html/cleaver.html"},{"authors":"Stephen Nyangoma","name":"clippda","license":"GPL (>=2)","tags":["Proteomics","OneChannel","Preprocessing","DifferentialExpression","MultipleComparison"],"description":"Methods for the nalysis of data from clinical proteomic\nprofiling studies. The focus is on the studies of human\nsubjects, which are often observational case-control by design\nand have technical replicates. A method for sample size\ndetermination for planning these studies is proposed. It\nincorporates routines for adjusting for the expected\nheterogeneities and imbalances in the data and the\nwithin-sample replicate correlations.","downloads_month":159,"downloads_total":40944,"page":"http://bioconductor.org/packages/release/bioc/html/clippda.html"},{"authors":"Habil Zare and Alex Hu","name":"Clomial","license":"GPL (>= 2)","tags":["Genetics","GeneticVariability","Sequencing","Clustering","MultipleComparison","Bayesian","DNASeq","ExomeSeq","TargetedResequencing","ImmunoOncology"],"description":"Clomial fits binomial distributions to counts obtained\nfrom Next Gen Sequencing data of multiple samples of the same\ntumor. The trained parameters can be interpreted to infer the\nclonal structure of the tumor.","downloads_month":155,"downloads_total":20238,"page":"http://bioconductor.org/packages/release/bioc/html/Clomial.html"},{"authors":"Irina Ostrovnaya","name":"Clonality","license":"GPL-3","tags":["Microarray","CopyNumberVariation","Classification","aCGH"],"description":"Statistical tests for clonality versus independence of\ntumors from the same patient based on their LOH or genomewide\ncopy number profiles","downloads_month":150,"downloads_total":34816,"page":"http://bioconductor.org/packages/release/bioc/html/Clonality.html"},{"authors":"Charles Plessy <plessy@riken.jp>","name":"clonotypeR","license":"file LICENSE","tags":["Sequencing"],"description":"High throughput analysis of T cell antigen receptor\nsequences The genes encoding T cell receptors are created by\nsomatic recombination, generating an immense combination of V,\n(D) and J segments. Additional processes during the\nrecombination create extra sequence diversity between the V an\nJ segments. Collectively, this hyper-variable region is called\nthe CDR3 loop. The purpose of this package is to process and\nquantitatively analyse millions of V-CDR3-J combination, called\nclonotypes, from multiple sequence libraries.","downloads_month":142,"downloads_total":23870,"page":"http://bioconductor.org/packages/release/bioc/html/clonotypeR.html"},{"authors":"Noah Hoffman","name":"clst","license":"GPL-3","tags":["Classification"],"description":"Package for modified nearest-neighbor classification based\non calculation of a similarity threshold distinguishing\nwithin-group from between-group comparisons.","downloads_month":148,"downloads_total":34598,"page":"http://bioconductor.org/packages/release/bioc/html/clst.html"},{"authors":"Noah Hoffman","name":"clstutils","license":"GPL-3","tags":["Sequencing","Classification","Visualization","QualityControl"],"description":"Tools for performing taxonomic assignment based on\nphylogeny using pplacer and clst.","downloads_month":144,"downloads_total":33106,"page":"http://bioconductor.org/packages/release/bioc/html/clstutils.html"},{"authors":"Tobias Depke [aut, cre], Raimo Franke [ctb] and Mark Broenstrup\n[ths]","name":"CluMSID","license":"MIT + file LICENSE","tags":["Metabolomics","Preprocessing","Clustering"],"description":"CluMSID is a tool that aids the identification of features\nin untargeted LC-MS/MS analysis by the use of MS2 spectra\nsimilarity and unsupervised statistical methods. It offers\nfunctions for a complete and customisable workflow from raw\ndata to visualisations and is interfaceable with the xmcs\nfamily of preprocessing packages.","downloads_month":43,"downloads_total":178,"page":"http://bioconductor.org/packages/release/bioc/html/CluMSID.html"},{"authors":"Aurora Torrente and Alvis Brazma.","name":"clustComp","license":"GPL (>= 2)","tags":["GeneExpression","Clustering","Visualization"],"description":"clustComp is a package that implements several techniques\nfor the comparison and visualisation of relationships between\ndifferent clustering results, either flat versus flat or\nhierarchical versus flat. These relationships among clusters\nare displayed using a weighted bi-graph, in which the nodes\nrepresent the clusters and the edges connect pairs of nodes\nwith non-empty intersection; the weight of each edge is the\nnumber of elements in that intersection and is displayed\nthrough the edge thickness. The best layout of the bi-graph is\nprovided by the barycentre algorithm, which minimises the\nweighted number of crossings. In the case of comparing a\nhierarchical and a non-hierarchical clustering, the dendrogram\nis pruned at different heights, selected by exploring the tree\nby depth-first search, starting at the root. Branches are\ndecided to be split according to the value of a scoring\nfunction, that can be based either on the aesthetics of the\nbi-graph or on the mutual information between the hierarchical\nand the flat clusterings. A mapping between groups of clusters\nfrom each side is constructed with a greedy algorithm, and can\nbe additionally visualised.","downloads_month":159,"downloads_total":8744,"page":"http://bioconductor.org/packages/release/bioc/html/clustComp.html"},{"authors":"Elizabeth Purdom [aut, cre, cph], Davide Risso [aut] and Marla\nJohnson [ctb]","name":"clusterExperiment","license":"Artistic-2.0","tags":["Clustering","RNASeq","Sequencing","Software","SingleCell"],"description":"Provides functionality for running and comparing many\ndifferent clusterings of single-cell sequencing data or other\nlarge mRNA Expression data sets.","downloads_month":634,"downloads_total":19452,"page":"http://bioconductor.org/packages/release/bioc/html/clusterExperiment.html"},{"authors":"Adrian Pasculescu","name":"ClusterJudge","license":"Artistic-2.0","tags":["Software","StatisticalMethod","Clustering","GeneExpression","GO"],"description":"ClusterJudge implements the functions, examples and other\nsoftware published as an algorithm by Gibbons, FD and Roth FP.\nThe article is called \"Judging the Quality of Gene\nExpression-Based Clustering Methods Using Gene Annotation\" and\nit appeared in Genome Research, vol. 12, pp1574-1581 (2002).\nSee package?ClusterJudge for an overview.","downloads_month":136,"downloads_total":3260,"page":"http://bioconductor.org/packages/release/bioc/html/ClusterJudge.html"},{"authors":"Guangchuang Yu [aut, cre, cph]\n(<https://orcid.org/0000-0002-6485-8781>), Li-Gen Wang [ctb] and Giovanni Dall'Olio [ctb] (formula interface of compareCluster)","name":"clusterProfiler","license":"Artistic-2.0","tags":["Annotation","Clustering","GeneSetEnrichment","GO","KEGG","MultipleComparison","Pathways","Reactome","Visualization"],"description":"This package implements methods to analyze and visualize\nfunctional profiles (GO and KEGG) of gene and gene clusters.","downloads_month":9726,"downloads_total":424538,"page":"http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html"},{"authors":"Thomas J. Hardcastle & Irene Papatheodorou","name":"clusterSeq","license":"GPL-3","tags":["Sequencing","DifferentialExpression","MultipleComparison","Clustering","GeneExpression"],"description":"Identification of clusters of co-expressed genes based on\ntheir expression across multiple (replicated) biological\nsamples.","downloads_month":144,"downloads_total":5516,"page":"http://bioconductor.org/packages/release/bioc/html/clusterSeq.html"},{"authors":"Jason T. Serviss [aut, cre] and Jesper R. Gadin [aut]","name":"ClusterSignificance","license":"GPL-3","tags":["Clustering","Classification","PrincipalComponent","StatisticalMethod"],"description":"The ClusterSignificance package provides tools to assess\nif class clusters in dimensionality reduced data\nrepresentations have a separation different from permuted data.\nThe term class clusters here refers to, clusters of points\nrepresenting known classes in the data. This is particularly\nuseful to determine if a subset of the variables, e.g. genes in\na specific pathway, alone can separate samples into these\nestablished classes. ClusterSignificance accomplishes this by,\nprojecting all points onto a one dimensional line. Cluster\nseparations are then scored and the probability of the seen\nseparation being due to chance is evaluated using a permutation\nmethod.","downloads_month":142,"downloads_total":9228,"page":"http://bioconductor.org/packages/release/bioc/html/ClusterSignificance.html"},{"authors":"James W. MacDonald, Debashis Ghosh and Mark Smolkin","name":"clusterStab","license":"Artistic-2.0","tags":["Clustering"],"description":"This package can be used to estimate the number of\nclusters in a set of microarray data, as well as test the\nstability of these clusters.","downloads_month":248,"downloads_total":61078,"page":"http://bioconductor.org/packages/release/bioc/html/clusterStab.html"},{"authors":"Martin Slawski <ms@cs.uni-sb.de>, Anne-Laure Boulesteix\n<boulesteix@ibe.med.uni-muenchen.de> and Christoph Bernau\n<bernau@ibe.med.uni-muenchen.de>.","name":"CMA","license":"GPL (>= 2)","tags":["Classification","DecisionTree"],"description":"This package provides a comprehensive collection of\nvarious microarray-based classification algorithms both from\nMachine Learning and Statistics. Variable Selection,\nHyperparameter tuning, Evaluation and Comparison can be\nperformed combined or stepwise in a user-friendly environment.","downloads_month":241,"downloads_total":67454,"page":"http://bioconductor.org/packages/release/bioc/html/CMA.html"},{"authors":"Andreas Mitterecker and Djork-Arne Clevert","name":"cn.farms","license":"LGPL (>= 2.0)","tags":["Microarray","CopyNumberVariation"],"description":"This package implements the cn.FARMS algorithm for copy\nnumber variation (CNV) analysis. cn.FARMS allows to analyze the\nmost common Affymetrix (250K-SNP6.0) array types, supports\nhigh-performance computing using snow and ff.","downloads_month":163,"downloads_total":36058,"page":"http://bioconductor.org/packages/release/bioc/html/cn.farms.html"},{"authors":"Guenter Klambauer","name":"cn.mops","license":"LGPL (>= 2.0)","tags":["Sequencing","CopyNumberVariation","Homo_sapiens","CellBiology","HapMap","Genetics"],"description":"cn.mops (Copy Number estimation by a Mixture Of PoissonS)\nis a data processing pipeline for copy number variations and\naberrations (CNVs and CNAs) from next generation sequencing\n(NGS) data. The package supplies functions to convert BAM files\ninto read count matrices or genomic ranges objects, which are\nthe input objects for cn.mops. cn.mops models the depths of\ncoverage across samples at each genomic position. Therefore, it\ndoes not suffer from read count biases along chromosomes. Using\na Bayesian approach, cn.mops decomposes read variations across\nsamples into integer copy numbers and noise by its mixture\ncomponents and Poisson distributions, respectively. cn.mops\nguarantees a low FDR because wrong detections are indicated by\nhigh noise and filtered out. cn.mops is very fast and written\nin C++.","downloads_month":385,"downloads_total":68820,"page":"http://bioconductor.org/packages/release/bioc/html/cn.mops.html"},{"authors":"Stefano Berri <sberri@illumina.com>, Henry M. Wood\n<H.M.Wood@leeds.ac.uk> and Arief Gusnanto <a.gusnanto@leeds.ac.uk>","name":"CNAnorm","license":"GPL-2","tags":["CopyNumberVariation","Sequencing","Coverage","Normalization","WholeGenome","DNASeq","GenomicVariation"],"description":"Performs ratio, GC content correction and normalization of\ndata obtained using low coverage (one read every 100-10,000 bp)\nhigh troughput sequencing. It performs a \"discrete\"\nnormalization looking for the ploidy of the genome. It will\nalso provide tumour content if at least two ploidy states can\nbe found.","downloads_month":155,"downloads_total":39124,"page":"http://bioconductor.org/packages/release/bioc/html/CNAnorm.html"},{"authors":"Ge Tan <ge_tan@live.com>","name":"CNEr","license":"GPL-2 | file LICENSE","tags":["GeneRegulation","Visualization","DataImport"],"description":"Large-scale identification and advanced visualization of\nsets of conserved noncoding elements.","downloads_month":2146,"downloads_total":101074,"page":"http://bioconductor.org/packages/release/bioc/html/CNEr.html"},{"authors":"A. MacNamara","name":"CNORdt","license":"GPL-2","tags":["ImmunoOncology","CellBasedAssays","CellBiology","Proteomics","TimeCourse"],"description":"This add-on to the package CellNOptR handles time-course\ndata, as opposed to steady state data in CellNOptR. It scales\nthe simulation step to allow comparison and model fitting for\ntime-course data. Future versions will optimize delays and\nstrengths for each edge.","downloads_month":203,"downloads_total":30148,"page":"http://bioconductor.org/packages/release/bioc/html/CNORdt.html"},{"authors":"F.Eduati","name":"CNORfeeder","license":"GPL-3","tags":["ImmunoOncology","CellBasedAssays","CellBiology","Proteomics","Bioinformatics","NetworkInference"],"description":"This package integrates literature-constrained and\ndata-driven methods to infer signalling networks from\nperturbation experiments. It permits to extends a given network\nwith links derived from the data via various inference methods\nand uses information on physical interactions of proteins to\nguide and validate the integration of links.","downloads_month":212,"downloads_total":28550,"page":"http://bioconductor.org/packages/release/bioc/html/CNORfeeder.html"},{"authors":"M. Morris and T. Cokelaer","name":"CNORfuzzy","license":"GPL-2","tags":["Network"],"description":"This package is an extension to CellNOptR.  It contains\nadditional functionality needed to simulate and train a prior\nknowledge network to experimental data using constrained fuzzy\nlogic (cFL, rather than Boolean logic as is the case in\nCellNOptR).  Additionally, this package will contain functions\nto use for the compilation of multiple optimization results\n(either Boolean or cFL).","downloads_month":158,"downloads_total":29154,"page":"http://bioconductor.org/packages/release/bioc/html/CNORfuzzy.html"},{"authors":"David Henriques and Thomas Cokelaer","name":"CNORode","license":"GPL-2","tags":["ImmunoOncology","CellBasedAssays","CellBiology","Proteomics","Bioinformatics","TimeCourse"],"description":"ODE add-on to CellNOptR","downloads_month":279,"downloads_total":34140,"page":"http://bioconductor.org/packages/release/bioc/html/CNORode.html"},{"authors":"Stephen Cristiano, Jacob Carey, and Rob Scharpf","name":"CNPBayes","license":"Artistic-2.0","tags":["CopyNumberVariation","Bayesian"],"description":"Bayesian hierarchical mixture models for batch effects and\ncopy number.","downloads_month":149,"downloads_total":11002,"page":"http://bioconductor.org/packages/release/bioc/html/CNPBayes.html"},{"authors":"Jianhua Zhang","name":"CNTools","license":"LGPL","tags":["Microarray","CopyNumberVariation"],"description":"This package provides tools to convert the output of\nsegmentation analysis using DNAcopy to a matrix structure with\noverlapping segments as rows and samples as columns so that\nother computational analyses can be applied to segmented data","downloads_month":323,"downloads_total":85076,"page":"http://bioconductor.org/packages/release/bioc/html/CNTools.html"},{"authors":"Daniele Merico <daniele.merico@gmail.com> and Robert Ziman\n<rziman@gmail.com>; packaged by Joseph Lugo\n<joseph.r.lugo@gmail.com>","name":"cnvGSA","license":"LGPL","tags":["MultipleComparison"],"description":"This package is intended to facilitate gene-set\nassociation with rare CNVs in case-control studies.","downloads_month":147,"downloads_total":31410,"page":"http://bioconductor.org/packages/release/bioc/html/cnvGSA.html"},{"authors":"Cristiano Oliveira [aut], Thomas Wolf [aut, cre], Albrecht\nStenzinger [ctb], Volker Endris [ctb], Nicole Pfarr [ctb], Benedikt Brors [ths] and Wilko Weichert [ths]","name":"CNVPanelizer","license":"GPL-3","tags":["Classification","Sequencing","Normalization","CopyNumberVariation","Coverage"],"description":"A method that allows for the use of a collection of\nnon-matched normal tissue samples. Our approach uses a\nnon-parametric bootstrap subsampling of the available reference\nsamples to estimate the distribution of read counts from\ntargeted sequencing. As inspired by random forest, this is\ncombined with a procedure that subsamples the amplicons\nassociated with each of the targeted genes. The obtained\ninformation allows us to reliably classify the copy number\naberrations on the gene level.","downloads_month":217,"downloads_total":15136,"page":"http://bioconductor.org/packages/release/bioc/html/CNVPanelizer.html"},{"authors":"Ludwig Geistlinger [aut, cre], Vinicius Henrique da Silva\n[aut], Marcel Ramos [ctb] and Levi Waldron [ctb]","name":"CNVRanger","license":"Artistic-2.0","tags":["CopyNumberVariation","DifferentialExpression","GeneExpression","GenomeWideAssociation","GenomicVariation","Microarray","RNASeq","SNP"],"description":"The CNVRanger package implements a comprehensive tool\nsuite for CNV analysis. This includes functionality for\nsummarizing individual CNV calls across a population, assessing\noverlap with functional genomic regions, and association\nanalysis with gene expression and quantitative phenotypes.","downloads_month":0,"downloads_total":6,"page":"http://bioconductor.org/packages/release/bioc/html/CNVRanger.html"},{"authors":"Hoang Tan Nguyen, Tony R Merriman and Mik Black","name":"CNVrd2","license":"GPL-2","tags":["CopyNumberVariation","SNP","Sequencing","Software","Coverage","LinkageDisequilibrium","Clustering."],"description":"CNVrd2 uses next-generation sequencing data to measure\nhuman gene copy number for multiple samples, indentify SNPs\ntagging copy number variants and detect copy number polymorphic\ngenomic regions.","downloads_month":163,"downloads_total":30832,"page":"http://bioconductor.org/packages/release/bioc/html/CNVrd2.html"},{"authors":"Chris Barnes <christopher.barnes@imperial.ac.uk> and Vincent\nPlagnol <vincent.plagnol@cimr.cam.ac.uk>","name":"CNVtools","license":"GPL-3","tags":["GeneticVariability"],"description":"This package is meant to facilitate the testing of Copy\nNumber Variant data for genetic association, typically in\ncase-control studies.","downloads_month":164,"downloads_total":46682,"page":"http://bioconductor.org/packages/release/bioc/html/CNVtools.html"},{"authors":"Manuela Benary, Stefan Kroeger, Yuehien Lee and Robert Lehmann","name":"cobindR","license":"Artistic-2.0","tags":["ChIPSeq","CellBiology","MultipleComparison","SequenceMatching"],"description":"Finding and analysing co-occuring motifs of transcription\nfactor binding sites in groups of genes","downloads_month":160,"downloads_total":23570,"page":"http://bioconductor.org/packages/release/bioc/html/cobindR.html"},{"authors":"B. Ding and R. Gentleman","name":"CoCiteStats","license":"CPL","tags":["Software"],"description":"A collection of software tools for dealing with\nco-citation data.","downloads_month":170,"downloads_total":52944,"page":"http://bioconductor.org/packages/release/bioc/html/CoCiteStats.html"},{"authors":"John Lawson [aut, cre] and Nathan Sheffield [aut]\n(http://www.databio.org)","name":"COCOA","license":"GPL-3","tags":["PrincipalComponent","GenomicVariation","DNAMethylation","GeneRegulation","GenomeAnnotation","SystemsBiology","FunctionalGenomics","ChIPSeq","MethylSeq","Sequencing","Epigenetics","ImmunoOncology"],"description":"COCOA is a method for understanding variation among\nsamples and can be used with data that includes genomic\ncoordinates such as DNA methylation. On a high level, COCOA\nuses a database of \"region sets\" and principal component\nanalysis (PCA) of your data to identify sources of variation\namong samples. A region set is a set of genomic regions that\nshare a biological annotation, for instance, transcription\nfactor binding regions, histone modification regions, or open\nchromatin regions. COCOA works in both supervised (known groups\nof samples) and unsupervised (no groups) situations and can be\nused as a complement to \"differential\" methods that find\ndiscrete differences between groups. COCOA can identify\nbiologically meaningful sources of variation between samples\nand increase understanding of variation in your data.","downloads_month":100,"downloads_total":830,"page":"http://bioconductor.org/packages/release/bioc/html/COCOA.html"},{"authors":"Diego Diez","name":"codelink","license":"GPL-2","tags":["Microarray","OneChannel","DataImport","Preprocessing"],"description":"This package facilitates reading, preprocessing and\nmanipulating Codelink microarray data. The raw data must be\nexported as text file using the Codelink software.","downloads_month":158,"downloads_total":51240,"page":"http://bioconductor.org/packages/release/bioc/html/codelink.html"},{"authors":"Yuchao Jiang and Nancy R. Zhang","name":"CODEX","license":"GPL-2","tags":["ImmunoOncology","ExomeSeq","Normalization","QualityControl","CopyNumberVariation"],"description":"A normalization and copy number variation calling\nprocedure for whole exome DNA sequencing data. CODEX relies on\nthe availability of multiple samples processed using the same\nsequencing pipeline for normalization, and does not require\nmatched controls. The normalization model in CODEX includes\nterms that specifically remove biases due to GC content, exon\nlength and targeting and amplification efficiency, and latent\nsystemic artifacts. CODEX also includes a Poisson\nlikelihood-based recursive segmentation procedure that\nexplicitly models the count-based exome sequencing data.","downloads_month":365,"downloads_total":22106,"page":"http://bioconductor.org/packages/release/bioc/html/CODEX.html"},{"authors":"Juan David Henao [aut, cre], Liliana Lopez-Kleine [aut] and Andres\nPinzon-Velasco [aut]","name":"coexnet","license":"LGPL","tags":["GeneExpression","Microarray","DifferentialExpression","GraphAndNetwork","NetworkInference","SystemsBiology","Normalization","Network"],"description":"Extracts the gene expression matrix from GEO DataSets\n(.CEL files) as a AffyBatch object. Additionally, can make the\nnormalization process using two different methods (vsn and\nrma). The summarization (pass from multi-probe to one gene)\nuses two different criteria (Maximum value and Median of the\nsamples expression data) and the process of gene differentially\nexpressed analisys using two methods (sam and acde). The\nconstruction of the co-expression network can be conduced using\ntwo different methods, Pearson Correlation Coefficient (PCC) or\nMutual Information (MI) and choosing a threshold value using a\ngraph theory approach.","downloads_month":156,"downloads_total":4132,"page":"http://bioconductor.org/packages/release/bioc/html/coexnet.html"},{"authors":"Thomas Sherman, Wai-shing Lee, Conor Kelton, Ondrej Maxian, Jacob Carey, Genevieve Stein-O'Brien, Michael Considine, Maggie\nWodicka, John Stansfield, Shawn Sivy, Carlo Colantuoni, Alexander Favorov, Mike Ochs and Elana Fertig","name":"CoGAPS","license":"GPL (==2)","tags":["GeneExpression","Transcription","GeneSetEnrichment","DifferentialExpression","Bayesian","Clustering","TimeCourse","RNASeq","Microarray","MultipleComparison","DimensionReduction","ImmunoOncology"],"description":"Coordinated Gene Activity in Pattern Sets (CoGAPS)\nimplements a Bayesian MCMC matrix factorization algorithm,\nGAPS, and links it to gene set statistic methods to infer\nbiological process activity.  It can be used to perform sparse\nmatrix factorization on any data, and when this data represents\nbiomolecules, to do gene set analysis.","downloads_month":273,"downloads_total":26366,"page":"http://bioconductor.org/packages/release/bioc/html/CoGAPS.html"},{"authors":"Zhilong Jia [aut, cre] and Michael Barnes [aut]","name":"cogena","license":"LGPL-3","tags":["Clustering","GeneSetEnrichment","GeneExpression","Visualization","Pathways","KEGG","GO","Microarray","Sequencing","SystemsBiology","DataRepresentation","DataImport"],"description":"cogena is a workflow for co-expressed gene-set enrichment\nanalysis. It aims to discovery smaller scale, but highly\ncorrelated cellular events that may be of great biological\nrelevance. A novel pipeline for drug discovery and drug\nrepositioning based on the cogena workflow is proposed.\nParticularly, candidate drugs can be predicted based on the\ngene expression of disease-related data, or other similar drugs\ncan be identified based on the gene expression of drug-related\ndata. Moreover, the drug mode of action can be disclosed by the\nassociated pathway analysis. In summary, cogena is a flexible\nworkflow for various gene set enrichment analysis for\nco-expressed genes, with a focus on pathway/GO analysis and\ndrug repositioning.","downloads_month":178,"downloads_total":16588,"page":"http://bioconductor.org/packages/release/bioc/html/cogena.html"},{"authors":"Yingying Wei and Michael Ochs","name":"coGPS","license":"GPL-2","tags":["Microarray","DifferentialExpression"],"description":"Gene Set Enrichment Analysis of P-value based statistics\nfor outlier gene detection in dataset merged from multiple\nstudies","downloads_month":153,"downloads_total":30488,"page":"http://bioconductor.org/packages/release/bioc/html/coGPS.html"},{"authors":"Charles Warden <cwarden@coh.org>, Yate-Ching Yuan\n<yyuan@coh.org> and Xiwei Wu <xwu@coh.org>","name":"COHCAP","license":"GPL-3","tags":["DNAMethylation","Microarray","MethylSeq","Epigenetics","DifferentialMethylation"],"description":"COHCAP (pronounced \"co-cap\") provides a pipeline to\nanalyze single-nucleotide resolution methylation data (Illumina\n450k/EPIC methylation array, targeted BS-Seq, etc.). It\nprovides differential methylation for CpG Sites, differential\nmethylation for CpG Islands, integration with gene expression\ndata, with visualizaton options. Discussion Group:\nhttps://sourceforge.net/p/cohcap/discussion/bioconductor/","downloads_month":166,"downloads_total":25446,"page":"http://bioconductor.org/packages/release/bioc/html/COHCAP.html"},{"authors":"Zuguang Gu","name":"cola","license":"MIT + file LICENSE","tags":["Clustering","GeneExpression","Classification","Software"],"description":"Subgroup classification is a basic task in genomic data\nanalysis, especially for gene expression data and methylation\ndata. It can predict novel subgroups when there is nothing\nknown about the data or it can test consistency between\npredicted subgroups with known annotations. The cola package\nprovides a general framework for subgroup classification by\nconsensus clustering. It has following features: 1. It\nmodularize the consensus clustering processes that various\nmethods can be easily integrated. 2. It provides rich\nvisualizations for interpreting the results. 3. It allows\nrunning multiple methods at the same time and provides\nfunctionalities to compare results in a straightforward way. 4.\nIt provides a new method to extract features which are more\nefficient to separate subgroups. 5. It allows doing\npartitioning in a hierarchical way to detect subgroups with\nrelatively smaller difference. 6. It generates detailed reports\nfor the complete analysis.","downloads_month":0,"downloads_total":36,"page":"http://bioconductor.org/packages/release/bioc/html/cola.html"},{"authors":"Tiphaine C. Martin [aut, cre], Thomas Hardiman [aut], Idil Yet\n[aut], Pei-Chien Tsai [aut] and Jordana T. Bell [aut]","name":"coMET","license":"GPL (>= 2)","tags":["Software","DifferentialMethylation","Visualization","Sequencing","Genetics","FunctionalGenomics","Microarray","MethylationArray","MethylSeq","ChIPSeq","DNASeq","RiboSeq","RNASeq","ExomeSeq","DNAMethylation","GenomeWideAssociation","MotifAnnotation"],"description":"Visualisation of EWAS results in a genomic region. In\naddition to phenotype-association P-values, coMET also\ngenerates plots of co-methylation patterns and provides a\nseries of annotation tracks. It can be used to other omic-wide\nassociation scans as long as the data can be translated to\ngenomic level and for any species.","downloads_month":220,"downloads_total":24168,"page":"http://bioconductor.org/packages/release/bioc/html/coMET.html"},{"authors":"Benjamin Johnson [aut, cre], Tim Triche [aut], Kasper Hansen\n[aut] and Jean-Philippe Fortin [aut]","name":"compartmap","license":"GPL-3 + file LICENSE","tags":["ImmunoOncology","Genetics","Epigenetics","ATACSeq","MethylSeq","MethylationArray"],"description":"Compartmap performs shrunken A/B compartment inference\nfrom ATAC-seq and methylation arrays.","downloads_month":135,"downloads_total":1082,"page":"http://bioconductor.org/packages/release/bioc/html/compartmap.html"},{"authors":"Lynn Lin, Kevin Ushey, Greg Finak and Ravio Kolde (pheatmap)","name":"COMPASS","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry"],"description":"COMPASS is a statistical framework that enables unbiased\nanalysis of antigen-specific T-cell subsets. COMPASS uses a\nBayesian hierarchical framework to model all observed\ncell-subsets and select the most likely to be antigen-specific\nwhile regularizing the small cell counts that often arise in\nmulti-parameter space. The model provides a posterior\nprobability of specificity for each cell subset and each\nsample, which can be used to profile a subject's immune\nresponse to external stimuli such as infection or vaccination.","downloads_month":232,"downloads_total":23584,"page":"http://bioconductor.org/packages/release/bioc/html/COMPASS.html"},{"authors":"Charlotte Soneson","name":"compcodeR","license":"GPL (>= 2)","tags":["ImmunoOncology","RNASeq","DifferentialExpression"],"description":"This package provides extensive functionality for\ncomparing results obtained by different methods for\ndifferential expression analysis of RNAseq data. It also\ncontains functions for simulating count data and interfaces to\nseveral packages for performing the differential expression\nanalysis.","downloads_month":234,"downloads_total":24144,"page":"http://bioconductor.org/packages/release/bioc/html/compcodeR.html"},{"authors":"Mattia Pelizzola [aut], Kamal Kishore [aut and cre]","name":"compEpiTools","license":"GPL","tags":["GeneExpression","Sequencing","Visualization","GenomeAnnotation","Coverage"],"description":"Tools for computational epigenomics developed for the\nanalysis, integration and simultaneous visualization of various\n(epi)genomics data types across multiple genomic regions in\nmultiple samples.","downloads_month":173,"downloads_total":21028,"page":"http://bioconductor.org/packages/release/bioc/html/compEpiTools.html"},{"authors":"Sam D. Bassett [aut], Ashley J. Waardenberg [aut and cre]","name":"CompGO","license":"GPL-2","tags":["GeneSetEnrichment","MultipleComparison","GO","Visualization"],"description":"This package contains functions to accomplish several\ntasks. It is able to download full genome databases from UCSC,\nimport .bed files easily, annotate these .bed file regions with\ngenes (plus distance) from aforementioned database dumps,\ninterface with DAVID to create functional annotation and gene\nontology enrichment charts based on gene lists (such as those\ngenerated from input .bed files) and finally visualise and\ncompare these enrichments using either directed acyclic graphs\nor scatterplots.","downloads_month":149,"downloads_total":23754,"page":"http://bioconductor.org/packages/release/bioc/html/CompGO.html"},{"authors":"Zuguang Gu","name":"ComplexHeatmap","license":"MIT + file LICENSE","tags":["Software","Visualization","Sequencing"],"description":"Complex heatmaps are efficient to visualize associations\nbetween different sources of data sets and reveal potential\npatterns. Here the ComplexHeatmap package provides a highly\nflexible way to arrange multiple heatmaps and supports various\nannotation graphics.","downloads_month":6025,"downloads_total":266606,"page":"http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html"},{"authors":"Diogo P. P. Branco [aut and cre]","name":"condcomp","license":"GPL (>=2) | file LICENSE","tags":["ImmunoOncology","Clustering","SingleCell","Visualization"],"description":"For a given clustered data, which can also be split into\ntwo conditions, this package provides a way to perform a\ncondition comparison on said clustered data. The comparison is\nperformed on each cluster. Several statistics are used and,\nwhen analysed in conjunction, they might give some insight\nregarding the heterogeneity of some of the clusters.","downloads_month":90,"downloads_total":814,"page":"http://bioconductor.org/packages/release/bioc/html/condcomp.html"},{"authors":"Diana LOW and Efthimios MOTAKIS","name":"CONFESS","license":"GPL-2","tags":["ImmunoOncology","GeneExpression","DataImport","CellBiology","Clustering","RNASeq","QualityControl","Visualization","TimeCourse","Regression","Classification"],"description":"Single Cell Fluidigm Spot Detector.","downloads_month":169,"downloads_total":8520,"page":"http://bioconductor.org/packages/release/bioc/html/CONFESS.html"},{"authors":"Tim Peters","name":"consensus","license":"BSD_3_clause + file LICENSE","tags":["ImmunoOncology","QualityControl","Regression","DataRepresentation","GeneExpression","Microarray","RNASeq"],"description":"An implementation of the American Society for Testing and\nMaterials (ASTM) Standard E691 for interlaboratory testing\nprocedures, designed for cross-platform genomic measurements.\nGiven three (3) or more genomic platforms or laboratory\nprotocols, this package provides interlaboratory testing\nprocedures giving per-locus comparisons for sensitivity and\nprecision between platforms.","downloads_month":95,"downloads_total":786,"page":"http://bioconductor.org/packages/release/bioc/html/consensus.html"},{"authors":"Matt Wilkerson <mdwilkerson@outlook.com> and Peter Waltman\n<waltman@soe.ucsc.edu>","name":"ConsensusClusterPlus","license":"GPL version 2","tags":["Software","Clustering"],"description":"algorithm for determining cluster count and membership by\nstability evidence in unsupervised analysis","downloads_month":3043,"downloads_total":190722,"page":"http://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html"},{"authors":"Ashley J. Waardenberg","name":"consensusDE","license":"GPL-3","tags":["Transcriptomics","MultipleComparison","Clustering"],"description":"This package allows users to perform DE analysis using\nmultiple algorithms. It seeks consensus from multiple methods.\nCurrently it supports \"Voom\", \"EdgeR\" and \"DESeq\", but can be\neasily extended. It uses RUV-seq (optional) to remove batch\neffects.","downloads_month":160,"downloads_total":1172,"page":"http://bioconductor.org/packages/release/bioc/html/consensusDE.html"},{"authors":"Gregory M Chen, Lavanya Kannan, Ludwig Geistlinger, Victor\nKofia, Levi Waldron and Benjamin Haibe-Kains","name":"consensusOV","license":"Artistic-2.0","tags":["Classification","Clustering","DifferentialExpression","GeneExpression","Microarray","Transcriptomics"],"description":"This package implements four major subtype classifiers for\nhigh-grade serous (HGS) ovarian cancer as described by Helland\net al. (PLoS One, 2011), Bentink et al. (PLoS One, 2012),\nVerhaak et al. (J Clin Invest, 2013), and Konecny et al. (J\nNatl Cancer Inst, 2014). In addition, the package implements a\nconsensus classifier, which consolidates and improves on the\nrobustness of the proposed subtype classifiers, thereby\nproviding reliable stratification of patients with HGS ovarian\ntumors of clearly defined subtype.","downloads_month":148,"downloads_total":3754,"page":"http://bioconductor.org/packages/release/bioc/html/consensusOV.html"},{"authors":"Astrid Deschenes [cre, aut], Fabien Claude Lamaze [ctb], Pascal\nBelleau [aut] and Arnaud Droit [aut]","name":"consensusSeekeR","license":"Artistic-2.0","tags":["BiologicalQuestion","ChIPSeq","Genetics","MultipleComparison","Transcription","PeakDetection","Sequencing","Coverage"],"description":"This package compares genomic positions and genomic ranges\nfrom multiple experiments to extract common regions. The size\nof the analyzed region is adjustable as well as the number of\nexperiences in which a feature must be present in a potential\nregion to tag this region as a consensus region.","downloads_month":150,"downloads_total":9234,"page":"http://bioconductor.org/packages/release/bioc/html/consensusSeekeR.html"},{"authors":"Kieran O'Neill, Mark Hills and Mike Gottlieb","name":"contiBAIT","license":"BSD_2_clause + file LICENSE","tags":["ImmunoOncology","CellBasedAssays","QualityControl","WholeGenome","Genetics","GenomeAssembly"],"description":"Using strand inheritance data from multiple single cells\nfrom the organism whose genome is to be assembled, contiBAIT\ncan cluster unbridged contigs together into putative\nchromosomes, and order the contigs within those chromosomes.","downloads_month":194,"downloads_total":8688,"page":"http://bioconductor.org/packages/release/bioc/html/contiBAIT.html"},{"authors":"Volker Hovestadt and Marc Zapatka","name":"conumee","license":"GPL (>= 2)","tags":["CopyNumberVariation","DNAMethylation","MethylationArray","Microarray","Normalization","Preprocessing","QualityControl","Software"],"description":"This package contains a set of processing and plotting\nmethods for performing copy-number variation (CNV) analysis\nusing Illumina 450k or EPIC methylation arrays.","downloads_month":223,"downloads_total":18530,"page":"http://bioconductor.org/packages/release/bioc/html/conumee.html"},{"authors":"Gordon Smyth <smyth@wehi.edu.au>, James Wettenhall\n<wettenhall@wehi.edu.au>, Yee Hwa (Jean Yang)\n<jean@biostat.ucsf.edu> and Martin Morgan\n<Martin.Morgan@RoswellPark.org>","name":"convert","license":"LGPL","tags":["Infrastructure","Microarray","TwoChannel"],"description":"Define coerce methods for microarray data objects.","downloads_month":408,"downloads_total":109798,"page":"http://bioconductor.org/packages/release/bioc/html/convert.html"},{"authors":"James W. MacDonald","name":"copa","license":"Artistic-2.0","tags":["OneChannel","TwoChannel","DifferentialExpression","Visualization"],"description":"COPA is a method to find genes that undergo recurrent\nfusion in a given cancer type by finding pairs of genes that\nhave mutually exclusive outlier profiles.","downloads_month":175,"downloads_total":51668,"page":"http://bioconductor.org/packages/release/bioc/html/copa.html"},{"authors":"Gro Nilsen, Knut Liestoel and Ole Christian Lingjaerde.","name":"copynumber","license":"Artistic-2.0","tags":["aCGH","SNP","CopyNumberVariation","Genetics","Visualization"],"description":"Penalized least squares regression is applied to fit\npiecewise constant curves to copy number data to locate genomic\nregions of constant copy number. Procedures are available for\nindividual segmentation of each sample, joint segmentation of\nseveral samples and joint segmentation of the two data tracks\nfrom SNP-arrays. Several plotting functions are available for\nvisualization of the data and the segmentation results.","downloads_month":495,"downloads_total":51454,"page":"http://bioconductor.org/packages/release/bioc/html/copynumber.html"},{"authors":"Thomas Kuilman","name":"CopywriteR","license":"GPL-2","tags":["ImmunoOncology","TargetedResequencing","ExomeSeq","CopyNumberVariation","Preprocessing","Visualization","Coverage"],"description":"CopywriteR extracts DNA copy number information from\ntargeted sequencing by utiizing off-target reads. It allows for\nextracting uniformly distributed copy number information, can\nbe used without reference, and can be applied to sequencing\ndata obtained from various techniques including chromatin\nimmunoprecipitation and target enrichment on small gene panels.\nThereby, CopywriteR constitutes a widely applicable alternative\nto available copy number detection tools.","downloads_month":279,"downloads_total":20614,"page":"http://bioconductor.org/packages/release/bioc/html/CopywriteR.html"},{"authors":"Anamaria Elek [cre, aut], Maja Kuzman [aut] and Kristian\nVlahovicek [aut]","name":"coRdon","license":"Artistic-2.0","tags":["Software","Metagenomics","GeneExpression","GeneSetEnrichment","GenePrediction","Visualization","KEGG","Pathways","Genetics\nCellBiology","BiomedicalInformatics","ImmunoOncology"],"description":"Tool for analysis of codon usage in various unannotated or\nKEGG/COG annotated DNA sequences. Calculates different measures\nof CU bias and CU-based predictors of gene expressivity, and\nperforms gene set enrichment analysis for annotated sequences.\nImplements several methods for visualization of CU and\nenrichment analysis results.","downloads_month":140,"downloads_total":1256,"page":"http://bioconductor.org/packages/release/bioc/html/coRdon.html"},{"authors":"Remy Nicolle, Thibault Venzac and Mohamed Elati","name":"CoRegNet","license":"GPL-3","tags":["NetworkInference","NetworkEnrichment","GeneRegulation","GeneExpression","GraphAndNetwork","SystemsBiology","Network","Visualization","Transcription"],"description":"This package provides methods to identify active\ntranscriptional programs. Methods and classes are provided to\nimport or infer large scale co-regulatory network from\ntranscriptomic data. The specificity of the encoded networks is\nto model Transcription Factor cooperation. External regulation\nevidences (TFBS, ChIP,...) can be integrated to assess the\ninferred network and refine it if necessary. Transcriptional\nactivity of the regulators in the network can be estimated\nusing an measure of their influence in a given sample. Finally,\nan interactive UI can be used to navigate through the network\nof cooperative regulators and to visualize their activity in a\nspecific sample or subgroup sample. The proposed visualization\ntool can be used to integrate gene expression, transcriptional\nactivity, copy number status, sample classification and a\ntranscriptional network including co-regulation information.","downloads_month":171,"downloads_total":19124,"page":"http://bioconductor.org/packages/release/bioc/html/CoRegNet.html"},{"authors":"Hongkai Ji and Yingying Wei","name":"Cormotif","license":"GPL-2","tags":["Microarray","DifferentialExpression"],"description":"It fits correlation motif model to multiple studies to\ndetect study specific differential expression patterns.","downloads_month":140,"downloads_total":33318,"page":"http://bioconductor.org/packages/release/bioc/html/Cormotif.html"},{"authors":"Zhenpeng Li","name":"CorMut","license":"GPL-2","tags":["Sequencing"],"description":"CorMut provides functions for computing kaks for\nindividual sites or specific amino acids and detecting\ncorrelated mutations among them. Three methods are provided for\ndetecting correlated mutations ,including conditional selection\npressure, mutual information and Jaccard index. The computation\nconsists of two steps: First, the positive selection sites are\ndetected; Second, the mutation correlations are computed among\nthe positive selection sites. Note that the first step is\noptional. Meanwhile, CorMut facilitates the comparison of the\ncorrelated mutations between two conditions by the means of\ncorrelated mutation network. The reference sequence should be\nthe first sequence of the sequence file, and does not allow the\npresence of gap.","downloads_month":147,"downloads_total":29832,"page":"http://bioconductor.org/packages/release/bioc/html/CorMut.html"},{"authors":"Dongxiao Zhu and Youjuan Li","name":"CORREP","license":"GPL (>= 2)","tags":["Microarray","Clustering","GraphAndNetwork"],"description":"Multivariate correlation estimation and statistical\ninference. See package vignette.","downloads_month":144,"downloads_total":44628,"page":"http://bioconductor.org/packages/release/bioc/html/CORREP.html"},{"authors":"Andrea Rau, Cathy Maugis-Rabusseau and Antoine Godichon-Baggioni","name":"coseq","license":"GPL (>=3)","tags":["GeneExpression","RNASeq","Sequencing","Software","ImmunoOncology"],"description":"Co-expression analysis for expression profiles arising\nfrom high-throughput sequencing data. Feature (e.g., gene)\nprofiles are clustered using adapted transformations and\nmixture models or a K-means algorithm, and model selection\ncriteria (to choose an appropriate number of clusters) are\nprovided.","downloads_month":265,"downloads_total":7350,"page":"http://bioconductor.org/packages/release/bioc/html/coseq.html"},{"authors":"David Fischer <dajofischer@googlemail.com>, Christian Panse\n<cp@fgcz.ethz.ch> and Endre Laczko <endre.laczko@fgcz.uzh.ch>","name":"cosmiq","license":"GPL-3","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"cosmiq is a tool for the preprocessing of liquid- or gas -\nchromatography mass spectrometry (LCMS/GCMS) data with a focus\non metabolomics or lipidomics applications. To improve the\ndetection of low abundant signals, cosmiq generates master maps\nof the mZ/RT space from all acquired runs before a peak\ndetection algorithm is applied. The result is a more robust\nidentification and quantification of low-intensity MS signals\ncompared to conventional approaches where peak picking is\nperformed in each LCMS/GCMS file separately. The cosmiq package\nbuilds on the xcmsSet object structure and can be therefore\nintegrated well with the package xcms as an alternative\npreprocessing step.","downloads_month":185,"downloads_total":17650,"page":"http://bioconductor.org/packages/release/bioc/html/cosmiq.html"},{"authors":"Marco Frasca and Giorgio Valentini -- Universita' degli Studi\ndi Milano","name":"COSNet","license":"GPL (>= 2)","tags":["GraphAndNetwork","Classification","Network","NeuralNetwork"],"description":"Package that implements the COSNet classification\nalgorithm. The algorithm predicts node labels in partially\nlabeled graphs where few positives are available for the class\nbeing predicted.","downloads_month":142,"downloads_total":15870,"page":"http://bioconductor.org/packages/release/bioc/html/COSNet.html"},{"authors":"Kushal Dey [aut, cre], Joyce Hsiao [aut] and Matthew Stephens\n[aut]","name":"CountClust","license":"GPL (>= 2)","tags":["ImmunoOncology","RNASeq","GeneExpression","Clustering","Sequencing","StatisticalMethod","Software","Visualization"],"description":"Fits grade of membership models (GoM, also known as\nadmixture models) to cluster RNA-seq gene expression count\ndata, identifies characteristic genes driving cluster\nmemberships, and provides a visual summary of the cluster\nmemberships.","downloads_month":233,"downloads_total":15178,"page":"http://bioconductor.org/packages/release/bioc/html/CountClust.html"},{"authors":"Charlotte Soneson","name":"countsimQC","license":"GPL (>=2)","tags":["Microbiome","RNASeq","SingleCell","ExperimentalDesign","QualityControl","ReportWriting","Visualization","ImmunoOncology"],"description":"countsimQC provides functionality to create a\ncomprehensive report comparing a broad range of characteristics\nacross a collection of count matrices. One important use case\nis the comparison of one or more synthetic count matrices to a\nreal count matrix, possibly the one underlying the simulations.\nHowever, any collection of count matrices can be compared.","downloads_month":137,"downloads_total":1082,"page":"http://bioconductor.org/packages/release/bioc/html/countsimQC.html"},{"authors":"C. Pacini","name":"covEB","license":"GPL-3","tags":["ImmunoOncology","Bayesian","Microarray","RNASeq","Preprocessing","Software","GeneExpression","StatisticalMethod"],"description":"Using bayesian methods to estimate correlation matrices\nassuming that they can be written and estimated as block\ndiagonal matrices. These block diagonal matrices are determined\nusing shrinkage parameters that values below this parameter to\nzero.","downloads_month":177,"downloads_total":7126,"page":"http://bioconductor.org/packages/release/bioc/html/covEB.html"},{"authors":"Ernesto Lowy","name":"CoverageView","license":"Artistic-2.0","tags":["ImmunoOncology","Visualization","RNASeq","ChIPSeq","Sequencing","Technology","Software"],"description":"This package provides a framework for the visualization of\ngenome coverage profiles. It can be used for ChIP-seq\nexperiments, but it can be also used for genome-wide nucleosome\npositioning experiments or other experiment types where it is\nimportant to have a framework in order to inspect how the\ncoverage distributed across the genome","downloads_month":225,"downloads_total":19548,"page":"http://bioconductor.org/packages/release/bioc/html/CoverageView.html"},{"authors":"Lara Urban <lara.h.urban@ebi.ac.uk>","name":"covRNA","license":"GPL (>= 2)","tags":["GeneExpression","Transcription"],"description":"This package provides the analysis methods fourthcorner\nand RLQ analysis for large-scale transcriptomic data.","downloads_month":148,"downloads_total":6984,"page":"http://bioconductor.org/packages/release/bioc/html/covRNA.html"},{"authors":"Caitlin McHugh, Jessica Larson, and Jason Hackney","name":"cpvSNP","license":"Artistic-2.0","tags":["Genetics","StatisticalMethod","Pathways","GeneSetEnrichment","GenomicVariation"],"description":"Gene set analysis methods exist to combine SNP-level\nassociation p-values into gene sets, calculating a single\nassociation p-value for each gene set. This package implements\ntwo such methods that require only the calculated SNP p-values,\nthe gene set(s) of interest, and a correlation matrix (if\ndesired). One method (GLOSSI) requires independent SNPs and the\nother (VEGAS) can take into account correlation (LD) among the\nSNPs. Built-in plotting functions are available to help users\nvisualize results.","downloads_month":150,"downloads_total":13192,"page":"http://bioconductor.org/packages/release/bioc/html/cpvSNP.html"},{"authors":"Jean (Zhijin) Wu and Kasper Daniel Hansen","name":"cqn","license":"Artistic-2.0","tags":["ImmunoOncology","RNASeq","Preprocessing","DifferentialExpression"],"description":"A normalization tool for RNA-Seq data, implementing the\nconditional quantile normalization method.","downloads_month":573,"downloads_total":54590,"page":"http://bioconductor.org/packages/release/bioc/html/cqn.html"},{"authors":"Henrik Failmezger <failmezger@mpipz.mpg.de>, Yinyin Yuan\n<Yinyin.Yuan@cancer.org.uk>, Oscar Rueda\n<oscar.rueda@cancer.org.uk> and Florian Markowetz\n<Florian.Markowetz@cancer.org.uk>","name":"CRImage","license":"Artistic-2.0","tags":["CellBiology","Classification"],"description":"CRImage provides functionality to process and analyze\nimages, in particular to classify cells in biological images.\nFurthermore, in the context of tumor images, it provides\nfunctionality to calculate tumour cellularity.","downloads_month":194,"downloads_total":46538,"page":"http://bioconductor.org/packages/release/bioc/html/CRImage.html"},{"authors":"Lihua Julie Zhu, Benjamin R. Holmes, Hervé Pagès, Michael\nLawrence, Isana Veksler-Lublinsky, Victor Ambros and Neil Aronin\nand Michael Brodsky","name":"CRISPRseek","license":"GPL (>= 2)","tags":["ImmunoOncology","GeneRegulation","SequenceMatching","CRISPR"],"description":"The package includes functions to find potential guide\nRNAs for input target sequences, optionally filter guide RNAs\nwithout restriction enzyme cut site, or without paired guide\nRNAs, genome-wide search for off-targets, score, rank, fetch\nflank sequence and indicate whether the target and off-targets\nare located in exon region or not. Potential guide RNAs are\nannotated with total score of the top5 and topN off-targets,\ndetailed topN mismatch sites, restriction enzyme cut sites, and\npaired guide RNAs. This package leverages Biostrings and\nBSgenome packages.","downloads_month":304,"downloads_total":29992,"page":"http://bioconductor.org/packages/release/bioc/html/CRISPRseek.html"},{"authors":"Sophie Wigmore <Sophie.Wigmore@umassmed.edu>, Alper Kucukural\n<alper.kucukural@umassmed.edu>, Lihua Julie Zhu\n<julie.zhu@umassmed.edu>, Michael Brodsky\n<Michael.Brodsky@umassmed.edu> and Manuel Garber\n<Manuel.Garber@umassmed.edu>","name":"crisprseekplus","license":"GPL-3 + file LICENSE","tags":["GeneRegulation","SequenceMatching","Software"],"description":"Bioinformatics platform containing interface to work with\noffTargetAnalysis and compare2Sequences in the CRISPRseek\npackage, and GUIDEseqAnalysis.","downloads_month":140,"downloads_total":7092,"page":"http://bioconductor.org/packages/release/bioc/html/crisprseekplus.html"},{"authors":"Helen Lindsay [aut and cre]","name":"CrispRVariants","license":"GPL-2","tags":["ImmunoOncology","CRISPR","GenomicVariation","VariantDetection","GeneticVariability","DataRepresentation","Visualization"],"description":"CrispRVariants provides tools for analysing the results of\na CRISPR-Cas9 mutagenesis sequencing experiment, or other\nsequencing experiments where variants within a given region are\nof interest. These tools allow users to localize variant allele\ncombinations with respect to any genomic location (e.g. the\nCas9 cut site), plot allele combinations and calculate mutation\nrates with flexible filtering of unrelated variants.","downloads_month":271,"downloads_total":12224,"page":"http://bioconductor.org/packages/release/bioc/html/CrispRVariants.html"},{"authors":"Benilton S Carvalho, Robert Scharpf, Matt Ritchie, Ingo\nRuczinski and Rafael A Irizarry","name":"crlmm","license":"Artistic-2.0","tags":["Microarray","Preprocessing","SNP","CopyNumberVariation"],"description":"Faster implementation of CRLMM specific to SNP 5.0 and 6.0\narrays, as well as a copy number tool specific to 5.0, 6.0, and\nIllumina platforms.","downloads_month":292,"downloads_total":106970,"page":"http://bioconductor.org/packages/release/bioc/html/crlmm.html"},{"authors":"Alex Pickering","name":"crossmeta","license":"MIT + file LICENSE","tags":["GeneExpression","Transcription","DifferentialExpression","Microarray","TissueMicroarray","OneChannel","Annotation","BatchEffect","Preprocessing","GUI"],"description":"Implements cross-platform and cross-species meta-analyses\nof Affymentrix, Illumina, and Agilent microarray data. This\npackage automates common tasks such as downloading,\nnormalizing, and annotating raw GEO data. The user then selects\ncontrol and treatment samples in order to perform differential\nexpression/pathway analyses for all comparisons. After\nanalysing each contrast seperately, the user can select tissue\nsources for each contrast and specify any tissue sources that\nshould be grouped for the subsequent meta-analyses. Finally,\neffect size and pathway meta-analyses can proceed and the\nresults graphically explored.","downloads_month":185,"downloads_total":12032,"page":"http://bioconductor.org/packages/release/bioc/html/crossmeta.html"},{"authors":"Jose M Muino","name":"CSAR","license":"Artistic-2.0","tags":["ChIPSeq","Transcription","Genetics"],"description":"Statistical tools for ChIP-seq data analysis. The package\nincludes the statistical method described in Kaufmann et al.\n(2009) PLoS Biology: 7(4):e1000090. Briefly, Taking the average\nDNA fragment size subjected to sequencing into account, the\nsoftware calculates genomic single-nucleotide read-enrichment\nvalues. After normalization, sample and control are compared\nusing a test based on the Poisson distribution. Test statistic\nthresholds to control the false discovery rate are obtained\nthrough random permutation.","downloads_month":240,"downloads_total":54842,"page":"http://bioconductor.org/packages/release/bioc/html/CSAR.html"},{"authors":"Aaron Lun [aut, cre] and Gordon Smyth [aut]","name":"csaw","license":"GPL-3","tags":["MultipleComparison","ChIPSeq","Normalization","Sequencing","Coverage","Genetics","Annotation","DifferentialPeakCalling"],"description":"Detection of differentially bound regions in ChIP-seq data\nwith sliding windows, with methods for normalization and proper\nFDR control.","downloads_month":820,"downloads_total":62354,"page":"http://bioconductor.org/packages/release/bioc/html/csaw.html"},{"authors":"Chandler Zuo and Sunduz Keles","name":"CSSP","license":"GPL-2","tags":["ChIPSeq","Sequencing","QualityControl","Bayesian"],"description":"Power computation for ChIP-Seq data based on Bayesian\nestimation for local poisson counting process.","downloads_month":144,"downloads_total":24380,"page":"http://bioconductor.org/packages/release/bioc/html/CSSP.html"},{"authors":"Antoine Lucas <antoinelucas@gmail.com> and Laurent Gautier\n<laurent@cbs.dtu.dk>","name":"ctc","license":"GPL-2","tags":["Microarray","Clustering","Classification","DataImport","Visualization"],"description":"Tools for export and import classification trees and\nclusters to other programs","downloads_month":478,"downloads_total":111814,"page":"http://bioconductor.org/packages/release/bioc/html/ctc.html"},{"authors":"Carles Hernandez-Ferrer [aut, cre] and Jaun R. Gonzalez [aut]","name":"CTDquerier","license":"MIT + file LICENSE","tags":["Software","BiomedicalInformatics","Infrastructure","DataImport","DataRepresentation","GeneSetEnrichment","NetworkEnrichment","Pathways","Network","GO","KEGG"],"description":"Package to retrieve and visualize data from the\nComparative Toxicogenomics Database (http://ctdbase.org/). The\ndownloaded data is formated as DataFrames for further\ndownstream analyses.","downloads_month":137,"downloads_total":2152,"page":"http://bioconductor.org/packages/release/bioc/html/CTDquerier.html"},{"authors":"Bernardo P. de Almeida [aut], Nuno Saraiva-Agostinho [aut, cre], Nuno L. Barbosa-Morais [aut and led]","name":"cTRAP","license":"MIT + file LICENSE","tags":["DifferentialExpression","GeneExpression","RNASeq","Transcriptomics","Pathways","ImmunoOncology"],"description":"Compare differential gene expression results with those\nfrom known cellular perturbations (such as gene knock-down,\noverexpression or small molecules) derived from the\nConnectivity Map. Such analyses allow not only to infer the\nmolecular causes of the observed difference in gene expression\nbut also to identify small molecules that could drive or revert\nspecific transcriptomic alterations.","downloads_month":106,"downloads_total":728,"page":"http://bioconductor.org/packages/release/bioc/html/cTRAP.html"},{"authors":"Michal Sharabi-Schwager [aut, cre] and Ron Ophir [aut]","name":"ctsGE","license":"GPL-2","tags":["ImmunoOncology","GeneExpression","Transcription","DifferentialExpression","GeneSetEnrichment","Genetics","Bayesian","Clustering","TimeCourse","Sequencing","RNASeq"],"description":"Methodology for supervised clustering of potentially many\npredictor variables, such as genes etc., in time series\ndatasets Provides functions that help the user assigning genes\nto predefined set of model profiles.","downloads_month":197,"downloads_total":8202,"page":"http://bioconductor.org/packages/release/bioc/html/ctsGE.html"},{"authors":"L. Goff, C. Trapnell and D. Kelley","name":"cummeRbund","license":"Artistic-2.0","tags":["HighThroughputSequencing","HighThroughputSequencingData","RNAseq","RNAseqData","GeneExpression","DifferentialExpression","Infrastructure","DataImport","DataRepresentation","Visualization","Bioinformatics","Clustering","MultipleComparisons","QualityControl"],"description":"Allows for persistent storage, access, exploration, and\nmanipulation of Cufflinks high-throughput sequencing data.  In\naddition, provides numerous plotting functions for commonly\nused visualizations.","downloads_month":1596,"downloads_total":1021342,"page":"http://bioconductor.org/packages/release/bioc/html/cummeRbund.html"},{"authors":"Xiaojing Wang","name":"customProDB","license":"Artistic-2.0","tags":["ImmunoOncology","Sequencing","MassSpectrometry","Proteomics","SNP","RNASeq","Software","Transcription","AlternativeSplicing","FunctionalGenomics"],"description":"Database search is the most widely used approach for\npeptide and protein identification in mass spectrometry-based\nproteomics studies. Our previous study showed that\nsample-specific protein databases derived from RNA-Seq data can\nbetter approximate the real protein pools in the samples and\nthus improve protein identification. More importantly, single\nnucleotide variations, short insertion and deletions and novel\njunctions identified from RNA-Seq data make protein database\nmore complete and sample-specific. Here, we report an R package\ncustomProDB that enables the easy generation of customized\ndatabases from RNA-Seq data for proteomics search. This work\nbridges genomics and proteomics studies and facilitates\ncross-omics data integration.","downloads_month":265,"downloads_total":32510,"page":"http://bioconductor.org/packages/release/bioc/html/customProDB.html"},{"authors":"Andreas Mock [aut and cre]","name":"CVE","license":"GPL-3","tags":["BiomedicalInformatics"],"description":"Shiny app for interactive variant prioritisation in\nprecision oncology. The input file for CVE is the output file\nof the recently released Oncotator Variant Annotation tool\nsummarising variant-centric information from 14 different\npublicly available resources relevant for cancer researches.\nInteractive priortisation in CVE is based on known germline and\ncancer variants, DNA repair genes and functional prediction\nscores. An optional feature of CVE is the exploration of the\ntumour-specific pathway context that is facilitated using\nco-expression modules generated from publicly available\ntranscriptome data. Finally druggability of prioritised\nvariants is assessed using the Drug Gene Interaction Database\n(DGIdb).","downloads_month":142,"downloads_total":8318,"page":"http://bioconductor.org/packages/release/bioc/html/CVE.html"},{"authors":"Matthias Futschik <mfutschik@ualg.pt>","name":"cycle","license":"GPL-2","tags":["Microarray","TimeCourse"],"description":"Package for assessing the statistical significance of\nperiodic expression based on Fourier analysis and comparison\nwith data generated by different background models","downloads_month":173,"downloads_total":41540,"page":"http://bioconductor.org/packages/release/bioc/html/cycle.html"},{"authors":"Aaron Lun [aut and cre]","name":"cydar","license":"GPL-3","tags":["ImmunoOncology","FlowCytometry","MultipleComparison","Proteomics","SingleCell"],"description":"Identifies differentially abundant populations between\nsamples and groups in mass cytometry data. Provides methods for\ncounting cells into hyperspheres, controlling the spatial false\ndiscovery rate, and visualizing changes in abundance in the\nhigh-dimensional marker space.","downloads_month":259,"downloads_total":7174,"page":"http://bioconductor.org/packages/release/bioc/html/cydar.html"},{"authors":"Zicheng Hu","name":"CytoDx","license":"GPL-2","tags":["ImmunoOncology","CellBiology","FlowCytometry","StatisticalMethod","Software","CellBasedAssays","Regression","Classification","Survival"],"description":"This package provides functions that predict clinical\noutcomes using single cell data (such as flow cytometry data,\nRNA single cell sequencing data) without the requirement of\ncell gating or clustering.","downloads_month":147,"downloads_total":2128,"page":"http://bioconductor.org/packages/release/bioc/html/CytoDx.html"},{"authors":"K.A. Stam [aut, cre] and G. Beyrend [aut]","name":"cytofast","license":"GPL-3","tags":["FlowCytometry","Visualization","Clustering"],"description":"Multi-parametric flow and mass cytometry allows\nexceptional high-resolution exploration of the cellular\ncomposition of the immune system. Together with tools like\nFlowSOM and Cytosplore it is possible to identify novel cell\ntypes. By introducing cytofast we hope to offer a workflow for\nvisualization and quantification of cell clusters for an\nefficient discovery of cell populations associated with\ndiseases or other clinical outcomes.","downloads_month":0,"downloads_total":74,"page":"http://bioconductor.org/packages/release/bioc/html/cytofast.html"},{"authors":"Mike Jiang","name":"cytolib","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","DataImport","Preprocessing","DataRepresentation"],"description":"This package provides the core data structure and API to\nrepresent and interact with the gated cytometry data.","downloads_month":1110,"downloads_total":25982,"page":"http://bioconductor.org/packages/release/bioc/html/cytolib.html"},{"authors":"Mike Jiang","name":"CytoML","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","DataImport","DataRepresentation"],"description":"Uses platform-specific implemenations of the GatingML2.0\nstandard to exchange gated cytometry data with other software\nplatforms.","downloads_month":230,"downloads_total":9158,"page":"http://bioconductor.org/packages/release/bioc/html/CytoML.html"},{"authors":"Benjamin Callahan <benjamin.j.callahan@gmail.com>, Paul\nMcMurdie and Susan Holmes","name":"dada2","license":"LGPL-3","tags":["ImmunoOncology","Microbiome","Sequencing","Classification","Metagenomics"],"description":"The dada2 package infers exact amplicon sequence variants\n(ASVs) from high-throughput amplicon sequencing data, replacing\nthe coarser and less accurate OTU clustering approach. The\ndada2 pipeline takes as input demultiplexed fastq files, and\noutputs the sequence variants and their sample-wise abundances\nafter removing substitution and chimera errors. Taxonomic\nclassification is available via a native implementation of the\nRDP naive Bayesian classifier, and species-level assignment to\n16S rRNA gene fragments by exact matching.","downloads_month":3201,"downloads_total":98246,"page":"http://bioconductor.org/packages/release/bioc/html/dada2.html"},{"authors":"Jianhong Ou, Alexey Stukalov, Niraj Nirala, Usha Acharya and Lihua\nJulie Zhu","name":"dagLogo","license":"GPL (>=2)","tags":["SequenceMatching","Visualization"],"description":"Visualize significant conserved amino acid sequence\npattern in groups based on probability theory.","downloads_month":140,"downloads_total":22532,"page":"http://bioconductor.org/packages/release/bioc/html/dagLogo.html"},{"authors":"Jobst Landgrebe <jlandgr1@gwdg.de> and Frank Bretz\n<bretz@bioinf.uni-hannover.de>","name":"daMA","license":"GPL (>= 2)","tags":["Microarray","TwoChannel","DifferentialExpression"],"description":"This package contains functions for the efficient design\nof factorial two-colour microarray experiments and for the\nstatistical analysis of factorial microarray data. Statistical\ndetails are described in Bretz et al. (2003, submitted)","downloads_month":139,"downloads_total":45682,"page":"http://bioconductor.org/packages/release/bioc/html/daMA.html"},{"authors":"Mattia Chiesa <mattia.chiesa@hotmail.it> and Luca Piacentini\n<luca.piacentini@cardiologicomonzino.it>","name":"DaMiRseq","license":"GPL (>= 2)","tags":["Sequencing","RNASeq","Classification","ImmunoOncology"],"description":"The DaMiRseq package offers a tidy pipeline of data mining\nprocedures to identify transcriptional biomarkers and exploit\nthem for both binary and multi-class classification purposes.\nThe package accepts any kind of data presented as a table of\nraw counts and allows including both continous and factorial\nvariables that occur with the experimental setting. A series of\nfunctions enable the user to clean up the data by filtering\ngenomic features and samples, to adjust data by identifying and\nremoving the unwanted source of variation (i.e. batches and\nconfounding factors) and to select the best predictors for\nmodeling. Finally, a \"stacking\" ensemble learning technique is\napplied to build a robust classification model. Every step\nincludes a checkpoint that the user may exploit to assess the\neffects of data management by looking at diagnostic plots, such\nas clustering and heatmaps, RLE boxplots, MDS or correlation\nplot.","downloads_month":178,"downloads_total":6518,"page":"http://bioconductor.org/packages/release/bioc/html/DaMiRseq.html"},{"authors":"Samuel Wieczorek [cre, aut], Florence Combes [aut], Thomas\nBurger [aut], Cosmin Lazar [ctb] and Alexia Dorffer [ctb]","name":"DAPAR","license":"Artistic-2.0","tags":["Proteomics","Normalization","Preprocessing","MassSpectrometry","QualityControl","GO","DataImport"],"description":"This package contains a collection of functions for the\nvisualisation and the statistical analysis of proteomic data.","downloads_month":238,"downloads_total":17168,"page":"http://bioconductor.org/packages/release/bioc/html/DAPAR.html"},{"authors":"Yan Jiao, Katherine Lawler, Andrew E Teschendorff and Charles\nShijie Zheng","name":"DART","license":"GPL-2","tags":["GeneExpression","DifferentialExpression","GraphAndNetwork","Pathways"],"description":"Denoising Algorithm based on Relevance network Topology\n(DART) is an algorithm designed to evaluate the consistency of\nprior information molecular signatures (e.g in-vitro\nperturbation expression signatures) in independent molecular\ndata (e.g gene expression data sets). If consistent, a pruning\nnetwork strategy is then used to infer the activation status of\nthe molecular signature in individual samples.","downloads_month":159,"downloads_total":33914,"page":"http://bioconductor.org/packages/release/bioc/html/DART.html"},{"authors":"Kun Liang","name":"DBChIP","license":"GPL (>= 2)","tags":["ChIPSeq","Sequencing","Transcription","Genetics"],"description":"DBChIP detects differentially bound sharp binding sites\nacross multiple conditions, with or without matching control\nsamples.","downloads_month":194,"downloads_total":35910,"page":"http://bioconductor.org/packages/release/bioc/html/DBChIP.html"},{"authors":"Jiehuan Sun [aut, cre], Jose Herazo-Maya [aut], Xiu Huang\n[aut], Naftali Kaminski [aut], and Hongyu Zhao [aut]","name":"dcGSA","license":"GPL-2","tags":["ImmunoOncology","GeneSetEnrichment","Microarray","StatisticalMethod","Sequencing","RNASeq","GeneExpression"],"description":"Distance-correlation based Gene Set Analysis for\nlongitudinal gene expression profiles. In longitudinal studies,\nthe gene expression profiles were collected at each visit from\neach subject and hence there are multiple measurements of the\ngene expression profiles for each subject. The dcGSA package\ncould be used to assess the associations between gene sets and\nclinical outcomes of interest by fully taking advantage of the\nlongitudinal nature of both the gene expression profiles and\nclinical outcomes.","downloads_month":173,"downloads_total":8418,"page":"http://bioconductor.org/packages/release/bioc/html/dcGSA.html"},{"authors":"Bernd Klaus [aut, cre], Christophe Chabbert [aut] and Sebastian\nGibb [ctb]","name":"DChIPRep","license":"MIT + file LICENCE","tags":["Sequencing","ChIPSeq","WholeGenome"],"description":"The DChIPRep package implements a methodology to assess\ndifferences between chromatin modification profiles in\nreplicated ChIP-Seq studies as described in Chabbert et. al -\nhttp://www.dx.doi.org/10.15252/msb.20145776. A detailed\ndescription of the method is given in the software paper at\nhttps://doi.org/10.7717/peerj.1981","downloads_month":152,"downloads_total":11588,"page":"http://bioconductor.org/packages/release/bioc/html/DChIPRep.html"},{"authors":"Jitao David Zhang, Rudolf Biczok, and Markus Ruschhaupt","name":"ddCt","license":"LGPL-3","tags":["GeneExpression","DifferentialExpression","MicrotitrePlateAssay","qPCR"],"description":"The Delta-Delta-Ct (ddCt) Algorithm is an approximation\nmethod to determine relative gene expression with quantitative\nreal-time PCR (qRT-PCR) experiments. Compared to other\napproaches, it requires no standard curve for each\nprimer-target pair, therefore reducing the working load and yet\nreturning accurate enough results as long as the assumptions of\nthe amplification efficiency hold. The ddCt package implements\na pipeline to collect, analyse and visualize qRT-PCR results,\nfor example those from TaqMan SDM software, mainly using the\nddCt method. The pipeline can be either invoked by a script in\ncommand-line or through the API consisting of S4-Classes,\nmethods and functions.","downloads_month":368,"downloads_total":55568,"page":"http://bioconductor.org/packages/release/bioc/html/ddCt.html"},{"authors":"Benedikt G. Brink [aut, cre], Justin Meskas [ctb] and Ryan R.\nBrinkman [ctb]","name":"ddPCRclust","license":"Artistic-2.0","tags":["ddPCR","Clustering"],"description":"The ddPCRclust algorithm can automatically quantify the\nCPDs of non-orthogonal ddPCR reactions with up to four targets.\nIn order to determine the correct droplet count for each\ntarget, it is crucial to both identify all clusters and label\nthem correctly based on their position. For more information on\nwhat data can be analyzed and how a template needs to be\nformatted, please check the vignette.","downloads_month":123,"downloads_total":2150,"page":"http://bioconductor.org/packages/release/bioc/html/ddPCRclust.html"},{"authors":"Alper Kucukural <alper.kucukural@umassmed.edu>, Onur Yukselen\n<onur.yukselen@umassmed.edu> and Manuel Garber\n<manuel.garber@umassmed.edu>","name":"debrowser","license":"GPL-3 + file LICENSE","tags":["Sequencing","ChIPSeq","RNASeq","DifferentialExpression","GeneExpression","Clustering","ImmunoOncology"],"description":"Bioinformatics platform containing interactive plots and\ntables for differential gene and region expression studies.\nAllows visualizing expression data much more deeply in an\ninteractive and faster way. By changing the parameters, users\ncan easily discover different parts of the data that like never\nhave been done before. Manually creating and looking these\nplots takes time. With DEBrowser users can prepare plots\nwithout writing any code. Differential expression, PCA and\nclustering analysis are made on site and the results are shown\nin various plots such as scatter, bar, box, volcano, ma plots\nand Heatmaps.","downloads_month":821,"downloads_total":18194,"page":"http://bioconductor.org/packages/release/bioc/html/debrowser.html"},{"authors":"Erik Wright","name":"DECIPHER","license":"GPL-3","tags":["ImmunoOncology","Clustering","Genetics","Sequencing","DataImport","Visualization","Microarray","QualityControl","qPCR","Alignment","WholeGenome","Microbiome"],"description":"A toolset for deciphering and managing biological\nsequences.","downloads_month":1630,"downloads_total":89450,"page":"http://bioconductor.org/packages/release/bioc/html/DECIPHER.html"},{"authors":"Francisco Jose Campos-Laborie, Jose Manuel Sanchez-Santos and\nJavier De Las Rivas. Bioinformatics, Functional Genomics\nGroup. Cancer Research Center (CiC-IBMCC and CSIC/USAL).\nSalamanca. Spain.","name":"deco","license":"GPL (>=3)","tags":["Software","FeatureExtraction","Clustering","MultipleComparison","DifferentialExpression","Transcriptomics","BiomedicalInformatics","Proteomics","Bayesian","GeneExpression","Transcription","Sequencing","Microarray","ExonArray","RNASeq","MicroRNAArray","mRNAMicroarray"],"description":"This package discovers differential features in hetero-\nand homogeneous omic data by a two-step method including\nsubsampling LIMMA and NSCA. DECO reveals feature associations\nto hidden subclasses not exclusively related to higher\nderegulation levels.","downloads_month":42,"downloads_total":160,"page":"http://bioconductor.org/packages/release/bioc/html/deco.html"},{"authors":"Guofeng Meng","name":"DEComplexDisease","license":"GPL-3","tags":["DNASeq","WholeGenome","FunctionalGenomics","DifferentialExpression","GeneExpression","Clustering"],"description":"It is designed to find the differential expressed genes\n(DEGs) for complex disease, which is characterized by the\nheterogeneous genomic expression profiles. Different from the\nestablished DEG analysis tools, it does not assume the patients\nof complex diseases to share the common DEGs. By applying a\nbi-clustering algorithm, DECD finds the DEGs shared by as many\npatients. In this way, DECD describes the DEGs of complex\ndisease in a novel syntax, e.g. a gene list composed of 200\ngenes are differentially expressed in 30% percent of studied\ncomplex disease. Applying the DECD analysis results, users are\npossible to find the patients affected by the same mechanism\nbased on the shared signatures.","downloads_month":128,"downloads_total":2236,"page":"http://bioconductor.org/packages/release/bioc/html/DEComplexDisease.html"},{"authors":"Rosario M. Piro [aut, cre] and Sandra Krueger [ctb]","name":"decompTumor2Sig","license":"GPL-2","tags":["Software","SNP","Sequencing","DNASeq","GenomicVariation","SomaticMutation","BiomedicalInformatics","Genetics","BiologicalQuestion","StatisticalMethod"],"description":"Uses quadratic programming for signature refitting, i.e.,\nto decompose the mutation catalog from an individual tumor\nsample into a set of given mutational signatures (either\nAlexandrov-model signatures or Shiraishi-model signatures),\ncomputing weights that reflect the contributions of the\nsignatures to the mutation load of the tumor.","downloads_month":26,"downloads_total":208,"page":"http://bioconductor.org/packages/release/bioc/html/decompTumor2Sig.html"},{"authors":"Ting Gong <tinggong@gmail.com> Joseph D. Szustakowski\n<joseph.szustakowski@novartis.com>","name":"DeconRNASeq","license":"GPL-2","tags":["DifferentialExpression"],"description":"DeconSeq is an R package for deconvolution of\nheterogeneous tissues based on mRNA-Seq data. It modeled\nexpression levels from heterogeneous cell populations in\nmRNA-Seq as the weighted average of expression from different\nconstituting cell types and predicted cell type proportions of\nsingle expression profiles.","downloads_month":276,"downloads_total":33594,"page":"http://bioconductor.org/packages/release/bioc/html/DeconRNASeq.html"},{"authors":"Benjamin Callahan <benjamin.j.callahan@gmail.com> and Nicole Marie\nDavis","name":"decontam","license":"Artistic-2.0","tags":["ImmunoOncology","Microbiome","Sequencing","Classification","Metagenomics"],"description":"Simple statistical identification of contaminating\nsequence features in marker-gene or metagenomics data. Works on\nany kind of feature derived from environmental sequencing data\n(e.g. ASVs, OTUs, taxonomic groups, MAGs,...). Requires DNA\nquantitation data or sequenced negative control samples.","downloads_month":218,"downloads_total":3030,"page":"http://bioconductor.org/packages/release/bioc/html/decontam.html"},{"authors":"Yuanyuan Xiao <yxiao@itsa.ucsf.edu> and Jean Yee Hwa Yang\n<jeany@maths.usyd.edu.au>.","name":"DEDS","license":"LGPL","tags":["Microarray","DifferentialExpression"],"description":"This library contains functions that calculate various\nstatistics of differential expression for microarray data,\nincluding t statistics, fold change, F statistics, SAM,\nmoderated t and F statistics and B statistics. It also\nimplements a new methodology called DEDS (Differential\nExpression via Distance Summary), which selects differentially\nexpressed genes by integrating and summarizing a set of\nstatistics using a weighted distance approach.","downloads_month":185,"downloads_total":56880,"page":"http://bioconductor.org/packages/release/bioc/html/DEDS.html"},{"authors":"Felipe Albrecht and Markus List","name":"DeepBlueR","license":"GPL (>=2.0)","tags":["DataImport","DataRepresentation","ThirdPartyClient","GeneRegulation","GenomeAnnotation","CpGIsland","DNAMethylation","Epigenetics","Annotation","Preprocessing"],"description":"Accessing the DeepBlue Epigenetics Data Server through R.","downloads_month":218,"downloads_total":9842,"page":"http://bioconductor.org/packages/release/bioc/html/DeepBlueR.html"},{"authors":"Niko Beerenwinkel [ths], David Jones [ctb], Inigo Martincorena\n[ctb], Moritz Gerstung [aut and cre]","name":"deepSNV","license":"GPL-3","tags":["GeneticVariability","SNP","Sequencing","Genetics","DataImport"],"description":"This package provides provides quantitative variant\ncallers for detecting subclonal mutations in ultra-deep (>=100x\ncoverage) sequencing experiments. The deepSNV algorithm is used\nfor a comparative setup with a control experiment of the same\nloci and uses a beta-binomial model and a likelihood ratio test\nto discriminate sequencing errors and subclonal SNVs. The\nshearwater algorithm computes a Bayes classifier based on a\nbeta-binomial model for variant calling with multiple samples\nfor precisely estimating model parameters - such as local error\nrates and dispersion - and prior knowledge, e.g. from variation\ndata bases such as COSMIC.","downloads_month":209,"downloads_total":42242,"page":"http://bioconductor.org/packages/release/bioc/html/deepSNV.html"},{"authors":"Andrzej Oleś","name":"DEFormats","license":"GPL-3","tags":["ImmunoOncology","DifferentialExpression","GeneExpression","RNASeq","Sequencing","Transcription"],"description":"Convert between different data formats used by\ndifferential gene expression analysis tools.","downloads_month":432,"downloads_total":17218,"page":"http://bioconductor.org/packages/release/bioc/html/DEFormats.html"},{"authors":"Laurent Jacob, Pierre Neuvial and Sandrine Dudoit","name":"DEGraph","license":"GPL-3","tags":["Microarray","DifferentialExpression","GraphAndNetwork","Network","NetworkEnrichment","DecisionTree"],"description":"DEGraph implements recent hypothesis testing methods which\ndirectly assess whether a particular gene network is\ndifferentially expressed between two conditions. This is to be\ncontrasted with the more classical two-step approaches which\nfirst test individual genes, then test gene sets for enrichment\nin differentially expressed genes. These recent methods take\ninto account the topology of the network to yield more powerful\ndetection procedures. DEGraph provides methods to easily test\nall KEGG pathways for differential expression on any gene\nexpression data set and tools to visualize the results.","downloads_month":120,"downloads_total":42024,"page":"http://bioconductor.org/packages/release/bioc/html/DEGraph.html"},{"authors":"Lorena Pantano [aut, cre], John Hutchinson [ctb], Victor\nBarrera [ctb], Mary Piper [ctb], Radhika Khetani [ctb], Kenneth\nDaily [ctb], Thanneer Malai Perumal [ctb], Rory Kirchner [ctb] and Michael Steinbaugh [ctb]","name":"DEGreport","license":"MIT + file LICENSE","tags":["DifferentialExpression","Visualization","RNASeq","ReportWriting","GeneExpression","ImmunoOncology"],"description":"Creation of a HTML report of differential expression\nanalyses of count data. It integrates some of the code\nmentioned in DESeq2 and edgeR vignettes, and report a ranked\nlist of genes according to the fold changes mean and\nvariability for each selected gene.","downloads_month":560,"downloads_total":30010,"page":"http://bioconductor.org/packages/release/bioc/html/DEGreport.html"},{"authors":"Likun Wang <wanglk@hsc.pku.edu.cn> and Xi Wang\n<wang-xi05@mails.tsinghua.edu.cn>.","name":"DEGseq","license":"LGPL (>=2)","tags":["RNASeq","Preprocessing","GeneExpression","DifferentialExpression","ImmunoOncology"],"description":"DEGseq is an R package to identify differentially\nexpressed genes from RNA-Seq data.","downloads_month":456,"downloads_total":157008,"page":"http://bioconductor.org/packages/release/bioc/html/DEGseq.html"},{"authors":"Hervé Pagès <hpages@fredhutch.org>, with contributions from\nPeter Hickey <peter.hickey@gmail.com> and Aaron Lun\n<infinite.monkeys.with.keyboards@gmail.com>","name":"DelayedArray","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation","Annotation","GenomeAnnotation"],"description":"Wrapping an array-like object (typically an on-disk\nobject) in a DelayedArray object allows one to perform common\narray operations on it without loading the object in memory. In\norder to reduce memory usage and optimize performance,\noperations on the object are either delayed or executed using a\nblock processing mechanism. Note that this also works on\nin-memory array-like objects like DataFrame objects (typically\nwith Rle columns), Matrix objects, and ordinary arrays and data\nframes.","downloads_month":27606,"downloads_total":923620,"page":"http://bioconductor.org/packages/release/bioc/html/DelayedArray.html"},{"authors":"Qian Liu [aut, cre], Hervé Pagès [ctb] and Martin Morgan [ctb]","name":"DelayedDataFrame","license":"GPL-3","tags":["Infrastructure","DataRepresentation"],"description":"Based on the standard DataFrame metaphor, we are trying to\nimplement the feature of delayed operation on the\nDelayedDataFrame, with a slot of lazyIndex, which saves the\nmapping indexes for each column of DelayedDataFrame. Methods\nlike show, validity check, [/[[ subsetting, rbind/cbind are\nimplemented for DelayedDataFrame to be operated around\nlazyIndex. The listData slot stays untouched until a\nrealization call e.g., DataFrame constructor OR as.list() is\ninvoked.","downloads_month":34,"downloads_total":166,"page":"http://bioconductor.org/packages/release/bioc/html/DelayedDataFrame.html"},{"authors":"Peter Hickey <peter.hickey@gmail.com>","name":"DelayedMatrixStats","license":"MIT + file LICENSE","tags":["Infrastructure","DataRepresentation","Software"],"description":"A port of the 'matrixStats' API for use with DelayedMatrix\nobjects from the 'DelayedArray' package. High-performing\nfunctions operating on rows and columns of DelayedMatrix\nobjects, e.g. col / rowMedians(), col / rowRanks(), and col /\nrowSds(). Functions optimized per data type and for subsetted\ncalculations such that both memory usage and processing time is\nminimized.","downloads_month":4528,"downloads_total":78548,"page":"http://bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html"},{"authors":"Diana Low and Efthymios Motakis","name":"deltaGseg","license":"GPL-2","tags":["Proteomics","TimeCourse","Visualization","Clustering"],"description":"Identifying distinct subpopulations through multiscale\ntime series analysis","downloads_month":143,"downloads_total":26076,"page":"http://bioconductor.org/packages/release/bioc/html/deltaGseg.html"},{"authors":"Jung Hoon Woo <jw2853@columbia.edu> and Yishai Shimoni\n<ys2559@columbia.edu>","name":"DeMAND","license":"file LICENSE","tags":["SystemsBiology","NetworkEnrichment","GeneExpression","StatisticalMethod","Network"],"description":"DEMAND predicts Drug MoA by interrogating a cell context\nspecific regulatory network with a small number (N >= 6) of\ncompound-induced gene expression signatures, to elucidate\nspecific proteins whose interactions in the network is\ndysregulated by the compound.","downloads_month":151,"downloads_total":11654,"page":"http://bioconductor.org/packages/release/bioc/html/DeMAND.html"},{"authors":"Arne Smits [cre, aut] and Wolfgang Huber [aut]","name":"DEP","license":"Artistic-2.0","tags":["ImmunoOncology","Proteomics","MassSpectrometry","DifferentialExpression","DataRepresentation"],"description":"This package provides an integrated analysis workflow for\nrobust and reproducible analysis of mass spectrometry\nproteomics data for differential protein expression or\ndifferential enrichment. It requires tabular input (e.g. txt\nfiles) as generated by quantitative analysis softwares of raw\nmass spectrometry data, such as MaxQuant or IsobarQuant.\nFunctions are provided for data preparation, filtering,\nvariance normalization and imputation of missing values, as\nwell as statistical testing of differentially enriched /\nexpressed proteins. It also includes tools to check\nintermediate steps in the workflow, such as normalization and\nmissing values imputation. Finally, visualization tools are\nprovided to explore the results, including heatmap, volcano\nplot and barplot representations. For scientists with limited\nexperience in R, the package also contains wrapper functions\nthat entail the complete analysis workflow and generate a\nreport. Even easier to use are the interactive Shiny apps that\nare provided by the package.","downloads_month":537,"downloads_total":8972,"page":"http://bioconductor.org/packages/release/bioc/html/DEP.html"},{"authors":"Yafeng Zhu","name":"DEqMS","license":"LGPL","tags":["ImmunoOncology","Proteomics","MassSpectrometry","Preprocessing","DifferentialExpression","MultipleComparison","Normalization","Bayesian"],"description":"DEqMS is developped on top of Limma. However, Limma\nassumes same prior variance for all genes. In proteomics, the\naccuracy of protein abundance estimates varies by the number of\npeptides/PSMs quantified in both label-free and labelled data.\nProteins quantification by multiple peptides or PSMs are more\naccurate. DEqMS package is able to estimate different prior\nvariances for proteins quantified by different number of\nPSMs/peptides, therefore acchieving better accuracy. The\npackage can be applied to analyze both label-free and labelled\nproteomics data.","downloads_month":149,"downloads_total":1266,"page":"http://bioconductor.org/packages/release/bioc/html/DEqMS.html"},{"authors":"Leonardo Collado-Torres [aut, cre], Alyssa C. Frazee [ctb], Andrew E. Jaffe [aut], Jeffrey T. Leek [aut and ths]","name":"derfinder","license":"Artistic-2.0","tags":["DifferentialExpression","Sequencing","RNASeq","ChIPSeq","DifferentialPeakCalling","Software","ImmunoOncology"],"description":"This package provides functions for annotation-agnostic\ndifferential expression analysis of RNA-seq data. Two\nimplementations of the DER Finder approach are included in this\npackage: (1) single base-level F-statistics and (2) DER\nidentification at the expressed regions-level. The DER Finder\napproach can also be used to identify differentially bounded\nChIP-seq peaks.","downloads_month":537,"downloads_total":43192,"page":"http://bioconductor.org/packages/release/bioc/html/derfinder.html"},{"authors":"Leonardo Collado-Torres [aut, cre], Andrew E. Jaffe [aut], Jeffrey T. Leek [aut and ths]","name":"derfinderHelper","license":"Artistic-2.0","tags":["DifferentialExpression","Sequencing","RNASeq","Software","ImmunoOncology"],"description":"Helper package for speeding up the derfinder package when\nusing multiple cores.","downloads_month":505,"downloads_total":35626,"page":"http://bioconductor.org/packages/release/bioc/html/derfinderHelper.html"},{"authors":"Leonardo Collado-Torres [aut, cre], Andrew E. Jaffe [aut], Jeffrey T. Leek [aut and ths]","name":"derfinderPlot","license":"Artistic-2.0","tags":["DifferentialExpression","Sequencing","RNASeq","Software","Visualization","ImmunoOncology"],"description":"This package provides plotting functions for results from\nthe derfinder package.","downloads_month":179,"downloads_total":20754,"page":"http://bioconductor.org/packages/release/bioc/html/derfinderPlot.html"},{"authors":"Dario Righelli [aut, cre], John Koberstein [aut], Bruce Gomes\n[aut], Nancy Zhang [aut], Claudia Angelini [aut], Lucia Peixoto\n[aut] and Davide Risso [aut]","name":"DEScan2","license":"Artistic-2.0","tags":["ImmunoOncology","PeakDetection","Epigenetics","Software","Sequencing","Coverage"],"description":"Integrated peak and differential caller, specifically\ndesigned for broad epigenomic signals.","downloads_month":185,"downloads_total":2458,"page":"http://bioconductor.org/packages/release/bioc/html/DEScan2.html"},{"authors":"Simon Anders and EMBL Heidelberg <sanders@fs.tum.de>","name":"DESeq","license":"GPL (>= 3)","tags":["ImmunoOncology","Sequencing","ChIPSeq","RNASeq","SAGE","DifferentialExpression"],"description":"Estimate variance-mean dependence in count data from\nhigh-throughput sequencing assays and test for differential\nexpression based on a model using the negative binomial\ndistribution","downloads_month":7756,"downloads_total":955910,"page":"http://bioconductor.org/packages/release/bioc/html/DESeq.html"},{"authors":"Michael Love, Simon Anders and Wolfgang Huber","name":"DESeq2","license":"LGPL (>= 3)","tags":["Sequencing","RNASeq","ChIPSeq","GeneExpression","Transcription","Normalization","DifferentialExpression","Bayesian","Regression","PrincipalComponent","Clustering","ImmunoOncology"],"description":"Estimate variance-mean dependence in count data from\nhigh-throughput sequencing assays and test for differential\nexpression based on a model using the negative binomial\ndistribution.","downloads_month":25234,"downloads_total":1625550,"page":"http://bioconductor.org/packages/release/bioc/html/DESeq2.html"},{"authors":"Zhun Miao <miaoz13@mails.tsinghua.edu.cn>","name":"DEsingle","license":"GPL-2","tags":["DifferentialExpression","GeneExpression","SingleCell","ImmunoOncology","RNASeq","Transcriptomics","Sequencing","Preprocessing","Software"],"description":"DEsingle is an R package for differential expression (DE)\nanalysis of single-cell RNA-seq (scRNA-seq) data. It defines\nand detects 3 types of differentially expressed genes between\ntwo groups of single cells, with regard to different expression\nstatus (DEs), differential expression abundance (DEa), and\ngeneral differential expression (DEg). DEsingle employs\nZero-Inflated Negative Binomial model to estimate the\nproportion of real and dropout zeros and to define and detect\nthe 3 types of DE genes. Results showed that DEsingle\noutperforms existing methods for scRNA-seq DE analysis, and can\nreveal different types of DE genes that are enriched in\ndifferent biological functions.","downloads_month":222,"downloads_total":3032,"page":"http://bioconductor.org/packages/release/bioc/html/DEsingle.html"},{"authors":"Philipp Angerer [cre, aut]\n(<https://orcid.org/0000-0002-0369-2888>), Laleh Haghverdi\n[ctb], Maren Büttner [ctb]\n(<https://orcid.org/0000-0002-6189-3792>), Fabian Theis [ctb]\n(<https://orcid.org/0000-0002-2419-1943>), Carsten Marr [ctb]\n(<https://orcid.org/0000-0003-2154-4552>) and Florian Büttner\n[ctb] (<https://orcid.org/0000-0001-5587-6761>)","name":"destiny","license":"GPL","tags":["CellBiology","CellBasedAssays","Clustering","Software","Visualization"],"description":"Create and plot diffusion maps.","downloads_month":1381,"downloads_total":79434,"page":"http://bioconductor.org/packages/release/bioc/html/destiny.html"},{"authors":"Aristidis G. Vrahatis and Panos Balomenos","name":"DEsubs","license":"GPL-3","tags":["SystemsBiology","GraphAndNetwork","Pathways","KEGG","GeneExpression","NetworkEnrichment","Network","RNASeq","DifferentialExpression","Normalization","ImmunoOncology"],"description":"DEsubs is a network-based systems biology package that\nextracts disease-perturbed subpathways within a pathway network\nas recorded by RNA-seq experiments. It contains an extensive\nand customizable framework covering a broad range of operation\nmodes at all stages of the subpathway analysis, enabling a\ncase-specific approach. The operation modes refer to the\npathway network construction and processing, the subpathway\nextraction, visualization and enrichment analysis with regard\nto various biological and pharmacological features. Its\ncapabilities render it a tool-guide for both the modeler and\nexperimentalist for the identification of more robust\nsystems-level biomarkers for complex diseases.","downloads_month":165,"downloads_total":8702,"page":"http://bioconductor.org/packages/release/bioc/html/DEsubs.html"},{"authors":"Simon Anders <sanders@fs.tum.de> and Alejandro Reyes\n<alejandro.reyes.ds@gmail.com>","name":"DEXSeq","license":"GPL (>= 3)","tags":["ImmunoOncology","Sequencing","RNASeq","DifferentialExpression","AlternativeSplicing","DifferentialSplicing","GeneExpression","Visualization"],"description":"The package is focused on finding differential exon usage\nusing RNA-seq exon counts between samples with different\nexperimental designs. It provides functions that allows the\nuser to make the necessary statistical tests based on a model\nthat uses the negative binomial distribution to estimate the\nvariance between biological replicates and generalized linear\nmodels for testing. The package also provides functions for the\nvisualization and exploration of the results.","downloads_month":1504,"downloads_total":234994,"page":"http://bioconductor.org/packages/release/bioc/html/DEXSeq.html"},{"authors":"Guenter Klambauer","name":"dexus","license":"LGPL (>= 2.0)","tags":["ImmunoOncology","Sequencing","RNASeq","GeneExpression","DifferentialExpression","CellBiology","Classification","QualityControl"],"description":"DEXUS identifies differentially expressed genes in RNA-Seq\ndata under all possible study designs such as studies without\nreplicates, without sample groups, and with unknown conditions.\nDEXUS works also for known conditions, for example for RNA-Seq\ndata with two or multiple conditions. RNA-Seq read count data\ncan be provided both by the S4 class Count Data Set and by read\ncount matrices. Differentially expressed transcripts can be\nvisualized by heatmaps, in which unknown conditions,\nreplicates, and samples groups are also indicated. This\nsoftware is fast since the core algorithm is written in C. For\nvery large data sets, a parallel version of DEXUS is provided\nin this package. DEXUS is a statistical model that is selected\nin a Bayesian framework by an EM algorithm. DEXUS does not need\nreplicates to detect differentially expressed transcripts,\nsince the replicates (or conditions) are estimated by the EM\nmethod for each transcript. The method provides an\ninformative/non-informative value to extract differentially\nexpressed transcripts at a desired significance level or power.","downloads_month":195,"downloads_total":28966,"page":"http://bioconductor.org/packages/release/bioc/html/dexus.html"},{"authors":"R. Alvarez-Gonzalez, D. Glez-Pena, F. Diaz and F. Fdez-Riverola","name":"DFP","license":"GPL-2","tags":["Microarray","DifferentialExpression"],"description":"This package provides a supervised technique able to\nidentify differentially expressed genes, based on the\nconstruction of \\emph{Fuzzy Patterns} (FPs). The Fuzzy Patterns\nare built by means of applying 3 Membership Functions to\ndiscretized gene expression values.","downloads_month":144,"downloads_total":44460,"page":"http://bioconductor.org/packages/release/bioc/html/DFP.html"},{"authors":"Rory Stark<rory.stark@cruk.cam.ac.uk> and Gord Brown\n<gdbzork@gmail.com>","name":"DiffBind","license":"Artistic-2.0","tags":["Sequencing","ChIPSeq","DifferentialPeakCalling"],"description":"Compute differentially bound sites from multiple ChIP-seq\nexperiments using affinity (quantitative) data. Also enables\noccupancy (overlap) analysis and plotting functions.","downloads_month":1262,"downloads_total":152490,"page":"http://bioconductor.org/packages/release/bioc/html/DiffBind.html"},{"authors":"Wenbin Wei, Sandeep Amberkar and Winston Hide","name":"diffcoexp","license":"GPL (>2)","tags":["GeneExpression","DifferentialExpression","Transcription","Microarray","OneChannel","TwoChannel","RNASeq","Sequencing","Coverage","ImmunoOncology"],"description":"A tool for the identification of differentially\ncoexpressed links (DCLs) and differentially coexpressed genes\n(DCGs). DCLs are gene pairs with significantly different\ncorrelation coefficients under two conditions. DCGs are genes\nwith significantly more DCLs than by chance.","downloads_month":133,"downloads_total":2252,"page":"http://bioconductor.org/packages/release/bioc/html/diffcoexp.html"},{"authors":"Lukas M. Weber [aut and cre]\n(<https://orcid.org/0000-0002-3282-1730>)","name":"diffcyt","license":"MIT + file LICENSE","tags":["ImmunoOncology","FlowCytometry","Proteomics","SingleCell","CellBasedAssays","CellBiology","Clustering","FeatureExtraction","Software"],"description":"Statistical methods for differential discovery analyses in\nhigh-dimensional cytometry data (including flow cytometry, mass\ncytometry or CyTOF, and oligonucleotide-tagged cytometry),\nbased on (i) high-resolution clustering and (ii) empirical\nBayes moderated tests adapted from transcriptomics.","downloads_month":278,"downloads_total":3764,"page":"http://bioconductor.org/packages/release/bioc/html/diffcyt.html"},{"authors":"Choudary Jagarlamudi","name":"diffGeneAnalysis","license":"GPL","tags":["Microarray","DifferentialExpression"],"description":"Analyze microarray data","downloads_month":133,"downloads_total":46692,"page":"http://bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html"},{"authors":"Aaron Lun [aut, cre] and Gordon Smyth [aut]","name":"diffHic","license":"GPL-3","tags":["MultipleComparison","Preprocessing","Sequencing","Coverage","Alignment","Normalization","Clustering","HiC"],"description":"Detects differential interactions across biological\nconditions in a Hi-C experiment. Methods are provided for read\nalignment and data pre-processing into interaction counts.\nStatistical analysis is based on edgeR and supports\nnormalization and filtering. Several visualization options are\nalso available.","downloads_month":240,"downloads_total":20256,"page":"http://bioconductor.org/packages/release/bioc/html/diffHic.html"},{"authors":"c( person(\"Martin\", \"Nettling\", role = c(\"aut\", \"cre\"), email =\n\"martin.nettling@informatik.uni-halle.de\"), person(\"Hendrik\", \"Treutler\", role = c(\"aut\", \"cre\"), email =\n\"hendrik.treutler@ipb-halle.de\"), person(\"Jan\", \"Grau\", role =\nc(\"aut\", \"ctb\"), email = \"grau@informatik.uni-halle.de\"), person(\"Andrey\", \"Lando\", role = c(\"aut\", \"ctb\"), email =\n\"dronte@autosome.ru\"), person(\"Jens\", \"Keilwagen\", role =\nc(\"aut\", \"ctb\"), email = \"jens.keilwagen@julius-kuehn.de\"), person(\"Stefan\", \"Posch\", role = \"aut\", email =\n\"posch@informatik.uni-halle.de\"), person(\"Ivo\", \"Grosse\", role\n= \"aut\" and email = \"grosse@informatik.uni-halle.de\"))","name":"DiffLogo","license":"GPL (>= 2)","tags":["Software","SequenceMatching","MultipleComparison","MotifAnnotation","Visualization","Alignment"],"description":"DiffLogo is an easy-to-use tool to visualize motif\ndifferences.","downloads_month":144,"downloads_total":11048,"page":"http://bioconductor.org/packages/release/bioc/html/DiffLogo.html"},{"authors":"Caleb Lareau [aut, cre] and Martin Aryee [aut]","name":"diffloop","license":"MIT + file LICENSE","tags":["Preprocessing","QualityControl","Visualization","DataImport","DataRepresentation","GO"],"description":"A suite of tools for subsetting, visualizing, annotating,\nand statistically analyzing the results of one or more ChIA-PET\nexperiments or other assays that infer chromatin loops.","downloads_month":199,"downloads_total":10354,"page":"http://bioconductor.org/packages/release/bioc/html/diffloop.html"},{"authors":"Sergio Picart-Armada and Alexandre Perera-Lluna","name":"diffuStats","license":"GPL-3","tags":["Network","GeneExpression","GraphAndNetwork"],"description":"Label propagation approaches are a widely used procedure\nin computational biology for giving context to molecular\nentities using network data. Node labels, which can derive from\ngene expression, genome-wide association studies, protein\ndomains or metabolomics profiling, are propagated to their\nneighbours in the network, effectively smoothing the scores\nthrough prior annotated knowledge and prioritising novel\ncandidates. The R package diffuStats contains a collection of\ndiffusion kernels and scoring approaches that facilitates their\ncomputation and benchmarking.","downloads_month":148,"downloads_total":3918,"page":"http://bioconductor.org/packages/release/bioc/html/diffuStats.html"},{"authors":"Mariano J Alvarez <reef103@gmail.com>","name":"diggit","license":"file LICENSE","tags":["SystemsBiology","NetworkEnrichment","GeneExpression","FunctionalPrediction","GeneRegulation"],"description":"Inference of Genetic Variants Driving Cellullar Phenotypes\nby the DIGGIT algorithm","downloads_month":156,"downloads_total":13678,"page":"http://bioconductor.org/packages/release/bioc/html/diggit.html"},{"authors":"Katherine Icay [aut and cre]","name":"Director","license":"GPL-3 + file LICENSE","tags":["Visualization"],"description":"Director is an R package designed to streamline the\nvisualization of molecular effects in regulatory cascades. It\nutilizes the R package htmltools and a modified Sankey plugin\nof the JavaScript library D3 to provide a fast and easy,\nbrowser-enabled solution to discovering potentially interesting\ndownstream effects of regulatory and/or co-expressed molecules.\nThe diagrams are robust, interactive, and packaged as\nhighly-portable HTML files that eliminate the need for\nthird-party software to view. This enables a straightforward\napproach for scientists to interpret the data produced, and\nbioinformatics developers an alternative means to present\nrelevant data.","downloads_month":139,"downloads_total":6572,"page":"http://bioconductor.org/packages/release/bioc/html/Director.html"},{"authors":"Martin Morgan <martin.morgan@roswellpark.org>","name":"DirichletMultinomial","license":"LGPL-3","tags":["ImmunoOncology","Microbiome","Sequencing","Clustering","Classification","Metagenomics"],"description":"Dirichlet-multinomial mixture models can be used to\ndescribe variability in microbial metagenomic data. This\npackage is an interface to code originally made available by\nHolmes, Harris, and Quince, 2012, PLoS ONE 7(2): 1-15, as\ndiscussed further in the man page for this package,\n?DirichletMultinomial.","downloads_month":1162,"downloads_total":72000,"page":"http://bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html"},{"authors":"Charlotte Siska [cre, aut] and Katerina Kechris [aut]","name":"discordant","license":"GPL (>= 2)","tags":["ImmunoOncology","BiologicalQuestion","StatisticalMethod","mRNAMicroarray","Microarray","Genetics","RNASeq"],"description":"Discordant is a method to determine differential\ncorrelation of molecular feature pairs from -omics data using\nmixture models. Algorithm is explained further in Siska et al.","downloads_month":199,"downloads_total":5762,"page":"http://bioconductor.org/packages/release/bioc/html/discordant.html"},{"authors":"Wikum Dinalankara <wdinala1@jhmi.edu>, Luigi Marchionni\n<marchion@jhu.edu> and Qian Ke <qke1@jhu.edu>","name":"divergence","license":"GPL-2","tags":["Software","StatisticalMethod"],"description":"This package provides functionality for performing\ndivergence analysis as presented in Dinalankara et al,\n\"Digitizing omics profiles by divergence from a baseline\", PANS\n2018. This allows the user to simplify high dimensional omics\ndata into a binary or ternary format which encapsulates how the\ndata is divergent from a specified baseline group with the same\nunivariate or multivariate features.","downloads_month":0,"downloads_total":34,"page":"http://bioconductor.org/packages/release/bioc/html/divergence.html"},{"authors":"Jeffrey T. Leek <jleek@jhsph.edu>","name":"dks","license":"GPL","tags":["MultipleComparison","QualityControl"],"description":"The dks package consists of a set of diagnostic functions\nfor multiple testing methods. The functions can be used to\ndetermine if the p-values produced by a multiple testing\nprocedure are correct. These functions are designed to be\napplied to simulated data. The functions require the entire set\nof p-values from multiple simulated studies, so that the joint\ndistribution can be evaluated.","downloads_month":136,"downloads_total":31172,"page":"http://bioconductor.org/packages/release/bioc/html/dks.html"},{"authors":"Farhad Shokoohi","name":"DMCHMM","license":"GPL-3","tags":["DifferentialMethylation","Sequencing","HiddenMarkovModel","Coverage"],"description":"A pipeline for identifying differentially methylated CpG\nsites using Hidden Markov Model in bisulfite sequencing data.","downloads_month":148,"downloads_total":4036,"page":"http://bioconductor.org/packages/release/bioc/html/DMCHMM.html"},{"authors":"Nicolae Radu Zabet <n.r.zabet@gen.cam.ac.uk>, Jonathan Michael\nFoonlan Tsang <jmft2@cam.ac.uk>, Alessandro Pio Greco\n<apgrec@essex.ac.uk> and Ryan Merritt <rmerri@essex.ac.uk>","name":"DMRcaller","license":"GPL-3","tags":["DifferentialMethylation","DNAMethylation","Software","Sequencing","Coverage"],"description":"Uses Bisulfite sequencing data in two conditions and\nidentifies differentially methylated regions between the\nconditions in CG and non-CG context. The input is the CX report\nfiles produced by Bismark and the output is a list of DMRs\nstored as GRanges objects.","downloads_month":300,"downloads_total":16008,"page":"http://bioconductor.org/packages/release/bioc/html/DMRcaller.html"},{"authors":"Tim Peters","name":"DMRcate","license":"file LICENSE","tags":["DifferentialMethylation","GeneExpression","Microarray","MethylationArray","Genetics","DifferentialExpression","GenomeAnnotation","DNAMethylation","OneChannel","TwoChannel","MultipleComparison","QualityControl","TimeCourse"],"description":"De novo identification and extraction of differentially\nmethylated regions (DMRs) from the human genome using Whole\nGenome Bisulphite Sequencing (WGBS) and Illumina Infinium Array\n(450K and EPIC) data. Provides functionality for filtering\nprobes possibly confounded by SNPs and cross-hybridisation.\nIncludes GRanges generation and plotting functions.","downloads_month":958,"downloads_total":73068,"page":"http://bioconductor.org/packages/release/bioc/html/DMRcate.html"},{"authors":"Martin Rijlaarsdam [aut, cre], Yvonne vd Zwan [aut], Lambert\nDorssers [aut] and Leendert Looijenga [aut]","name":"DMRforPairs","license":"GPL (>= 2)","tags":["Microarray","DNAMethylation","DifferentialMethylation","ReportWriting","Visualization","Annotation"],"description":"DMRforPairs (formerly DMR2+) allows researchers to compare\nn>=2 unique samples with regard to their methylation profile.\nThe (pairwise) comparison of n unique single samples\ndistinguishes DMRforPairs from other existing pipelines as\nthese often compare groups of samples in either single CpG\nlocus or region based analysis. DMRforPairs defines regions of\ninterest as genomic ranges with sufficient probes located in\nclose proximity to each other. Probes in one region are\noptionally annotated to the same functional class(es).\nDifferential methylation is evaluated by comparing the\nmethylation values within each region between individual\nsamples and (if the difference is sufficiently large), testing\nthis difference formally for statistical significance.","downloads_month":144,"downloads_total":20612,"page":"http://bioconductor.org/packages/release/bioc/html/DMRforPairs.html"},{"authors":"Christian M Page [aut, cre], Linda Vos [aut], Trine B Rounge\n[ctb, dtc], Hanne F Harbo [ths] and Bettina K Andreassen [aut]","name":"DMRScan","license":"GPL-3","tags":["Software","Technology","Sequencing","WholeGenome"],"description":"This package detects significant differentially methylated\nregions (for both qualitative and quantitative traits), using a\nscan statistic with underlying Poisson heuristics. The scan\nstatistic will depend on a sequence of window sizes (# of CpGs\nwithin each window) and on a threshold for each window size.\nThis threshold can be calculated by three different means: i)\nanalytically using Siegmund et.al (2012) solution (preferred),\nii) an important sampling as suggested by Zhang (2008), and a\niii) full MCMC modeling of the data, choosing between a number\nof different options for modeling the dependency between each\nCpG.","downloads_month":145,"downloads_total":5116,"page":"http://bioconductor.org/packages/release/bioc/html/DMRScan.html"},{"authors":"Keegan Korthauer <keegan@jimmy.harvard.edu>, Sutirtha\nChakraborty <statistuta@gmail.com>, Yuval Benjamini\n<yuvalbenj@gmail.com> and Rafael Irizarry <rafa@jimmy.harvard.edu>","name":"dmrseq","license":"MIT + file LICENSE","tags":["ImmunoOncology","DNAMethylation","Epigenetics","MultipleComparison","Software","Sequencing","DifferentialMethylation","WholeGenome","Regression","FunctionalGenomics"],"description":"This package implements an approach for scanning the\ngenome to detect and perform accurate inference on\ndifferentially methylated regions from Whole Genome Bisulfite\nSequencing data. The method is based on comparing detected\nregions to a pooled null distribution, that can be implemented\neven when as few as two samples per population are available.\nRegion-level statistics are obtained by fitting a generalized\nleast squares (GLS) regression model with a nested\nautoregressive correlated error structure for the effect of\ninterest on transformed methylation proportions.","downloads_month":338,"downloads_total":4782,"page":"http://bioconductor.org/packages/release/bioc/html/dmrseq.html"},{"authors":"Céline Trébeau [cre] (<https://orcid.org/0000-0001-6795-5379>), Jacques Boutet de Monvel [aut]\n(<https://orcid.org/0000-0001-6182-3527>), Fabienne Wong Jun\nTai [ctb] and Raphaël Etournay [aut]\n(<https://orcid.org/0000-0002-2441-9274>)","name":"DNABarcodeCompatibility","license":"GPL-2","tags":["Preprocessing","Sequencing"],"description":"The package allows one to obtain optimized combinations of\nDNA barcodes to be used for multiplex sequencing. In each\nbarcode combination, barcodes are pooled with respect to\nIllumina chemistry constraints. Combinations can be filtered to\nkeep those that are robust against substitution and\ninsertion/deletion errors thereby facilitating the\ndemultiplexing step. In addition, the package provides an\noptimizer function to further favor the selection of barcode\ncombinations with least redundancy of DNA barcodes.","downloads_month":24,"downloads_total":152,"page":"http://bioconductor.org/packages/release/bioc/html/DNABarcodeCompatibility.html"},{"authors":"Tilo Buschmann <tilo.buschmann.ac@gmail.com>","name":"DNABarcodes","license":"GPL-2","tags":["Preprocessing","Sequencing"],"description":"The package offers a function to create DNA barcode sets\ncapable of correcting insertion, deletion, and substitution\nerrors. Existing barcodes can be analysed regarding their\nminimal, maximal and average distances between barcodes.\nFinally, reads that start with a (possibly mutated) barcode can\nbe demultiplexed, i.e., assigned to their original reference\nbarcode.","downloads_month":210,"downloads_total":13618,"page":"http://bioconductor.org/packages/release/bioc/html/DNABarcodes.html"},{"authors":"Venkatraman E. Seshan and Adam Olshen","name":"DNAcopy","license":"GPL (>= 2)","tags":["Microarray","CopyNumberVariation"],"description":"Implements the circular binary segmentation (CBS)\nalgorithm to segment DNA copy number data and identify genomic\nregions with abnormal copy number.","downloads_month":3189,"downloads_total":559012,"page":"http://bioconductor.org/packages/release/bioc/html/DNAcopy.html"},{"authors":"Tsu-Pei Chiu and Federico Comoglio","name":"DNAshapeR","license":"GPL-2","tags":["StructuralPrediction","DNA3DStructure","Software"],"description":"DNAhapeR is an R/BioConductor package for ultra-fast,\nhigh-throughput predictions of DNA shape features. The package\nallows to predict, visualize and encode DNA shape features for\nstatistical learning.","downloads_month":210,"downloads_total":12016,"page":"http://bioconductor.org/packages/release/bioc/html/DNAshapeR.html"},{"authors":"Marija Buljan and Peter Blattmann","name":"DominoEffect","license":"GPL (>= 3)","tags":["Software","SomaticMutation","Proteomics","SequenceMatching","Alignment"],"description":"The functions support identification and annotation of\nhotspot residues in proteins. These are individual amino acids\nthat accumulate mutations at a much higher rate than their\nsurrounding regions.","downloads_month":134,"downloads_total":2308,"page":"http://bioconductor.org/packages/release/bioc/html/DominoEffect.html"},{"authors":"Levi Waldron [aut, cre], Markus Reister [aut, ctb] and Marcel\nRamos [ctb]","name":"doppelgangR","license":"GPL (>=2.0)","tags":["ImmunoOncology","RNASeq","Microarray","GeneExpression","QualityControl"],"description":"The main function is doppelgangR(), which takes as minimal\ninput a list of ExpressionSet object, and searches all list\npairs for duplicated samples.  The search is based on the\ngenomic data (exprs(eset)), phenotype/clinical data\n(pData(eset)), and \"smoking guns\" - supposedly unique\nidentifiers found in pData(eset).","downloads_month":199,"downloads_total":9028,"page":"http://bioconductor.org/packages/release/bioc/html/doppelgangR.html"},{"authors":"Daniel Gatti, Karl Broman, Andrey Shabalin and Petr Simecek","name":"DOQTL","license":"GPL-3","tags":["GeneticVariability","SNP","Genetics","HiddenMarkovModel"],"description":"DOQTL is a quantitative trait locus (QTL) mapping pipeline\ndesigned for Diversity Outbred mice and other multi-parent\noutbred populations. The package reads in data from genotyping\narrays and perform haplotype reconstruction using a hidden\nMarkov model (HMM). The haplotype probabilities from the HMM\nare then used to perform linkage mapping. When founder\nsequences are available, DOQTL can use the haplotype\nreconstructions to impute the founder sequences onto DO genomes\nand perform association mapping.","downloads_month":186,"downloads_total":21296,"page":"http://bioconductor.org/packages/release/bioc/html/DOQTL.html"},{"authors":"Bruno Contrino and Piero Ricchiuto","name":"Doscheda","license":"GPL-3","tags":["ImmunoOncology","Proteomics","Normalization","Preprocessing","MassSpectrometry","QualityControl","DataImport","Regression"],"description":"Doscheda focuses on quantitative chemoproteomics used to\ndetermine protein interaction profiles of small molecules from\nwhole cell or tissue lysates using Mass Spectrometry data. The\npackage provides a shiny application to run the pipeline,\nseveral visualisations and a downloadable report of an\nexperiment.","downloads_month":176,"downloads_total":3642,"page":"http://bioconductor.org/packages/release/bioc/html/Doscheda.html"},{"authors":"Guangchuang Yu [aut, cre], Li-Gen Wang [ctb], Vladislav Petyuk\n[ctb] and Giovanni Dall'Olio [ctb]","name":"DOSE","license":"Artistic-2.0","tags":["Annotation","Visualization","MultipleComparison","GeneSetEnrichment","Pathways","Software"],"description":"This package implements five methods proposed by Resnik,\nSchlicker, Jiang, Lin and Wang respectively for measuring\nsemantic similarities among DO terms and gene products.\nEnrichment analyses including hypergeometric model and gene set\nenrichment analysis are also implemented for discovering\ndisease associations of high-throughput biological data.","downloads_month":8498,"downloads_total":386216,"page":"http://bioconductor.org/packages/release/bioc/html/DOSE.html"},{"authors":"AJ Vaestermark and JR Walters.","name":"doseR","license":"GPL","tags":["Infrastructure","Software","DataRepresentation","Sequencing","GeneExpression","SystemsBiology","DifferentialExpression"],"description":"doseR package is a next generation sequencing package for\nsex chromosome dosage compensation which can be applied broadly\nto detect shifts in gene expression among an arbitrary number\nof pre-defined groups of loci. doseR is a differential gene\nexpression package for count data, that detects directional\nshifts in expression for multiple, specific subsets of genes,\nbroad utility in systems biology research. doseR has been\nprepared to manage the nature of the data and the desired set\nof inferences. doseR uses S4 classes to store count data from\nsequencing experiment. It contains functions to normalize and\nfilter count data, as well as to plot and calculate statistics\nof count data. It contains a framework for linear modeling of\ncount data. The package has been tested using real and\nsimulated data.","downloads_month":0,"downloads_total":2,"page":"http://bioconductor.org/packages/release/bioc/html/doseR.html"},{"authors":"Paul Brennan [aut and cre]","name":"drawProteins","license":"MIT + file LICENSE","tags":["Visualization","FunctionalPrediction","Proteomics"],"description":"This package draws protein schematics from Uniprot API\noutput. From the JSON returned by the GET command, it creates a\ndataframe from the Uniprot Features API. This dataframe can\nthen be used by geoms based on ggplot2 and base R to draw\nprotein schematics.","downloads_month":159,"downloads_total":3028,"page":"http://bioconductor.org/packages/release/bioc/html/drawProteins.html"},{"authors":"Malgorzata Nowicka [aut and cre]","name":"DRIMSeq","license":"GPL (>= 3)","tags":["ImmunoOncology","SNP","AlternativeSplicing","DifferentialSplicing","Genetics","RNASeq","Sequencing","WorkflowStep","MultipleComparison","GeneExpression","DifferentialExpression"],"description":"The package provides two frameworks. One for the\ndifferential transcript usage analysis between different\nconditions and one for the tuQTL analysis. Both are based on\nmodeling the counts of genomic features (i.e., transcripts)\nwith the Dirichlet-multinomial distribution. The package also\nmakes available functions for visualization and exploration of\nthe data and results.","downloads_month":514,"downloads_total":14748,"page":"http://bioconductor.org/packages/release/bioc/html/DRIMSeq.html"},{"authors":"Ali Bashashati, Reza Haffari, Jiarui Ding, Gavin Ha, Kenneth\nLiu, Jamie Rosner and Sohrab Shah","name":"DriverNet","license":"GPL-3","tags":["Network"],"description":"DriverNet is a package to predict functional important\ndriver genes in cancer by integrating genome data (mutation and\ncopy number variation data) and transcriptome data (gene\nexpression data). The different kinds of data are combined by\nan influence graph, which is a gene-gene interaction network\ndeduced from pathway data. A greedy algorithm is used to find\nthe possible driver genes, which may mutated in a larger number\nof patients and these mutations will push the gene expression\nvalues of the connected genes to some extreme values.","downloads_month":149,"downloads_total":28926,"page":"http://bioconductor.org/packages/release/bioc/html/DriverNet.html"},{"authors":"Aaron Lun [aut, cre], Jonathan Griffiths [ctb] and Davis McCarthy\n[ctb]","name":"DropletUtils","license":"GPL-3","tags":["ImmunoOncology","SingleCell","Sequencing","RNASeq","GeneExpression","Transcriptomics","DataImport","Coverage"],"description":"Provides a number of utility functions for handling\nsingle-cell (RNA-seq) data from droplet technologies such as\n10X Genomics. This includes data loading, identification of\ncells from empty droplets, removal of barcode-swapped\npseudo-cells, and downsampling of the count matrix.","downloads_month":1044,"downloads_total":15040,"page":"http://bioconductor.org/packages/release/bioc/html/DropletUtils.html"},{"authors":"C. Pacini","name":"DrugVsDisease","license":"GPL-3","tags":["Microarray","GeneExpression","Clustering"],"description":"This package generates ranked lists of differential gene\nexpression for either disease or drug profiles. Input data can\nbe downloaded from Array Express or GEO, or from local CEL\nfiles. Ranked lists of differential expression and associated\np-values are calculated using Limma. Enrichment scores\n(Subramanian et al. PNAS 2005) are calculated to a reference\nset of default drug or disease profiles, or a set of custom\ndata supplied by the user. Network visualisation of significant\nscores are output in Cytoscape format.","downloads_month":164,"downloads_total":28512,"page":"http://bioconductor.org/packages/release/bioc/html/DrugVsDisease.html"},{"authors":"Hao Wu <hao.wu@emory.edu>","name":"DSS","license":"GPL","tags":["ImmunoOncology","Sequencing","RNASeq","ChIPSeq","DNAMethylation","GeneExpression","DifferentialExpression","DifferentialMethylation"],"description":"DSS is an R library performing differntial analysis for\ncount-based sequencing data. It detectes differentially\nexpressed genes (DEGs) from RNA-seq, and differentially\nmethylated loci or regions (DML/DMRs) from bisulfite sequencing\n(BS-seq). The core of DSS is a new dispersion shrinkage method\nfor estimating the dispersion parameter from Gamma-Poisson or\nBeta-Binomial distributions.","downloads_month":1465,"downloads_total":80770,"page":"http://bioconductor.org/packages/release/bioc/html/DSS.html"},{"authors":"Bjoern Schwalb, Benedikt Zacher, Sebastian Duemcke and Achim\nTresch","name":"DTA","license":"Artistic-2.0","tags":["Microarray","DifferentialExpression","GeneExpression","Transcription"],"description":"Dynamic Transcriptome Analysis (DTA) can monitor the\ncellular response to perturbations with higher sensitivity and\ntemporal resolution than standard transcriptomics. The package\nimplements the underlying kinetic modeling approach capable of\nthe precise determination of synthesis- and decay rates from\nindividual microarray or RNAseq measurements.","downloads_month":134,"downloads_total":32502,"page":"http://bioconductor.org/packages/release/bioc/html/DTA.html"},{"authors":"Eric J. Kort and Yarong Yang","name":"dualKS","license":"LGPL (>= 2.0)","tags":["Microarray","Classification"],"description":"This package implements a Kolmogorov Smirnov rank-sum\nbased algorithm for training (i.e. discriminant\nanalysis--identification of genes that discriminate between\nclasses) and classification of gene expression data sets.  One\nof the chief strengths of this approach is that it is amenable\nto the \"multiclass\" problem. That is, it can discriminate\nbetween more than 2 classes.","downloads_month":142,"downloads_total":40840,"page":"http://bioconductor.org/packages/release/bioc/html/dualKS.html"},{"authors":"Quanhu Sheng, Yu Shyr and Xi Chen","name":"DupChecker","license":"GPL (>= 2)","tags":["Preprocessing"],"description":"Meta-analysis has become a popular approach for\nhigh-throughput genomic data analysis because it often can\nsignificantly increase power to detect biological signals or\npatterns in datasets. However, when using public-available\ndatabases for meta-analysis, duplication of samples is an often\nencountered problem, especially for gene expression data. Not\nremoving duplicates would make study results questionable. We\ndeveloped a Bioconductor package DupChecker that efficiently\nidentifies duplicated samples by generating MD5 fingerprints\nfor raw data.","downloads_month":140,"downloads_total":15356,"page":"http://bioconductor.org/packages/release/bioc/html/DupChecker.html"},{"authors":"Sergi Sayols <sergisayolspuig@gmail.com> and Holger Klein\n<holger.klein@gmail.com>","name":"dupRadar","license":"GPL-3","tags":["Technology","Sequencing","RNASeq","QualityControl","ImmunoOncology"],"description":"Duplication rate quality control for RNA-Seq datasets.","downloads_month":155,"downloads_total":79742,"page":"http://bioconductor.org/packages/release/bioc/html/dupRadar.html"},{"authors":"Philip Lijnzaad and Thanasis Margaritis","name":"dyebias","license":"GPL-3","tags":["Microarray","TwoChannel","QualityControl","Preprocessing"],"description":"Many two-colour hybridizations suffer from a dye bias that\nis both gene-specific and slide-specific. The former depends on\nthe content of the nucleotide used for labeling; the latter\ndepends on the labeling percentage. The slide-dependency was\nhitherto not recognized, and made addressing the artefact\nimpossible.  Given a reasonable number of dye-swapped pairs of\nhybridizations, or of same vs. same hybridizations, both the\ngene- and slide-biases can be estimated and corrected using the\nGASSCO method (Margaritis et al., Mol. Sys. Biol. 5:266 (2009),\ndoi:10.1038/msb.2009.21)","downloads_month":145,"downloads_total":42930,"page":"http://bioconductor.org/packages/release/bioc/html/dyebias.html"},{"authors":"R. Gentleman and Jeff Gentry","name":"DynDoc","license":"Artistic-2.0","tags":["ReportWriting","Infrastructure"],"description":"A set of functions to create and interact with dynamic\ndocuments and vignettes.","downloads_month":879,"downloads_total":639186,"page":"http://bioconductor.org/packages/release/bioc/html/DynDoc.html"},{"authors":"Le Pape Sylvain","name":"EasyqpcR","license":"GPL (>=2)","tags":["qPCR","GeneExpression"],"description":"This package is based on the qBase algorithms published by\nHellemans et al. in 2007. The EasyqpcR package allows you to\nimport easily qPCR data files as described in the vignette.\nThereafter, you can calculate amplification efficiencies,\nrelative quantities and their standard errors, normalization\nfactors based on the best reference genes choosen (using the\nSLqPCR package), and then the normalized relative quantities,\nthe NRQs scaled to your control and their standard errors. This\npackage has been created for low-throughput qPCR data analysis.","downloads_month":282,"downloads_total":43738,"page":"http://bioconductor.org/packages/release/bioc/html/EasyqpcR.html"},{"authors":"Nicolas Delhomme, Ismael Padioleau, Bastian Schiffthaler and Niklas Maehler","name":"easyRNASeq","license":"Artistic-2.0","tags":["GeneExpression","RNASeq","Genetics","Preprocessing","ImmunoOncology"],"description":"Calculates the coverage of high-throughput short-reads\nagainst a genome of reference and summarizes it per feature of\ninterest (e.g. exon, gene, transcript). The data can be\nnormalized as 'RPKM' or by the 'DESeq' or 'edgeR' package.","downloads_month":297,"downloads_total":85028,"page":"http://bioconductor.org/packages/release/bioc/html/easyRNASeq.html"},{"authors":"Ming Yuan, Michael Newton, Deepayan Sarkar and Christina\nKendziorski","name":"EBarrays","license":"GPL (>= 2)","tags":["Clustering","DifferentialExpression"],"description":"EBarrays provides tools for the analysis of\nreplicated/unreplicated microarray data.","downloads_month":225,"downloads_total":69618,"page":"http://bioconductor.org/packages/release/bioc/html/EBarrays.html"},{"authors":"John A. Dawson","name":"EBcoexpress","license":"GPL (>= 2)","tags":["Bayesian"],"description":"An Empirical Bayesian Approach to Differential\nCo-Expression Analysis at the Gene-Pair Level","downloads_month":159,"downloads_total":32178,"page":"http://bioconductor.org/packages/release/bioc/html/EBcoexpress.html"},{"authors":"Andrzej Oleś, Gregoire Pau, Mike Smith, Oleg Sklyar, Wolfgang\nHuber, with contributions from Joseph Barry and Philip A.\nMarais","name":"EBImage","license":"LGPL","tags":["Visualization"],"description":"EBImage provides general purpose functionality for image\nprocessing and analysis. In the context of (high-throughput)\nmicroscopy-based cellular assays, EBImage offers tools to\nsegment cells and extract quantitative cellular descriptors.\nThis allows the automation of such tasks using the R\nprogramming language and facilitates the use of other tools in\nthe R environment for signal processing, statistical modeling,\nmachine learning and visualization with image data.","downloads_month":4710,"downloads_total":479578,"page":"http://bioconductor.org/packages/release/bioc/html/EBImage.html"},{"authors":"Arfa Mehmood, Asta Laiho and Laura L. Elo","name":"EBSEA","license":"GPL-2","tags":["Software","DifferentialExpression","GeneExpression","Sequencing"],"description":"Calculates differential expression of genes based on exon\ncounts of genes obtained from RNA-seq sequencing data.","downloads_month":135,"downloads_total":7978,"page":"http://bioconductor.org/packages/release/bioc/html/EBSEA.html"},{"authors":"Ning Leng and Christina Kendziorski","name":"EBSeq","license":"Artistic-2.0","tags":["ImmunoOncology","StatisticalMethod","DifferentialExpression","MultipleComparison","RNASeq","Sequencing"],"description":"Differential Expression analysis at both gene and isoform\nlevel using RNA-seq data","downloads_month":1043,"downloads_total":94798,"page":"http://bioconductor.org/packages/release/bioc/html/EBSeq.html"},{"authors":"Ning Leng and Christina Kendziorski","name":"EBSeqHMM","license":"Artistic-2.0","tags":["ImmunoOncology","StatisticalMethod","DifferentialExpression","MultipleComparison","RNASeq","Sequencing","GeneExpression","Bayesian","HiddenMarkovModel","TimeCourse"],"description":"The EBSeqHMM package implements an auto-regressive hidden\nMarkov model for statistical analysis in ordered RNA-seq\nexperiments (e.g. time course or spatial course data). The\nEBSeqHMM package provides functions to identify genes and\nisoforms that have non-constant expression profile over the\ntime points/positions, and cluster them into expression paths.","downloads_month":234,"downloads_total":19636,"page":"http://bioconductor.org/packages/release/bioc/html/EBSeqHMM.html"},{"authors":"Laurent Gautier","name":"ecolitk","license":"GPL (>= 2)","tags":["Annotation","Visualization"],"description":"Meta-data and tools to work with E. coli. The tools are\nmostly plotting functions to work with circular genomes. They\ncan used with other genomes/plasmids.","downloads_month":155,"downloads_total":45972,"page":"http://bioconductor.org/packages/release/bioc/html/ecolitk.html"},{"authors":"Davide Risso [aut, cre, cph], Sandrine Dudoit [aut] and Ludwig\nGeistlinger [ctb]","name":"EDASeq","license":"Artistic-2.0","tags":["ImmunoOncology","Sequencing","RNASeq","Preprocessing","QualityControl","DifferentialExpression"],"description":"Numerical and graphical summaries of RNA-Seq read data.\nWithin-lane normalization procedures to adjust for GC-content\neffect (or other gene-level effects) on read counts: loess\nrobust local regression, global-scaling, and full-quantile\nnormalization (Risso et al., 2011). Between-lane normalization\nprocedures to adjust for distributional differences between\nlanes (e.g., sequencing depth): global-scaling and\nfull-quantile normalization (Bullard et al., 2010).","downloads_month":4825,"downloads_total":232378,"page":"http://bioconductor.org/packages/release/bioc/html/EDASeq.html"},{"authors":"Li Juntao, Luo Huaien, Chia Kuan Hui Burton and Niranjan Nagarajan","name":"EDDA","license":"GPL (>= 2)","tags":["ImmunoOncology","Sequencing","ExperimentalDesign","Normalization","RNASeq","ChIPSeq"],"description":"EDDA can aid in the design of a range of common\nexperiments such as RNA-seq, Nanostring assays, RIP-seq and\nMetagenomic sequencing, and enables researchers to\ncomprehensively investigate the impact of experimental\ndecisions on the ability to detect differential abundance. This\nwork was published on 3 December 2014 at Genome Biology under\nthe title \"The importance of study design for detecting\ndifferentially abundant features in high-throughput\nexperiments\" (http://genomebiology.com/2014/15/12/527).","downloads_month":203,"downloads_total":21650,"page":"http://bioconductor.org/packages/release/bioc/html/EDDA.html"},{"authors":"John D. Storey, Jeffrey T. Leek and Andrew J. Bass","name":"edge","license":"MIT + file LICENSE","tags":["MultipleComparison","DifferentialExpression","TimeCourse","Regression","GeneExpression","DataImport"],"description":"The edge package implements methods for carrying out\ndifferential expression analyses of genome-wide gene expression\nstudies. Significance testing using the optimal discovery\nprocedure and generalized likelihood ratio tests (equivalent to\nF-tests and t-tests) are implemented for general study designs.\nSpecial functions are available to facilitate the analysis of\ncommon study designs, including time course experiments. Other\npackages such as snm, sva, and qvalue are integrated in edge to\nprovide a wide range of tools for gene expression analysis.","downloads_month":256,"downloads_total":21322,"page":"http://bioconductor.org/packages/release/bioc/html/edge.html"},{"authors":"Yunshun Chen <yuchen@wehi.edu.au>, Aaron Lun\n<alun@wehi.edu.au>, Davis McCarthy <dmccarthy@wehi.edu.au>, Xiaobei Zhou <xiaobei.zhou@uzh.ch>, Mark Robinson\n<mark.robinson@imls.uzh.ch> and Gordon Smyth <smyth@wehi.edu.au>","name":"edgeR","license":"GPL (>=2)","tags":["GeneExpression","Transcription","AlternativeSplicing","Coverage","DifferentialExpression","DifferentialSplicing","DifferentialMethylation","GeneSetEnrichment","Pathways","Genetics","DNAMethylation","Bayesian","Clustering","ChIPSeq","Regression","TimeCourse","Sequencing","RNASeq","BatchEffect","SAGE","Normalization","QualityControl","MultipleComparison","BiomedicalInformatics","CellBiology","FunctionalGenomics","Epigenetics","Genetics","ImmunoOncology","SystemsBiology","Transcriptomics"],"description":"Differential expression analysis of RNA-seq expression\nprofiles with biological replication. Implements a range of\nstatistical methodology based on the negative binomial\ndistributions, including empirical Bayes estimation, exact\ntests, generalized linear models and quasi-likelihood tests. As\nwell as RNA-seq, it be applied to differential signal analysis\nof other types of genomic data that produce counts, including\nChIP-seq, Bisulfite-seq, SAGE and CAGE.","downloads_month":21801,"downloads_total":2262834,"page":"http://bioconductor.org/packages/release/bioc/html/edgeR.html"},{"authors":"Xiaoyuan Zhou, Guofeng Meng, Christine Nardini and Hongkang Mei","name":"eegc","license":"GPL-2","tags":["Microarray","Sequencing","RNASeq","DifferentialExpression","GeneRegulation","GeneSetEnrichment","GeneExpression","GeneTarget"],"description":"This package has been developed to evaluate cellular\nengineering processes for direct differentiation of stem cells\nor conversion (transdifferentiation) of somatic cells to\nprimary cells based on high throughput gene expression data\nscreened either by DNA microarray or RNA sequencing. The\npackage takes gene expression profiles as inputs from three\ntypes of samples: (i) somatic or stem cells to be\n(trans)differentiated (input of the engineering process), (ii)\ninduced cells to be evaluated (output of the engineering\nprocess) and (iii) target primary cells (reference for the\noutput). The package performs differential gene expression\nanalysis for each pair-wise sample comparison to identify and\nevaluate the transcriptional differences among the 3 types of\nsamples (input, output, reference). The ideal goal is to have\ninduced and primary reference cell showing overlapping\nprofiles, both very different from the original cells.","downloads_month":135,"downloads_total":7242,"page":"http://bioconductor.org/packages/release/bioc/html/eegc.html"},{"authors":"Sara Ballouz [aut, cre], Melanie Weber [aut, ctb], Paul\nPavlidis [aut], Jesse Gillis [aut and ctb]","name":"EGAD","license":"GPL-2","tags":["Software","FunctionalGenomics","SystemsBiology","GenePrediction","FunctionalPrediction","NetworkEnrichment","GraphAndNetwork","Network"],"description":"The package implements a series of highly efficient tools\nto calculate functional properties of networks based on guilt\nby association methods.","downloads_month":141,"downloads_total":10598,"page":"http://bioconductor.org/packages/release/bioc/html/EGAD.html"},{"authors":"Monther Alhamdoosh, Luyi Tian, Milica Ng and Matthew Ritchie","name":"EGSEA","license":"GPL-3","tags":["ImmunoOncology","DifferentialExpression","GO","GeneExpression","GeneSetEnrichment","Genetics","Microarray","MultipleComparison","OneChannel","Pathways","RNASeq","Sequencing","Software","SystemsBiology","TwoChannel","Metabolomics","Proteomics","KEGG","GraphAndNetwork","GeneSignaling","GeneTarget","NetworkEnrichment","Network","Classification"],"description":"This package implements the Ensemble of Gene Set\nEnrichment Analyses (EGSEA) method for gene set testing.","downloads_month":457,"downloads_total":20282,"page":"http://bioconductor.org/packages/release/bioc/html/EGSEA.html"},{"authors":"Kevin Horan, Yiqun Cao and Tyler Backman","name":"eiR","license":"Artistic-2.0","tags":["Cheminformatics","BiomedicalInformatics","Pharmacogenetics","Pharmacogenomics","MicrotitrePlateAssay","CellBasedAssays","Visualization","Infrastructure","DataImport","Clustering","Proteomics","Metabolomics"],"description":"The eiR package provides utilities for accelerated\nstructure similarity searching of very large small molecule\ndata sets using an embedding and indexing approach.","downloads_month":157,"downloads_total":17882,"page":"http://bioconductor.org/packages/release/bioc/html/eiR.html"},{"authors":"Gabor Csardi <csardi.gabor@gmail.com>","name":"eisa","license":"GPL (>= 2)","tags":["Classification","Visualization","Microarray","GeneExpression"],"description":"The Iterative Signature Algorithm (ISA) is a biclustering\nmethod; it finds correlated blocks (transcription modules) in\ngene expression (or other tabular) data. The ISA is capable of\nfinding overlapping modules and it is resilient to noise. This\npackage provides a convenient interface to the ISA, using\nstandard BioConductor data structures; and also contains\nvarious visualization tools that can be used with other\nbiclustering algorithms.","downloads_month":184,"downloads_total":46868,"page":"http://bioconductor.org/packages/release/bioc/html/eisa.html"},{"authors":"Xiangli Zhang, Natalie Bjorklund, Graham Alvare, Tom Ryzdak, Richard Sparling and Brian Fristensky","name":"ELBOW","license":"file LICENSE","tags":["ImmunoOncology","Technology","Microarray","RNASeq","Sequencing","Sequencing","Software","MultiChannel","OneChannel","TwoChannel","GeneExpression"],"description":"Elbow an improved fold change test that uses cluster\nanalysis and pattern recognition to set cut off limits that are\nderived directly from intrareplicate variance without assuming\na normal distribution for as few as 2 biological replicates.\nElbow also provides the same consistency as fold testing in\ncross platform analysis. Elbow has lower false positive and\nfalse negative rates than standard fold testing when both are\nevaluated using T testing and Statistical Analysis of\nMicroarray using 12 replicates (six replicates each for initial\nand final conditions). Elbow provides a null value based on\ninitial condition replicates and gives error bounds for results\nto allow better evaluation of significance.","downloads_month":153,"downloads_total":19964,"page":"http://bioconductor.org/packages/release/bioc/html/ELBOW.html"},{"authors":"Tiago Chedraoui Silva [aut, cre], Lijing Yao [aut], Simon\nCoetzee [aut], Nicole Gull [ctb], Hui Shen [ctb], Peter Laird\n[ctb], Peggy Farnham [aut], Dechen Li [ctb] and Benjamin Berman\n[aut]","name":"ELMER","license":"GPL-3","tags":["DNAMethylation","GeneExpression","MotifAnnotation","Software","GeneRegulation","Transcription","Network"],"description":"ELMER is designed to use DNA methylation and gene\nexpression from a large number of samples to infere regulatory\nelement landscape and transcription factor network in primary\ntissue.","downloads_month":480,"downloads_total":26642,"page":"http://bioconductor.org/packages/release/bioc/html/ELMER.html"},{"authors":"Sadhika Malladi [aut, cre], Daniel Schmolze [aut, cre], Andrew\nBeck [aut] and Sheida Nabavi [aut]","name":"EMDomics","license":"MIT + file LICENSE","tags":["Software","DifferentialExpression","GeneExpression","Microarray"],"description":"The EMDomics algorithm is used to perform a supervised\nmulti-class analysis to measure the magnitude and statistical\nsignificance of observed continuous genomics data between\ngroups. Usually the data will be gene expression values from\narray-based or sequence-based experiments, but data from other\ntypes of experiments can also be analyzed (e.g. copy number\nvariation). Traditional methods like Significance Analysis of\nMicroarrays (SAM) and Linear Models for Microarray Data (LIMMA)\nuse significance tests based on summary statistics (mean and\nstandard deviation) of the distributions. This approach lacks\npower to identify expression differences between groups that\nshow high levels of intra-group heterogeneity. The Earth\nMover's Distance (EMD) algorithm instead computes the \"work\"\nneeded to transform one distribution into another, thus\nproviding a metric of the overall difference in shape between\ntwo distributions. Permutation of sample labels is used to\ngenerate q-values for the observed EMD scores. This package\nalso incorporates the Komolgorov-Smirnov (K-S) test and the\nCramer von Mises test (CVM), which are both common distribution\ncomparison tests.","downloads_month":148,"downloads_total":13540,"page":"http://bioconductor.org/packages/release/bioc/html/EMDomics.html"},{"authors":"William Poole","name":"EmpiricalBrownsMethod","license":"MIT + file LICENSE","tags":["StatisticalMethod","GeneExpression","Pathways"],"description":"Combining P-values from multiple statistical tests is\ncommon in bioinformatics. However, this procedure is\nnon-trivial for dependent P-values. This package implements an\nempirical adaptation of Brown’s Method (an extension of\nFisher’s Method) for combining dependent P-values which is\nappropriate for highly correlated data sets found in\nhigh-throughput biological experiments.","downloads_month":185,"downloads_total":10414,"page":"http://bioconductor.org/packages/release/bioc/html/EmpiricalBrownsMethod.html"},{"authors":"Charles Joly Beauparlant\n<charles.joly-beauparlant@crchul.ulaval.ca>, Audrey Lemacon\n<lemacon.audrey@ulaval.ca>, Louis Gendron\n<louisg.212@gmail.com> Astrid-Louise Deschenes, <astrid-louise.deschenes@crchudequebec.ulaval.ca> and Arnaud\nDroit <arnaud.droit@crchudequebec.ulaval.ca >","name":"ENCODExplorer","license":"Artistic-2.0","tags":["Infrastructure","DataImport"],"description":"This package allows user to quickly access ENCODE project\nfiles metadata and give access to helper functions to query the\nENCODE rest api, download ENCODE datasets and save the database\nin SQLite format.","downloads_month":190,"downloads_total":16544,"page":"http://bioconductor.org/packages/release/bioc/html/ENCODExplorer.html"},{"authors":"Kevin Blighe","name":"EnhancedVolcano","license":"GPL-3","tags":["RNASeq","GeneExpression","Transcription","DifferentialExpression","ImmunoOncology"],"description":"Volcano plots represent a useful way to visualise the\nresults of differential expression analyses. Here, we present a\nhighly-configurable function that produces publication-ready\nvolcano plots. EnhancedVolcano will attempt to fit as many\ntranscript names in the plot window as possible, thus avoiding\nclogging up the plot with labels that could not otherwise have\nbeen read. Other functionality allows the user to identify up\nto 3 different types of attributes in the same plot space via\ncolour, shape, and shade parameter configurations.","downloads_month":569,"downloads_total":4514,"page":"http://bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html"},{"authors":"Zongli Xu [cre, aut], Liang Niu [aut], Leping Li [ctb] and Jack\nTaylor [ctb]","name":"ENmix","license":"Artistic-2.0","tags":["DNAMethylation","Preprocessing","QualityControl","TwoChannel","Microarray","OneChannel","MethylationArray","BatchEffect","Normalization","DataImport","Regression","PrincipalComponent","Epigenetics","MultiChannel","DifferentialMethylation","ImmunoOncology"],"description":"The ENmix package provides a set of quality control and\ndata pre-processing tools for Illumina HumanMethylation450 and\nMethylationEPIC Beadchips. It includes ENmix background\ncorrection, RELIC dye bias correction, RCP probe-type bias\nadjustment, along with a number of additional tools. These\nfunctions can be used to remove unwanted experimental noise and\nthus to improve accuracy and reproducibility of methylation\nmeasures. ENmix functions are flexible and transparent. Users\nhave option to choose a single pipeline command to finish all\ndata pre-processing steps (including background correction,\ndye-bias adjustment, inter-array normalization and probe-type\nbias correction) or to use individual functions sequentially to\nperform data pre-processing in a more customized manner. In\naddition the ENmix package has selectable complementary\nfunctions for efficient data visualization (such as data\ndistribution plots); quality control (identifing and filtering\nlow quality data points, samples, probes, and outliers, along\nwith imputation of missing values); identification of probes\nwith multimodal distributions due to SNPs or other factors;\nexploration of data variance structure using principal\ncomponent regression analysis plot; preparation of experimental\nfactors related surrogate control variables to be adjusted in\ndownstream statistical analysis; and an efficient algorithm\noxBS-MLE to estimate 5-methylcytosine and\n5-hydroxymethylcytosine level.","downloads_month":328,"downloads_total":18522,"page":"http://bioconductor.org/packages/release/bioc/html/ENmix.html"},{"authors":"Zuguang Gu","name":"EnrichedHeatmap","license":"MIT + file LICENSE","tags":["Software","Visualization","Sequencing","GenomeAnnotation","Coverage"],"description":"Enriched heatmap is a special type of heatmap which\nvisualizes the enrichment of genomic signals on specific target\nregions. Here we implement enriched heatmap by ComplexHeatmap\npackage. Since this type of heatmap is just a normal heatmap\nbut with some special settings, with the functionality of\nComplexHeatmap, it would be much easier to customize the\nheatmap as well as concatenating to a list of heatmaps to show\ncorrespondance between different data sources.","downloads_month":234,"downloads_total":16542,"page":"http://bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html"},{"authors":"Ludwig Geistlinger [aut, cre], Gergely Csaba [aut], Mara\nSantarelli [ctb], Marcel Ramos [ctb], Levi Waldron [ctb] and Ralf\nZimmer [aut]","name":"EnrichmentBrowser","license":"Artistic-2.0","tags":["ImmunoOncology","Microarray","RNASeq","GeneExpression","DifferentialExpression","Pathways","GraphAndNetwork","Network","GeneSetEnrichment","NetworkEnrichment","Visualization","ReportWriting"],"description":"The EnrichmentBrowser package implements essential\nfunctionality for the enrichment analysis of gene expression\ndata. The analysis combines the advantages of set-based and\nnetwork-based enrichment analysis in order to derive\nhigh-confidence gene sets and biological pathways that are\ndifferentially regulated in the expression data under\ninvestigation. Besides, the package facilitates the\nvisualization and exploration of such sets and pathways.","downloads_month":573,"downloads_total":37266,"page":"http://bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html"},{"authors":"Guangchuang Yu [aut and cre]\n(<https://orcid.org/0000-0002-6485-8781>)","name":"enrichplot","license":"Artistic-2.0","tags":["Annotation","GeneSetEnrichment","GO","KEGG","Pathways","Software","Visualization"],"description":"The 'enrichplot' package implements several visualization\nmethods for interpreting functional enrichment results obtained\nfrom ORA or GSEA analysis. All the visualization methods are\ndeveloped based on 'ggplot2' graphics.","downloads_month":14517,"downloads_total":150572,"page":"http://bioconductor.org/packages/release/bioc/html/enrichplot.html"},{"authors":"Johannes Rainer <johannes.rainer@eurac.edu> with contributions\nfrom Tim Triche, Sebastian Gibb, Laurent Gatto and Christian\nWeichenberger.","name":"ensembldb","license":"LGPL","tags":["Genetics","AnnotationData","Sequencing","Coverage"],"description":"The package provides functions to create and use\ntranscript centric annotation databases/packages. The\nannotation for the databases are directly fetched from Ensembl\nusing their Perl API. The functionality and data is similar to\nthat of the TxDb packages from the GenomicFeatures package,\nbut, in addition to retrieve all gene/transcript models and\nannotations from the database, ensembldb provides a filter\nframework allowing to retrieve annotations for specific entries\nlike genes encoded on a chromosome region or transcript models\nof lincRNA genes. EnsDb databases built with ensembldb contain\nalso protein annotations and mappings between proteins and\ntheir encoding transcripts. Finally, ensembldb provides\nfunctions to map between genomic, transcript and protein\ncoordinates.","downloads_month":8034,"downloads_total":458876,"page":"http://bioconductor.org/packages/release/bioc/html/ensembldb.html"},{"authors":"Valerie Obenchain and Lori Shepherd","name":"ensemblVEP","license":"Artistic-2.0","tags":["Annotation","VariantAnnotation","SNP"],"description":"Query the Ensembl Variant Effect Predictor via the perl\nAPI.","downloads_month":305,"downloads_total":42146,"page":"http://bioconductor.org/packages/release/bioc/html/ensemblVEP.html"},{"authors":"Alex Lisovich and Roger Day","name":"ENVISIONQuery","license":"GPL-2","tags":["Annotation"],"description":"Tools to retrieve data from ENVISION, the Database for\nAnnotation, Visualization and Integrated Discovery portal","downloads_month":121,"downloads_total":32704,"page":"http://bioconductor.org/packages/release/bioc/html/ENVISIONQuery.html"},{"authors":"Andrew E. Teschendorff <a.teschendorff@ucl.ac.uk> and Shijie C.\nZheng<shijieczheng@gmail.com>","name":"EpiDISH","license":"GPL-2","tags":["DNAMethylation","MethylationArray","Epigenetics","DifferentialMethylation","ImmunoOncology"],"description":"EpiDISH is a R package to infer the proportions of a\npriori known cell subtypes present in a sample representing a\nmixture of such cell-types. And it also provides a method that\nallows the identification of differentially methylated\ncell-types and their directionality of change.","downloads_month":209,"downloads_total":4812,"page":"http://bioconductor.org/packages/release/bioc/html/EpiDISH.html"},{"authors":"Hans-Ulrich Klein and Martin Schaefer","name":"epigenomix","license":"LGPL-3","tags":["ChIPSeq","GeneExpression","DifferentialExpression","Classification"],"description":"A package for the integrative analysis of RNA-seq or\nmicroarray based gene transcription and histone modification\ndata obtained by ChIP-seq. The package provides methods for\ndata preprocessing and matching as well as methods for fitting\nbayesian mixture models in order to detect genes with\ndifferences in both data types.","downloads_month":160,"downloads_total":27086,"page":"http://bioconductor.org/packages/release/bioc/html/epigenomix.html"},{"authors":"Xiaowen Chen [aut, cre], Haitham Ashoor [aut], Ryan Musich\n[aut], Mingsheng Zhang [aut], Jiahui Wang [aut] and Sheng Li [aut]","name":"epihet","license":"Artistic-2.0","tags":["DNAMethylation","Epigenetics","MethylSeq","Sequencing","Software"],"description":"epihet is an R-package that calculates the epigenetic\nheterogeneity between cancer cells and/or normal cells. The\nfunctions establish a pipeline that take in bisulfite\nsequencing data from multiple samples and use the data to track\nsimilarities and differences in epipolymorphism,proportion of\ndiscordantly methylated sequencing reads (PDR),and Shannon\nentropy values at epialleles that are shared between the\nsamples.epihet can be used to perform analysis on the data by\ncreating pheatmaps, box plots, PCA plots, and t-SNE plots. MA\nplots can also be created by calculating the differential\nheterogeneity of the samples. And we construct co-epihet\nnetwork and perform network analysis.","downloads_month":10,"downloads_total":42,"page":"http://bioconductor.org/packages/release/bioc/html/epihet.html"},{"authors":"Madeline Diekmann & Martin Pirkl","name":"epiNEM","license":"GPL-3","tags":["Pathways","SystemsBiology","NetworkInference","Network"],"description":"epiNEM is an extension of the original Nested Effects\nModels (NEM). EpiNEM is able to take into account double\nknockouts and infer more complex network signalling pathways.","downloads_month":146,"downloads_total":5220,"page":"http://bioconductor.org/packages/release/bioc/html/epiNEM.html"},{"authors":"Hector Corrada Bravo, Florin Chelaru, Llewellyn Smith, Naomi\nGoldstein, Jayaram Kancherla, Morgan Walter and Brian Gottfried","name":"epivizr","license":"Artistic-2.0","tags":["Visualization","Infrastructure","GUI"],"description":"This package provides connections to the epiviz web app\n(http://epiviz.cbcb.umd.edu) for interactive visualization of\ngenomic data. Objects in R/bioc interactive sessions can be\ndisplayed in genome browser tracks or plots to be explored by\nnavigation through genomic regions. Fundamental Bioconductor\ndata structures are supported (e.g., GenomicRanges and\nRangedSummarizedExperiment objects), while providing an easy\nmechanism to support other data structures (through package\nepivizrData). Visualizations (using d3.js) can be easily added\nto the web app as well.","downloads_month":205,"downloads_total":32730,"page":"http://bioconductor.org/packages/release/bioc/html/epivizr.html"},{"authors":"Brian Gottfried [aut], Jayaram Kancherla [aut], Hector Corrada\nBravo [aut and cre]","name":"epivizrChart","license":"Artistic-2.0","tags":["Visualization","GUI"],"description":"This package provides an API for interactive visualization\nof genomic data using epiviz web components. Objects in\nR/BioConductor can be used to generate interactive R\nmarkdown/notebook documents or can be visualized in the R\nStudio's default viewer.","downloads_month":139,"downloads_total":3516,"page":"http://bioconductor.org/packages/release/bioc/html/epivizrChart.html"},{"authors":"Hector Corrada Bravo [aut, cre] and Florin Chelaru [aut]","name":"epivizrData","license":"MIT + file LICENSE","tags":["Infrastructure","Visualization"],"description":"Serve data from Bioconductor Objects through a WebSocket\nconnection.","downloads_month":203,"downloads_total":12288,"page":"http://bioconductor.org/packages/release/bioc/html/epivizrData.html"},{"authors":"Hector Corrada Bravo [aut and cre]","name":"epivizrServer","license":"MIT + file LICENSE","tags":["Infrastructure","Visualization"],"description":"This package provides objects to manage WebSocket\nconnections to epiviz apps. Other epivizr package use this\ninfrastructure.","downloads_month":182,"downloads_total":11408,"page":"http://bioconductor.org/packages/release/bioc/html/epivizrServer.html"},{"authors":"Hector Corrada Bravo and Jayaram Kancherla","name":"epivizrStandalone","license":"MIT + file LICENSE","tags":["Visualization","Infrastructure","GUI"],"description":"This package imports the epiviz visualization JavaScript\napp for genomic data interactive visualization. The\n'epivizrServer' package is used to provide a web server running\ncompletely within R. This standalone version allows to browse\narbitrary genomes through genome annotations provided by\nBioconductor packages.","downloads_month":177,"downloads_total":9780,"page":"http://bioconductor.org/packages/release/bioc/html/epivizrStandalone.html"},{"authors":"Sarah Munro and Steve Lund","name":"erccdashboard","license":"GPL (>=2)","tags":["ImmunoOncology","GeneExpression","Transcription","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","Genetics","Microarray","mRNAMicroarray","RNASeq","BatchEffect","MultipleComparison","QualityControl"],"description":"Technical performance metrics for differential gene\nexpression experiments using External RNA Controls Consortium\n(ERCC) spike-in ratio mixtures.","downloads_month":218,"downloads_total":21552,"page":"http://bioconductor.org/packages/release/bioc/html/erccdashboard.html"},{"authors":"Zixuan Shao [aut and cre]","name":"ERSSA","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","GeneExpression","Transcription","DifferentialExpression","RNASeq","MultipleComparison","QualityControl"],"description":"The ERSSA package takes user supplied RNA-seq differential\nexpression dataset and calculates the number of differentially\nexpressed genes at varying biological replicate levels. This\nallows the user to determine, without relying on any a priori\nassumptions, whether sufficient differential detection has been\nacheived with their RNA-seq dataset.","downloads_month":129,"downloads_total":1114,"page":"http://bioconductor.org/packages/release/bioc/html/ERSSA.html"},{"authors":"Zheng Wei and Wei Zhang","name":"esATAC","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","Sequencing","DNASeq","QualityControl","Alignment","Preprocessing","Coverage","ATACSeq","DNaseSeq"],"description":"This package provides a framework and complete preset\npipeline for quantification and analysis of ATAC-seq Reads. It\ncovers raw sequencing reads preprocessing (FASTQ files), reads\nalignment (Rbowtie2), aligned reads file operations (SAM, BAM,\nand BED files), peak calling (F-seq), genome annotations\n(Motif, GO, SNP analysis) and quality control report. The\npackage is managed by dataflow graph. It is easy for user to\npass variables seamlessly between processes and understand the\nworkflow. Users can process FASTQ files through end-to-end\npreset pipeline which produces a pretty HTML report for quality\ncontrol and preliminary statistical results, or customize\nworkflow starting from any intermediate stages with esATAC\nfunctions easily and flexibly.","downloads_month":541,"downloads_total":13732,"page":"http://bioconductor.org/packages/release/bioc/html/esATAC.html"},{"authors":"Laure Cougnaud <laure.cougnaud@openanalytics.eu>","name":"esetVis","license":"GPL-3","tags":["Visualization","DataRepresentation","DimensionReduction","PrincipalComponent","Pathways"],"description":"Utility functions for visualization of expressionSet (or\nSummarizedExperiment) Bioconductor object, including spectral\nmap, tsne and linear discriminant analysis. Static plot via the\nggplot2 package or interactive via the ggvis or rbokeh packages\nare available.","downloads_month":150,"downloads_total":7874,"page":"http://bioconductor.org/packages/release/bioc/html/esetVis.html"},{"authors":"Juan Pablo Romero, Juan Ferrer-Bonsoms, Ander Muniategui, Fernando Carazo, Ander Aramburu and Angel Rubio","name":"EventPointer","license":"Artistic-2.0","tags":["AlternativeSplicing","DifferentialSplicing","mRNAMicroarray","RNASeq","Transcription","Sequencing","TimeCourse","ImmunoOncology"],"description":"EventPointer is an R package to identify alternative\nsplicing events that involve either simple (case-control\nexperiment) or complex experimental designs such as time course\nexperiments and studies including paired-samples. The algorithm\ncan be used to analyze data from either junction arrays\n(Affymetrix Arrays) or sequencing data (RNA-Seq). The software\nreturns a data.frame with the detected alternative splicing\nevents: gene name, type of event (cassette, alternative\n3',...,etc), genomic position, statistical significance and\nincrement of the percent spliced in (Delta PSI) for all the\nevents. The algorithm can generate a series of files to\nvisualize the detected alternative splicing events in IGV. This\neases the interpretation of results and the design of primers\nfor standard PCR validation.","downloads_month":165,"downloads_total":6132,"page":"http://bioconductor.org/packages/release/bioc/html/EventPointer.html"},{"authors":"R. Matthew Tanner, William L. Stanford, and Theodore J. Perkins","name":"ExCluster","license":"GPL-3","tags":["ImmunoOncology","DifferentialSplicing","RNASeq","Software"],"description":"ExCluster flattens Ensembl and GENCODE GTF files into GFF\nfiles, which are used to count reads per non-overlapping exon\nbin from BAM files. This read counting is done using the\nfunction featureCounts from the package Rsubread. Library sizes\nare normalized across all biological replicates, and ExCluster\nthen compares two different conditions to detect signifcantly\ndifferentially spliced genes. This process requires at least\ntwo independent biological repliates per condition, and\nExCluster accepts only exactly two conditions at a time.\nExCluster ultimately produces false discovery rates (FDRs) per\ngene, which are used to detect significance. Exon log2 fold\nchange (log2FC) means and variances may be plotted for each\nsignificantly differentially spliced gene, which helps\nscientists develop hypothesis and target differential splicing\nevents for RT-qPCR validation in the wet lab.","downloads_month":112,"downloads_total":974,"page":"http://bioconductor.org/packages/release/bioc/html/ExCluster.html"},{"authors":"Sylvain Gubian <DL.RSupport@pmi.com>, Alain Sewer\n<DL.RSupport@pmi.com> and PMP SA","name":"ExiMiR","license":"GPL-2","tags":["Microarray","OneChannel","TwoChannel","Preprocessing","GeneExpression","Transcription"],"description":"This package contains functions for reading raw data in\nImaGene TXT format obtained from Exiqon miRCURY LNA arrays,\nannotating them with appropriate GAL files, and normalizing\nthem using a spike-in probe-based method. Other platforms and\ndata formats are also supported.","downloads_month":148,"downloads_total":35906,"page":"http://bioconductor.org/packages/release/bioc/html/ExiMiR.html"},{"authors":"Michael Love","name":"exomeCopy","license":"GPL (>= 2)","tags":["CopyNumberVariation","Sequencing","Genetics"],"description":"Detection of copy number variants (CNV) from exome\nsequencing samples, including unpaired samples.  The package\nimplements a hidden Markov model which uses positional\ncovariates, such as background read depth and GC-content, to\nsimultaneously normalize and segment the samples into regions\nof constant copy count.","downloads_month":430,"downloads_total":59448,"page":"http://bioconductor.org/packages/release/bioc/html/exomeCopy.html"},{"authors":"Lin Zhang <lin.zhang@cumt.edu.cn>, Lian Liu\n<liulian19860905@163.com> and Jia Meng <jia.meng@xjtlu.edu.cn>","name":"exomePeak","license":"GPL-2","tags":["Sequencing","HighThroughputSequencing","Methylseq","RNAseq"],"description":"The package is developed for the analysis of\naffinity-based epitranscriptome shortgun sequencing data from\nMeRIP-seq (maA-seq). It was built on the basis of the exomePeak\nMATLAB package (Meng, Jia, et al. \"Exome-based analysis for RNA\nepigenome sequencing data.\" Bioinformatics 29.12 (2013):\n1565-1567.) with new functions for differential analysis of two\nexperimental conditions to unveil the dynamics in\npost-transcriptional regulation of the RNA methylome. The\nexomePeak R-package accepts and statistically supports multiple\nbiological replicates, internally removes PCR artifacts and\nmulti-mapping reads, outputs exome-based binding sites (RNA\nmethylation sites) and detects differential\npost-transcriptional RNA modification sites between two\nexperimental conditions in term of percentage rather the\nabsolute amount. The package is still under active development,\nand we welcome all biology and computation scientist for all\nkinds of collaborations and communications. Please feel free to\ncontact Dr. Jia Meng <jia.meng@hotmail.com> if you have any\nquestions.","downloads_month":232,"downloads_total":30924,"page":"http://bioconductor.org/packages/release/bioc/html/exomePeak.html"},{"authors":"Bioconductor Package Maintainer <maintainer@bioconductor.org>","name":"ExperimentHub","license":"Artistic-2.0","tags":["Infrastructure","DataImport","GUI","ThirdPartyClient"],"description":"This package provides a client for the Bioconductor\nExperimentHub web resource. ExperimentHub provides a central\nlocation where curated data from experiments, publications or\ntraining courses can be accessed. Each resource has associated\nmetadata, tags and date of modification. The client creates and\nmanages a local cache of files retrieved enabling quick and\nreproducible access.","downloads_month":934,"downloads_total":30378,"page":"http://bioconductor.org/packages/release/bioc/html/ExperimentHub.html"},{"authors":"Bioconductor Maintainer [cre]","name":"ExperimentHubData","license":"Artistic-2.0","tags":["Infrastructure","DataImport","GUI","ThirdPartyClient"],"description":"Functions to add metadata to ExperimentHub db and resource\nfiles to AWS S3 buckets.","downloads_month":156,"downloads_total":9188,"page":"http://bioconductor.org/packages/release/bioc/html/ExperimentHubData.html"},{"authors":"Michael Lawrence, Eun-kyung Lee, Dianne Cook, Jihong Kim, Hogeun An, and Dongshin Kim","name":"explorase","license":"GPL-2","tags":["Visualization","Microarray","GUI"],"description":"explore and analyze *omics data with R and GGobi","downloads_month":113,"downloads_total":48432,"page":"http://bioconductor.org/packages/release/bioc/html/explorase.html"},{"authors":"Maria Keays","name":"ExpressionAtlas","license":"GPL (>= 3)","tags":["ExpressionData","ExperimentData","SequencingData","MicroarrayData","ArrayExpress"],"description":"This package is for searching for datasets in EMBL-EBI\nExpression Atlas, and downloading them into R for further\nanalysis. Each Expression Atlas dataset is represented as a\nSimpleList object with one element per platform. Sequencing\ndata is contained in a SummarizedExperiment object, while\nmicroarray data is contained in an ExpressionSet or MAList\nobject.","downloads_month":169,"downloads_total":10328,"page":"http://bioconductor.org/packages/release/bioc/html/ExpressionAtlas.html"},{"authors":"Andreas Luscher <andreas.luescher@a3.epfl.ch>","name":"ExpressionView","license":"GPL (>= 2)","tags":["Classification","Visualization","Microarray","GeneExpression","GO","KEGG"],"description":"ExpressionView visualizes possibly overlapping biclusters\nin a gene expression matrix. It can use the result of the ISA\nmethod (eisa package) or the algorithms in the biclust package\nor others. The viewer itself was developed using Adobe Flex and\nruns in a flash-enabled web browser.","downloads_month":160,"downloads_total":39796,"page":"http://bioconductor.org/packages/release/bioc/html/ExpressionView.html"},{"authors":"Sepp Hochreiter <hochreit@bioinf.jku.at>","name":"fabia","license":"LGPL (>= 2.1)","tags":["StatisticalMethod","Microarray","DifferentialExpression","MultipleComparison","Clustering","Visualization"],"description":"Biclustering by \"Factor Analysis for Bicluster\nAcquisition\" (FABIA). FABIA is a model-based technique for\nbiclustering, that is clustering rows and columns\nsimultaneously. Biclusters are found by factor analysis where\nboth the factors and the loading matrix are sparse. FABIA is a\nmultiplicative model that extracts linear dependencies between\nsamples and feature patterns. It captures realistic\nnon-Gaussian data distributions with heavy tails as observed in\ngene expression measurements. FABIA utilizes well understood\nmodel selection techniques like the EM algorithm and\nvariational approaches and is embedded into a Bayesian\nframework. FABIA ranks biclusters according to their\ninformation content and separates spurious biclusters from true\nbiclusters. The code is written in C.","downloads_month":1168,"downloads_total":65840,"page":"http://bioconductor.org/packages/release/bioc/html/fabia.html"},{"authors":"Denise Scholtens","name":"factDesign","license":"LGPL","tags":["Microarray","DifferentialExpression"],"description":"This package provides a set of tools for analyzing data\nfrom a factorial designed microarray experiment, or any\nmicroarray experiment for which a linear model is appropriate.\nThe functions can be used to evaluate tests of contrast of\nbiological interest and perform single outlier detection.","downloads_month":169,"downloads_total":55088,"page":"http://bioconductor.org/packages/release/bioc/html/factDesign.html"},{"authors":"J. Rainer, D. Taliun and C.X. Weichenberger","name":"FamAgg","license":"MIT + file LICENSE","tags":["Genetics"],"description":"Framework providing basic pedigree analysis and plotting\nutilities as well as a variety of methods to evaluate familial\naggregation of traits in large pedigrees.","downloads_month":173,"downloads_total":9306,"page":"http://bioconductor.org/packages/release/bioc/html/FamAgg.html"},{"authors":"Djork-Arne Clevert <okko@clevert.de>","name":"farms","license":"LGPL (>= 2.1)","tags":["GeneExpression","Microarray","Preprocessing","QualityControl"],"description":"The package provides the summarization algorithm called\nFactor Analysis for Robust Microarray Summarization (FARMS) and\na novel unsupervised feature selection criterion called\n\"I/NI-calls\"","downloads_month":160,"downloads_total":41374,"page":"http://bioconductor.org/packages/release/bioc/html/farms.html"},{"authors":"Tina Gunderson","name":"fastLiquidAssociation","license":"GPL-2","tags":["Software","GeneExpression","Genetics","Pathways","CellBiology"],"description":"This package extends the function of the LiquidAssociation\npackage for genome-wide application. It integrates a screening\nmethod into the LA analysis to reduce the number of triplets to\nbe examined for a high LA value and provides code for use in\nsubsequent significance analyses.","downloads_month":140,"downloads_total":17702,"page":"http://bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html"},{"authors":"Leandro Roser [aut, cre], Fernán Agüero [aut] and Daniel Sánchez\n[aut]","name":"FastqCleaner","license":"GPL (>= 2)","tags":["QualityControl","Sequencing","Software","SangerSeq","SequenceMatching"],"description":"An interactive web application for quality control,\nfiltering and trimming of FASTQ files. This user-friendly tool\ncombines a pipeline for data processing based on Biostrings and\nShortRead infrastructure, with a cutting-edge visual\nenvironment. Single-Read and Paired-End files can be locally\nprocessed. Diagnostic interactive plots (CG content, per-base\nsequence quality, etc.) are provided for both the input and\noutput files.","downloads_month":107,"downloads_total":1016,"page":"http://bioconductor.org/packages/release/bioc/html/FastqCleaner.html"},{"authors":"Guenter Klambauer","name":"fastseg","license":"LGPL (>= 2.0)","tags":["Classification","CopyNumberVariation"],"description":"fastseg implements a very fast and efficient segmentation\nalgorithm. It has similar functionality as DNACopy (Olshen and\nVenkatraman 2004), but is considerably faster and more\nflexible. fastseg can segment data from DNA microarrays and\ndata from next generation sequencing for example to detect copy\nnumber segments. Further it can segment data from RNA\nmicroarrays like tiling arrays to identify transcripts. Most\ngenerally, it can segment data given as a matrix or as a\nvector. Various data formats can be used as input to fastseg\nlike expression set objects for microarrays or GRanges for\nsequencing data. The segmentation criterion of fastseg is based\non a statistical test in a Bayesian framework, namely the cyber\nt-test (Baldi 2001). The speed-up arises from the facts, that\nsampling is not necessary in for fastseg and that a dynamic\nprogramming approach is used for calculation of the segments'\nfirst and higher order moments.","downloads_month":978,"downloads_total":69556,"page":"http://bioconductor.org/packages/release/bioc/html/fastseg.html"},{"authors":"Tiago Lubiana [aut, cre], Helder Nakaya [aut and ths]","name":"FCBF","license":"MIT + file LICENSE","tags":["ImmunoOncology","GeneTarget","FeatureExtraction","Classification","GeneExpression","SingleCell"],"description":"This package provides a simple R implementation for the\nFast Correlation Based Filter described in Yu, L. and Liu, H.;\nFeature Selection for High-Dimensional Data: A Fast Correlation\nBased Filter Solution,Proc. 20th Intl. Conf. Mach. Learn.\n(ICML-2003), Washington DC, 2003 The current package is an\nintent to make easier for bioinformaticians to use FCBF for\nfeature selection, especially regarding transcriptomic\ndata.This implies discretizing expression (function\ndiscretize_exprs) before calculating the features that explain\nthe class, but are not predictable by other features. The\nfunctions are implemented based on the algorithm of Yu and Liu,\n2003 and Rajarshi Guha's implementation from 13/05/2005\navailable (as of 26/08/2018) at\nhttp://www.rguha.net/code/R/fcbf.R .","downloads_month":123,"downloads_total":1032,"page":"http://bioconductor.org/packages/release/bioc/html/FCBF.html"},{"authors":"Pedro Madrigal <bioinformatics.engineer@gmail.com>","name":"fCCAC","license":"Artistic-2.0","tags":["Transcription","Genetics","Sequencing","Coverage"],"description":"An application of functional canonical correlation\nanalysis to assess covariance of nucleic acid sequencing\ndatasets such as chromatin immunoprecipitation followed by deep\nsequencing (ChIP-seq). The package can be used as well with\nother types of sequencing data such as neMeRIP-seq (PMID:\n29489750) or with single cell RNA-seq or epigenome data\nprovided in bigWig format.","downloads_month":142,"downloads_total":6834,"page":"http://bioconductor.org/packages/release/bioc/html/fCCAC.html"},{"authors":"Shaojun Tang","name":"fCI","license":"GPL (>= 2)","tags":["Proteomics"],"description":"(f-divergence Cutoff Index), is to find DEGs in the\ntranscriptomic & proteomic data, and identify DEGs by computing\nthe difference between the distribution of fold-changes for the\ncontrol-control and remaining (non-differential) case-control\ngene expression ratio data. fCI provides several advantages\ncompared to existing methods.","downloads_month":158,"downloads_total":10876,"page":"http://bioconductor.org/packages/release/bioc/html/fCI.html"},{"authors":"Yoav Benjamini, Effi Kenigsberg, Anat Reiner and Daniel Yekutieli","name":"fdrame","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"This package contains two main functions. The first is\nfdr.ma which takes normalized expression data array,\nexperimental design and computes adjusted p-values It returns\nthe fdr adjusted p-values and plots, according to the methods\ndescribed in (Reiner, Yekutieli and Benjamini 2002). The\nsecond, is fdr.gui() which creates a simple graphic user\ninterface to access fdr.ma","downloads_month":153,"downloads_total":50128,"page":"http://bioconductor.org/packages/release/bioc/html/fdrame.html"},{"authors":"Sergio Picart-Armada [aut, cre], Francesc Fernandez-Albert\n[aut] and Alexandre Perera-Lluna [aut]","name":"FELLA","license":"GPL-3","tags":["Software","Metabolomics","GraphAndNetwork","KEGG","GO","Pathways","Network","NetworkEnrichment"],"description":"Enrichment of metabolomics data using KEGG entries. Given\na set of affected compounds, FELLA suggests affected reactions,\nenzymes, modules and pathways using label propagation in a\nknowledge model network. The resulting subnetwork can be\nvisualised and exported.","downloads_month":166,"downloads_total":2376,"page":"http://bioconductor.org/packages/release/bioc/html/FELLA.html"},{"authors":"Andrew E. Teschendorff and Zhen Yang","name":"FEM","license":"GPL (>=2)","tags":["SystemsBiology","NetworkEnrichment","DifferentialMethylation","DifferentialExpression"],"description":"The FEM package performs a systems-level integrative\nanalysis of DNA methylation and gene expression data. It seeks\nmodules of functionally related genes which exhibit\ndifferential promoter DNA methylation and differential\nexpression, where an inverse association between promoter DNA\nmethylation and gene expression is assumed. For full details,\nsee Jiao et al Bioinformatics 2014.","downloads_month":603,"downloads_total":40580,"page":"http://bioconductor.org/packages/release/bioc/html/FEM.html"},{"authors":"Levi Waldron","name":"ffpe","license":"GPL (>2)","tags":["Microarray","GeneExpression","QualityControl"],"description":"Identify low-quality data using metrics developed for\nexpression data derived from Formalin-Fixed, Paraffin-Embedded\n(FFPE) data.  Also a function for making Concordance at the Top\nplots (CAT-plots).","downloads_month":162,"downloads_total":33688,"page":"http://bioconductor.org/packages/release/bioc/html/ffpe.html"},{"authors":"Sara Aibar, Celia Fontanillo, Conrad Droste and Javier De Las\nRivas.","name":"FGNet","license":"GPL (>= 2)","tags":["Annotation","GO","Pathways","GeneSetEnrichment","Network","Visualization","FunctionalGenomics","NetworkEnrichment","Clustering"],"description":"Build and visualize functional gene and term networks from\nclustering of enrichment analyses in multiple annotation\nspaces. The package includes a graphical user interface (GUI)\nand functions to perform the functional enrichment analysis\nthrough DAVID, GeneTerm Linker, gage (GSEA) and topGO.","downloads_month":208,"downloads_total":34210,"page":"http://bioconductor.org/packages/release/bioc/html/FGNet.html"},{"authors":"Gennady Korotkevich [aut], Vladimir Sukhov [aut], Alexey\nSergushichev [aut and cre]","name":"fgsea","license":"MIT + file LICENCE","tags":["GeneExpression","DifferentialExpression","GeneSetEnrichment","Pathways"],"description":"The package implements an algorithm for fast gene set\nenrichment analysis. Using the fast algorithm allows to make\nmore permutations and get more fine grained p-values, which\nallows to use accurate stantard approaches to multiple\nhypothesis correction.","downloads_month":10267,"downloads_total":269676,"page":"http://bioconductor.org/packages/release/bioc/html/fgsea.html"},{"authors":"Thomas Lin Pedersen","name":"FindMyFriends","license":"GPL (>=2)","tags":["ComparativeGenomics","Clustering","DataRepresentation","GenomicVariation","SequenceMatching","GraphAndNetwork"],"description":"A framework for doing microbial comparative genomics in R.\nThe main purpose of the package is assisting in the creation of\npangenome matrices where genes from related organisms are\ngrouped by similarity, as well as the analysis of these data.\nFindMyFriends provides many novel approaches to doing pangenome\nanalysis and supports a gene grouping algorithm that scales\nlinearly, thus making the creation of huge pangenomes feasible.","downloads_month":196,"downloads_total":14040,"page":"http://bioconductor.org/packages/release/bioc/html/FindMyFriends.html"},{"authors":"Karesh Arunakirinathan <akaresh88@gmail.com> and Andreas Heindl\n<andreas.heindl@icr.ac.uk>","name":"FISHalyseR","license":"Artistic-2.0","tags":["CellBiology"],"description":"FISHalyseR provides functionality to process and analyse\ndigital cell culture images, in particular to quantify FISH\nprobes within nuclei. Furthermore, it extract the spatial\nlocation of each nucleus as well as each probe enabling spatial\nco-localisation analysis.","downloads_month":153,"downloads_total":12904,"page":"http://bioconductor.org/packages/release/bioc/html/FISHalyseR.html"},{"authors":"Anqi Zhu, Avi Srivastava, Joseph Ibrahim, Rob Patro and Michael\nLove","name":"fishpond","license":"GPL-2","tags":["Sequencing","RNASeq","GeneExpression","Transcription","Normalization","Regression","MultipleComparison","BatchEffect","Visualization","DifferentialExpression","DifferentialSplicing","AlternativeSplicing","SingleCell"],"description":"Fishpond contains methods for differential transcript and\ngene expression analysis of RNA-seq data using inferential\nreplicates for uncertainty of abundance quantification, as\ngenerated by Gibbs sampling or bootstrap sampling.","downloads_month":0,"downloads_total":8,"page":"http://bioconductor.org/packages/release/bioc/html/fishpond.html"},{"authors":"Ferhat Ay [aut] (Python original, https://noble.gs.washington.edu/proj/fit-hi-c/), Timothy L.\nBailey [aut], William S. Noble [aut], Ruyu Tan [aut, cre and trl]\n(R port)","name":"FitHiC","license":"GPL (>= 2)","tags":["DNA3DStructure","Software"],"description":"Fit-Hi-C is a tool for assigning statistical confidence\nestimates to intra-chromosomal contact maps produced by\ngenome-wide genome architecture assays such as Hi-C.","downloads_month":173,"downloads_total":8700,"page":"http://bioconductor.org/packages/release/bioc/html/FitHiC.html"},{"authors":"Mark Robinson <mark.robinson@imls.uzh.ch> and Riccardo Romoli\n<riccardo.romoli@unifi.it>","name":"flagme","license":"LGPL (>= 2)","tags":["ImmunoOncology","DifferentialExpression","MassSpectrometry"],"description":"Fragment-level analysis of gas chromatography - mass\nspectrometry metabolomics data","downloads_month":183,"downloads_total":45440,"page":"http://bioconductor.org/packages/release/bioc/html/flagme.html"},{"authors":"Gianni Monaco and Hao Chen","name":"flowAI","license":"GPL (>= 2)","tags":["FlowCytometry","QualityControl","BiomedicalInformatics","ImmunoOncology"],"description":"The package is able to perform an automatic or interactive\nquality control on FCS data acquired using flow cytometry\ninstruments. By evaluating three different properties: 1) flow\nrate, 2) signal acquisition, 3) dynamic range, the quality\ncontrol enables the detection and removal of anomalies.","downloads_month":522,"downloads_total":15260,"page":"http://bioconductor.org/packages/release/bioc/html/flowAI.html"},{"authors":"Nikolas Pontikos","name":"flowBeads","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry","CellBasedAssays"],"description":"This package extends flowCore to provide functionality\nspecific to bead data. One of the goals of this package is to\nautomate analysis of bead data for the purpose of\nnormalisation.","downloads_month":190,"downloads_total":23416,"page":"http://bioconductor.org/packages/release/bioc/html/flowBeads.html"},{"authors":"Kieran O'Neill","name":"flowBin","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","FlowCytometry"],"description":"Software to combine flow cytometry data that has been\nmultiplexed into multiple tubes with common markers between\nthem, by establishing common bins across tubes in terms of the\ncommon markers, then determining expression within each tube\nfor each bin in terms of the tube-specific markers.","downloads_month":197,"downloads_total":20506,"page":"http://bioconductor.org/packages/release/bioc/html/flowBin.html"},{"authors":"Federico Marini [aut and cre]\n(<https://orcid.org/0000-0003-3252-7758>)","name":"flowcatchR","license":"BSD_3_clause + file LICENSE","tags":["Software","Visualization","CellBiology","Classification","Infrastructure","GUI"],"description":"flowcatchR is a set of tools to analyze in vivo microscopy\nimaging data, focused on tracking flowing blood cells. It\nguides the steps from segmentation to calculation of features,\nfiltering out particles not of interest, providing also a set\nof utilities to help checking the quality of the performed\noperations (e.g. how good the segmentation was). It allows\ninvestigating the issue of tracking flowing cells such as in\nblood vessels, to categorize the particles in flowing, rolling\nand adherent. This classification is applied in the study of\nphenomena such as hemostasis and study of thrombosis\ndevelopment. Moreover, flowcatchR presents an integrated\nworkflow solution, based on the integration with a Shiny App\nand Jupyter notebooks, which is delivered alongside the\npackage, and can enable fully reproducible bioimage analysis in\nthe R environment.","downloads_month":157,"downloads_total":17364,"page":"http://bioconductor.org/packages/release/bioc/html/flowcatchR.html"},{"authors":"Joachim Schumann <joachim.schumann@ufz.de>, Christin Koch\n<christin.koch@ufz.de>, Ingo Fetzer\n<info.fetzer@stockholmresilience.su.se> and Susann Müller\n<susann.mueller@ufz.de>","name":"flowCHIC","license":"GPL-2","tags":["ImmunoOncology","CellBasedAssays","Clustering","FlowCytometry","Software","Visualization"],"description":"A package to analyze flow cytometric data of complex\nmicrobial communities based on histogram images","downloads_month":191,"downloads_total":17480,"page":"http://bioconductor.org/packages/release/bioc/html/flowCHIC.html"},{"authors":"Justin Meskas and Radina Droumeva","name":"flowCL","license":"Artistic-2.0","tags":["FlowCytometry","ImmunoOncology"],"description":"Semantic labelling of flow cytometric cell populations.","downloads_month":356,"downloads_total":25006,"page":"http://bioconductor.org/packages/release/bioc/html/flowCL.html"},{"authors":"Kipper Fletez-Brant","name":"flowClean","license":"Artistic-2.0","tags":["FlowCytometry","QualityControl"],"description":"A quality control tool for flow cytometry data based on\ncompositional data analysis.","downloads_month":266,"downloads_total":21628,"page":"http://bioconductor.org/packages/release/bioc/html/flowClean.html"},{"authors":"Raphael Gottardo <raph@stat.ubc.ca>, Kenneth Lo\n<c.lo@stat.ubc.ca> and Greg Finak <gfinak@fhcrc.org>","name":"flowClust","license":"Artistic-2.0","tags":["ImmunoOncology","Clustering","Visualization","FlowCytometry"],"description":"Robust model-based clustering using a t-mixture model with\nBox-Cox transformation. Note: users should have GSL installed.\nWindows users: 'consult the README file available in the inst\ndirectory of the source distribution for necessary\nconfiguration instructions'.","downloads_month":803,"downloads_total":96712,"page":"http://bioconductor.org/packages/release/bioc/html/flowClust.html"},{"authors":"B Ellis [aut], Perry Haaland [aut], Florian Hahne [aut], Nathan\nLe Meur [aut], Nishant Gopalakrishnan [aut], Josef Spidlen\n[aut], Mike Jiang [aut, cre], Greg Finak [aut] and Samuel\nGranjeaud [ctb]","name":"flowCore","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry","CellBasedAssays"],"description":"Provides S4 data structures and basic functions to deal\nwith flow cytometry data.","downloads_month":18626,"downloads_total":8416208,"page":"http://bioconductor.org/packages/release/bioc/html/flowCore.html"},{"authors":"Joachim Schumann <joachim.schumann@ufz.de>, Christin Koch\n<christin.koch@ufz.de>, Susanne Günther\n<susanne.guenther@ufz.de>, Ingo Fetzer\n<ingo.fetzer@stockholmresilience.su.se> and Susann Müller\n<susann.mueller@ufz.de>","name":"flowCyBar","license":"GPL-2","tags":["ImmunoOncology","CellBasedAssays","Clustering","FlowCytometry","Software","Visualization"],"description":"A package to analyze flow cytometric data using gate\ninformation to follow population/community dynamics","downloads_month":142,"downloads_total":20214,"page":"http://bioconductor.org/packages/release/bioc/html/flowCyBar.html"},{"authors":"Mehrnoush Malek and M. Jafar Taghiyar","name":"flowDensity","license":"Artistic-2.0","tags":["ImmunoOncology","Bioinformatics","FlowCytometry","CellBiology","Clustering","Cancer","FlowCytData","StemCells","DensityGating"],"description":"This package provides tools for automated sequential\ngating analogous to the manual gating strategy based on the\ndensity of the data.","downloads_month":450,"downloads_total":28842,"page":"http://bioconductor.org/packages/release/bioc/html/flowDensity.html"},{"authors":"Davide Rambaldi","name":"flowFit","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays"],"description":"This package estimate the proliferation of a cell\npopulation in cell-tracking dye studies. The package uses an R\nimplementation of the Levenberg-Marquardt algorithm\n(minpack.lm) to fit a set of peaks (corresponding to different\ngenerations of cells) over the proliferation-tracking dye\ndistribution in a FACS experiment.","downloads_month":168,"downloads_total":23630,"page":"http://bioconductor.org/packages/release/bioc/html/flowFit.html"},{"authors":"Herb Holyst <holyst@mail.med.upenn.edu> and Wade Rogers\n<rogersw@mail.med.upenn.edu>","name":"flowFP","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays","Clustering","Visualization"],"description":"Fingerprint generation of flow cytometry data, used to\nfacilitate the application of machine learning and datamining\ntools for flow cytometry.","downloads_month":267,"downloads_total":48006,"page":"http://bioconductor.org/packages/release/bioc/html/flowFP.html"},{"authors":"Chiaowen Joyce Hsiao, Yu Qian, and Richard H. Scheuermann","name":"flowMap","license":"GPL (>=2)","tags":["ImmunoOncology","MultipleComparison","FlowCytometry"],"description":"flowMap quantifies the similarity of cell populations\nacross multiple flow cytometry samples using a nonparametric\nmultivariate statistical test. The method is able to map cell\npopulations of different size, shape, and proportion across\nmultiple flow cytometry samples. The algorithm can be\nincorporate in any flow cytometry work flow that requires\naccurat quantification of similarity between cell populations.","downloads_month":163,"downloads_total":23420,"page":"http://bioconductor.org/packages/release/bioc/html/flowMap.html"},{"authors":"Ariful Azad","name":"flowMatch","license":"Artistic-2.0","tags":["ImmunoOncology","Clustering","FlowCytometry"],"description":"Matching cell populations and building meta-clusters and\ntemplates from a collection of FC samples.","downloads_month":195,"downloads_total":21138,"page":"http://bioconductor.org/packages/release/bioc/html/flowMatch.html"},{"authors":"Nima Aghaeepour","name":"flowMeans","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBiology","Clustering"],"description":"Identifies cell populations in Flow Cytometry data using\nnon-parametric clustering and segmented-regression-based change\npoint detection. Note: R 2.11.0 or newer is required.","downloads_month":533,"downloads_total":60270,"page":"http://bioconductor.org/packages/release/bioc/html/flowMeans.html"},{"authors":"Greg Finak <gfinak@fhcrc.org> and Raphael Gottardo\n<rgottard@fhcrc.org>","name":"flowMerge","license":"Artistic-2.0","tags":["ImmunoOncology","Clustering","FlowCytometry"],"description":"Merging of mixture components for model-based automated\ngating of flow cytometry data using the flowClust framework.\nNote: users should have a working copy of flowClust 2.0\ninstalled.","downloads_month":312,"downloads_total":53376,"page":"http://bioconductor.org/packages/release/bioc/html/flowMerge.html"},{"authors":"Yongchao Ge<yongchao.ge@gmail.com>","name":"flowPeaks","license":"Artistic-1.0","tags":["ImmunoOncology","FlowCytometry","Clustering","Gating"],"description":"A fast and automatic clustering to classify the cells into\nsubpopulations based on finding the peaks from the overall\ndensity function generated by K-means.","downloads_month":287,"downloads_total":36696,"page":"http://bioconductor.org/packages/release/bioc/html/flowPeaks.html"},{"authors":"Tyler Smith <tyler@plantarum.ca>","name":"flowPloidy","license":"GPL-3","tags":["FlowCytometry","GUI","Regression","Visualization"],"description":"Determine sample ploidy via flow cytometry histogram\nanalysis. Reads Flow Cytometry Standard (FCS) files via the\nflowCore bioconductor package, and provides functions for\ndetermining the DNA ploidy of samples based on internal\nstandards.","downloads_month":165,"downloads_total":7486,"page":"http://bioconductor.org/packages/release/bioc/html/flowPloidy.html"},{"authors":"N. Hawkins and S. Self","name":"flowPlots","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays","Visualization","DataRepresentation"],"description":"Graphical displays with embedded statistical tests for\ngated ICS flow cytometry data, and a data class which stores\n\"stacked\" data and has methods for computing summary measures\non stacked data, such as marginal and polyfunctional degree\ndata.","downloads_month":151,"downloads_total":34558,"page":"http://bioconductor.org/packages/release/bioc/html/flowPlots.html"},{"authors":"Josef Spidlen, Faysal El Khettabi, Wayne Moore, David Parks and Ryan Brinkman","name":"flowQB","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","Regression","PeakDetection","QualityControl","MultiChannel","OneChannel"],"description":"flowQB is a fully automated R Bioconductor package to\ncalculate automatically the detector efficiency (Q), optical\nbackground (B) and intrinsic CV of the beads.","downloads_month":154,"downloads_total":29464,"page":"http://bioconductor.org/packages/release/bioc/html/flowQB.html"},{"authors":"Josef Spidlen [aut and cre]","name":"FlowRepositoryR","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry"],"description":"This package provides an interface to search and download\ndata and annotations from FlowRepository (flowrepository.org).\nIt uses the FlowRepository programming interface to communicate\nwith a FlowRepository server.","downloads_month":191,"downloads_total":13906,"page":"http://bioconductor.org/packages/release/bioc/html/FlowRepositoryR.html"},{"authors":"Sofie Van Gassen, Britt Callebaut and Yvan Saeys","name":"FlowSOM","license":"GPL (>= 2)","tags":["ImmunoOncology","CellBiology","FlowCytometry","Clustering","Visualization","Software","CellBasedAssays"],"description":"FlowSOM offers visualization options for cytometry data,\nby using Self-Organizing Map clustering and Minimal Spanning\nTrees.","downloads_month":1461,"downloads_total":49528,"page":"http://bioconductor.org/packages/release/bioc/html/FlowSOM.html"},{"authors":"Florian Hahne, Nishant Gopalakrishnan, Alireza Hadj\nKhodabakhshi, Chao-Jen Wong and Kyongryun Lee","name":"flowStats","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays"],"description":"Methods and functionality to analyse flow data that is\nbeyond the basic infrastructure provided by the flowCore\npackage.","downloads_month":825,"downloads_total":156680,"page":"http://bioconductor.org/packages/release/bioc/html/flowStats.html"},{"authors":"R. Clay Wright [aut, cre], Nick Bolten [aut] and Edith\nPierre-Jerome [aut]","name":"flowTime","license":"Artistic-2.0","tags":["FlowCytometry","TimeCourse","Visualization","DataImport","CellBasedAssays"],"description":"This package was developed for analysis of both dynamic\nand steady state experiments examining the function of gene\nregulatory networks in yeast (strain W303) expressing\nfluorescent reporter proteins using a BD Accuri C6 and SORP\ncytometers. However, the functions are for the most part\ngeneral and may be adapted for analysis of other organisms\nusing other flow cytometers. Functions in this package\nfacilitate the annotation of flow cytometry data with\nexperimental metadata, as is requisite for dissemination and\ngeneral ease-of-use. Functions for creating, saving and loading\ngate sets are also included. In the past, we have typically\ngenerated summary statistics for each flowset for each\ntimepoint and then annotated and analyzed these summary\nstatistics. This method loses a great deal of the power that\ncomes from the large amounts of individual cell data generated\nin flow cytometry, by essentially collapsing this data into a\nbulk measurement after subsetting. In addition to these summary\nfunctions, this package also contains functions to facilitate\nannotation and analysis of steady-state or time-lapse data\nutilizing all of the data collected from the thousands of\nindividual cells in each sample.","downloads_month":194,"downloads_total":5146,"page":"http://bioconductor.org/packages/release/bioc/html/flowTime.html"},{"authors":"Greg Finak <gfinak@fredhutch.org>, Juan Manuel-Perez\n<jperez@ircm.qc.ca> and Raphael Gottardo <rgottard@fredhutch.org>","name":"flowTrans","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry"],"description":"Profile maximum likelihood estimation of parameters for\nflow cytometry data transformations.","downloads_month":245,"downloads_total":41714,"page":"http://bioconductor.org/packages/release/bioc/html/flowTrans.html"},{"authors":"Nima Aghaeepour, Kieran O'Neill and Adrin Jalali","name":"flowType","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry"],"description":"Phenotyping Flow Cytometry Assays using multidimentional\nexpansion of single dimentional partitions.","downloads_month":230,"downloads_total":40364,"page":"http://bioconductor.org/packages/release/bioc/html/flowType.html"},{"authors":"J. Spidlen., N. Gopalakrishnan, F. Hahne, B. Ellis, R.\nGentleman, M. Dalphin, N. Le Meur, B. Purcell and W. Jiang","name":"flowUtils","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry","CellBasedAssays","DecisionTree"],"description":"Provides utilities for flow cytometry data.","downloads_month":1408,"downloads_total":91552,"page":"http://bioconductor.org/packages/release/bioc/html/flowUtils.html"},{"authors":"B. Ellis, R. Gentleman, F. Hahne, N. Le Meur, D. Sarkar and M.\nJiang","name":"flowViz","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry","CellBasedAssays","Visualization"],"description":"Provides visualization tools for flow cytometry data.","downloads_month":1672,"downloads_total":259696,"page":"http://bioconductor.org/packages/release/bioc/html/flowViz.html"},{"authors":"Ariful Azad","name":"flowVS","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays","Microarray"],"description":"Per-channel variance stabilization from a collection of\nflow cytometry samples by Bertlett test for homogeneity of\nvariances. The approach is applicable to microarrays data as\nwell.","downloads_month":201,"downloads_total":15752,"page":"http://bioconductor.org/packages/release/bioc/html/flowVS.html"},{"authors":"Greg Finak and Mike Jiang","name":"flowWorkspace","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","DataImport","Preprocessing","DataRepresentation"],"description":"This package is designed to facilitate comparison of\nautomated gating methods against manual gating done in flowJo.\nThis package allows you to import basic flowJo workspaces into\nBioConductor and replicate the gating from flowJo using the\nflowCore functionality. Gating hierarchies, groups of samples,\ncompensation, and transformation are performed so that the\noutput matches the flowJo analysis.","downloads_month":1341,"downloads_total":207768,"page":"http://bioconductor.org/packages/release/bioc/html/flowWorkspace.html"},{"authors":"Yan Wang, Tyler Backman, Kevin Horan and Thomas Girke","name":"fmcsR","license":"Artistic-2.0","tags":["Cheminformatics","BiomedicalInformatics","Pharmacogenetics","Pharmacogenomics","MicrotitrePlateAssay","CellBasedAssays","Visualization","Infrastructure","DataImport","Clustering","Proteomics","Metabolomics"],"description":"The fmcsR package introduces an efficient maximum common\nsubstructure (MCS) algorithms combined with a novel matching\nstrategy that allows for atom and/or bond mismatches in the\nsubstructures shared among two small molecules. The resulting\nflexible MCSs (FMCSs) are often larger than strict MCSs,\nresulting in the identification of more common features in\ntheir source structures, as well as a higher sensitivity in\nfinding compounds with weak structural similarities. The fmcsR\npackage provides several utilities to use the FMCS algorithm\nfor pairwise compound comparisons, structure similarity\nsearching and clustering.","downloads_month":290,"downloads_total":47054,"page":"http://bioconductor.org/packages/release/bioc/html/fmcsR.html"},{"authors":"Oscar Krijgsman","name":"focalCall","license":"GPL-2","tags":["Microarray","Preprocessing","Visualization","Sequencing"],"description":"Detection of genomic focal aberrations in high-resolution\nDNA copy number data","downloads_month":139,"downloads_total":16180,"page":"http://bioconductor.org/packages/release/bioc/html/focalCall.html"},{"authors":"Daniil Wiebe <daniil.wiebe@gmail.com> [aut and cre]","name":"FoldGO","license":"GPL-3","tags":["DifferentialExpression","GeneExpression","GO","Software"],"description":"FoldGO is a package designed to annotate gene sets derived\nfrom expression experiments and identify fold-change-specific\nGO terms.","downloads_month":92,"downloads_total":694,"page":"http://bioconductor.org/packages/release/bioc/html/FoldGO.html"},{"authors":"Felix A. Klein and EMBL Heidelberg","name":"FourCSeq","license":"GPL (>= 3)","tags":["Software","Preprocessing","Sequencing"],"description":"FourCSeq is an R package dedicated to the analysis of\n(multiplexed) 4C sequencing data. The package provides a\npipeline to detect specific interactions between DNA elements\nand identify differential interactions between conditions. The\nstatistical analysis in R starts with individual bam files for\neach sample as inputs. To obtain these files, the package\ncontains a python script (extdata/python/demultiplex.py) to\ndemultiplex libraries and trim off primer sequences. With a\nstandard alignment software the required bam files can be then\nbe generated.","downloads_month":186,"downloads_total":20372,"page":"http://bioconductor.org/packages/release/bioc/html/FourCSeq.html"},{"authors":"Futao Zhang","name":"FRGEpistasis","license":"GPL-2","tags":["Genetics","NetworkInference","GeneticVariability","Software"],"description":"A Tool for Epistasis Analysis Based on Functional\nRegression Model","downloads_month":145,"downloads_total":18942,"page":"http://bioconductor.org/packages/release/bioc/html/FRGEpistasis.html"},{"authors":"Matthew N. McCall <mccallm@gmail.com>, Rafael A. Irizarry\n<rafa@jhu.edu> and with contributions from Terry Therneau","name":"frma","license":"GPL (>= 2)","tags":["Software","Microarray","Preprocessing"],"description":"Preprocessing and analysis for single microarrays and\nmicroarray batches.","downloads_month":397,"downloads_total":96376,"page":"http://bioconductor.org/packages/release/bioc/html/frma.html"},{"authors":"Matthew N. McCall <mccallm@gmail.com> and Rafael A. Irizarry\n<rafa@jhu.edu>","name":"frmaTools","license":"GPL (>= 2)","tags":["Software","Microarray","Preprocessing"],"description":"Tools for advanced use of the frma package.","downloads_month":146,"downloads_total":43216,"page":"http://bioconductor.org/packages/release/bioc/html/frmaTools.html"},{"authors":"Alice Parodi [aut, cre], Marco Morelli [aut, cre], Laura M.\nSangalli [aut], Piercesare Secchi [aut] and Simone Vantini [aut]","name":"FunChIP","license":"Artistic-2.0","tags":["StatisticalMethod","Clustering","ChIPSeq"],"description":"Preprocessing and smoothing of ChIP-Seq peaks and\nefficient implementation of the k-mean alignment algorithm to\nclassify them.","downloads_month":143,"downloads_total":6840,"page":"http://bioconductor.org/packages/release/bioc/html/FunChIP.html"},{"authors":"Simon G. Coetzee <simon@simoncoetzee.com>, Houtan Noushmehr and PhD <houtan@usp.br>","name":"FunciSNP","license":"GPL-3","tags":["Infrastructure","DataRepresentation","DataImport","SequenceMatching","Annotation"],"description":"FunciSNP integrates information from GWAS, 1000genomes and\nchromatin feature to identify functional SNP in coding or\nnon-coding regions.","downloads_month":154,"downloads_total":34920,"page":"http://bioconductor.org/packages/release/bioc/html/FunciSNP.html"},{"authors":"Celia Greenwood <celia.greenwood@mcgill.ca>, Stepan Grinek\n<stepan.grinek@ladydavis.ca>, Maxime Turgeon\n<maxime.turgeon@mail.mcgill.ca> and Kathleen Klein\n<kathleen.klein@mail.mcgill.ca>","name":"funtooNorm","license":"GPL-3","tags":["DNAMethylation","Preprocessing","Normalization"],"description":"Provides a function to normalize Illumina Infinium Human\nMethylation 450 BeadChip (Illumina 450K), correcting for tissue\nand/or cell type.","downloads_month":140,"downloads_total":4986,"page":"http://bioconductor.org/packages/release/bioc/html/funtooNorm.html"},{"authors":"Welliton Souza [aut, cre], Benilton Carvalho [ctb] and Cristiane\nRocha [ctb]","name":"GA4GHclient","license":"GPL (>= 2)","tags":["DataRepresentation","ThirdPartyClient"],"description":"GA4GHclient provides an easy way to access public data\nservers through Global Alliance for Genomics and Health (GA4GH)\ngenomics API. It provides low-level access to GA4GH API and\ntranslates response data into Bioconductor-based class objects.","downloads_month":139,"downloads_total":5356,"page":"http://bioconductor.org/packages/release/bioc/html/GA4GHclient.html"},{"authors":"Welliton Souza [aut, cre], Benilton Carvalho [ctb], Cristiane\nRocha [ctb] and Elizabeth Borgognoni [ctb]","name":"GA4GHshiny","license":"GPL-3","tags":["GUI"],"description":"GA4GHshiny package provides an easy way to interact with\ndata servers based on Global Alliance for Genomics and Health\n(GA4GH) genomics API through a Shiny application. It also\nintegrates with Beacon Network.","downloads_month":137,"downloads_total":3798,"page":"http://bioconductor.org/packages/release/bioc/html/GA4GHshiny.html"},{"authors":"David Rossell <rosselldavid@gmail.com>.","name":"gaga","license":"GPL (>= 2)","tags":["ImmunoOncology","OneChannel","MassSpectrometry","MultipleComparison","DifferentialExpression","Classification"],"description":"Implements the GaGa model for high-throughput data\nanalysis, including differential expression analysis,\nsupervised gene clustering and classification. Additionally, it\nperforms sequential sample size calculations using the GaGa and\nLNNGV models (the latter from EBarrays package).","downloads_month":206,"downloads_total":47722,"page":"http://bioconductor.org/packages/release/bioc/html/gaga.html"},{"authors":"Weijun Luo","name":"gage","license":"GPL (>=2.0)","tags":["ImmunoOncology","Pathways","GO","DifferentialExpression","Microarray","OneChannel","TwoChannel","RNASeq","Genetics","MultipleComparison","GeneSetEnrichment","GeneExpression","SystemsBiology","Sequencing"],"description":"GAGE is a published method for gene set (enrichment or\nGSEA) or pathway analysis. GAGE is generally applicable\nindependent of microarray or RNA-Seq data attributes including\nsample sizes, experimental designs, assay platforms, and other\ntypes of heterogeneity, and consistently achieves superior\nperformance over other frequently used methods. In gage\npackage, we provide functions for basic GAGE analysis, result\nprocessing and presentation. We have also built pipeline\nroutines for of multiple GAGE analyses in a batch, comparison\nbetween parallel analyses, and combined analysis of\nheterogeneous data from different sources/studies. In addition,\nwe provide demo microarray data and commonly used gene set data\nbased on KEGG pathways and GO terms. These funtions and data\nare also useful for gene set analysis using other methods.","downloads_month":1734,"downloads_total":180412,"page":"http://bioconductor.org/packages/release/bioc/html/gage.html"},{"authors":"Paul Shannon <pshannon@systemsbiology.org>","name":"gaggle","license":"GPL version 2 or newer","tags":["ThirdPartyClient","Visualization","Annotation","GraphAndNetwork","DataImport"],"description":"This package contains functions enabling data exchange\nbetween R and Gaggle enabled bioinformatics software, including\nCytoscape, Firegoose and Gaggle Genome Browser.","downloads_month":124,"downloads_total":50454,"page":"http://bioconductor.org/packages/release/bioc/html/gaggle.html"},{"authors":"Sandro Morganella et al.","name":"gaia","license":"GPL-2","tags":["aCGH","CopyNumberVariation"],"description":"This package allows to assess the statistical significance\nof chromosomal aberrations.","downloads_month":290,"downloads_total":39514,"page":"http://bioconductor.org/packages/release/bioc/html/gaia.html"},{"authors":"Casper Peters [aut, cre], Finn Drabløs [aut] and Rezvan Ehsani\n[aut]","name":"GAPGOM","license":"MIT + file LICENSE","tags":["GO","GeneExpression","GenePrediction"],"description":"Collection of various measures and tools for lncRNA\nannotation prediction put inside a redistributable R package.\nThe package contains two main algorithms; lncRNA2GOA and\nTopoICSim. lncRNA2GOA tries to annotate novel genes (in this\nspecific case lncRNAs) by using various correlation/geometric\nscoring methods on correlated expression data. After\ncorrelating/scoring, the results are annotated and enriched.\nTopoICSim is a topologically based method, that compares gene\nsimilarity based on the topology of the GO DAG by information\ncontent (IC) between GO terms.","downloads_month":0,"downloads_total":92,"page":"http://bioconductor.org/packages/release/bioc/html/GAPGOM.html"},{"authors":"Jon Bohlin","name":"GAprediction","license":"GPL (>=2)","tags":["ImmunoOncology","DNAMethylation","Epigenetics","Regression","BiomedicalInformatics"],"description":"[GAprediction] predicts gestational age using Illumina\nHumanMethylation450 CpG data.","downloads_month":152,"downloads_total":6364,"page":"http://bioconductor.org/packages/release/bioc/html/GAprediction.html"},{"authors":"Sandro Morganella <sm22@sanger.ac.uk>","name":"garfield","license":"GPL-3","tags":["Software","StatisticalMethod","Annotation","FunctionalPrediction","GenomeAnnotation"],"description":"GARFIELD is a non-parametric functional enrichment\nanalysis approach described in the paper GARFIELD: GWAS\nanalysis of regulatory or functional information enrichment\nwith LD correction. Briefly, it is a method that leverages GWAS\nfindings with regulatory or functional annotations (primarily\nfrom ENCODE and Roadmap epigenomics data) to find features\nrelevant to a phenotype of interest. It performs greedy pruning\nof GWAS SNPs (LD r2 > 0.1) and then annotates them based on\nfunctional information overlap. Next, it quantifies Fold\nEnrichment (FE) at various GWAS significance cutoffs and\nassesses them by permutation testing, while matching for minor\nallele frequency, distance to nearest transcription start site\nand number of LD proxies (r2 > 0.8).","downloads_month":137,"downloads_total":8044,"page":"http://bioconductor.org/packages/release/bioc/html/garfield.html"},{"authors":"Mattia Chiesa <mattia.chiesa@hotmail.it> and Luca Piacentini\n<luca.piacentini@cardiologicomonzino.it>","name":"GARS","license":"GPL (>= 2)","tags":["Classification","FeatureExtraction","Clustering"],"description":"Feature selection aims to identify and remove redundant,\nirrelevant and noisy variables from high-dimensional datasets.\nSelecting informative features affects the subsequent\nclassification and regression analyses by improving their\noverall performances. Several methods have been proposed to\nperform feature selection: most of them relies on univariate\nstatistics, correlation, entropy measurements or the usage of\nbackward/forward regressions. Herein, we propose an efficient,\nrobust and fast method that adopts stochastic optimization\napproaches for high-dimensional. GARS is an innovative\nimplementation of a genetic algorithm that selects robust\nfeatures in high-dimensional and challenging datasets.","downloads_month":100,"downloads_total":1926,"page":"http://bioconductor.org/packages/release/bioc/html/GARS.html"},{"authors":"Nima Aghaeepour <naghaeep@gmail.com> and Erin F. Simonds\n<erin.simonds@gmail.com>","name":"GateFinder","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBiology","Clustering"],"description":"Given a vector of cluster memberships for a cell\npopulation, identifies a sequence of gates (polygon filters on\n2D scatter plots) for isolation of that cell type.","downloads_month":173,"downloads_total":2516,"page":"http://bioconductor.org/packages/release/bioc/html/GateFinder.html"},{"authors":"Mingxiang Teng and Rafael A. Irizarry","name":"gcapc","license":"GPL-3","tags":["Sequencing","ChIPSeq","BatchEffect","PeakDetection"],"description":"Peak calling for ChIP-seq data with consideration of\npotential GC bias in sequencing reads. GC bias is first\nestimated with generalized linear mixture models using\neffective GC strategy, then applied into peak significance\nestimation.","downloads_month":139,"downloads_total":5534,"page":"http://bioconductor.org/packages/release/bioc/html/gcapc.html"},{"authors":"Wei Hao, Minsun Song and John D. Storey","name":"gcatest","license":"GPL-3","tags":["SNP","DimensionReduction","PrincipalComponent","GenomeWideAssociation"],"description":"GCAT is an association test for genome wide association\nstudies that controls for population structure under a general\nclass of trait. models.","downloads_month":127,"downloads_total":9854,"page":"http://bioconductor.org/packages/release/bioc/html/gcatest.html"},{"authors":"Thomas Sandmann <sandmann.t@gmail.com>, Richard Bourgon\n<bourgon.richard@gene.com> and Sarah Kummerfeld\n<kummerfeld.sarah@gene.com>","name":"gCMAP","license":"Artistic-2.0","tags":["Microarray","Software","Pathways","Annotation"],"description":"The gCMAP package provides a toolkit for comparing\ndifferential gene expression profiles through gene set\nenrichment analysis. Starting from normalized microarray or\nRNA-seq gene expression values (stored in lists of\nExpressionSet and CountDataSet objects) the package performs\ndifferential expression analysis using the limma or DESeq\npackages. Supplying a simple list of gene identifiers, global\ndifferential expression profiles or data from complete\nexperiments as input, users can use a unified set of several\nwell-known gene set enrichment analysis methods to retrieve\nexperiments with similar changes in gene expression. To take\ninto account the directionality of gene expression changes,\ngCMAPQuery introduces the SignedGeneSet class, directly\nextending GeneSet from the GSEABase package.  To increase\nperformance of large queries, multiple gene sets are stored as\nsparse incidence matrices within CMAPCollection eSets. gCMAP\noffers implementations of 1. Fisher's exact test (Fisher, J R\nStat Soc, 1922) 2. The \"connectivity map\" method (Lamb et al,\nScience, 2006) 3. Parametric and non-parametric t-statistic\nsummaries (Jiang & Gentleman, Bioinformatics, 2007) and 4.\nWilcoxon / Mann-Whitney rank sum statistics (Wilcoxon,\nBiometrics Bulletin, 1945) as well as wrappers for the 5.\ncamera (Wu & Smyth, Nucleic Acid Res, 2012) 6. mroast and romer\n(Wu et al, Bioinformatics, 2010) functions from the limma\npackage and 7. wraps the gsea method from the mgsa package\n(Bauer et al, NAR, 2010). All methods return CMAPResult\nobjects, an S4 class inheriting from AnnotatedDataFrame,\ncontaining enrichment statistics as well as annotation data and\nproviding simple high-level summary plots.","downloads_month":199,"downloads_total":29462,"page":"http://bioconductor.org/packages/release/bioc/html/gCMAP.html"},{"authors":"Thomas Sandmann","name":"gCMAPWeb","license":"Artistic-2.0","tags":["GUI","GeneSetEnrichment","Visualization","GeneExpression","Transcription","Microarray","DifferentialExpression"],"description":"The gCMAPWeb R package provides a graphical user interface\nfor the gCMAP package. gCMAPWeb uses the Rook package and can\nbe used either on a local machine, leveraging R's internal web\nserver, or run on a dedicated rApache web server installation.\ngCMAPWeb allows users to search their own data sources and\ninstructions to generate reference datasets from public\nrepositories are included with the package. The package\nsupports three common types of analyses, specifically queries\nwith 1. one or two sets of query gene identifiers, whose\nmembers are expected to show changes in gene expression in a\nconsistent direction. For example, an up-regulated gene set\nmight contain genes activated by a transcription factor, a\ndown-regulated geneset targets repressed by the same factor. 2.\na single set of query gene identifiers, whose members are\nexpected to show divergent differential expression\n(non-directional query). For example, members of a particular\nsignaling pathway, some of which may be up- some down-regulated\nin response to a stimulus. 3. a query with the complete results\nof a differential expression profiling experiment. For example,\ngene identifiers and z-scores from a previous perturbation\nexperiment. gCMAPWeb accepts three types of identifiers:\nEntreIds, gene Symbols and microarray probe ids and can be\nconfigured to work with any species supported by Bioconductor.\nFor each query submission, significantly similar reference\ndatasets will be identified and reported in graphical and\ntabular form.","downloads_month":124,"downloads_total":22856,"page":"http://bioconductor.org/packages/release/bioc/html/gCMAPWeb.html"},{"authors":"Russell Bainer, Dariusz Ratman, Steve Lianoglou and Peter Haverty","name":"gCrisprTools","license":"Artistic-2.0","tags":["ImmunoOncology","CRISPR","PooledScreens","ExperimentalDesign","BiomedicalInformatics","CellBiology","FunctionalGenomics","Pharmacogenomics","Pharmacogenetics","SystemsBiology","DifferentialExpression","GeneSetEnrichment","Genetics","MultipleComparison","Normalization","Preprocessing","QualityControl","RNASeq","Regression","Software","Visualization"],"description":"Set of tools for evaluating pooled high-throughput\nscreening experiments, typically employing CRISPR/Cas9 or shRNA\nexpression cassettes. Contains methods for interrogating\nlibrary and cassette behavior within an experiment, identifying\ndifferentially abundant cassettes, aggregating signals to\nidentify candidate targets for empirical validation, hypothesis\ntesting, and comprehensive reporting.","downloads_month":296,"downloads_total":8436,"page":"http://bioconductor.org/packages/release/bioc/html/gCrisprTools.html"},{"authors":"Jean(ZHIJIN) Wu and Rafael Irizarry with contributions from James\nMacDonald <jmacdon@med.umich.edu> Jeff Gentry","name":"gcrma","license":"LGPL","tags":["Microarray","OneChannel","Preprocessing"],"description":"Background adjustment using sequence information","downloads_month":2485,"downloads_total":959156,"page":"http://bioconductor.org/packages/release/bioc/html/gcrma.html"},{"authors":"Ruidong Li, Han Qu, Shibo Wang, Julong Wei, Le Zhang, Renyuan\nMa, Jianming Lu, Jianguo Zhu, Wei-De Zhong and Zhenyu Jia","name":"GDCRNATools","license":"Artistic-2.0","tags":["ImmunoOncology","GeneExpression","DifferentialExpression","GeneRegulation","GeneTarget","NetworkInference","Survival","Visualization","GeneSetEnrichment","NetworkEnrichment","Network","RNASeq","GO","KEGG"],"description":"This is an easy-to-use package for downloading,\norganizing, and integrative analyzing RNA expression data in\nGDC with an emphasis on deciphering the lncRNA-mRNA related\nceRNA regulatory network in cancer. Three databases of\nlncRNA-miRNA interactions including spongeScan, starBase, and\nmiRcode, as well as three databases of mRNA-miRNA interactions\nincluding miRTarBase, starBase, and miRcode are incorporated\ninto the package for ceRNAs network construction. limma, edgeR,\nand DESeq2 can be used to identify differentially expressed\ngenes/miRNAs. Functional enrichment analyses including GO,\nKEGG, and DO can be performed based on the clusterProfiler and\nDO packages. Both univariate CoxPH and KM survival analyses of\nmultiple genes can be implemented in the package. Besides some\nroutine visualization functions such as volcano plot, bar plot,\nand KM plot, a few simply shiny apps are developed to\nfacilitate visualization of results on a local webpage.","downloads_month":511,"downloads_total":8046,"page":"http://bioconductor.org/packages/release/bioc/html/GDCRNATools.html"},{"authors":"Qian Liu [aut, cre], Martin Morgan [aut] and Hervé Pagès [aut]","name":"GDSArray","license":"GPL-3","tags":["Infrastructure","DataRepresentation","Sequencing","GenotypingArray"],"description":"GDS files are widely used to represent genotyping or\nsequence data. The GDSArray package implements the `GDSArray`\nclass to represent nodes in GDS files in a matrix-like\nrepresentation that allows easy manipulation (e.g., subsetting,\nmathematical transformation) in _R_. The data remains on disk\nuntil needed, so that very large files can be processed.","downloads_month":141,"downloads_total":2212,"page":"http://bioconductor.org/packages/release/bioc/html/GDSArray.html"},{"authors":"Xiuwen Zheng [aut, cre], Stephanie Gogarten [ctb], Jean-loup\nGailly, Mark Adler [ctb] (for the included zlib sources), Yann Collet [ctb] (for the included LZ4 sources) and xz\ncontributors (for the included liblzma sources)","name":"gdsfmt","license":"LGPL-3","tags":["Infrastructure","DataImport"],"description":"This package provides a high-level R interface to\nCoreArray Genomic Data Structure (GDS) data files, which are\nportable across platforms with hierarchical structure to store\nmultiple scalable array-oriented data sets with metadata\ninformation. It is suited for large-scale datasets, especially\nfor data which are much larger than the available random-access\nmemory. The gdsfmt package offers the efficient operations\nspecifically designed for integers of less than 8 bits, since a\ndiploid genotype, like single-nucleotide polymorphism (SNP),\nusually occupies fewer bits than a byte. Data compression and\ndecompression are available with relatively efficient random\naccess. It is also allowed to read a GDS file in parallel with\nmultiple R processes supported by the package parallel.","downloads_month":1930,"downloads_total":119716,"page":"http://bioconductor.org/packages/release/bioc/html/gdsfmt.html"},{"authors":"Markus Boenn","name":"geecc","license":"GPL (>= 2)","tags":["ImmunoOncology","BiologicalQuestion","GeneSetEnrichment","WorkflowStep","GO","StatisticalMethod","GeneExpression","Transcription","RNASeq","Microarray"],"description":"Use log-linear models to perform hypergeometric and\nchi-squared tests for gene set enrichments for two (based on\ncontingency tables) or three categories (contingency cubes).\nCategories can be differentially expressed genes, GO terms,\nsequence length, GC content, chromosomal position, phylostrata,\ndivergence-strata, ....","downloads_month":144,"downloads_total":15424,"page":"http://bioconductor.org/packages/release/bioc/html/geecc.html"},{"authors":"Hong Pan, Joanna D Holbrook, Neerja Karnani and Chee-Keong Kwoh","name":"GEM","license":"Artistic-2.0","tags":["MethylSeq","MethylationArray","GenomeWideAssociation","Regression","DNAMethylation","SNP","GeneExpression","GUI"],"description":"Tools for analyzing EWAS, methQTL and GxE genome widely.","downloads_month":164,"downloads_total":7338,"page":"http://bioconductor.org/packages/release/bioc/html/GEM.html"},{"authors":"Ana Patricia Gomez Mayen <pgomez@ifc.unam.mx>, \\\\ Gustavo Corral\nGuille <gcorral@ifc.unam.mx>, \\\\ Lina Riego Ruiz\n<lriego@ifc.unam.mx> and \\\\ Gerardo Coello Coutino\n<gcoello@ifc.unam.mx>","name":"genArise","license":"file LICENSE","tags":["Microarray","TwoChannel","Preprocessing"],"description":"genArise is an easy to use tool for dual color microarray\ndata. Its GUI-Tk based environment let any non-experienced user\nperforms a basic, but not simple, data analysis just following\na wizard. In addition it provides some tools for the developer.","downloads_month":155,"downloads_total":45652,"page":"http://bioconductor.org/packages/release/bioc/html/genArise.html"},{"authors":"Gabriel Becker [aut, cre] and Michael Lawrence [aut]","name":"genbankr","license":"Artistic-2.0","tags":["Infrastructure","DataImport"],"description":"Reads Genbank files.","downloads_month":323,"downloads_total":16216,"page":"http://bioconductor.org/packages/release/bioc/html/genbankr.html"},{"authors":"Ariane L. Moore, Jack Kuipers and Niko Beerenwinkel","name":"GeneAccord","license":"file LICENSE","tags":["BiomedicalInformatics","GeneticVariability","GenomicVariation","SomaticMutation","FunctionalGenomics","Genetics","MathematicalBiology","SystemsBiology","FeatureExtraction","PatternLogic","Pathways"],"description":"A statistical framework to examine the combinations of\nclones that co-exist in tumors. More precisely, the algorithm\nfinds pairs of genes that are mutated in the same tumor but in\ndifferent clones, i.e. their subclonal mutation profiles are\nmutually exclusive. We refer to this as clonally exclusive. It\nmeans that the mutations occurred in different branches of the\ntumor phylogeny, indicating parallel evolution of the clones.\nOur statistical framework assesses whether a pattern of clonal\nexclusivity occurs more often than expected by chance alone\nacross a cohort of patients. The required input data are the\nmutated gene-to-clone assignments from a cohort of cancer\npatients, which were obtained by running phylogenetic tree\ninference methods. Reconstructing the evolutionary history of a\ntumor and detecting the clones is challenging. For\nnondeterministic algorithms, repeated tree inference runs may\nlead to slightly different mutation-to-clone assignments.\nTherefore, our algorithm was designed to allow the input of\nmultiple gene-to-clone assignments per patient. They may have\nbeen generated by repeatedly performing the tree inference, or\nby sampling from the posterior distribution of trees. The tree\ninference methods designate the mutations to individual clones.\nThe mutations can then be mapped to genes or pathways. Hence\nour statistical framework can be applied on the gene level, or\non the pathway level to detect clonally exclusive pairs of\npathways. If a pair is significantly clonally exclusive, it\npoints towards the fact that this specific clone configuration\nconfers a selective advantage, possibly through synergies\nbetween the clones with these mutations.","downloads_month":91,"downloads_total":750,"page":"http://bioconductor.org/packages/release/bioc/html/GeneAccord.html"},{"authors":"Lei Huang, Gang Feng, Pan Du, Tian Xia, Xishu Wang, Jing, Wen, Warren Kibbe and Simon Lin","name":"GeneAnswers","license":"LGPL (>= 2)","tags":["Infrastructure","DataRepresentation","Visualization","GraphsAndNetworks"],"description":"GeneAnswers provides an integrated tool for biological or\nmedical interpretation of the given one or more groups of genes\nby means of statistical test.","downloads_month":232,"downloads_total":86194,"page":"http://bioconductor.org/packages/release/bioc/html/GeneAnswers.html"},{"authors":"Arthur Wuster","name":"geneAttribution","license":"Artistic-2.0","tags":["SNP","GenePrediction","GenomeWideAssociation","VariantAnnotation","GenomicVariation"],"description":"Identification of the most likely gene or genes through\nwhich variation at a given genomic locus in the human genome\nacts. The most basic functionality assumes that the closer gene\nis to the input locus, the more likely the gene is to be\ncausative. Additionally, any empirical data that links genomic\nregions to genes (e.g. eQTL or genome conformation data) can be\nused if it is supplied in the UCSC .BED file format.","downloads_month":194,"downloads_total":7246,"page":"http://bioconductor.org/packages/release/bioc/html/geneAttribution.html"},{"authors":"Evert van den Broek and Stef van Lieshout","name":"GeneBreak","license":"GPL-2","tags":["aCGH","CopyNumberVariation","DNASeq","Genetics","Sequencing","WholeGenome","Visualization"],"description":"Recurrent breakpoint gene detection on copy number\naberration profiles.","downloads_month":140,"downloads_total":10438,"page":"http://bioconductor.org/packages/release/bioc/html/GeneBreak.html"},{"authors":"R Kuiper","name":"geneClassifiers","license":"GPL-2","tags":["GeneExpression","BiomedicalInformatics","Classification","Survival","Microarray"],"description":"This packages aims for easy accessible application of\nclassifiers which have been published in literature using an\nExpressionSet as input.","downloads_month":134,"downloads_total":4784,"page":"http://bioconductor.org/packages/release/bioc/html/geneClassifiers.html"},{"authors":"Yang Cao","name":"GeneExpressionSignature","license":"GPL-2","tags":["GeneExpression"],"description":"This package gives the implementations of the gene\nexpression signature and its distance to each. Gene expression\nsignature is represented as a list of genes whose expression is\ncorrelated with a biological state of interest. And its\ndistance is defined using a nonparametric, rank-based\npattern-matching strategy based on the Kolmogorov-Smirnov\nstatistic. Gene expression signature and its distance can be\nused to detect similarities among the signatures of drugs,\ndiseases, and biological states of interest.","downloads_month":155,"downloads_total":36088,"page":"http://bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html"},{"authors":"R. Gentleman, V. Carey, W. Huber and F. Hahne","name":"genefilter","license":"Artistic-2.0","tags":["Microarray"],"description":"Some basic functions for filtering genes.","downloads_month":31248,"downloads_total":2877394,"page":"http://bioconductor.org/packages/release/bioc/html/genefilter.html"},{"authors":"Deena M.A. Gendoo, Natchar Ratanasirigulchai, Markus S.\nSchroder, Laia Pare, Joel S. Parker, Aleix Prat, and Benjamin\nHaibe-Kains","name":"genefu","license":"Artistic-2.0","tags":["DifferentialExpression","GeneExpression","Visualization","Clustering","Classification"],"description":"Description: This package contains functions implementing\nvarious tasks usually required by gene expression analysis,\nespecially in breast cancer studies: gene mapping between\ndifferent microarray platforms, identification of molecular\nsubtypes, implementation of published gene signatures, gene\nselection, and survival analysis.","downloads_month":431,"downloads_total":63760,"page":"http://bioconductor.org/packages/release/bioc/html/genefu.html"},{"authors":"Zhenpeng Li and Haixiu Huang","name":"GeneGA","license":"GPL version 2","tags":["GeneExpression"],"description":"R based Genetic algorithm for gene expression optimization\nby considering both mRNA secondary structure and codon usage\nbias, GeneGA includes the information of highly expressed genes\nof almost 200 genomes. Meanwhile, Vienna RNA Package is needed\nto ensure GeneGA to function properly.","downloads_month":136,"downloads_total":20980,"page":"http://bioconductor.org/packages/release/bioc/html/GeneGA.html"},{"authors":"Mathieu Emily, Nicolas Sounac, Florian Kroell and Magalie\nHouee-Bigot","name":"GeneGeneInteR","license":"GPL (>= 2)","tags":["GenomeWideAssociation","SNP","Genetics","GeneticVariability"],"description":"The aim of this package is to propose several methods for\ntesting gene-gene interaction in case-control association\nstudies. Such a test can be done by aggregating SNP-SNP\ninteraction tests performed at the SNP level (SSI) or by using\ngene-gene multidimensionnal methods (GGI) methods. The package\nalso proposes tools for a graphic display of the results.","downloads_month":136,"downloads_total":7606,"page":"http://bioconductor.org/packages/release/bioc/html/GeneGeneInteR.html"},{"authors":"Lara Lusa <lusa@ifom-firc.it>, R. Gentleman and M. Ruschhaupt","name":"GeneMeta","license":"Artistic-2.0","tags":["Sequencing","GeneExpression","Microarray"],"description":"A collection of meta-analysis tools for analysing high\nthroughput experimental data","downloads_month":192,"downloads_total":60386,"page":"http://bioconductor.org/packages/release/bioc/html/GeneMeta.html"},{"authors":"Jianhong Ou, Haibo Liu, Heidi A Tissenbaum and Lihua Julie Zhu","name":"GeneNetworkBuilder","license":"GPL (>= 2)","tags":["Sequencing","Microarray","GraphAndNetwork"],"description":"Appliation for discovering direct or indirect targets of\ntranscription factors using ChIP-chip or ChIP-seq, and\nmicroarray or RNA-seq gene expression data. Inputting a list of\ngenes of potential targets of one TF from ChIP-chip or\nChIP-seq, and the gene expression results, GeneNetworkBuilder\ngenerates a regulatory network of the TF.","downloads_month":176,"downloads_total":31840,"page":"http://bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html"},{"authors":"Li Shen and Mount Sinai <shenli.sam@gmail.com>","name":"GeneOverlap","license":"GPL-3","tags":["MultipleComparison","Visualization"],"description":"Test two sets of gene lists and visualize the results.","downloads_month":372,"downloads_total":27450,"page":"http://bioconductor.org/packages/release/bioc/html/GeneOverlap.html"},{"authors":"Rodrigo Dalmolin and Mauro Castro","name":"geneplast","license":"GPL (>= 2)","tags":["Genetics","GeneRegulation","SystemsBiology"],"description":"Geneplast is designed for evolutionary and plasticity\nanalysis based on orthologous groups distribution in a given\nspecies tree. It uses Shannon information theory and orthologs\nabundance to estimate the Evolutionary Plasticity Index.\nAdditionally, it implements the Bridge algorithm to determine\nthe evolutionary root of a given gene based on its orthologs\ndistribution.","downloads_month":141,"downloads_total":7146,"page":"http://bioconductor.org/packages/release/bioc/html/geneplast.html"},{"authors":"R. Gentleman and Biocore","name":"geneplotter","license":"Artistic-2.0","tags":["Visualization"],"description":"Functions for plotting genomic data","downloads_month":16588,"downloads_total":1978506,"page":"http://bioconductor.org/packages/release/bioc/html/geneplotter.html"},{"authors":"Gregory J. Hather <ghather@stat.berkeley.edu> and with\ncontributions from Art B. Owen <art@stat.stanford.edu> and\nTerence P. Speed <terry@stat.berkeley.edu>","name":"geneRecommender","license":"GPL (>= 2)","tags":["Microarray","Clustering"],"description":"This package contains a targeted clustering algorithm for\nthe analysis of microarray data. The algorithm can aid in the\ndiscovery of new genes with similar functions to a given list\nof genes already known to have closely related functions.","downloads_month":146,"downloads_total":47940,"page":"http://bioconductor.org/packages/release/bioc/html/geneRecommender.html"},{"authors":"Lasse Folkersen and Diego Diez","name":"GeneRegionScan","license":"GPL (>= 2)","tags":["Microarray","DataImport","SNP","OneChannel","Visualization"],"description":"A package with focus on analysis of discrete regions of\nthe genome. This package is useful for investigation of one or\na few genes using Affymetrix data, since it will extract probe\nlevel data using the Affymetrix Power Tools application and\nwrap these data into a ProbeLevelSet. A ProbeLevelSet directly\nextends the expressionSet, but includes additional information\nabout the sequence of each probe and the probe set it is\nderived from. The package includes a number of functions used\nfor plotting these probe level data as a function of location\nalong sequences of mRNA-strands. This can be used for analysis\nof variable splicing, and is especially well suited for use\nwith exon-array data.","downloads_month":154,"downloads_total":42944,"page":"http://bioconductor.org/packages/release/bioc/html/GeneRegionScan.html"},{"authors":"Charles Berry","name":"geneRxCluster","license":"GPL (>= 2)","tags":["Sequencing","Clustering","Genetics"],"description":"Detect Differential Clustering of Genomic Sites such as\ngene therapy integrations.  The package provides some functions\nfor exploring genomic insertion sites originating from two\ndifferent sources. Possibly, the two sources are two different\ngene therapy vectors.  Vectors are preferred that target\nsensitive regions less frequently, motivating the search for\nlocalized clusters of insertions and comparison of the clusters\nformed by integration of different vectors.  Scan statistics\nallow the discovery of spatial differences in clustering and\ncalculation of False Discovery Rates (FDRs) providing\nstatistical methods for comparing retroviral vectors. A scan\nstatistic for comparing two vectors using multiple window\nwidths to detect clustering differentials and compute FDRs is\nimplemented here.","downloads_month":141,"downloads_total":18986,"page":"http://bioconductor.org/packages/release/bioc/html/geneRxCluster.html"},{"authors":"Jarrett Morrow <remdj@channing.harvard.edu>, Weiliang Qiu\n<stwxq@channing.harvard.edu>, Wenqing He <whe@stats.uwo.ca>, Xiaogang Wang <stevenw@mathstat.yorku.ca> and Ross Lazarus\n<ross.lazarus@channing.harvard.edu>.","name":"GeneSelectMMD","license":"GPL (>= 2)","tags":["DifferentialExpression"],"description":"Gene selection based on a mixture of marginal\ndistributions","downloads_month":165,"downloads_total":42996,"page":"http://bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html"},{"authors":"Matthew P. Conomos, Stephanie M. Gogarten, Lisa Brown, Han\nChen, Ken Rice, Tamar Sofer, Timothy Thornton and Chaoyu Yu","name":"GENESIS","license":"GPL-3","tags":["SNP","GeneticVariability","Genetics","StatisticalMethod","DimensionReduction","PrincipalComponent","GenomeWideAssociation","QualityControl","BiocViews"],"description":"The GENESIS package provides methodology for estimating,\ninferring, and accounting for population and pedigree structure\nin genetic analyses.  The current implementation provides\nfunctions to perform PC-AiR (Conomos et al., 2015, Gen Epi) and\nPC-Relate (Conomos et al., 2016, AJHG). PC-AiR performs a\nPrincipal Components Analysis on genome-wide SNP data for the\ndetection of population structure in a sample that may contain\nknown or cryptic relatedness. Unlike standard PCA, PC-AiR\naccounts for relatedness in the sample to provide accurate\nancestry inference that is not confounded by family structure.\nPC-Relate uses ancestry representative principal components to\nadjust for population structure/ancestry and accurately\nestimate measures of recent genetic relatedness such as kinship\ncoefficients, IBD sharing probabilities, and inbreeding\ncoefficients. Additionally, functions are provided to perform\nefficient variance component estimation and mixed model\nassociation testing for both quantitative and binary\nphenotypes.","downloads_month":555,"downloads_total":44208,"page":"http://bioconductor.org/packages/release/bioc/html/GENESIS.html"},{"authors":"Beth Signal","name":"GeneStructureTools","license":"BSD_3_clause + file LICENSE","tags":["ImmunoOncology","Software","DifferentialSplicing","FunctionalPrediction","Transcriptomics","AlternativeSplicing","RNASeq"],"description":"GeneStructureTools can be used to create in silico\nalternative splicing events, and analyse potential effects this\nhas on functional gene products.","downloads_month":166,"downloads_total":2380,"page":"http://bioconductor.org/packages/release/bioc/html/GeneStructureTools.html"},{"authors":"Sara Aibar, Celia Fontanillo, Javier De Las Rivas.\nBioinformatics and Functional Genomics Group. Cancer Research\nCenter (CiC-IBMCC and CSIC/USAL). Salamanca. Spain.","name":"geNetClassifier","license":"GPL (>= 2)","tags":["Classification","DifferentialExpression","Microarray"],"description":"Comprehensive package to automatically train and validate\na multi-class SVM classifier based on gene expression data.\nProvides transparent selection of gene markers, their\ncoexpression networks, and an interface to query the\nclassifier.","downloads_month":197,"downloads_total":31354,"page":"http://bioconductor.org/packages/release/bioc/html/geNetClassifier.html"},{"authors":"Gregory Warnes <warnes@bst.rochester.edu> David Duffy\n<davidD@qumr.edu.au> and Michael Man <michael.man@pfizer.com>\nWeiliang Qiu <stwxq@channing.harvard.edu> Ross Lazarus\n<ross.lazarus@channing.harvard.edu>","name":"GeneticsDesign","license":"GPL-2","tags":["Genetics"],"description":"This package contains functions useful for designing\ngenetics studies, including power and sample-size calculations.","downloads_month":157,"downloads_total":52492,"page":"http://bioconductor.org/packages/release/bioc/html/GeneticsDesign.html"},{"authors":"Gregor Gorjanc, David A. Henderson <DNADavenator@GMail.Com> and with code contributions by Brian Kinghorn and Andrew Percy (see\nfile COPYING)","name":"GeneticsPed","license":"LGPL (>= 2.1) | file LICENSE","tags":["Genetics"],"description":"Classes and methods for handling pedigree data. It also\nincludes functions to calculate genetic relationship measures\nas relationship and inbreeding coefficients and other\nutilities. Note that package is not yet stable. Use it with\ncare!","downloads_month":196,"downloads_total":59062,"page":"http://bioconductor.org/packages/release/bioc/html/GeneticsPed.html"},{"authors":"Bohdan Khomtchouk [aut, cre] and William Koehler [aut]","name":"geneXtendeR","license":"GPL (>= 3)","tags":["ChIPSeq","Genetics","Annotation","GenomeAnnotation","DifferentialPeakCalling","Coverage","PeakDetection","ChipOnChip","HistoneModification","DataImport","NaturalLanguageProcessing","Visualization","GO","Software"],"description":"geneXtendeR optimizes the functional annotation of\nChIP-seq peaks by exploring relative differences in annotating\nChIP-seq peak sets to variable-length gene bodies.  In contrast\nto prior techniques, geneXtendeR considers peak annotations\nbeyond just the closest gene, allowing users to see peak\nsummary statistics for the first-closest gene, second-closest\ngene, ..., n-closest gene whilst ranking the output according\nto biologically relevant events and iteratively comparing the\nfidelity of peak-to-gene overlap across a user-defined range of\nupstream and downstream extensions on the original boundaries\nof each gene's coordinates.  Since different ChIP-seq peak\ncallers produce different differentially enriched peaks with a\nlarge variance in peak length distribution and total peak\ncount, annotating peak lists with their nearest genes can often\nbe a noisy process.  As such, the goal of geneXtendeR is to\nrobustly link differentially enriched peaks with their\nrespective genes, thereby aiding experimental follow-up and\nvalidation in designing primers for a set of prospective gene\ncandidates during qPCR.","downloads_month":139,"downloads_total":11822,"page":"http://bioconductor.org/packages/release/bioc/html/geneXtendeR.html"},{"authors":"Van Anh Huynh-Thu, Sara Aibar and Pierre Geurts","name":"GENIE3","license":"GPL (>= 2)","tags":["NetworkInference","SystemsBiology","DecisionTree","Regression","Network","GraphAndNetwork","GeneExpression"],"description":"This package implements the GENIE3 algorithm for inferring\ngene regulatory networks from expression data.","downloads_month":497,"downloads_total":16364,"page":"http://bioconductor.org/packages/release/bioc/html/GENIE3.html"},{"authors":"Wei Sun and ZhengZheng Tang","name":"genoCN","license":"GPL (>=2)","tags":["Microarray","Genetics"],"description":"Simultaneous identification of copy number states and\ngenotype calls for regions of either copy number variations or\ncopy number aberrations","downloads_month":132,"downloads_total":37462,"page":"http://bioconductor.org/packages/release/bioc/html/genoCN.html"},{"authors":"Georg Stricker [aut, cre], Alexander Engelhardt [aut] and Julien\nGagneur [aut]","name":"GenoGAM","license":"GPL-2","tags":["Regression","DifferentialPeakCalling","ChIPSeq","DifferentialExpression","Genetics","Epigenetics","WholeGenome","ChipOnChip","ImmunoOncology"],"description":"This package allows statistical analysis of genome-wide\ndata with smooth functions using generalized additive models\nbased on the implementation from the R-package 'mgcv'. It\nprovides methods for the statistical analysis of ChIP-Seq data\nincluding inference of protein occupancy, and pointwise and\nregion-wise differential analysis. Estimation of dispersion and\nsmoothing parameters is performed by cross-validation. Scaling\nof generalized additive model fitting to whole chromosomes is\nachieved by parallelization over overlapping genomic intervals.","downloads_month":180,"downloads_total":9460,"page":"http://bioconductor.org/packages/release/bioc/html/GenoGAM.html"},{"authors":"Altuna Akalin [aut, cre], Vedran Franke [aut, cre], Katarzyna\nWreczycka [aut], Alexander Gosdschan [ctb], Liz Ing-Simmons\n[ctb] and Bozena Mika-Gospodorz [ctb]","name":"genomation","license":"Artistic-2.0","tags":["Annotation","Sequencing","Visualization","CpGIsland"],"description":"A package for summary and annotation of genomic intervals.\nUsers can visualize and quantify genomic intervals over\npre-defined functional regions, such as promoters, exons,\nintrons, etc. The genomic intervals represent regions with a\ndefined chromosome position, which may be associated with a\nscore, such as aligned reads from HT-seq experiments, TF\nbinding sites, methylation scores, etc. The package can use any\ntabular genomic feature data as long as it has minimal\ninformation on the locations of genomic intervals. In addition,\nIt can use BAM or BigWig files as input.","downloads_month":488,"downloads_total":38030,"page":"http://bioconductor.org/packages/release/bioc/html/genomation.html"},{"authors":"Steffen Durinck <sdurinck@gmail.com> and James Bullard\n<bullard@berkeley.edu>","name":"GenomeGraphs","license":"Artistic-2.0","tags":["Visualization","Microarray"],"description":"Genomic data analyses requires integrated visualization of\nknown genomic information and new experimental data.\nGenomeGraphs uses the biomaRt package to perform live\nannotation queries to Ensembl and translates this to e.g.\ngene/transcript structures in viewports of the grid graphics\npackage. This results in genomic information plotted together\nwith your data.  Another strength of GenomeGraphs is to plot\ndifferent data types such as array CGH, gene expression,\nsequencing and other data, together in one plot using the same\ngenome coordinate system.","downloads_month":527,"downloads_total":131048,"page":"http://bioconductor.org/packages/release/bioc/html/GenomeGraphs.html"},{"authors":"Sonali Arora, Martin Morgan, Marc Carlson and H. Pagès","name":"GenomeInfoDb","license":"Artistic-2.0","tags":["Genetics","DataRepresentation","Annotation","GenomeAnnotation"],"description":"Contains data and functions that define and allow\ntranslation between different chromosome sequence naming\nconventions (e.g., \"chr1\" versus \"1\"), including a function\nthat attempts to place sequence names in their natural, rather\nthan lexicographic, order.","downloads_month":34356,"downloads_total":2672630,"page":"http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html"},{"authors":"Julien Gagneur <gagneur@genzentrum.lmu.de>, Joern Toedling, Richard Bourgon and Nicolas Delhomme","name":"genomeIntervals","license":"Artistic-2.0","tags":["DataImport","Infrastructure","Genetics"],"description":"This package defines classes for representing genomic\nintervals and provides functions and methods for working with\nthese. Note: The package provides the basic infrastructure for\nand is enhanced by the package 'girafe'.","downloads_month":380,"downloads_total":132058,"page":"http://bioconductor.org/packages/release/bioc/html/genomeIntervals.html"},{"authors":"Chris Stubben","name":"genomes","license":"GPL-3","tags":["Annotation","Genetics"],"description":"Download genome and assembly reports from NCBI","downloads_month":177,"downloads_total":43584,"page":"http://bioconductor.org/packages/release/bioc/html/genomes.html"},{"authors":"Hervé Pagès, Valerie Obenchain and Martin Morgan","name":"GenomicAlignments","license":"Artistic-2.0","tags":["ImmunoOncology","Genetics","Infrastructure","DataImport","Sequencing","RNASeq","SNP","Coverage","Alignment"],"description":"Provides efficient containers for storing and manipulating\nshort genomic alignments (typically obtained by aligning short\nreads to a reference genome). This includes read counting,\ncomputing the coverage, junction detection, and working with\nthe nucleotide content of the alignments.","downloads_month":29191,"downloads_total":1790314,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html"},{"authors":"Martin Morgan [aut], Sean Davis [aut and cre]","name":"GenomicDataCommons","license":"Artistic-2.0","tags":["DataImport","Sequencing"],"description":"Programmatically access the NIH / NCI Genomic Data Commons\nRESTful service.","downloads_month":995,"downloads_total":21860,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicDataCommons.html"},{"authors":"M. Carlson, H. Pagès, P. Aboyoun, S. Falcon, M. Morgan, D.\nSarkar, M. Lawrence and V. Obenchain","name":"GenomicFeatures","license":"Artistic-2.0","tags":["Genetics","Infrastructure","Annotation","Sequencing","GenomeAnnotation"],"description":"A set of tools and methods for making and manipulating\ntranscript centric annotations. With these tools the user can\neasily download the genomic locations of the transcripts, exons\nand cds of a given organism, from either the UCSC Genome\nBrowser or a BioMart database (more sources will be supported\nin the future). This information is then stored in a local\ndatabase that keeps track of the relationship between\ntranscripts, exons, cds and genes. Flexible methods are\nprovided for extracting the desired features in a convenient\nformat.","downloads_month":22480,"downloads_total":2641716,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html"},{"authors":"Bioconductor Package Maintainer [aut, cre], Valerie Obenchain\n[aut], Michael Love [aut], Lori Shepherd [aut] and Martin Morgan\n[aut]","name":"GenomicFiles","license":"Artistic-2.0","tags":["Genetics","Infrastructure","DataImport","Sequencing","Coverage"],"description":"This package provides infrastructure for parallel\ncomputations distributed 'by file' or 'by range'. User defined\nMAPPER and REDUCER functions provide added flexibility for data\ncombination and manipulation.","downloads_month":1206,"downloads_total":88922,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicFiles.html"},{"authors":"Harmston, N., Ing-Simmons, E., Perry, M., Baresic, A., Lenhard and B.","name":"GenomicInteractions","license":"GPL-3","tags":["Software","Infrastructure","DataImport","DataRepresentation","HiC"],"description":"R package for handling Genomic interaction data, such as\nChIA-PET/Hi-C, annotating genomic features with interaction\ninformation and producing various plots / statistics.","downloads_month":260,"downloads_total":25926,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicInteractions.html"},{"authors":"P. Aboyoun, H. Pagès, and M. Lawrence","name":"GenomicRanges","license":"Artistic-2.0","tags":["Genetics","Infrastructure","Sequencing","Annotation","Coverage","GenomeAnnotation"],"description":"The ability to efficiently represent and manipulate\ngenomic annotations and alignments is playing a central role\nwhen it comes to analyzing high-throughput sequencing data\n(a.k.a. NGS data). The GenomicRanges package defines general\npurpose containers for storing and manipulating genomic\nintervals and variables defined along a genome. More\nspecialized containers for representing and manipulating short\nalignments against a reference genome, or a matrix-like\nsummarization of an experiment, are defined in the\nGenomicAlignments and SummarizedExperiment packages,\nrespectively. Both packages build on top of the GenomicRanges\ninfrastructure.","downloads_month":37571,"downloads_total":3678796,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html"},{"authors":"Robert Castelo [aut, cre] and Pau Puigdevall [ctb]","name":"GenomicScores","license":"Artistic-2.0","tags":["Infrastructure","Genetics","Annotation","Sequencing","Coverage"],"description":"Provide infrastructure to store and access genomewide\nposition-specific scores within R and Bioconductor.","downloads_month":469,"downloads_total":17864,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicScores.html"},{"authors":"Peter Hickey <peter.hickey@gmail.com>, with contributions from\nMarcin Cieslik and Hervé Pagès.","name":"GenomicTuples","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation","Sequencing"],"description":"GenomicTuples defines general purpose containers for\nstoring genomic tuples. It aims to provide functionality for\ntuples of genomic co-ordinates that are analogous to those\navailable for genomic ranges in the GenomicRanges Bioconductor\npackage.","downloads_month":158,"downloads_total":17442,"page":"http://bioconductor.org/packages/release/bioc/html/GenomicTuples.html"},{"authors":"James Bullard and Kasper Daniel Hansen","name":"Genominator","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Tools for storing, accessing, analyzing and visualizing\ngenomic data.","downloads_month":170,"downloads_total":56556,"page":"http://bioconductor.org/packages/release/bioc/html/Genominator.html"},{"authors":"Peter M. Haverty","name":"genoset","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation","Microarray","SNP","CopyNumberVariation"],"description":"GenoSet provides an extension of the\nRangedSummarizedExperiment class with additional API features.\nThis class provides convenient and fast methods for working\nwith segmented genomic data. Additionally, GenoSet provides the\nclass RleDataFrame which stores runs of data along the genome\nfor multiple samples and provides very fast summaries of\narbitrary row sets (regions of the genome).","downloads_month":409,"downloads_total":68880,"page":"http://bioconductor.org/packages/release/bioc/html/genoset.html"},{"authors":"Jennifer Tom [aut and cre]","name":"genotypeeval","license":"file LICENSE","tags":["Genetics","BatchEffect","Sequencing","SNP","VariantAnnotation","DataImport"],"description":"Takes in a gVCF or VCF and reports metrics to assess\nquality of calls.","downloads_month":137,"downloads_total":10594,"page":"http://bioconductor.org/packages/release/bioc/html/genotypeeval.html"},{"authors":"Simo Kitanovski","name":"genphen","license":"GPL (>= 2)","tags":["GenomeWideAssociation","Regression","Classification","SupportVectorMachine","Genetics","SequenceMatching","Bayesian","FeatureExtraction","Sequencing"],"description":"Genetic association studies are an essential tool for\nstudying the relationship between genotypes and phenotypes.\nWith genphen we can jointly study multiple phenotypes of\ndifferent types, by quantifying the association between\ndifferent genotypes and each phenotype using a hybrid method\nwhich uses statistical learning techniques such as random\nforest and support vector machines, and Bayesian inference\nusing hierarchical models.","downloads_month":121,"downloads_total":8044,"page":"http://bioconductor.org/packages/release/bioc/html/genphen.html"},{"authors":"Chakravarthi Kanduri","name":"GenRank","license":"Artistic-2.0","tags":["GeneExpression","SNP","CopyNumberVariation","Microarray","Sequencing","Software","Genetics"],"description":"Methods for ranking genes based on convergent evidence\nobtained from multiple independent evidence layers. This\npackage adapts three methods that are popular for\nmeta-analysis.","downloads_month":145,"downloads_total":8282,"page":"http://bioconductor.org/packages/release/bioc/html/GenRank.html"},{"authors":"Zachary Skidmore [aut, cre], Alex Wagner [aut], Robert Lesurf\n[aut], Katie Campbell [aut], Jason Kunisaki [aut], Obi Griffith\n[aut] and Malachi Griffith [aut]","name":"GenVisR","license":"GPL-3 + file LICENSE","tags":["Infrastructure","DataRepresentation","Classification","DNASeq"],"description":"Produce highly customizable publication quality graphics\nfor genomic data primarily at the cohort level.","downloads_month":585,"downloads_total":29962,"page":"http://bioconductor.org/packages/release/bioc/html/GenVisR.html"},{"authors":"Jack Zhu and Sean Davis","name":"GEOmetadb","license":"Artistic-2.0","tags":["Infrastructure"],"description":"The NCBI Gene Expression Omnibus (GEO) represents the\nlargest public repository of microarray data. However, finding\ndata of interest can be challenging using current tools.\nGEOmetadb is an attempt to make access to the metadata\nassociated with samples, platforms, and datasets much more\nfeasible. This is accomplished by parsing all the NCBI GEO\nmetadata into a SQLite database that can be stored and queried\nlocally. GEOmetadb is simply a thin wrapper around the SQLite\ndatabase along with associated documentation. Finally, the\nSQLite database is updated regularly as new data is added to\nGEO and can be downloaded at will for the most up-to-date\nmetadata. GEOmetadb paper:\nhttp://bioinformatics.oxfordjournals.org/cgi/content/short/24/23/2798","downloads_month":1416,"downloads_total":96736,"page":"http://bioconductor.org/packages/release/bioc/html/GEOmetadb.html"},{"authors":"Sean Davis <sdavis2@mail.nih.gov>","name":"GEOquery","license":"GPL-2","tags":["Microarray","DataImport","OneChannel","TwoChannel","SAGE"],"description":"The NCBI Gene Expression Omnibus (GEO) is a public\nrepository of microarray data.  Given the rich and varied\nnature of this resource, it is only natural to want to apply\nBioConductor tools to these data.  GEOquery is the bridge\nbetween GEO and BioConductor.","downloads_month":11654,"downloads_total":1022986,"page":"http://bioconductor.org/packages/release/bioc/html/GEOquery.html"},{"authors":"Alexandre Kuhn <alexandre.m.kuhn@gmail.com>","name":"GEOsubmission","license":"GPL (>= 2)","tags":["Microarray"],"description":"Helps to easily submit a microarray dataset and the\nassociated sample information to GEO by preparing a single file\nfor upload (direct deposit).","downloads_month":142,"downloads_total":37962,"page":"http://bioconductor.org/packages/release/bioc/html/GEOsubmission.html"},{"authors":"Francesco Napolitano <franapoli@gmail.com>","name":"gep2pep","license":"GPL-3","tags":["GeneExpression","DifferentialExpression","GeneSetEnrichment","DimensionReduction","Pathways"],"description":"Pathway Expression Profiles (PEPs) are based on the\nexpression of pathways (defined as sets of genes) as opposed to\nindividual genes. This package converts gene expression\nprofiles to PEPs and performs enrichment analysis of both\npathways and experimental conditions, such as \"drug set\nenrichment analysis\" and \"gene2drug\" drug discovery analysis\nrespectively.","downloads_month":133,"downloads_total":2174,"page":"http://bioconductor.org/packages/release/bioc/html/gep2pep.html"},{"authors":"Fabian Schmich","name":"gespeR","license":"GPL-3","tags":["ImmunoOncology","CellBasedAssays","Preprocessing","GeneTarget","Regression","Visualization"],"description":"Estimates gene-specific phenotypes from off-target\nconfounded RNAi screens. The phenotype of each siRNA is modeled\nbased on on-targeted and off-targeted genes, using a\nregularized linear regression model.","downloads_month":226,"downloads_total":14728,"page":"http://bioconductor.org/packages/release/bioc/html/gespeR.html"},{"authors":"Wei Q. Deng and Guillaume Pare","name":"GEWIST","license":"GPL-2","tags":["MultipleComparison","Genetics"],"description":"This 'GEWIST' package provides statistical tools to\nefficiently optimize SNP prioritization for gene-gene and\ngene-environment interactions.","downloads_month":137,"downloads_total":28488,"page":"http://bioconductor.org/packages/release/bioc/html/GEWIST.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"GGBase","license":"Artistic-2.0","tags":["Genetics","Infrastructure"],"description":"infrastructure","downloads_month":203,"downloads_total":77936,"page":"http://bioconductor.org/packages/release/bioc/html/GGBase.html"},{"authors":"Tengfei Yin [aut], Michael Lawrence [aut, ths, cre], Dianne\nCook [aut, ths] and Johannes Rainer [ctb]","name":"ggbio","license":"Artistic-2.0","tags":["Infrastructure","Visualization"],"description":"The ggbio package extends and specializes the grammar of\ngraphics for biological data. The graphics are designed to\nanswer common scientific questions, in particular those often\nasked of high throughput genomics data. All core Bioconductor\ndata structures are supported, where appropriate. The package\nsupports detailed views of particular genomic regions, as well\nas genome-wide overviews. Supported overviews include ideograms\nand grand linear views. High-level plots include sequence\nfragment length, edge-linked interval to data view, mismatch\npileup, and several splicing summaries.","downloads_month":3009,"downloads_total":389762,"page":"http://bioconductor.org/packages/release/bioc/html/ggbio.html"},{"authors":"Mike Jiang","name":"ggcyto","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays","Infrastructure","Visualization"],"description":"With the dedicated fortify method implemented for flowSet,\nncdfFlowSet and GatingSet classes, both raw and gated flow\ncytometry data can be plotted directly with ggplot. ggcyto\nwrapper and some customed layers also make it easy to add gates\nand population statistics to the plot.","downloads_month":787,"downloads_total":29888,"page":"http://bioconductor.org/packages/release/bioc/html/ggcyto.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"GGtools","license":"Artistic-2.0","tags":["Genetics","GeneExpression","GeneticVariability","SNP"],"description":"software and data for analyses in genetics of gene\nexpression and/or DNA methylation","downloads_month":189,"downloads_total":72676,"page":"http://bioconductor.org/packages/release/bioc/html/GGtools.html"},{"authors":"Guangchuang Yu [aut, cre, cph]\n(<https://orcid.org/0000-0002-6485-8781>), Tommy Tsan-Yuk Lam\n[aut, ths], Justin Silverman [ctb], Bradley Jones [ctb] and Watal\nM. Iwasaki [ctb]","name":"ggtree","license":"Artistic-2.0","tags":["Alignment","Annotation","Clustering","DataImport","MultipleSequenceAlignment","ReproducibleResearch","Software","Visualization"],"description":"'ggtree' extends the 'ggplot2' plotting system which\nimplemented the grammar of graphics. 'ggtree' is designed for\nvisualization and annotation of phylogenetic trees with their\ncovariates and other associated data.","downloads_month":3792,"downloads_total":166276,"page":"http://bioconductor.org/packages/release/bioc/html/ggtree.html"},{"authors":"Shijia Zhu","name":"GIGSEA","license":"LGPL-3","tags":["GeneSetEnrichment","SNP","VariantAnnotation","GeneExpression","GeneRegulation","Regression","DifferentialExpression"],"description":"We presented the Genotype-imputed Gene Set Enrichment\nAnalysis (GIGSEA), a novel method that uses\nGWAS-and-eQTL-imputed trait-associated differential gene\nexpression to interrogate gene set enrichment for the\ntrait-associated SNPs. By incorporating eQTL from large gene\nexpression studies, e.g. GTEx, GIGSEA appropriately addresses\nsuch challenges for SNP enrichment as gene size, gene boundary,\nSNP distal regulation, and multiple-marker regulation. The\nweighted linear regression model, taking as weights both\nimputation accuracy and model completeness, was used to perform\nthe enrichment test, properly adjusting the bias due to\nredundancy in different gene sets. The permutation test,\nfurthermore, is used to evaluate the significance of\nenrichment, whose efficiency can be largely elevated by\nexpressing the computational intensive part in terms of large\nmatrix operation. We have shown the appropriate type I error\nrates for GIGSEA (<5%), and the preliminary results also\ndemonstrate its good performance to uncover the real signal.","downloads_month":79,"downloads_total":686,"page":"http://bioconductor.org/packages/release/bioc/html/GIGSEA.html"},{"authors":"Joern Toedling, with contributions from Constance Ciaudo, Olivier Voinnet, Edith Heard, Emmanuel Barillot, and Wolfgang\nHuber","name":"girafe","license":"Artistic-2.0","tags":["Sequencing"],"description":"The package 'girafe' deals with the genome-level\nrepresentation of aligned reads from next-generation sequencing\ndata. It contains an object class for enabling a detailed\ndescription of genome intervals with aligned reads and\nfunctions for comparing, visualising, exporting and working\nwith such intervals and the aligned reads. As such, the package\ninteracts with and provides a link between the packages\nShortRead, IRanges and genomeIntervals.","downloads_month":273,"downloads_total":57216,"page":"http://bioconductor.org/packages/release/bioc/html/girafe.html"},{"authors":"Bhakti Dwivedi and Jeanne Kowalski","name":"GISPA","license":"GPL-2","tags":["StatisticalMethod","GeneSetEnrichment","GenomeWideAssociation"],"description":"GISPA is a method intended for the researchers who are\ninterested in defining gene sets with similar, a priori\nspecified molecular profile. GISPA method has been previously\npublished in Nucleic Acid Research (Kowalski et al., 2016;\nPMID: 26826710).","downloads_month":154,"downloads_total":5208,"page":"http://bioconductor.org/packages/release/bioc/html/GISPA.html"},{"authors":"Philippe Hupe","name":"GLAD","license":"GPL-2","tags":["Microarray","CopyNumberVariation"],"description":"Analysis of array CGH data : detection of breakpoints in\ngenomic profiles and assignment of a status (gain, normal or\nloss) to each chromosomal regions identified.","downloads_month":481,"downloads_total":112550,"page":"http://bioconductor.org/packages/release/bioc/html/GLAD.html"},{"authors":"Shian Su [aut, cre], Matthew Ritchie [aut], Charity Law [aut] and Stuart Lee [ctb]","name":"Glimma","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","DifferentialExpression","GeneExpression","Microarray","ReportWriting","RNASeq","Sequencing","Visualization"],"description":"This package generates interactive visualisations for\nanalysis of RNA-sequencing data using output from limma, edgeR\nor DESeq2 packages in an HTML page. The interactions are built\non top of the popular static representations of analysis\nresults in order to provide additional information.","downloads_month":1732,"downloads_total":57612,"page":"http://bioconductor.org/packages/release/bioc/html/Glimma.html"},{"authors":"André Veríssimo [aut, cre], Susana Vinga [aut], Eunice\nCarrasquinha [ctb] and Marta Lopes [ctb]","name":"glmSparseNet","license":"GPL (>=3)","tags":["Software","StatisticalMethod","DimensionReduction","Regression","Classification","Survival","Network","GraphAndNetwork"],"description":"glmSparseNet is an R-package that generalizes sparse\nregression models when the features (e.g. genes) have a graph\nstructure (e.g. protein-protein interactions), by including\nnetwork-based regularizers. glmSparseNet uses the glmnet\nR-package, by including centrality measures of the network as\npenalty weights in the regularization. The current version\nimplements regularization based on node degree, i.e. the\nstrength and/or number of its associated edges, either by\npromoting hubs in the solution or orphan genes in the solution.\nAll the glmnet distribution families are supported, namely\n\"gaussian\", \"poisson\", \"binomial\", \"multinomial\", \"cox\", and\n\"mgaussian\".","downloads_month":98,"downloads_total":902,"page":"http://bioconductor.org/packages/release/bioc/html/glmSparseNet.html"},{"authors":"U. Mansmann, R. Meister, M. Hummel, R. Scheufele and with\ncontributions from S. Knueppel","name":"GlobalAncova","license":"GPL (>= 2)","tags":["Microarray","OneChannel","DifferentialExpression","Pathways","Regression"],"description":"The association between a variable of interest (e.g. two\ngroups) and the global pattern of a group of variables (e.g. a\ngene set) is tested via a global F-test. We give the following\narguments in support of the GlobalAncova approach: After\nappropriate normalisation, gene-expression-data appear rather\nsymmetrical and outliers are no real problem, so least squares\nshould be rather robust. ANCOVA with interaction yields\nsaturated data modelling e.g. different means per group and\ngene. Covariate adjustment can help to correct for possible\nselection bias. Variance homogeneity and uncorrelated residuals\ncannot be expected. Application of ordinary least squares gives\nunbiased, but no longer optimal estimates\n(Gauss-Markov-Aitken). Therefore, using the classical F-test is\ninappropriate, due to correlation. The test statistic however\nmirrors deviations from the null hypothesis. In combination\nwith a permutation approach, empirical significance levels can\nbe approximated. Alternatively, an approximation yields\nasymptotic p-values. The framework is generalized to groups of\ncategorical variables or even mixed data by a likelihood ratio\napproach. Closed and hierarchical testing procedures are\nsupported. This work was supported by the NGFN grant 01 GR\n0459, BMBF, Germany and BMBF grant 01ZX1309B, Germany.","downloads_month":508,"downloads_total":88216,"page":"http://bioconductor.org/packages/release/bioc/html/GlobalAncova.html"},{"authors":"Armin Rauschenberger [aut and cre]","name":"globalSeq","license":"GPL-3","tags":["GeneExpression","ExonArray","DifferentialExpression","GenomeWideAssociation","Transcriptomics","DimensionReduction","Regression","Sequencing","WholeGenome","RNASeq","ExomeSeq","miRNA","MultipleComparison"],"description":"The method may be conceptualised as a test of overall\nsignificance in regression analysis, where the response\nvariable is overdispersed and the number of explanatory\nvariables exceeds the sample size. Useful for testing for\nassociation between RNA-Seq and high-dimensional data.","downloads_month":148,"downloads_total":8432,"page":"http://bioconductor.org/packages/release/bioc/html/globalSeq.html"},{"authors":"Jelle Goeman, Jan Oosting, with contributions by Livio\nFinos and Aldo Solari and Dominic Edelmann","name":"globaltest","license":"GPL (>= 2)","tags":["Microarray","OneChannel","Bioinformatics","DifferentialExpression","GO","Pathways"],"description":"The global test tests groups of covariates (or features)\nfor association with a response variable. This package\nimplements the test with diagnostic plots and multiple testing\nutilities, along with several functions to facilitate the use\nof this test for gene set testing of GO and KEGG terms.","downloads_month":1355,"downloads_total":204674,"page":"http://bioconductor.org/packages/release/bioc/html/globaltest.html"},{"authors":"Cory Barr, Thomas Wu and Michael Lawrence","name":"gmapR","license":"Artistic-2.0","tags":["Alignment"],"description":"GSNAP and GMAP are a pair of tools to align short-read\ndata written by Tom Wu.  This package provides convenience\nmethods to work with GMAP and GSNAP from within R. In addition,\nit provides methods to tally alignment results on a\nper-nucleotide basis using the bam_tally tool.","downloads_month":326,"downloads_total":27682,"page":"http://bioconductor.org/packages/release/bioc/html/gmapR.html"},{"authors":"Yuan-De Tan","name":"GMRP","license":"GPL (>= 2)","tags":["Sequencing","Regression","SNP"],"description":"Perform Mendelian randomization analysis of multiple SNPs\nto determine risk factors causing disease of study and to\nexclude confounding variabels and perform path analysis to\nconstruct path of risk factors to the disease.","downloads_month":152,"downloads_total":7890,"page":"http://bioconductor.org/packages/release/bioc/html/GMRP.html"},{"authors":"Kevin Rue-Albrecht [aut, cre], Tharvesh M.L. Ali [ctb], Paul A.\nMcGettigan [ctb], Belinda Hernandez [ctb], David A. Magee\n[ctb], Nicolas C. Nalpas [ctb], Andrew Parnell [ctb], Stephen\nV. Gordon [ths] and David E. MacHugh [ths]","name":"GOexpress","license":"GPL (>= 3)","tags":["Software","GeneExpression","Transcription","DifferentialExpression","GeneSetEnrichment","DataRepresentation","Clustering","TimeCourse","Microarray","Sequencing","RNASeq","Annotation","MultipleComparison","Pathways","GO","Visualization","ImmunoOncology"],"description":"The package contains methods to visualise the expression\nprofile of genes from a microarray or RNA-seq experiment, and\noffers a supervised clustering approach to identify GO terms\ncontaining genes with expression levels that best classify two\nor more predefined groups of samples. Annotations for the genes\npresent in the expression dataset may be obtained from Ensembl\nthrough the biomaRt package, if not provided by the user. The\ndefault random forest framework is used to evaluate the\ncapacity of each gene to cluster samples according to the\nfactor of interest. Finally, GO terms are scored by averaging\nthe rank (alternatively, score) of their respective gene sets\nto cluster the samples. P-values may be computed to assess the\nsignificance of GO term ranking. Visualisation function include\ngene expression profile, gene ontology-based heatmaps, and\nhierarchical clustering of experimental samples using gene\nexpression data.","downloads_month":297,"downloads_total":26402,"page":"http://bioconductor.org/packages/release/bioc/html/GOexpress.html"},{"authors":"Steffi Grote","name":"GOfuncR","license":"GPL (>= 2)","tags":["GeneSetEnrichment","GO"],"description":"GOfuncR performs a gene ontology enrichment analysis based\non the ontology enrichment software FUNC. GO-annotations are\nobtained from OrganismDb or OrgDb packages ('Homo.sapiens' by\ndefault); the GO-graph is included in the package and updated\nregularly (11-Oct-2018). GOfuncR provides the standard\ncandidate vs. background enrichment analysis using the\nhypergeometric test, as well as three additional tests: (i) the\nWilcoxon rank-sum test that is used when genes are ranked, (ii)\na binomial test that is used when genes are associated with two\ncounts and (iii) a Chi-square or Fisher's exact test that is\nused in cases when genes are associated with four counts. To\ncorrect for multiple testing and interdependency of the tests,\nfamily-wise error rates are computed based on random\npermutations of the gene-associated variables. GOfuncR also\nprovides tools for exploring the ontology graph and the\nannotations, and options to take gene-length or spatial\nclustering of genes into account. It is also possible to\nprovide custom gene coordinates, annotations and ontologies.","downloads_month":222,"downloads_total":3218,"page":"http://bioconductor.org/packages/release/bioc/html/GOfuncR.html"},{"authors":"Jing Wang <jingwang.uestc@gmail.com>","name":"GOFunction","license":"GPL (>= 2)","tags":["GO","Pathways","Microarray","GeneSetEnrichment"],"description":"The GO-function package provides a tool to address the\nredundancy that result from the GO structure or multiple\nannotation genes and derive biologically relevant functions\nfrom the statistically significant functions based on some\nintuitive assumption and statistical testing.","downloads_month":180,"downloads_total":40982,"page":"http://bioconductor.org/packages/release/bioc/html/GOFunction.html"},{"authors":"Lidia Chrabaszcz","name":"GOpro","license":"GPL-3","tags":["Annotation","Clustering","GO","GeneExpression","GeneSetEnrichment","MultipleComparison"],"description":"Find the most characteristic gene ontology terms for\ngroups of human genes. This package was created as a part of\nthe thesis which was developed under the auspices of MI^2 Group\n(http://mi2.mini.pw.edu.pl/,\nhttps://github.com/geneticsMiNIng).","downloads_month":159,"downloads_total":7960,"page":"http://bioconductor.org/packages/release/bioc/html/GOpro.html"},{"authors":"Alex Sanchez, Jordi Ocana and Miquel Salicru","name":"goProfiles","license":"GPL-2","tags":["Annotation","GO","GeneExpression","GeneSetEnrichment","GraphAndNetwork","Microarray","MultipleComparison","Pathways","Software"],"description":"The package implements methods to compare lists of genes\nbased on comparing the corresponding 'functional profiles'.","downloads_month":248,"downloads_total":61338,"page":"http://bioconductor.org/packages/release/bioc/html/goProfiles.html"},{"authors":"Guangchuang Yu [aut, cre], Alexey Stukalov [ctb], Chuanle Xiao\n[ctb] and Lluís Revilla Sancho [ctb]","name":"GOSemSim","license":"Artistic-2.0","tags":["Annotation","GO","Clustering","Pathways","Network","Software"],"description":"The semantic comparisons of Gene Ontology (GO) annotations\nprovide quantitative ways to compute similarities between genes\nand gene groups, and have became important basis for many\nbioinformatics analysis approaches. GOSemSim is an R package\nfor semantic similarity computation among GO terms, sets of GO\nterms, gene products and gene clusters. GOSemSim implemented\nfive methods proposed by Resnik, Schlicker, Jiang, Lin and Wang\nrespectively.","downloads_month":6010,"downloads_total":362428,"page":"http://bioconductor.org/packages/release/bioc/html/GOSemSim.html"},{"authors":"Matthew Young","name":"goseq","license":"LGPL (>= 2)","tags":["ImmunoOncology","Sequencing","GO","GeneExpression","Transcription","RNASeq"],"description":"Detects Gene Ontology and/or other user defined categories\nwhich are over/under represented in RNA-seq data","downloads_month":2263,"downloads_total":205234,"page":"http://bioconductor.org/packages/release/bioc/html/goseq.html"},{"authors":"Holger Froehlich <frohlich@bit.uni-bonn.de>","name":"GOSim","license":"GPL (>= 2)","tags":["GO","Clustering","Software","Pathways"],"description":"This package implements several functions useful for\ncomputing similarities between GO terms and gene products based\non their GO annotation. Moreover it allows for computing a GO\nenrichment analysis","downloads_month":210,"downloads_total":33636,"page":"http://bioconductor.org/packages/release/bioc/html/GOSim.html"},{"authors":"Brian D. Bennett and Pierre R. Bushel","name":"goSTAG","license":"GPL-3","tags":["GeneExpression","DifferentialExpression","GeneSetEnrichment","Clustering","Microarray","mRNAMicroarray","RNASeq","Visualization","GO","ImmunoOncology"],"description":"Gene lists derived from the results of genomic analyses\nare rich in biological information. For instance,\ndifferentially expressed genes (DEGs) from a microarray or\nRNA-Seq analysis are related functionally in terms of their\nresponse to a treatment or condition. Gene lists can vary in\nsize, up to several thousand genes, depending on the robustness\nof the perturbations or how widely different the conditions are\nbiologically. Having a way to associate biological relatedness\nbetween hundreds and thousands of genes systematically is\nimpractical by manually curating the annotation and function of\neach gene. Over-representation analysis (ORA) of genes was\ndeveloped to identify biological themes. Given a Gene Ontology\n(GO) and an annotation of genes that indicate the categories\neach one fits into, significance of the over-representation of\nthe genes within the ontological categories is determined by a\nFisher's exact test or modeling according to a hypergeometric\ndistribution. Comparing a small number of enriched biological\ncategories for a few samples is manageable using Venn diagrams\nor other means for assessing overlaps. However, with hundreds\nof enriched categories and many samples, the comparisons are\nlaborious. Furthermore, if there are enriched categories that\nare shared between samples, trying to represent a common theme\nacross them is highly subjective. goSTAG uses GO subtrees to\ntag and annotate genes within a set. goSTAG visualizes the\nsimilarities between the over-representation of DEGs by\nclustering the p-values from the enrichment statistical tests\nand labels clusters with the GO term that has the most paths to\nthe root within the subtree generated from all the GO terms in\nthe cluster.","downloads_month":155,"downloads_total":5688,"page":"http://bioconductor.org/packages/release/bioc/html/goSTAG.html"},{"authors":"Robert Gentleman [aut], Seth Falcon [ctb], Robert Castelo\n[ctb] and Bioconductor Package Maintainer [cre]","name":"GOstats","license":"Artistic-2.0","tags":["Annotation","GO","MultipleComparison","GeneExpression","Microarray","Pathways","GeneSetEnrichment","GraphAndNetwork"],"description":"A set of tools for interacting with GO and microarray\ndata. A variety of basic manipulation tools for graphs,\nhypothesis testing and other simple calculations.","downloads_month":4531,"downloads_total":555942,"page":"http://bioconductor.org/packages/release/bioc/html/GOstats.html"},{"authors":"Raivo Kolde <raivo.kolde@eesti.ee>","name":"GOsummaries","license":"GPL (>= 2)","tags":["GeneExpression","Clustering","GO","Visualization"],"description":"A package to visualise Gene Ontology (GO) enrichment\nanalysis results on gene lists arising from different analyses\nsuch clustering or PCA. The significant GO categories are\nvisualised as word clouds that can be combined with different\nplots summarising the underlying data.","downloads_month":207,"downloads_total":22234,"page":"http://bioconductor.org/packages/release/bioc/html/GOsummaries.html"},{"authors":"Borbala Mifsud and Robert Sugar","name":"GOTHiC","license":"GPL-3","tags":["Sequencing","Preprocessing","Epigenetics","HiC"],"description":"This is a Hi-C analysis package using a cumulative\nbinomial test to detect interactions between distal genomic\nloci that have significantly more reads than expected by chance\nin Hi-C experiments. It takes mapped paired NGS reads as input\nand gives back the list of significant interactions for a given\nbin size in the genome.","downloads_month":285,"downloads_total":28046,"page":"http://bioconductor.org/packages/release/bioc/html/GOTHiC.html"},{"authors":"Yee Hwa (Jean) Yang <jean@biostat.ucsf.edu> and Agnes Paquet\n<paquetagnes@yahoo.com>","name":"goTools","license":"GPL-2","tags":["Microarray","GO","Visualization"],"description":"Wraper functions for description/comparison of oligo ID\nlist using Gene Ontology database","downloads_month":191,"downloads_total":70630,"page":"http://bioconductor.org/packages/release/bioc/html/goTools.html"},{"authors":"Sun Ah Kim [aut, cre, cph], Yun Joo Yoo [aut and cph]","name":"gpart","license":"MIT + file LICENSE","tags":["Software","Clustering"],"description":"we provide a new SNP sequence partitioning method which\npartitions the whole SNP sequence based on not only LD block\nstructures but also gene location information. The LD block\nconstruction for GPART is performed using Big-LD algorithm,\nwith additional improvement from previous version reported in\nKim et al.(2017). We also add a visualization tool to show the\nLD heatmap with the information of LD block boundaries and gene\nlocations in the package.","downloads_month":95,"downloads_total":804,"page":"http://bioconductor.org/packages/release/bioc/html/gpart.html"},{"authors":"Beiying Ding","name":"gpls","license":"Artistic-2.0","tags":["Classification","Microarray","Regression"],"description":"Classification using generalized partial least squares for\ntwo-group and multi-group (more than 2 group) classification.","downloads_month":251,"downloads_total":67214,"page":"http://bioconductor.org/packages/release/bioc/html/gpls.html"},{"authors":"Alfredo Kalaitzis <alkalait@gmail.com>","name":"gprege","license":"AGPL-3","tags":["Microarray","Preprocessing","Bioinformatics","DifferentialExpression","TimeCourse"],"description":"The gprege package implements the methodology described in\nKalaitzis & Lawrence (2011) \"A simple approach to ranking\ndifferentially expressed gene expression time-courses through\nGaussian process regression\". The software fits two GPs with\nthe an RBF (+ noise diagonal) kernel on each profile. One GP\nkernel is initialised wih a short lengthscale hyperparameter,\nsignal variance as the observed variance and a zero noise\nvariance. It is optimised via scaled conjugate gradients\n(netlab). A second GP has fixed hyperparameters: zero\ninverse-width, zero signal variance and noise variance as the\nobserved variance. The log-ratio of marginal likelihoods of the\ntwo hypotheses acts as a score of differential expression for\nthe profile. Comparison via ROC curves is performed against\nBATS (Angelini et.al, 2007). A detailed discussion of the\nranking approach and dataset used can be found in the paper\n(http://www.biomedcentral.com/1471-2105/12/180).","downloads_month":134,"downloads_total":29672,"page":"http://bioconductor.org/packages/release/bioc/html/gprege.html"},{"authors":"Britta Velten [aut, cre] and Wolfgang Huber [aut]","name":"graper","license":"GPL (>= 2)","tags":["Regression","Bayesian","Classification"],"description":"This package enables regression and classification on\nhigh-dimensional data with different relative strengths of\npenalization for different feature groups, such as different\nassays or omic types. The optimal relative strengths are chosen\nadaptively. Optimisation is performed using a variational Bayes\napproach.","downloads_month":32,"downloads_total":132,"page":"http://bioconductor.org/packages/release/bioc/html/graper.html"},{"authors":"R. Gentleman, Elizabeth Whalen, W. Huber and S. Falcon","name":"graph","license":"Artistic-2.0","tags":["GraphAndNetwork"],"description":"A package that implements some simple graph handling\ncapabilities.","downloads_month":18862,"downloads_total":1935772,"page":"http://bioconductor.org/packages/release/bioc/html/graph.html"},{"authors":"Joern P. Meier <mail@ionflux.org>, Michal Kolar, Ville\nMustonen, Michael Laessig, and Johannes Berg.","name":"GraphAlignment","license":"file LICENSE","tags":["GraphAndNetwork","Network"],"description":"Graph alignment is an extension package for the R\nprogramming environment which provides functions for finding an\nalignment between two networks based on link and node\nsimilarity scores. (J. Berg and M. Laessig, \"Cross-species\nanalysis of biological networks by Bayesian alignment\", PNAS\n103 (29), 10967-10972 (2006))","downloads_month":144,"downloads_total":44972,"page":"http://bioconductor.org/packages/release/bioc/html/GraphAlignment.html"},{"authors":"R. Balasubramanian, T. LaFramboise and D. Scholtens","name":"GraphAT","license":"LGPL","tags":["Network","GraphAndNetwork"],"description":"Functions and data used in Balasubramanian, et al. (2004)","downloads_month":137,"downloads_total":46026,"page":"http://bioconductor.org/packages/release/bioc/html/GraphAT.html"},{"authors":"Gabriele Sales <gabriele.sales@unipd.it>, Enrica Calura\n<enrica.calura@gmail.com> and Chiara Romualdi\n<chiara.romualdi@unipd.it>","name":"graphite","license":"AGPL-3","tags":["Pathways","ThirdPartyClient","GraphAndNetwork","Network","Reactome","KEGG","BioCarta","Metabolomics"],"description":"Graph objects from pathway topology derived from Biocarta,\nHumanCyc, KEGG, NCI, Panther, PathBank, PharmGKB, Reactome and\nSMPDB databases.","downloads_month":2202,"downloads_total":128366,"page":"http://bioconductor.org/packages/release/bioc/html/graphite.html"},{"authors":"Gregory Ryslik and Hongyu Zhao","name":"GraphPAC","license":"GPL-2","tags":["Clustering","Proteomics"],"description":"Identifies mutational clusters of amino acids in a protein\nwhile utilizing the proteins tertiary structure via a graph\ntheoretical model.","downloads_month":151,"downloads_total":26700,"page":"http://bioconductor.org/packages/release/bioc/html/GraphPAC.html"},{"authors":"Edward Morrissey","name":"GRENITS","license":"GPL (>= 2)","tags":["NetworkInference","GeneRegulation","TimeCourse","GraphAndNetwork","GeneExpression","Network","Bayesian"],"description":"The package offers four network inference statistical\nmodels using Dynamic Bayesian Networks and Gibbs Variable\nSelection: a linear interaction model, two linear interaction\nmodels with added experimental noise (Gaussian and Student\ndistributed) for the case where replicates are available and a\nnon-linear interaction model.","downloads_month":151,"downloads_total":33856,"page":"http://bioconductor.org/packages/release/bioc/html/GRENITS.html"},{"authors":"Gord Brown <gdbzork@gmail.com>","name":"GreyListChIP","license":"Artistic-2.0","tags":["ChIPSeq","Alignment","Preprocessing","DifferentialPeakCalling","Sequencing","GenomeAnnotation","Coverage"],"description":"Identify regions of ChIP experiments with high signal in\nthe input, that lead to spurious peaks during peak calling.\nRemove reads aligning to these regions prior to peak calling,\nfor cleaner ChIP analysis.","downloads_month":166,"downloads_total":13824,"page":"http://bioconductor.org/packages/release/bioc/html/GreyListChIP.html"},{"authors":"Nicholas Clark","name":"GRmetrics","license":"GPL-3","tags":["ImmunoOncology","CellBasedAssays","CellBiology","Software","TimeCourse","Visualization"],"description":"Functions for calculating and visualizing growth-rate\ninhibition (GR) metrics.","downloads_month":253,"downloads_total":10070,"page":"http://bioconductor.org/packages/release/bioc/html/GRmetrics.html"},{"authors":"Charles G. Danko, Minho Chae, Andre Martins and W. Lee Kraus","name":"groHMM","license":"GPL-3","tags":["Sequencing","Software"],"description":"A pipeline for the analysis of GRO-seq data.","downloads_month":157,"downloads_total":18468,"page":"http://bioconductor.org/packages/release/bioc/html/groHMM.html"},{"authors":"Mark A. van de Wiel <mark.vdwiel@vumc.nl> and Putri W. Novianti\n<p.novianti@vumc.nl>","name":"GRridge","license":"GPL-3","tags":["Classification","Regression","Survival","Bayesian","RNASeq","GenePrediction","GeneExpression","Pathways","GeneSetEnrichment","GO","KEGG","GraphAndNetwork","ImmunoOncology"],"description":"This package allows the use of multiple sources of co-data\n(e.g. external p-values, gene lists, annotation) to improve\nprediction of binary, continuous and survival response using\n(logistic, linear or Cox) group-regularized ridge regression.\nIt also facilitates post-hoc variable selection and prediction\ndiagnostics by cross-validation using ROC curves and AUC.","downloads_month":133,"downloads_total":5234,"page":"http://bioconductor.org/packages/release/bioc/html/GRridge.html"},{"authors":"Billy Heung Wing Chang","name":"GSALightning","license":"GPL (>=2)","tags":["Software","BiologicalQuestion","GeneSetEnrichment","DifferentialExpression","GeneExpression","Transcription"],"description":"GSALightning provides a fast implementation of\npermutation-based gene set analysis for two-sample problem.\nThis package is particularly useful when testing simultaneously\na large number of gene sets, or when a large number of\npermutations is necessary for more accurate p-values\nestimation.","downloads_month":135,"downloads_total":7886,"page":"http://bioconductor.org/packages/release/bioc/html/GSALightning.html"},{"authors":"Yasir Rahmatallah <yrahmatallah@uams.edu> and Galina Glazko\n<gvglazko@uams.edu>","name":"GSAR","license":"GPL (>=2)","tags":["Software","StatisticalMethod","DifferentialExpression"],"description":"Gene set analysis using specific alternative hypotheses.\nTests for differential expression, scale and net correlation\nstructure.","downloads_month":252,"downloads_total":21558,"page":"http://bioconductor.org/packages/release/bioc/html/GSAR.html"},{"authors":"Zhicheng Ji and Hongkai Ji","name":"GSCA","license":"GPL(>=2)","tags":["GeneExpression","Visualization","GUI"],"description":"GSCA takes as input several lists of activated and\nrepressed genes. GSCA then searches through a compendium of\npublicly available gene expression profiles for biological\ncontexts that are enriched with a specified pattern of gene\nexpression. GSCA provides both traditional R functions and\ninteractive, user-friendly user interface.","downloads_month":135,"downloads_total":18988,"page":"http://bioconductor.org/packages/release/bioc/html/GSCA.html"},{"authors":"Martin Morgan, Seth Falcon and Robert Gentleman","name":"GSEABase","license":"Artistic-2.0","tags":["GeneExpression","GeneSetEnrichment","GraphAndNetwork","GO","KEGG"],"description":"This package provides classes and methods to support Gene\nSet Enrichment Analysis (GSEA).","downloads_month":5980,"downloads_total":692562,"page":"http://bioconductor.org/packages/release/bioc/html/GSEABase.html"},{"authors":"Ludwig Geistlinger [aut, cre], Gergely Csaba [aut], Mara\nSantarelli [ctb], Lucas Schiffer [ctb], Marcel Ramos [ctb], Ralf Zimmer [aut] and Levi Waldron [aut]","name":"GSEABenchmarkeR","license":"Artistic-2.0","tags":["ImmunoOncology","Microarray","RNASeq","GeneExpression","DifferentialExpression","Pathways","GraphAndNetwork","Network","GeneSetEnrichment","NetworkEnrichment","Visualization","ReportWriting"],"description":"The GSEABenchmarkeR package implements an extendable\nframework for reproducible evaluation of set- and network-based\nmethods for enrichment analysis of gene expression data. This\nincludes support for the efficient execution of these methods\non comprehensive real data compendia (microarray and RNA-seq)\nusing parallel computation on standard workstations and\ninstitutional computer grids. Methods can then be assessed with\nrespect to runtime, statistical significance, and relevance of\nthe results for the phenotypes investigated.","downloads_month":183,"downloads_total":3134,"page":"http://bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html"},{"authors":"Assaf Oron and Robert Gentleman (with contributions from S. Falcon\nand Z. Jiang)","name":"GSEAlm","license":"Artistic-2.0","tags":["Microarray"],"description":"Models and methods for fitting linear models to gene\nexpression data, together with tools for computing and using\nvarious regression diagnostics.","downloads_month":262,"downloads_total":63076,"page":"http://bioconductor.org/packages/release/bioc/html/GSEAlm.html"},{"authors":"Dongmin Jung","name":"gsean","license":"Artistic-2.0","tags":["Software","StatisticalMethod","Network","GraphAndNetwork","GeneSetEnrichment","GeneExpression","NetworkEnrichment","Pathways","DifferentialExpression"],"description":"Biological molecules in a living organism seldom work\nindividually. They usually interact each other in a cooperative\nway. Biological process is too complicated to understand\nwithout considering such interactions. Thus, network-based\nprocedures can be seen as powerful methods for studying complex\nprocess. However, many methods are devised for analyzing\nindividual genes. It is said that techniques based on\nbiological networks such as gene co-expression are more precise\nways to represent information than those using lists of genes\nonly. This package is aimed to integrate the gene expression\nand biological network. A biological network is constructed\nfrom gene expression data and it is used for Gene Set\nEnrichment Analysis.","downloads_month":114,"downloads_total":2132,"page":"http://bioconductor.org/packages/release/bioc/html/gsean.html"},{"authors":"Bahman Afsari <bahman@jhu.edu> and Elana J. Fertig\n<ejfertig@jhmi.edu>","name":"GSReg","license":"GPL-2","tags":["GeneRegulation","Pathways","GeneExpression","GeneticVariability","GeneSetEnrichment","AlternativeSplicing"],"description":"A package for gene set analysis based on the variability\nof expressions as well as a method to detect Alternative\nSplicing Events . It implements DIfferential RAnk Conservation\n(DIRAC) and gene set Expression Variation Analysis (EVA)\nmethods. For detecting Differentially Spliced genes, it\nprovides an implementation of the Spliced-EVA (SEVA).","downloads_month":146,"downloads_total":15198,"page":"http://bioconductor.org/packages/release/bioc/html/GSReg.html"},{"authors":"Julian Gehring, Kilian Bartholome, Clemens Kreutz and Jens Timmer","name":"GSRI","license":"GPL-3","tags":["Microarray","Transcription","DifferentialExpression","GeneSetEnrichment","GeneRegulation"],"description":"The GSRI package estimates the number of differentially\nexpressed genes in gene sets, utilizing the concept of the Gene\nSet Regulation Index (GSRI).","downloads_month":163,"downloads_total":37760,"page":"http://bioconductor.org/packages/release/bioc/html/GSRI.html"},{"authors":"Justin Guinney [aut, cre], Robert Castelo [aut] and Joan Fernandez\n[ctb]","name":"GSVA","license":"GPL (>= 2)","tags":["Microarray","Pathways","GeneSetEnrichment"],"description":"Gene Set Variation Analysis (GSVA) is a non-parametric,\nunsupervised method for estimating variation of gene set\nenrichment through the samples of a expression data set. GSVA\nperforms a change in coordinate systems, transforming the data\nfrom a gene by sample matrix to a gene-set by sample matrix,\nthereby allowing the evaluation of pathway enrichment for each\nsample. This new matrix of GSVA enrichment scores facilitates\napplying standard analytical methods like functional\nenrichment, survival analysis, clustering, CNV-pathway analysis\nor cross-tissue pathway analysis, in a pathway-centric manner.","downloads_month":1542,"downloads_total":106926,"page":"http://bioconductor.org/packages/release/bioc/html/GSVA.html"},{"authors":"Zuguang Gu","name":"gtrellis","license":"MIT + file LICENSE","tags":["Software","Visualization","Sequencing"],"description":"Genome level Trellis graph visualizes genomic data\nconditioned by genomic categories (e.g. chromosomes). For each\ngenomic category, multiple dimensional data which are\nrepresented as tracks describe different features from\ndifferent aspects. This package provides high flexibility to\narrange genomic categories and to add self-defined graphics in\nthe plot.","downloads_month":202,"downloads_total":16382,"page":"http://bioconductor.org/packages/release/bioc/html/gtrellis.html"},{"authors":"Lihua Julie Zhu, Michael Lawrence, Ankit Gupta, Hervé Pagès, Alper Kucukural, Manuel Garber and Scot A. Wolfe","name":"GUIDEseq","license":"GPL (>= 2)","tags":["ImmunoOncology","GeneRegulation","Sequencing","WorkflowStep","CRISPR"],"description":"The package implements GUIDE-seq analysis workflow\nincluding functions for obtaining unique insertion sites (proxy\nof cleavage sites), estimating the locations of the insertion\nsites, aka, peaks, merging estimated insertion sites from plus\nand minus strand, and performing off target search of the\nextended regions around insertion sites.","downloads_month":207,"downloads_total":12424,"page":"http://bioconductor.org/packages/release/bioc/html/GUIDEseq.html"},{"authors":"Jia Meng","name":"Guitar","license":"GPL-2","tags":["ImmunoOncology","Sequencing","SplicedAlignment","Alignment","DataImport","RNASeq","MethylSeq","QualityControl","Transcription","Coverage"],"description":"The package is designed for visualization of RNA-related\ngenomic features with respect to the landmarks of RNA\ntranscripts, i.e., transcription starting site, start codon,\nstop codon and transcription ending site.","downloads_month":219,"downloads_total":12350,"page":"http://bioconductor.org/packages/release/bioc/html/Guitar.html"},{"authors":"Florian Hahne, Steffen Durinck, Robert Ivanek, Arne Mueller, Steve Lianoglou, Ge Tan <ge.tan09@imperial.ac.uk>, Lance\nParsons <lparsons@princeton.edu> and Shraddha Pai\n<shraddha.pai@utoronto.ca>","name":"Gviz","license":"Artistic-2.0","tags":["Visualization","Microarray"],"description":"Genomic data analyses requires integrated visualization of\nknown genomic information and new experimental data.  Gviz uses\nthe biomaRt and the rtracklayer packages to perform live\nannotation queries to Ensembl and UCSC and translates this to\ne.g. gene/transcript structures in viewports of the grid\ngraphics package. This results in genomic information plotted\ntogether with your data.","downloads_month":5202,"downloads_total":903382,"page":"http://bioconductor.org/packages/release/bioc/html/Gviz.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"gwascat","license":"Artistic-2.0","tags":["Genetics"],"description":"Represent and model data in the EMBL-EBI GWAS catalog.","downloads_month":387,"downloads_total":50734,"page":"http://bioconductor.org/packages/release/bioc/html/gwascat.html"},{"authors":"Stephanie M. Gogarten, Cathy Laurie, Tushar Bhangale, Matthew\nP. Conomos, Cecelia Laurie, Michael Lawrence, Caitlin McHugh, Ian Painter, Xiuwen Zheng, Jess Shen, Rohit Swarnkar, Adrienne\nStilp and Sarah Nelson","name":"GWASTools","license":"Artistic-2.0","tags":["SNP","GeneticVariability","QualityControl","Microarray"],"description":"Classes for storing very large GWAS data sets and\nannotation, and functions for GWAS data cleaning and analysis.","downloads_month":749,"downloads_total":91614,"page":"http://bioconductor.org/packages/release/bioc/html/GWASTools.html"},{"authors":"Abbas Rizvi, Ezgi Karaesmen, Martin Morgan and Lara\nSucheston-Campbell","name":"gwasurvivr","license":"Artistic-2.0","tags":["GenomeWideAssociation","Survival","Regression","Genetics","SNP","GeneticVariability","Pharmacogenomics","BiomedicalInformatics"],"description":"gwasurvivr is a package to perform survival analysis using\nCox proportional hazard models on imputed genetic data.","downloads_month":92,"downloads_total":1086,"page":"http://bioconductor.org/packages/release/bioc/html/gwasurvivr.html"},{"authors":"Sepp Hochreiter <hochreit@bioinf.jku.at>","name":"hapFabia","license":"LGPL (>= 2.1)","tags":["Genetics","GeneticVariability","SNP","Sequencing","Sequencing","Visualization","Clustering","SequenceMatching","Software"],"description":"A package to identify very short IBD segments in large\nsequencing data by FABIA biclustering. Two haplotypes are\nidentical by descent (IBD) if they share a segment that both\ninherited from a common ancestor. Current IBD methods reliably\ndetect long IBD segments because many minor alleles in the\nsegment are concordant between the two haplotypes. However,\nmany cohort studies contain unrelated individuals which share\nonly short IBD segments. This package provides software to\nidentify short IBD segments in sequencing data. Knowledge of\nshort IBD segments are relevant for phasing of genotyping data,\nassociation studies, and for population genetics, where they\nshed light on the evolutionary history of humans. The package\nsupports VCF formats, is based on sparse matrix operations, and\nprovides visualization of haplotype clusters in different\nformats.","downloads_month":150,"downloads_total":30040,"page":"http://bioconductor.org/packages/release/bioc/html/hapFabia.html"},{"authors":"Josh Bowden [aut], Jason Ross [aut, cre] and Yalchin Oytam [aut]","name":"Harman","license":"GPL-3 + file LICENCE","tags":["BatchEffect","Microarray","MultipleComparison","PrincipalComponent","Normalization","Preprocessing","DNAMethylation","Transcription","Software","StatisticalMethod"],"description":"Harman is a PCA and constrained optimisation based\ntechnique that maximises the removal of batch effects from\ndatasets, with the constraint that the probability of\novercorrection (i.e. removing genuine biological signal along\nwith batch noise) is kept to a fraction which is set by the\nend-user.","downloads_month":460,"downloads_total":10450,"page":"http://bioconductor.org/packages/release/bioc/html/Harman.html"},{"authors":"Mayte Suarez-Farinas, Maurizio Pellegrino, Knut M. Wittkowski and Marcelo O. Magnasco","name":"Harshlight","license":"GPL (>= 2)","tags":["Microarray","QualityControl","Preprocessing","OneChannel","ReportWriting"],"description":"The package is used to detect extended, diffuse and\ncompact blemishes on microarray chips. Harshlight automatically\nmarks the areas in a collection of chips (affybatch objects)\nand a corrected AffyBatch object is returned, in which the\ndefected areas are substituted with NAs or the median of the\nvalues of the same probe in the other chips in the collection.\nThe new version handle the substitute value as whole matrix to\nsolve the memory problem.","downloads_month":142,"downloads_total":46060,"page":"http://bioconductor.org/packages/release/bioc/html/Harshlight.html"},{"authors":"Hervé Pagès","name":"HDF5Array","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation","DataImport","Sequencing","RNASeq","Coverage","Annotation","GenomeAnnotation","SingleCell","ImmunoOncology"],"description":"Implements the HDF5Array and TENxMatrix classes, 2\nconvenient and memory-efficient array-like containers for\non-disk representation of HDF5 datasets. HDF5Array is for\ndatasets that use the conventional (i.e. dense) HDF5\nrepresentation. TENxMatrix is for datasets that use the\nHDF5-based sparse matrix representation from 10x Genomics (e.g.\nthe 1.3 Million Brain Cell Dataset). Both containers being\nDelayedArray extensions, they support all operations supported\nby DelayedArray objects. These operations can be either delayed\nor block-processed.","downloads_month":5752,"downloads_total":141096,"page":"http://bioconductor.org/packages/release/bioc/html/HDF5Array.html"},{"authors":"Anestis Touloumis [cre, aut], John C. Marioni [aut] and Simon\nTavar\\'{e} [aut]","name":"HDTD","license":"GPL-3","tags":["DifferentialExpression","Genetics","GeneExpression","Microarray","Sequencing","StatisticalMethod","Software"],"description":"Characterization of intra-individual variability using\nphysiologically relevant measurements provides important\ninsights into fundamental biological questions ranging from\ncell type identity to tumor development. For each individual,\nthe data measurements can be written as a matrix with the\ndifferent subsamples of the individual recorded in the columns\nand the different phenotypic units recorded in the rows.\nDatasets of this type are called high-dimensional transposable\ndata. The HDTD package provides functions for conducting\nstatistical inference for the mean relationship between the row\nand column variables and for the covariance structure within\nand between the row and column variables.","downloads_month":135,"downloads_total":15580,"page":"http://bioconductor.org/packages/release/bioc/html/HDTD.html"},{"authors":"Malcolm Perry <mgperry32@gmail.com>","name":"heatmaps","license":"Artistic-2.0","tags":["Visualization","SequenceMatching","FunctionalGenomics"],"description":"This package provides functions for plotting heatmaps of\ngenome-wide data across genomic intervals, such as ChIP-seq\nsignals at peaks or across promoters. Many functions are also\nprovided for investigating sequence features.","downloads_month":316,"downloads_total":10100,"page":"http://bioconductor.org/packages/release/bioc/html/heatmaps.html"},{"authors":"Alexander Ploner <Alexander.Ploner@ki.se>","name":"Heatplus","license":"GPL (>= 2)","tags":["Microarray","Visualization"],"description":"Display a rectangular heatmap (intensity plot) of a data\nmatrix. By default, both samples (columns) and features (row)\nof the matrix are sorted according to a hierarchical\nclustering, and the corresponding dendrogram is plotted.\nOptionally, panels with additional information about samples\nand features can be added to the plot.","downloads_month":896,"downloads_total":188140,"page":"http://bioconductor.org/packages/release/bioc/html/Heatplus.html"},{"authors":"Michael Lawrence","name":"HelloRanges","license":"GPL (>= 2)","tags":["Sequencing","Annotation","Coverage","GenomeAnnotation","DataImport","SequenceMatching","VariantAnnotation"],"description":"Translates bedtools command-line invocations to R code\ncalling functions from the Bioconductor *Ranges infrastructure.\nThis is intended to educate novice Bioconductor users and to\ncompare the syntax and semantics of the two frameworks.","downloads_month":192,"downloads_total":9098,"page":"http://bioconductor.org/packages/release/bioc/html/HelloRanges.html"},{"authors":"Reid F. Thompson <reid.thompson@gmail.com>, John M. Greally\n<john.greally@einstein.yu.edu> and with contributions from Mark\nReimers <mreimers@vcu.edu>","name":"HELP","license":"GPL (>= 2)","tags":["CpGIsland","DNAMethylation","Microarray","TwoChannel","DataImport","QualityControl","Preprocessing","Visualization"],"description":"The package contains a modular pipeline for analysis of\nHELP microarray data, and includes graphical and mathematical\ntools with more general applications.","downloads_month":159,"downloads_total":45228,"page":"http://bioconductor.org/packages/release/bioc/html/HELP.html"},{"authors":"HyungJun Cho <hcho@virginia.edu> and Jae K. Lee\n<jaeklee@virginia.edu>","name":"HEM","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression"],"description":"This package fits heterogeneous error models for analysis\nof microarray data","downloads_month":139,"downloads_total":45304,"page":"http://bioconductor.org/packages/release/bioc/html/HEM.html"},{"authors":"Nirav V Malani <malnirav@gmail.com>","name":"hiAnnotator","license":"GPL (>= 2)","tags":["Software","Annotation"],"description":"hiAnnotator contains set of functions which allow users to\nannotate a GRanges object with custom set of annotations. The\nbasic philosophy of this package is to take two GRanges objects\n(query & subject) with common set of seqnames (i.e.\nchromosomes) and return associated annotation per seqnames and\nrows from the query matching seqnames and rows from the subject\n(i.e. genes or cpg islands). The package comes with three types\nof annotation functions which calculates if a position from\nquery is: within a feature, near a feature, or count features\nin defined window sizes. Moreover, each function is equipped\nwith parallel backend to utilize the foreach package. In\naddition, the package is equipped with wrapper functions, which\nfinds appropriate columns needed to make a GRanges object from\na common data frame.","downloads_month":193,"downloads_total":17766,"page":"http://bioconductor.org/packages/release/bioc/html/hiAnnotator.html"},{"authors":"Xiuwen Zheng [aut, cre, cph], Bruce Weir [ctb and ths]","name":"HIBAG","license":"GPL-3","tags":["Genetics","StatisticalMethod"],"description":"It is a software package for imputing HLA types using SNP\ndata, and relies on a training set of HLA and SNP genotypes.\nHIBAG can be used by researchers with published parameter\nestimates instead of requiring access to large training sample\ndatasets. It combines the concepts of attribute bagging, an\nensemble classifier method, with haplotype inference for SNPs\nand HLA types. Attribute bagging is a technique which improves\nthe accuracy and stability of classifier ensembles using\nbootstrap aggregating and random variable selection.","downloads_month":248,"downloads_total":17690,"page":"http://bioconductor.org/packages/release/bioc/html/HIBAG.html"},{"authors":"Koustav Pal [aut, cre], Carmen Livi [ctb] and Ilario Tagliaferri\n[ctb]","name":"HiCBricks","license":"MIT + file LICENSE","tags":["DataImport","Infrastructure","Software","Technology","Sequencing","HiC"],"description":"A flexible framework for storing and accessing\nhigh-resolution Hi-C data through HDF files. HiCBricks allows\nimport of Hi-C data through various formats such as the 2D\nmatrix format or a generalized n-column table formats. In terms\nof access, HiCBricks offers functions to retrieve values from\ngenomic loci separated by a certain distance, or the ability to\nfetch matrix subsets using word alike terms. HiCBricks will at\na later point offer the ability to fetch multiple matrix\nsubsets using fewer calls. It offers the capacity to store\nGenomicRanges that may be associated to a particular Hi-C\nexperiment, to do basic ranges overlap (any, within) with the\nHi-C experiment associated Ranges object and also to store any\nmetadata that users may think to be relevant for their Hi-C\nexperiment. Finally, you can do TAD calls with LSD and create\npretty heatmaps.","downloads_month":91,"downloads_total":628,"page":"http://bioconductor.org/packages/release/bioc/html/HiCBricks.html"},{"authors":"John Stansfield [aut, cre], Kellen Cresswell [aut] and Mikhail\nDozmorov [aut]","name":"HiCcompare","license":"MIT + file LICENSE","tags":["Software","HiC","Sequencing","Normalization"],"description":"HiCcompare provides functions for joint normalization and\ndifference detection in multiple Hi-C datasets. HiCcompare\noperates on processed Hi-C data in the form of\nchromosome-specific chromatin interaction matrices. It accepts\nthree-column tab-separated text files storing chromatin\ninteraction matrices in a sparse matrix format which are\navailable from several sources. HiCcompare is designed to give\nthe user the ability to perform a comparative analysis on the\n3-Dimensional structure of the genomes of cells in different\nbiological states.`HiCcompare` differs from other packages that\nattempt to compare Hi-C data in that it works on processed data\nin chromatin interaction matrix format instead of pre-processed\nsequencing data. In addition, `HiCcompare` provides a\nnon-parametric method for the joint normalization and removal\nof biases between two Hi-C datasets for the purpose of\ncomparative analysis. `HiCcompare` also provides a simple yet\nrobust method for detecting differences between Hi-C datasets.","downloads_month":202,"downloads_total":6654,"page":"http://bioconductor.org/packages/release/bioc/html/HiCcompare.html"},{"authors":"Tao Yang [aut and cre]","name":"hicrep","license":"GPL (>= 2.0)","tags":["Sequencing","HiC","QualityControl"],"description":"Hi-C is a powerful technology for studying genome-wide\nchromatin interactions. However, current methods for assessing\nHi-C data reproducibility can produce misleading results\nbecause they ignore spatial features in Hi-C data, such as\ndomain structure and distance-dependence. We present a novel\nreproducibility measure that systematically takes these\nfeatures into consideration. This measure can assess pairwise\ndifferences between Hi-C matrices under a wide range of\nsettings, and can be used to determine optimal sequencing\ndepth. Compared to existing approaches, it consistently shows\nhigher accuracy in distinguishing subtle differences in\nreproducibility and depicting interrelationships of cell\nlineages than existing approaches. This R package `hicrep`\nimplements our approach.","downloads_month":153,"downloads_total":5998,"page":"http://bioconductor.org/packages/release/bioc/html/hicrep.html"},{"authors":"Laura Buzdugan","name":"hierGWAS","license":"GPL-3","tags":["SNP","LinkageDisequilibrium","Clustering"],"description":"Testing individual SNPs, as well as arbitrarily large\ngroups of SNPs in GWA studies, using a joint model of all SNPs.\nThe method controls the FWER, and provides an automatic,\ndata-driven refinement of the SNP clusters to smaller groups or\nsingle markers.","downloads_month":144,"downloads_total":10448,"page":"http://bioconductor.org/packages/release/bioc/html/hierGWAS.html"},{"authors":"Claude Renaux [aut, cre], Laura Buzdugan [aut], Markus Kalisch\n[aut] and Peter Bühlmann [aut]","name":"hierinf","license":"GPL-3 | file LICENSE","tags":["Clustering","GenomeWideAssociation","LinkageDisequilibrium","Regression","SNP"],"description":"Tools to perform hierarchical inference for one or\nmultiple studies / data sets based on high-dimensional\nmultivariate (generalised) linear models. A possible\napplication is to perform hierarchical inference for GWA\nstudies to find significant groups or single SNPs (if the\nsignal is strong) in a data-driven and automated procedure. The\nmethod is based on an efficient hierarchical multiple testing\ncorrection and controls the FWER. The functions can easily be\nrun in parallel.","downloads_month":74,"downloads_total":620,"page":"http://bioconductor.org/packages/release/bioc/html/hierinf.html"},{"authors":"Zuguang Gu","name":"HilbertCurve","license":"MIT + file LICENSE","tags":["Software","Visualization","Sequencing","Coverage","GenomeAnnotation"],"description":"Hilbert curve is a type of space-filling curves that fold\none dimensional axis into a two dimensional space, but with\nstill preserves the locality. This package aims to provide an\neasy and flexible way to visualize data through Hilbert curve.","downloads_month":183,"downloads_total":13988,"page":"http://bioconductor.org/packages/release/bioc/html/HilbertCurve.html"},{"authors":"Simon Anders <sanders@fs.tum.de>","name":"HilbertVis","license":"GPL (>= 3)","tags":["Visualization"],"description":"Functions to visualize long vectors of integer data by\nmeans of Hilbert curves","downloads_month":338,"downloads_total":100568,"page":"http://bioconductor.org/packages/release/bioc/html/HilbertVis.html"},{"authors":"Simon Anders <sanders@fs.tum.de>","name":"HilbertVisGUI","license":"GPL (>= 3)","tags":["Visualization"],"description":"An interactive tool to visualize long vectors of integer\ndata by means of Hilbert curves","downloads_month":129,"downloads_total":43618,"page":"http://bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html"},{"authors":"Marta R. Hidalgo [aut, cre], José Carbonell-Caballero [ctb], Francisco Salavert [ctb], Alicia Amadoz [ctb], Çankut Cubuk\n[ctb] and Joaquin Dopazo [ctb]","name":"hipathia","license":"GPL-2","tags":["Pathways","GraphAndNetwork","GeneExpression","GeneSignaling","GO"],"description":"Hipathia is a method for the computation of signal\ntransduction along signaling pathways from transcriptomic data.\nThe method is based on an iterative algorithm which is able to\ncompute the signal intensity passing through the nodes of a\nnetwork by taking into account the level of expression of each\ngene and the intensity of the signal arriving to it. It also\nprovides a new approach to functional analysis allowing to\ncompute the signal arriving to the functions annotated to each\npathway.","downloads_month":147,"downloads_total":2828,"page":"http://bioconductor.org/packages/release/bioc/html/hipathia.html"},{"authors":"Nirav V Malani <malnirav@gmail.com>","name":"hiReadsProcessor","license":"GPL-3","tags":["Sequencing","Preprocessing"],"description":"hiReadsProcessor contains set of functions which allow\nusers to process LM-PCR products sequenced using any platform.\nGiven an excel/txt file containing parameters for\ndemultiplexing and sample metadata, the functions automate\ntrimming of adaptors and identification of the genomic product.\nGenomic products are further processed for QC and abundance\nquantification.","downloads_month":147,"downloads_total":14278,"page":"http://bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html"},{"authors":"Xiangyu Luo <xyluo1991@gmail.com>, Can Yang <macyang@ust.hk> and Yingying Wei <yweicuhk@gmail.com>","name":"HIREewas","license":"GPL (>= 2)","tags":["DNAMethylation","DifferentialMethylation","FeatureExtraction"],"description":"In epigenome-wide association studies, the measured\nsignals for each sample are a mixture of methylation profiles\nfrom different cell types. The current approaches to the\nassociation detection only claim whether a\ncytosine-phosphate-guanine (CpG) site is associated with the\nphenotype or not, but they cannot determine the cell type in\nwhich the risk-CpG site is affected by the phenotype. We\npropose a solid statistical method, HIgh REsolution (HIRE),\nwhich not only substantially improves the power of association\ndetection at the aggregated level as compared to the existing\nmethods but also enables the detection of risk-CpG sites for\nindividual cell types. The \"HIREewas\" R package is to implement\nHIRE model in R.","downloads_month":88,"downloads_total":698,"page":"http://bioconductor.org/packages/release/bioc/html/HIREewas.html"},{"authors":"Nicolas Servant","name":"HiTC","license":"Artistic-2.0","tags":["Sequencing","HighThroughputSequencing","HiC"],"description":"The HiTC package was developed to explore high-throughput\n'C' data such as 5C or Hi-C. Dedicated R classes as well as\nstandard methods for quality controls, normalization,\nvisualization, and further analysis are also provided.","downloads_month":342,"downloads_total":49558,"page":"http://bioconductor.org/packages/release/bioc/html/HiTC.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu>","name":"hmdbQuery","license":"Artistic-2.0","tags":["Metabolomics","Infrastructure"],"description":"Define utilities for exploration of human metabolome\ndatabase, including functions to retrieve specific metabolite\nentries and data snapshots with pairwise associations\n(metabolite-gene,-protein,-disease).","downloads_month":137,"downloads_total":2430,"page":"http://bioconductor.org/packages/release/bioc/html/hmdbQuery.html"},{"authors":"Daniel Lai, Gavin Ha and Sohrab Shah","name":"HMMcopy","license":"GPL-3","tags":["Sequencing","Preprocessing","Visualization","CopyNumberVariation","Microarray"],"description":"Corrects GC and mappability biases for readcounts (i.e.\ncoverage) in non-overlapping windows of fixed length for single\nwhole genome samples, yielding a rough estimate of copy number\nfor furthur analysis.  Designed for rapid correction of high\ncoverage whole genome tumour and normal samples.","downloads_month":392,"downloads_total":42224,"page":"http://bioconductor.org/packages/release/bioc/html/HMMcopy.html"},{"authors":"Katherine S. Pollard, with Mark J. van der Laan\n<laan@stat.berkeley.edu> and Greg Wall","name":"hopach","license":"GPL (>= 2)","tags":["Clustering"],"description":"The HOPACH clustering algorithm builds a hierarchical tree\nof clusters by recursively partitioning a data set, while\nordering and possibly collapsing clusters at each level. The\nalgorithm uses the Mean/Median Split Silhouette (MSS) criteria\nto identify the level of the tree with maximally homogeneous\nclusters. It also runs the tree down to produce a final ordered\nlist of the elements. The non-parametric bootstrap allows one\nto estimate the probability that each element belongs to each\ncluster (fuzzy clustering).","downloads_month":351,"downloads_total":84714,"page":"http://bioconductor.org/packages/release/bioc/html/hopach.html"},{"authors":"Anh Nhat Tran [aut and cre]","name":"HPAanalyze","license":"GPL-3 + file LICENSE","tags":["Proteomics","CellBiology","Visualization","Software"],"description":"Provide functions for retrieving, exploratory analyzing\nand visualizing the Human Protein Atlas data.","downloads_month":107,"downloads_total":828,"page":"http://bioconductor.org/packages/release/bioc/html/HPAanalyze.html"},{"authors":"Laurent Gatto","name":"hpar","license":"Artistic-2.0","tags":["Proteomics","Homo_sapiens","CellBiology"],"description":"The hpar package provides a simple R interface to and data\nfrom the Human Protein Atlas project.","downloads_month":363,"downloads_total":44966,"page":"http://bioconductor.org/packages/release/bioc/html/hpar.html"},{"authors":"Heidi Dvinge and Paul Bertone","name":"HTqPCR","license":"Artistic-2.0","tags":["MicrotitrePlateAssay","DifferentialExpression","GeneExpression","DataImport","QualityControl","Preprocessing","Visualization","MultipleComparison","qPCR"],"description":"Analysis of Ct values from high throughput quantitative\nreal-time PCR (qPCR) assays across multiple conditions or\nreplicates. The input data can be from spatially-defined\nformats such ABI TaqMan Low Density Arrays or OpenArray;\nLightCycler from Roche Applied Science; the CFX plates from\nBio-Rad Laboratories; conventional 96- or 384-well plates; or\nmicrofluidic devices such as the Dynamic Arrays from Fluidigm\nCorporation. HTqPCR handles data loading, quality assessment,\nnormalization, visualization and parametric or non-parametric\ntesting for statistical significance in Ct values between\nfeatures (e.g. genes, microRNAs).","downloads_month":382,"downloads_total":72648,"page":"http://bioconductor.org/packages/release/bioc/html/HTqPCR.html"},{"authors":"Xin Wang <xinwang2hms@gmail.com>, Camille Terfve\n<cdat2@cam.ac.uk>, John C. Rose <jcr53@cam.ac.uk> and Florian\nMarkowetz <Florian.Markowetz@cruk.cam.ac.uk>","name":"HTSanalyzeR","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","MultipleComparison"],"description":"This package provides classes and methods for gene set\nover-representation, enrichment and network analyses on\nhigh-throughput screens. The over-representation analysis is\nperformed based on hypergeometric tests. The enrichment\nanalysis is based on the GSEA algorithm (Subramanian et al.\nPNAS 2005). The network analysis identifies enriched\nsubnetworks based on algorithms from the BioNet package\n(Beisser et al., Bioinformatics 2010). A pipeline is also\nspecifically designed for cellHTS2 object to perform\nintegrative network analyses of high-throughput RNA\ninterference screens. The users can build their own analysis\npipeline for their own data set based on this package.","downloads_month":330,"downloads_total":53354,"page":"http://bioconductor.org/packages/release/bioc/html/HTSanalyzeR.html"},{"authors":"Evarist Planet, Camille Stephan-Otto, Oscar Reina, Oscar\nFlores and David Rossell","name":"htSeqTools","license":"GPL (>=2)","tags":["Sequencing","QualityControl"],"description":"We provide efficient, easy-to-use tools for\nHigh-Throughput Sequencing (ChIP-seq, RNAseq etc.). These\ninclude MDS plots (analogues to PCA), detecting inefficient\nimmuno-precipitation or over-amplification artifacts, tools to\nidentify and test for genomic regions with large accumulation\nof reads, and visualization of coverage profiles.","downloads_month":256,"downloads_total":47630,"page":"http://bioconductor.org/packages/release/bioc/html/htSeqTools.html"},{"authors":"Andrea Rau, Melina Gallopin, Gilles Celeux, and Florence\nJaffrezic","name":"HTSFilter","license":"Artistic-2.0","tags":["Sequencing","RNASeq","Preprocessing","DifferentialExpression","GeneExpression","Normalization","ImmunoOncology"],"description":"This package implements a filtering procedure for\nreplicated transcriptome sequencing data based on a global\nJaccard similarity index in order to identify genes with low,\nconstant levels of expression across one or more experimental\nconditions.","downloads_month":416,"downloads_total":42002,"page":"http://bioconductor.org/packages/release/bioc/html/HTSFilter.html"},{"authors":"Stan Pounds <stanley.pounds@stjude.org> and Demba Fofana\n<demba.fofana@stjude.org>","name":"HybridMTest","license":"GPL Version 2 or later","tags":["GeneExpression","Genetics","Microarray"],"description":"Performs hybrid multiple testing that incorporates method\nselection and assumption evaluations into the analysis using\nempirical Bayes probability (EBP) estimates obtained by\nGrenander density estimation. For instance, for 3-group\ncomparison analysis, Hybrid Multiple testing considers EBPs as\nweighted EBPs between F-test and H-test with EBPs from Shapiro\nWilk test of normality as weigth. Instead of just using EBPs\nfrom F-test only or using H-test only, this methodology\ncombines both types of EBPs through EBPs from Shapiro Wilk test\nof normality. This methodology uses then the law of total EBPs.","downloads_month":171,"downloads_total":29994,"page":"http://bioconductor.org/packages/release/bioc/html/HybridMTest.html"},{"authors":"Paul Murrell","name":"hyperdraw","license":"GPL (>= 2)","tags":["Visualization","GraphAndNetwork"],"description":"Functions for visualizing hypergraphs.","downloads_month":197,"downloads_total":44694,"page":"http://bioconductor.org/packages/release/bioc/html/hyperdraw.html"},{"authors":"Seth Falcon and Robert Gentleman","name":"hypergraph","license":"Artistic-2.0","tags":["GraphAndNetwork"],"description":"A package that implements some simple capabilities for\nrepresenting and manipulating hypergraphs.","downloads_month":276,"downloads_total":74646,"page":"http://bioconductor.org/packages/release/bioc/html/hypergraph.html"},{"authors":"Yingying Wei and Hongkai Ji","name":"iASeq","license":"GPL-2","tags":["ImmunoOncology","SNP","RNASeq","ChIPSeq"],"description":"It fits correlation motif model to multiple RNAseq or\nChIPseq studies to improve detection of allele-specific events\nand describe correlation patterns across studies.","downloads_month":132,"downloads_total":30466,"page":"http://bioconductor.org/packages/release/bioc/html/iASeq.html"},{"authors":"Donghyung Lee [aut, cre], Anthony Cheng [aut], Nathan Lawlor\n[aut] and Duygu Ucar [aut]","name":"iasva","license":"GPL-2","tags":["Preprocessing","QualityControl","BatchEffect","RNASeq","Software","StatisticalMethod","FeatureExtraction","ImmunoOncology"],"description":"Iteratively Adjusted Surrogate Variable Analysis (IA-SVA)\nis a statistical framework to uncover hidden sources of\nvariation even when these sources are correlated. IA-SVA\nprovides a flexible methodology to i) identify a hidden factor\nfor unwanted heterogeneity while adjusting for all known\nfactors; ii) test the significance of the putative hidden\nfactor for explaining the unmodeled variation in the data; and\niii), if significant, use the estimated factor as an additional\nknown factor in the next iteration to uncover further hidden\nfactors.","downloads_month":131,"downloads_total":1140,"page":"http://bioconductor.org/packages/release/bioc/html/iasva.html"},{"authors":"Daniel Gusenleitner and Aedin Culhane","name":"iBBiG","license":"Artistic-2.0","tags":["Clustering","Annotation","GeneSetEnrichment"],"description":"iBBiG is a bi-clustering algorithm which is optimizes for\nbinary data analysis. We apply it to meta-gene set analysis of\nlarge numbers of gene expression datasets. The iterative\nalgorithm extracts groups of phenotypes from multiple studies\nthat are associated with similar gene sets. iBBiG does not\nrequire prior knowledge of the number or scale of clusters and\nallows discovery of clusters with diverse sizes","downloads_month":236,"downloads_total":35978,"page":"http://bioconductor.org/packages/release/bioc/html/iBBiG.html"},{"authors":"Kircicegi Korkmaz, Volkan Atalay and Rengul Cetin Atalay.","name":"ibh","license":"GPL (>= 2)","tags":["QualityControl","DataImport","GraphAndNetwork","NetworkEnrichment"],"description":"This package contains methods for calculating Interaction\nBased Homogeneity to evaluate fitness of gene lists to an\ninteraction network which is useful for evaluation of\nclustering results and gene list analysis. BioGRID interactions\nare used in the calculation. The user can also provide their\nown interactions.","downloads_month":124,"downloads_total":31554,"page":"http://bioconductor.org/packages/release/bioc/html/ibh.html"},{"authors":"Marie-Pier Scott-Boyer and Greg Imholte","name":"iBMQ","license":"Artistic-2.0","tags":["Microarray","Preprocessing","GeneExpression","SNP"],"description":"integrated Bayesian Modeling of eQTL data","downloads_month":96,"downloads_total":24132,"page":"http://bioconductor.org/packages/release/bioc/html/iBMQ.html"},{"authors":"Paige Maas, Parichoy Pal Choudhury and Nilanjan Chatterjee and\nWilliam Wheeler","name":"iCARE","license":"GPL-3 + file LICENSE","tags":["Software","StatisticalMethod","GenomeWideAssociation"],"description":"An R package to compute Individualized Coherent Absolute\nRisk Estimators.","downloads_month":189,"downloads_total":9512,"page":"http://bioconductor.org/packages/release/bioc/html/iCARE.html"},{"authors":"R. Gentleman and Alain Vandal","name":"Icens","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Many functions for computing the NPMLE for censored and\ntruncated data.","downloads_month":462,"downloads_total":172006,"page":"http://bioconductor.org/packages/release/bioc/html/Icens.html"},{"authors":"Vivek Bhardwaj [aut and cre]","name":"icetea","license":"GPL-3 + file LICENSE","tags":["ImmunoOncology","Transcription","GeneExpression","Sequencing","RNASeq","Transcriptomics","DifferentialExpression"],"description":"icetea (Integrating Cap Enrichment with Transcript\nExpression Analysis) provides functions for end-to-end analysis\nof multiple 5'-profiling methods such as CAGE, RAMPAGE and\nMAPCap, beginning from raw reads to detection of transcription\nstart sites using replicates. It also allows performing\ndifferential TSS detection between group of samples, therefore,\nintegrating the mRNA cap enrichment information with transcript\nexpression analysis.","downloads_month":126,"downloads_total":1158,"page":"http://bioconductor.org/packages/release/bioc/html/icetea.html"},{"authors":"Weiliang Qiu [aut, cre], Brandon Guo [aut, ctb], Christopher\nAnderson [aut, ctb], Barbara Klanderman [aut, ctb], Vincent\nCarey [aut, ctb], Benjamin Raby [aut and ctb]","name":"iCheck","license":"GPL (>= 2)","tags":["GeneExpression","DifferentialExpression","Microarray","Preprocessing","DNAMethylation","OneChannel","TwoChannel","QualityControl"],"description":"QC pipeline and data analysis tools for high-dimensional\nIllumina mRNA expression data.","downloads_month":152,"downloads_total":12416,"page":"http://bioconductor.org/packages/release/bioc/html/iCheck.html"},{"authors":"Qianxing Mo","name":"iChip","license":"GPL (>= 2)","tags":["ChIPchip","OneChannel","AgilentChip","Microarray"],"description":"Hidden Ising models are implemented to identify enriched\ngenomic regions in ChIP-chip data.  They can be used to analyze\nthe data from multiple platforms (e.g., Affymetrix, Agilent,\nand NimbleGen), and the data with single to multiple\nreplicates.","downloads_month":136,"downloads_total":36284,"page":"http://bioconductor.org/packages/release/bioc/html/iChip.html"},{"authors":"Qianxing Mo and Ronglai Shen","name":"iClusterPlus","license":"GPL (>= 2)","tags":["Microarray","Clustering"],"description":"Integrative clustering of multiple genomic data using a\njoint latent variable model.","downloads_month":276,"downloads_total":44078,"page":"http://bioconductor.org/packages/release/bioc/html/iClusterPlus.html"},{"authors":"Zilu Zhou and Nancy Zhang","name":"iCNV","license":"GPL-2","tags":["ImmunoOncology","ExomeSeq","WholeGenome","SNP","CopyNumberVariation","HiddenMarkovModel"],"description":"Integrative copy number variation (CNV) detection from\nmultiple platform and experimental design.","downloads_month":179,"downloads_total":2404,"page":"http://bioconductor.org/packages/release/bioc/html/iCNV.html"},{"authors":"Charlotte Soneson [aut and cre]","name":"iCOBRA","license":"GPL (>=2)","tags":["Classification"],"description":"This package provides functions for calculation and\nvisualization of performance metrics for evaluation of ranking\nand binary classification (assignment) methods. It also\ncontains a shiny application for interactive exploration of\nresults.","downloads_month":185,"downloads_total":9500,"page":"http://bioconductor.org/packages/release/bioc/html/iCOBRA.html"},{"authors":"Federico Marini [aut and cre]\n(<https://orcid.org/0000-0003-3252-7758>)","name":"ideal","license":"MIT + file LICENSE","tags":["ImmunoOncology","GeneExpression","DifferentialExpression","RNASeq","Sequencing","Visualization","QualityControl","GUI","GeneSetEnrichment","ReportWriting"],"description":"This package provides functions for an Interactive\nDifferential Expression AnaLysis of RNA-sequencing datasets, to\nextract quickly and effectively information downstream the step\nof differential expression. A Shiny application encapsulates\nthe whole package.","downloads_month":207,"downloads_total":7644,"page":"http://bioconductor.org/packages/release/bioc/html/ideal.html"},{"authors":"Shraddha Pai <shraddha.pai@utoronto.ca> and Jingliang Ren","name":"IdeoViz","license":"GPL-2","tags":["Visualization","Microarray"],"description":"Plots data associated with arbitrary genomic intervals\nalong chromosomal ideogram.","downloads_month":206,"downloads_total":19946,"page":"http://bioconductor.org/packages/release/bioc/html/IdeoViz.html"},{"authors":"Karl J. Dykema <karl.dykema@vai.org>","name":"idiogram","license":"GPL-2","tags":["Visualization"],"description":"A package for plotting genomic data by chromosomal\nlocation","downloads_month":152,"downloads_total":48102,"page":"http://bioconductor.org/packages/release/bioc/html/idiogram.html"},{"authors":"Alex Lisovich and Roger Day","name":"IdMappingAnalysis","license":"GPL-2","tags":["Annotation","MultipleComparison"],"description":"Identifier mapping performance analysis","downloads_month":122,"downloads_total":29070,"page":"http://bioconductor.org/packages/release/bioc/html/IdMappingAnalysis.html"},{"authors":"Alex Lisovich and Roger Day","name":"IdMappingRetrieval","license":"GPL-2","tags":["Annotation","MultipleComparison"],"description":"Data retrieval for identifier mapping performance analysis","downloads_month":119,"downloads_total":30404,"page":"http://bioconductor.org/packages/release/bioc/html/IdMappingRetrieval.html"},{"authors":"Yi-Pin Lai [aut], Liang-Bo Wang [aut, cre], Tzu-Pin Lu [aut] and Eric Y. Chuang [aut]","name":"iGC","license":"GPL-2","tags":["Software","Biological Question","DifferentialExpression","GenomicVariation","AssayDomain","CopyNumberVariation","GeneExpression","ResearchField","Genetics","Technology","Microarray","Sequencing","WorkflowStep","MultipleComparison"],"description":"This package is intended to identify differentially\nexpressed genes driven by Copy Number Alterations from samples\nwith both gene expression and CNA data.","downloads_month":140,"downloads_total":10376,"page":"http://bioconductor.org/packages/release/bioc/html/iGC.html"},{"authors":"Paul Shannon","name":"igvR","license":"MIT + file LICENSE","tags":["Visualization","ThirdPartyClient","GenomeBrowsers"],"description":"Access to igv.js, the Integrative Genomics Viewer running\nin a web browser.","downloads_month":142,"downloads_total":2270,"page":"http://bioconductor.org/packages/release/bioc/html/igvR.html"},{"authors":"Nikos Ignatiadis [aut, cre] and Wolfgang Huber [aut]","name":"IHW","license":"Artistic-2.0","tags":["ImmunoOncology","MultipleComparison","RNASeq"],"description":"Independent hypothesis weighting (IHW) is a multiple\ntesting procedure that increases power compared to the method\nof Benjamini and Hochberg by assigning data-driven weights to\neach hypothesis. The input to IHW is a two-column table of\np-values and covariates. The covariate can be any\ncontinuous-valued or categorical variable that is thought to be\ninformative on the statistical properties of each hypothesis\ntest, while it is independent of the p-value under the null\nhypothesis.","downloads_month":1201,"downloads_total":42148,"page":"http://bioconductor.org/packages/release/bioc/html/IHW.html"},{"authors":"Keith Baggerly [aut], Henrik Bengtsson [aut], Kasper Daniel\nHansen [aut, cre], Matt Ritchie [aut], Mike L. Smith [aut] and Tim\nTriche Jr. [ctb]","name":"illuminaio","license":"GPL-2","tags":["Infrastructure","DataImport","Microarray","ProprietaryPlatforms"],"description":"Tools for parsing Illumina's microarray output files,\nincluding IDAT.","downloads_month":3749,"downloads_total":348292,"page":"http://bioconductor.org/packages/release/bioc/html/illuminaio.html"},{"authors":"Gregoire Pau, Xian Zhang, Michael Boutros and Wolfgang Huber","name":"imageHTS","license":"LGPL-2.1","tags":["ImmunoOncology","Software","CellBasedAssays","Preprocessing","Visualization"],"description":"imageHTS is an R package dedicated to the analysis of\nhigh-throughput microscopy-based screens. The package provides\na modular and extensible framework to segment cells, extract\nquantitative cell features, predict cell types and browse\nscreen data through web interfaces. Designed to operate in\ndistributed environments, imageHTS provides a standardized\naccess to remote data and facilitates the dissemination of\nhigh-throughput microscopy-based datasets.","downloads_month":219,"downloads_total":39040,"page":"http://bioconductor.org/packages/release/bioc/html/imageHTS.html"},{"authors":"Seonggyun Han and Younghee Lee","name":"IMAS","license":"GPL-2","tags":["ImmunoOncology","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","GeneExpression","GeneRegulation","Regression","RNASeq","Sequencing","SNP","Software","Transcription"],"description":"Integrative analysis of Multi-omics data for Alternative\nsplicing.","downloads_month":196,"downloads_total":5502,"page":"http://bioconductor.org/packages/release/bioc/html/IMAS.html"},{"authors":"Audrey Lemacon <audrey.lemacon.1@ulaval.ca>, Charles Joly\nBeauparlant <charles.joly-beauparlant@crchul.ulaval.ca> and Arnaud\nDroit <arnaud.droit@crchuq.ulaval.ca>","name":"Imetagene","license":"Artistic-2.0 | file LICENSE","tags":["ChIPSeq","Genetics","MultipleComparison","Coverage","Alignment","Sequencing"],"description":"This package provide a graphical user interface to the\nmetagene package. This will allow people with minimal R\nexperience to easily complete metagene analysis.","downloads_month":127,"downloads_total":10302,"page":"http://bioconductor.org/packages/release/bioc/html/Imetagene.html"},{"authors":"Minoo Ashtiani, Payman Nickchi, Abdollah Safari, Mehdi Mirzaie and Mohieddin Jafari","name":"IMMAN","license":"Artistic-2.0","tags":["SequenceMatching","Alignment","SystemsBiology","GraphAndNetwork","Network","Proteomics"],"description":"Reconstructing Interlog Protein Network (IPN) integrated\nfrom several Protein protein Interaction Networks (PPINs).\nUsing this package, overlaying different PPINs to mine\nconserved common networks between diverse species will be\napplicable.","downloads_month":143,"downloads_total":2036,"page":"http://bioconductor.org/packages/release/bioc/html/IMMAN.html"},{"authors":"Greg Finak [aut], Renan Sauteraud [aut], Mike Jiang [aut], Gil\nGuday [aut], Leo Dashevskiy [aut], Evan Henrich [aut], Ju Yeong\nKim [aut], Lauren Wolfe [aut], Raphael Gottardo [aut], ImmuneSpace Package Maintainer [cre and cph]","name":"ImmuneSpaceR","license":"GPL-2","tags":["DataImport","DataRepresentation","ThirdPartyClient"],"description":"Provides a convenient API for accessing data sets within\nImmuneSpace (www.immunespace.org), the data repository and\nanalysis platform of the Human Immunology Project Consortium\n(HIPC).","downloads_month":214,"downloads_total":9648,"page":"http://bioconductor.org/packages/release/bioc/html/ImmuneSpaceR.html"},{"authors":"Till Soerensen <till-antoni.soerensen@charite.de>","name":"immunoClust","license":"Artistic-2.0","tags":["Clustering","FlowCytometry","CellBasedAssays","ImmunoOncology"],"description":"Model based clustering and meta-clustering of Flow\nCytometry Data","downloads_month":162,"downloads_total":13648,"page":"http://bioconductor.org/packages/release/bioc/html/immunoClust.html"},{"authors":"Natalja Kurbatova and Jeremy Mason","name":"IMPCdata","license":"file LICENSE","tags":["ExperimentData"],"description":"Package contains methods for data retrieval from IMPC\nDatabase.","downloads_month":125,"downloads_total":13866,"page":"http://bioconductor.org/packages/release/bioc/html/IMPCdata.html"},{"authors":"Jil Sander [aut, cre] and Nir Yosef [aut]","name":"ImpulseDE","license":"GPL-3","tags":["Software","StatisticalMethod","TimeCourse"],"description":"ImpulseDE is suited to capture single impulse-like\npatterns in high throughput time series datasets. By fitting a\nrepresentative impulse model to each gene, it reports\ndifferentially expressed genes whether across time points in a\nsingle experiment or between two time courses from two\nexperiments. To optimize the running time, the code makes use\nof clustering steps and multi-threading.","downloads_month":136,"downloads_total":6924,"page":"http://bioconductor.org/packages/release/bioc/html/ImpulseDE.html"},{"authors":"David S Fischer [aut, cre], Fabian J Theis [ctb] and Nir Yosef\n[ctb]","name":"ImpulseDE2","license":"Artistic-2.0","tags":["ImmunoOncology","Software","StatisticalMethod","TimeCourse","Sequencing","DifferentialExpression","GeneExpression","CellBiology","CellBasedAssays"],"description":"ImpulseDE2 is a differential expression algorithm for\nlongitudinal count data sets which arise in sequencing\nexperiments such as RNA-seq, ChIP-seq, ATAC-seq and DNaseI-seq.\nImpulseDE2 is based on a negative binomial noise model with\ndispersion trend smoothing by DESeq2 and uses the impulse model\nto constrain the mean expression trajectory of each gene. The\nimpulse model was empirically found to fit global expression\nchanges in cells after environmental and developmental stimuli\nand is therefore appropriate in most cell biological scenarios.\nThe constraint on the mean expression trajectory prevents\noverfitting to small expression fluctuations. Secondly,\nImpulseDE2 has higher statistical testing power than\ngeneralized linear model-based differential expression\nalgorithms which fit time as a categorial variable if more than\nsix time points are sampled because of the fixed number of\nparameters.","downloads_month":304,"downloads_total":6930,"page":"http://bioconductor.org/packages/release/bioc/html/ImpulseDE2.html"},{"authors":"Trevor Hastie, Robert Tibshirani, Balasubramanian Narasimhan and Gilbert Chu","name":"impute","license":"GPL-2","tags":["Microarray"],"description":"Imputation for microarray data (currently KNN only)","downloads_month":9841,"downloads_total":1394072,"page":"http://bioconductor.org/packages/release/bioc/html/impute.html"},{"authors":"Zhenzhi Li <zzrickli@gmail.com>","name":"INDEED","license":"Artistic-2.0","tags":["ImmunoOncology","Software","ResearchField","BiologicalQuestion","StatisticalMethod","DifferentialExpression","MassSpectrometry","Metabolomics"],"description":"An Implementation of Integrated Differential Expression\nand Differential Network Analysis of Omic Data. The\ndifferential network is obtained based on partial correlation\nor correlation.","downloads_month":118,"downloads_total":744,"page":"http://bioconductor.org/packages/release/bioc/html/INDEED.html"},{"authors":"Jianhong Ou, Sung Mi Park, Michael R. Green and Lihua Julie Zhu","name":"InPAS","license":"GPL (>= 2)","tags":["RNASeq","Sequencing","AlternativeSplicing","Coverage","DifferentialSplicing","GeneRegulation","Transcription","ImmunoOncology"],"description":"Alternative polyadenylation (APA) is one of the important\npost-transcriptional regulation mechanisms which occurs in most\nhuman genes. InPAS facilitates the discovery of novel APA sites\nfrom RNAseq data. It leverages cleanUpdTSeq to fine tune\nidentified APA sites.","downloads_month":145,"downloads_total":13738,"page":"http://bioconductor.org/packages/release/bioc/html/InPAS.html"},{"authors":"Ju-Hyun Park","name":"INPower","license":"GPL-2 + file LICENSE","tags":["SNP"],"description":"An R package for computing the number of susceptibility\nSNPs and power of future studies","downloads_month":135,"downloads_total":17836,"page":"http://bioconductor.org/packages/release/bioc/html/INPower.html"},{"authors":"Stefano de Pretis","name":"INSPEcT","license":"GPL-2","tags":["Sequencing","RNASeq","GeneRegulation","TimeCourse","SystemsBiology"],"description":"INSPEcT (INference of Synthesis, Processing and\ndEgradation rates from Transcriptomic data) RNA-seq data in\ntime-course experiments or different conditions, with or\nwithout the support of nascent RNA data.","downloads_month":225,"downloads_total":11730,"page":"http://bioconductor.org/packages/release/bioc/html/INSPEcT.html"},{"authors":"Konstantin Okonechnikov, Serap Erkek and Lukas Chavez","name":"InTAD","license":"GPL (>=2)","tags":["Epigenetics","Sequencing","ChIPSeq","RNASeq","HiC","GeneExpression"],"description":"The package is focused on the detection of correlation\nbetween expressed genes and selected epigenomic signals i.e.\nenhancers obtained from ChIP-seq data within topologically\nassociated domains (TADs). Various parameters can be controlled\nto investigate the influence of external factors and\nvisualization plots are available for each analysis step.","downloads_month":132,"downloads_total":2276,"page":"http://bioconductor.org/packages/release/bioc/html/InTAD.html"},{"authors":"Wen Yao <ywhzau@gmail.com>","name":"intansv","license":"Artistic-2.0","tags":["Genetics","Annotation","Sequencing","Software"],"description":"This package provides efficient tools to read and\nintegrate structural variations predicted by popular softwares.\nAnnotation and visulation of structural variations are also\nimplemented in the package.","downloads_month":146,"downloads_total":26176,"page":"http://bioconductor.org/packages/release/bioc/html/intansv.html"},{"authors":"Aaron Lun [aut, cre], Malcolm Perry [aut] and Elizabeth\nIng-Simmons [aut]","name":"InteractionSet","license":"GPL-3","tags":["Infrastructure","DataRepresentation","Software","HiC"],"description":"Provides the GInteractions, InteractionSet and\nContactMatrix objects and associated methods for storing and\nmanipulating genomic interaction data from Hi-C and ChIA-PET\nexperiments.","downloads_month":474,"downloads_total":23474,"page":"http://bioconductor.org/packages/release/bioc/html/InteractionSet.html"},{"authors":"Shawn Balcome and Marc Carlson","name":"interactiveDisplay","license":"Artistic-2.0","tags":["GO","GeneExpression","Microarray","Sequencing","Classification","Network","QualityControl","Visualization","Visualization","Genetics","DataRepresentation","GUI","AnnotationData"],"description":"The interactiveDisplay package contains the methods needed\nto generate interactive Shiny based display methods for\nBioconductor objects.","downloads_month":264,"downloads_total":58576,"page":"http://bioconductor.org/packages/release/bioc/html/interactiveDisplay.html"},{"authors":"Shawn Balcome and Marc Carlson","name":"interactiveDisplayBase","license":"Artistic-2.0","tags":["GO","GeneExpression","Microarray","Sequencing","Classification","Network","QualityControl","Visualization","Visualization","Genetics","DataRepresentation","GUI","AnnotationData"],"description":"The interactiveDisplayBase package contains the the basic\nmethods needed to generate interactive Shiny based display\nmethods for Bioconductor objects.","downloads_month":3657,"downloads_total":355588,"page":"http://bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html"},{"authors":"Ali Oghabian <Ali.Oghabian@Helsinki.Fi>, Dario Greco\n<dario.greco@helsinki.fi> and Mikko Frilander\n<Mikko.Frilander@helsinki.fi>","name":"IntEREst","license":"GPL-2","tags":["Software","AlternativeSplicing","Coverage","DifferentialSplicing","Sequencing","RNASeq","Alignment","Normalization","DifferentialExpression","ImmunoOncology"],"description":"This package performs Intron-Exon Retention analysis on\nRNA-seq data (.bam files).","downloads_month":175,"downloads_total":5804,"page":"http://bioconductor.org/packages/release/bioc/html/IntEREst.html"},{"authors":"Bing Wang, Julie Sullivan, Rachel Lyne and Konstantinos Kyritsis","name":"InterMineR","license":"LGPL","tags":["GeneExpression","SNP","GeneSetEnrichment","DifferentialExpression","GeneRegulation","GenomeAnnotation","GenomeWideAssociation","FunctionalPrediction","AlternativeSplicing","ComparativeGenomics","FunctionalGenomics","Proteomics","SystemsBiology","Microarray","MultipleComparison","Pathways","GO","KEGG","Reactome","Visualization"],"description":"Databases based on the InterMine platform such as FlyMine,\nmodMine (modENCODE), RatMine, YeastMine, HumanMine and\nTargetMine are integrated databases of genomic, expression and\nprotein data for various organisms. Integrating data makes it\npossible to run sophisticated data mining queries that span\ndomains of biological knowledge. This R package provides\ninterfaces with these databases through webservices. It makes\nmost from the correspondence of the data frame object in R and\nthe table object in databases, while hiding the details of data\nexchange through XML or JSON.","downloads_month":205,"downloads_total":4528,"page":"http://bioconductor.org/packages/release/bioc/html/InterMineR.html"},{"authors":"Surajit Bhattacharya and Daniel Cox","name":"IntramiRExploreR","license":"GPL-2","tags":["Software","Microarray","GeneTarget","StatisticalMethod","GeneExpression","GenePrediction"],"description":"Intra-miR-ExploreR, an integrative miRNA target prediction\nbioinformatics tool, identifies targets combining expression\nand biophysical interactions of a given microRNA (miR). Using\nthe tool, we have identified targets for 92 intragenic miRs in\nD. melanogaster, using available microarray expression data,\nfrom Affymetrix 1 and Affymetrix2 microarray array platforms,\nproviding a global perspective of intragenic miR targets in\nDrosophila. Predicted targets are grouped according to\nbiological functions using the DAVID Gene Ontology tool and are\nranked based on a biologically relevant scoring system,\nenabling the user to identify functionally relevant targets for\na given miR.","downloads_month":111,"downloads_total":3056,"page":"http://bioconductor.org/packages/release/bioc/html/IntramiRExploreR.html"},{"authors":"Alejandro Caceres","name":"inveRsion","license":"GPL (>= 2)","tags":["Microarray","SNP"],"description":"Package to find genetic inversions in genotype (SNP array)\ndata.","downloads_month":137,"downloads_total":32446,"page":"http://bioconductor.org/packages/release/bioc/html/inveRsion.html"},{"authors":"Mike Smith [aut and cre]","name":"IONiseR","license":"MIT + file LICENSE","tags":["QualityControl","DataImport","Sequencing"],"description":"IONiseR provides tools for the quality assessment of\nOxford Nanopore MinION data. It extracts summary statistics\nfrom a set of fast5 files and can be used either before or\nafter base calling.  In addition to standard summaries of the\nread-types produced, it provides a number of plots for\nvisualising metrics relative to experiment run time or\nspatially over the surface of a flowcell.","downloads_month":172,"downloads_total":15662,"page":"http://bioconductor.org/packages/release/bioc/html/IONiseR.html"},{"authors":"Gregory Ryslik and Hongyu Zhao","name":"iPAC","license":"GPL-2","tags":["Clustering","Proteomics"],"description":"iPAC is a novel tool to identify somatic amino acid\nmutation clustering within proteins while taking into account\nprotein structure.","downloads_month":165,"downloads_total":30804,"page":"http://bioconductor.org/packages/release/bioc/html/iPAC.html"},{"authors":"Steffen Klasberg","name":"ipdDb","license":"Artistic-2.0","tags":["GenomicVariation","SequenceMatching","VariantAnnotation","DataRepresentation"],"description":"All alleles from the IPD IMGT/HLA\n<https://www.ebi.ac.uk/ipd/imgt/hla/> and IPD KIR\n<https://www.ebi.ac.uk/ipd/kir/> database for Homo sapiens.\nReference: Robinson J, Maccari G, Marsh SGE, Walter L, Blokhuis\nJ, Bimber B, Parham P, De Groot NG, Bontrop RE, Guethlein LA,\nand Hammond JA KIR Nomenclature in non-human species\nImmunogenetics (2018), in preparation.","downloads_month":83,"downloads_total":770,"page":"http://bioconductor.org/packages/release/bioc/html/ipdDb.html"},{"authors":"Gunnar Libiseller <Gunnar.Libiseller@joanneum.at>, Christoph\nMagnes <christoph.magnes@joanneum.at> and Thomas Riebenbauer\n<Thomas.Riebenbauer@joanneum.at>","name":"IPO","license":"GPL (>= 2) + file LICENSE","tags":["ImmunoOncology","Metabolomics","MassSpectrometry"],"description":"The outcome of XCMS data processing strongly depends on\nthe parameter settings. IPO (`Isotopologue Parameter\nOptimization`) is a parameter optimization tool that is\napplicable for different kinds of samples and liquid\nchromatography coupled to high resolution mass spectrometry\ndevices, fast and free of labeling steps. IPO uses natural,\nstable 13C isotopes to calculate a peak picking score.\nRetention time correction is optimized by minimizing the\nrelative retention time differences within features and\ngrouping parameters are optimized by maximizing the number of\nfeatures showing exactly one peak from each injection of a\npooled sample. The different parameter settings are achieved by\ndesign of experiment. The resulting scores are evaluated using\nresponse surface models.","downloads_month":310,"downloads_total":12642,"page":"http://bioconductor.org/packages/release/bioc/html/IPO.html"},{"authors":"Martin Slawski <ms@cs.uni-saarland.de>, Rene Hussong\n<rene.hussong@uni.lu>, Andreas Hildebrandt\n<andreas.hildebrandt@uni-mainz.de> and Matthias Hein\n<hein@cs.uni-saarland.de>","name":"IPPD","license":"GPL (version 2 or later)","tags":["Proteomics"],"description":"The package provides functionality to extract isotopic\npeak patterns from raw mass spectra. This is done by fitting a\nlarge set of template basis functions to the raw spectrum using\neither nonnegative least squares or least absolute deviation\nfittting. The package offers a flexible function which tries to\nestimate model parameters in a way tailored to the peak shapes\nin the data. The package also provides functionality to process\nLCMS runs.","downloads_month":277,"downloads_total":41158,"page":"http://bioconductor.org/packages/release/bioc/html/IPPD.html"},{"authors":"H. Pagès, P. Aboyoun and M. Lawrence","name":"IRanges","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"Provides efficient low-level and highly reusable S4\nclasses for storing, manipulating and aggregating over\nannotated ranges of integers. Implements an algebra of range\noperations, including efficient algorithms for finding overlaps\nand nearest neighbors. Defines efficient list-like classes for\nstoring, transforming and aggregating large grouped data, i.e.,\ncollections of atomic vectors and DataFrames.","downloads_month":46777,"downloads_total":5299662,"page":"http://bioconductor.org/packages/release/bioc/html/IRanges.html"},{"authors":"Kevin Rue-Albrecht [aut], Federico Marini [aut], Charlotte\nSoneson [aut, cre] and Aaron Lun [aut]","name":"iSEE","license":"MIT + file LICENSE","tags":["ImmunoOncology","Visualization","GUI","DimensionReduction","FeatureExtraction","Clustering","Transcription","GeneExpression","Transcriptomics","SingleCell","CellBasedAssays"],"description":"Provides functions for creating an interactive Shiny-based\ngraphical user interface for exploring data stored in\nSummarizedExperiment objects, including row- and column-level\nmetadata. Particular attention is given to single-cell data in\na SingleCellExperiment object with visualization of\ndimensionality reduction results.","downloads_month":269,"downloads_total":4414,"page":"http://bioconductor.org/packages/release/bioc/html/iSEE.html"},{"authors":"Qianxing Mo","name":"iSeq","license":"GPL (>= 2)","tags":["ChIPSeq","Sequencing"],"description":"Bayesian hidden Ising models are implemented to identify\nIP-enriched genomic regions from ChIP-seq data. They can be\nused to analyze ChIP-seq data with and without controls and\nreplicates.","downloads_month":141,"downloads_total":35492,"page":"http://bioconductor.org/packages/release/bioc/html/iSeq.html"},{"authors":"Florian P Breitwieser <florian.bw@gmail.com>, Jacques\nColinge <jacques.colinge@inserm.fr>, with contributions from\nAlexey Stukalov <stukalov@biochem.mpg.de> and Xavier Robin\n<xavier.robin@unige.ch> and Florent Gluck\n<florent.gluck@unige.ch>","name":"isobar","license":"LGPL-2","tags":["ImmunoOncology","Proteomics","MassSpectrometry","Bioinformatics","MultipleComparisons","QualityControl"],"description":"isobar provides methods for preprocessing, normalization,\nand report generation for the analysis of quantitative mass\nspectrometry proteomics data labeled with isobaric tags, such\nas iTRAQ and TMT. Features modules for integrating and\nvalidating PTM-centric datasets (isobar-PTM). More information\non http://www.ms-isobar.org.","downloads_month":428,"downloads_total":48018,"page":"http://bioconductor.org/packages/release/bioc/html/isobar.html"},{"authors":"Christian Kohler [cre, aut] and Paul Heinrich [aut]","name":"IsoCorrectoR","license":"GPL-3","tags":["Software","Metabolomics","MassSpectrometry","Preprocessing","ImmunoOncology"],"description":"IsoCorrectoR performs the correction of mass spectrometry\ndata from stable isotope labeling/tracing metabolomics\nexperiments with regard to natural isotope abundance and tracer\nimpurity. Data from both MS and MS/MS measurements can be\ncorrected (with any tracer isotope: 13C, 15N, 18O...), as well\nas high resolution MS data from multiple-tracer experiments\n(e.g. 13C and 15N used simultaneously). See the Bioconductor\npackage IsoCorrectoRGUI for a graphical user interface to\nIsoCorrectoR. IsoCorrectoRGUI is currently only available in\nthe devel version of Bioconductor.","downloads_month":96,"downloads_total":680,"page":"http://bioconductor.org/packages/release/bioc/html/IsoCorrectoR.html"},{"authors":"Christian Kohler [cre, aut], Paul Kuerner [aut] and Paul Heinrich\n[aut]","name":"IsoCorrectoRGUI","license":"GPL-3","tags":["Software","Metabolomics","MassSpectrometry","Preprocessing","GUI","ImmunoOncology"],"description":"IsoCorrectoRGUI is a Graphical User Interface for the\nIsoCorrectoR package. IsoCorrectoR performs the correction of\nmass spectrometry data from stable isotope labeling/tracing\nmetabolomics experiments with regard to natural isotope\nabundance and tracer impurity. Data from both MS and MS/MS\nmeasurements can be corrected (with any tracer isotope: 13C,\n15N, 18O...), as well as high resolution MS data from\nmultiple-tracer experiments (e.g. 13C and 15N used\nsimultaneously).","downloads_month":29,"downloads_total":148,"page":"http://bioconductor.org/packages/release/bioc/html/IsoCorrectoRGUI.html"},{"authors":"Kristoffer Vitting-Seerup","name":"IsoformSwitchAnalyzeR","license":"GPL (>= 2)","tags":["GeneExpression","Transcription","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","Visualization","StatisticalMethod","TranscriptomeVariant","BiomedicalInformatics","FunctionalGenomics","SystemsBiology","Transcriptomics","RNASeq","Annotation","FunctionalPrediction","GenePrediction","DataImport","MultipleComparison","BatchEffect"],"description":"IsoformSwitchAnalyzeR enables identification and analysis\nof alternative splicing and isoform switches with predicted\nfunctional consequences (such as gain/loss of protein domains\netc) from quantification by Kallisto, Salmon,\nCufflinks/Cuffdiff, RSEM etc.","downloads_month":305,"downloads_total":8390,"page":"http://bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html"},{"authors":"Setia Pramana, Dan Lin, Philippe Haldermans, Tobias Verbeke and Martin Otava","name":"IsoGeneGUI","license":"GPL-2","tags":["Microarray","DifferentialExpression","GUI"],"description":"The IsoGene Graphical User Interface (IsoGene-GUI) is a\nuser friendly interface of the IsoGene package which is aimed\nto identify for genes with a monotonic trend in the expression\nlevels with respect to the increasing doses. Additionally, GUI\nextension of original package contains various tools to perform\nclustering of dose-response profiles. Testing is addressed\nthrough several test statistics: global likelihood ratio test\n(E2), Bartholomew 1961, Barlow et al. 1972 and Robertson et al.\n1988), Williams (1971, 1972), Marcus (1976), the M (Hu et al.\n2005) and the modified M (Lin et al. 2007). The p-values of the\nglobal likelihood ratio test (E2) are obtained using the exact\ndistribution and permutations. The other four test statistics\nare obtained using permutations. Several p-values adjustment\nare provided: Bonferroni, Holm (1979), Hochberg (1988), and\nSidak procedures for controlling the family-wise Type I error\nrate (FWER), and BH (Benjamini and Hochberg 1995) and BY\n(Benjamini and Yekutieli 2001) procedures are used for\ncontrolling the FDR. The inference is based on resampling\nmethods, which control the False Discovery Rate (FDR), for both\npermutations (Ge et al., 2003) and the Significance Analysis of\nMicroarrays (SAM, Tusher et al., 2001). Clustering methods are\noutsourced from CRAN packages ORCME, ORIClust. The package\nORCME is based on delta-clustering method (Cheng and Church,\n2000) and ORIClust on Order Restricted Information Criterion\n(Liu et al., 2009), both perform same task but from different\nperspective and their outputs are clusters of genes.\nAdditionally, profile selection for given gene based on\nGeneralized ORIC (Kuiper et al., 2014) from package goric and\npermutation test for E2 based on package orQA are included in\nIsoGene-GUI. None of these four packages has GUI.","downloads_month":154,"downloads_total":35978,"page":"http://bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html"},{"authors":"Christelle Reynès [aut, cre], Marine Rohmer [aut] and Guilhem\nKister [aut]","name":"ISoLDE","license":"GPL (>= 2.0)","tags":["ImmunoOncology","GeneExpression","Transcription","GeneSetEnrichment","Genetics","Sequencing","RNASeq","MultipleComparison","SNP","GeneticVariability","Epigenetics","MathematicalBiology","GeneRegulation"],"description":"This package provides ISoLDE a new method for identifying\nimprinted genes. This method is dedicated to data arising from\nRNA sequencing technologies. The ISoLDE package implements\noriginal statistical methodology described in the publication\nbelow.","downloads_month":130,"downloads_total":7660,"page":"http://bioconductor.org/packages/release/bioc/html/ISoLDE.html"},{"authors":"Lorena Pantano [aut, cre] and Georgia Escaramis [aut]","name":"isomiRs","license":"MIT + file LICENSE","tags":["miRNA","RNASeq","DifferentialExpression","Clustering","ImmunoOncology"],"description":"Characterization of miRNAs and isomiRs, clustering and\ndifferential expression.","downloads_month":196,"downloads_total":9776,"page":"http://bioconductor.org/packages/release/bioc/html/isomiRs.html"},{"authors":"Guillem Rigaill and Philippe Hupe","name":"ITALICS","license":"GPL-2","tags":["Microarray","CopyNumberVariation"],"description":"A Method to normalize of Affymetrix GeneChip Human Mapping\n100K and 500K set","downloads_month":136,"downloads_total":43534,"page":"http://bioconductor.org/packages/release/bioc/html/ITALICS.html"},{"authors":"Ka Yee Yeung, University of Washington, Seattle, WA, with\ncontributions from Adrian Raftery and Ian Painter","name":"iterativeBMA","license":"GPL (>= 2)","tags":["Microarray","Classification"],"description":"The iterative Bayesian Model Averaging (BMA) algorithm is\na variable selection and classification algorithm with an\napplication of classifying 2-class microarray samples, as\ndescribed in Yeung, Bumgarner and Raftery (Bioinformatics 2005,\n21: 2394-2402).","downloads_month":153,"downloads_total":43844,"page":"http://bioconductor.org/packages/release/bioc/html/iterativeBMA.html"},{"authors":"Amalia Annest, University of Washington, Tacoma, WA Ka Yee\nYeung, University of Washington, Seattle and WA","name":"iterativeBMAsurv","license":"GPL (>= 2)","tags":["Microarray"],"description":"The iterative Bayesian Model Averaging (BMA) algorithm for\nsurvival analysis is a variable selection method for applying\nsurvival analysis to microarray data.","downloads_month":138,"downloads_total":42436,"page":"http://bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html"},{"authors":"Hongxu Ding and Andrea Califano","name":"iterClust","license":"file LICENSE","tags":["StatisticalMethod","Clustering"],"description":"A framework for performing clustering analysis\niteratively.","downloads_month":165,"downloads_total":3440,"page":"http://bioconductor.org/packages/release/bioc/html/iterClust.html"},{"authors":"Jiacheng Chuan [aut and cre]","name":"iteremoval","license":"GPL-2","tags":["StatisticalMethod"],"description":"The package provides a flexible algorithm to screen\nfeatures of two distinct groups in consideration of overfitting\nand overall performance. It was originally tailored for\nmethylation locus screening of NGS data, and it can also be\nused as a generic method for feature selection. Each step of\nthe algorithm provides a default method for simple\nimplemention, and the method can be replaced by a user defined\nfunction.","downloads_month":121,"downloads_total":2108,"page":"http://bioconductor.org/packages/release/bioc/html/iteremoval.html"},{"authors":"Seonggyun Han and Sangsoo Kim","name":"IVAS","license":"GPL-2","tags":["ImmunoOncology","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","GeneExpression","GeneRegulation","Regression","RNASeq","Sequencing","SNP","Software","Transcription"],"description":"Identification of genetic variants affecting alternative\nsplicing.","downloads_month":215,"downloads_total":13290,"page":"http://bioconductor.org/packages/release/bioc/html/IVAS.html"},{"authors":"Vince Carey","name":"ivygapSE","license":"Artistic-2.0","tags":["Transcription","Software","Visualization","Survival","GeneExpression","Sequencing"],"description":"Define a SummarizedExperiment and exploratory app for\nIvy-GAP glioblastoma image, expression, and clinical data.","downloads_month":133,"downloads_total":3096,"page":"http://bioconductor.org/packages/release/bioc/html/ivygapSE.html"},{"authors":"Marzia A Cremona, Alessia Pini, Francesca Chiaromonte and Simone\nVantini","name":"IWTomics","license":"GPL (>=2)","tags":["StatisticalMethod","MultipleComparison","DifferentialExpression","DifferentialMethylation","DifferentialPeakCalling","GenomeAnnotation","DataImport"],"description":"Implementation of the Interval-Wise Testing (IWT) for\nomics data. This inferential procedure tests for differences in\n\"Omics\" data between two groups of genomic regions (or between\na group of genomic regions and a reference center of symmetry),\nand does not require fixing location and scale at the outset.","downloads_month":142,"downloads_total":5100,"page":"http://bioconductor.org/packages/release/bioc/html/IWTomics.html"},{"authors":"Ewa Szczurek","name":"joda","license":"GPL (>= 2)","tags":["Microarray","Pathways","GraphAndNetwork","StatisticalMethod","NetworkInference"],"description":"Package 'joda' implements three steps of an algorithm\ncalled JODA. The algorithm computes gene deregulation scores.\nFor each gene, its deregulation score reflects how strongly an\neffect of a certain regulator's perturbation on this gene\ndiffers between two different cell populations. The algorithm\nutilizes regulator knockdown expression data as well as\nknowledge about signaling pathways in which the regulators are\ninvolved (formalized in a simple matrix model).","downloads_month":132,"downloads_total":32786,"page":"http://bioconductor.org/packages/release/bioc/html/joda.html"},{"authors":"Stephen Hartley [aut, cre] (PhD), Simon Anders [cph] and Alejandro\nReyes [cph]","name":"JunctionSeq","license":"file LICENSE","tags":["ImmunoOncology","Sequencing","RNASeq","DifferentialExpression"],"description":"A Utility for Detection and Visualization of Differential\nExon or Splice-Junction Usage in RNA-Seq data.","downloads_month":368,"downloads_total":14738,"page":"http://bioconductor.org/packages/release/bioc/html/JunctionSeq.html"},{"authors":"Bernat Gel <bgel@igtp.cat>","name":"karyoploteR","license":"Artistic-2.0","tags":["Visualization","CopyNumberVariation","Sequencing","Coverage","DNASeq","ChIPSeq","MethylSeq","DataImport","OneChannel"],"description":"karyoploteR creates karyotype plots of arbitrary genomes\nand offers a complete set of functions to plot arbitrary data\non them. It mimicks many R base graphics functions coupling\nthem with a coordinate change function automatically mapping\nthe chromosome and data coordinates into the plot coordinates.\nIn addition to the provided data plotting functions, it is easy\nto add new ones.","downloads_month":618,"downloads_total":21822,"page":"http://bioconductor.org/packages/release/bioc/html/karyoploteR.html"},{"authors":"Jorma de Ronde, Christiaan Klijn and Arno Velds","name":"KCsmart","license":"GPL-3","tags":["CopyNumberVariation","Visualization","aCGH","Microarray"],"description":"Multi sample aCGH analysis package using kernel\nconvolution","downloads_month":150,"downloads_total":43388,"page":"http://bioconductor.org/packages/release/bioc/html/KCsmart.html"},{"authors":"Johannes Palme","name":"kebabs","license":"GPL (>= 2.1)","tags":["SupportVectorMachine","Classification","Clustering","Regression"],"description":"The package provides functionality for kernel-based\nanalysis of DNA, RNA, and amino acid sequences via SVM-based\nmethods. As core functionality, kebabs implements following\nsequence kernels: spectrum kernel, mismatch kernel, gappy pair\nkernel, and motif kernel. Apart from an efficient\nimplementation of standard position-independent functionality,\nthe kernels are extended in a novel way to take the position of\npatterns into account for the similarity measure. Because of\nthe flexibility of the kernel formulation, other kernels like\nthe weighted degree kernel or the shifted weighted degree\nkernel with constant weighting of positions are included as\nspecial cases. An annotation-specific variant of the kernels\nuses annotation information placed along the sequence together\nwith the patterns in the sequence. The package allows for the\ngeneration of a kernel matrix or an explicit feature\nrepresentation in dense or sparse format for all available\nkernels which can be used with methods implemented in other R\npackages. With focus on SVM-based methods, kebabs provides a\nframework which simplifies the usage of existing SVM\nimplementations in kernlab, e1071, and LiblineaR. Binary and\nmulti-class classification as well as regression tasks can be\nused in a unified way without having to deal with the different\nfunctions, parameters, and formats of the selected SVM. As\nsupport for choosing hyperparameters, the package provides\ncross validation - including grouped cross validation, grid\nsearch and model selection functions. For easier biological\ninterpretation of the results, the package computes feature\nweights for all SVMs and prediction profiles which show the\ncontribution of individual sequence positions to the prediction\nresult and indicate the relevance of sequence sections for the\nlearning result and the underlying biological functions.","downloads_month":267,"downloads_total":26510,"page":"http://bioconductor.org/packages/release/bioc/html/kebabs.html"},{"authors":"Jitao David Zhang and with inputs from Paul Shannon","name":"KEGGgraph","license":"GPL (>= 2)","tags":["Pathways","GraphAndNetwork","Visualization","KEGG"],"description":"KEGGGraph is an interface between KEGG pathway and graph\nobject as well as a collection of tools to analyze, dissect and\nvisualize these graphs. It parses the regularly updated KGML\n(KEGG XML) files into graph models maintaining all essential\npathway attributes. The package offers functionalities\nincluding parsing, graph operation, visualization and etc.","downloads_month":3759,"downloads_total":327998,"page":"http://bioconductor.org/packages/release/bioc/html/KEGGgraph.html"},{"authors":"Shana White","name":"KEGGlincs","license":"GPL-3","tags":["NetworkInference","GeneExpression","DataRepresentation","ThirdPartyClient","CellBiology","GraphAndNetwork","Pathways","KEGG","Network"],"description":"See what is going on 'under the hood' of KEGG pathways by\nexplicitly re-creating the pathway maps from information\nobtained from KGML files.","downloads_month":136,"downloads_total":7692,"page":"http://bioconductor.org/packages/release/bioc/html/KEGGlincs.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"keggorthology","license":"Artistic-2.0","tags":["Pathways","GraphAndNetwork","Visualization","KEGG"],"description":"graphical representation of the Feb 2010 KEGG Orthology.\nThe KEGG orthology is a set of pathway IDs that are not to be\nconfused with the KEGG ortholog IDs.","downloads_month":166,"downloads_total":46334,"page":"http://bioconductor.org/packages/release/bioc/html/keggorthology.html"},{"authors":"Shilin Zhao, Yan Guo and Yu Shyr","name":"KEGGprofile","license":"GPL (>= 2)","tags":["Pathways","KEGG"],"description":"KEGGprofile is an annotation and visualization tool which\nintegrated the expression profiles and the function annotation\nin KEGG pathway maps. The multi-types and multi-groups\nexpression data can be visualized in one pathway map.\nKEGGprofile facilitated more detailed analysis about the\nspecific function changes inner pathway or temporal\ncorrelations in different genes and samples.","downloads_month":328,"downloads_total":46684,"page":"http://bioconductor.org/packages/release/bioc/html/KEGGprofile.html"},{"authors":"Dan Tenenbaum","name":"KEGGREST","license":"Artistic-2.0","tags":["Annotation","Pathways","ThirdPartyClient","KEGG"],"description":"A package that provides a client interface to the KEGG\nREST server. Based on KEGGSOAP by J. Zhang, R. Gentleman, and\nMarc Carlson, and KEGG (python package) by Aurelien Mazurie.","downloads_month":4981,"downloads_total":423780,"page":"http://bioconductor.org/packages/release/bioc/html/KEGGREST.html"},{"authors":"Maria Laura Zingaretti, Johanna Altair Demey-Zambrano, Jose\nLuis Vicente-Villardon and Jhonny Rafael Demey","name":"kimod","license":"GPL (>=2)","tags":["Microarray","Visualization","GeneExpression","ExperimentData","Proteomics"],"description":"This package allows to work with mixed omics data\n(transcriptomics, proteomics, microarray-chips, rna-seq data),\nintroducing the following improvements: distance options (for\nnumeric and/or categorical variables) for each of the tables,\nbootstrap resampling techniques on the residuals matrices for\nall methods, that enable perform confidence ellipses for the\nprojection of individuals, variables and biplot methodology to\nproject variables (gene expression) on the compromise. Since\nthe main purpose of the package is to use these techniques to\nomic data analysis, it includes an example data from four\ndifferent microarray platforms (i.e.,Agilent, Affymetrix HGU\n95, Affymetrix HGU 133 and Affymetrix HGU 133plus 2.0) on the\nNCI-60 cell lines.NCI60_4arrays is a list containing the NCI-60\nmicroarray data with only few hundreds of genes randomly\nselected in each platform to keep the size of the package\nsmall. The data are the same that the package omicade4 used to\nimplement the co-inertia analysis. The references in packages\nfollow the style of the APA-6th norm.","downloads_month":136,"downloads_total":8036,"page":"http://bioconductor.org/packages/release/bioc/html/kimod.html"},{"authors":"Ashley J. Waardenberg [aut and cre]","name":"KinSwingR","license":"GPL-3","tags":["Proteomics","SequenceMatching","Network"],"description":"KinSwingR integrates phosphosite data derived from\nmass-spectrometry data and kinase-substrate predictions to\npredict kinase activity. Several functions allow the user to\nbuild PWM models of kinase-subtrates, statistically infer\nPWM:substrate matches, and integrate these data to infer kinase\nactivity.","downloads_month":90,"downloads_total":652,"page":"http://bioconductor.org/packages/release/bioc/html/KinSwingR.html"},{"authors":"Clara Benoit-Pilven [aut], Camille Marchet [aut], Janice\nKielbassa [aut], Lilia Brinza [aut], Audric Cologne [aut], Aurélie Siberchicot [aut, cre], Vincent Lacroix [aut], Frank\nPicard [ctb], Laurent Jacob [ctb] and Vincent Miele [ctb]","name":"kissDE","license":"GPL (>= 2)","tags":["AlternativeSplicing","DifferentialSplicing","ExperimentalDesign","GenomicVariation","RNASeq","Transcriptomics"],"description":"Retrieves condition-specific variants in RNA-seq data\n(SNVs, alternative-splicings, indels). It has been developed as\na post-treatment of 'KisSplice' but can also be used with\nuser's own data.","downloads_month":151,"downloads_total":2394,"page":"http://bioconductor.org/packages/release/bioc/html/kissDE.html"},{"authors":"Yann Ruffieux, contributions from Debjani Bhowmick, Anthony C.\nDavison, and Darlene R. Goldstein","name":"lapmix","license":"GPL (>= 2)","tags":["Microarray","OneChannel","DifferentialExpression"],"description":"Laplace mixture modelling of microarray experiments. A\nhierarchical Bayesian approach is used, and the hyperparameters\nare estimated using empirical Bayes. The main purpose is to\nidentify differentially expressed genes.","downloads_month":135,"downloads_total":43996,"page":"http://bioconductor.org/packages/release/bioc/html/lapmix.html"},{"authors":"Cyril Dalmasso","name":"LBE","license":"GPL-2","tags":["MultipleComparison"],"description":"LBE is an efficient procedure for estimating the\nproportion of true null hypotheses, the false discovery rate\n(and so the q-values) in the framework of estimating procedures\nbased on the marginal distribution of the p-values without\nassumption for the alternative hypothesis.","downloads_month":230,"downloads_total":58436,"page":"http://bioconductor.org/packages/release/bioc/html/LBE.html"},{"authors":"Eric Frichot <eric.frichot@gmail.com> and Olivier Francois\n<olivier.francois@grenoble-inp.fr>","name":"LEA","license":"GPL-3","tags":["Software","StatisticalMethod","Clustering","Regression"],"description":"LEA is an R package dedicated to landscape genomics and\necological association tests. LEA can run analyses of\npopulation structure and genomewide tests for local adaptation.\nThe package includes statistical methods for estimating\nancestry coefficients from large genotypic matrices and for\nevaluating the number of ancestral populations (snmf, pca). It\nperforms statistical tests using latent factor mixed models for\nidentifying genetic polymorphisms that exhibit association with\nenvironmental gradients or phenotypic traits (lfmm). LEA is\nmainly based on optimized C programs that can scale with the\ndimension of large data sets.","downloads_month":404,"downloads_total":34084,"page":"http://bioconductor.org/packages/release/bioc/html/LEA.html"},{"authors":"Elodie Darbo, Denis Seyres and Aitor Gonzalez","name":"LedPred","license":"MIT | file LICENSE","tags":["SupportVectorMachine","Software","MotifAnnotation","ChIPSeq","Sequencing","Classification"],"description":"This package aims at creating a predictive model of\nregulatory sequences used to score unknown sequences based on\nthe content of DNA motifs, next-generation sequencing (NGS)\npeaks and signals and other numerical scores of the sequences\nusing supervised classification. The package contains a\nworkflow based on the support vector machine (SVM) algorithm\nthat maps features to sequences, optimize SVM parameters and\nfeature number and creates a model that can be stored and used\nto score the regulatory potential of unknown sequences.","downloads_month":141,"downloads_total":10592,"page":"http://bioconductor.org/packages/release/bioc/html/LedPred.html"},{"authors":"Julian Gehring, Clemens Kreutz and Jens Timmer","name":"les","license":"GPL-3","tags":["Microarray","DifferentialExpression","ChIPchip","DNAMethylation","Transcription"],"description":"The 'les' package estimates Loci of Enhanced Significance\n(LES) in tiling microarray data. These are regions of\nregulation such as found in differential transcription,\nCHiP-chip, or DNA modification analysis. The package provides a\nuniversal framework suitable for identifying differential\neffects in tiling microarray data sets, and is independent of\nthe underlying statistics at the level of single probes.","downloads_month":169,"downloads_total":36630,"page":"http://bioconductor.org/packages/release/bioc/html/les.html"},{"authors":"Jose Rafael Pilan <rafael.pilan@unesp.br>, Agnes Alessandra\nSekijima Takeda <agnes.takeda@unesp.br> and Jose Luiz Rybarczyk\nFilho <jose.luiz@unesp.br>","name":"levi","license":"GPL (>= 2)","tags":["GeneExpression","Sequencing","Network"],"description":"The tool integrates data from biological networks with\ntranscriptomes, displaying a heatmap with surface curves to\nevidence the altered regions.","downloads_month":80,"downloads_total":618,"page":"http://bioconductor.org/packages/release/bioc/html/levi.html"},{"authors":"Wei Hao, Minsun Song and John D. Storey","name":"lfa","license":"GPL-3","tags":["SNP","DimensionReduction","PrincipalComponent"],"description":"LFA is a method for a PCA analogue on Binomial data via\nestimation of latent structure in the natural parameter.","downloads_month":312,"downloads_total":25562,"page":"http://bioconductor.org/packages/release/bioc/html/lfa.html"},{"authors":"Gordon Smyth [cre, aut], Yifang Hu [ctb], Matthew Ritchie [ctb], Jeremy Silver [ctb], James Wettenhall [ctb], Davis McCarthy\n[ctb], Di Wu [ctb], Wei Shi [ctb], Belinda Phipson [ctb], Aaron\nLun [ctb], Natalie Thorne [ctb], Alicia Oshlack [ctb], Carolyn\nde Graaf [ctb], Yunshun Chen [ctb], Mette Langaas [ctb], Egil\nFerkingstad [ctb], Marcus Davy [ctb], Francois Pepin [ctb] and Dongseok Choi [ctb]","name":"limma","license":"GPL (>=2)","tags":["ExonArray","GeneExpression","Transcription","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","GeneSetEnrichment","DataImport","Bayesian","Clustering","Regression","TimeCourse","Microarray","MicroRNAArray","mRNAMicroarray","OneChannel","ProprietaryPlatforms","TwoChannel","Sequencing","RNASeq","BatchEffect","MultipleComparison","Normalization","Preprocessing","QualityControl","BiomedicalInformatics","CellBiology","Cheminformatics","Epigenetics","FunctionalGenomics","Genetics","ImmunoOncology","Metabolomics","Proteomics","SystemsBiology","Transcriptomics"],"description":"Data analysis, linear models and differential expression\nfor microarray data.","downloads_month":32585,"downloads_total":4132300,"page":"http://bioconductor.org/packages/release/bioc/html/limma.html"},{"authors":"James Wettenhall [aut], Gordon Smyth [aut] and Keith Satterley\n[ctb]","name":"limmaGUI","license":"GPL (>=2)","tags":["GUI","GeneExpression","DifferentialExpression","DataImport","Bayesian","Regression","TimeCourse","Microarray","mRNAMicroarray","TwoChannel","BatchEffect","MultipleComparison","Normalization","Preprocessing","QualityControl"],"description":"A Graphical User Interface for differential expression\nanalysis of two-color microarray data using the limma package.","downloads_month":211,"downloads_total":79782,"page":"http://bioconductor.org/packages/release/bioc/html/limmaGUI.html"},{"authors":"Manuel Goepferich and Carl Herrmann","name":"LINC","license":"Artistic-2.0","tags":["Software","BiologicalQuestion","GeneRegulation","GeneExpression"],"description":"This package provides methods to compute co-expression\nnetworks of lincRNAs and protein-coding genes. Biological terms\nassociated with the sets of protein-coding genes predict the\nbiological contexts of lincRNAs according to the 'Guilty by\nAssociation' approach.","downloads_month":142,"downloads_total":7348,"page":"http://bioconductor.org/packages/release/bioc/html/LINC.html"},{"authors":"David S Fischer [aut, cre], Fabian Theis [ctb] and Nir Yosef [ctb]","name":"LineagePulse","license":"Artistic-2.0","tags":["ImmunoOncology","Software","StatisticalMethod","TimeCourse","Sequencing","DifferentialExpression","GeneExpression","CellBiology","CellBasedAssays","SingleCell"],"description":"LineagePulse is a differential expression and expression\nmodel fitting package tailored to single-cell RNA-seq data\n(scRNA-seq). LineagePulse accounts for batch effects, drop-out\nand variable sequencing depth. One can use LineagePulse to\nperform longitudinal differential expression analysis across\npseudotime as a continuous coordinate or between discrete\ngroups of cells (e.g. pre-defined clusters or experimental\nconditions). Expression model fits can be directly extracted\nfrom LineagePulse.","downloads_month":161,"downloads_total":2474,"page":"http://bioconductor.org/packages/release/bioc/html/LineagePulse.html"},{"authors":"Shun Hang Yip <shunyip@bu.edu>, Panwen Wang\n<pwwang@pwwang.com>, Jean-Pierre Kocher\n<Kocher.JeanPierre@mayo.edu>, Pak Chung Sham <pcsham@hku.hk> and Junwen Wang <junwen@uw.edu>","name":"Linnorm","license":"MIT + file LICENSE","tags":["ImmunoOncology","Sequencing","ChIPSeq","RNASeq","DifferentialExpression","GeneExpression","Genetics","Normalization","Software","Transcription","BatchEffect","PeakDetection","Clustering","Network","SingleCell"],"description":"Linnorm is an R package for the analysis of RNA-seq,\nscRNA-seq, ChIP-seq count data or any large scale count data.\nIt normalizes and performs variance-stabilizing transformation\non such datasets. In addition to the transformtion function\n(Linnorm), the following pipelines are implemented: 1. Library\nsize/Batch effect normalization (Linnorm.Norm), 2. Cell\nsubpopluation analysis and visualization using t-SNE or PCA\nK-means clustering or Hierarchical clustering (Linnorm.tSNE,\nLinnorm.PCA, Linnorm.HClust), 3. Differential expression\nanalysis or differential peak detection using limma\n(Linnorm.limma), 4. Highly variable gene discovery and\nvisualization (Linnorm.HVar), 5. Gene correlation network\nanalysis and visualization (Linnorm.Cor), 6. Stable gene\nselection for scRNA-seq data; for users without or do not want\nto rely on spike-in genes (Linnorm.SGenes). 7. Data imputation.\n(under development) (Linnorm.DataImput). Linnorm can work with\nraw count, CPM, RPKM, FPKM and TPM. Additionally, the RnaXSim\nfunction is included for simulating RNA-seq data for the\nevaluation of DEG analysis methods.","downloads_month":297,"downloads_total":12608,"page":"http://bioconductor.org/packages/release/bioc/html/Linnorm.html"},{"authors":"Yen-Yi Ho <yho@jhsph.edu>","name":"LiquidAssociation","license":"GPL (>=3)","tags":["Pathways","GeneExpression","CellBiology","Genetics","Network","TimeCourse"],"description":"The package contains functions for calculate direct and\nmodel-based estimators for liquid association. It also provides\nfunctions for testing the existence of liquid association given\na gene triplet data.","downloads_month":154,"downloads_total":39876,"page":"http://bioconductor.org/packages/release/bioc/html/LiquidAssociation.html"},{"authors":"Cristobal Fresno and Elmer A. Fernandez","name":"lmdme","license":"GPL (>=2)","tags":["Microarray","OneChannel","TwoChannel","Visualization","DifferentialExpression","ExperimentData","Cancer"],"description":"linear ANOVA decomposition of Multivariate Designed\nExperiments implementation based on limma lmFit. Features:\ni)Flexible formula type interface, ii) Fast limma based\nimplementation, iii) p-values for each estimated coefficient\nlevels in each factor, iv) F values for factor effects and v)\nplotting functions for PCA and PLS.","downloads_month":138,"downloads_total":29156,"page":"http://bioconductor.org/packages/release/bioc/html/lmdme.html"},{"authors":"David Rocke, Geun Cheol Lee, John Tillinghast, Blythe\nDurbin-Johnson, and Shiquan Wu","name":"LMGene","license":"LGPL","tags":["Microarray","DifferentialExpression","Preprocessing"],"description":"LMGene package for analysis of microarray data using a\nlinear model and glog data transformation","downloads_month":184,"downloads_total":50848,"page":"http://bioconductor.org/packages/release/bioc/html/LMGene.html"},{"authors":"James Collins [aut, cre], Helen Fredricks [aut], Bethanie\nEdwards [aut] and Benjamin Van Mooy [aut]","name":"LOBSTAHS","license":"GPL (>= 3) + file LICENSE","tags":["ImmunoOncology","MassSpectrometry","Metabolomics","Lipidomics","DataImport"],"description":"LOBSTAHS is a multifunction package for screening,\nannotation, and putative identification of mass spectral\nfeatures in large, HPLC-MS lipid datasets. In silico data for a\nwide range of lipids, oxidized lipids, and oxylipins can be\ngenerated from user-supplied structural criteria with a\ndatabase generation function. LOBSTAHS then applies these\ndatabases to assign putative compound identities to features in\nany high-mass accuracy dataset that has been processed using\nxcms and CAMERA. Users can then apply a series of orthogonal\nscreening criteria based on adduct ion formation patterns,\nchromatographic retention time, and other properties, to\nevaluate and assign confidence scores to this list of\npreliminary assignments. During the screening routine, LOBSTAHS\nrejects assignments that do not meet the specified criteria,\nidentifies potential isomers and isobars, and assigns a variety\nof annotation codes to assist the user in evaluating the\naccuracy of each assignment.","downloads_month":182,"downloads_total":8012,"page":"http://bioconductor.org/packages/release/bioc/html/LOBSTAHS.html"},{"authors":"Tianlei Xu","name":"loci2path","license":"Artistic-2.0","tags":["FunctionalGenomics","Genetics","GeneSetEnrichment","Software","GeneExpression","Sequencing","Coverage","BioCarta"],"description":"loci2path performs statistics-rigorous enrichment analysis\nof eQTLs in genomic regions of interest. Using eQTL collections\nprovided by the Genotype-Tissue Expression (GTEx) project and\npathway collections from MSigDB.","downloads_month":126,"downloads_total":2318,"page":"http://bioconductor.org/packages/release/bioc/html/loci2path.html"},{"authors":"Holger Schwender and Tobias Tietz","name":"logicFS","license":"LGPL (>= 2)","tags":["SNP","Classification","Genetics"],"description":"Identification of interactions between binary variables\nusing Logic Regression. Can, e.g., be used to find interesting\nSNP interactions. Contains also a bagging version of logic\nregression for classification.","downloads_month":280,"downloads_total":61896,"page":"http://bioconductor.org/packages/release/bioc/html/logicFS.html"},{"authors":"Tobias Guennel <tguennel@vcu.edu>","name":"logitT","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression"],"description":"The logitT library implements the Logit-t algorithm\nintroduced in --A high performance test of differential gene\nexpression for oligonucleotide arrays-- by William J Lemon,\nSandya Liyanarachchi and Ming You for use with Affymetrix data\nstored in an AffyBatch object in R.","downloads_month":129,"downloads_total":42212,"page":"http://bioconductor.org/packages/release/bioc/html/logitT.html"},{"authors":"Kushal Dey [aut, cre], Dongyue Xie [aut], Peter Carbonetto\n[ctb] and Matthew Stephens [aut]","name":"Logolas","license":"GPL (>= 2)","tags":["SequenceMatching","Alignment","Software","Visualization","Bayesian"],"description":"Produces logo plots highlighting both enrichment and\ndepletion of characters, allows for plotting of string symbols,\nand performs scaling of position-weights adaptively, along with\nseveral fun stylizations.","downloads_month":176,"downloads_total":14602,"page":"http://bioconductor.org/packages/release/bioc/html/Logolas.html"},{"authors":"Yinyin Yuan","name":"lol","license":"GPL-2","tags":["StatisticalMethod"],"description":"Various optimization methods for Lasso inference with\nmatrix warpper","downloads_month":157,"downloads_total":32392,"page":"http://bioconductor.org/packages/release/bioc/html/lol.html"},{"authors":"Nathan Sheffield <http://www.databio.org> [aut, cre] and Christoph\nBock [ctb]","name":"LOLA","license":"GPL-3","tags":["GeneSetEnrichment","GeneRegulation","GenomeAnnotation","SystemsBiology","FunctionalGenomics","ChIPSeq","MethylSeq","Sequencing"],"description":"Provides functions for testing overlap of sets of genomic\nregions with public and custom region set (genomic ranges)\ndatabases. This makes it possible to do automated enrichment\nanalysis for genomic region sets, thus facilitating\ninterpretation of functional genomics and epigenomics data.","downloads_month":227,"downloads_total":16116,"page":"http://bioconductor.org/packages/release/bioc/html/LOLA.html"},{"authors":"Martin Morgan and Daniel Van Twisk","name":"LoomExperiment","license":"Artistic-2.0","tags":["ImmunoOncology","DataRepresentation","DataImport","Infrastructure","SingleCell"],"description":"The LoomExperiment class provide a means to easily convert\nBioconductor's \"Experiment\" classes to loom files and vice\nversa.","downloads_month":137,"downloads_total":1126,"page":"http://bioconductor.org/packages/release/bioc/html/LoomExperiment.html"},{"authors":"Stefano de Pretis and Giorgio Melloni","name":"LowMACA","license":"GPL-3","tags":["SomaticMutation","SequenceMatching","WholeGenome","Sequencing","Alignment","DataImport","MultipleSequenceAlignment"],"description":"The LowMACA package is a simple suite of tools to\ninvestigate and analyze the mutation profile of several\nproteins or pfam domains via consensus alignment. You can\nconduct an hypothesis driven exploratory analysis using our\npackage simply providing a set of genes or pfam domains of your\ninterest.","downloads_month":119,"downloads_total":13364,"page":"http://bioconductor.org/packages/release/bioc/html/LowMACA.html"},{"authors":"Nitin Jain <emailnitinjain@gmail.com>, Michael O'Connell\n<michaelo@warath.com> and Jae K. Lee <jaeklee@virginia.edu>.\nIncludes R source code contributed by HyungJun Cho\n<hcho@virginia.edu>","name":"LPE","license":"LGPL","tags":["Microarray","DifferentialExpression"],"description":"This LPE library is used to do significance analysis of\nmicroarray data with small number of replicates. It uses\nresampling based FDR adjustment, and gives less conservative\nresults than traditional 'BH' or 'BY' procedures. Data accepted\nis raw data in txt format from MAS4, MAS5 or dChip. Data can\nalso be supplied after normalization. LPE library is primarily\nused for analyzing data between two conditions. To use it for\npaired data, see LPEP library. For using LPE in multiple\nconditions, use HEM library.","downloads_month":157,"downloads_total":59888,"page":"http://bioconductor.org/packages/release/bioc/html/LPE.html"},{"authors":"Carl Murie <carl.murie@mcgill.ca> and Robert Nadon\n<robert.nadon@mcgill.ca>","name":"LPEadj","license":"LGPL","tags":["Microarray","Proteomics"],"description":"Two options are added to the LPE algorithm. The original\nLPE method sets all variances below the max variance in the\nordered distribution of variances to the maximum variance. in\nLPEadj this option is turned off by default.  The second option\nis to use a variance adjustment based on sample size rather\nthan pi/2.  By default the LPEadj uses the sample size based\nvariance adjustment.","downloads_month":132,"downloads_total":42482,"page":"http://bioconductor.org/packages/release/bioc/html/LPEadj.html"},{"authors":"Bettina Knapp, Marta R. A. Matos, Johanna Mazur and Lars Kaderali","name":"lpNet","license":"Artistic License 2.0","tags":["NetworkInference"],"description":"lpNet aims at infering biological networks, in particular\nsignaling and gene networks. For that it takes perturbation\ndata, either steady-state or time-series, as input and\ngenerates an LP model which allows the inference of signaling\nnetworks. For parameter identification either leave-one-out\ncross-validation or stratified n-fold cross-validation can be\nused.","downloads_month":131,"downloads_total":24818,"page":"http://bioconductor.org/packages/release/bioc/html/lpNet.html"},{"authors":"Vladislav Kim [aut, cre], Ted Ralphs [ctb], Menal Guzelsoy\n[ctb], Ashutosh Mahajan [ctb], Reinhard Harter [ctb], Kurt\nHornik [ctb], Cyrille Szymanski [ctb] and Stefan Theussl [ctb]","name":"lpsymphony","license":"EPL","tags":["Infrastructure","ThirdPartyClient"],"description":"This package was derived from Rsymphony_0.1-17 from CRAN.\nThese packages provide an R interface to SYMPHONY, an\nopen-source linear programming solver written in C++. The main\ndifference between this package and Rsymphony is that it\nincludes the solver source code (SYMPHONY version 5.6), while\nRsymphony expects to find header and library files on the\nusers' system. Thus the intention of lpsymphony is to provide\nan easy to install interface to SYMPHONY. For Windows,\nprecompiled DLLs are included in this package.","downloads_month":926,"downloads_total":45478,"page":"http://bioconductor.org/packages/release/bioc/html/lpsymphony.html"},{"authors":"Koki Tsuyuzaki","name":"LRBaseDbi","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Interface to construct LRBase package (LRBase.XXX.eg.db).","downloads_month":105,"downloads_total":848,"page":"http://bioconductor.org/packages/release/bioc/html/LRBaseDbi.html"},{"authors":"Pan Du, Richard Bourgon, Gang Feng and Simon Lin","name":"lumi","license":"LGPL (>= 2)","tags":["Microarray","OneChannel","Preprocessing","DNAMethylation","QualityControl","TwoChannel"],"description":"The lumi package provides an integrated solution for the\nIllumina microarray data analysis. It includes functions of\nIllumina BeadStudio (GenomeStudio) data input, quality control,\nBeadArray-specific variance stabilization, normalization and\ngene annotation at the probe level. It also includes the\nfunctions of processing Illumina methylation microarrays,\nespecially Illumina Infinium methylation microarrays.","downloads_month":1813,"downloads_total":448970,"page":"http://bioconductor.org/packages/release/bioc/html/lumi.html"},{"authors":"Stefano Calza, Suo Chen and Yudi Pawitan","name":"LVSmiRNA","license":"GPL-2","tags":["Microarray","AgilentChip","OneChannel","Preprocessing"],"description":"Normalization of Agilent miRNA arrays.","downloads_month":148,"downloads_total":37404,"page":"http://bioconductor.org/packages/release/bioc/html/LVSmiRNA.html"},{"authors":"David Coffey <dcoffey@fredhutch.org>","name":"LymphoSeq","license":"Artistic-2.0","tags":["Software","Technology","Sequencing","TargetedResequencing","Alignment","MultipleSequenceAlignment"],"description":"This R package analyzes high-throughput sequencing of T\nand B cell receptor complementarity determining region 3 (CDR3)\nsequences generated by Adaptive Biotechnologies' ImmunoSEQ\nassay.  Its input comes from tab-separated value (.tsv) files\nexported from the ImmunoSEQ analyzer.","downloads_month":170,"downloads_total":8918,"page":"http://bioconductor.org/packages/release/bioc/html/LymphoSeq.html"},{"authors":"Christopher John and David Watson","name":"M3C","license":"AGPL-3","tags":["Clustering","GeneExpression","Transcription","RNASeq","Sequencing","ImmunoOncology"],"description":"M3C is a consensus clustering algorithm that uses a Monte\nCarlo simulation to eliminate overfitting and can reject the\nnull hypothesis K=1.","downloads_month":176,"downloads_total":3556,"page":"http://bioconductor.org/packages/release/bioc/html/M3C.html"},{"authors":"Tom Mayo","name":"M3D","license":"Artistic License 2.0","tags":["DNAMethylation","DifferentialMethylation","Coverage","CpGIsland"],"description":"This package identifies statistically significantly\ndifferentially methylated regions of CpGs. It uses kernel\nmethods (the Maximum Mean Discrepancy) to measure differences\nin methylation profiles, and relates these to inter-replicate\nchanges, whilst accounting for variation in coverage profiles.","downloads_month":149,"downloads_total":16038,"page":"http://bioconductor.org/packages/release/bioc/html/M3D.html"},{"authors":"Tallulah Andrews <tallulandrews@gmail.com>","name":"M3Drop","license":"GPL (>=2)","tags":["ImmunoOncology","RNASeq","Sequencing","Transcriptomics","GeneExpression","Software","DifferentialExpression","DimensionReduction","FeatureExtraction"],"description":"This package fits a Michaelis-Menten model to the pattern\nof dropouts in single-cell RNASeq data. This model is used as a\nnull to identify significantly variable (i.e. differentially\nexpressed) genes for use in downstream analysis, such as\nclustering cells.","downloads_month":421,"downloads_total":14532,"page":"http://bioconductor.org/packages/release/bioc/html/M3Drop.html"},{"authors":"Hao Wu, modified by Hyuna Yang, Keith Sheppard with ideas\nfrom Gary Churchill and Katie Kerr and Xiangqin Cui.","name":"maanova","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","Clustering"],"description":"Analysis of N-dye Micro Array experiment using mixed model\neffect. Containing analysis of variance, permutation and\nbootstrap, cluster and consensus tree.","downloads_month":168,"downloads_total":59892,"page":"http://bioconductor.org/packages/release/bioc/html/maanova.html"},{"authors":"Benjamin Georgi, Matthias Heinig, Stefan Roepcke, Sebastian\nSchmeier and Joern Toedling","name":"macat","license":"Artistic-2.0","tags":["Microarray","DifferentialExpression","Visualization"],"description":"This library contains functions to investigate links\nbetween differential gene expression and the chromosomal\nlocalization of the genes. MACAT is motivated by the common\nobservation of phenomena involving large chromosomal regions in\ntumor cells. MACAT is the implementation of a statistical\napproach for identifying significantly differentially expressed\nchromosome regions. The functions have been tested on a\npublicly available data set about acute lymphoblastic leukemia\n(Yeoh et al.Cancer Cell 2002), which is provided in the library\n'stjudem'.","downloads_month":160,"downloads_total":47024,"page":"http://bioconductor.org/packages/release/bioc/html/macat.html"},{"authors":"Alexander Ploner <Alexander.Ploner@ki.se>","name":"maCorrPlot","license":"GPL (>= 2)","tags":["Microarray","Preprocessing","Visualization"],"description":"Graphically displays correlation in microarray data that\nis due to insufficient normalization","downloads_month":139,"downloads_total":43946,"page":"http://bioconductor.org/packages/release/bioc/html/maCorrPlot.html"},{"authors":"Ioannis Vardaxis","name":"MACPET","license":"GPL-3","tags":["Software","DNA3DStructure","PeakDetection","StatisticalMethod","Clustering","Classification","HiC"],"description":"The MACPET package can be used for complete interaction\nanalysis for ChIA-PET data. MACPET reads ChIA-PET data in BAM\nor SAM format and separates the data into Self-ligated, Intra-\nand Inter-chromosomal PETs. Furthermore, MACPET breaks the\ngenome into regions and applies 2D mixture models for\nidentifying candidate peaks/binding sites using skewed\ngeneralized students-t distributions (SGT). It then uses a\nlocal poisson model for finding significant binding sites.\nFinally it runs an additive interaction-analysis model for\ncalling for significant interactions between those peaks.\nMACPET is mainly written in C++, and it also supports the\nBiocParallel package.","downloads_month":124,"downloads_total":2080,"page":"http://bioconductor.org/packages/release/bioc/html/MACPET.html"},{"authors":"Aedin Culhane","name":"made4","license":"Artistic-2.0","tags":["Clustering","Classification","MultipleComparison"],"description":"Multivariate data analysis and graphical display of\nmicroarray data. Functions include between group analysis and\ncoinertia analysis. It contains functions that require ADE4.","downloads_month":536,"downloads_total":122188,"page":"http://bioconductor.org/packages/release/bioc/html/made4.html"},{"authors":"Yu Kong, Adam Auton and John Murray Greally","name":"MADSEQ","license":"GPL(>=2)","tags":["GenomicVariation","SomaticMutation","VariantDetection","Bayesian","CopyNumberVariation","Sequencing","Coverage"],"description":"The MADSEQ package provides a group of hierarchical\nBayeisan models for the detection of mosaic aneuploidy, the\ninference of the type of aneuploidy and also for the\nquantification of the fraction of aneuploid cells in the\nsample.","downloads_month":139,"downloads_total":7086,"page":"http://bioconductor.org/packages/release/bioc/html/MADSEQ.html"},{"authors":"Anand Mayakonda <anand_mt@hotmail.com>","name":"maftools","license":"MIT + file LICENSE","tags":["DataRepresentation","DNASeq","Visualization","DriverMutation","VariantAnnotation","FeatureExtraction","Classification","SomaticMutation","Sequencing","FunctionalGenomics","Survival"],"description":"Analyze and visualize Mutation Annotation Format (MAF)\nfiles from large scale sequencing studies. This package\nprovides various functions to perform most commonly used\nanalyses in cancer genomics and to create feature rich\ncustomizable visualzations with minimal effort.","downloads_month":1930,"downloads_total":62642,"page":"http://bioconductor.org/packages/release/bioc/html/maftools.html"},{"authors":"Wubing Zhang, Feizhen Wu and Binbin Wang","name":"MAGeCKFlute","license":"GPL (>=3)","tags":["Software","FunctionalGenomics","CRISPR","ImmunoOncology","BatchEffect","QualityControl","Normalization","GeneSetEnrichment","Pathways","Visualization"],"description":"MAGeCKFlute is designed to surporting downstream analysis,\nutilizing the gene summary data provided through MAGeCK or\nMAGeCK-VISPR. Quality control, normalization, and screen hit\nidentification for CRISPR screen data are performed in\npipeline. Identified hits within the pipeline are categorized\nbased on experimental design, and are subsequently interpreted\nby functional enrichment analysis.","downloads_month":215,"downloads_total":3662,"page":"http://bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html"},{"authors":"Gustavo H. Esteves <gesteves@gmail.com>, with contributions\nfrom Roberto Hirata Jr <hirata@ime.usp.br>, E. Jordao Neves\n<neves@ime.usp.br>, Elier B. Cristo <elier@ime.usp.br>, Ana C.\nSimoes <anakqui@ime.usp.br> and Lucas Fahham\n<fahham@linux.ime.usp.br>","name":"maigesPack","license":"GPL (>= 2)","tags":["Microarray","TwoChannel","Preprocessing","ThirdPartyClient","DifferentialExpression","Clustering","Classification","GraphAndNetwork"],"description":"This package uses functions of various other packages\ntogether with other functions in a coordinated way to handle\nand analyse cDNA microarray data","downloads_month":154,"downloads_total":46856,"page":"http://bioconductor.org/packages/release/bioc/html/maigesPack.html"},{"authors":"Francesc Fernandez-Albert, Rafael Llorach, Cristina\nAndres-LaCueva and Alexandre Perera","name":"MAIT","license":"GPL-2","tags":["ImmunoOncology","MassSpectrometry","Metabolomics","Software"],"description":"The MAIT package contains functions to perform end-to-end\nstatistical analysis of LC/MS Metabolomic Data. Special\nemphasis is put on peak annotation and in modular function\ndesign of the functions.","downloads_month":274,"downloads_total":23058,"page":"http://bioconductor.org/packages/release/bioc/html/MAIT.html"},{"authors":"Rafael A. Irizarry <rafa@jhu.edu>, Laurent Gautier\n<laurent@cbs.dtu.dk>, Wolfgang Huber\n<w.huber@dkfz-heidelberg.de> and Ben Bolstad <bmb@bmbolstad.com>","name":"makecdfenv","license":"GPL (>= 2)","tags":["OneChannel","DataImport","Preprocessing"],"description":"This package has two functions. One reads a Affymetrix\nchip description file (CDF) and creates a hash table\nenvironment containing the location/probe set membership\nmapping. The other creates a package that automatically loads\nthat environment.","downloads_month":275,"downloads_total":123960,"page":"http://bioconductor.org/packages/release/bioc/html/makecdfenv.html"},{"authors":"Pierre Neuvial <pierre.neuvial@genopole.cnrs.fr> and Philippe Hupe\n<philippe.hupe@curie.fr>","name":"MANOR","license":"GPL-2","tags":["Microarray","TwoChannel","DataImport","QualityControl","Preprocessing","CopyNumberVariation"],"description":"Importation, normalization, visualization, and quality\ncontrol functions to correct identified sources of variability\nin array-CGH experiments.","downloads_month":141,"downloads_total":46354,"page":"http://bioconductor.org/packages/release/bioc/html/MANOR.html"},{"authors":"Ginger Armbrust and Adrian Marchetti","name":"manta","license":"Artistic-2.0","tags":["ImmunoOncology","DifferentialExpression","RNASeq","Genetics","GeneExpression","Sequencing","QualityControl","DataImport","Visualization"],"description":"Tools for robust comparative metatranscriptomics.","downloads_month":183,"downloads_total":31048,"page":"http://bioconductor.org/packages/release/bioc/html/manta.html"},{"authors":"Brian Steinmeyer and William Shannon","name":"MantelCorr","license":"GPL (>= 2)","tags":["Clustering"],"description":"Computes Mantel cluster correlations from a (p x n)\nnumeric data matrix (e.g. microarray gene-expression data).","downloads_month":131,"downloads_total":43148,"page":"http://bioconductor.org/packages/release/bioc/html/MantelCorr.html"},{"authors":"Argiris Sakellariou","name":"mAPKL","license":"GPL (>= 2)","tags":["FeatureExtraction","DifferentialExpression","Microarray","GeneExpression"],"description":"We propose a hybrid FS method (mAP-KL), which combines\nmultiple hypothesis testing and affinity propagation\n(AP)-clustering algorithm along with the Krzanowski & Lai\ncluster quality index, to select a small yet informative subset\nof genes.","downloads_month":159,"downloads_total":12976,"page":"http://bioconductor.org/packages/release/bioc/html/mAPKL.html"},{"authors":"Adi Laurentiu Tarca <atarca@med.wayne.edu>","name":"maPredictDSC","license":"GPL-2","tags":["Microarray","Classification"],"description":"This package implements the classification pipeline of the\nbest overall team (Team221) in the IMPROVER Diagnostic\nSignature Challenge. Additional functionality is added to\ncompare 27 combinations of data preprocessing, feature\nselection and classifier types.","downloads_month":142,"downloads_total":22472,"page":"http://bioconductor.org/packages/release/bioc/html/maPredictDSC.html"},{"authors":"Maia Smith [aut and cre]","name":"mapscape","license":"GPL-3","tags":["Visualization"],"description":"MapScape integrates clonal prevalence, clonal hierarchy,\nanatomic and mutational information to provide interactive\nvisualization of spatial clonal evolution. There are four\ninputs to MapScape: (i) the clonal phylogeny, (ii) clonal\nprevalences, (iii) an image reference, which may be a medical\nimage or drawing and (iv) pixel locations for each sample on\nthe referenced image. Optionally, MapScape can accept a data\ntable of mutations for each clone and their variant allele\nfrequencies in each sample. The output of MapScape consists of\na cropped anatomical image surrounded by two representations of\neach tumour sample. The first, a cellular aggregate, visually\ndisplays the prevalence of each clone. The second shows a\nskeleton of the clonal phylogeny while highlighting only those\nclones present in the sample. Together, these representations\nenable the analyst to visualize the distribution of clones\nthroughout anatomic space.","downloads_month":122,"downloads_total":5446,"page":"http://bioconductor.org/packages/release/bioc/html/mapscape.html"},{"authors":"Yee Hwa (Jean) Yang <jeany@maths.usyd.edu.au> with\ncontributions from Agnes Paquet and Sandrine Dudoit.","name":"marray","license":"LGPL","tags":["Microarray","TwoChannel","Preprocessing"],"description":"Class definitions for two-color spotted microarray data.\nFuctions for data input, diagnostic plots, normalization and\nquality checking.","downloads_month":2163,"downloads_total":795166,"page":"http://bioconductor.org/packages/release/bioc/html/marray.html"},{"authors":"Hector Climente-Gonzalez [aut, cre] and Chloe-Agathe Azencott\n[aut]","name":"martini","license":"MIT + file LICENSE","tags":["Software","GenomeWideAssociation","SNP","GeneticVariability","Genetics","FeatureExtraction","GraphAndNetwork","Network"],"description":"martini deals with the low power inherent to GWAS studies\nby using prior knowledge represented as a network. SNPs are the\nvertices of the network, and the edges represent biological\nrelationships between them (genomic adjacency, belonging to the\nsame gene, physical interaction between protein products). The\nnetwork is scanned using SConES, which looks for groups of SNPs\nmaximally associated with the phenotype, that form a close\nsubnetwork.","downloads_month":148,"downloads_total":2270,"page":"http://bioconductor.org/packages/release/bioc/html/martini.html"},{"authors":"Diogo Veiga [aut and cre]","name":"maser","license":"MIT + file LICENSE","tags":["AlternativeSplicing","Transcriptomics","Visualization"],"description":"This package provides functionalities for analysis,\nannotation and visualizaton of alternative splicing events.","downloads_month":105,"downloads_total":1288,"page":"http://bioconductor.org/packages/release/bioc/html/maser.html"},{"authors":"Ana Conesa <aconesa@cipf.es> and Maria Jose Nueda <mj.nueda@ua.es>","name":"maSigPro","license":"GPL (>= 2)","tags":["Microarray","RNA-Seq","Differential Expression","TimeCourse"],"description":"maSigPro is a regression based approach to find genes for\nwhich there are significant gene expression profile differences\nbetween experimental groups in time course microarray and\nRNA-Seq experiments.","downloads_month":457,"downloads_total":98214,"page":"http://bioconductor.org/packages/release/bioc/html/maSigPro.html"},{"authors":"Michael Dannemann <michael_dannemann@eva.mpg.de>","name":"maskBAD","license":"GPL (>= 2)","tags":["Microarray"],"description":"Package includes functions to analyze and mask microarray\nexpression data.","downloads_month":136,"downloads_total":29470,"page":"http://bioconductor.org/packages/release/bioc/html/maskBAD.html"},{"authors":"Reid F. Thompson <rthompso@aecom.yu.edu> and John M. Greally\n<jgreally@aecom.yu.edu>","name":"MassArray","license":"GPL (>=2)","tags":["ImmunoOncology","DNAMethylation","SNP","MassSpectrometry","Genetics","DataImport","Visualization"],"description":"This package is designed for the import, quality control,\nanalysis, and visualization of methylation data generated using\nSequenom's MassArray platform.  The tools herein contain a\nhighly detailed amplicon prediction for optimal assay design.\nAlso included are quality control measures of data, such as\nprimer dimer and bisulfite conversion efficiency estimation.\nMethylation data are calculated using the same algorithms\ncontained in the EpiTyper software package.  Additionally,\nautomatic SNP-detection can be used to flag potentially\nconfounded data from specific CG sites.  Visualization includes\nbarplots of methylation data as well as UCSC Genome\nBrowser-compatible BED tracks.  Multiple assays can be\npositionally combined for integrated analysis.","downloads_month":150,"downloads_total":38946,"page":"http://bioconductor.org/packages/release/bioc/html/MassArray.html"},{"authors":"Sam Buckberry","name":"massiR","license":"GPL-3","tags":["Software","Microarray","GeneExpression","Clustering","Classification","QualityControl"],"description":"Predicts the sex of samples in gene expression microarray\ndatasets","downloads_month":145,"downloads_total":19852,"page":"http://bioconductor.org/packages/release/bioc/html/massiR.html"},{"authors":"Pan Du, Warren Kibbe and Simon Lin","name":"MassSpecWavelet","license":"LGPL (>= 2)","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"Processing Mass Spectrometry spectrum by using wavelet\nbased algorithm","downloads_month":2014,"downloads_total":138910,"page":"http://bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html"},{"authors":"Andrew McDavid [aut, cre], Greg Finak [aut] and Masanao Yajima\n[aut]","name":"MAST","license":"GPL(>= 2)","tags":["GeneExpression","DifferentialExpression","GeneSetEnrichment","RNASeq","Transcriptomics","SingleCell"],"description":"Methods and models for handling zero-inflated single cell\nassay data.","downloads_month":1086,"downloads_total":37486,"page":"http://bioconductor.org/packages/release/bioc/html/MAST.html"},{"authors":"Luigi Marchionni <marchion@jhu.edu> and Anuj Gupta\n<agupta52@jhu.edu>","name":"matchBox","license":"Artistic-2.0","tags":["Software","Annotation","Microarray","MultipleComparison","Visualization"],"description":"The matchBox package enables comparing ranked vectors of\nfeatures, merging multiple datasets, removing redundant\nfeatures, using CAT-plots and Venn diagrams, and computing\nstatistical significance.","downloads_month":120,"downloads_total":26770,"page":"http://bioconductor.org/packages/release/bioc/html/matchBox.html"},{"authors":"Elena Grassi","name":"MatrixRider","license":"GPL-3","tags":["GeneRegulation","Genetics","MotifAnnotation"],"description":"Calculates a single number for a whole sequence that\nreflects the propensity of a DNA binding protein to interact\nwith it. The DNA binding protein has to be described with a PFM\nmatrix, for example gotten from Jaspar.","downloads_month":138,"downloads_total":12374,"page":"http://bioconductor.org/packages/release/bioc/html/MatrixRider.html"},{"authors":"Kylie A. Bemis <k.bemis@northeastern.edu>","name":"matter","license":"Artistic-2.0","tags":["Software","Infrastructure"],"description":"Memory-efficient reading, writing, and manipulation of\nstructured binary data on disk as vectors, matrices, arrays,\nlists, and data frames.","downloads_month":250,"downloads_total":9950,"page":"http://bioconductor.org/packages/release/bioc/html/matter.html"},{"authors":"Jan Sauer and Bernd Fischer","name":"MaxContrastProjection","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","Preprocessing","Software","Visualization"],"description":"A problem when recording 3D fluorescent microscopy images\nis how to properly present these results in 2D. Maximum\nintensity projections are a popular method to determine the\nfocal plane of each pixel in the image. The problem with this\napproach, however, is that out-of-focus elements will still be\nvisible, making edges and fine structures difficult to detect.\nThis package aims to resolve this problem by using the contrast\naround a given pixel to determine the focal plane, allowing for\na much cleaner structure detection than would be otherwise\npossible. For convenience, this package also contains functions\nto perform various other types of projections, including a\nmaximum intensity projection.","downloads_month":176,"downloads_total":5228,"page":"http://bioconductor.org/packages/release/bioc/html/MaxContrastProjection.html"},{"authors":"Tao Wang and Mengjie Chen","name":"MBAmethyl","license":"Artistic-2.0","tags":["DNAMethylation","MethylationArray"],"description":"This package provides a function for reconstructing DNA\nmethylation values from raw measurements. It iteratively\nimplements the group fused lars to smooth related-by-location\nmethylation values and the constrained least squares to remove\nprobe affinity effect across multiple sequences.","downloads_month":122,"downloads_total":13942,"page":"http://bioconductor.org/packages/release/bioc/html/MBAmethyl.html"},{"authors":"Oleg Mayba and Houston Gilbert","name":"MBASED","license":"Artistic-2.0","tags":["Sequencing","GeneExpression","Transcription"],"description":"The package implements MBASED algorithm for detecting\nallele-specific gene expression from RNA count data, where\nallele counts at individual loci (SNVs) are integrated into a\ngene-specific measure of ASE, and utilizes simulations to\nappropriately assess the statistical significance of observed\nASE.","downloads_month":163,"downloads_total":17918,"page":"http://bioconductor.org/packages/release/bioc/html/MBASED.html"},{"authors":"Yang Xie <Yang.Xie@UTSouthwestern.edu>","name":"MBCB","license":"GPL (>= 2)","tags":["Microarray","Preprocessing"],"description":"This package provides a model-based background correction\nmethod, which incorporates the negative control beads to\npre-process Illumina BeadArray data.","downloads_month":154,"downloads_total":35784,"page":"http://bioconductor.org/packages/release/bioc/html/MBCB.html"},{"authors":"P.M.V. Rancoita <rancoita.paola@gmail.com> and with contributions\nfrom M. Hutter <marcus.hutter@anu.edu.au>","name":"mBPCR","license":"GPL (>= 2)","tags":["aCGH","SNP","Microarray","CopyNumberVariation"],"description":"Estimates the DNA copy number profile using mBPCR to\ndetect regions with copy number changes","downloads_month":158,"downloads_total":38974,"page":"http://bioconductor.org/packages/release/bioc/html/mBPCR.html"},{"authors":"Yuan-De Tan","name":"MBttest","license":"GPL-3","tags":["Sequencing","DifferentialExpression","MultipleComparison","SAGE","GeneExpression","Transcription","AlternativeSplicing","Coverage","DifferentialSplicing"],"description":"MBttest method was developed from beta t-test method of\nBaggerly et al(2003). Compared to baySeq (Hard castle and Kelly\n2010), DESeq (Anders and Huber 2010) and exact test (Robinson\nand Smyth 2007, 2008) and the GLM of McCarthy et al(2012),\nMBttest is of high work efficiency,that is, it has high power,\nhigh conservativeness of FDR estimation and high stability.\nMBttest is suit- able to transcriptomic data, tag data, SAGE\ndata (count data) from small samples or a few replicate\nlibraries. It can be used to identify genes, mRNA isoforms or\ntags differentially expressed between two conditions.","downloads_month":135,"downloads_total":7566,"page":"http://bioconductor.org/packages/release/bioc/html/MBttest.html"},{"authors":"Wade K. Copeland, Vandhana Krishnan, Daniel Beck, Matt Settles, James Foster, Kyu-Chul Cho, Mitch Day, Roxana Hickey, Ursel\nM.E. Schutte, Xia Zhou, Chris Williams, Larry J. Forney, Zaid\nAbdo and Poor Man's GUI (PMG) base code by John Verzani with\ncontributions by Yvonnick Noel","name":"mcaGUI","license":"GPL (>= 2)","tags":["GUI","Visualization","Clustering","Sequencing"],"description":"Microbial community analysis GUI for R using gWidgets.","downloads_month":144,"downloads_total":33986,"page":"http://bioconductor.org/packages/release/bioc/html/mcaGUI.html"},{"authors":"Robert Bentham","name":"MCbiclust","license":"GPL-2","tags":["ImmunoOncology","Clustering","Microarray","StatisticalMethod","Software","RNASeq","GeneExpression"],"description":"Custom made algorithm and associated methods for finding,\nvisualising and analysing biclusters in large gene expression\ndata sets. Algorithm is based on with a supplied gene set of\nsize n, finding the maximum strength correlation matrix\ncontaining m samples from the data set.","downloads_month":188,"downloads_total":6434,"page":"http://bioconductor.org/packages/release/bioc/html/MCbiclust.html"},{"authors":"Marc Johannes, Markus Ruschhaupt, Holger Froehlich, Ulrich\nMansmann, Andreas Buness, Patrick Warnat, Wolfgang Huber, Axel\nBenner and Tim Beissbarth","name":"MCRestimate","license":"GPL (>= 2)","tags":["Classification"],"description":"This package includes a function for combining\npreprocessing and classification methods to calculate\nmisclassification errors","downloads_month":149,"downloads_total":45400,"page":"http://bioconductor.org/packages/release/bioc/html/MCRestimate.html"},{"authors":"Jordi Martorell-Marugán and Pedro Carmona-Sáez","name":"mCSEA","license":"GPL-2","tags":["ImmunoOncology","DifferentialMethylation","DNAMethylation","Epigenetics","Genetics","GenomeAnnotation","MethylationArray","Microarray","MultipleComparison","TwoChannel"],"description":"Identification of diferentially methylated regions (DMRs)\nin predefined regions (promoters, CpG islands...) from the\nhuman genome using Illumina's 450K or EPIC microarray data.\nProvides methods to rank CpG probes based on linear models and\nincludes plotting functions.","downloads_month":171,"downloads_total":2586,"page":"http://bioconductor.org/packages/release/bioc/html/mCSEA.html"},{"authors":"David Montaner <dmontaner@cipf.es>","name":"mdgsa","license":"GPL","tags":["GeneSetEnrichment","Annotation","Pathways","GO"],"description":"Functions to preform a Gene Set Analysis in several\ngenomic dimensions. Including methods for miRNAs.","downloads_month":176,"downloads_total":14362,"page":"http://bioconductor.org/packages/release/bioc/html/mdgsa.html"},{"authors":"Melissa Lever [aut], Pedro Russo [aut], Helder Nakaya [aut and cre]","name":"mdp","license":"GPL-3","tags":["BiomedicalInformatics","QualityControl","Transcriptomics","SystemsBiology","Microarray","QualityControl"],"description":"The Molecular Degree of Perturbation webtool quantifies\nthe heterogeneity of samples. It takes a data.frame of omic\ndata that contains at least two classes (control and test) and\nassigns a score to all samples based on how perturbed they are\ncompared to the controls. It is based on the Molecular Distance\nto Health (Pankla et al. 2009), and expands on this algorithm\nby adding the options to calculate the z-score using the\nmodified z-score (using median absolute deviation), change the\nz-score zeroing threshold, and look at genes that are most\nperturbed in the test versus control classes.","downloads_month":158,"downloads_total":2262,"page":"http://bioconductor.org/packages/release/bioc/html/mdp.html"},{"authors":"Justin Harrington","name":"mdqc","license":"LGPL (>= 2)","tags":["Microarray","QualityControl"],"description":"MDQC is a multivariate quality assessment method for\nmicroarrays based on quality control (QC) reports. The\nMahalanobis distance of an array's quality attributes is used\nto measure the similarity of the quality of that array against\nthe quality of the other arrays. Then, arrays with unusually\nhigh distances can be flagged as potentially low-quality.","downloads_month":129,"downloads_total":48460,"page":"http://bioconductor.org/packages/release/bioc/html/mdqc.html"},{"authors":"Jack M.. Fu [aut and cre]","name":"MDTS","license":"Artistic-2.0","tags":["StatisticalMethod","Technology","Sequencing","TargetedResequencing","Coverage","DataImport"],"description":"A package for the detection of de novo copy number\ndeletions in targeted sequencing of trios with high sensitivity\nand positive predictive value.","downloads_month":118,"downloads_total":1964,"page":"http://bioconductor.org/packages/release/bioc/html/MDTS.html"},{"authors":"Carlos Ruiz-Arenas [aut, cre] and Juan R. Gonzalez [aut]","name":"MEAL","license":"Artistic-2.0","tags":["DNAMethylation","Microarray","Software","WholeGenome"],"description":"Package to integrate methylation and expression data. It\ncan also perform methylation or expression analysis alone.\nSeveral plotting functionalities are included as well as a new\nregion analysis based on redundancy analysis. Effect of SNPs on\na region can also be estimated.","downloads_month":136,"downloads_total":11534,"page":"http://bioconductor.org/packages/release/bioc/html/MEAL.html"},{"authors":"Beiying Ding","name":"MeasurementError.cor","license":"LGPL","tags":["StatisticalMethod"],"description":"Two-stage measurement error model for correlation\nestimation with smaller bias than the usual sample correlation","downloads_month":130,"downloads_total":42472,"page":"http://bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html"},{"authors":"Lukas Chavez, Matthias Lienhard, Joern Dietrich and Isaac Lopez\nMoyado","name":"MEDIPS","license":"GPL (>=2)","tags":["DNAMethylation","CpGIsland","DifferentialExpression","Sequencing","ChIPSeq","Preprocessing","QualityControl","Visualization","Microarray","Genetics","Coverage","GenomeAnnotation","CopyNumberVariation","SequenceMatching"],"description":"MEDIPS was developed for analyzing data derived from\nmethylated DNA immunoprecipitation (MeDIP) experiments followed\nby sequencing (MeDIP-seq). However, MEDIPS provides\nfunctionalities for the analysis of any kind of quantitative\nsequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others)\nincluding calculation of differential coverage between groups\nof samples and saturation and correlation analysis.","downloads_month":321,"downloads_total":70670,"page":"http://bioconductor.org/packages/release/bioc/html/MEDIPS.html"},{"authors":"Mattia Pelizzola and Annette Molinaro","name":"MEDME","license":"GPL (>= 2)","tags":["Microarray","CpGIsland","DNAMethylation"],"description":"Description: MEDME allows the prediction of absolute and\nrelative methylation levels based on measures obtained by\nMeDIP-microarray experiments","downloads_month":136,"downloads_total":45090,"page":"http://bioconductor.org/packages/release/bioc/html/MEDME.html"},{"authors":"Jose Egea, David Henriques, Alexandre Fdez. Villaverde and Thomas\nCokelaer","name":"MEIGOR","license":"GPL-3","tags":["SystemsBiology"],"description":"Global Optimization","downloads_month":181,"downloads_total":17500,"page":"http://bioconductor.org/packages/release/bioc/html/MEIGOR.html"},{"authors":"C. A. Kapourani [aut and cre]","name":"Melissa","license":"GPL-3","tags":["ImmunoOncology","DNAMethylation","GeneExpression","GeneRegulation","Epigenetics","Genetics","Clustering","FeatureExtraction","Regression","RNASeq","Bayesian","KEGG","Sequencing","Coverage","SingleCell"],"description":"Melissa is a Baysian probabilistic model for jointly\nclustering and imputing single cell methylomes. This is done by\ntaking into account local correlations via a Generalised Linear\nModel approach and global similarities using a mixture\nmodelling approach.","downloads_month":0,"downloads_total":44,"page":"http://bioconductor.org/packages/release/bioc/html/Melissa.html"},{"authors":"Xiaogang Zhong <zhong@ams.jhu.edu> Leslie Cope <cope@jhu.edu>\nElizabeth Garrett <esg@jhu.edu> Giovanni Parmigiani\n<gp@jhu.edu>","name":"MergeMaid","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","Visualization"],"description":"The functions in this R extension are intended for\ncross-study comparison of gene expression array data. Required\nfrom the user is gene expression matrices, their corresponding\ngene-id vectors and other useful information, and they could be\n'list','matrix', or 'ExpressionSet'. The main function is\n'mergeExprs' which transforms the input objects into data in\nthe merged format, such that common genes in different datasets\ncan be easily found. And the function 'intcor' calculate the\ncorrelation coefficients. Other functions use the output from\n'modelOutcome' to graphically display the results and\ncross-validate associations of gene expression data with\nsurvival.","downloads_month":177,"downloads_total":66232,"page":"http://bioconductor.org/packages/release/bioc/html/MergeMaid.html"},{"authors":"Ville-Petteri Makinen, Le Shu, Yuqi Zhao, Zeyneb Kurt, Bin\nZhang and Xia Yang","name":"Mergeomics","license":"GPL (>= 2)","tags":["Software"],"description":"The Mergeomics pipeline serves as a flexible framework for\nintegrating multidimensional omics-disease associations,\nfunctional genomics, canonical pathways and gene-gene\ninteraction networks to generate mechanistic hypotheses. It\nincludes two main parts, 1) Marker set enrichment analysis\n(MSEA); 2) Weighted Key Driver Analysis (wKDA).","downloads_month":161,"downloads_total":8704,"page":"http://bioconductor.org/packages/release/bioc/html/Mergeomics.html"},{"authors":"Koki Tsuyuzaki","name":"MeSHDbi","license":"Artistic-2.0","tags":["Annotation","AnnotationData","Infrastructure"],"description":"The package is unified implementation of MeSH.db,\nMeSH.AOR.db, and MeSH.PCR.db and also is interface to construct\nGene-MeSH package (MeSH.XXX.eg.db). loadMeSHDbiPkg import\nsqlite file and generate MeSH.XXX.eg.db.","downloads_month":275,"downloads_total":28746,"page":"http://bioconductor.org/packages/release/bioc/html/MeSHDbi.html"},{"authors":"Guangchuang Yu [aut and cre]","name":"meshes","license":"Artistic-2.0","tags":["Annotation","Clustering","MultipleComparison","Software"],"description":"MeSH (Medical Subject Headings) is the NLM controlled\nvocabulary used to manually index articles for MEDLINE/PubMed.\nMeSH terms were associated by Entrez Gene ID by three methods,\ngendoo, gene2pubmed and RBBH. This association is fundamental\nfor enrichment and semantic analyses. meshes supports\nenrichment analysis (over-representation and gene set\nenrichment analysis) of gene list or whole expression profile.\nThe semantic comparisons of MeSH terms provide quantitative\nways to compute similarities between genes and gene groups.\nmeshes implemented five methods proposed by Resnik, Schlicker,\nJiang, Lin and Wang respectively and supports more than 70\nspecies.","downloads_month":157,"downloads_total":8058,"page":"http://bioconductor.org/packages/release/bioc/html/meshes.html"},{"authors":"Koki Tsuyuzaki, Itoshi Nikaido and Gota Morota","name":"meshr","license":"Artistic-2.0","tags":["AnnotationData","FunctionalAnnotation","Bioinformatics","Statistics","Annotation","MultipleComparisons","MeSHDb"],"description":"A set of annotation maps describing the entire MeSH\nassembled using data from MeSH.","downloads_month":221,"downloads_total":26814,"page":"http://bioconductor.org/packages/release/bioc/html/meshr.html"},{"authors":"Mark Pinese [aut], Mark Pinese [cre] and Mark Pinese [cph]","name":"messina","license":"EPL (>= 1.0)","tags":["GeneExpression","DifferentialExpression","BiomedicalInformatics","Classification","Survival"],"description":"Messina is a collection of algorithms for constructing\noptimally robust single-gene classifiers, and for identifying\ndifferential expression in the presence of outliers or unknown\nsample subgroups.  The methods have application in identifying\nlead features to develop into clinical tests (both diagnostic\nand prognostic), and in identifying differential expression\nwhen a fraction of samples show unusual patterns of expression.","downloads_month":138,"downloads_total":18118,"page":"http://bioconductor.org/packages/release/bioc/html/messina.html"},{"authors":"Debashis Ghosh <ghoshd@psu.edu> Hyungwon Choi\n<hyung_won_choi@nuhs.edu.sg>","name":"metaArray","license":"LGPL-2","tags":["Microarray","DifferentialExpression"],"description":"1) Data transformation for meta-analysis of microarray\nData: Transformation of gene expression data to signed\nprobability scale (MCMC/EM methods) 2) Combined differential\nexpression on raw scale: Weighted Z-score after stabilizing\nmean-variance relation within platform","downloads_month":169,"downloads_total":63046,"page":"http://bioconductor.org/packages/release/bioc/html/metaArray.html"},{"authors":"Raphael Aggio <ragg005@aucklanduni.ac.nz>","name":"Metab","license":"GPL (>=2)","tags":["ImmunoOncology","Metabolomics","MassSpectrometry","AMDIS","GCMS"],"description":"Metab is an R package for high-throughput processing of\nmetabolomics data analysed by the Automated Mass Spectral\nDeconvolution and Identification System (AMDIS)\n(http://chemdata.nist.gov/mass-spc/amdis/downloads/). In\naddition, it performs statistical hypothesis test (t-test) and\nanalysis of variance (ANOVA). Doing so, Metab considerably\nspeed up the data mining process in metabolomics and produces\nbetter quality results. Metab was developed using interactive\nfeatures, allowing users with lack of R knowledge to appreciate\nits functionalities.","downloads_month":183,"downloads_total":17984,"page":"http://bioconductor.org/packages/release/bioc/html/Metab.html"},{"authors":"Michael Nodzenski, Anna Reisetter and Denise Scholtens","name":"metabomxtr","license":"GPL-2","tags":["ImmunoOncology","Metabolomics","MassSpectrometry"],"description":"The functions in this package return optimized parameter\nestimates and log likelihoods for mixture models of truncated\ndata with normal or lognormal distributions.","downloads_month":203,"downloads_total":16438,"page":"http://bioconductor.org/packages/release/bioc/html/metabomxtr.html"},{"authors":"Andrea Rodriguez-Martinez, Rafael Ayala, Joram M. Posma, Ana L.\nNeves, Maryam Anwar, Jeremy K. Nicholson and Marc-Emmanuel Dumas","name":"MetaboSignal","license":"GPL-3","tags":["GraphAndNetwork","GeneSignaling","GeneTarget","Network","Pathways","KEGG","Reactome","Software"],"description":"MetaboSignal is an R package that allows merging,\nanalyzing and customizing metabolic and signaling KEGG\npathways. It is a network-based approach designed to explore\nthe topological relationship between genes (signaling- or\nenzymatic-genes) and metabolites, representing a powerful tool\nto investigate the genetic landscape and regulatory networks of\nmetabolic phenotypes.","downloads_month":159,"downloads_total":8150,"page":"http://bioconductor.org/packages/release/bioc/html/MetaboSignal.html"},{"authors":"Anna Cichonska <anna.cichonska@helsinki.fi>","name":"metaCCA","license":"MIT + file LICENSE","tags":["GenomeWideAssociation","SNP","Genetics","Regression","StatisticalMethod","Software"],"description":"metaCCA performs multivariate analysis of a single or\nmultiple GWAS based on univariate regression coefficients. It\nallows multivariate representation of both phenotype and\ngenotype. metaCCA extends the statistical technique of\ncanonical correlation analysis to the setting where original\nindividual-level records are not available, and employs a\ncovariance shrinkage algorithm to achieve robustness.","downloads_month":136,"downloads_total":8486,"page":"http://bioconductor.org/packages/release/bioc/html/metaCCA.html"},{"authors":"Zicheng Hu, Chethan Jujjavarapu, Sanchita Bhattacharya and Atul J.\nButte","name":"MetaCyto","license":"GPL (>= 2)","tags":["ImmunoOncology","CellBiology","FlowCytometry","Clustering","StatisticalMethod","Software","CellBasedAssays","Preprocessing"],"description":"This package provides functions for preprocessing,\nautomated gating and meta-analysis of cytometry data. It also\nprovides functions that facilitate the collection of cytometry\ndata from the ImmPort database.","downloads_month":206,"downloads_total":4044,"page":"http://bioconductor.org/packages/release/bioc/html/MetaCyto.html"},{"authors":"Charles Joly Beauparlant\n<charles.joly-beauparlant@crchul.ulaval.ca>, Fabien Claude\nLamaze <fabien.lamaze.1@ulaval.ca>, Rawane Samb\n<rawane.samb.1@ulaval.ca>, Cedric Lippens\n<lippens.cedric@protonmail>, Astrid Louise Deschenes\n<Astrid-Louise.Deschenes@crchudequebec.ulaval.ca> and Arnaud\nDroit <arnaud.droit@crchuq.ulaval.ca>.","name":"metagene","license":"Artistic-2.0 | file LICENSE","tags":["ChIPSeq","Genetics","MultipleComparison","Coverage","Alignment","Sequencing"],"description":"This package produces metagene plots to compare the\nbehavior of DNA-interacting proteins at selected groups of\ngenes/features. Bam files are used to increase the resolution.\nMultiple combination of group of bam files and/or group of\ngenomic regions can be compared in a single analysis.\nBootstraping analysis is used to compare the groups and locate\nregions with statistically different enrichment profiles.","downloads_month":153,"downloads_total":19998,"page":"http://bioconductor.org/packages/release/bioc/html/metagene.html"},{"authors":"Nathan D. Olson, Joseph Nathaniel Paulson and Hector Corrada Bravo","name":"metagenomeFeatures","license":"Artistic-2.0","tags":["ImmunoOncology","Microbiome","Metagenomics","Annotation","Infrastructure","Sequencing","Software"],"description":"metagenomeFeatures was developed for use in exploring the\ntaxonomic annotations for a marker-gene metagenomic sequence\ndataset. The package can be used to explore the taxonomic\ncomposition of a marker-gene database or annotated sequences\nfrom a marker-gene metagenome experiment.","downloads_month":216,"downloads_total":11454,"page":"http://bioconductor.org/packages/release/bioc/html/metagenomeFeatures.html"},{"authors":"Joseph Nathaniel Paulson, Nathan D. Olson, Justin Wagner, Hisham Talukder, Mihai Pop and Hector Corrada Bravo","name":"metagenomeSeq","license":"Artistic-2.0","tags":["ImmunoOncology","Classification","Clustering","GeneticVariability","DifferentialExpression","Microbiome","Metagenomics","Normalization","Visualization","MultipleComparison","Sequencing","Software"],"description":"metagenomeSeq is designed to determine features (be it\nOperational Taxanomic Unit (OTU), species, etc.) that are\ndifferentially abundant between two or more groups of multiple\nsamples. metagenomeSeq is designed to address the effects of\nboth normalization and under-sampling of microbial communities\non disease association detection and the testing of feature\ncorrelations.","downloads_month":1115,"downloads_total":100060,"page":"http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html"},{"authors":"John R. Stevens and Gabriel Nicholas","name":"metahdep","license":"GPL-3","tags":["Microarray","DifferentialExpression"],"description":"Tools for meta-analysis in the presence of hierarchical\n(and/or sampling) dependence, including with gene expression\nstudies","downloads_month":129,"downloads_total":40374,"page":"http://bioconductor.org/packages/release/bioc/html/metahdep.html"},{"authors":"Ron Wehrens [aut, cre] (author of GC-MS part), Pietro\nFranceschi [aut] (author of LC-MS part), Nir Shahaf [ctb], Matthias Scholz [ctb], Georg Weingart [ctb] (development of\nGC-MS approach), Elisabete Carvalho [ctb] (testing and feedback\nof GC-MS pipeline)","name":"metaMS","license":"GPL (>= 2)","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"MS-based metabolomics data processing and compound\nannotation pipeline.","downloads_month":247,"downloads_total":23506,"page":"http://bioconductor.org/packages/release/bioc/html/metaMS.html"},{"authors":"Megan Crow [aut, cre], Sara Ballouz [ctb], Manthan Shah [ctb] and Jesse Gillis [aut]","name":"MetaNeighbor","license":"MIT + file LICENSE","tags":["ImmunoOncology","GeneExpression","GO","MultipleComparison","SingleCell","Transcriptomics"],"description":"MetaNeighbor allows users to quantify cell type\nreplicability across datasets using neighbor voting.","downloads_month":195,"downloads_total":2690,"page":"http://bioconductor.org/packages/release/bioc/html/MetaNeighbor.html"},{"authors":"Koki Tsuyuzaki and Itoshi Nikaido","name":"metaSeq","license":"Artistic-2.0","tags":["RNASeq","DifferentialExpression","Sequencing","ImmunoOncology"],"description":"The probabilities by one-sided NOISeq are combined by\nFisher's method or Stouffer's method","downloads_month":141,"downloads_total":23278,"page":"http://bioconductor.org/packages/release/bioc/html/metaSeq.html"},{"authors":"Panagiotis Moulos <moulos@fleming.gr>","name":"metaseqR","license":"GPL (>= 3)","tags":["ImmunoOncology","Software","GeneExpression","DifferentialExpression","WorkflowStep","Preprocessing","QualityControl","Normalization","ReportWriting","RNASeq"],"description":"Provides an interface to several normalization and\nstatistical testing packages for RNA-Seq gene expression data.\nAdditionally, it creates several diagnostic plots, performs\nmeta-analysis by combinining the results of several statistical\ntests and reports the results in an interactive way.","downloads_month":219,"downloads_total":23548,"page":"http://bioconductor.org/packages/release/bioc/html/metaseqR.html"},{"authors":"Hector Corrada Bravo [cre, aut], Florin Chelaru [aut], Justin\nWagner [aut], Jayaram Kancherla [aut] and Joseph Paulson [aut]","name":"metavizr","license":"MIT + file LICENSE","tags":["Visualization","Infrastructure","GUI","Metagenomics","ImmunoOncology"],"description":"This package provides Websocket communication to the\nmetaviz web app (http://metaviz.cbcb.umd.edu) for interactive\nvisualization of metagenomics data. Objects in R/bioc\ninteractive sessions can be displayed in plots and data can be\nexplored using a facetzoom visualization. Fundamental\nBioconductor data structures are supported (e.g., MRexperiment\nobjects), while providing an easy mechanism to support other\ndata structures. Visualizations (using d3.js) can be easily\nadded to the web app as well.","downloads_month":171,"downloads_total":6144,"page":"http://bioconductor.org/packages/release/bioc/html/metavizr.html"},{"authors":"Thomas Naake <thomasnaake@googlemail.com> and Emmanuel Gaquerel\n<emmanuel.gaquerel@cos.uni-heidelberg.de>","name":"MetCirc","license":"GPL-2","tags":["ImmunoOncology","Metabolomics","MassSpectrometry","Visualization"],"description":"MetCirc comprises a workflow to interactively explore\nhigh-resolution MS/MS metabolomics data: create an MSP object,\na format for MS/MS library data, bin m/z values of precursors,\ncalculate similarity between precursors based on the normalised\ndot product and visualise similarities in a circular layout.\nWithin the interactive framework the user can annotate MS/MS\nfeatures based on their similarity to (known) related MS/MS\nfeatures.","downloads_month":155,"downloads_total":7414,"page":"http://bioconductor.org/packages/release/bioc/html/MetCirc.html"},{"authors":"Aaron Taudt","name":"methimpute","license":"Artistic-2.0","tags":["ImmunoOncology","Software","DNAMethylation","Epigenetics","HiddenMarkovModel","Sequencing","Coverage"],"description":"This package implements functions for calling methylation\nfor all cytosines in the genome.","downloads_month":161,"downloads_total":3600,"page":"http://bioconductor.org/packages/release/bioc/html/methimpute.html"},{"authors":"Pascal Belleau, Astrid Deschênes and Arnaud Droit","name":"methInheritSim","license":"Artistic-2.0","tags":["BiologicalQuestion","Epigenetics","DNAMethylation","DifferentialMethylation","MethylSeq","Software","ImmunoOncology","StatisticalMethod","WholeGenome","Sequencing"],"description":"Simulate a multigeneration methylation case versus control\nexperiment with inheritance relation using a real control\ndataset.","downloads_month":153,"downloads_total":3814,"page":"http://bioconductor.org/packages/release/bioc/html/methInheritSim.html"},{"authors":"Mohammad Tanvir Ahamed [aut, trl], Anna Danielsson [aut], Szilárd Nemes [aut, trl], Helena Carén [aut, cre and cph]","name":"MethPed","license":"GPL-2","tags":["ImmunoOncology","DNAMethylation","Classification","Epigenetics"],"description":"Classification of pediatric tumors into biologically\ndefined subtypes is challenging and multifaceted approaches are\nneeded. For this aim, we developed a diagnostic classifier\nbased on DNA methylation profiles. We offer MethPed as an\neasy-to-use toolbox that allows researchers and clinical\ndiagnosticians to test single samples as well as large cohorts\nfor subclass prediction of pediatric brain tumors.  The current\nversion of MethPed can classify the following tumor\ndiagnoses/subgroups: Diffuse Intrinsic Pontine Glioma (DIPG),\nEpendymoma, Embryonal tumors with multilayered rosettes (ETMR),\nGlioblastoma (GBM), Medulloblastoma (MB) - Group 3 (MB_Gr3),\nGroup 4 (MB_Gr3), Group WNT (MB_WNT), Group SHH (MB_SHH) and\nPilocytic Astrocytoma (PiloAstro).","downloads_month":176,"downloads_total":8248,"page":"http://bioconductor.org/packages/release/bioc/html/MethPed.html"},{"authors":"Muhammad Ahmer Jamil with Contribution of Prof. Holger Frohlich\nand Priv.-Doz. Dr. Osman El-Maarri","name":"MethTargetedNGS","license":"Artistic-2.0","tags":["ResearchField","Genetics","Sequencing","Alignment","SequenceMatching","DataImport"],"description":"Perform step by step methylation analysis of Next\nGeneration Sequencing data.","downloads_month":136,"downloads_total":12082,"page":"http://bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html"},{"authors":"A. Zackay and C. Steinhoff","name":"methVisual","license":"GPL (>= 2)","tags":["DNAMethylation","Clustering","Classification"],"description":"The package 'methVisual' allows the visualization of DNA\nmethylation data after bisulfite sequencing.","downloads_month":152,"downloads_total":39026,"page":"http://bioconductor.org/packages/release/bioc/html/methVisual.html"},{"authors":"Pan Du and Richard Bourgon","name":"methyAnalysis","license":"Artistic-2.0","tags":["Microarray","DNAMethylation","Visualization"],"description":"The methyAnalysis package aims for the DNA methylation\ndata analysis and visualization. A MethyGenoSet class is\ndefined to keep the chromosome location information together\nwith the data. The package also includes functions of\nestimating the methylation levels from Methy-Seq data.","downloads_month":239,"downloads_total":50274,"page":"http://bioconductor.org/packages/release/bioc/html/methyAnalysis.html"},{"authors":"Maarten van Iterson [aut, cre], Elmar Tobi[ctb], Roderick\nSlieker[ctb], Wouter den Hollander[ctb] and Rene Luijk[ctb] and\nBas Heijmans[ctb]","name":"MethylAid","license":"GPL (>= 2)","tags":["DNAMethylation","MethylationArray","Microarray","TwoChannel","QualityControl","BatchEffect","Visualization","GUI"],"description":"A visual and interactive web application using RStudio's\nshiny package. Bad quality samples are detected using\nsample-dependent and sample-independent controls present on the\narray and user adjustable thresholds. In depth exploration of\nbad quality samples can be performed using several interactive\ndiagnostic plots of the quality control probes present on the\narray. Furthermore, the impact of any batch effect provided by\nthe user can be explored.","downloads_month":248,"downloads_total":19698,"page":"http://bioconductor.org/packages/release/bioc/html/MethylAid.html"},{"authors":"Xu Ren [aut, cre] and Pei Fen Kuan [aut]","name":"methylGSA","license":"GPL-2","tags":["DNAMethylation","DifferentialMethylation","GeneSetEnrichment","Regression","GeneRegulation","Pathways"],"description":"The main functions for methylGSA are methylglm and\nmethylRRA. methylGSA implements logistic regression adjusting\nnumber of probes as a covariate. methylRRA adjusts multiple\np-values of each gene by Robust Rank Aggregation. For more\ndetailed help information, please see the vignette.","downloads_month":102,"downloads_total":1046,"page":"http://bioconductor.org/packages/release/bioc/html/methylGSA.html"},{"authors":"Astrid Deschênes, Pascal Belleau and Arnaud Droit","name":"methylInheritance","license":"Artistic-2.0","tags":["BiologicalQuestion","Epigenetics","DNAMethylation","DifferentialMethylation","MethylSeq","Software","ImmunoOncology","StatisticalMethod","WholeGenome","Sequencing"],"description":"Permutation analysis, based on Monte Carlo sampling, for\ntesting the hypothesis that the number of conserved\ndifferentially methylated elements, between several\ngenerations, is associated to an effect inherited from a\ntreatment and that stochastic effect can be dismissed.","downloads_month":145,"downloads_total":5258,"page":"http://bioconductor.org/packages/release/bioc/html/methylInheritance.html"},{"authors":"Altuna Akalin [aut, cre], Matthias Kormaksson [aut], Sheng Li\n[aut], Arsene Wabo [ctb], Adrian Bierling [aut] and Alexander\nGosdschan [aut]","name":"methylKit","license":"Artistic-2.0","tags":["DNAMethylation","Sequencing","MethylSeq"],"description":"methylKit is an R package for DNA methylation analysis and\nannotation from high-throughput bisulfite sequencing. The\npackage is designed to deal with sequencing data from RRBS and\nits variants, but also target-capture methods and whole genome\nbisulfite sequencing. It also has functions to analyze\nbase-pair resolution 5hmC data from experimental protocols such\nas oxBS-Seq and TAB-Seq. Methylation calling can be performed\ndirectly from Bismark aligned BAM files.","downloads_month":1015,"downloads_total":45568,"page":"http://bioconductor.org/packages/release/bioc/html/methylKit.html"},{"authors":"Olivier Gevaert","name":"MethylMix","license":"GPL-2","tags":["DNAMethylation","StatisticalMethod","DifferentialMethylation","GeneRegulation","GeneExpression","MethylationArray","DifferentialExpression","Pathways","Network"],"description":"MethylMix is an algorithm implemented to identify hyper\nand hypomethylated genes for a disease. MethylMix is based on a\nbeta mixture model to identify methylation states and compares\nthem with the normal DNA methylation state. MethylMix uses a\nnovel statistic, the Differential Methylation value or DM-value\ndefined as the difference of a methylation state with the\nnormal methylation state. Finally, matched gene expression data\nis used to identify, besides differential, functional\nmethylation states by focusing on methylation changes that\neffect gene expression. References: Gevaert 0. MethylMix: an R\npackage for identifying DNA methylation-driven genes.\nBioinformatics (Oxford, England). 2015;31(11):1839-41.\ndoi:10.1093/bioinformatics/btv020. Gevaert O, Tibshirani R,\nPlevritis SK. Pancancer analysis of DNA methylation-driven\ngenes using MethylMix. Genome Biology. 2015;16(1):17.\ndoi:10.1186/s13059-014-0579-8.","downloads_month":225,"downloads_total":19836,"page":"http://bioconductor.org/packages/release/bioc/html/MethylMix.html"},{"authors":"Yan Zhou, Bo Zhang, Nan Lin, BaoXue Zhang and Ting Wang","name":"methylMnM","license":"GPL-3","tags":["Software","DNAMethylation","Sequencing"],"description":"To give the exactly p-value and q-value of MeDIP-seq and\nMRE-seq data for different samples comparation.","downloads_month":138,"downloads_total":22018,"page":"http://bioconductor.org/packages/release/bioc/html/methylMnM.html"},{"authors":"Kamal Kishore","name":"methylPipe","license":"GPL(>=2)","tags":["MethylSeq","DNAMethylation","Coverage","Sequencing"],"description":"Memory efficient analysis of base resolution DNA\nmethylation data in both the CpG and non-CpG sequence context.\nIntegration of DNA methylation data derived from any\nmethodology providing base- or low-resolution data.","downloads_month":188,"downloads_total":23688,"page":"http://bioconductor.org/packages/release/bioc/html/methylPipe.html"},{"authors":"Lukas Burger, Dimos Gaidatzis, Dirk Schubeler and Michael\nStadler","name":"MethylSeekR","license":"GPL (>=2)","tags":["Sequencing","MethylSeq","DNAMethylation"],"description":"This is a package for the discovery of regulatory regions\nfrom Bis-seq data","downloads_month":171,"downloads_total":30658,"page":"http://bioconductor.org/packages/release/bioc/html/MethylSeekR.html"},{"authors":"Sean Davis, Pan Du, Sven Bilke, Tim Triche, Jr. and Moiz Bootwalla","name":"methylumi","license":"GPL-2","tags":["DNAMethylation","TwoChannel","Preprocessing","QualityControl","CpGIsland"],"description":"This package provides classes for holding and manipulating\nIllumina methylation data.  Based on eSet, it can contain MIAME\ninformation, sample information, feature information, and\nmultiple matrices of data.  An \"intelligent\" import function,\nmethylumiR can read the Illumina text files and create a\nMethyLumiSet. methylumIDAT can directly read raw IDAT files\nfrom HumanMethylation27 and HumanMethylation450 microarrays.\nNormalization, background correction, and quality control\nfeatures for GoldenGate, Infinium, and Infinium HD arrays are\nalso included.","downloads_month":2009,"downloads_total":319294,"page":"http://bioconductor.org/packages/release/bioc/html/methylumi.html"},{"authors":"Nima Hejazi [aut, cre, cph], Mark van der Laan [aut, ths], Alan\nHubbard [ctb, ths] and Rachael Phillips [ctb]","name":"methyvim","license":"file LICENSE","tags":["Clustering","DNAMethylation","DifferentialMethylation","MethylationArray","MethylSeq"],"description":"This package provides facilities for differential\nmethylation analysis based on variable importance measures\n(VIMs), a class of statistical target parameters that arise in\ncausal inference. The estimation and inference procedures\nprovided are nonparametric, relying on ensemble machine\nlearning to flexibly assess functional relationships among\ncovariates and the outcome of interest. These tools can be\napplied to differential methylation at the level of CpG sites,\nto obtain valid statistical inference even after corrections\nfor multiple hypothesis testing.","downloads_month":139,"downloads_total":3524,"page":"http://bioconductor.org/packages/release/bioc/html/methyvim.html"},{"authors":"Zhenzhi Li <zzrickli@gmail.com>","name":"MetID","license":"Artistic-2.0","tags":["AssayDomain","BiologicalQuestion","Infrastructure","ResearchField","StatisticalMethod","Technology","WorkflowStep","Network","KEGG"],"description":"This package uses an innovative network-based approach\nthat will enhance our ability to determine the identities of\nsignificant ions detected by LC-MS.","downloads_month":69,"downloads_total":572,"page":"http://bioconductor.org/packages/release/bioc/html/MetID.html"},{"authors":"Thomas Naake [aut and cre]","name":"MetNet","license":"GPL-2","tags":["ImmunoOncology","Metabolomics","MassSpectrometry","Network","Regression"],"description":"MetNet contains functionality to infer metabolic network\ntopologies from quantitative data and high-resolution\nmass/charge information. Using statistical models (including\ncorrelation, mutual information, regression and Bayes\nstatistics) and quantitative data (intensity values of\nfeatures) adjacency matrices are inferred that can be combined\nto a consensus matrix. Mass differences calculated between\nmass/charge values of features will be matched against a data\nframe of supplied mass/charge differences referring to\ntransformations of enzymatic activities. In a third step, the\ntwo matrices are combined to form a adjacency matrix inferred\nfrom both quantitative and structure information.","downloads_month":134,"downloads_total":1090,"page":"http://bioconductor.org/packages/release/bioc/html/MetNet.html"},{"authors":"Kieran Campbell [aut and cre]","name":"mfa","license":"GPL (>= 2)","tags":["ImmunoOncology","RNASeq","GeneExpression","Bayesian","SingleCell"],"description":"MFA models genomic bifurcations using a Bayesian\nhierarchical mixture of factor analysers.","downloads_month":198,"downloads_total":4502,"page":"http://bioconductor.org/packages/release/bioc/html/mfa.html"},{"authors":"Matthias Futschik <matthias.futschik@sysbiolab.eu>","name":"Mfuzz","license":"GPL-2","tags":["Microarray","Clustering","TimeCourse","Preprocessing","Visualization"],"description":"Package for noise-robust soft clustering of gene\nexpression time-series data (including a graphical user\ninterface)","downloads_month":493,"downloads_total":129156,"page":"http://bioconductor.org/packages/release/bioc/html/Mfuzz.html"},{"authors":"Khadija El Amrani","name":"MGFM","license":"GPL-3","tags":["Genetics","GeneExpression","Microarray"],"description":"The package is designed to detect marker genes from\nMicroarray gene expression data sets","downloads_month":159,"downloads_total":15522,"page":"http://bioconductor.org/packages/release/bioc/html/MGFM.html"},{"authors":"Khadija El Amrani","name":"MGFR","license":"GPL-3","tags":["ImmunoOncology","Genetics","GeneExpression","RNASeq"],"description":"The package is designed to detect marker genes from\nRNA-seq data.","downloads_month":195,"downloads_total":7156,"page":"http://bioconductor.org/packages/release/bioc/html/MGFR.html"},{"authors":"Sebastian Bauer <mail@sebastianbauer.info> and Julien Gagneur\n<gagneur@genzentrum.lmu.de>","name":"mgsa","license":"Artistic-2.0","tags":["Pathways","GO","GeneSetEnrichment"],"description":"Model-based Gene Set Analysis (MGSA) is a Bayesian\nmodeling approach for gene set enrichment. The package mgsa\nimplements MGSA and tools to use MGSA together with the Gene\nOntology.","downloads_month":155,"downloads_total":38136,"page":"http://bioconductor.org/packages/release/bioc/html/mgsa.html"},{"authors":"Jonathon Blake <blake@embl.de>","name":"MiChip","license":"GPL (>= 2)","tags":["Microarray","Preprocessing"],"description":"This package takes the MiChip miRNA microarray .grp\nscanner output files and parses these out, providing summary\nand plotting functions to analyse MiChip hybridizations. A set\nof hybridizations is packaged into an ExpressionSet allowing it\nto be used by other BioConductor packages.","downloads_month":132,"downloads_total":38230,"page":"http://bioconductor.org/packages/release/bioc/html/MiChip.html"},{"authors":"Leo Lahti [aut, cre] and Sudarshan Shetty [aut]","name":"microbiome","license":"BSD_2_clause + file LICENSE","tags":["Clustering","Metagenomics","Microbiome","Sequencing","SystemsBiology","ImmunoOncology"],"description":"Utilities for microbiome analysis.","downloads_month":742,"downloads_total":19078,"page":"http://bioconductor.org/packages/release/bioc/html/microbiome.html"},{"authors":"R. Gentleman and S. Falcon","name":"microRNA","license":"Artistic-2.0","tags":["Infrastructure","GenomeAnnotation","SequenceMatching"],"description":"Different data resources for microRNAs and some functions\nfor manipulating them.","downloads_month":237,"downloads_total":83802,"page":"http://bioconductor.org/packages/release/bioc/html/microRNA.html"},{"authors":"Juan C. Rodriguez, Cristobal Fresno, Andrea S. Llera and Elmer\nA. Fernandez","name":"MIGSA","license":"GPL (>= 2)","tags":["Software","GeneSetEnrichment","Visualization","GeneExpression","Microarray","RNASeq","KEGG"],"description":"Massive and Integrative Gene Set Analysis. The MIGSA\npackage allows to perform a massive and integrative gene set\nanalysis over several expression and gene sets simultaneously.\nIt provides a common gene expression analytic framework that\ngrants a comprehensive and coherent analysis. Only a minimal\nuser parameter setting is required to perform both singular and\ngene set enrichment analyses in an integrative manner by means\nof the best available methods, i.e. dEnricher and mGSZ\nrespectively. The greatest strengths of this big omics data\ntool are the availability of several functions to explore,\nanalyze and visualize its results in order to facilitate the\ndata mining task over huge information sources. MIGSA package\nalso provides several functions that allow to easily load the\nmost updated gene sets from several repositories.","downloads_month":157,"downloads_total":5952,"page":"http://bioconductor.org/packages/release/bioc/html/MIGSA.html"},{"authors":"Aaron Wolen [aut, cre and cph]","name":"mimager","license":"MIT + file LICENSE","tags":["Infrastructure","Visualization","Microarray"],"description":"Easily visualize and inspect microarrays for spatial\nartifacts.","downloads_month":149,"downloads_total":4892,"page":"http://bioconductor.org/packages/release/bioc/html/mimager.html"},{"authors":"Greg Finak <gfinak@fhcrc.org>","name":"MIMOSA","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","CellBasedAssays"],"description":"Modeling count data using Dirichlet-multinomial and\nbeta-binomial mixtures with applications to single-cell assays.","downloads_month":202,"downloads_total":20544,"page":"http://bioconductor.org/packages/release/bioc/html/MIMOSA.html"},{"authors":"Anne Biton","name":"MineICA","license":"GPL-2","tags":["Visualization","MultipleComparison"],"description":"The goal of MineICA is to perform Independent Component\nAnalysis (ICA) on multiple transcriptome datasets, integrating\nadditional data (e.g molecular, clinical and pathological).\nThis Integrative ICA helps the biological interpretation of the\ncomponents by studying their association with variables (e.g\nsample annotations) and gene sets, and enables the comparison\nof components from different datasets using correlation-based\ngraph.","downloads_month":159,"downloads_total":26912,"page":"http://bioconductor.org/packages/release/bioc/html/MineICA.html"},{"authors":"Patrick E. Meyer, Frederic Lafitte and Gianluca Bontempi","name":"minet","license":"file LICENSE","tags":["Microarray","GraphAndNetwork","Network","NetworkInference"],"description":"This package implements various algorithms for inferring\nmutual information networks from data.","downloads_month":1190,"downloads_total":139872,"page":"http://bioconductor.org/packages/release/bioc/html/minet.html"},{"authors":"Kasper Daniel Hansen [cre, aut], Martin Aryee [aut], Rafael A.\nIrizarry [aut], Andrew E. Jaffe [ctb], Jovana Maksimovic [ctb], E. Andres Houseman [ctb], Jean-Philippe Fortin [ctb], Tim\nTriche [ctb], Shan V. Andrews [ctb] and Peter F. Hickey [ctb]","name":"minfi","license":"Artistic-2.0","tags":["ImmunoOncology","DNAMethylation","DifferentialMethylation","Epigenetics","Microarray","MethylationArray","MultiChannel","TwoChannel","DataImport","Normalization","Preprocessing","QualityControl"],"description":"Tools to analyze & visualize Illumina Infinium methylation\narrays.","downloads_month":4589,"downloads_total":1108278,"page":"http://bioconductor.org/packages/release/bioc/html/minfi.html"},{"authors":"Robert B Scharpf and Ingo Ruczinski","name":"MinimumDistance","license":"Artistic-2.0","tags":["Microarray","SNP","CopyNumberVariation"],"description":"Analysis of de novo copy number variants in trios from\nhigh-dimensional genotyping platforms.","downloads_month":143,"downloads_total":31630,"page":"http://bioconductor.org/packages/release/bioc/html/MinimumDistance.html"},{"authors":"HyungJun Cho <hj4cho@korea.ac.kr>, Sukwoo Kim\n<s4kim@korea.ac.kr>, Mat Soukup <soukup@fda.gov>, and Jae K.\nLee <jaeklee@virginia.edu>","name":"MiPP","license":"GPL (>= 2)","tags":["Microarray","Classification"],"description":"This package finds optimal sets of genes that seperate\nsamples into two or more classes.","downloads_month":132,"downloads_total":48342,"page":"http://bioconductor.org/packages/release/bioc/html/MiPP.html"},{"authors":"Nathan Sheffield <http://www.databio.org> [aut], Christoph Bock\n[ctb], John Lawson [aut and cre]","name":"MIRA","license":"GPL-3","tags":["ImmunoOncology","DNAMethylation","GeneRegulation","GenomeAnnotation","SystemsBiology","FunctionalGenomics","ChIPSeq","MethylSeq","Sequencing","Epigenetics","Coverage"],"description":"MIRA measures the degree of \"dip\" in methylation level\nsurrounding a regulatory site of interest, such as a\ntranscription factor binding site, for instances of that type\nof site across the genome which can then be used to infer\nregulatory activity.","downloads_month":202,"downloads_total":4226,"page":"http://bioconductor.org/packages/release/bioc/html/MIRA.html"},{"authors":"Y-h. Taguchi <tag@granular.com>","name":"MiRaGE","license":"GPL","tags":["ImmunoOncology","Microarray","GeneExpression","RNASeq","Sequencing","SAGE"],"description":"The package contains functions for inferece of target gene\nregulation by miRNA, based on only target gene expression\nprofile.","downloads_month":189,"downloads_total":29326,"page":"http://bioconductor.org/packages/release/bioc/html/MiRaGE.html"},{"authors":"Taosheng Xu<taosheng.x@gmail.com> and Thuc\nLe<Thuc.Le@unisa.edu.au>","name":"miRBaseConverter","license":"GPL (>= 2)","tags":["Software","miRNA"],"description":"A comprehensive tool for converting and retrieving the\nmiRNA Name, Accession, Sequence, Version, History and Family\ninformation in different miRBase versions. It can process a\nhuge number of miRNAs in a short time without other depends.","downloads_month":173,"downloads_total":3948,"page":"http://bioconductor.org/packages/release/bioc/html/miRBaseConverter.html"},{"authors":"Matthew N. McCall <mccallm@gmail.com> and Lauren Kemperman\n<lkemperm@u.rochester.edu>","name":"miRcomp","license":"GPL-3 | file LICENSE","tags":["Software","qPCR","Preprocessing","QualityControl"],"description":"Based on a large miRNA dilution study, this package\nprovides tools to read in the raw amplification data and use\nthese data to assess the performance of methods that estimate\nexpression from the amplification curves.","downloads_month":140,"downloads_total":10320,"page":"http://bioconductor.org/packages/release/bioc/html/miRcomp.html"},{"authors":"Diana Diaz <dmd at wayne dot edu>","name":"mirIntegrator","license":"GPL (>=3)","tags":["Network","Microarray","GraphAndNetwork","Pathways","KEGG"],"description":"Tools for augmenting signaling pathways to perform pathway\nanalysis of microRNA and mRNA expression levels.","downloads_month":151,"downloads_total":10956,"page":"http://bioconductor.org/packages/release/bioc/html/mirIntegrator.html"},{"authors":"Thuc Duy Le, Junpeng Zhang, Mo Chen and Vu Viet Hoang Pham","name":"miRLAB","license":"GPL (>=2)","tags":["miRNA","GeneExpression","NetworkInference","Network"],"description":"Provide tools exploring miRNA-mRNA relationships,\nincluding popular miRNA target prediction methods, ensemble\nmethods that integrate individual methods, functions to get\ndata from online resources, functions to validate the results,\nand functions to conduct enrichment analyses.","downloads_month":141,"downloads_total":12334,"page":"http://bioconductor.org/packages/release/bioc/html/miRLAB.html"},{"authors":"Dusan Randjelovic [aut and cre]","name":"miRmine","license":"GPL (>= 3)","tags":["Homo_sapiens_Data","RNASeqData","SequencingData","ExpressionData"],"description":"miRmine database is a collection of expression profiles\nfrom different publicly available miRNA-seq datasets, Panwar et\nal (2017) miRmine: A Database of Human miRNA Expression,\nprepared with this data package as RangedSummarizedExperiment.","downloads_month":133,"downloads_total":3194,"page":"http://bioconductor.org/packages/release/bioc/html/miRmine.html"},{"authors":"Stefan Haunsberger [aut and cre]","name":"miRNAmeConverter","license":"Artistic-2.0","tags":["Preprocessing","miRNA"],"description":"Translating mature miRNA names to different miRBase\nversions, sequence retrieval, checking names for validity and\ndetecting miRBase version of a given set of names (data from\nhttp://www.mirbase.org/).","downloads_month":131,"downloads_total":8722,"page":"http://bioconductor.org/packages/release/bioc/html/miRNAmeConverter.html"},{"authors":"James M. Ward <jmw86069@gmail.com> with contributions from\nYunling Shi, Cindy Richards and John P. Cogswell","name":"miRNApath","license":"LGPL-2.1","tags":["Annotation","Pathways","DifferentialExpression","NetworkEnrichment","miRNA"],"description":"This package provides pathway enrichment techniques for\nmiRNA expression data. Specifically, the set of methods handles\nthe many-to-many relationship between miRNAs and the multiple\ngenes they are predicted to target (and thus affect.)  It also\nhandles the gene-to-pathway relationships separately. Both\nsteps are designed to preserve the additive effects of miRNAs\non genes, many miRNAs affecting one gene, one miRNA affecting\nmultiple genes, or many miRNAs affecting many genes.","downloads_month":154,"downloads_total":51690,"page":"http://bioconductor.org/packages/release/bioc/html/miRNApath.html"},{"authors":"Maciej Pajak and T. Ian Simpson","name":"miRNAtap","license":"GPL-2","tags":["Software","Classification","Microarray","Sequencing","miRNA"],"description":"The package facilitates implementation of workflows\nrequiring miRNA predictions, it allows to integrate ranked\nmiRNA target predictions from multiple sources available online\nand aggregate them with various methods which improves quality\nof predictions above any of the single sources. Currently\npredictions are available for Homo sapiens, Mus musculus and\nRattus norvegicus (the last one through homology translation).","downloads_month":233,"downloads_total":21968,"page":"http://bioconductor.org/packages/release/bioc/html/miRNAtap.html"},{"authors":"Junpeng Zhang [aut and cre]","name":"miRSM","license":"GPL-3","tags":["GeneExpression","BiomedicalInformatics","Clustering","GeneSetEnrichment","Microarray","Software","GeneRegulation","GeneTarget"],"description":"The package aims to identify miRNA sponge modules by\nintegrating expression data and miRNA-target binding\ninformation. It provides several functions to study miRNA\nsponge modules, including popular methods for inferring gene\nmodules (candidate miRNA sponge modules), and a function to\nidentify miRNA sponge modules, as well as a function to conduct\nfunctional analysis of miRNA sponge modules.","downloads_month":80,"downloads_total":674,"page":"http://bioconductor.org/packages/release/bioc/html/miRSM.html"},{"authors":"Junpeng Zhang","name":"miRsponge","license":"GPL-3","tags":["GeneExpression","BiomedicalInformatics","NetworkEnrichment","Survival","Microarray","Software"],"description":"The name of the miRsponge package has been changed into\n\"miRspongeR\". Future updates will be seen in the miRspongeR\npackage (http://bioconductor.org/packages/miRspongeR/). This\npackage provides several functions to study miRNA sponge (also\ncalled ceRNA or miRNA decoy), including popular methods for\nidentifying miRNA sponge interactions, and the integrative\nmethod to integrate miRNA sponge interactions from different\nmethods, as well as the functions to validate miRNA sponge\ninteractions, and infer miRNA sponge modules, conduct\nenrichment analysis of modules, and conduct survival analysis\nof modules.","downloads_month":144,"downloads_total":3746,"page":"http://bioconductor.org/packages/release/bioc/html/miRsponge.html"},{"authors":"Junpeng Zhang","name":"miRspongeR","license":"GPL-3","tags":["GeneExpression","BiomedicalInformatics","NetworkEnrichment","Survival","Microarray","Software"],"description":"The previous name of the miRspongeR package is called\n\"miRsponge\". This package provides several functions to study\nmiRNA sponge (also called ceRNA or miRNA decoy), including\npopular methods for identifying miRNA sponge interactions, and\nthe integrative method to integrate miRNA sponge interactions\nfrom different methods, as well as the functions to validate\nmiRNA sponge interactions, and infer miRNA sponge modules,\nconduct enrichment analysis of modules, and conduct survival\nanalysis of modules.","downloads_month":0,"downloads_total":6,"page":"http://bioconductor.org/packages/release/bioc/html/miRspongeR.html"},{"authors":"Yue Li","name":"Mirsynergy","license":"GPL-2","tags":["Clustering"],"description":"Detect synergistic miRNA regulatory modules by overlapping\nneighbourhood expansion.","downloads_month":142,"downloads_total":19236,"page":"http://bioconductor.org/packages/release/bioc/html/Mirsynergy.html"},{"authors":"Belinda Phipson and Jovana Maksimovic","name":"missMethyl","license":"GPL-2","tags":["Normalization","DNAMethylation","MethylationArray","GenomicVariation","GeneticVariability","DifferentialMethylation","GeneSetEnrichment"],"description":"Normalisation and testing for differential variability and\ndifferential methylation for data from Illumina's Infinium\nHumanMethylation450 array. The normalisation procedure is\nsubset-quantile within-array normalisation (SWAN), which allows\nInfinium I and II type probes on a single array to be\nnormalised together. The test for differential variability is\nbased on an empirical Bayes version of Levene's test.\nDifferential methylation testing is performed using RUV, which\ncan adjust for systematic errors of unknown origin in\nhigh-dimensional data by using negative control probes. Gene\nontology analysis is performed by taking into account the\nnumber of probes per gene on the array.","downloads_month":1015,"downloads_total":62608,"page":"http://bioconductor.org/packages/release/bioc/html/missMethyl.html"},{"authors":"Ignacio Gonzalez and Valentin Voillet","name":"missRows","license":"Artistic-2.0","tags":["Software","StatisticalMethod","DimensionReduction","PrincipalComponent","MathematicalBiology","Visualization"],"description":"The missRows package implements the MI-MFA method to deal\nwith missing individuals ('biological units') in multi-omics\ndata integration. The MI-MFA method generates multiple imputed\ndatasets from a Multiple Factor Analysis model, then the yield\nresults are combined in a single consensus solution. The\npackage provides functions for estimating coordinates of\nindividuals and variables, imputing missing individuals, and\nvarious diagnostic plots to inspect the pattern of missingness\nand visualize the uncertainty due to missing values.","downloads_month":120,"downloads_total":1946,"page":"http://bioconductor.org/packages/release/bioc/html/missRows.html"},{"authors":"Gregoire Pau","name":"mitoODE","license":"LGPL","tags":["ImmunoOncology","ExperimentData","TimeCourse","CellBasedAssays","Preprocessing"],"description":"The package contains the methods to fit a cell-cycle model\non cell count data and the code to reproduce the results shown\nin our paper \"Dynamical modelling of phenotypes in a\ngenome-wide RNAi live-cell imaging assay\" by Pau, G., Walter,\nT., Neumann, B., Heriche, J.-K., Ellenberg, J., & Huber, W.,\nBMC Bioinformatics (2013), 14(1), 308.\ndoi:10.1186/1471-2105-14-308","downloads_month":127,"downloads_total":20408,"page":"http://bioconductor.org/packages/release/bioc/html/mitoODE.html"},{"authors":"Kim-Anh Le Cao, Florian Rohart, Ignacio Gonzalez, Sebastien\nDejean with key contributors Benoit Gautier, Francois Bartolo\nand contributions from Pierre Monget, Jeff Coquery, FangZou\nYao and Benoit Liquet.","name":"mixOmics","license":"GPL (>= 2)","tags":["ImmunoOncology","Microarray","Sequencing","Metabolomics","Metagenomics","Proteomics","GenePrediction","MultipleComparison","Classification","Regression"],"description":"Multivariate methods are well suited to large omics data\nsets where the number of variables (e.g. genes, proteins,\nmetabolites) is much larger than the number of samples\n(patients, cells, mice). They have the appealing properties of\nreducing the dimension of the data by using instrumental\nvariables (components), which are defined as combinations of\nall variables. Those components are then used to produce useful\ngraphical outputs that enable better understanding of the\nrelationships and correlation structures between the different\ndata sets that are integrated. mixOmics offers a wide range of\nmultivariate methods for the exploration and integration of\nbiological datasets with a particular focus on variable\nselection. The package proposes several sparse multivariate\nmodels we have developed to identify the key variables that are\nhighly correlated, and/or explain the biological outcome of\ninterest. The data that can be analysed with mixOmics may come\nfrom high throughput sequencing technologies, such as omics\ndata (transcriptomics, metabolomics, proteomics, metagenomics\netc) but also beyond the realm of omics (e.g. spectral\nimaging). The methods implemented in mixOmics can also handle\nmissing values without having to delete entire rows with\nmissing data. A non exhaustive list of methods include variants\nof generalised Canonical Correlation Analysis, sparse Partial\nLeast Squares and sparse Discriminant Analysis. Recently we\nimplemented integrative methods to combine multiple data sets:\nN-integration with variants of Generalised Canonical\nCorrelation Analysis and P-integration with variants of\nmulti-group Partial Least Squares.","downloads_month":2203,"downloads_total":15112,"page":"http://bioconductor.org/packages/release/bioc/html/mixOmics.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu>, Robert Gentleman, Jess Mar, and contributions from Jason Vertrees\n<jv@cs.dartmouth.edu> and Laurent Gatto <lg390@cam.ac.uk>","name":"MLInterfaces","license":"LGPL","tags":["Classification","Clustering"],"description":"This package provides uniform interfaces to machine\nlearning code for data in R and Bioconductor containers.","downloads_month":544,"downloads_total":117350,"page":"http://bioconductor.org/packages/release/bioc/html/MLInterfaces.html"},{"authors":"Irene Zeng [aut, cre] and Thomas Lumley [ctb]","name":"mlm4omics","license":"GPL-3","tags":["ImmunoOncology","Bayesian","CopyNumberVariation","Classification","Regression","MassSpectrometry","Proteomics","Software"],"description":"To conduct Bayesian inference regression for responses\nwith multilevel explanatory variables and missing values; It\nuses function from 'Stan', a software to implement posterior\nsampling using Hamiltonian MC and its variation Non-U-Turn\nalgorithms. It implements the posterior sampling of regression\ncoefficients from the multilevel regression models. The package\nhas two main functions to handle not-missing-at-random missing\nresponses and left-censored with not-missing-at random\nresponses. The purpose is to provide a similar format as the\nother R regression functions but using 'Stan' models.","downloads_month":35,"downloads_total":370,"page":"http://bioconductor.org/packages/release/bioc/html/mlm4omics.html"},{"authors":"Nandini Raghavan, Tobias Verbeke, An De Bondt with\ncontributions by Javier Cabrera, Dhammika Amaratunga, Tine\nCasneuf and Willem Ligtenberg","name":"MLP","license":"GPL-3","tags":["Genetics","Reactome","KEGG"],"description":"Mean Log P Analysis","downloads_month":182,"downloads_total":37570,"page":"http://bioconductor.org/packages/release/bioc/html/MLP.html"},{"authors":"Gokmen Zararsiz, Dincer Goksuluk, Selcuk Korkmaz, Vahap Eldem, Izzet Parug Duru, Ahmet Ozturk and Ahmet Ergun Karaagaoglu","name":"MLSeq","license":"GPL(>=2)","tags":["ImmunoOncology","Sequencing","RNASeq","Classification","Clustering"],"description":"This package applies several machine learning methods,\nincluding SVM, bagSVM, Random Forest and CART to RNA-Seq data.","downloads_month":328,"downloads_total":25860,"page":"http://bioconductor.org/packages/release/bioc/html/MLSeq.html"},{"authors":"Gabriele Schweikert [cre, aut] and David Kuo [aut]","name":"MMDiff2","license":"Artistic-2.0","tags":["ChIPSeq","DifferentialPeakCalling","Sequencing","Software"],"description":"This package detects statistically significant differences\nbetween read enrichment profiles in different ChIP-Seq samples.\nTo take advantage of shape differences it uses Kernel methods\n(Maximum Mean Discrepancy, MMD).","downloads_month":141,"downloads_total":8376,"page":"http://bioconductor.org/packages/release/bioc/html/MMDiff2.html"},{"authors":"Guy Brock <guy.brock@louisville.edu>, Partha Mukhopadhyay\n<p0mukh01@louisville.edu>, Vasyl Pihur\n<vasyl.pihur@louisville.edu>, Robert M. Greene\n<Dr.Bob.Greene@gmail.com>, and M. Michele Pisano\n<mmpisa01@louisville.edu>","name":"MmPalateMiRNA","license":"GPL-3","tags":["Microarray","TwoChannel","QualityControl","Preprocessing","DifferentialExpression","MultipleComparison","Clustering","GO","Pathways","ReportWriting","SequenceMatching"],"description":"R package compendium for the analysis of murine palate\nmiRNA two-color expression data.","downloads_month":140,"downloads_total":32810,"page":"http://bioconductor.org/packages/release/bioc/html/MmPalateMiRNA.html"},{"authors":"Dong Li, James B. Brown, Luisa Orsini, Zhisong Pan and Guyu Hu and\nShan He","name":"MODA","license":"GPL (>= 2)","tags":["GeneExpression","Microarray","DifferentialExpression","Network"],"description":"MODA can be used to estimate and construct\ncondition-specific gene co-expression networks, and identify\ndifferentially expressed subnetworks as conserved or condition\nspecific modules which are potentially associated with relevant\nbiological processes.","downloads_month":134,"downloads_total":8260,"page":"http://bioconductor.org/packages/release/bioc/html/MODA.html"},{"authors":"Chen Meng","name":"mogsa","license":"GPL-2","tags":["GeneExpression","PrincipalComponent","StatisticalMethod","Clustering","Software"],"description":"This package provide a method for doing gene set analysis\nbased on multiple omics data.","downloads_month":163,"downloads_total":13958,"page":"http://bioconductor.org/packages/release/bioc/html/mogsa.html"},{"authors":"Cole Trapnell","name":"monocle","license":"Artistic-2.0","tags":["ImmunoOncology","Sequencing","RNASeq","GeneExpression","DifferentialExpression","Infrastructure","DataImport","DataRepresentation","Visualization","Clustering","MultipleComparison","QualityControl"],"description":"Monocle performs differential expression and time-series\nanalysis for single-cell expression experiments. It orders\nindividual cells according to progress through a biological\nprocess, without knowing ahead of time which genes define\nprogress through that process. Monocle also performs\ndifferential expression analysis, clustering, visualization,\nand other useful tasks on single cell expression data.  It is\ndesigned to work with RNA-Seq and qPCR data, but could be used\nwith other types as well.","downloads_month":3528,"downloads_total":128912,"page":"http://bioconductor.org/packages/release/bioc/html/monocle.html"},{"authors":"Antonio Colaprico*, Catharina Olsen*, Claudia Cava, Thilde\nTerkelsen, Laura Cantini, Andre Olsen, Gloria Bertoli, Andrei\nZinovyev, Emmanuel Barillot, Isabella Castiglioni, Elena\nPapaleo and Gianluca Bontempi","name":"MoonlightR","license":"GPL (>= 3)","tags":["DNAMethylation","DifferentialMethylation","GeneRegulation","GeneExpression","MethylationArray","DifferentialExpression","Pathways","Network","Survival","GeneSetEnrichment","NetworkEnrichment"],"description":"Motivation: The understanding of cancer mechanism requires\nthe identification of genes playing a role in the development\nof the pathology and the characterization of their role\n(notably oncogenes and tumor suppressors). Results: We present\nan R/bioconductor package called MoonlightR which returns a\nlist of candidate driver genes for specific cancer types on the\nbasis of TCGA expression data. The method first infers gene\nregulatory networks and then carries out a functional\nenrichment analysis (FEA) (implementing an upstream regulator\nanalysis, URA) to score the importance of well-known biological\nprocesses with respect to the studied cancer type. Eventually,\nby means of random forests, MoonlightR predicts two specific\nroles for the candidate driver genes: i) tumor suppressor genes\n(TSGs) and ii) oncogenes (OCGs). As a consequence, this\nmethodology does not only identify genes playing a dual role\n(e.g. TSG in one cancer type and OCG in another) but also helps\nin elucidating the biological processes underlying their\nspecific roles. In particular, MoonlightR can be used to\ndiscover OCGs and TSGs in the same cancer type. This may help\nin answering the question whether some genes change role\nbetween early stages (I, II) and late stages (III, IV) in\nbreast cancer. In the future, this analysis could be useful to\ndetermine the causes of different resistances to\nchemotherapeutic treatments.","downloads_month":157,"downloads_total":7914,"page":"http://bioconductor.org/packages/release/bioc/html/MoonlightR.html"},{"authors":"Philipp Eser and Achim Tresch","name":"MoPS","license":"GPL-3","tags":["GeneRegulation","Classification","TimeCourse","Regression"],"description":"Identification and characterization of periodic\nfluctuations in time-series data.","downloads_month":135,"downloads_total":14894,"page":"http://bioconductor.org/packages/release/bioc/html/MoPS.html"},{"authors":"Dongjun Chung, Pei Fen Kuan, Rene Welch and Sunduz Keles","name":"mosaics","license":"GPL (>= 2)","tags":["ChIPseq","Sequencing","Transcription","Genetics","Bioinformatics"],"description":"This package provides functions for fitting MOSAiCS and\nMOSAiCS-HMM, a statistical framework to analyze one-sample or\ntwo-sample ChIP-seq data of transcription factor binding and\nhistone modification.","downloads_month":218,"downloads_total":504982,"page":"http://bioconductor.org/packages/release/bioc/html/mosaics.html"},{"authors":"Simon Gert Coetzee [aut and cre] Dennis J. Hazelett [aut]","name":"motifbreakR","license":"GPL-2","tags":["ChIPSeq","Visualization","MotifAnnotation"],"description":"We introduce motifbreakR, which allows the biologist to\njudge in the first place whether the sequence surrounding the\npolymorphism is a good match, and in the second place how much\ninformation is gained or lost in one allele of the polymorphism\nrelative to another. MotifbreakR is both flexible and\nextensible over previous offerings; giving a choice of\nalgorithms for interrogation of genomes with motifs from public\nsources that users can choose from; these are 1) a weighted-sum\nprobability matrix, 2) log-probabilities, and 3) weighted by\nrelative entropy. MotifbreakR can predict effects for novel or\npreviously described variants in public databases, making it\nsuitable for tasks beyond the scope of its original design.\nLastly, it can be used to interrogate any genome curated within\nBioconductor (currently there are 22).","downloads_month":253,"downloads_total":14264,"page":"http://bioconductor.org/packages/release/bioc/html/motifbreakR.html"},{"authors":"Wolfgang Kopp [aut and cre]","name":"motifcounter","license":"GPL-2","tags":["Transcription","MotifAnnotation","SequenceMatching","Software"],"description":"'motifcounter' provides motif matching, motif counting and\nmotif enrichment functionality based on position frequency\nmatrices. The main features of the packages include the\nutilization of higher-order background models and accounting\nfor self-overlapping motif matches when determining motif\nenrichment. The background model allows to capture dinucleotide\n(or higher-order nucleotide) composition adequately which may\nreduced model biases and misleading results compared to using\nsimple GC background models. When conducting a motif enrichment\nanalysis based on the motif match count, the package relies on\na compound Poisson distribution or alternatively a\ncombinatorial model. These distribution account for\nself-overlapping motif structures as exemplified by repeat-like\nor palindromic motifs, and allow to determine the p-value and\nfold-enrichment for a set of observed motif matches.","downloads_month":165,"downloads_total":26912,"page":"http://bioconductor.org/packages/release/bioc/html/motifcounter.html"},{"authors":"Paul Shannon and Matt Richards","name":"MotifDb","license":"Artistic-2.0 | file LICENSE","tags":["MotifAnnotation"],"description":"More than 9900 annotated position frequency matrices from\n14 public sources, for multiple organisms.","downloads_month":511,"downloads_total":90376,"page":"http://bioconductor.org/packages/release/bioc/html/MotifDb.html"},{"authors":"Alicia Schep [aut, cre] and Stanford University [cph]","name":"motifmatchr","license":"GPL-3 + file LICENSE","tags":["MotifAnnotation"],"description":"Quickly find motif matches for many motifs and many\nsequences. Wraps C++ code from the MOODS motif calling library,\nwhich was developed by Pasi Rastas, Janne Korhonen, and Petri\nMartinmäki.","downloads_month":397,"downloads_total":8854,"page":"http://bioconductor.org/packages/release/bioc/html/motifmatchr.html"},{"authors":"Zizhen Yao","name":"motifRG","license":"Artistic-2.0","tags":["Transcription","MotifDiscovery"],"description":"Tools for discriminative motif discovery using regression\nmethods","downloads_month":247,"downloads_total":36628,"page":"http://bioconductor.org/packages/release/bioc/html/motifRG.html"},{"authors":"Jianhong Ou, Michael Brodsky, Scot Wolfe and Lihua Julie Zhu","name":"motifStack","license":"GPL (>= 2)","tags":["SequenceMatching","Visualization","Sequencing","Microarray","Alignment","ChIPchip","ChIPSeq","MotifAnnotation","DataImport"],"description":"The motifStack package is designed for graphic\nrepresentation of multiple motifs with different similarity\nscores. It works with both DNA/RNA sequence motif and amino\nacid sequence motif. In addition, it provides the flexibility\nfor users to customize the graphic parameters such as the font\ntype and symbol colors.","downloads_month":769,"downloads_total":98636,"page":"http://bioconductor.org/packages/release/bioc/html/motifStack.html"},{"authors":"Eloi Mercier and Raphael Gottardo","name":"MotIV","license":"GPL-2","tags":["Microarray","ChIPchip","ChIPSeq","GenomicSequence","MotifAnnotation"],"description":"This package makes use of STAMP for comparing a set of\nmotifs to a given database (e.g. JASPAR). It can also be used\nto visualize motifs, motif distributions, modules and filter\nmotifs.","downloads_month":859,"downloads_total":95428,"page":"http://bioconductor.org/packages/release/bioc/html/MotIV.html"},{"authors":"Peijie Lin, Sylvain Foret and Conrad Burden","name":"MPFE","license":"GPL (>= 3)","tags":["HighThroughputSequencingData","DNAMethylation","MethylSeq"],"description":"Estimate distribution of methylation patterns from a table\nof counts from a bisulphite sequencing experiment given a\nnon-conversion rate and read error rate.","downloads_month":124,"downloads_total":14192,"page":"http://bioconductor.org/packages/release/bioc/html/MPFE.html"},{"authors":"Leslie Myint [cre, aut] and Kasper D. Hansen [aut]","name":"mpra","license":"Artistic-2.0","tags":["Software","GeneRegulation","Sequencing","FunctionalGenomics"],"description":"Tools for data management, count preprocessing, and\ndifferential analysis in massively parallel report assays\n(MPRA).","downloads_month":145,"downloads_total":3608,"page":"http://bioconductor.org/packages/release/bioc/html/mpra.html"},{"authors":"Tal Ashuach [aut, cre], David S Fischer [aut], Nir Yosef [ctb], Fabian J Theis [ctb] and ","name":"MPRAnalyze","license":"GPL-3","tags":["ImmunoOncology","Software","StatisticalMethod","Sequencing","GeneExpression","CellBiology","CellBasedAssays","DifferentialExpression"],"description":"MPRAnalyze provides statistical framework for the analysis\nof data generated by Massively Parallel Reporter Assays\n(MPRAs), used to directly measure enhancer activity. MPRAnalyze\ncan be used for quantification of enhancer activity,\nclassification of active enhancers and comparative analyses of\nenhancer activity between conditions. MPRAnalyze construct a\nnested pair of generalized linear models (GLMs) to relate the\nDNA and RNA observations, easily adjustable to various\nexperimental designs and conditions, and provides a set of\nrigorous statistical testig schemes.","downloads_month":122,"downloads_total":982,"page":"http://bioconductor.org/packages/release/bioc/html/MPRAnalyze.html"},{"authors":"Enrico Bonatesta, Christoph Horejs-Kainrath and Ulrich Bodenhofer","name":"msa","license":"GPL (>= 2)","tags":["MultipleSequenceAlignment","Alignment","MultipleComparison","Sequencing"],"description":"The 'msa' package provides a unified R/Bioconductor\ninterface to the multiple sequence alignment algorithms\nClustalW, ClustalOmega, and Muscle. All three algorithms are\nintegrated in the package, therefore, they do not depend on any\nexternal software tools and are available for all major\nplatforms. The multiple sequence alignment algorithms are\ncomplemented by a function for pretty-printing multiple\nsequence alignments using the LaTeX package TeXshade.","downloads_month":1165,"downloads_total":84978,"page":"http://bioconductor.org/packages/release/bioc/html/msa.html"},{"authors":"Benjamin Mayne","name":"msgbsR","license":"GPL-2","tags":["ImmunoOncology","DifferentialMethylation","DataImport","Epigenetics","MethylSeq"],"description":"Pipeline for the anaysis of a MS-GBS experiment.","downloads_month":172,"downloads_total":5026,"page":"http://bioconductor.org/packages/release/bioc/html/msgbsR.html"},{"authors":"Thomas Lin Pedersen","name":"MSGFgui","license":"GPL (>= 2)","tags":["ImmunoOncology","MassSpectrometry","Proteomics","GUI","Visualization"],"description":"This package makes it possible to perform analyses using\nthe MSGFplus package in a GUI environment. Furthermore it\nenables the user to investigate the results using interactive\nplots, summary statistics and filtering. Lastly it exposes the\ncurrent results to another R session so the user can seamlessly\nintegrate the gui into other workflows.","downloads_month":351,"downloads_total":24812,"page":"http://bioconductor.org/packages/release/bioc/html/MSGFgui.html"},{"authors":"Thomas Lin Pedersen","name":"MSGFplus","license":"GPL (>= 2)","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"This package contains function to perform peptide\nidentification using the MS-GF+ algorithm. The package contains\nfunctionality for building up a parameter set both in code and\nthrough a simple GUI, as well as running the algorithm in\nbatches, potentially asynchronously.","downloads_month":496,"downloads_total":30146,"page":"http://bioconductor.org/packages/release/bioc/html/MSGFplus.html"},{"authors":"Josep Gregori, Alex Sanchez, and Josep Villanueva","name":"msmsEDA","license":"GPL-2","tags":["ImmunoOncology","Software","MassSpectrometry","Proteomics"],"description":"Exploratory data analysis to assess the quality of a set\nof LC-MS/MS experiments, and visualize de influence of the\ninvolved factors.","downloads_month":396,"downloads_total":32450,"page":"http://bioconductor.org/packages/release/bioc/html/msmsEDA.html"},{"authors":"Josep Gregori, Alex Sanchez, and Josep Villanueva","name":"msmsTests","license":"GPL-2","tags":["ImmunoOncology","Software","MassSpectrometry","Proteomics"],"description":"Statistical tests for label-free LC-MS/MS data by spectral\ncounts, to discover differentially expressed proteins between\ntwo biological conditions. Three tests are available: Poisson\nGLM regression, quasi-likelihood GLM regression, and the\nnegative binomial of the edgeR package.The three models admit\nblocking factors to control for nuissance variables.To assure a\ngood level of reproducibility a post-test filter is available,\nwhere we may set the minimum effect size considered biologicaly\nrelevant, and the minimum expression of the most abundant\ncondition.","downloads_month":353,"downloads_total":31870,"page":"http://bioconductor.org/packages/release/bioc/html/msmsTests.html"},{"authors":"Laurent Gatto, Johannes Rainer, Sebastian Gibb with\ncontributions from Guangchuang Yu, Samuel Wieczorek, Vasile-Cosmin Lazar, Vladislav Petyuk, Thomas Naake, Richie\nCotton, Arne Smits, Martina Fisher, Ludger Goeminne and Adriaan\nSticker and Lieven Clement.","name":"MSnbase","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","Proteomics","MassSpectrometry","QualityControl","DataImport"],"description":"MSnbase provides infrastructure for manipulation,\nprocessing and visualisation of mass spectrometry and\nproteomics data, ranging from raw to quantitative and annotated\ndata.","downloads_month":3256,"downloads_total":202542,"page":"http://bioconductor.org/packages/release/bioc/html/MSnbase.html"},{"authors":"Vlad Petyuk with contributions from Laurent Gatto","name":"MSnID","license":"Artistic-2.0","tags":["Proteomics","MassSpectrometry","ImmunoOncology"],"description":"Extracts MS/MS ID data from mzIdentML (leveraging mzID\npackage) or text files. After collating the search results from\nmultiple datasets it assesses their identification quality and\noptimize filtering criteria to achieve the maximum number of\nidentifications while not exceeding a specified false discovery\nrate. Also contains a number of utilities to explore the MS/MS\nresults and assess missed and irregular enzymatic cleavages,\nmass measurement accuracy, etc.","downloads_month":402,"downloads_total":28770,"page":"http://bioconductor.org/packages/release/bioc/html/MSnID.html"},{"authors":"Thomas N. Lawson, Ralf Weber, Martin Jones, Mark Viant and Warwick\nDunn","name":"msPurity","license":"GPL (>= 2)","tags":["MassSpectrometry","Metabolomics","Software"],"description":"Assess the contribution of the targeted precursor in\nfragmentation acquired or anticipated isolation windows using a\nmetric called \"precursor purity\". Also provides simple\nprocessing steps (averaging, filtering, blank subtraction, etc)\nfor DI-MS data. Works for both LC-MS(/MS) and DI-MS(/MS) data.\nSpectral matching of fragmentation spectra can also be run\nagainst a SQLite database of library spectra.","downloads_month":216,"downloads_total":8152,"page":"http://bioconductor.org/packages/release/bioc/html/msPurity.html"},{"authors":"Meena Choi [aut, cre], Cyril Galitzine [aut], Ting Huang [aut], Tsung-Heng Tsai [aut] and Olga Vitek [aut]","name":"MSstats","license":"Artistic-2.0","tags":["ImmunoOncology","MassSpectrometry","Proteomics","Software","Normalization","QualityControl","TimeCourse"],"description":"A set of tools for statistical relative protein\nsignificance analysis in DDA, SRM and DIA experiments.","downloads_month":465,"downloads_total":49476,"page":"http://bioconductor.org/packages/release/bioc/html/MSstats.html"},{"authors":"Eralp Dogu [aut, cre], Sara Taheri [aut] and Olga Vitek [aut]","name":"MSstatsQC","license":"Artistic License 2.0","tags":["ImmunoOncology","Software","QualityControl","Proteomics","MassSpectrometry"],"description":"MSstatsQC is an R package which provides longitudinal\nsystem suitability monitoring and quality control tools for\nproteomic experiments.","downloads_month":167,"downloads_total":4808,"page":"http://bioconductor.org/packages/release/bioc/html/MSstatsQC.html"},{"authors":"Eralp Dogu [aut, cre], Sara Taheri [aut] and Olga Vitek [aut]","name":"MSstatsQCgui","license":"Artistic License 2.0","tags":["ImmunoOncology","Software","QualityControl","Proteomics","MassSpectrometry","GUI"],"description":"MSstatsQCgui is a Shiny app which provides longitudinal\nsystem suitability monitoring and quality control tools for\nproteomic experiments.","downloads_month":131,"downloads_total":1914,"page":"http://bioconductor.org/packages/release/bioc/html/MSstatsQCgui.html"},{"authors":"Ting Huang [aut, cre], Meena Choi [aut] and Sicheng Hao [aut]","name":"MSstatsTMT","license":"Artistic-2.0","tags":["MassSpectrometry","Proteomics","Software"],"description":"Tools for protein significance analysis in shotgun mass\nspectrometry-based proteomic experiments with tandem mass tag\n(TMT) labeling.","downloads_month":159,"downloads_total":1226,"page":"http://bioconductor.org/packages/release/bioc/html/MSstatsTMT.html"},{"authors":"Tim Triche [cre, aut], Noor Sohail [aut], Ben Johnson [aut], Tim Vickers [ctb] and Azif Zubair [ctb]","name":"MTseeker","license":"GPL-3","tags":["Genetics","Metabolomics","VariantAnnotation"],"description":"Variant analysis tools for mitochondrial genetics.","downloads_month":68,"downloads_total":466,"page":"http://bioconductor.org/packages/release/bioc/html/MTseeker.html"},{"authors":"Claudio Isella","name":"Mulcom","license":"GPL-2","tags":["StatisticalMethod","MultipleComparison","Microarray","DifferentialExpression","GeneExpression"],"description":"Identification of differentially expressed genes and false\ndiscovery rate (FDR) calculation by Multiple Comparison test","downloads_month":244,"downloads_total":43876,"page":"http://bioconductor.org/packages/release/bioc/html/Mulcom.html"},{"authors":"Marcel Ramos [aut, cre], Levi Waldron [aut] and MultiAssay SIG\n[ctb]","name":"MultiAssayExperiment","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"MultiAssayExperiment harmonizes data management of\nmultiple assays performed on an overlapping set of specimens.\nIt provides a familiar Bioconductor user experience by\nextending concepts from SummarizedExperiment, supporting an\nopen-ended mix of standard data classes for individual assays,\nand allowing subsetting by genomic ranges or rownames.","downloads_month":1442,"downloads_total":44194,"page":"http://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html"},{"authors":"Nathan Lawlor [aut, cre], Peiyong Guan [aut], Alec Fabbri\n[aut], Krish Karuturi [aut] and Joshy George [aut]","name":"multiClust","license":"GPL (>= 2)","tags":["FeatureExtraction","Clustering","GeneExpression","Survival"],"description":"Clustering is carried out to identify patterns in\ntranscriptomics profiles to determine clinically relevant\nsubgroups of patients. Feature (gene) selection is a critical\nand an integral part of the process. Currently, there are many\nfeature selection and clustering methods to identify the\nrelevant genes and perform clustering of samples. However,\nchoosing an appropriate methodology is difficult. In addition,\nextensive feature selection methods have not been supported by\nthe available packages. Hence, we developed an integrative\nR-package called multiClust that allows researchers to\nexperiment with the choice of combination of methods for gene\nselection and clustering with ease. Using multiClust, we\nidentified the best performing clustering methodology in the\ncontext of clinical outcome. Our observations demonstrate that\nsimple methods such as variance-based ranking perform well on\nthe majority of data sets, provided that the appropriate number\nof genes is selected. However, different gene ranking and\nselection methods remain relevant as no methodology works for\nall studies.","downloads_month":183,"downloads_total":11358,"page":"http://bioconductor.org/packages/release/bioc/html/multiClust.html"},{"authors":"Carlos Ruiz-Arenas [aut, cre], Carles Hernandez-Ferrer [aut] and Juan R. Gonzalez [aut]","name":"MultiDataSet","license":"file LICENSE","tags":["Software","DataRepresentation"],"description":"Implementation of the BRGE's (Bioinformatic Research Group\nin Epidemiology from Center for Research in Environmental\nEpidemiology) MultiDataSet and ResultSet. MultiDataSet is\ndesigned for integrating multi omics data sets and ResultSet is\na container for omics results. This package contains base\nclasses for MEAL and rexposome packages.","downloads_month":187,"downloads_total":11444,"page":"http://bioconductor.org/packages/release/bioc/html/MultiDataSet.html"},{"authors":"John Stansfield [aut, cre] and Mikhail Dozmorov [aut]","name":"multiHiCcompare","license":"MIT + file LICENSE","tags":["Software","HiC","Sequencing","Normalization"],"description":"multiHiCcompare provides functions for joint normalization\nand difference detection in multiple Hi-C datasets. This\nextension of the original HiCcompare package now allows for\nHi-C experiments with more than 2 groups and multiple samples\nper group. multiHiCcompare operates on processed Hi-C data in\nthe form of sparse upper triangular matrices. It accepts four\ncolumn (chromosome, region1, region2, IF) tab-separated text\nfiles storing chromatin interaction matrices. multiHiCcompare\nprovides cyclic loess and fast loess (fastlo) methods adapted\nto jointly normalizing Hi-C data. Additionally, it provides a\ngeneral linear model (GLM) framework adapting the edgeR package\nto detect differences in Hi-C data in a distance dependent\nmanner.","downloads_month":98,"downloads_total":822,"page":"http://bioconductor.org/packages/release/bioc/html/multiHiCcompare.html"},{"authors":"Simina M. Boca, Ruth Heller and Joshua N. Sampson","name":"MultiMed","license":"GPL (>= 2) + file LICENSE","tags":["MultipleComparison","StatisticalMethod","Software"],"description":"Implements methods for testing multiple mediators","downloads_month":139,"downloads_total":14524,"page":"http://bioconductor.org/packages/release/bioc/html/MultiMed.html"},{"authors":"Yuanbin Ru [aut], Matt Mulvahill [cre, aut], Spencer Mahaffey\n[aut], Katerina Kechris [aut, cph and ths]","name":"multiMiR","license":"MIT + file LICENSE","tags":["miRNAData","Homo_sapiens_Data","Mus_musculus_Data","Rattus_norvegicus_Data","OrganismData"],"description":"A collection of microRNAs/targets from external resources,\nincluding validated microRNA-target databases (miRecords,\nmiRTarBase and TarBase), predicted microRNA-target databases\n(DIANA-microT, ElMMo, MicroCosm, miRanda, miRDB, PicTar, PITA\nand TargetScan) and microRNA-disease/drug databases\n(miR2Disease, Pharmaco-miR VerSe and PhenomiR).","downloads_month":213,"downloads_total":5362,"page":"http://bioconductor.org/packages/release/bioc/html/multiMiR.html"},{"authors":"Jing Wang <jingwang.uestc@gmail.com>","name":"multiOmicsViz","license":"LGPL","tags":["Software","Visualization","SystemsBiology"],"description":"Calculate the spearman correlation between the source\nomics data and other target omics data, identify the\nsignificant correlations and plot the significant correlations\non the heat map in which the x-axis and y-axis are ordered by\nthe chromosomal location.","downloads_month":138,"downloads_total":5090,"page":"http://bioconductor.org/packages/release/bioc/html/multiOmicsViz.html"},{"authors":"Mizanur Khondoker <mizanur.khondoker@ed.ac.uk>, Chris Glasbey and Bruce Worton.","name":"multiscan","license":"GPL (>= 2)","tags":["Microarray","Preprocessing"],"description":"Estimates gene expressions from several laser scans of the\nsame microarray","downloads_month":132,"downloads_total":41124,"page":"http://bioconductor.org/packages/release/bioc/html/multiscan.html"},{"authors":"Katherine S. Pollard, Houston N. Gilbert, Yongchao Ge, Sandra\nTaylor and Sandrine Dudoit","name":"multtest","license":"LGPL","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"Non-parametric bootstrap and permutation resampling-based\nmultiple testing procedures (including empirical Bayes methods)\nfor controlling the family-wise error rate (FWER), generalized\nfamily-wise error rate (gFWER), tail probability of the\nproportion of false positives (TPPFP), and false discovery rate\n(FDR).  Several choices of bootstrap-based null distribution\nare implemented (centered, centered and scaled,\nquantile-transformed). Single-step and step-wise methods are\navailable. Tests based on a variety of t- and F-statistics\n(including t-statistics based on regression parameters from\nlinear and survival models as well as those based on\ncorrelation parameters) are included.  When probing hypotheses\nwith t-statistics, users may also select a potentially faster\nnull distribution which is multivariate normal with mean zero\nand variance covariance matrix derived from the vector\ninfluence function.  Results are reported in terms of adjusted\np-values, confidence regions and test statistic cutoffs. The\nprocedures are directly applicable to identifying\ndifferentially expressed genes in DNA microarray experiments.","downloads_month":11771,"downloads_total":2138510,"page":"http://bioconductor.org/packages/release/bioc/html/multtest.html"},{"authors":"Algorithm by Robert C. Edgar. R port by Alex T. Kalinka.","name":"muscle","license":"Unlimited","tags":["MultipleSequenceAlignment","Alignment","Sequencing","Genetics","SequenceMatching","DataImport"],"description":"MUSCLE performs multiple sequence alignments of nucleotide\nor amino acid sequences.","downloads_month":415,"downloads_total":45398,"page":"http://bioconductor.org/packages/release/bioc/html/muscle.html"},{"authors":"Francis Blokzijl, Roel Janssen, Ruben van Boxtel and Edwin Cuppen","name":"MutationalPatterns","license":"MIT + file LICENSE","tags":["Genetics","SomaticMutation"],"description":"An extensive toolset for the characterization and\nvisualization of a wide range of mutational patterns in base\nsubstitution catalogs.","downloads_month":379,"downloads_total":16526,"page":"http://bioconductor.org/packages/release/bioc/html/MutationalPatterns.html"},{"authors":"Elizabeth Whalen","name":"MVCClass","license":"LGPL","tags":["Visualization","Infrastructure","GraphAndNetwork"],"description":"Creates classes used in model-view-controller (MVC) design","downloads_month":128,"downloads_total":45448,"page":"http://bioconductor.org/packages/release/bioc/html/MVCClass.html"},{"authors":"Andrea Rodriguez-Martinez, Joram M. Posma, Rafael Ayala, Ana L.\nNeves, Maryam Anwar, Jeremy K. Nicholson and Marc-Emmanuel Dumas","name":"MWASTools","license":"CC BY-NC-ND 4.0","tags":["Metabolomics","Lipidomics","Cheminformatics","SystemsBiology","QualityControl"],"description":"MWASTools provides a complete pipeline to perform\nmetabolome-wide association studies. Key functionalities of the\npackage include: quality control analysis of metabonomic data;\nMWAS using different association models (partial correlations;\ngeneralized linear models); model validation using\nnon-parametric bootstrapping; visualization of MWAS results;\nNMR metabolite identification using STOCSY; and biological\ninterpretation of MWAS results.","downloads_month":175,"downloads_total":6872,"page":"http://bioconductor.org/packages/release/bioc/html/MWASTools.html"},{"authors":"Adam Mark, Ryan Thompson, Cyrus Afrasiabi and Chunlei Wu","name":"mygene","license":"Artistic-2.0","tags":["Annotation"],"description":"MyGene.Info_ provides simple-to-use REST web services to\nquery/retrieve gene annotation data. It's designed with\nsimplicity and performance emphasized. *mygene*, is an\neasy-to-use R wrapper to access MyGene.Info_ services.","downloads_month":648,"downloads_total":51774,"page":"http://bioconductor.org/packages/release/bioc/html/mygene.html"},{"authors":"Adam Mark","name":"myvariant","license":"Artistic-2.0","tags":["VariantAnnotation","Annotation","GenomicVariation"],"description":"MyVariant.info is a comprehensive aggregation of variant\nannotation resources. myvariant is a wrapper for querying\nMyVariant.info services","downloads_month":189,"downloads_total":12850,"page":"http://bioconductor.org/packages/release/bioc/html/myvariant.html"},{"authors":"Thomas Lin Pedersen, Vladislav A Petyuk with contributions from\nLaurent Gatto and Sebastian Gibb.","name":"mzID","license":"GPL (>= 2)","tags":["ImmunoOncology","DataImport","MassSpectrometry","Proteomics"],"description":"A parser for mzIdentML files implemented using the XML\npackage. The parser tries to be general and able to handle all\ntypes of mzIdentML files with the drawback of having less\n'pretty' output than a vendor specific parser. Please contact\nthe maintainer with any problems and supply an mzIdentML file\nso the problems can be fixed quickly.","downloads_month":2626,"downloads_total":126190,"page":"http://bioconductor.org/packages/release/bioc/html/mzID.html"},{"authors":"Bernd Fischer, Steffen Neumann, Laurent Gatto, Qiang Kou and Johannes Rainer","name":"mzR","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","DataImport","Proteomics","Metabolomics","MassSpectrometry"],"description":"mzR provides a unified API to the common file formats and\nparsers available for mass spectrometry data. It comes with a\nwrapper for the ISB random access parser for mass spectrometry\nmzXML, mzData and mzML files. The package contains the original\ncode written by the ISB, and a subset of the proteowizard\nlibrary for mzML and mzIdentML. The netCDF reading code has\npreviously been used in XCMS.","downloads_month":3534,"downloads_total":1372620,"page":"http://bioconductor.org/packages/release/bioc/html/mzR.html"},{"authors":"Jianhong Ou, Haibo Liu, Jun Yu, Hervé Pagès, Paul Kaufman and Lihua Julie Zhu","name":"NADfinder","license":"GPL (>= 2)","tags":["Sequencing","DNASeq","GeneRegulation","PeakDetection"],"description":"Nucleolus is an important structure inside the nucleus in\neukaryotic cells. It is the site for transcribing rDNA into\nrRNA and for assembling ribosomes, aka ribosome biogenesis. In\naddition, nucleoli are dynamic hubs through which numerous\nproteins shuttle and contact specific non-rDNA genomic loci.\nDeep sequencing analyses of DNA associated with isolated\nnucleoli (NAD- seq) have shown that specific loci, termed\nnucleolus- associated domains (NADs) form frequent three-\ndimensional associations with nucleoli. NAD-seq has been used\nto study the biological functions of NAD and the dynamics of\nNAD distribution during embryonic stem cell (ESC)\ndifferentiation. Here, we developed a Bioconductor package\nNADfinder for bioinformatic analysis of the NAD-seq data,\nincluding normalization, smoothing, peak calling, peak trimming\nand annotation.","downloads_month":141,"downloads_total":5700,"page":"http://bioconductor.org/packages/release/bioc/html/NADfinder.html"},{"authors":"hong wang <hong.wang@uky.edu>, tingting zhai\n<tingting.zhai@uky.edu> and chi wang <chi.wang@uky.edu>","name":"NanoStringDiff","license":"GPL","tags":["DifferentialExpression","Normalization"],"description":"This Package utilizes a generalized linear model(GLM) of\nthe negative binomial family to characterize count data and\nallows for multi-factor design. NanoStrongDiff incorporate size\nfactors, calculated from positive controls and housekeeping\ncontrols, and background level, obtained from negative\ncontrols, in the model framework so that all the normalization\ninformation provided by NanoString nCounter Analyzer is fully\nutilized.","downloads_month":202,"downloads_total":13540,"page":"http://bioconductor.org/packages/release/bioc/html/NanoStringDiff.html"},{"authors":"Dorothee Nickles <nickles.dorothee@gene.com>, Thomas Sandmann\n<sandmann.thomas@gene.com>, Robert Ziman\n<ziman.robert@gene.com> and Richard Bourgon\n<bourgon.richard@gene.com>","name":"NanoStringQCPro","license":"Artistic-2.0","tags":["Microarray","mRNAMicroarray","Preprocessing","Normalization","QualityControl","ReportWriting"],"description":"NanoStringQCPro provides a set of quality metrics that can\nbe used to assess the quality of NanoString mRNA gene\nexpression data -- i.e. to identify outlier probes and outlier\nsamples. It also provides different background subtraction and\nnormalization approaches for this data. It outputs suggestions\nfor flagging samples/probes and an easily sharable html quality\ncontrol output.","downloads_month":229,"downloads_total":17890,"page":"http://bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html"},{"authors":"Pedro Madrigal <bioinformatics.engineer@gmail.com> and Pawel\nKrajewski <pkra@igr.poznan.pl>","name":"NarrowPeaks","license":"Artistic-2.0","tags":["Visualization","ChIPSeq","Transcription","Genetics","Sequencing","Sequencing"],"description":"The package applies a functional version of principal\ncomponent analysis (FPCA) to: (1) Postprocess data in wiggle\ntrack format, commonly produced by generic ChIP-seq peak\ncallers, by applying FPCA over a set of read-enriched regions\n(ChIP-seq peaks). This is done to study variability of the the\npeaks, or to shorten their genomic locations accounting for a\ngiven proportion of variation among the enrichment-score\nprofiles. (2) Analyse differential variation between multiple\nChIP-seq samples with replicates. The function\n'narrowpeaksDiff' quantifies differences between the shapes,\nand uses Hotelling's T2 tests on the functional principal\ncomponent scores to identify significant differences across\nconditions. An application of the package for Arabidopsis\ndatasets is described in Mateos, Madrigal, et al. (2015) Genome\nBiology: 16:31.","downloads_month":150,"downloads_total":31420,"page":"http://bioconductor.org/packages/release/bioc/html/NarrowPeaks.html"},{"authors":"Gabriela A. Merino and Elmer A. Fernandez","name":"NBSplice","license":"GPL (>=2)","tags":["Software","StatisticalMethod","AlternativeSplicing","Regression","DifferentialExpression","DifferentialSplicing","RNASeq","ImmunoOncology"],"description":"The package proposes a differential splicing evaluation\nmethod based on isoform quantification. It applies generalized\nlinear models with negative binomial distribution to infer\nchanges in isoform relative expression.","downloads_month":126,"downloads_total":1142,"page":"http://bioconductor.org/packages/release/bioc/html/NBSplice.html"},{"authors":"Mike Jiang, Greg Finak and N. Gopalakrishnan","name":"ncdfFlow","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry"],"description":"Provides HDF5 storage based methods and functions for\nmanipulation of flow cytometry data.","downloads_month":1280,"downloads_total":148778,"page":"http://bioconductor.org/packages/release/bioc/html/ncdfFlow.html"},{"authors":"Laurent Jacob","name":"NCIgraph","license":"GPL-3","tags":["Pathways","GraphAndNetwork"],"description":"Provides various methods to load the pathways from the NCI\nPathways Database in R graph objects and to re-format them.","downloads_month":111,"downloads_total":41282,"page":"http://bioconductor.org/packages/release/bioc/html/NCIgraph.html"},{"authors":"Florian Auer <florian.auer@informatik.uni-augsburg.de>, Frank\nKramer <frank.kramer@informatik.uni-augsburg.de>, Alex Ishkin\n<aleksandr.ishkin@thomsonreuters.com> and Dexter Pratt\n<depratt@ucsc.edu>","name":"ndexr","license":"BSD","tags":["Pathways","DataImport","Network"],"description":"This package offers an interface to NDEx servers, e.g. the\npublic server at http://ndexbio.org/. It can retrieve and save\nnetworks via the API. Networks are offered as RCX object and as\nigraph representation.","downloads_month":186,"downloads_total":4422,"page":"http://bioconductor.org/packages/release/bioc/html/ndexr.html"},{"authors":"Sahar Ansari <saharansari@wayne.edu> and Sorin Draghici\n<sorin@wayne.edu>","name":"NeighborNet","license":"CC BY-NC-ND 4.0","tags":["Software","GeneExpression","StatisticalMethod","GraphAndNetwork"],"description":"Identify the putative mechanism explaining the active\ninteractions between genes in the investigated phenotype.","downloads_month":77,"downloads_total":642,"page":"http://bioconductor.org/packages/release/bioc/html/NeighborNet.html"},{"authors":"Holger Froehlich, Florian Markowetz, Achim Tresch, Theresa\nNiederberger, Christian Bender, Matthias Maneck, Claudio\nLottaz and Tim Beissbarth","name":"nem","license":"GPL (>= 2)","tags":["Microarray","Bioinformatics","GraphsAndNetworks","Pathways","SystemsBiology","NetworkInference"],"description":"The package 'nem' allows to reconstruct features of\npathways from the nested structure of perturbation effects. It\ntakes as input (1.) a set of pathway components, which were\nperturbed, and (2.) phenotypic readout of these perturbations\n(e.g. gene expression, protein expression). The output is a\ndirected graph representing the phenotypic hierarchy.","downloads_month":249,"downloads_total":56176,"page":"http://bioconductor.org/packages/release/bioc/html/nem.html"},{"authors":"Pau Bellot, Catharina Olsen and Patrick Meyer","name":"netbenchmark","license":"CC BY-NC-SA 4.0","tags":["Microarray","GraphAndNetwork","Network","NetworkInference","GeneExpression"],"description":"This package implements a benchmarking of several gene\nnetwork inference algorithms from gene expression data.","downloads_month":138,"downloads_total":14140,"page":"http://bioconductor.org/packages/release/bioc/html/netbenchmark.html"},{"authors":"Shailesh tripathi and Frank Emmert-Streib","name":"netbiov","license":"GPL (>= 2)","tags":["GraphAndNetwork","Network","Software","Visualization"],"description":"A package that provides an effective visualization of\nlarge biological networks","downloads_month":170,"downloads_total":17668,"page":"http://bioconductor.org/packages/release/bioc/html/netbiov.html"},{"authors":"Nicolas Staedler and Frank Dondelinger","name":"nethet","license":"GPL-2","tags":["Clustering","GraphAndNetwork"],"description":"Package nethet is an implementation of statistical solid\nmethodology enabling the analysis of network heterogeneity from\nhigh-dimensional data. It combines several implementations of\nrecent statistical innovations useful for estimation and\ncomparison of networks in a heterogeneous, high-dimensional\nsetting. In particular, we provide code for formal two-sample\ntesting in Gaussian graphical models (differential network and\nGGM-GSA; Stadler and Mukherjee, 2013, 2014) and make a novel\nnetwork-based clustering algorithm available (mixed graphical\nlasso, Stadler and Mukherjee, 2013).","downloads_month":165,"downloads_total":13108,"page":"http://bioconductor.org/packages/release/bioc/html/nethet.html"},{"authors":"Ahmed Mohamed <mohamed@kuicr.kyoto-u.ac.jp>, Tim Hancock\n<timothy.hancock@kuicr.kyoto-u.ac.jp>, Ichigaku Takigawa\n<takigawa@kuicr.kyoto-u.ac.jp> and Nicolas Wicker\n<nicolas.wicker@unistra.fr>","name":"NetPathMiner","license":"GPL (>= 2)","tags":["GraphAndNetwork","Pathways","Network","Clustering","Classification"],"description":"NetPathMiner is a general framework for network path\nmining using genome-scale networks. It constructs networks from\nKGML, SBML and BioPAX files, providing three network\nrepresentations, metabolic, reaction and gene representations.\nNetPathMiner finds active paths and applies machine learning\nmethods to summarize found paths for easy interpretation. It\nalso provides static and interactive visualizations of networks\nand paths to aid manual investigation.","downloads_month":161,"downloads_total":23196,"page":"http://bioconductor.org/packages/release/bioc/html/NetPathMiner.html"},{"authors":"Fabian Schmich","name":"netprioR","license":"GPL-3","tags":["ImmunoOncology","CellBasedAssays","Preprocessing","Network"],"description":"A model for semi-supervised prioritisation of genes\nintegrating network data, phenotypes and additional prior\nknowledge about TP and TN gene labels from the literature or\nexperts.","downloads_month":135,"downloads_total":6304,"page":"http://bioconductor.org/packages/release/bioc/html/netprioR.html"},{"authors":"Simon Dirmeier [aut and cre]","name":"netReg","license":"GPL-3 | BSL-1.0 + file LICENSE","tags":["Software","StatisticalMethod","Regression","FeatureExtraction","Network","GraphAndNetwork"],"description":"netReg fits linear regression models using\nnetwork-penalization. Graph prior knowledge, in the form of\nbiological networks, is being incorporated into the loss\nfunction of the linear model. The networks describe biological\nrelationships such as co-regulation or dependency of the same\ntranscription factors/metabolites/etc. yielding a part sparse\nand part smooth solution for coefficient profiles.","downloads_month":124,"downloads_total":4822,"page":"http://bioconductor.org/packages/release/bioc/html/netReg.html"},{"authors":"Leo Lahti, Olli-Pekka Huovilainen, Antonio Gusmao and Juuso\nParkkinen","name":"netresponse","license":"GPL (>=2)","tags":["CellBiology","Clustering","GeneExpression","Genetics","Network","GraphAndNetwork","DifferentialExpression","Microarray","Transcription"],"description":"Algorithms for functional network analysis. Includes an\nimplementation of a variational Dirichlet process Gaussian\nmixture model for nonparametric mixture modeling.","downloads_month":143,"downloads_total":35266,"page":"http://bioconductor.org/packages/release/bioc/html/netresponse.html"},{"authors":"Jing Wang <jing.wang.2@vanderbilt.edu>","name":"NetSAM","license":"LGPL","tags":["Visualization","Network"],"description":"The NetSAM (Network Seriation and Modularization) package\ntakes an edge-list representation of a network as an input,\nperforms network seriation and modularization analysis, and\ngenerates as files that can be used as an input for the\none-dimensional network visualization tool NetGestalt\n(http://www.netgestalt.org) or other network analysis.","downloads_month":144,"downloads_total":22280,"page":"http://bioconductor.org/packages/release/bioc/html/NetSAM.html"},{"authors":"Jonathan Ronen [aut, cre] and Altuna Akalin [aut]","name":"netSmooth","license":"GPL-3","tags":["Network","GraphAndNetwork","SingleCell","RNASeq","GeneExpression","Sequencing","Transcriptomics","Normalization","Preprocessing","Clustering","DimensionReduction"],"description":"netSmooth is an R package for network smoothing of single\ncell RNA sequencing data. Using bio networks such as\nprotein-protein interactions as priors for gene co-expression,\nnetsmooth improves cell type identification from noisy, sparse\nscRNAseq data.","downloads_month":173,"downloads_total":2378,"page":"http://bioconductor.org/packages/release/bioc/html/netSmooth.html"},{"authors":"Chris Fraley, Wm. Chad Young, Ling-Hong Hung, Kaiyuan Shi, Ka\nYee Yeung and Adrian Raftery (with contributions from Kenneth Lo)","name":"networkBMA","license":"GPL (>= 2)","tags":["GraphsAndNetwork","NetworkInference","GeneExpression","GeneTarget","Network","Bayesian"],"description":"An extension of Bayesian Model Averaging (BMA) for network\nconstruction using time series gene expression data. Includes\nassessment functions and sample test data.","downloads_month":140,"downloads_total":31672,"page":"http://bioconductor.org/packages/release/bioc/html/networkBMA.html"},{"authors":"Xiaobei Zhao [aut, cre and cph]","name":"NGScopy","license":"GPL (>=2)","tags":["ImmunoOncology","CopyNumberVariation","DNASeq","TargetedResequencing","ExomeSeq","WholeGenome","Sequencing"],"description":"NGScopy provides a quantitative caller for detecting copy\nnumber variations in next generation sequencing (NGS),\nincluding whole genome sequencing (WGS), whole exome sequencing\n(WES) and targeted panel sequencing (TPS). The caller can be\nparallelized by chromosomes to use multiple processors/cores on\none computer.","downloads_month":152,"downloads_total":15818,"page":"http://bioconductor.org/packages/release/bioc/html/NGScopy.html"},{"authors":"Adi Laurentiu Tarca <atarca@med.wayne.edu>","name":"nnNorm","license":"LGPL","tags":["Microarray","TwoChannel","Preprocessing"],"description":"This package allows to detect and correct for spatial and\nintensity biases with two-channel microarray data. The\nnormalization method implemented in this package is based on\nrobust neural networks fitting.","downloads_month":130,"downloads_total":44374,"page":"http://bioconductor.org/packages/release/bioc/html/nnNorm.html"},{"authors":"Sonia Tarazona, Pedro Furio-Tari, Maria Jose Nueda, Alberto\nFerrer and Ana Conesa","name":"NOISeq","license":"Artistic-2.0","tags":["ImmunoOncology","RNASeq","DifferentialExpression","Visualization","Sequencing"],"description":"Analysis of RNA-seq expression data or other similar kind\nof data. Exploratory plots to evualuate saturation, count\ndistribution, expression per chromosome, type of detected\nfeatures, features length, etc. Differential expression between\ntwo experimental conditions with no parametric assumptions.","downloads_month":1002,"downloads_total":98514,"page":"http://bioconductor.org/packages/release/bioc/html/NOISeq.html"},{"authors":"Matthew N. McCall <mccallm@gmail.com> and Valeriia Sherina\n<valery.sherina@gmail.com>","name":"nondetects","license":"GPL-3","tags":["Software","AssayDomain","GeneExpression","Technology","qPCR","WorkflowStep","Preprocessing"],"description":"Methods to model and impute non-detects in the results of\nqPCR experiments.","downloads_month":167,"downloads_total":20670,"page":"http://bioconductor.org/packages/release/bioc/html/nondetects.html"},{"authors":"Jonathan Alexander Heiss","name":"normalize450K","license":"BSD_2_clause + file LICENSE","tags":["Normalization","DNAMethylation","Microarray","TwoChannel","Preprocessing","MethylationArray"],"description":"Precise measurements are important for epigenome-wide\nstudies investigating DNA methylation in whole blood samples,\nwhere effect sizes are expected to be small in magnitude. The\n450K platform is often affected by batch effects and proper\npreprocessing is recommended. This package provides functions\nto read and normalize 450K '.idat' files. The normalization\ncorrects for dye bias and biases related to signal intensity\nand methylation of probes using local regression. No adjustment\nfor probe type bias is performed to avoid the trade-off of\nprecision for accuracy of beta-values.","downloads_month":124,"downloads_total":7484,"page":"http://bioconductor.org/packages/release/bioc/html/normalize450K.html"},{"authors":"Jakob Willforss","name":"NormalyzerDE","license":"Artistic-2.0","tags":["Normalization","MultipleComparison","Visualization","Bayesian","Proteomics","Metabolomics","DifferentialExpression"],"description":"NormalyzerDE provides screening of normalization methods\nfor LC-MS based expression data. It calculates a range of\nnormalized matrices using both existing approaches and a novel\ntime-segmented approach, calculates performance measures and\ngenerates an evaluation report. Furthermore, it provides an\neasy utility for Limma- or ANOVA- based differential expression\nanalysis.","downloads_month":139,"downloads_total":1050,"page":"http://bioconductor.org/packages/release/bioc/html/NormalyzerDE.html"},{"authors":"Matthias Kohl, James Perkins and Nor Izayu Abdul Rahman","name":"NormqPCR","license":"LGPL-3","tags":["MicrotitrePlateAssay","GeneExpression","qPCR"],"description":"Functions for the selection of optimal reference genes and\nthe normalisation of real-time quantitative PCR data.","downloads_month":477,"downloads_total":56266,"page":"http://bioconductor.org/packages/release/bioc/html/NormqPCR.html"},{"authors":"Johannes Helmuth [aut, cre] and Ho-Ryun Chung [aut]","name":"normr","license":"GPL-2","tags":["Bayesian","DifferentialPeakCalling","Classification","DataImport","ChIPSeq","RIPSeq","FunctionalGenomics","Genetics","MultipleComparison","Normalization","PeakDetection","Preprocessing","Alignment"],"description":"Robust normalization and difference calling procedures for\nChIP-seq and alike data. Read counts are modeled jointly as a\nbinomial mixture model with a user-specified number of\ncomponents. A fitted background estimate accounts for the\neffect of enrichment in certain regions and, therefore,\nrepresents an appropriate null hypothesis. This robust\nbackground is used to identify significantly enriched or\ndepleted regions.","downloads_month":174,"downloads_total":7568,"page":"http://bioconductor.org/packages/release/bioc/html/normr.html"},{"authors":"Jessica Larson and Art Owen","name":"npGSEA","license":"Artistic-2.0","tags":["GeneSetEnrichment","Microarray","StatisticalMethod","Pathways"],"description":"Current gene set enrichment methods rely upon permutations\nfor inference.  These approaches are computationally expensive\nand have minimum achievable p-values based on the number of\npermutations, not on the actual observed statistics.  We have\nderived three parametric approximations to the permutation\ndistributions of two gene set enrichment test statistics.  We\nare able to reduce the computational burden and granularity\nissues of permutation testing with our method, which is\nimplemented in this package. npGSEA calculates gene set\nenrichment statistics and p-values without the computational\ncost of permutations.  It is applicable in settings where one\nor many gene sets are of interest.  There are also built-in\nplotting functions to help users visualize results.","downloads_month":161,"downloads_total":21820,"page":"http://bioconductor.org/packages/release/bioc/html/npGSEA.html"},{"authors":"Wei Xiao, Yin Jin, Darong Lai, Xinyi Yang, Yuanhua Liu and Christine Nardini","name":"NTW","license":"GPL-2","tags":["Preprocessing"],"description":"This package predicts the gene-gene interaction network\nand identifies the direct transcriptional targets of the\nperturbation using an ODE (Ordinary Differential Equation)\nbased method.","downloads_month":124,"downloads_total":32522,"page":"http://bioconductor.org/packages/release/bioc/html/NTW.html"},{"authors":"Rawane Samb [aut], Astrid Deschênes [cre, aut], Pascal Belleau\n[aut] and Arnaud Droit [aut]","name":"nucleoSim","license":"Artistic-2.0","tags":["Genetics","Sequencing","Software","StatisticalMethod","Alignment"],"description":"This package can generate a synthetic map with reads\ncovering the nucleosome regions as well as a synthetic map with\nforward and reverse reads emulating next-generation sequencing.\nThe user has choice between three different distributions for\nthe read positioning: Normal, Student and Uniform.","downloads_month":129,"downloads_total":8428,"page":"http://bioconductor.org/packages/release/bioc/html/nucleoSim.html"},{"authors":"Oscar Flores and Ricard Illa","name":"nucleR","license":"LGPL (>= 3)","tags":["NucleosomePositioning","Coverage","ChIPSeq","Microarray","Sequencing","Genetics","QualityControl","DataImport"],"description":"Nucleosome positioning for Tiling Arrays and NGS\nexperiments.","downloads_month":193,"downloads_total":37432,"page":"http://bioconductor.org/packages/release/bioc/html/nucleR.html"},{"authors":"Hiroaki Kato and Takeshi Urano","name":"nuCpos","license":"GPL-2","tags":["Genetics","Epigenetics","NucleosomePositioning","HiddenMarkovModel","ImmunoOncology"],"description":"nuCpos, a derivative of NuPoP, is an R package for\nprediction of nucleosome positions. In nuCpos, a duration\nhidden Markov model is trained with a chemical map of\nnucleosomes either from budding yeast, fission yeast, or mouse\nembryonic stem cells. nuCpos outputs the Viterbi (most\nprobable) path of nucleosome-linker states, predicted\nnucleosome occupancy scores and histone binding affinity (HBA)\nscores as NuPoP does. nuCpos can also calculate local and whole\nnucleosomal HBA scores for a given 147-bp sequence.\nFurthermore, effect of genetic alterations on nucleosome\noccupancy can be predicted with this package. The parental\npackage NuPoP, which is based on an MNase-seq-based map of\nbudding yeast nucleosomes, was developed by Ji-Ping Wang and\nLiqun Xi, licensed under GPL-2.","downloads_month":87,"downloads_total":644,"page":"http://bioconductor.org/packages/release/bioc/html/nuCpos.html"},{"authors":"Ji-Ping Wang <jzwang@northwestern.edu>; Liqun Xi\n<lxi700@northwestern.edu>","name":"NuPoP","license":"GPL-2","tags":["Genetics","Visualization","Classification"],"description":"NuPoP is an R package for Nucleosome Positioning\nPrediction.This package is built upon a duration hidden Markov\nmodel proposed in Xi et al, 2010; Wang et al, 2008. The core of\nthe package was written in Fotran. In addition to the R\npackage, a stand-alone Fortran software tool is also available\nat http://nucleosome.stats.northwestern.edu.","downloads_month":142,"downloads_total":34230,"page":"http://bioconductor.org/packages/release/bioc/html/NuPoP.html"},{"authors":"Oliver Will <oliverrreader@gmail.com>","name":"occugene","license":"GPL (>= 2)","tags":["Annotation","Pathways"],"description":"Statistical tools for building random mutagenesis\nlibraries for prokaryotes. The package has functions for\nhandling the occupancy distribution for a multinomial and for\nestimating the number of essential genes in random transposon\nmutagenesis libraries.","downloads_month":122,"downloads_total":40320,"page":"http://bioconductor.org/packages/release/bioc/html/occugene.html"},{"authors":"Yudi Pawitan <Yudi.Pawitan@ki.se> and Alexander Ploner\n<Alexander.Ploner@ki.se>","name":"OCplus","license":"LGPL","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"This package allows to characterize the operating\ncharacteristics of a microarray experiment, i.e. the trade-off\nbetween false discovery rate and the power to detect truly\nregulated genes. The package includes tools both for planned\nexperiments (for sample size assessment) and for already\ncollected data (identification of differentially expressed\ngenes).","downloads_month":244,"downloads_total":61040,"page":"http://bioconductor.org/packages/release/bioc/html/OCplus.html"},{"authors":"José Jiménez","name":"odseq","license":"MIT + file LICENSE","tags":["Alignment","MultipleSequenceAlignment"],"description":"Performs outlier detection of sequences in a multiple\nsequence alignment using bootstrap of predefined distance\nmetrics. Outlier sequences can make downstream analyses\nunreliable or make the alignments less accurate while they are\nbeing constructed. This package implements the OD-seq algorithm\nproposed by Jehl et al (doi 10.1186/s12859-015-0702-1) for\naligned sequences and a variant using string kernels for\nunaligned sequences.","downloads_month":124,"downloads_total":7596,"page":"http://bioconductor.org/packages/release/bioc/html/odseq.html"},{"authors":"Michael F. Ochs <ochsm@tcnj.edu>","name":"OGSA","license":"GPL (== 2)","tags":["GeneExpression","Microarray","CopyNumberVariation"],"description":"OGSA provides a global estimate of pathway deregulation in\ncancer subtypes by integrating the estimates of significance\nfor individual pathway members that have been identified by\noutlier analysis.","downloads_month":138,"downloads_total":10174,"page":"http://bioconductor.org/packages/release/bioc/html/OGSA.html"},{"authors":"Benilton Carvalho and Rafael Irizarry","name":"oligo","license":"LGPL (>= 2)","tags":["Microarray","OneChannel","TwoChannel","Preprocessing","SNP","DifferentialExpression","ExonArray","GeneExpression","DataImport"],"description":"A package to analyze oligonucleotide arrays\n(expression/SNP/tiling/exon) at probe-level. It currently\nsupports Affymetrix (CEL files) and NimbleGen arrays (XYS\nfiles).","downloads_month":3110,"downloads_total":392830,"page":"http://bioconductor.org/packages/release/bioc/html/oligo.html"},{"authors":"Benilton Carvalho and Robert Scharpf","name":"oligoClasses","license":"GPL (>= 2)","tags":["Infrastructure"],"description":"This package contains class definitions, validity checks,\nand initialization methods for classes used by the oligo and\ncrlmm packages.","downloads_month":2954,"downloads_total":367248,"page":"http://bioconductor.org/packages/release/bioc/html/oligoClasses.html"},{"authors":"Matthias Futschik <mfutschik@ualg.pt>","name":"OLIN","license":"GPL-2","tags":["Microarray","TwoChannel","QualityControl","Preprocessing","Visualization"],"description":"Functions for normalisation of two-color microarrays by\noptimised local regression and for detection of artefacts in\nmicroarray data","downloads_month":134,"downloads_total":47132,"page":"http://bioconductor.org/packages/release/bioc/html/OLIN.html"},{"authors":"Matthias Futschik <mfutschik@ualg.pt>","name":"OLINgui","license":"GPL-2","tags":["Microarray","TwoChannel","QualityControl","Preprocessing","Visualization"],"description":"Graphical user interface for the OLIN package","downloads_month":143,"downloads_total":44112,"page":"http://bioconductor.org/packages/release/bioc/html/OLINgui.html"},{"authors":"Klara Kaleb","name":"OmaDB","license":"GPL-3","tags":["Software","ComparativeGenomics","FunctionalGenomics","Genetics","Annotation","GO","FunctionalPrediction"],"description":"A package for the orthology prediction data download from\nOMA database.","downloads_month":160,"downloads_total":2204,"page":"http://bioconductor.org/packages/release/bioc/html/OmaDB.html"},{"authors":"Chen Meng, Aedin Culhane and Amin M. Gholami.","name":"omicade4","license":"GPL-2","tags":["Software","Clustering","Classification","MultipleComparison"],"description":"Multiple co-inertia analysis of omics datasets","downloads_month":160,"downloads_total":27848,"page":"http://bioconductor.org/packages/release/bioc/html/omicade4.html"},{"authors":"Ying Hu <yhu@mail.nih.gov> Chunhua Yan <yanch@mail.nih.gov>","name":"OmicCircos","license":"GPL-2","tags":["Visualization","Statistics","Annotation"],"description":"OmicCircos is an R application and package for generating\nhigh-quality circular plots for omics data.","downloads_month":353,"downloads_total":48362,"page":"http://bioconductor.org/packages/release/bioc/html/OmicCircos.html"},{"authors":"Daniel Giguere [aut, cre], Jean Macklaim [aut] and Greg Gloor\n[aut]","name":"omicplotR","license":"MIT + file LICENSE","tags":["Software","DifferentialExpression","GeneExpression","GUI","RNASeq","DNASeq","Metagenomics","Transcriptomics","Bayesian","Microbiome","Visualization","Sequencing"],"description":"A Shiny app for visual exploration of omic datasets as\ncompositions, and differential abundance analysis using ALDEx2.\nUseful for exploring RNA-seq, meta-RNA-seq, 16s rRNA gene\nsequencing with visualizations such as principal component\nanalysis biplots (coloured using metadata for visualizing each\nvariable), dendrograms and stacked bar plots, and effect plots\n(ALDEx2). Input is a table of counts and metadata file (if\nmetadata exists), with options to filter data by count or by\nmetadata to remove low counts, or to visualize select samples\naccording to selected metadata.","downloads_month":170,"downloads_total":2336,"page":"http://bioconductor.org/packages/release/bioc/html/omicplotR.html"},{"authors":"Carles Hernandez-Ferrer [aut, cre] and Juan R. González [aut]","name":"omicRexposome","license":"MIT + file LICENSE","tags":["ImmunoOncology","WorkflowStep","MultipleComparison","Visualization","GeneExpression","DifferentialExpression","DifferentialMethylation","GeneRegulation","Epigenetics","Proteomics","Transcriptomics","StatisticalMethod","Regression"],"description":"omicRexposome systematizes the association evaluation\nbetween exposures and omic data, taking advantage of\nMultiDataSet for coordinated data management, rexposome for\nexposome data definition and limma for association testing.\nAlso to perform data integration mixing exposome and omic data\nusing multi co-inherent analysis (omicade4) and multi-canonical\ncorrelation analysis (PMA).","downloads_month":175,"downloads_total":3138,"page":"http://bioconductor.org/packages/release/bioc/html/omicRexposome.html"},{"authors":"Charles E. Determan Jr. <cdetermanjr@gmail.com>","name":"OmicsMarkeR","license":"GPL-3","tags":["Metabolomics","Classification","FeatureExtraction"],"description":"Tools for classification and feature selection for 'omics'\nlevel datasets.  It is a tool to provide multiple multivariate\nclassification and feature selection techniques complete with\nmultiple stability metrics and aggregation techniques.  It is\nprimarily designed for analysis of metabolomics datasets but\npotentially extendable to proteomics and transcriptomics\napplications.","downloads_month":184,"downloads_total":14690,"page":"http://bioconductor.org/packages/release/bioc/html/OmicsMarkeR.html"},{"authors":"Subhadeep Das [aut, cre] and Dr. Sucheta Tripathy [ctb]","name":"OMICsPCA","license":"GPL-3","tags":["ImmunoOncology","MultipleComparison","PrincipalComponent","DataRepresentation","Workflow","Visualization","DimensionReduction","Clustering","BiologicalQuestion","EpigeneticsWorkflow","Transcription","GeneticVariability","GUI","BiomedicalInformatics","Epigenetics","FunctionalGenomics","SingleCell"],"description":"OMICsPCA is an analysis pipeline designed to integrate\nmulti OMICs experiments done on various subjects (e.g. Cell\nlines, individuals), treatments (e.g. disease/control) or time\npoints and to analyse such integrated data from various various\nangles and perspectives. In it's core OMICsPCA uses Principal\nComponent Analysis (PCA) to integrate multiomics experiments\nfrom various sources and thus has ability to over data\ninsufficiency issues by using the ingegrated data as\nrepresentatives. OMICsPCA can be used in various application\nincluding analysis of overall distribution of OMICs assays\nacross various samples /individuals /time points; grouping\nassays by user-defined conditions; identification of source of\nvariation, similarity/dissimilarity between assays, variables\nor individuals.","downloads_month":98,"downloads_total":638,"page":"http://bioconductor.org/packages/release/bioc/html/OMICsPCA.html"},{"authors":"Maarten van Iterson [aut] and Davy Cats [cre]","name":"omicsPrint","license":"GPL (>= 2)","tags":["QualityControl","Genetics","Epigenetics","Transcriptomics","DNAMethylation","Transcription","GeneticVariability","ImmunoOncology"],"description":"omicsPrint provides functionality for cross omic genetic\nfingerprinting, for example, to verify sample relationships\nbetween multiple omics data types, i.e. genomic, transcriptomic\nand epigenetic (DNA methylation).","downloads_month":166,"downloads_total":2416,"page":"http://bioconductor.org/packages/release/bioc/html/omicsPrint.html"},{"authors":"Eugenia Galeota","name":"Onassis","license":"GPL-2","tags":["Annotation","DataImport","Clustering","Network","Software","GeneTarget"],"description":"A package that allows the annotation of text with ontology\nterms (mainly from OBO ontologies) and the computation of\nsemantic similarity measures based on the structure of the\nontology between different annotated samples.","downloads_month":119,"downloads_total":3478,"page":"http://bioconductor.org/packages/release/bioc/html/Onassis.html"},{"authors":"Daniel Pique, John Greally and Jessica Mar","name":"oncomix","license":"GPL-3","tags":["GeneExpression","Sequencing"],"description":"This package helps identify mRNAs that are overexpressed\nin subsets of tumors relative to normal tissue. Ideal inputs\nwould be paired tumor-normal data from the same tissue from\nmany patients (>15 pairs). This unsupervised approach relies on\nthe observation that oncogenes are characteristically\noverexpressed in only a subset of tumors in the population, and\nmay help identify oncogene candidates purely based on\ndifferences in mRNA expression between previously unknown\nsubtypes.","downloads_month":132,"downloads_total":3176,"page":"http://bioconductor.org/packages/release/bioc/html/oncomix.html"},{"authors":"Luca De Sano [aut], Carlo Gambacorti Passerini [ctb], Rocco\nPiazza [ctb], Daniele Ramazzotti [aut, cre] and Roberta Spinelli\n[ctb]","name":"OncoScore","license":"file LICENSE","tags":["BiomedicalInformatics"],"description":"OncoScore is a tool to measure the association of genes to\ncancer based on citation frequencies in biomedical literature.\nThe score is evaluated from PubMed literature by dynamically\nupdatable web queries.","downloads_month":139,"downloads_total":9184,"page":"http://bioconductor.org/packages/release/bioc/html/OncoScore.html"},{"authors":"Ramon Diaz-Uriarte [aut, cre] and Mark Taylor [ctb]","name":"OncoSimulR","license":"GPL (>= 3)","tags":["BiologicalQuestion","SomaticMutation"],"description":"Functions for forward population genetic simulation in\nasexual populations, with special focus on cancer progression.\nFitness can be an arbitrary function of genetic interactions\nbetween multiple genes or modules of genes, including\nepistasis, order restrictions in mutation accumulation, and\norder effects.  Mutation rates can differ between genes, and we\ncan include mutator/antimutator genes (to model mutator\nphenotypes). Simulations use continuous-time models and can\ninclude driver and passenger genes and modules.  Also included\nare functions for: simulating random DAGs of the type found in\nOncogenetic Tress, Conjunctive Bayesian Networks, and other\ntumor progression models; plotting and sampling from single or\nmultiple realizations of the simulations, including single-cell\nsampling; plotting the parent-child relationships of the\nclones; generating random fitness landscapes (Rough Mount Fuji,\nHouse of Cards, and additive models) and plotting them.","downloads_month":189,"downloads_total":16368,"page":"http://bioconductor.org/packages/release/bioc/html/OncoSimulR.html"},{"authors":"Cheng Yang, Evan Newell and Yong Kee Tan","name":"oneSENSE","license":"GPL (>=3)","tags":["ImmunoOncology","Software","FlowCytometry","GUI","DimensionReduction"],"description":"A graphical user interface that facilitates the\ndimensional reduction method based on the t-distributed\nstochastic neighbor embedding (t-SNE) algorithm, for\ncategorical analysis of mass cytometry data. With One-SENSE,\nmeasured parameters are grouped into predefined categories, and\ncells are projected onto a space composed of one dimension for\neach category. Each dimension is informative and can be\nannotated through the use of heatplots aligned in parallel to\neach axis, allowing for simultaneous visualization of two\ncatergories across a two-dimensional plot. The cellular\noccupancy of the resulting plots alllows for direct assessment\nof the relationships between the categories.","downloads_month":154,"downloads_total":3434,"page":"http://bioconductor.org/packages/release/bioc/html/oneSENSE.html"},{"authors":"David Robertson [aut, cre], Adel Javanmard [aut], Andrea\nMontanari [aut] and Natasha Karp [aut]","name":"onlineFDR","license":"GPL-3","tags":["MultipleComparison","Software","StatisticalMethod"],"description":"This package allows users to control the false discovery\nrate for online hypothesis testing, where hypotheses arrive\nsequentially in a stream, as presented by Javanmard and\nMontanari (2015, 2018) and Ramdas et al. (2017). In this\nframework, a null hypothesis is rejected based only on the\nprevious decisions, as the future p-values and the number of\nhypotheses to be tested are unknown.","downloads_month":76,"downloads_total":658,"page":"http://bioconductor.org/packages/release/bioc/html/onlineFDR.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu>","name":"ontoProc","license":"Artistic-2.0","tags":["Infrastructure","GO"],"description":"Support harvesting of diverse bioinformatic ontologies,\nmaking particular use of the ontologyIndex package on CRAN. We\nprovide snapshots of key ontologies for terms about cells, cell\nlines, chemical compounds, and anatomy, to help analyze\ngenome-scale experiments, particularly cell x compound screens.\nAnother purpose is to strengthen development of compelling use\ncases for richer interfaces to emerging ontologies.","downloads_month":139,"downloads_total":3358,"page":"http://bioconductor.org/packages/release/bioc/html/ontoProc.html"},{"authors":"Mike Jiang, John Ramey, Greg Finak and Raphael Gottardo","name":"openCyto","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","DataImport","Preprocessing","DataRepresentation"],"description":"This package is designed to facilitate the automated\ngating methods in sequential way to mimic the manual gating\nstrategy.","downloads_month":649,"downloads_total":73798,"page":"http://bioconductor.org/packages/release/bioc/html/openCyto.html"},{"authors":"Matthias Döring [aut, cre] and Nico Pfeifer [aut]","name":"openPrimeR","license":"GPL-2","tags":["Software","Technology"],"description":"An implementation of methods for designing, evaluating,\nand comparing primer sets for multiplex PCR. Primers are\ndesigned by solving a set cover problem such that the number of\ncovered template sequences is maximized with the smallest\npossible set of primers. To guarantee that high-quality primers\nare generated, only primers fulfilling constraints on their\nphysicochemical properties are selected. A Shiny app providing\na user interface for the functionalities of this package is\nprovided by the 'openPrimeRui' package.","downloads_month":168,"downloads_total":4246,"page":"http://bioconductor.org/packages/release/bioc/html/openPrimeR.html"},{"authors":"Matthias Döring [aut, cre] and Nico Pfeifer [aut]","name":"openPrimeRui","license":"GPL-2","tags":["Software","Technology"],"description":"A Shiny application providing methods for designing,\nevaluating, and comparing primer sets for multiplex polymerase\nchain reaction. Primers are designed by solving a set cover\nproblem such that the number of covered template sequences is\nmaximized with the smallest possible set of primers. To\nguarantee that high-quality primers are generated, only primers\nfulfilling constraints on their physicochemical properties are\nselected.","downloads_month":128,"downloads_total":3168,"page":"http://bioconductor.org/packages/release/bioc/html/openPrimeRui.html"},{"authors":"Henry Loeffler-Wirth <wirth@izbi.uni-leipzig.de>, Hoang Thanh\nLe <le@izbi.uni-leipzig.de> and Martin Kalcher\n<mkalcher@porkbox.net>","name":"oposSOM","license":"GPL (>=2)","tags":["GeneExpression","DifferentialExpression","GeneSetEnrichment","DataRepresentation","Visualization"],"description":"This package translates microarray expression data into\nmetadata of reduced dimension. It provides various\nsample-centered and group-centered visualizations, sample\nsimilarity analyses and functional enrichment analyses. The\nunderlying SOM algorithm combines feature clustering,\nmultidimensional scaling and dimension reduction, along with\nstrong visualization capabilities. It enables extraction and\ndescription of functional expression modules inherent in the\ndata.","downloads_month":179,"downloads_total":18082,"page":"http://bioconductor.org/packages/release/bioc/html/oposSOM.html"},{"authors":"Chenwei Wang [aut], Alperen Taciroglu [aut], Stefan R Maetschke\n[aut], Colleen C Nelson [aut], Mark Ragan [aut], Melissa Davis\n[aut], Soroor Hediyeh zadeh [cre] and Momeneh Foroutan [ctr]","name":"oppar","license":"GPL-2","tags":["Pathways","GeneSetEnrichment","SystemsBiology","GeneExpression","Software"],"description":"The R implementation of mCOPA package published by Wang et\nal. (2012). Oppar provides methods for Cancer Outlier profile\nAnalysis. Although initially developed to detect outlier genes\nin cancer studies, methods presented in oppar can be used for\noutlier profile analysis in general. In addition, tools are\nprovided for gene set enrichment and pathway analysis.","downloads_month":137,"downloads_total":7790,"page":"http://bioconductor.org/packages/release/bioc/html/oppar.html"},{"authors":"Mohamad Hasan [aut, cre] and Paul Schliekelman [aut]","name":"OPWeight","license":"Artistic-2.0","tags":["ImmunoOncology","BiomedicalInformatics","MultipleComparison","Regression","RNASeq","SNP"],"description":"This package perform weighted-pvalue based multiple\nhypothesis test and provides corresponding information such as\nranking probability, weight, significant tests, etc . To\nconduct this testing procedure, the testing method apply a\nprobabilistic relationship between the test rank and the\ncorresponding test effect size.","downloads_month":391,"downloads_total":10696,"page":"http://bioconductor.org/packages/release/bioc/html/OPWeight.html"},{"authors":"Xinan Yang, Stefanie Scheid and Claudio Lottaz","name":"OrderedList","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"Detection of similarities between ordered lists of genes.\nThereby, either simple lists can be compared or gene expression\ndata can be used to deduce the lists. Significance of\nsimilarities is evaluated by shuffling lists or by resampling\nin microarray data, respectively.","downloads_month":283,"downloads_total":68378,"page":"http://bioconductor.org/packages/release/bioc/html/OrderedList.html"},{"authors":"Kornel Labun [aut, cre, cph], Haakon Tjeldnes [aut, dtc], Katarzyna Chyzynska [ctb, dtc], Evind Valen [ths and fnd]","name":"ORFik","license":"MIT + file LICENSE","tags":["ImmunoOncology","Software","Sequencing","RiboSeq","RNASeq","FunctionalGenomics","Coverage","Alignment","DataImport"],"description":"Tools for manipulation of RiboSeq, RNASeq and CageSeq\ndata. ORFik is extremely fast through use of C, data.table and\nGenomicRanges. Package allows to reassign starts of the\ntranscripts with the use of CageSeq data, automatic shifting of\nRiboSeq reads, finding of Open Reading Frames for whole genomes\nand much more.","downloads_month":180,"downloads_total":2810,"page":"http://bioconductor.org/packages/release/bioc/html/ORFik.html"},{"authors":"Martin Morgan [aut, cre] and Yubo Cheng [ctb]","name":"Organism.dplyr","license":"Artistic-2.0","tags":["Annotation","Sequencing","GenomeAnnotation"],"description":"This package provides an alternative interface to\nBioconductor 'annotation' resources, in particular the gene\nidentifier mapping functionality of the 'org' packages (e.g.,\norg.Hs.eg.db) and the genome coordinate functionality of the\n'TxDb' packages (e.g., TxDb.Hsapiens.UCSC.hg38.knownGene).","downloads_month":474,"downloads_total":13296,"page":"http://bioconductor.org/packages/release/bioc/html/Organism.dplyr.html"},{"authors":"Marc Carlson, Hervé Pagès, Martin Morgan and Valerie Obenchain","name":"OrganismDbi","license":"Artistic-2.0","tags":["Annotation","Infrastructure"],"description":"The package enables a simple unified interface to several\nannotation packages each of which has its own schema by taking\nadvantage of the fact that each of these packages implements a\nselect methods.","downloads_month":3504,"downloads_total":307196,"page":"http://bioconductor.org/packages/release/bioc/html/OrganismDbi.html"},{"authors":"Li Yan","name":"OSAT","license":"Artistic-2.0","tags":["DataRepresentation","Visualization","ExperimentalDesign","QualityControl"],"description":"A sizable genomics study such as microarray often involves\nthe use of multiple batches (groups) of experiment due to\npractical complication. To minimize batch effects, a careful\nexperiment design should ensure the even distribution of\nbiological groups and confounding factors across batches. OSAT\n(Optimal Sample Assignment Tool) is developed to facilitate the\nallocation of collected samples to different batches. With\nminimum steps, it produces setup that optimizes the even\ndistribution of samples in groups of biological interest into\ndifferent batches, reducing the confounding or correlation\nbetween batches and the biological variables of interest. It\ncan also optimize the even distribution of confounding factors\nacross batches. Our tool can handle challenging instances where\nincomplete and unbalanced sample collections are involved as\nwell as ideal balanced RCBD. OSAT provides a number of\npredefined layout for some of the most commonly used genomics\nplatform. Related paper can be find at\nhttp://www.biomedcentral.com/1471-2164/13/689 .","downloads_month":169,"downloads_total":28600,"page":"http://bioconductor.org/packages/release/bioc/html/OSAT.html"},{"authors":"Ning Leng","name":"Oscope","license":"Artistic-2.0","tags":["ImmunoOncology","StatisticalMethod","RNASeq","Sequencing","GeneExpression"],"description":"Oscope is a statistical pipeline developed to identifying\nand recovering the base cycle profiles of oscillating genes in\nan unsynchronized single cell RNA-seq experiment. The Oscope\npipeline includes three modules: a sine model module to search\nfor candidate oscillator pairs; a K-medoids clustering module\nto cluster candidate oscillators into groups; and an extended\nnearest insertion module to recover the base cycle order for\neach oscillator group.","downloads_month":208,"downloads_total":12308,"page":"http://bioconductor.org/packages/release/bioc/html/Oscope.html"},{"authors":"Daniel Beck, Matt Settles, and James A. Foster","name":"OTUbase","license":"Artistic-2.0","tags":["Sequencing","DataImport"],"description":"Provides a platform for Operational Taxonomic Unit based\nanalysis","downloads_month":147,"downloads_total":37066,"page":"http://bioconductor.org/packages/release/bioc/html/OTUbase.html"},{"authors":"HyungJun Cho <hj4cho@korea.ac.kr>","name":"OutlierD","license":"GPL (>= 2)","tags":["Microarray"],"description":"This package detects outliers using quantile regression on\nthe M-A scatterplots of high-throughput data.","downloads_month":154,"downloads_total":48028,"page":"http://bioconductor.org/packages/release/bioc/html/OutlierD.html"},{"authors":"Felix Brechtmann [aut, cre], Christian Mertes [aut, cre], Ange\nMatuseviciute [aut, cre], Vicente Yepez [aut] and Julien Gagneur\n[aut]","name":"OUTRIDER","license":"MIT + file LICENSE","tags":["ImmunoOncology","RNASeq","Transcriptomics","Alignment","Sequencing","GeneExpression","Genetics"],"description":"Identification of aberrant gene expression in RNA-seq\ndata. Read count expectations are modeled by an autoencoder to\ncontrol for confounders in the data. Given these expectations,\nthe RNA-seq read counts are assumed to follow a negative\nbinomial distribution with a gene-specific dispersion. Outliers\nare then identified as read counts that significantly deviate\nfrom this distribution. Furthermore, OUTRIDER provides useful\nplotting functions to analyze and visualize the results.","downloads_month":131,"downloads_total":1030,"page":"http://bioconductor.org/packages/release/bioc/html/OUTRIDER.html"},{"authors":"Lulu Chen <luluchen@vt.edu>","name":"OVESEG","license":"GPL-2","tags":["Software","MultipleComparison","CellBiology","GeneExpression"],"description":"An R package for multiple-group comparison to detect\ntissue/cell-specific marker genes among subtypes. It provides\nfunctions to compute OVESEG-test statistics, derive component\nweights in the mixture null distribution model and estimate\np-values from weightedly aggregated permutations. Obtained\nposterior probabilities of component null hypotheses can also\nportrait all kinds of upregulation patterns among subtypes.","downloads_month":0,"downloads_total":64,"page":"http://bioconductor.org/packages/release/bioc/html/OVESEG.html"},{"authors":"Michael Turewicz [aut, cre], Martin Eisenacher [ctb and cre]","name":"PAA","license":"BSD_3_clause + file LICENSE","tags":["Classification","Microarray","OneChannel","Proteomics"],"description":"PAA imports single color (protein) microarray data that\nhas been saved in gpr file format - esp. ProtoArray data. After\npreprocessing (background correction, batch filtering,\nnormalization) univariate feature preselection is performed\n(e.g., using the \"minimum M statistic\" approach - hereinafter\nreferred to as \"mMs\"). Subsequently, a multivariate feature\nselection is conducted to discover biomarker candidates.\nTherefore, either a frequency-based backwards elimination\naproach or ensemble feature selection can be used. PAA provides\na complete toolbox of analysis tools including several\ndifferent plots for results examination and evaluation.","downloads_month":189,"downloads_total":20132,"page":"http://bioconductor.org/packages/release/bioc/html/PAA.html"},{"authors":"Adi Laurentiu Tarca <atarca@med.wayne.edu>; Zhonghui Xu\n<zhonghui.xu@gmail.com>","name":"PADOG","license":"GPL (>= 2)","tags":["Microarray","OneChannel","TwoChannel"],"description":"This package implements a general purpose gene set\nanalysis method called PADOG that downplays the importance of\ngenes that apear often accross the sets of genes to be\nanalyzed. The package provides also a benchmark for gene set\nanalysis methods in terms of sensitivity and ranking using 24\npublic datasets from KEGGdzPathwaysGEO package.","downloads_month":333,"downloads_total":39082,"page":"http://bioconductor.org/packages/release/bioc/html/PADOG.html"},{"authors":"Levon Demirdjian, Shihao Shen, Ying Nian Wu and Yi Xing","name":"PAIRADISE","license":"MIT + file LICENSE","tags":["RNASeq","DifferentialExpression","AlternativeSplicing","StatisticalMethod","ImmunoOncolgy"],"description":"This package implements the PAIRADISE procedure for\ndetecting differential isoform expression between matched\nreplicates in paired RNA-Seq data.","downloads_month":4,"downloads_total":82,"page":"http://bioconductor.org/packages/release/bioc/html/PAIRADISE.html"},{"authors":"Michal Burda","name":"paircompviz","license":"GPL (>=3.0)","tags":["GraphAndNetwork"],"description":"This package provides visualization of the results from\nthe multiple (i.e. pairwise) comparison tests such as\npairwise.t.test, pairwise.prop.test or pairwise.wilcox.test.\nThe groups being compared are visualized as nodes in Hasse\ndiagram. Such approach enables very clear and vivid depiction\nof which group is significantly greater than which others,\nespecially if comparing a large number of groups.","downloads_month":146,"downloads_total":22350,"page":"http://bioconductor.org/packages/release/bioc/html/paircompviz.html"},{"authors":"Dan Schlauch, Joseph N. Paulson, Albert Young, John\nQuackenbush and Kimberly Glass","name":"pandaR","license":"GPL-2","tags":["StatisticalMethod","GraphAndNetwork","Microarray","GeneRegulation","NetworkInference","GeneExpression","Transcription","Network"],"description":"Runs PANDA, an algorithm for discovering novel network\nstructure by combining information from multiple complementary\ndata sources.","downloads_month":165,"downloads_total":14418,"page":"http://bioconductor.org/packages/release/bioc/html/pandaR.html"},{"authors":"Verena Haunschmid and Gundula Povysil","name":"panelcn.mops","license":"LGPL (>= 2.0)","tags":["Sequencing","CopyNumberVariation","CellBiology","GenomicVariation","VariantDetection","Genetics"],"description":"CNV detection tool for targeted NGS panel data. Extension\nof the cn.mops package.","downloads_month":187,"downloads_total":3894,"page":"http://bioconductor.org/packages/release/bioc/html/panelcn.mops.html"},{"authors":"Peter Warren","name":"panp","license":"GPL (>= 2)","tags":["Infrastructure"],"description":"A function to make gene presence/absence calls based on\ndistance from negative strand matching probesets (NSMP) which\nare derived from Affymetrix annotation. PANP is applied after\ngene expression values are created, and therefore can be used\nafter any preprocessing method such as MAS5 or GCRMA, or\nPM-only methods like RMA. NSMP sets have been established for\nthe HGU133A and HGU133-Plus-2.0 chipsets to date.","downloads_month":149,"downloads_total":50372,"page":"http://bioconductor.org/packages/release/bioc/html/panp.html"},{"authors":"Xin Wang <xin_wang@hms.harvard.edu>","name":"PANR","license":"Artistic-2.0","tags":["ImmunoOncology","NetworkInference","Visualization","GraphAndNetwork","Clustering","CellBasedAssays"],"description":"This package provides S4 classes and methods for inferring\nfunctional gene networks with edges encoding posterior beliefs\nof gene association types and nodes encoding perturbation\neffects.","downloads_month":170,"downloads_total":31322,"page":"http://bioconductor.org/packages/release/bioc/html/PANR.html"},{"authors":"Thomas Lin Pedersen","name":"PanVizGenerator","license":"GPL (>= 2)","tags":["ComparativeGenomics","GUI","Visualization"],"description":"PanViz is a JavaScript based visualisation tool for\nfunctionaly annotated pangenomes. PanVizGenerator is a\ncompanion for PanViz that facilitates the necessary data\npreprocessing step necessary to create a working PanViz\nvisualization. The output is fully self-contained so the\nrecipient of the visualization does not need R or\nPanVizGenerator installed.","downloads_month":155,"downloads_total":8668,"page":"http://bioconductor.org/packages/release/bioc/html/PanVizGenerator.html"},{"authors":"Raphael Aggio","name":"PAPi","license":"GPL(>= 2)","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"The Pathway Activity Profiling - PAPi - is an R package\nfor predicting the activity of metabolic pathways based solely\non a metabolomics data set containing a list of metabolites\nidentified and their respective abundances in different\nbiological samples. PAPi generates hypothesis that improves the\nfinal biological interpretation. See Aggio, R.B.M; Ruggiero, K.\nand Villas-Boas, S.G. (2010) - Pathway Activity Profiling\n(PAPi): from metabolite profile to metabolic pathway activity.\nBioinformatics.","downloads_month":190,"downloads_total":27364,"page":"http://bioconductor.org/packages/release/bioc/html/PAPi.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"parody","license":"Artistic-2.0","tags":["MultipleComparison"],"description":"routines for univariate and multivariate outlier detection\nwith a focus on parametric methods, but support for some\nmethods based on resistant statistics","downloads_month":163,"downloads_total":59318,"page":"http://bioconductor.org/packages/release/bioc/html/parody.html"},{"authors":"Oliver Philipp [aut, cre] and Ina Koch [ctb]","name":"Path2PPI","license":"GPL (>= 2)","tags":["NetworkInference","SystemsBiology","Network","Proteomics","Pathways"],"description":"Package to predict protein-protein interaction (PPI)\nnetworks in target organisms for which only a view information\nabout PPIs is available. Path2PPI predicts PPI networks based\non sets of proteins which can belong to a certain pathway from\nwell-established model organisms. It helps to combine and\ntransfer information of a certain pathway or biological process\nfrom several reference organisms to one target organism.\nPath2PPI only depends on the sequence similarity of the\ninvolved proteins.","downloads_month":145,"downloads_total":11394,"page":"http://bioconductor.org/packages/release/bioc/html/Path2PPI.html"},{"authors":"Yotam Drier","name":"pathifier","license":"Artistic-1.0","tags":["Network"],"description":"Pathifier is an algorithm that infers pathway deregulation\nscores for each tumor sample on the basis of expression data.\nThis score is determined, in a context-specific manner, for\nevery particular dataset and type of cancer that is being\ninvestigated. The algorithm transforms gene-level information\ninto pathway-level information, generating a compact and\nbiologically relevant representation of each sample.","downloads_month":296,"downloads_total":48608,"page":"http://bioconductor.org/packages/release/bioc/html/pathifier.html"},{"authors":"Bhaskar Dutta <bhaskar.dutta@gmail.com>, Anders Wallqvist\n<awallqvist@bhsai.org>, and Jaques Reifman <jreifman@bhsai.org>","name":"PathNet","license":"GPL-3","tags":["Pathways","DifferentialExpression","MultipleComparison","KEGG","NetworkEnrichment","Network"],"description":"PathNet uses topological information present in pathways\nand differential expression levels of genes (obtained from\nmicroarray experiment) to identify pathways that are 1)\nsignificantly enriched and 2) associated with each other in the\ncontext of differential expression. The algorithm is described\nin: PathNet: A tool for pathway analysis using topological\ninformation. Dutta B, Wallqvist A, and Reifman J. Source Code\nfor Biology and Medicine 2012 Sep 24;7(1):10.","downloads_month":143,"downloads_total":28006,"page":"http://bioconductor.org/packages/release/bioc/html/PathNet.html"},{"authors":"Solaiappan Manimaran <manimaran_1975@hotmail.com>, Matthew\nBendall <bendall@gwmail.gwu.edu>, Sandro Valenzuela Diaz\n<sandrolvalenzuelad@gmail.com>, Eduardo Castro\n<castronallar@gmail.com>, Tyler Faits <tfaits@gmail.com>, Yue\nZhao <jasonzhao0307@gmail.com>, Anthony Nicholas Federico\n<anfed@bu.edu> and W. Evan Johnson <wej@bu.edu>","name":"PathoStat","license":"GPL (>= 2)","tags":["Microbiome","Metagenomics","GraphAndNetwork","Microarray","PatternLogic","PrincipalComponent","Sequencing","Software","Visualization","RNASeq","ImmunoOncology"],"description":"The purpose of this package is to perform Statistical\nMicrobiome Analysis on metagenomics results from sequencing\ndata samples. In particular, it supports analyses on the\nPathoScope generated report files. PathoStat provides various\nfunctionalities including Relative Abundance charts, Diversity\nestimates and plots, tests of Differential Abundance, Time\nSeries visualization, and Core OTU analysis.","downloads_month":162,"downloads_total":8432,"page":"http://bioconductor.org/packages/release/bioc/html/PathoStat.html"},{"authors":"Gabriel Altschuler and Sokratis Kariotis","name":"pathprint","license":"GPL","tags":["Transcription","GeneExpression","KEGG","Reactome"],"description":"Algorithms to convert a gene expression array provided as\nan expression table or a GEO reference to a 'pathway\nfingerprint', a vector of discrete ternary scores representing\nhigh (1), low(-1) or insignificant (0) expression in a suite of\npathways.","downloads_month":121,"downloads_total":4638,"page":"http://bioconductor.org/packages/release/bioc/html/pathprint.html"},{"authors":"Li Long <lilong@isb-sib.ch>","name":"pathRender","license":"LGPL","tags":["GraphAndNetwork","Pathways","Visualization"],"description":"build graphs from pathway databases, render them by\nRgraphviz.","downloads_month":142,"downloads_total":46008,"page":"http://bioconductor.org/packages/release/bioc/html/pathRender.html"},{"authors":"Laurence de Torrente, Samuel Zimmerman and Jessica Mar","name":"pathVar","license":"LGPL (>= 2.0)","tags":["GeneticVariability","GeneSetEnrichment","Pathways"],"description":"This package contains the functions to find the pathways\nthat have significantly different variability than a reference\ngene set. It also finds the categories from this pathway that\nare significant where each category is a cluster of genes. The\ngenes are separated into clusters by their level of\nvariability.","downloads_month":140,"downloads_total":9834,"page":"http://bioconductor.org/packages/release/bioc/html/pathVar.html"},{"authors":"Weijun Luo","name":"pathview","license":"GPL (>=3.0)","tags":["Pathways","GraphAndNetwork","Visualization","GeneSetEnrichment","DifferentialExpression","GeneExpression","Microarray","RNASeq","Genetics","Metabolomics","Proteomics","SystemsBiology","Sequencing"],"description":"Pathview is a tool set for pathway based data integration\nand visualization. It maps and renders a wide variety of\nbiological data on relevant pathway graphs. All users need is\nto supply their data and specify the target pathway. Pathview\nautomatically downloads the pathway graph data, parses the data\nfile, maps user data to the pathway, and render pathway graph\nwith the mapped data. In addition, Pathview also seamlessly\nintegrates with pathway and gene set (enrichment) analysis\ntools for large-scale and fully automated analysis.","downloads_month":5155,"downloads_total":311076,"page":"http://bioconductor.org/packages/release/bioc/html/pathview.html"},{"authors":"Aimin Yan, Xi Chen and Lily Wang","name":"PathwaySplice","license":"LGPL(>=2)","tags":["ImmunoOncology","AlternativeSplicing","DifferentialSplicing","GeneSetEnrichment","GO","RNASeq","Sequencing","Software","Visualization","NetworkEnrichment","Network","Pathways","GraphAndNetwork","Regression"],"description":"Pathway analysis of alternative splicing would be biased\nwithout accounting for the different number of exons associated\nwith each gene, because genes with higher number of exons are\nmore likely to be included in the 'significant' gene list in\nalternative splicing. PathwaySplice is an R package that: (1)\nperforms pathway analysis that explicitly adjusts for the\nnumber of exons associated with each gene (2) visualizes\nselection bias due to different number of exons for each gene\n(3) formally tests for presence of bias using logistic\nregression (4) supports gene sets based on the Gene Ontology\nterms, as well as more broadly defined gene sets (e.g. MSigDB)\nor user defined gene sets (5) identifies the significant genes\ndriving pathway significance (6) organizes significant pathways\nwith an enrichment map, where pathways with large number of\noverlapping genes are grouped together in a network graph","downloads_month":179,"downloads_total":4442,"page":"http://bioconductor.org/packages/release/bioc/html/PathwaySplice.html"},{"authors":"Augustin Luna","name":"paxtoolsr","license":"LGPL-3","tags":["GeneSetEnrichment","GraphAndNetwork","Pathways","Software","SystemsBiology","NetworkEnrichment","Network","Reactome","KEGG"],"description":"The package provides a set of R functions for interacting\nwith BioPAX OWL files using Paxtools and the querying Pathway\nCommons (PC) molecular interaction database that are hosted by\nthe Computational Biology Center at Memorial Sloan-Kettering\nCancer Center (MSKCC). Pathway Commons databases include: BIND,\nBioGRID, CORUM, CTD, DIP, DrugBank, HPRD, HumanCyc, IntAct,\nKEGG, MirTarBase, Panther, PhosphoSitePlus, Reactome, RECON,\nTRANSFAC.","downloads_month":189,"downloads_total":28480,"page":"http://bioconductor.org/packages/release/bioc/html/paxtoolsr.html"},{"authors":"Laurent Gatto [aut], Sebastian Gibb [aut and cre]","name":"Pbase","license":"GPL-3","tags":["ImmunoOncology","Infrastructure","Proteomics","MassSpectrometry","Visualization","DataImport","DataRepresentation"],"description":"A set of classes and functions to investigate and\nunderstand protein sequence data in the context of a proteomics\nexperiment.","downloads_month":200,"downloads_total":18232,"page":"http://bioconductor.org/packages/release/bioc/html/Pbase.html"},{"authors":"Federico Marini [aut and cre]\n(<https://orcid.org/0000-0003-3252-7758>)","name":"pcaExplorer","license":"MIT + file LICENSE","tags":["ImmunoOncology","Visualization","RNASeq","DimensionReduction","PrincipalComponent","QualityControl","GUI","ReportWriting"],"description":"This package provides functionality for interactive\nvisualization of RNA-seq datasets based on Principal Components\nAnalysis. The methods provided allow for quick information\nextraction and effective data exploration. A Shiny application\nencapsulates the whole analysis.","downloads_month":590,"downloads_total":24414,"page":"http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html"},{"authors":"Morten Hansen and Jorgen Olsen","name":"pcaGoPromoter","license":"GPL (>= 2)","tags":["GeneExpression","Microarray","GO","Visualization"],"description":"This package contains functions to ease the analyses of\nDNA micro arrays. It utilizes principal component analysis as\nthe initial multivariate analysis, followed by functional\ninterpretation of the principal component dimensions with\noverrepresentation analysis for GO terms and regulatory\ninterpretations using overrepresentation analysis of predicted\ntranscription factor binding sites with the primo algorithm.","downloads_month":213,"downloads_total":36344,"page":"http://bioconductor.org/packages/release/bioc/html/pcaGoPromoter.html"},{"authors":"Wolfram Stacklies, Henning Redestig and Kevin Wright","name":"pcaMethods","license":"GPL (>= 3)","tags":["Bayesian"],"description":"Provides Bayesian PCA, Probabilistic PCA, Nipals PCA,\nInverse Non-Linear PCA and the conventional SVD PCA. A cluster\nbased method for missing value estimation is included for\ncomparison. BPCA, PPCA and NipalsPCA may be used to perform PCA\non incomplete data as well as for accurate missing value\nestimation. A set of methods for printing and plotting the\nresults is also provided. All PCA methods make use of the same\ndata structure (pcaRes) to provide a common interface to the\nPCA results. Initiated at the Max-Planck Institute for\nMolecular Plant Physiology, Golm, Germany.","downloads_month":5439,"downloads_total":388494,"page":"http://bioconductor.org/packages/release/bioc/html/pcaMethods.html"},{"authors":"Matthew Page and Patrice Godard","name":"PCAN","license":"CC BY-NC-ND 4.0","tags":["Annotation","Sequencing","Genetics","FunctionalPrediction","VariantAnnotation","Pathways","Network"],"description":"Phenotypes comparison based on a pathway consensus\napproach. Assess the relationship between candidate genes and a\nset of phenotypes based on additional genes related to the\ncandidate (e.g. Pathways or network neighbors).","downloads_month":132,"downloads_total":8186,"page":"http://bioconductor.org/packages/release/bioc/html/PCAN.html"},{"authors":"Kevin Blighe and Myles Lewis","name":"PCAtools","license":"GPL-3","tags":["RNASeq","GeneExpression","Transcription"],"description":"Principal Components Analysis (PCA) is a very powerful\ntechnique that has wide applicability in data science,\nbioinformatics, and further afield. It was initially developed\nto analyse large volumes of data in order to tease out the\ndifferences/relationships between the logical entities being\nanalysed. It extracts the fundamental structure of the data\nwithout the need to build any model to represent it. This\n'summary' of the data is arrived at through a process of\nreduction that can transform the large number of variables into\na lesser number that are uncorrelated, i.e., the principal\ncomponents, whilst at the same time being capable of easy\ninterpretation on the original data.","downloads_month":4,"downloads_total":72,"page":"http://bioconductor.org/packages/release/bioc/html/PCAtools.html"},{"authors":"Sarah Song and Mik Black","name":"pcot2","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","KEGG","GeneExpression","Network"],"description":"PCOT2 is a permutation-based method for investigating\nchanges in the activity of multi-gene networks. It utilizes\ninter-gene correlation information to detect significant\nalterations in gene network activities. Currently it can be\napplied to two-sample comparisons.","downloads_month":231,"downloads_total":51218,"page":"http://bioconductor.org/packages/release/bioc/html/pcot2.html"},{"authors":"Nolwenn Le Meur and Robert Gentleman","name":"PCpheno","license":"Artistic-2.0","tags":["GraphAndNetwork","Proteomics","Network"],"description":"Tools to integrate, annotate, and link phenotypes to\ncellular organizational units such as protein complexes and\npathways.","downloads_month":147,"downloads_total":43826,"page":"http://bioconductor.org/packages/release/bioc/html/PCpheno.html"},{"authors":"Sokratis Kariotis, Yered Pita-Juarez, Winston Hide and Wenbin Wei","name":"pcxn","license":"MIT + file LICENSE","tags":["ExperimentData","ExpressionData","MicroarrayData","GEO","Homo_sapiens_Data","OneChannelData","PathwayInteractionDatabase"],"description":"Discover the correlated pathways/gene sets of a single\npathway/gene set or discover correlation relationships among\nmultiple pathways/gene sets. Draw a heatmap or create a network\nof your query and extract members of each pathway/gene set\nfound in the available collections (MSigDB H hallmark, MSigDB\nC2 Canonical pathways, MSigDB C5 GO BP and Pathprint).","downloads_month":123,"downloads_total":3214,"page":"http://bioconductor.org/packages/release/bioc/html/pcxn.html"},{"authors":"Seth Falcon, Vince Carey, Matt Settles, Kristof de Beuf and Benilton Carvalho","name":"pdInfoBuilder","license":"Artistic-2.0","tags":["Annotation","Infrastructure"],"description":"Builds platform design information packages. These consist\nof a SQLite database containing feature-level data such as x, y\nposition on chip and featureSet ID. The database also\nincorporates featureSet-level annotation data. The products of\nthis packages are used by the oligo pkg.","downloads_month":242,"downloads_total":74742,"page":"http://bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html"},{"authors":"Tomi Suomi, Jukka Hiissa and Laura L. Elo","name":"PECA","license":"GPL (>= 2)","tags":["Software","Proteomics","Microarray","DifferentialExpression","GeneExpression","ExonArray","DifferentialSplicing"],"description":"Calculates Probe-level Expression Change Averages (PECA)\nto identify differential expression in Affymetrix gene\nexpression microarray studies or in proteomic studies using\npeptide-level mesurements respectively.","downloads_month":163,"downloads_total":20644,"page":"http://bioconductor.org/packages/release/bioc/html/PECA.html"},{"authors":"Manon Martin [aut, cre], Bernadette Govaerts [aut, ths], Benoît\nLegat [aut], Pascal de Tullio [dtc], Bruno Boulanger [ctb], Paul H.C. Eilers [ctb] and Julien Vanwinsberghe [ctb]","name":"PepsNMR","license":"GPL-2 | file LICENSE","tags":["Software","Preprocessing","Visualization","Metabolomics","DataImport"],"description":"This package provides R functions for common pre-procssing\nsteps that are applied on 1H-NMR data. It also provides a\nfunction to read the FID signals directly in the Bruker format.","downloads_month":104,"downloads_total":790,"page":"http://bioconductor.org/packages/release/bioc/html/PepsNMR.html"},{"authors":"Raphael Gottardo, Gregory C Imholte, Renan Sauteraud and Mike\nJiang","name":"pepStat","license":"Artistic-2.0","tags":["Microarray","Preprocessing"],"description":"Statistical analysis of peptide microarrays","downloads_month":157,"downloads_total":15948,"page":"http://bioconductor.org/packages/release/bioc/html/pepStat.html"},{"authors":"Xiaojing Wang","name":"pepXMLTab","license":"Artistic-2.0","tags":["ImmunoOncology","Proteomics","MassSpectrometry"],"description":"Parsing pepXML files based one XML package. The package\ntries to handle pepXML files generated from different\nsoftwares. The output will be a peptide-spectrum-matching\ntabular file. The package also provide function to filter the\nPSMs based on FDR.","downloads_month":152,"downloads_total":16188,"page":"http://bioconductor.org/packages/release/bioc/html/pepXMLTab.html"},{"authors":"Simon Dirmeier [aut and cre]","name":"perturbatr","license":"GPL-3","tags":["ImmunoOncology","Regression","CellBasedAssays","Network"],"description":"perturbatr does stage-wise analysis of large-scale genetic\nperturbation screens for integrated data sets consisting of\nmultiple screens. For multiple integrated perturbation screens\na hierarchical model that considers the variance between\ndifferent biological conditions is fitted. The resulting list\nof gene effects is then further extended using a network\npropagation algorithm to correct for false negatives.","downloads_month":118,"downloads_total":1996,"page":"http://bioconductor.org/packages/release/bioc/html/perturbatr.html"},{"authors":"Shaohang Xu and Bo Wen","name":"PGA","license":"GPL-2","tags":["Proteomics","ImmunoOncology","MassSpectrometry","Software","ReportWriting","RNASeq","Sequencing"],"description":"This package provides functions for construction of\ncustomized protein databases based on RNA-Seq data with/without\ngenome guided, database searching, post-processing and report\ngeneration. This kind of customized protein database includes\nboth the reference database (such as Refseq or ENSEMBL) and the\nnovel peptide sequences form RNA-Seq data.","downloads_month":165,"downloads_total":12014,"page":"http://bioconductor.org/packages/release/bioc/html/PGA.html"},{"authors":"Gabriela Cohen-Freue <gcohen@stat.ubc.ca>","name":"pgca","license":"GPL (>= 2)","tags":["WorkflowStep","AssayDomain","Proteomics","MassSpectrometry","ImmunoOncology"],"description":"Protein Group Code Algorithm (PGCA) is a computationally\ninexpensive algorithm to merge protein summaries from multiple\nexperimental quantitative proteomics data. The algorithm\nconnects two or more groups with overlapping accession numbers.\nIn some cases, pairwise groups are mutually exclusive but they\nmay still be connected by another group (or set of groups) with\noverlapping accession numbers. Thus, groups created by PGCA\nfrom multiple experimental runs (i.e., global groups) are\ncalled \"connected\" groups. These identified global protein\ngroups enable the analysis of quantitative data available for\nprotein groups instead of unique protein identifiers.","downloads_month":128,"downloads_total":4474,"page":"http://bioconductor.org/packages/release/bioc/html/pgca.html"},{"authors":"Kyle Furge <kyle.furge@vai.org> and Karl Dykema\n<karl.dykema@vai.org>","name":"PGSEA","license":"GPL-2","tags":["Microarray"],"description":"Parametric Analysis of Gene Set Enrichment","downloads_month":358,"downloads_total":81096,"page":"http://bioconductor.org/packages/release/bioc/html/PGSEA.html"},{"authors":"Daria Zenkova [aut], Vladislav Kamenev [aut], Rita Sablina\n[ctb], Maxim Artyomov [aut], Alexey Sergushichev [aut and cre]","name":"phantasus","license":"MIT + file LICENSE","tags":["GeneExpression","GUI","Visualization","DataRepresentation","Transcriptomics","RNASeq","Microarray","Normalization","Clustering","DifferentialExpression","PrincipalComponent","ImmunoOncology"],"description":"Phantasus is a web-application for visual and interactive\ngene expression analysis. Phantasus is based on Morpheus – a\nweb-based software for heatmap visualisation and analysis,\nwhich was integrated with an R environment via OpenCPU API.\nAside from basic visualization and filtering methods, R-based\nmethods such as k-means clustering, principal component\nanalysis or differential expression analysis with limma package\nare supported.","downloads_month":125,"downloads_total":2196,"page":"http://bioconductor.org/packages/release/bioc/html/phantasus.html"},{"authors":"Petr Smirnov, Zhaleh Safikhani, Mark Freeman and Benjamin\nHaibe-Kains","name":"PharmacoGx","license":"Artistic-2.0","tags":["GeneExpression","Pharmacogenetics","Pharmacogenomics","Software","Classification"],"description":"Contains a set of functions to perform large-scale\nanalysis of pharmacogenomic data.","downloads_month":295,"downloads_total":12984,"page":"http://bioconductor.org/packages/release/bioc/html/PharmacoGx.html"},{"authors":"William S Chen","name":"phemd","license":"GPL-2","tags":["Clustering","ComparativeGenomics","Proteomics","Transcriptomics","Sequencing","DimensionReduction","SingleCell","DataRepresentation","Visualization","MultipleComparison"],"description":"Package for comparing and generating a low-dimensional\nembedding of multiple single-cell samples.","downloads_month":0,"downloads_total":38,"page":"http://bioconductor.org/packages/release/bioc/html/phemd.html"},{"authors":"Kieran Campbell","name":"phenopath","license":"Apache License (== 2.0)","tags":["ImmunoOncology","RNASeq","GeneExpression","Bayesian","SingleCell","PrincipalComponent"],"description":"PhenoPath infers genomic trajectories (pseudotimes) in the\npresence of heterogeneous genetic and environmental backgrounds\nand tests for interactions between them.","downloads_month":150,"downloads_total":4190,"page":"http://bioconductor.org/packages/release/bioc/html/phenopath.html"},{"authors":"Evarist Planet","name":"phenoTest","license":"GPL (>=2)","tags":["Microarray","DifferentialExpression","MultipleComparison","Clustering","Classification"],"description":"Tools to test correlation between gene expression and\nphenotype in a way that is efficient, structured, fast and\nscalable. GSEA is also provided.","downloads_month":221,"downloads_total":41258,"page":"http://bioconductor.org/packages/release/bioc/html/phenoTest.html"},{"authors":"Natalja Kurbatova, Natasha Karp, Jeremy Mason and Hamed\nHaselimashhadi","name":"PhenStat","license":"file LICENSE","tags":["StatisticalMethod"],"description":"Package contains methods for statistical analysis of\nphenotypic data.","downloads_month":157,"downloads_total":19468,"page":"http://bioconductor.org/packages/release/bioc/html/PhenStat.html"},{"authors":"Justin Silverman","name":"philr","license":"GPL-3","tags":["ImmunoOncology","Sequencing","Microbiome","Metagenomics","Software"],"description":"PhILR is short for Phylogenetic Isometric Log-Ratio\nTransform. This package provides functions for the analysis of\ncompositional data (e.g., data representing proportions of\ndifferent variables/parts). Specifically this package allows\nanalysis of compositional data where the parts can be related\nthrough a phylogenetic tree (as is common in microbiota survey\ndata) and makes available the Isometric Log Ratio transform\nbuilt from the phylogenetic tree and utilizing a weighted\nreference measure.","downloads_month":302,"downloads_total":8858,"page":"http://bioconductor.org/packages/release/bioc/html/philr.html"},{"authors":"Sohrab Saraei, Tomi Suomi, Otto Kauko and Laura L. Elo","name":"phosphonormalizer","license":"GPL (>= 2)","tags":["Software","StatisticalMethod","WorkflowStep","Normalization","Proteomics"],"description":"It uses the overlap between enriched and non-enriched\ndatasets to compensate for the bias introduced in global\nphosphorylation after applying median normalization.","downloads_month":119,"downloads_total":8092,"page":"http://bioconductor.org/packages/release/bioc/html/phosphonormalizer.html"},{"authors":"Paul J. McMurdie <joey711@gmail.com>, Susan Holmes\n<susan@stat.stanford.edu>, with contributions from Gregory\nJordan and Scott Chamberlain","name":"phyloseq","license":"AGPL-3","tags":["ImmunoOncology","Sequencing","Microbiome","Metagenomics","Clustering","Classification","MultipleComparison","GeneticVariability"],"description":"phyloseq provides a set of classes and tools to facilitate\nthe import, storage, analysis, and graphical display of\nmicrobiome census data.","downloads_month":7178,"downloads_total":394106,"page":"http://bioconductor.org/packages/release/bioc/html/phyloseq.html"},{"authors":"Hai Fang, the ULTRA-DD Consortium and Julian C Knight","name":"Pi","license":"GPL-3","tags":["Software","Genetics","GraphAndNetwork","Pathways","GeneExpression","GeneTarget","GenomeWideAssociation","LinkageDisequilibrium","Network","HiC"],"description":"Priority index or Pi is developed as a genomic-led target\nprioritisation system, with the focus on leveraging human\ngenetic data to prioritise potential drug targets at the gene,\npathway and network level. The long term goal is to use such\ninformation to enhance early-stage target validation. Based on\nevidence of disease association from genome-wide association\nstudies (GWAS), this prioritisation system is able to generate\nevidence to support identification of the specific modulated\ngenes (seed genes) that are responsible for the genetic\nassociation signal by utilising knowledge of linkage\ndisequilibrium (co-inherited genetic variants), distance of\nassociated variants from the gene, evidence of independent\ngenetic association with gene expression in disease-relevant\ntissues, cell types and states, and evidence of physical\ninteractions between disease-associated genetic variants and\ngene promoters based on genome-wide capture HiC-generated\npromoter interactomes in primary blood cell types. Seed genes\nare scored in an integrative way, quantifying the genetic\ninfluence. Scored seed genes are subsequently used as baits to\nrank seed genes plus additional (non-seed) genes; this is\nachieved by iteratively exploring the global connectivity of a\ngene interaction network. Genes with the highest priority are\nfurther used to identify/prioritise pathways that are\nsignificantly enriched with highly prioritised genes.\nPrioritised genes are also used to identify a gene network\ninterconnecting highly prioritised genes and a minimal number\nof less prioritised genes (which act as linkers bringing\ntogether highly prioritised genes).","downloads_month":126,"downloads_total":7204,"page":"http://bioconductor.org/packages/release/bioc/html/Pi.html"},{"authors":"Leif Varemo <piano.rpkg@gmail.com> and Intawat Nookaew\n<piano.rpkg@gmail.com>","name":"piano","license":"GPL (>=2)","tags":["Microarray","Preprocessing","QualityControl","DifferentialExpression","Visualization","GeneExpression","GeneSetEnrichment","Pathways"],"description":"Piano performs gene set analysis using various statistical\nmethods, from different gene level statistics and a wide range\nof gene-set collections. Furthermore, the Piano package\ncontains functions for combining the results of multiple runs\nof gene set analyses.","downloads_month":524,"downloads_total":70390,"page":"http://bioconductor.org/packages/release/bioc/html/piano.html"},{"authors":"Brian S. Yandell <yandell@stat.wisc.edu>","name":"pickgene","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression"],"description":"Functions to Analyze Microarray (Gene Expression) Data.","downloads_month":122,"downloads_total":42644,"page":"http://bioconductor.org/packages/release/bioc/html/pickgene.html"},{"authors":"Xuekui Zhang <xzhang@stat.ubc.ca> and Raphael Gottardo\n<rgottard@fhcrc.org>","name":"PICS","license":"Artistic-2.0","tags":["Clustering","Visualization","Sequencing","ChIPSeq"],"description":"Probabilistic inference of ChIP-Seq using an empirical\nBayes mixture model approach.","downloads_month":232,"downloads_total":62730,"page":"http://bioconductor.org/packages/release/bioc/html/PICS.html"},{"authors":"Habil Zare, Amir Foroushani, and Rupesh Agrahari","name":"Pigengene","license":"GPL (>=2)","tags":["GeneExpression","RNASeq","NetworkInference","Network","GraphAndNetwork","BiomedicalInformatics","SystemsBiology","Transcriptomics","Classification","Clustering","DecisionTree","DimensionReduction","PrincipalComponent","Microarray","Normalization","ImmunoOncology"],"description":"Pigengene package provides an efficient way to infer\nbiological signatures from gene expression profiles. The\nsignatures are independent from the underlying platform, e.g.,\nthe input can be microarray or RNA Seq data. It can even infer\nthe signatures using data from one platform, and evaluate them\non the other. Pigengene identifies the modules (clusters) of\nhighly coexpressed genes using coexpression network analysis,\nsummarizes the biological information of each module in an\neigengene, learns a Bayesian network that models the\nprobabilistic dependencies between modules, and builds a\ndecision tree based on the expression of eigengenes.","downloads_month":223,"downloads_total":8564,"page":"http://bioconductor.org/packages/release/bioc/html/Pigengene.html"},{"authors":"Xuekui Zhang <xuezhang@jhsph.edu>, Raphael Gottardo\n<raph@stat.ubc.ca> and Sangsoon Woo <swoo@fhcrc.org>","name":"PING","license":"Artistic-2.0","tags":["Clustering","StatisticalMethod","Visualization","Sequencing"],"description":"Probabilistic inference of ChIP-Seq using an empirical\nBayes mixture model approach.","downloads_month":139,"downloads_total":33444,"page":"http://bioconductor.org/packages/release/bioc/html/PING.html"},{"authors":"Olli-Pekka Huovilainen <ohuovila@gmail.com> and Leo Lahti\n<leo.lahti@iki.fi>","name":"pint","license":"BSD_2_clause + file LICENSE","tags":["aCGH","GeneExpression","Genetics","DifferentialExpression","Microarray"],"description":"Pairwise data integration for functional genomics,\nincluding tools for DNA/RNA/miRNA dependency screens.","downloads_month":137,"downloads_total":39106,"page":"http://bioconductor.org/packages/release/bioc/html/pint.html"},{"authors":"Seth Falcon [aut] and Bioconductor Package Maintainer [cre]","name":"pkgDepTools","license":"GPL-2","tags":["Infrastructure","GraphAndNetwork"],"description":"This package provides tools for computing and analyzing\ndependency relationships among R packages.  It provides tools\nfor building a graph-based representation of the dependencies\namong all packages in a list of CRAN-style package\nrepositories.  There are also utilities for computing\ninstallation order of a given package.  If the RCurl package is\navailable, an estimate of the download size required to install\na given package and its dependencies can be obtained.","downloads_month":170,"downloads_total":51664,"page":"http://bioconductor.org/packages/release/bioc/html/pkgDepTools.html"},{"authors":"Errol Strain, Florian Hahne, and Perry Haaland","name":"plateCore","license":"Artistic-2.0","tags":["ImmunoOncology","FlowCytometry","Infrastructure","CellBasedAssays"],"description":"Provides basic S4 data structures and routines for\nanalyzing plate based flow cytometry data.","downloads_month":176,"downloads_total":39568,"page":"http://bioconductor.org/packages/release/bioc/html/plateCore.html"},{"authors":"Daniel Bottomly [aut, cre], Marty Ferris [ctb], Beth Wilmot\n[aut] and Shannon McWeeney [aut]","name":"plethy","license":"GPL-3","tags":["DataImport","biocViews","Infastructure","DataRepresentation","TimeCourse"],"description":"This package provides the infrastructure and tools to\nimport, query and perform basic analysis of whole body\nplethysmography and metabolism data.  Currently support is\nlimited to data derived from Buxco respirometry instruments as\nexported by their FinePointe software.","downloads_month":159,"downloads_total":21854,"page":"http://bioconductor.org/packages/release/bioc/html/plethy.html"},{"authors":"Mattia Pelizzola <mattia.pelizzola@gmail.com> and Norman\nPavelka <normanpavelka@gmail.com>","name":"plgem","license":"GPL-2","tags":["ImmunoOncology","Microarray","DifferentialExpression","Proteomics","GeneExpression","MassSpectrometry"],"description":"The Power Law Global Error Model (PLGEM) has been shown to\nfaithfully model the variance-versus-mean dependence that\nexists in a variety of genome-wide datasets, including\nmicroarray and proteomics data. The use of PLGEM has been shown\nto improve the detection of differentially expressed genes or\nproteins in these datasets.","downloads_month":220,"downloads_total":48680,"page":"http://bioconductor.org/packages/release/bioc/html/plgem.html"},{"authors":"Affymetrix Inc., Crispin J Miller and PICR","name":"plier","license":"GPL (>= 2)","tags":["Software"],"description":"The PLIER (Probe Logarithmic Error Intensity Estimate)\nmethod produces an improved signal by accounting for\nexperimentally observed patterns in probe behavior and handling\nerror at the appropriately at low and high signal values.","downloads_month":306,"downloads_total":99926,"page":"http://bioconductor.org/packages/release/bioc/html/plier.html"},{"authors":"John D. Gagnon [aut and cre]","name":"plotGrouper","license":"GPL-3","tags":["ImmunoOncology","FlowCytometry","GraphAndNetwork","StatisticalMethod","DataImport","GUI","MultipleComparison"],"description":"A shiny app-based GUI wrapper for ggplot with built-in\nstatistical analysis. Import data from file and use dropdown\nmenus and checkboxes to specify the plotting variables, graph\ntype, and look of your plots. Once created, plots can be saved\nindependently or stored in a report that can be saved as a pdf.\nIf new data are added to the file, the report can be refreshed\nto include new data. Statistical tests can be selected and\nadded to the graphs. Analysis of flow cytometry data is\nespecially integrated with plotGrouper. Count data can be\ntransformed to return the absolute number of cells in a sample\n(this feature requires inclusion of the number of beads per\nsample and information about any dilution performed).","downloads_month":112,"downloads_total":920,"page":"http://bioconductor.org/packages/release/bioc/html/plotGrouper.html"},{"authors":"HyungJun Cho <hj4cho@korea.ac.kr> and Jae K. Lee\n<jaeklee@virginia.edu>","name":"PLPE","license":"GPL (>= 2)","tags":["Proteomics","Microarray","DifferentialExpression"],"description":"This package performs tests for paired high-throughput\ndata.","downloads_month":141,"downloads_total":43720,"page":"http://bioconductor.org/packages/release/bioc/html/PLPE.html"},{"authors":"Gwenael G.R. Leday","name":"plrs","license":"GPL (>=2.0)","tags":["Regression"],"description":"The present package implements a flexible framework for\nmodeling the relationship between DNA copy number and gene\nexpression data using Piecewise Linear Regression Splines\n(PLRS).","downloads_month":146,"downloads_total":24334,"page":"http://bioconductor.org/packages/release/bioc/html/plrs.html"},{"authors":"Magnus Astrand","name":"plw","license":"GPL-2","tags":["Microarray","OneChannel","TwoChannel","DifferentialExpression"],"description":"Probe level Locally moderated Weighted median-t (PLW) and\nLocally Moderated Weighted-t (LMW).","downloads_month":133,"downloads_total":42266,"page":"http://bioconductor.org/packages/release/bioc/html/plw.html"},{"authors":"Stuart Lee [aut, cre], Michael Lawrence [aut, ctb], Dianne Cook\n[aut and ctb]","name":"plyranges","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation","WorkflowStep","Coverage"],"description":"A dplyr-like interface for interacting with the common\nBioconductor classes Ranges and GenomicRanges. By providing a\ngrammatical and consistent way of manipulating these classes\ntheir accessiblity for new Bioconductor users is hopefully\nincreased.","downloads_month":257,"downloads_total":4044,"page":"http://bioconductor.org/packages/release/bioc/html/plyranges.html"},{"authors":"Anna Drewek","name":"pmm","license":"GPL-3","tags":["SystemsBiology","Regression"],"description":"The Parallel Mixed Model (PMM) approach is suitable for\nhit selection and cross-comparison of RNAi screens generated in\nexperiments that are performed in parallel under several\nconditions. For example, we could think of the measurements or\nreadouts from cells under RNAi knock-down, which are infected\nwith several pathogens or which are grown from different cell\nlines.","downloads_month":126,"downloads_total":11334,"page":"http://bioconductor.org/packages/release/bioc/html/pmm.html"},{"authors":"Ulrich Bodenhofer","name":"podkat","license":"GPL (>= 2)","tags":["Genetics","WholeGenome","Annotation","VariantAnnotation","Sequencing","DataImport"],"description":"This package provides an association test that is capable\nof dealing with very rare and even private variants. This is\naccomplished by a kernel-based approach that takes the\npositions of the variants into account. The test can be used\nfor pre-processed matrix data, but also directly for variant\ndata stored in VCF files. Association testing can be performed\nwhole-genome, whole-exome, or restricted to pre-defined regions\nof interest. The test is complemented by tools for analyzing\nand visualizing the results.","downloads_month":154,"downloads_total":13760,"page":"http://bioconductor.org/packages/release/bioc/html/podkat.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu>","name":"pogos","license":"Artistic-2.0","tags":["Pharmacogenomics","PooledScreens","ImmunoOncology"],"description":"Provide simple utilities for querying bhklab PharmacoDB,\nmodeling API outputs, and integrating to cell and compound\nontologies.","downloads_month":159,"downloads_total":2268,"page":"http://bioconductor.org/packages/release/bioc/html/pogos.html"},{"authors":"Alyssa C. Frazee, Andrew E. Jaffe, Rory Kirchner and Jeffrey T.\nLeek","name":"polyester","license":"Artistic-2.0","tags":["Sequencing","DifferentialExpression"],"description":"This package can be used to simulate RNA-seq reads from\ndifferential expression experiments with replicates. The reads\ncan then be aligned and used to perform comparisons of methods\nfor differential expression.","downloads_month":220,"downloads_total":24048,"page":"http://bioconductor.org/packages/release/bioc/html/polyester.html"},{"authors":"Conrad Burden","name":"Polyfit","license":"GPL (>= 3)","tags":["ImmunoOncology","DifferentialExpression","Sequencing","RNASeq","GeneExpression"],"description":"Polyfit is an add-on to the packages DESeq which ensures\nthe p-value distribution is uniform over the interval [0, 1]\nfor data satisfying the null hypothesis of no differential\nexpression, and uses an adpated Storey-Tibshiran method to\ncalculate q-values.","downloads_month":178,"downloads_total":15640,"page":"http://bioconductor.org/packages/release/bioc/html/Polyfit.html"},{"authors":"Xueyuan Cao <xueyuan.cao@stjude.org> and Stanley.pounds\n<stanley.pounds@stjude.org>","name":"POST","license":"GPL (>= 2)","tags":["Microarray","GeneExpression"],"description":"Perform orthogonal projection of high dimensional data of\na set, and statistical modeling of phenotye with projected\nvectors as predictor.","downloads_month":157,"downloads_total":4598,"page":"http://bioconductor.org/packages/release/bioc/html/POST.html"},{"authors":"Xu Qiao [aut, cre] and Laura Elo [cph]","name":"PowerExplorer","license":"Artistic-2.0","tags":["ImmunoOncology","RNASeq","Proteomics","DifferentialExpression","MultipleComparison","Sequencing","Coverage","ChIPSeq"],"description":"Estimate and predict power among groups and multiple\nsample sizes with simulated data, the simulations are operated\nbased on distribution parameters estimated from the provided\ninput dataset.","downloads_month":137,"downloads_total":2200,"page":"http://bioconductor.org/packages/release/bioc/html/PowerExplorer.html"},{"authors":"Hillary Koch","name":"powerTCR","license":"Artistic-2.0","tags":["Software","Clustering","BiomedicalInformatics"],"description":"This package provides a model for the clone size\ndistribution of the TCR repertoire. Further, it permits\ncomparative analysis of TCR repertoire libraries based on\ntheoretical model fits.","downloads_month":130,"downloads_total":2128,"page":"http://bioconductor.org/packages/release/bioc/html/powerTCR.html"},{"authors":"Dongmin Jung and Xijin Ge","name":"PPInfer","license":"Artistic-2.0","tags":["Software","StatisticalMethod","Network","GraphAndNetwork","GeneSetEnrichment","NetworkEnrichment","Pathways"],"description":"Interactions between proteins occur in many, if not most,\nbiological processes. Most proteins perform their functions in\nnetworks associated with other proteins and other biomolecules.\nThis fact has motivated the development of a variety of\nexperimental methods for the identification of protein\ninteractions. This variety has in turn ushered in the\ndevelopment of numerous different computational approaches for\nmodeling and predicting protein interactions. Sometimes an\nexperiment is aimed at identifying proteins closely related to\nsome interesting proteins. A network based statistical learning\nmethod is used to infer the putative functions of proteins from\nthe known functions of its neighboring proteins on a PPI\nnetwork. This package identifies such proteins often involved\nin the same or similar biological functions.","downloads_month":161,"downloads_total":5514,"page":"http://bioconductor.org/packages/release/bioc/html/PPInfer.html"},{"authors":"T. Chiang and D. Scholtens with contributions from W. Huber and\nL. Wang","name":"ppiStats","license":"Artistic-2.0","tags":["Proteomics","GraphAndNetwork","Network","NetworkInference"],"description":"Tools for the analysis of protein interaction data.","downloads_month":183,"downloads_total":50512,"page":"http://bioconductor.org/packages/release/bioc/html/ppiStats.html"},{"authors":"Jiri Hon, Matej Lexa and Tomas Martinek","name":"pqsfinder","license":"BSD_2_clause + file LICENSE","tags":["MotifDiscovery","SequenceMatching","GeneRegulation"],"description":"The main functionality of this package is to detect DNA\nsequence patterns that are likely to fold into an\nintramolecular G-quadruplex (G4). Unlike many other approaches,\nthis package is able to detect sequences responsible for G4s\nfolded from imperfect G-runs containing bulges or mismatches\nand as such is more sensitive than competing algorithms.","downloads_month":171,"downloads_total":9216,"page":"http://bioconductor.org/packages/release/bioc/html/pqsfinder.html"},{"authors":"Florian Hahne <florian.hahne@novartis.com>, Wolfgang Huber\n<huber@ebi.ac.uk>, Markus Ruschhaupt, Joern Toedling\n<toedling@ebi.ac.uk> and Joseph Barry <joseph.barry@embl.de>","name":"prada","license":"LGPL","tags":["ImmunoOncology","CellBasedAssays","Visualization"],"description":"Tools for analysing and navigating data from\nhigh-throughput phenotyping experiments based on cellular\nassays and fluorescent detection (flow cytometry (FACS),\nhigh-content screening microscopy).","downloads_month":523,"downloads_total":102936,"page":"http://bioconductor.org/packages/release/bioc/html/prada.html"},{"authors":"Karolis Uziela and Antti Honkela","name":"prebs","license":"Artistic-2.0","tags":["ImmunoOncology","Microarray","RNASeq","Sequencing","GeneExpression","Preprocessing"],"description":"The prebs package aims at making RNA-sequencing (RNA-seq)\ndata more comparable to microarray data. The comparability is\nachieved by summarizing sequencing-based expressions of probe\nregions using a modified version of RMA algorithm. The pipeline\ntakes mapped reads in BAM format as an input and produces\neither gene expressions or original microarray probe set\nexpressions as an output.","downloads_month":195,"downloads_total":26622,"page":"http://bioconductor.org/packages/release/bioc/html/prebs.html"},{"authors":"Giorgio Melloni, Alessandro Guida and Luca Mazzarella","name":"PrecisionTrialDrawer","license":"GPL-3","tags":["SomaticMutation","WholeGenome","Sequencing","DataImport","GeneExpression"],"description":"A suite of methods to design umbrella and basket trials\nfor preision oncology.","downloads_month":37,"downloads_total":166,"page":"http://bioconductor.org/packages/release/bioc/html/PrecisionTrialDrawer.html"},{"authors":"Francesco Ferrari <francesco.ferrari@ifom.eu>","name":"PREDA","license":"GPL-2","tags":["Software","CopyNumberVariation","GeneExpression","Genetics"],"description":"Package for the position related analysis of quantitative\nfunctional genomics data.","downloads_month":179,"downloads_total":34242,"page":"http://bioconductor.org/packages/release/bioc/html/PREDA.html"},{"authors":"Benjamin Haibe-Kains, Catharina Olsen, Gianluca Bontempi and John\nQuackenbush","name":"predictionet","license":"Artistic-2.0","tags":["GraphAndNetwork","NetworkInference"],"description":"This package contains a set of functions related to\nnetwork inference combining genomic data and prior information\nextracted from biomedical literature and structured biological\ndatabases. The main function is able to generate networks using\nBayesian or regression-based inference methods; while the\nformer is limited to < 100 of variables, the latter may infer\nnetworks with hundreds of variables. Several statistics at the\nedge and node levels have been implemented (edge stability,\npredictive ability of each node, ...) in order to help the user\nto focus on high quality subnetworks. Ultimately, this package\nis used in the 'Predictive Networks' web application developed\nby the Dana-Farber Cancer Institute in collaboration with\nEntagen.","downloads_month":112,"downloads_total":17314,"page":"http://bioconductor.org/packages/release/bioc/html/predictionet.html"},{"authors":"Ben Bolstad <bmb@bmbolstad.com>","name":"preprocessCore","license":"LGPL (>= 2)","tags":["Infrastructure"],"description":"A library of core preprocessing routines.","downloads_month":16216,"downloads_total":1991494,"page":"http://bioconductor.org/packages/release/bioc/html/preprocessCore.html"},{"authors":"Pumin Li [aut, cre] and Qi Xu [aut]","name":"primirTSS","license":"GPL-2","tags":["ImmunoOncology","Sequencing","RNASeq","Genetics","Preprocessing","Transcription","GeneRegulation"],"description":"A fast, convenient tool to identify the TSSs of miRNAs by\nintegrating the data of H3K4me3 and Pol II as well as combining\nthe conservation level and sequence feature, provided within\nboth command-line and graphical interfaces, which achieves a\nbetter performance than the previous non-cell-specific methods\non miRNA TSSs.","downloads_month":93,"downloads_total":692,"page":"http://bioconductor.org/packages/release/bioc/html/primirTSS.html"},{"authors":"Daryanaz Dargahi <daryanazdargahi@gmail.com>","name":"Prize","license":"Artistic-2.0","tags":["ImmunoOncology","Software","MultipleComparison","GeneExpression","CellBiology","RNASeq"],"description":"The high throughput studies often produce large amounts of\nnumerous genes and proteins of interest. While it is difficult\nto study and validate all of them. Analytic Hierarchy Process\n(AHP) offers a novel approach to narrowing down long lists of\ncandidates by prioritizing them based on how well they meet the\nresearch goal. AHP is a mathematical technique for organizing\nand analyzing complex decisions where multiple criteria are\ninvolved. The technique structures problems into a hierarchy of\nelements, and helps to specify numerical weights representing\nthe relative importance of each element. Numerical weight or\npriority derived from each element allows users to find\nalternatives that best suit their goal and their understanding\nof the problem.","downloads_month":156,"downloads_total":10024,"page":"http://bioconductor.org/packages/release/bioc/html/Prize.html"},{"authors":"Xiaojing Wang","name":"proBAMr","license":"Artistic-2.0","tags":["ImmunoOncology","Proteomics","MassSpectrometry","Software","Visualization"],"description":"Mapping PSMs back to genome. The package builds SAM file\nfrom shotgun proteomics data The package also provides function\nto prepare annotation from GTF file.","downloads_month":191,"downloads_total":17288,"page":"http://bioconductor.org/packages/release/bioc/html/proBAMr.html"},{"authors":"Xiaochun Li","name":"PROcess","license":"Artistic-2.0","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"A package for processing protein mass spectrometry data.","downloads_month":304,"downloads_total":97958,"page":"http://bioconductor.org/packages/release/bioc/html/PROcess.html"},{"authors":"Ulrich Bodenhofer","name":"procoil","license":"GPL (>= 2)","tags":["Proteomics","Classification","SupportVectorMachine"],"description":"The package allows for predicting whether a coiled coil\nsequence (amino acid sequence plus heptad register) is more\nlikely to form a dimer or more likely to form a trimer.\nAdditionally to the prediction itself, a prediction profile is\ncomputed which allows for determining the strengths to which\nthe individual residues are indicative for either class.\nPrediction profiles can also be visualized as curves or\nheatmaps.","downloads_month":145,"downloads_total":32344,"page":"http://bioconductor.org/packages/release/bioc/html/procoil.html"},{"authors":"Alexis Delabriere and Etienne Thevenot.","name":"proFIA","license":"CeCILL","tags":["ImmunoOncology","MassSpectrometry","Metabolomics","Lipidomics","Preprocessing","PeakDetection","Proteomics"],"description":"Flow Injection Analysis coupled to High-Resolution Mass\nSpectrometry is a promising approach for high-throughput\nmetabolomics. FIA- HRMS data, however, cannot be pre-processed\nwith current software tools which rely on liquid chromatography\nseparation, or handle low resolution data only. Here we present\nthe proFIA package, which implements a new methodology to\npre-process FIA-HRMS raw data (netCDF, mzData, mzXML, and mzML)\nincluding noise modelling and injection peak reconstruction,\nand generate the peak table. The workflow includes noise\nmodelling, band detection and filtering then signal matching\nand missing value imputation. The peak table can then be\nexported as a .tsv file for further analysis. Visualisations to\nassess the quality of the data and of the signal made are\neasely produced.","downloads_month":169,"downloads_total":8108,"page":"http://bioconductor.org/packages/release/bioc/html/proFIA.html"},{"authors":"Paal O. Westermark","name":"profileScoreDist","license":"MIT + file LICENSE","tags":["Software","GeneRegulation","StatisticalMethod"],"description":"Regularization and score distributions for position count\nmatrices.","downloads_month":124,"downloads_total":7434,"page":"http://bioconductor.org/packages/release/bioc/html/profileScoreDist.html"},{"authors":"Michael Schubert <mschu.dev@gmail.com>","name":"progeny","license":"Apache License (== 2.0) | file LICENSE","tags":["SystemsBiology","GeneExpression","FunctionalPrediction","GeneRegulation"],"description":"This package provides a function to infer pathway activity\nfrom gene expression using PROGENy. It contains the linear\nmodel we inferred in the publication \"Perturbation-response\ngenes reveal signaling footprints in cancer gene expression\".","downloads_month":244,"downloads_total":4240,"page":"http://bioconductor.org/packages/release/bioc/html/progeny.html"},{"authors":"Laurent Gatto, Oliver Crook and Lisa M. Breckels with\ncontributions from Thomas Burger and Samuel Wieczorek","name":"pRoloc","license":"GPL-2","tags":["ImmunoOncology","Proteomics","MassSpectrometry","Classification","Clustering","QualityControl"],"description":"The pRoloc package implements machine learning and\nvisualisation methods for the analysis and interogation of\nquantitiative mass spectrometry data to reliably infer protein\nsub-cellular localisation.","downloads_month":604,"downloads_total":49694,"page":"http://bioconductor.org/packages/release/bioc/html/pRoloc.html"},{"authors":"Lisa M Breckels, Thomas Naake and Laurent Gatto","name":"pRolocGUI","license":"GPL-2","tags":["Proteomics","Visualization","GUI"],"description":"The package pRolocGUI comprises functions to interactively\nvisualise organelle (spatial) proteomics data on the basis of\npRoloc, pRolocdata and shiny.","downloads_month":181,"downloads_total":19608,"page":"http://bioconductor.org/packages/release/bioc/html/pRolocGUI.html"},{"authors":"Stan Pounds <stanley.pounds@stjude.org> and Xueyuan Cao\n<xueyuan.cao@stjude.org>","name":"PROMISE","license":"GPL (>= 2)","tags":["Microarray","OneChannel","MultipleComparison","GeneExpression"],"description":"A general tool to identify genomic features with a\nspecific biologically interesting pattern of associations with\nmultiple endpoint variables as described in Pounds et. al.\n(2009) Bioinformatics 25: 2013-2019","downloads_month":146,"downloads_total":35370,"page":"http://bioconductor.org/packages/release/bioc/html/PROMISE.html"},{"authors":"Hao Wu","name":"PROPER","license":"GPL","tags":["ImmunoOncology","Sequencing","RNASeq","DifferentialExpression"],"description":"This package provide simulation based methods for\nevaluating the statistical power in differential expression\nanalysis from RNA-seq data.","downloads_month":212,"downloads_total":16110,"page":"http://bioconductor.org/packages/release/bioc/html/PROPER.html"},{"authors":"Lichy Han","name":"PROPS","license":"GPL-2","tags":["Classification","Bayesian","GeneExpression"],"description":"This package calculates probabilistic pathway scores using\ngene expression data. Gene expression values are aggregated\ninto pathway-based scores using Bayesian network\nrepresentations of biological pathways.","downloads_month":139,"downloads_total":3202,"page":"http://bioconductor.org/packages/release/bioc/html/PROPS.html"},{"authors":"Samuel Wieczorek [cre, aut], Florence Combes [aut] and Thomas\nBurger [aut]","name":"Prostar","license":"Artistic-2.0","tags":["ImmunoOncology","MassSpectrometry","Normalization","Preprocessing","R.utils Proteomics","GO","GUI"],"description":"This package provides a GUI interface for DAPAR.","downloads_month":213,"downloads_total":15888,"page":"http://bioconductor.org/packages/release/bioc/html/Prostar.html"},{"authors":"Gatto Laurent [aut, cre] and Sonali Arora [aut]","name":"ProteomicsAnnotationHubData","license":"Artistic-2.0","tags":["DataImport","Proteomics"],"description":"These recipes convert a variety and a growing number of\npublic proteomics data sets into easily-used standard\nBioconductor data structures.","downloads_month":168,"downloads_total":10626,"page":"http://bioconductor.org/packages/release/bioc/html/ProteomicsAnnotationHubData.html"},{"authors":"Yuliya V Karpievitch, Tim Stuart and Sufyaan Mohamed","name":"ProteoMM","license":"MIT","tags":["ImmunoOncology","MassSpectrometry","Proteomics","Normalization","DifferentialExpression"],"description":"ProteoMM is a statistical method to perform model-based\npeptide-level differential expression analysis of single or\nmultiple datasets. For multiple datasets ProteoMM produces a\nsingle fold change and p-value for each protein across multiple\ndatasets. ProteoMM provides functionality for normalization,\nmissing value imputation and differential expression.\nModel-based peptide-level imputation and differential\nexpression analysis component of package follows the analysis\ndescribed in “A statistical framework for protein quantitation\nin bottom-up MS based proteomics\" (Karpievitch et al.\nBioinformatics 2009). EigenMS normalisation is implemented as\ndescribed in \"Normalization of peak intensities in bottom-up\nMS-based proteomics using singular value decomposition.\"\n(Karpievitch et al. Bioinformatics 2009).","downloads_month":133,"downloads_total":1008,"page":"http://bioconductor.org/packages/release/bioc/html/ProteoMM.html"},{"authors":"Bo Wen <wenbo@genomics.cn> and Laurent Gatto <lg390@cam.ac.uk>","name":"proteoQC","license":"LGPL-2","tags":["ImmunoOncology","Proteomics","MassSpectrometry","QualityControl","Visualization","ReportWriting"],"description":"This package creates an HTML format QC report for\nMS/MS-based proteomics data. The report is intended to allow\nthe user to quickly assess the quality of proteomics data.","downloads_month":175,"downloads_total":18854,"page":"http://bioconductor.org/packages/release/bioc/html/proteoQC.html"},{"authors":"Laurent Gatto <lg390@cam.ac.uk>","name":"ProtGenerics","license":"Artistic-2.0","tags":["Infrastructure","Proteomics","MassSpectrometry"],"description":"S4 generic functions needed by Bioconductor proteomics\npackages.","downloads_month":7456,"downloads_total":325600,"page":"http://bioconductor.org/packages/release/bioc/html/ProtGenerics.html"},{"authors":"Alexandre Kuhn <alexandre.m.kuhn@gmail.com>","name":"PSEA","license":"Artistic-2.0","tags":["Software"],"description":"Deconvolution of gene expression data by\nPopulation-Specific Expression Analysis (PSEA).","downloads_month":154,"downloads_total":15324,"page":"http://bioconductor.org/packages/release/bioc/html/PSEA.html"},{"authors":"Nuno Saraiva-Agostinho [aut, cre], Nuno Luís Barbosa-Morais\n[aut, led, ths], André Falcão [ths], Lina Gallego Paez [ctb], Marie Bordone [ctb], Teresa Maia [ctb], Mariana Ferreira [ctb], Ana Carolina Leote [ctb] and Bernardo de Almeida [ctb]","name":"psichomics","license":"MIT + file LICENSE","tags":["Sequencing","RNASeq","AlternativeSplicing","DifferentialSplicing","Transcription","GUI","PrincipalComponent","Survival","BiomedicalInformatics","Transcriptomics","ImmunoOncology","Visualization","MultipleComparison","GeneExpression","DifferentialExpression"],"description":"Interactive R package with an intuitive Shiny-based\ngraphical interface for alternative splicing quantification and\nintegrative analyses of alternative splicing and gene\nexpression based on The Cancer Genome Atlas (TCGA), the\nGenotype-Tissue Expression project (GTEx), Sequence Read\nArchive (SRA) and user-provided data. The tool interactively\nperforms survival, dimensionality reduction and median- and\nvariance-based differential splicing and gene expression\nanalyses that benefit from the incorporation of clinical and\nmolecular sample-associated features (such as tumour stage or\nsurvival). Interactive visual access to genomic mapping and\nfunctional annotation of selected alternative splicing events\nis also included.","downloads_month":195,"downloads_total":8976,"page":"http://bioconductor.org/packages/release/bioc/html/psichomics.html"},{"authors":"Paul Shannon","name":"PSICQUIC","license":"Apache License 2.0","tags":["DataImport","GraphAndNetwork","ThirdPartyClient"],"description":"PSICQUIC is a project within the HUPO Proteomics Standard\nInitiative (HUPO-PSI).  It standardises programmatic access to\nmolecular interaction databases.","downloads_month":196,"downloads_total":27358,"page":"http://bioconductor.org/packages/release/bioc/html/PSICQUIC.html"},{"authors":"Alba Gutierrez-Sacristan [aut, cre], Carles Hernandez-Ferrer\n[aut], Jaun R. Gonzalez [aut] and Laura I. Furlong [aut]","name":"psygenet2r","license":"MIT + file LICENSE","tags":["Software","BiomedicalInformatics","Genetics","Infrastructure","DataImport","DataRepresentation"],"description":"Package to retrieve data from PsyGeNET database\n(www.psygenet.org) and to perform comorbidity studies with\nPsyGeNET's and user's data.","downloads_month":152,"downloads_total":9320,"page":"http://bioconductor.org/packages/release/bioc/html/psygenet2r.html"},{"authors":"Richard D. Pearson, Xuejun Liu, Magnus Rattray, Marta Milo, Neil D. Lawrence, Guido Sanguinetti and Li Zhang","name":"puma","license":"LGPL","tags":["Microarray","OneChannel","Preprocessing","DifferentialExpression","Clustering","ExonArray","GeneExpression","mRNAMicroarray","ChipOnChip","AlternativeSplicing","DifferentialSplicing","Bayesian","TwoChannel","DataImport","HTA2.0"],"description":"Most analyses of Affymetrix GeneChip data (including\ntranditional 3' arrays and exon arrays and Human Transcriptome\nArray 2.0) are based on point estimates of expression levels\nand ignore the uncertainty of such estimates. By propagating\nuncertainty to downstream analyses we can improve results from\nmicroarray analyses. For the first time, the puma package makes\na suite of uncertainty propagation methods available to a\ngeneral audience. In additon to calculte gene expression from\nAffymetrix 3' arrays, puma also provides methods to process\nexon arrays and produces gene and isoform expression for\nalternative splicing study. puma also offers improvements in\nterms of scope and speed of execution over previously available\nuncertainty propagation methods. Included are summarisation,\ndifferential expression detection, clustering and PCA methods,\ntogether with useful plotting functions.","downloads_month":245,"downloads_total":69036,"page":"http://bioconductor.org/packages/release/bioc/html/puma.html"},{"authors":"Markus Riester [aut, cre] and Angad P. Singh [aut]","name":"PureCN","license":"Artistic-2.0","tags":["CopyNumberVariation","Software","Sequencing","VariantAnnotation","VariantDetection","Coverage","ImmunoOncology"],"description":"This package estimates tumor purity, copy number, and loss\nof heterozygosity (LOH), and classifies single nucleotide\nvariants (SNVs) by somatic status and clonality. PureCN is\ndesigned for targeted short read sequencing data, integrates\nwell with standard somatic variant detection and copy number\npipelines, and has support for tumor samples without matching\nnormal samples.","downloads_month":301,"downloads_total":14434,"page":"http://bioconductor.org/packages/release/bioc/html/PureCN.html"},{"authors":"Jun Lu and Pierre R. Bushel","name":"pvac","license":"LGPL (>= 2.0)","tags":["Microarray","OneChannel","QualityControl"],"description":"The package contains the function for filtering genes by\nthe proportion of variation accounted for by the first\nprincipal component (PVAC).","downloads_month":143,"downloads_total":34034,"page":"http://bioconductor.org/packages/release/bioc/html/pvac.html"},{"authors":"Pierre Bushel <bushel@niehs.nih.gov>","name":"pvca","license":"LGPL (>= 2.0)","tags":["Microarray","BatchEffect"],"description":"This package contains the function to assess the batch\nsourcs by fitting all \"sources\" as random effects including\ntwo-way interaction terms in the Mixed Model(depends on lme4\npackage) to selected principal components, which were obtained\nfrom the original data correlation matrix. This package\naccompanies the book \"Batch Effects and Noise in Microarray\nExperiements, chapter 12.","downloads_month":284,"downloads_total":36072,"page":"http://bioconductor.org/packages/release/bioc/html/pvca.html"},{"authors":"Renan Sauteraud, Mike Jiang and Raphael Gottardo","name":"Pviz","license":"Artistic-2.0","tags":["Visualization","Proteomics","Microarray"],"description":"Pviz adapts the Gviz package for protein sequences and\ndata.","downloads_month":163,"downloads_total":18798,"page":"http://bioconductor.org/packages/release/bioc/html/Pviz.html"},{"authors":"Robert Stojnic and Diego Diez","name":"PWMEnrich","license":"LGPL (>= 2)","tags":["MotifAnnotation","SequenceMatching","Software"],"description":"A toolkit of high-level functions for DNA motif scanning\nand enrichment analysis built upon Biostrings. The main\nfunctionality is PWM enrichment analysis of already known PWMs\n(e.g. from databases such as MotifDb), but the package also\nimplements high-level functions for PWM scanning and\nvisualisation. The package does not perform \"de novo\" motif\ndiscovery, but is instead focused on using motifs that are\neither experimentally derived or computationally constructed by\nother tools.","downloads_month":227,"downloads_total":56704,"page":"http://bioconductor.org/packages/release/bioc/html/PWMEnrich.html"},{"authors":"Astrid Wachter <Astrid.Wachter@med.uni-goettingen.de>","name":"pwOmics","license":"GPL (>= 2)","tags":["SystemsBiology","Transcription","GeneTarget","GeneSignaling"],"description":"pwOmics performs pathway-based level-specific data\ncomparison of matching omics data sets based on pre-analysed\nuser-specified lists of differential genes/transcripts and\nphosphoproteins. A separate downstream analysis of\nphosphoproteomic data including pathway identification,\ntranscription factor identification and target gene\nidentification is opposed to the upstream analysis starting\nwith gene or transcript information as basis for identification\nof upstream transcription factors and potential proteomic\nregulators. The cross-platform comparative analysis allows for\ncomprehensive analysis of single time point experiments and\ntime-series experiments by providing static and dynamic\nanalysis tools for data integration. In addition, it provides\nfunctions to identify individual signaling axes based on data\nintegration.","downloads_month":154,"downloads_total":12720,"page":"http://bioconductor.org/packages/release/bioc/html/pwOmics.html"},{"authors":"Laurent Gatto [aut and cre]","name":"qcmetrics","license":"GPL-2","tags":["ImmunoOncology","Software","QualityControl","Proteomics","Microarray","MassSpectrometry","Visualization","ReportWriting"],"description":"The package provides a framework for generic quality\ncontrol of data. It permits to create, manage and visualise\nindividual or sets of quality control metrics and generate\nquality control reports in various formats.","downloads_month":202,"downloads_total":23708,"page":"http://bioconductor.org/packages/release/bioc/html/qcmetrics.html"},{"authors":"Ilari Scheinin [aut], Daoud Sie [aut, cre] and Henrik Bengtsson\n[aut]","name":"QDNAseq","license":"GPL","tags":["CopyNumberVariation","DNASeq","Genetics","GenomeAnnotation","Preprocessing","QualityControl","Sequencing"],"description":"Quantitative DNA sequencing for chromosomal aberrations.\nThe genome is divided into non-overlapping fixed-sized bins,\nnumber of sequence reads in each counted, adjusted with a\nsimultaneous two-dimensional loess correction for sequence\nmappability and GC content, and filtered to remove spurious\nregions in the genome. Downstream steps of segmentation and\ncalling are also implemented via packages DNAcopy and CGHcall,\nrespectively.","downloads_month":410,"downloads_total":36766,"page":"http://bioconductor.org/packages/release/bioc/html/QDNAseq.html"},{"authors":"Jessica Mar","name":"qpcrNorm","license":"LGPL (>= 2)","tags":["Preprocessing","GeneExpression"],"description":"The package contains functions to perform normalization of\nhigh-throughput qPCR data. Basic functions for processing raw\nCt data plus functions to generate diagnostic plots are also\navailable.","downloads_month":151,"downloads_total":44750,"page":"http://bioconductor.org/packages/release/bioc/html/qpcrNorm.html"},{"authors":"Robert Castelo [aut, cre] and Alberto Roverato [aut]","name":"qpgraph","license":"GPL (>= 2)","tags":["Microarray","GeneExpression","Transcription","Pathways","NetworkInference","GraphAndNetwork","GeneRegulation","Genetics","GeneticVariability","SNP","Software"],"description":"Estimate gene and eQTL networks from high-throughput\nexpression and genotyping assays.","downloads_month":303,"downloads_total":76744,"page":"http://bioconductor.org/packages/release/bioc/html/qpgraph.html"},{"authors":"Matthew Eldridge [aut], Kamal Kishore [aut], Ashley Sawle [aut and cre]","name":"qPLEXanalyzer","license":"GPL-2","tags":["ImmunoOncology","Proteomics","MassSpectrometry","Normalization","Preprocessing","QualityControl","DataImport"],"description":"Tools for quantitative proteomics data analysis generated\nfrom qPLEX-RIME method.","downloads_month":111,"downloads_total":948,"page":"http://bioconductor.org/packages/release/bioc/html/qPLEXanalyzer.html"},{"authors":"Vince Buffalo","name":"qrqc","license":"GPL (>=2)","tags":["Sequencing","QualityControl","DataImport","Preprocessing","Visualization"],"description":"Quickly scans reads and gathers statistics on base and\nquality frequencies, read length, k-mers by position, and\nfrequent sequences. Produces graphical output of statistics for\nuse in quality control pipelines, and an optional HTML quality\nreport. S4 SequenceSummary objects allow specific tests and\nfunctionality to be written around the data collected.","downloads_month":280,"downloads_total":55064,"page":"http://bioconductor.org/packages/release/bioc/html/qrqc.html"},{"authors":"Matthias Lienhard, Lukas Chavez and Ralf Herwig","name":"qsea","license":"GPL (>=2)","tags":["Sequencing","DNAMethylation","CpGIsland","ChIPSeq","Preprocessing","Normalization","QualityControl","Visualization","CopyNumberVariation","ChipOnChip","DifferentialMethylation"],"description":"qsea (quantitative sequencing enrichment analysis) was\ndeveloped as the successor of the MEDIPS package for analyzing\ndata derived from methylated DNA immunoprecipitation (MeDIP)\nexperiments followed by sequencing (MeDIP-seq). However, qsea\nprovides several functionalities for the analysis of other\nkinds of quantitative sequencing data (e.g. ChIP-seq, MBD-seq,\nCMS-seq and others) including calculation of differential\nenrichment between groups of samples.","downloads_month":154,"downloads_total":7504,"page":"http://bioconductor.org/packages/release/bioc/html/qsea.html"},{"authors":"Mercedes Guerrero-Murillo and Josep Gregori i Font","name":"QSutils","license":"file LICENSE","tags":["Software","Genetics","DNASeq","GeneticVariability","Sequencing","Alignment","SequenceMatching","DataImport"],"description":"Set of utility functions for viral quasispecies analysis\nwith NGS data. Most functions are equally useful for\nmetagenomic studies. There are three main types: (1) data\nmanipulation and exploration—functions useful for converting\nreads to haplotypes and frequencies, repairing reads,\nintersecting strand haplotypes, and visualizing haplotype\nalignments. (2) diversity indices—functions to compute\ndiversity and entropy, in which incidence, abundance, and\nfunctional indices are considered. (3) data\nsimulation—functions useful for generating random viral\nquasispecies data.","downloads_month":605,"downloads_total":5192,"page":"http://bioconductor.org/packages/release/bioc/html/QSutils.html"},{"authors":"Mike Jiang, Greg Finak and Raphael Gottardo","name":"QUALIFIER","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","FlowCytometry","CellBasedAssays"],"description":"Provides quality control and quality assessment tools for\ngated flow cytometry data.","downloads_month":220,"downloads_total":30528,"page":"http://bioconductor.org/packages/release/bioc/html/QUALIFIER.html"},{"authors":"Stephanie Hicks and Rafael Irizarry","name":"quantro","license":"GPL (>=3)","tags":["Normalization","Preprocessing","MultipleComparison","Microarray","Sequencing"],"description":"A data-driven test for the assumptions of quantile\nnormalization using raw data such as objects that inherit eSets\n(e.g. ExpressionSet, MethylSet). Group level information about\neach sample (such as Tumor / Normal status) must also be\nprovided because the test assesses if there are global\ndifferences in the distributions between the user-defined\ngroups.","downloads_month":232,"downloads_total":20778,"page":"http://bioconductor.org/packages/release/bioc/html/quantro.html"},{"authors":"Jan Oosting, Paul Eilers and Renee Menezes","name":"quantsmooth","license":"GPL-2","tags":["Visualization","CopyNumberVariation"],"description":"Implements quantile smoothing as introduced in: Quantile\nsmoothing of array CGH data; Eilers PH, de Menezes RX;\nBioinformatics. 2005 Apr 1;21(7):1146-53.","downloads_month":654,"downloads_total":105606,"page":"http://bioconductor.org/packages/release/bioc/html/quantsmooth.html"},{"authors":"Gregory Ryslik, Yuwei Cheng and Hongyu Zhao","name":"QuartPAC","license":"GPL-2","tags":["Clustering","Proteomics","SomaticMutation"],"description":"Identifies clustering of somatic mutations in proteins\nover the entire quaternary structure.","downloads_month":150,"downloads_total":12446,"page":"http://bioconductor.org/packages/release/bioc/html/QuartPAC.html"},{"authors":"Anita Lerch, Charlotte Soneson, Dimos Gaiditzis and Michael\nStadler","name":"QuasR","license":"GPL-2","tags":["Genetics","Preprocessing","Sequencing","ChIPSeq","RNASeq","MethylSeq","Coverage","Alignment","QualityControl","ImmunoOncology"],"description":"This package provides a framework for the quantification\nand analysis of Short Reads. It covers a complete workflow\nstarting from raw sequence reads, over creation of alignments\nand quality control plots, to the quantification of genomic\nregions of interest.","downloads_month":715,"downloads_total":82574,"page":"http://bioconductor.org/packages/release/bioc/html/QuasR.html"},{"authors":"Carl Tony Fakhry [cre, aut], Ping Chen [ths], Kourosh\nZarringhalam [aut and ths]","name":"QuaternaryProd","license":"GPL (>=3)","tags":["GraphAndNetwork","GeneExpression","Transcription"],"description":"QuaternaryProd is an R package that performs causal\nreasoning on biological networks, including publicly available\nnetworks such as STRINGdb. QuaternaryProd is an open-source\nalternative to commercial products such as Inginuity Pathway\nAnalysis. For a given a set of differentially expressed genes,\nQuaternaryProd computes the significance of upstream regulators\nin the network by performing causal reasoning using the\nQuaternary Dot Product Scoring Statistic (Quaternary\nStatistic), Ternary Dot product Scoring Statistic (Ternary\nStatistic) and Fisher's exact test (Enrichment test). The\nQuaternary Statistic handles signed, unsigned and ambiguous\nedges in the network. Ambiguity arises when the direction of\ncausality is unknown, or when the source node (e.g., a protein)\nhas edges with conflicting signs for the same target gene. On\nthe other hand, the Ternary Statistic provides causal reasoning\nusing the signed and unambiguous edges only. The Vignette\nprovides more details on the Quaternary Statistic and\nillustrates an example of how to perform causal reasoning using\nSTRINGdb.","downloads_month":141,"downloads_total":8386,"page":"http://bioconductor.org/packages/release/bioc/html/QuaternaryProd.html"},{"authors":"Yu Zhang [aut, cre] and Qin Ma [aut]","name":"QUBIC","license":"CC BY-NC-ND 4.0 + file LICENSE","tags":["StatisticalMethod","Microarray","DifferentialExpression","MultipleComparison","Clustering","Visualization","GeneExpression","Network"],"description":"The core function of this R package is to provide the\nimplementation of the well-cited and well-reviewed QUBIC\nalgorithm, aiming to deliver an effective and efficient\nbiclustering capability. This package also includes the\nfollowing related functions: (i) a qualitative representation\nof the input gene expression data, through a well-designed\ndiscretization way considering the underlying data property,\nwhich can be directly used in other biclustering programs; (ii)\nvisualization of identified biclusters using heatmap in support\nof overall expression pattern analysis; (iii) bicluster-based\nco-expression network elucidation and visualization, where\ndifferent correlation coefficient scores between a pair of\ngenes are provided; and (iv) a generalize output format of\nbiclusters and corresponding network can be freely downloaded\nso that a user can easily do following comprehensive functional\nenrichment analysis (e.g. DAVID) and advanced network\nvisualization (e.g. Cytoscape).","downloads_month":1075,"downloads_total":24518,"page":"http://bioconductor.org/packages/release/bioc/html/QUBIC.html"},{"authors":"Christopher Bolen, Gur Yaari, with contributions from Juilee\nThakar, Hailong Meng, Jacob Turner, Derek Blankenship and and\nSteven Kleinstein","name":"qusage","license":"GPL (>= 2)","tags":["GeneSetEnrichment","Microarray","RNASeq","Software"],"description":"This package is an implementation the Quantitative Set\nAnalysis for Gene Expression (QuSAGE) method described in\n(Yaari G. et al, Nucl Acids Res, 2013). This is a novel Gene\nSet Enrichment-type test, which is designed to provide a\nfaster, more accurate, and easier to understand test for gene\nexpression studies. qusage accounts for inter-gene correlations\nusing the Variance Inflation Factor technique proposed by Wu et\nal. (Nucleic Acids Res, 2012). In addition, rather than simply\nevaluating the deviation from a null hypothesis with a single\nnumber (a P value), qusage quantifies gene set activity with a\ncomplete probability density function (PDF). From this PDF, P\nvalues and confidence intervals can be easily extracted.\nPreserving the PDF also allows for post-hoc analysis (e.g.,\npair-wise comparisons of gene set activity) while maintaining\nstatistical traceability. Finally, while qusage is compatible\nwith individual gene statistics from existing methods (e.g.,\nLIMMA), a Welch-based method is implemented that is shown to\nimprove specificity. For questions, contact Chris Bolen\n(cbolen1@gmail.com) or Steven Kleinstein\n(steven.kleinstein@yale.edu)","downloads_month":515,"downloads_total":34644,"page":"http://bioconductor.org/packages/release/bioc/html/qusage.html"},{"authors":"John D. Storey with contributions from Andrew J. Bass, Alan\nDabney and David Robinson","name":"qvalue","license":"LGPL","tags":["MultipleComparisons"],"description":"This package takes a list of p-values resulting from the\nsimultaneous testing of many hypotheses and estimates their\nq-values and local FDR values. The q-value of a test measures\nthe proportion of false positives incurred (called the false\ndiscovery rate) when that particular test is called\nsignificant. The local FDR measures the posterior probability\nthe null hypothesis is true given the test's p-value. Various\nplots are automatically generated, allowing one to make\nsensible significance cut-offs. Several mathematical results\nhave recently been shown on the conservative accuracy of the\nestimated q-values from this software. The software can be\napplied to problems in genomics, brain imaging, astrophysics,\nand data mining.","downloads_month":12244,"downloads_total":851756,"page":"http://bioconductor.org/packages/release/bioc/html/qvalue.html"},{"authors":"Djekidel MN and Yang Chen et al.","name":"R3CPET","license":"GPL (>=2)","tags":["NetworkInference","GenePrediction","Bayesian","GraphAndNetwork","Network","GeneExpression","HiC"],"description":"The package provides a method to infer the set of proteins\nthat are more probably to work together to maintain chormatin\ninteraction given a ChIA-PET experiment results.","downloads_month":150,"downloads_total":12402,"page":"http://bioconductor.org/packages/release/bioc/html/R3CPET.html"},{"authors":"Supat Thongjuea, MRC WIMM Centre for Computational Biology, Weatherall Institute of Molecular Medicine, University of\nOxford and UK <supat.thongjuea@imm.ox.ac.uk>","name":"r3Cseq","license":"GPL-3","tags":["Preprocessing","Sequencing"],"description":"This package is used for the analysis of long-range\nchromatin interactions from 3C-seq assay.","downloads_month":233,"downloads_total":42718,"page":"http://bioconductor.org/packages/release/bioc/html/r3Cseq.html"},{"authors":"Hans-Ulrich Klein, Christoph Bartenhagen and Christian Ruckert","name":"R453Plus1Toolbox","license":"LGPL-3","tags":["Sequencing","Infrastructure","DataImport","DataRepresentation","Visualization","QualityControl","ReportWriting"],"description":"The R453Plus1 Toolbox comprises useful functions for the\nanalysis of data generated by Roche's 454 sequencing platform.\nIt adds functions for quality assurance as well as for\nannotation and visualization of detected variants,\ncomplementing the software tools shipped by Roche with their\nproduct. Further, a pipeline for the detection of structural\nvariants is provided.","downloads_month":150,"downloads_total":38656,"page":"http://bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html"},{"authors":"Daniel Lai and Irmtraud Meyer <irmtraud.meyer@cantab.net>","name":"R4RNA","license":"GPL-3","tags":["Alignment","MultipleSequenceAlignment","Preprocessing","Visualization","DataImport","DataRepresentation","MultipleComparison"],"description":"A package for RNA basepair analysis, including the\nvisualization of basepairs as arc diagrams for easy comparison\nand annotation of sequence and structure.  Arc diagrams can\nadditionally be projected onto multiple sequence alignments to\nassess basepair conservation and covariation, with numerical\nmethods for computing statistics for each.","downloads_month":127,"downloads_total":8486,"page":"http://bioconductor.org/packages/release/bioc/html/R4RNA.html"},{"authors":"Martin Morgan [aut, cre] and Marcel Ramos [aut]","name":"RaggedExperiment","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"This package provides a flexible representation of copy\nnumber, mutation, and other data that fit into the ragged array\nschema for genomic location data. The basic representation of\nsuch data provides a rectangular flat table interface to the\nuser with range information in the rows and samples/specimen in\nthe columns.","downloads_month":754,"downloads_total":22784,"page":"http://bioconductor.org/packages/release/bioc/html/RaggedExperiment.html"},{"authors":"Paul F. Thaben and Pål O. Westermark","name":"rain","license":"GPL-2","tags":["TimeCourse","Genetics","SystemsBiology","Proteomics","Microarray","MultipleComparison"],"description":"This package uses non-parametric methods to detect rhythms\nin time series. It deals with outliers, missing values and is\noptimized for time series comprising 10-100 measurements. As it\ndoes not assume expect any distinct waveform it is optimal or\ndetecting oscillating behavior (e.g. circadian or cell cycle)\nin e.g. genome- or proteome-wide biological measurements such\nas: micro arrays, proteome mass spectrometry, or metabolome\nmeasurements.","downloads_month":172,"downloads_total":19958,"page":"http://bioconductor.org/packages/release/bioc/html/rain.html"},{"authors":"Raphael Gottardo","name":"rama","license":"GPL (>= 2)","tags":["Microarray","TwoChannel","QualityControl","Preprocessing"],"description":"Robust estimation of cDNA microarray intensities with\nreplicates. The package uses a Bayesian hierarchical model for\nthe robust estimation. Outliers are modeled explicitly using a\nt-distribution, and the model also addresses classical issues\nsuch as design effects, normalization, transformation, and\nnonconstant variance.","downloads_month":131,"downloads_total":48952,"page":"http://bioconductor.org/packages/release/bioc/html/rama.html"},{"authors":"Andrey A Shabalin [aut, cre]\n(<https://orcid.org/0000-0003-0309-6821>), Shaunna L Clark\n[aut], Mohammad W Hattab [aut], Karolina A Aberg [aut] and Edwin J\nC G van den Oord [aut]","name":"ramwas","license":"LGPL-3","tags":["DNAMethylation","Sequencing","QualityControl","Coverage","Preprocessing","Normalization","BatchEffect","PrincipalComponent","DifferentialMethylation","Visualization"],"description":"A complete toolset for methylome-wide association studies\n(MWAS). It is specifically designed for data from enrichment\nbased methylation assays, but can be applied to other data as\nwell. The analysis pipeline includes seven steps: (1) scanning\naligned reads from BAM files, (2) calculation of quality\ncontrol measures, (3) creation of methylation score (coverage)\nmatrix, (4) principal component analysis for capturing batch\neffects and detection of outliers, (5) association analysis\nwith respect to phenotypes of interest while correcting for top\nPCs and known covariates, (6) annotation of significant\nfindings, and (7) multi-marker analysis (methylation risk\nscore) using elastic net. Additionally, RaMWAS include tools\nfor joint analysis of methlyation and genotype data. This work\nis published in Bioinformatics, Shabalin et al. (2018)\n<doi:10.1093/bioinformatics/bty069>.","downloads_month":145,"downloads_total":5462,"page":"http://bioconductor.org/packages/release/bioc/html/ramwas.html"},{"authors":"Alberto Valdeolivas Urbelz <alvaldeolivas@gmail.com>","name":"RandomWalkRestartMH","license":"GPL (>= 2)","tags":["GenePrediction","NetworkInference","SomaticMutation","BiomedicalInformatics","MathematicalBiology","SystemsBiology","GraphAndNetwork","Pathways","BioCarta","KEGG","Reactome","Network"],"description":"This package performs Random Walk with Restart on\nmultiplex and heterogeneous networks. It is described in the\nfollowing article: \"Random Walk With Restart On Multiplex And\nHeterogeneous Biological Networks\".\nhttps://www.biorxiv.org/content/early/2017/08/30/134734 .","downloads_month":138,"downloads_total":2134,"page":"http://bioconductor.org/packages/release/bioc/html/RandomWalkRestartMH.html"},{"authors":"Vincent Carey <stvjc@channing.harvard.edu> and Robert Gentleman","name":"randPack","license":"Artistic 2.0","tags":["StatisticalMethod"],"description":"A suite of classes and functions for randomizing patients\nin clinical trials.","downloads_month":185,"downloads_total":30774,"page":"http://bioconductor.org/packages/release/bioc/html/randPack.html"},{"authors":"Francesco Del Carratore\n<francesco.delcarratore@postgrad.manchester.ac.uk>, Andris\nJankevics <andris.jankevics@gmail.com> Fangxin Hong\n<fxhong@jimmy.harvard.edu>, Ben Wittner\n<Wittner.Ben@mgh.harvard.edu>, Rainer Breitling\n<rainer.breitling@manchester.ac.uk>, and Florian Battke\n<battke@informatik.uni-tuebingen.de>","name":"RankProd","license":"file LICENSE","tags":["DifferentialExpression","StatisticalMethod","Software","ResearchField","Metabolomics","Lipidomics","Proteomics","SystemsBiology","GeneExpression","Microarray","GeneSignaling"],"description":"Non-parametric method for identifying differentially\nexpressed (up- or down- regulated) genes based on the estimated\npercentage of false predictions (pfp). The method can combine\ndata sets from different origins (meta-analysis) to increase\nthe power of the identification.","downloads_month":681,"downloads_total":148502,"page":"http://bioconductor.org/packages/release/bioc/html/RankProd.html"},{"authors":"Adam Gudys and Tomasz Stokowy","name":"RareVariantVis","license":"Artistic-2.0","tags":["GenomicVariation","Sequencing","WholeGenome"],"description":"Second version of RareVariantVis package aims to provide\ncomprehensive information about rare variants for your genome\ndata. It annotates, filters and presents genomic variants\n(especially rare ones) in a global, per chromosome way. For\ndiscovered rare variants CRISPR guide RNAs are designed, so the\nuser can plan further functional studies. Large structural\nvariants, including copy number variants are also supported.\nPackage accepts variants directly from variant caller - for\nexample GATK or Speedseq. Output of package are lists of\nvariants together with adequate visualization. Visualization of\nvariants is performed in two ways - standard that outputs png\nfigures and interactive that uses JavaScript d3 package.\nInteractive visualization allows to analyze trio/family data,\nfor example in search for causative variants in rare Mendelian\ndiseases, in point-and-click interface. The package includes\nhomozygous region caller and allows to analyse whole human\ngenomes in less than 30 minutes on a desktop computer.\nRareVariantVis disclosed novel causes of several rare monogenic\ndisorders, including one with non-coding causative variant -\nkeratolythic winter erythema.","downloads_month":135,"downloads_total":11008,"page":"http://bioconductor.org/packages/release/bioc/html/RareVariantVis.html"},{"authors":"Julian Gehring, Simon Anders and Bernd Klaus","name":"Rariant","license":"GPL-3","tags":["Sequencing","StatisticalMethod","GenomicVariation","SomaticMutation","VariantDetection","Visualization"],"description":"The 'Rariant' package identifies single nucleotide\nvariants from sequencing data based on the difference of\nbinomially distributed mismatch rates between matched samples.","downloads_month":144,"downloads_total":19432,"page":"http://bioconductor.org/packages/release/bioc/html/Rariant.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu> and Wolfgang Huber\n<huber@ebi.ac.uk>","name":"RbcBook1","license":"Artistic-2.0","tags":["Software"],"description":"tools for building book","downloads_month":183,"downloads_total":57528,"page":"http://bioconductor.org/packages/release/bioc/html/RbcBook1.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu>, Li Long\n<li.long@isb-sib.ch> and R. Gentleman","name":"RBGL","license":"Artistic-2.0","tags":["GraphAndNetwork","Network"],"description":"A fairly extensive and comprehensive interface to the\ngraph algorithms contained in the BOOST library.","downloads_month":11121,"downloads_total":1160478,"page":"http://bioconductor.org/packages/release/bioc/html/RBGL.html"},{"authors":"Robert Gentleman","name":"RBioinf","license":"Artistic-2.0","tags":["GeneExpression","Microarray","Preprocessing","QualityControl","Classification","Clustering","MultipleComparison","Annotation"],"description":"Functions and datasets and examples to accompany the\nmonograph R For Bioinformatics.","downloads_month":168,"downloads_total":50680,"page":"http://bioconductor.org/packages/release/bioc/html/RBioinf.html"},{"authors":"Frank Kramer","name":"rBiopaxParser","license":"GPL (>= 2)","tags":["DataRepresentation"],"description":"Parses BioPAX files and represents them in R, at the\nmoment BioPAX level 2 and level 3 are supported.","downloads_month":237,"downloads_total":35418,"page":"http://bioconductor.org/packages/release/bioc/html/rBiopaxParser.html"},{"authors":"Dongmei Li and Chin-Yuan Liang","name":"RBM","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression"],"description":"Use A Resampling-Based Empirical Bayes Approach to Assess\nDifferential Expression in Two-Color Microarrays and RNA-Seq\ndata sets.","downloads_month":129,"downloads_total":12008,"page":"http://bioconductor.org/packages/release/bioc/html/RBM.html"},{"authors":"Florian Hahne, Anita Lerch and Michael B Stadler","name":"Rbowtie","license":"Artistic-2.0 | file LICENSE","tags":["Sequencing","Alignment"],"description":"This package provides an R wrapper around the popular\nbowtie short read aligner and around SpliceMap, a de novo\nsplice junction discovery and alignment tool. The package is\nused by the QuasR bioconductor package. We recommend to use the\nQuasR package instead of using Rbowtie directly.","downloads_month":473,"downloads_total":64808,"page":"http://bioconductor.org/packages/release/bioc/html/Rbowtie.html"},{"authors":"Zheng Wei and Wei Zhang","name":"Rbowtie2","license":"GPL (>= 3)","tags":["Sequencing","Alignment","Preprocessing"],"description":"This package provides an R wrapper of the popular bowtie2\nsequencing reads aligner and AdapterRemoval, a convenient tool\nfor rapid adapter trimming, identification, and read merging.","downloads_month":308,"downloads_total":8484,"page":"http://bioconductor.org/packages/release/bioc/html/Rbowtie2.html"},{"authors":"HyungJun Cho <hj4cho@korea.ac.kr>, Sukwoo Kim\n<s4kim@korea.ac.kr>, Soo-heang Eo <hanansh@korea.ac.kr> and Jaewoo\nKang <kangj@korea.ac.kr>","name":"rbsurv","license":"GPL (>= 2)","tags":["Microarray"],"description":"This package selects genes associated with survival.","downloads_month":197,"downloads_total":45538,"page":"http://bioconductor.org/packages/release/bioc/html/rbsurv.html"},{"authors":"Jonathan Cairns","name":"Rcade","license":"GPL-2","tags":["DifferentialExpression","GeneExpression","Transcription","ChIPSeq","Sequencing","Genetics"],"description":"Rcade (which stands for \"R-based analysis of ChIP-seq And\nDifferential Expression\") is a tool for integrating ChIP-seq\ndata with differential expression summary data, through a\nBayesian framework. A key application is in identifing the\ngenes targeted by a transcription factor of interest - that is,\nwe collect genes that are associated with a ChIP-seq peak, and\ndifferential expression under some perturbation related to that\nTF.","downloads_month":168,"downloads_total":29316,"page":"http://bioconductor.org/packages/release/bioc/html/Rcade.html"},{"authors":"Bora Uyar [aut, cre], Dilmurat Yusuf [aut], Ricardo Wurmus\n[aut] and Altuna Akalin [aut]","name":"RCAS","license":"Artistic-2.0","tags":["Software","GeneTarget","MotifAnnotation","MotifDiscovery","GO","Transcriptomics","GenomeAnnotation","GeneSetEnrichment","Coverage"],"description":"RCAS is an automated system that provides dynamic genome\nannotations for custom input files that contain transcriptomic\nregions. Such transcriptomic regions could be, for instance,\npeak regions detected by CLIP-Seq analysis that detect\nprotein-RNA interactions, RNA modifications (alias the\nepitranscriptome), CAGE-tag locations, or any other collection\nof target regions at the level of the transcriptome. RCAS is\ndesigned as a reporting tool for the functional analysis of\nRNA-binding sites detected by high-throughput experiments. It\ntakes as input a BED format file containing the genomic\ncoordinates of the RNA binding sites and a GTF file that\ncontains the genomic annotation features usually provided by\npublicly available databases such as Ensembl and UCSC. RCAS\nperforms overlap operations between the genomic coordinates of\nthe RNA binding sites and the genomic annotation features and\nproduces in-depth annotation summaries such as the distribution\nof binding sites with respect to gene features (exons, introns,\n5'/3' UTR regions, exon-intron boundaries, promoter regions,\nand whole transcripts). Moreover, by detecting the collection\nof targeted transcripts, RCAS can carry out functional\nannotation tables for enriched gene sets (annotated by the\nMolecular Signatures Database) and GO terms. As one of the most\nimportant questions that arise during protein-RNA interaction\nanalysis; RCAS has a module for detecting sequence motifs\nenriched in the targeted regions of the transcriptome. A full\ninteractive report in HTML format can be generated that\ncontains interactive figures and tables that are ready for\npublication purposes.","downloads_month":140,"downloads_total":8350,"page":"http://bioconductor.org/packages/release/bioc/html/RCAS.html"},{"authors":"Douaa Mugahid and Lars Kaderali","name":"RCASPAR","license":"GPL (>=3)","tags":["aCGH","GeneExpression","Genetics","Proteomics","Visualization"],"description":"The package is the R-version of the C-based software\n\\bold{CASPAR} (Kaderali,2006:\n\\url{http://bioinformatics.oxfordjournals.org/content/22/12/1495}).\nIt is meant to help predict survival times in the presence of\nhigh-dimensional explanatory covariates. The model is a\npiecewise baseline hazard Cox regression model with an Lq-norm\nbased prior that selects for the most important regression\ncoefficients, and in turn the most relevant covariates for\nsurvival analysis. It was primarily tried on gene expression\nand aCGH data, but can be used on any other type of\nhigh-dimensional data and in disciplines other than biology and\nmedicine.","downloads_month":131,"downloads_total":30818,"page":"http://bioconductor.org/packages/release/bioc/html/RCASPAR.html"},{"authors":"Augustin Luna, Vinodh Rajapakse and Fabricio Sousa","name":"rcellminer","license":"LGPL-3 + file LICENSE","tags":["aCGH","CellBasedAssays","CopyNumberVariation","GeneExpression","Pharmacogenomics","Pharmacogenetics","miRNA","Cheminformatics","Visualization","Software","SystemsBiology"],"description":"The NCI-60 cancer cell line panel has been used over the\ncourse of several decades as an anti-cancer drug screen. This\npanel was developed as part of the Developmental Therapeutics\nProgram (DTP, http://dtp.nci.nih.gov/) of the U.S. National\nCancer Institute (NCI). Thousands of compounds have been tested\non the NCI-60, which have been extensively characterized by\nmany platforms for gene and protein expression, copy number,\nmutation, and others (Reinhold, et al., 2012). The purpose of\nthe CellMiner project (http://discover.nci.nih.gov/ cellminer)\nhas been to integrate data from multiple platforms used to\nanalyze the NCI-60 and to provide a powerful suite of tools for\nexploration of NCI-60 data.","downloads_month":141,"downloads_total":16518,"page":"http://bioconductor.org/packages/release/bioc/html/rcellminer.html"},{"authors":"Frederic Commo [aut and cre]","name":"rCGH","license":"Artistic-2.0","tags":["aCGH","CopyNumberVariation","Preprocessing","FeatureExtraction"],"description":"A comprehensive pipeline for analyzing and interactively\nvisualizing genomic profiles generated through commercial or\ncustom aCGH arrays. As inputs, rCGH supports Agilent dual-color\nFeature Extraction files (.txt), from 44 to 400K, Affymetrix\nSNP6.0 and cytoScanHD probeset.txt, cychp.txt, and cnchp.txt\nfiles exported from ChAS or Affymetrix Power Tools. rCGH also\nsupports custom arrays, provided data complies with the\nexpected format. This package takes over all the steps required\nfor individual genomic profiles analysis, from reading files to\nprofiles segmentation and gene annotations. This package also\nprovides several visualization functions (static or\ninteractive) which facilitate individual profiles\ninterpretation. Input files can be in compressed format, e.g.\n.bz2 or .gz.","downloads_month":172,"downloads_total":12970,"page":"http://bioconductor.org/packages/release/bioc/html/rCGH.html"},{"authors":"Michael Mahr and Guenter Klambauer","name":"Rchemcpp","license":"GPL (>= 2.1)","tags":["ImmunoOncology","Bioinformatics","CellBasedAssays","Clustering","DataImport","Infrastructure","MicrotitrePlateAssay","Proteomics","Software","Visualization"],"description":"The Rchemcpp package implements the marginalized graph\nkernel and extensions, Tanimoto kernels, graph kernels,\npharmacophore and 3D kernels suggested for measuring the\nsimilarity of molecules.","downloads_month":202,"downloads_total":25172,"page":"http://bioconductor.org/packages/release/bioc/html/Rchemcpp.html"},{"authors":"Adrin Jalali and Nima Aghaeepour","name":"RchyOptimyx","license":"Artistic-2.0","tags":["FlowCytometry"],"description":"Constructs a hierarchy of cells using flow cytometry for\nmaximization of an external variable (e.g., a clinical outcome\nor a cytokine response).","downloads_month":174,"downloads_total":33106,"page":"http://bioconductor.org/packages/release/bioc/html/RchyOptimyx.html"},{"authors":"Sara Aibar, Gert Hulselmans, Stein Aerts. Laboratory of\nComputational Biology. VIB-KU Leuven Center for Brain & Disease\nResearch. Leuven and Belgium","name":"RcisTarget","license":"GPL-3","tags":["GeneRegulation","MotifAnnotation","Transcriptomics","Transcription","GeneSetEnrichment","GeneTarget"],"description":"RcisTarget identifies transcription factor binding motifs\n(TFBS) over-represented on a gene list. In a first step,\nRcisTarget selects DNA motifs that are significantly\nover-represented in the surroundings of the transcription start\nsite (TSS) of the genes in the gene-set. This is achieved by\nusing a database that contains genome-wide cross-species\nrankings for each motif. The motifs that are then annotated to\nTFs and those that have a high Normalized Enrichment Score\n(NES) are retained. Finally, for each motif and gene-set,\nRcisTarget predicts the candidate target genes (i.e. genes in\nthe gene-set that are ranked above the leading edge).","downloads_month":544,"downloads_total":10488,"page":"http://bioconductor.org/packages/release/bioc/html/RcisTarget.html"},{"authors":"Stijn Hawinkel <stijn.hawinkel@ugent.be>","name":"RCM","license":"GPL-2","tags":["Metagenomics","DimensionReduction","Microbiome","Visualization"],"description":"Combine ideas of log-linear analysis of contingency table,\nflexible response function estimation and empirical Bayes\ndispersion estimation for explorative visualization of\nmicrobiome datasets. The package includes unconstrained as well\nas constrained analysis.","downloads_month":20,"downloads_total":102,"page":"http://bioconductor.org/packages/release/bioc/html/RCM.html"},{"authors":"Nan Xiao [aut, cre], Dong-Sheng Cao [aut] and Qing-Song Xu [aut]","name":"Rcpi","license":"Artistic-2.0 | file LICENSE","tags":["Software","DataImport","DataRepresentation","FeatureExtraction","Cheminformatics","BiomedicalInformatics","Proteomics","GO","SystemsBiology"],"description":"Rcpi offers a molecular informatics toolkit with a\ncomprehensive integration of bioinformatics and\nchemoinformatics tools for drug discovery.","downloads_month":265,"downloads_total":35478,"page":"http://bioconductor.org/packages/release/bioc/html/Rcpi.html"},{"authors":"Qiang Hu [aut and cre]","name":"Rcwl","license":"GPL-2 | file LICENSE","tags":["Software","WorkflowStep"],"description":"The package can be a simple and user-friendly way to\nmanage command line tools and build data analysis pipelines in\nR using Common Workflow Language (CWL).","downloads_month":0,"downloads_total":46,"page":"http://bioconductor.org/packages/release/bioc/html/Rcwl.html"},{"authors":"Alexander Pico, Tanja Muetze, Ruth Isserlin, Shraddha Pai, Julia Gustavsen, Georgi Kolishovski and Paul Shannon","name":"RCy3","license":"MIT + file LICENSE","tags":["Visualization","GraphAndNetwork","ThirdPartyClient","Network"],"description":"Vizualize, analyze and explore networks using Cytoscape\nvia R.","downloads_month":595,"downloads_total":25996,"page":"http://bioconductor.org/packages/release/bioc/html/RCy3.html"},{"authors":"Paul Shannon","name":"RCyjs","license":"MIT + file LICENSE","tags":["Visualization","GraphAndNetwork","ThirdPartyClient"],"description":"Interactive viewing and exploration of graphs, connecting\nR to Cytoscape.js, using websockets.","downloads_month":157,"downloads_total":12490,"page":"http://bioconductor.org/packages/release/bioc/html/RCyjs.html"},{"authors":"Cristobal Fresno and Elmer A. Fernandez","name":"RDAVIDWebService","license":"GPL (>=2)","tags":["Visualization","DifferentialExpression","GraphAndNetwork"],"description":"Tools for retrieving data from the Database for\nAnnotation, Visualization and Integrated Discovery (DAVID)\nusing Web Services into R objects. This package offers the main\nfunctionalities of DAVID website including: i) user friendly\nconnectivity to upload gene/background list/s, change\ngene/background position, select current specie/s, select\nannotations, etc. ii) Reports of the submitted Gene List,\nAnnotation Category Summary, Gene/Term Clusters, Functional\nAnnotation Chart, Functional Annotation Table","downloads_month":747,"downloads_total":78350,"page":"http://bioconductor.org/packages/release/bioc/html/RDAVIDWebService.html"},{"authors":"Thomas Thurnherr, Franziska Singer, Daniel J. Stekhoven and and\nNiko Beerenwinkel","name":"rDGIdb","license":"MIT + file LICENSE","tags":["Software","ResearchField","Pharmacogenetics","Pharmacogenomics","FunctionalGenomics","WorkflowStep","Annotation"],"description":"The rDGIdb package provides a wrapper for the Drug Gene\nInteraction Database (DGIdb). For simplicity, the wrapper query\nfunction and output resembles the user interface and results\nformat provided on the DGIdb website (http://www.dgidb.org/).","downloads_month":143,"downloads_total":7604,"page":"http://bioconductor.org/packages/release/bioc/html/rDGIdb.html"},{"authors":"Anton Pervukhin <apervukh@minet.uni-jena.de> and Steffen Neumann\n<sneumann@ipb-halle.de>","name":"Rdisop","license":"GPL-2","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"Identification of metabolites using high precision mass\nspectrometry. MS Peaks are used to derive a ranked list of sum\nformulae, alternatively for a given sum formula the theoretical\nisotope distribution can be calculated to search in MS peak\nlists.","downloads_month":607,"downloads_total":65840,"page":"http://bioconductor.org/packages/release/bioc/html/Rdisop.html"},{"authors":"Christoph Bartenhagen","name":"RDRToolbox","license":"GPL (>= 2)","tags":["DimensionReduction","FeatureExtraction","Visualization","Clustering","Microarray"],"description":"A package for nonlinear dimension reduction using the\nIsomap and LLE algorithm. It also includes a routine for\ncomputing the Davis-Bouldin-Index for cluster validation, a\nplotting tool and a data generator for microarray gene\nexpression data and for the Swiss Roll dataset.","downloads_month":858,"downloads_total":58456,"page":"http://bioconductor.org/packages/release/bioc/html/RDRToolbox.html"},{"authors":"Guangchuang Yu [aut, cre] and Vladislav Petyuk [ctb]","name":"ReactomePA","license":"GPL-2","tags":["Pathways","Visualization","Annotation","MultipleComparison","GeneSetEnrichment","Reactome"],"description":"This package provides functions for pathway analysis based\non REACTOME pathway database. It implements enrichment\nanalysis, gene set enrichment analysis and several functions\nfor visualization.","downloads_month":11634,"downloads_total":238082,"page":"http://bioconductor.org/packages/release/bioc/html/ReactomePA.html"},{"authors":"Richard Cotton [cre, aut] and Aditya Bhagwat [aut]","name":"readat","license":"GPL-3","tags":["GeneExpression","DataImport","Proteomics","OneChannel","ProprietaryPlatforms"],"description":"This package contains functionality to import, transform\nand annotate data from ADAT files generated by the SomaLogic\nSOMAscan platform.","downloads_month":221,"downloads_total":8934,"page":"http://bioconductor.org/packages/release/bioc/html/readat.html"},{"authors":"James Perkins, Matthias Kohl and Nor Izayu Abdul Rahman","name":"ReadqPCR","license":"LGPL-3","tags":["DataImport","MicrotitrePlateAssay","GeneExpression","qPCR"],"description":"The package provides functions to read raw RT-qPCR data of\ndifferent platforms.","downloads_month":517,"downloads_total":63198,"page":"http://bioconductor.org/packages/release/bioc/html/ReadqPCR.html"},{"authors":"Kyle A. Furge <kyle.furge@vai.org> and Karl Dykema\n<karl.dykema@vai.org>","name":"reb","license":"GPL-2","tags":["Microarray","CopyNumberVariation","Visualization"],"description":"A set of functions to dentify regional expression biases","downloads_month":137,"downloads_total":43626,"page":"http://bioconductor.org/packages/release/bioc/html/reb.html"},{"authors":"Bill Wheeler [cre], Bin Zhu [aut], Lisa Mirabello [ctb] and Nilanjan Chatterjee [ctb]","name":"REBET","license":"GPL-2","tags":["Software","VariantAnnotation","SNP"],"description":"There is an increasing focus to investigate the\nassociation between rare variants and diseases. The REBET\npackage implements the subREgion-based BurdEn Test which is a\npowerful burden test that simultaneously identifies\nsusceptibility loci and sub-regions.","downloads_month":85,"downloads_total":688,"page":"http://bioconductor.org/packages/release/bioc/html/REBET.html"},{"authors":"Leonardo Collado-Torres [aut, cre], Abhinav Nellore [ctb], Andrew E. Jaffe [ctb], Margaret A. Taub [ctb], Kai Kammers\n[ctb], Shannon E. Ellis [ctb], Kasper Daniel Hansen [ctb], Ben\nLangmead [ctb], Jeffrey T. Leek [aut and ths]","name":"recount","license":"Artistic-2.0","tags":["Coverage","DifferentialExpression","GeneExpression","RNASeq","Sequencing","Software","DataImport","ImmunoOncology"],"description":"Explore and download data from the recount project\navailable at https://jhubiostatistics.shinyapps.io/recount/.\nUsing the recount package you can download\nRangedSummarizedExperiment objects at the gene, exon or\nexon-exon junctions level, the raw counts, the phenotype\nmetadata used, the urls to the sample coverage bigWig files or\nthe mean coverage bigWig file for a particular study. The\nRangedSummarizedExperiment objects can be used by different\npackages for performing differential expression analysis. Using\nhttp://bioconductor.org/packages/derfinder you can perform\nannotation-agnostic differential expression analyses with the\ndata from the recount project as described at\nhttp://www.nature.com/nbt/journal/v35/n4/full/nbt.3838.html.","downloads_month":504,"downloads_total":26600,"page":"http://bioconductor.org/packages/release/bioc/html/recount.html"},{"authors":"Panagiotis Moulos <moulos@fleming.gr>","name":"recoup","license":"GPL (>= 3)","tags":["ImmunoOncology","Software","GeneExpression","Preprocessing","QualityControl","RNASeq","ChIPSeq","Sequencing","Coverage"],"description":"recoup calculates and plots signal profiles created from\nshort sequence reads derived from Next Generation Sequencing\ntechnologies. The profiles provided are either sumarized curve\nprofiles or heatmap profiles. Currently, recoup supports\ngenomic profile plots for reads derived from ChIP-Seq and\nRNA-Seq experiments. The package uses ggplot2 and\nComplexHeatmap graphics facilities for curve and heatmap\ncoverage profiles respectively.","downloads_month":183,"downloads_total":8680,"page":"http://bioconductor.org/packages/release/bioc/html/recoup.html"},{"authors":"Mauro Castro, Xin Wang and Florian Markowetz","name":"RedeR","license":"GPL (>= 2)","tags":["Infrastructure","GraphAndNetwork","Software","Network","Visualization","DataRepresentation"],"description":"RedeR is an R-based package combined with a stand-alone\nJava application for interactive visualization and manipulation\nof modular structures, nested networks and multiple levels of\nhierarchical associations.","downloads_month":227,"downloads_total":56932,"page":"http://bioconductor.org/packages/release/bioc/html/RedeR.html"},{"authors":"Lihua Julie Zhu and Thomas Fazzio","name":"REDseq","license":"GPL (>=2)","tags":["Sequencing","SequenceMatching","Preprocessing"],"description":"The package includes functions to build restriction enzyme\ncut site (RECS) map, distribute mapped sequences on the map\nwith five different approaches, find enriched/depleted RECSs\nfor a sample, and identify differentially enriched/depleted\nRECSs between samples.","downloads_month":210,"downloads_total":41124,"page":"http://bioconductor.org/packages/release/bioc/html/REDseq.html"},{"authors":"Paul Shannon","name":"RefNet","license":"Artistic-2.0","tags":["GraphAndNetwork"],"description":"Molecular interactions with metadata, some archived, some\ndynamically obtained","downloads_month":142,"downloads_total":19244,"page":"http://bioconductor.org/packages/release/bioc/html/RefNet.html"},{"authors":"Kai-Ming Chang <kaiming@gmail.com>, Chris Harbron\n<Chris.Harbron@astrazeneca.com> and Marie C South\n<Marie.C.South@astrazeneca.com>","name":"RefPlus","license":"GPL (>= 2)","tags":["Microarray","OneChannel","Preprocessing"],"description":"The package contains functions for pre-processing\nAffymetrix data using the RMA+ and the RMA++ methods.","downloads_month":158,"downloads_total":50130,"page":"http://bioconductor.org/packages/release/bioc/html/RefPlus.html"},{"authors":"Anna Diez-Villanueva <adiez@iconcologia.net>, Roberto\nMalinverni <rmalinverni@carrerasresearch.org> and Bernat Gel\n<bgel@igtp.cat>","name":"regioneR","license":"Artistic-2.0","tags":["Genetics","ChIPSeq","DNASeq","MethylSeq","CopyNumberVariation"],"description":"regioneR offers a statistical framework based on\ncustomizable permutation tests to assess the association\nbetween genomic region sets and other genomic features.","downloads_month":1544,"downloads_total":88222,"page":"http://bioconductor.org/packages/release/bioc/html/regioneR.html"},{"authors":"Leonardo Collado-Torres [aut, cre], Andrew E. Jaffe [aut], Jeffrey T. Leek [aut and ths]","name":"regionReport","license":"Artistic-2.0","tags":["DifferentialExpression","Sequencing","RNASeq","Software","Visualization","Transcription","Coverage","ReportWriting","DifferentialMethylation","DifferentialPeakCalling","ImmunoOncology"],"description":"Generate HTML or PDF reports to explore a set of regions\nsuch as the results from annotation-agnostic expression\nanalysis of RNA-seq data at base-pair resolution performed by\nderfinder. You can also create reports for DESeq2 or edgeR\nresults.","downloads_month":233,"downloads_total":23268,"page":"http://bioconductor.org/packages/release/bioc/html/regionReport.html"},{"authors":"Lukas M. Weber [aut and cre]","name":"regsplice","license":"MIT + file LICENSE","tags":["ImmunoOncology","AlternativeSplicing","DifferentialExpression","DifferentialSplicing","Sequencing","RNASeq","Microarray","ExonArray","ExperimentalDesign","Software"],"description":"Statistical methods for detection of differential splicing\n(differential exon usage) in RNA-seq and exon microarray data,\nusing L1-regularization (lasso) to improve power.","downloads_month":189,"downloads_total":7050,"page":"http://bioconductor.org/packages/release/bioc/html/regsplice.html"},{"authors":"Yinan Zheng [aut, cre], Lei Liu [aut], Wei Zhang [aut], Warren\nKibbe [aut], Lifang Hou [aut and cph]","name":"REMP","license":"GPL-3","tags":["ImmunoOncology","DNAMethylation","Microarray","MethylationArray","GenePrediction","Sequencing","Alignment","Epigenetics","Normalization","Preprocessing","MultiChannel","TwoChannel","DifferentialMethylation","QualityControl","DataImport"],"description":"Machine learning-based tools to predict DNA methylation of\nlocus-specific repetitive elements (RE) by learning surrounding\ngenetic and epigenetic information. These tools provide\ngenomewide and single-base resolution of DNA methylation\nprediction on RE that are difficult to measure using\narray-based or sequencing-based platforms, which enables\nepigenome-wide association study (EWAS) and differentially\nmethylated region (DMR) analysis on RE.","downloads_month":194,"downloads_total":6300,"page":"http://bioconductor.org/packages/release/bioc/html/REMP.html"},{"authors":"Mark Robinson <mark.robinson@imls.uzh.ch>, Dario Strbenac\n<dario.strbenac@sydney.edu.au>, Aaron Statham\n<a.statham@garvan.org.au> and Andrea Riebler\n<andrea.riebler@math.ntnu.no>","name":"Repitools","license":"LGPL (>= 2)","tags":["DNAMethylation","GeneExpression","MethylSeq"],"description":"Tools for the analysis of enrichment-based epigenomic\ndata.  Features include summarization and visualization of\nepigenomic data across promoters according to gene expression\ncontext, finding regions of differential methylation/binding,\nBayMeth for quantifying methylation etc.","downloads_month":428,"downloads_total":80556,"page":"http://bioconductor.org/packages/release/bioc/html/Repitools.html"},{"authors":"Jason A. Hackney, Melanie Huntley, Jessica L. Larson, Christina\nChaivorapol, Gabriel Becker, and Josh Kaminker","name":"ReportingTools","license":"Artistic-2.0","tags":["ImmunoOncology","Software","Visualization","Microarray","RNASeq","GO","DataRepresentation","GeneSetEnrichment"],"description":"The ReportingTools software package enables users to\neasily display reports of analysis results generated from\nsources such as microarray and sequencing data.  The package\nallows users to create HTML pages that may be viewed on a web\nbrowser such as Safari, or in other formats readable by\nprograms such as Excel.  Users can generate tables with\nsortable and filterable columns, make and display plots, and\nlink table entries to other data sources such as NCBI or larger\nplots within the HTML page.  Using the package, users can also\nproduce a table of contents page to link various reports\ntogether for a particular project that can be viewed in a web\nbrowser.  For more examples, please visit our site: http://\nresearch-pub.gene.com/ReportingTools.","downloads_month":2114,"downloads_total":158528,"page":"http://bioconductor.org/packages/release/bioc/html/ReportingTools.html"},{"authors":"Christopher Cabanski, Keary Cavin and Chris Bizon","name":"ReQON","license":"GPL-2","tags":["Sequencing","HighThroughputSequencing","Preprocessing","QualityControl"],"description":"Algorithm for recalibrating the base quality scores for\naligned sequencing data in BAM format.","downloads_month":122,"downloads_total":31272,"page":"http://bioconductor.org/packages/release/bioc/html/ReQON.html"},{"authors":"Vincent Carey [aut], Shweta Gopaulakrishnan [cre and aut]","name":"restfulSE","license":"Artistic-2.0","tags":["ImmunoOncology","Infrastructure","SingleCell","Transcriptomics","Sequencing","Coverage"],"description":"This package provides functions and classes to interface\nwith remote data stores by operating on\nSummarizedExperiment-like objects.","downloads_month":213,"downloads_total":5548,"page":"http://bioconductor.org/packages/release/bioc/html/restfulSE.html"},{"authors":"Carles Hernandez-Ferrer [aut, cre] and Juan R. Gonzalez [aut]","name":"rexposome","license":"MIT + file LICENSE","tags":["Software","BiologicalQuestion","Infrastructure","DataImport","DataRepresentation","BiomedicalInformatics","ExperimentalDesign","MultipleComparison","Classification","Clustering"],"description":"Package that allows to explore the exposome and to perform\nassociation analyses between exposures and health outcomes.","downloads_month":153,"downloads_total":3322,"page":"http://bioconductor.org/packages/release/bioc/html/rexposome.html"},{"authors":"Fabienne Jabot-Hanin, Hugo Varet and Jean-Philippe Jais","name":"rfPred","license":"GPL (>=2 )","tags":["Software","Annotation","Classification"],"description":"Based on external numerous data files where rfPred scores\nare pre-calculated on all genomic positions of the human exome,\nthe package gives rfPred scores to missense variants identified\nby the chromosome, the position (hg19 version), the referent\nand alternative nucleotids and the uniprot identifier of the\nprotein. Note that for using the package, the user has to be\nconnected on the Internet or to download the TabixFile and\nindex (approximately 3.3 Go).","downloads_month":147,"downloads_total":21722,"page":"http://bioconductor.org/packages/release/bioc/html/rfPred.html"},{"authors":"Arnaud Droit, Raphael Gottardo, Gordon Robertson and Leiping Li","name":"rGADEM","license":"Artistic-2.0","tags":["Microarray","ChIPchip","Sequencing","ChIPSeq","MotifDiscovery"],"description":"rGADEM is an efficient de novo motif discovery tool for\nlarge-scale genomic sequence data. It is an open-source R\npackage, which is based on the GADEM software.","downloads_month":787,"downloads_total":103618,"page":"http://bioconductor.org/packages/release/bioc/html/rGADEM.html"},{"authors":"Dan Tenenbaum","name":"RGalaxy","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Given an R function and its manual page, make the\ndocumented function available in Galaxy.","downloads_month":163,"downloads_total":30726,"page":"http://bioconductor.org/packages/release/bioc/html/RGalaxy.html"},{"authors":"Hector Climente-Gonzalez [aut, cre], Dominik Gerhard Grimm\n[aut] and Chloe-Agathe Azencott [aut]","name":"Rgin","license":"MIT + file LICENSE","tags":["Software","GenomeWideAssociation","SNP","GeneticVariability","Genetics","FeatureExtraction","GraphAndNetwork","Network"],"description":"C++ implementation of SConES.","downloads_month":129,"downloads_total":1950,"page":"http://bioconductor.org/packages/release/bioc/html/Rgin.html"},{"authors":"Simone Pallotta and Marco Masseroli","name":"RGMQL","license":"Artistic-2.0","tags":["Software","Infrastructure","DataImport","Network","SingleCell"],"description":"This package brings the GenoMetric Query Language (GMQL)\nfunctionalities into the R environment. GMQL is a high-level,\ndeclarative language to manage heterogeneous genomic datasets\nfor biomedical purposes, using simple queries to process\ngenomic regions and their metadata and properties. GMQL adopts\nalgorithms efficiently designed for big data using\ncloud-computing technologies (like Apache Hadoop and Spark)\nallowing GMQL to run on modern infrastructures, in order to\nachieve scalability and high performance. It allows to create,\nmanipulate and extract genomic data from different data sources\nboth locally and remotely. Our RGMQL functions allow complex\nqueries and processing leveraging on the R idiomatic paradigm.\nThe RGMQL package also provides a rich set of ancillary classes\nthat allow sophisticated input/output management and sorting,\nsuch as: ASC, DESC, BAG, MIN, MAX, SUM, AVG, MEDIAN, STD, Q1,\nQ2, Q3 (and many others). Note that many RGMQL functions are\nnot directly executed in R environment, but are deferred until\nreal execution is issued.","downloads_month":110,"downloads_total":2186,"page":"http://bioconductor.org/packages/release/bioc/html/RGMQL.html"},{"authors":"Stephane Cano [aut, cre], Sylvain Gubian [aut] and Florian Martin\n[aut]","name":"RGraph2js","license":"GPL-2","tags":["Visualization","Network","GraphAndNetwork","ThirdPartyClient"],"description":"Generator of web pages which display interactive\nnetwork/graph visualizations with D3js, jQuery and Raphael.","downloads_month":144,"downloads_total":8074,"page":"http://bioconductor.org/packages/release/bioc/html/RGraph2js.html"},{"authors":"Kasper Daniel Hansen [cre, aut], Jeff Gentry [aut], Li Long\n[aut], Robert Gentleman [aut], Seth Falcon [aut], Florian Hahne\n[aut] and Deepayan Sarkar [aut]","name":"Rgraphviz","license":"EPL","tags":["GraphAndNetwork","Visualization"],"description":"Interfaces R with the AT and T graphviz library for\nplotting R graph objects from the graph package.","downloads_month":10394,"downloads_total":1367198,"page":"http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html"},{"authors":"Zuguang Gu","name":"rGREAT","license":"MIT + file LICENSE","tags":["GeneSetEnrichment","GO","Pathways","Software","Sequencing","WholeGenome","GenomeAnnotation","Coverage"],"description":"This package makes GREAT (Genomic Regions Enrichment of\nAnnotations Tool) analysis automatic by constructing a HTTP\nPOST request according to user's input and automatically\nretrieving results from GREAT web server.","downloads_month":246,"downloads_total":24208,"page":"http://bioconductor.org/packages/release/bioc/html/rGREAT.html"},{"authors":"Chengcheng Ma","name":"RGSEA","license":"GPL(>=3)","tags":["GeneSetEnrichment","StatisticalMethod","Classification"],"description":"Combining bootstrap aggregating and Gene set enrichment\nanalysis (GSEA), RGSEA is a classfication algorithm with high\nrobustness and no over-fitting problem. It performs well\nespecially for the data generated from different exprements.","downloads_month":199,"downloads_total":18178,"page":"http://bioconductor.org/packages/release/bioc/html/RGSEA.html"},{"authors":"Karl Stamm","name":"rgsepd","license":"GPL-3","tags":["ImmunoOncology","Software","DifferentialExpression","GeneSetEnrichment","RNASeq"],"description":"R/GSEPD is a bioinformatics package for R to help\ndisambiguate transcriptome samples (a matrix of RNA-Seq counts\nat transcript IDs) by automating differential expression (with\nDESeq2), then gene set enrichment (with GOSeq), and finally a\nN-dimensional projection to quantify in which ways each sample\nis like either treatment group.","downloads_month":209,"downloads_total":12812,"page":"http://bioconductor.org/packages/release/bioc/html/rgsepd.html"},{"authors":"Bernd Fischer [aut], Gregoire Pau [aut], Mike Smith [aut, cre], Martin Morgan [ctb] and Daniel van Twisk [ctb]","name":"rhdf5","license":"Artistic-2.0","tags":["Infrastructure","DataImport"],"description":"This package provides an interface between HDF5 and R.\nHDF5's main features are the ability to store and access very\nlarge and/or complex datasets and a wide variety of metadata on\nmass storage (disk) through a completely portable file format.\nThe rhdf5 package is thus suited for the exchange of large\nand/or complex datasets between R and other software package,\nand for letting R applications work on datasets that are larger\nthan the available RAM.","downloads_month":16651,"downloads_total":1008698,"page":"http://bioconductor.org/packages/release/bioc/html/rhdf5.html"},{"authors":"Samuela Pollack [cre, aut], Shweta Gopaulakrishnan [aut] and Vincent Carey [aut]","name":"rhdf5client","license":"Artistic-2.0","tags":["DataImport","Software"],"description":"Provides functionality for reading data from h5serv server\nfrom within R.","downloads_month":224,"downloads_total":5064,"page":"http://bioconductor.org/packages/release/bioc/html/rhdf5client.html"},{"authors":"Mike Smith","name":"Rhdf5lib","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Provides C and C++ hdf5 libraries.","downloads_month":13367,"downloads_total":239790,"page":"http://bioconductor.org/packages/release/bioc/html/Rhdf5lib.html"},{"authors":"Charlotte Soneson","name":"Rhisat2","license":"GPL-3","tags":["Alignment","Sequencing","SplicedAlignment"],"description":"An R interface to the HISAT2 spliced short-read aligner by\nKim et al. (2015). The package contains wrapper functions to\ncreate a genome index and to perform the read alignment to the\ngenerated index.","downloads_month":0,"downloads_total":138,"page":"http://bioconductor.org/packages/release/bioc/html/Rhisat2.html"},{"authors":"Nathaniel Hayden [led, aut], Martin Morgan [aut] and Bioconductor\nPackage Maintainer [cre]","name":"Rhtslib","license":"LGPL (>= 2)","tags":["DataImport","Sequencing"],"description":"This package provides version 1.7 of the 'HTSlib' C\nlibrary for high-throughput sequence analysis. The package is\nprimarily useful to developers of other R packages who wish to\nmake use of HTSlib. Motivation and instructions for use of this\npackage are in the vignette, vignette(package=\"Rhtslib\",\n\"Rhtslib\").","downloads_month":1804,"downloads_total":93830,"page":"http://bioconductor.org/packages/release/bioc/html/Rhtslib.html"},{"authors":"Martino Barenco","name":"rHVDM","license":"GPL-2","tags":["Microarray","GraphAndNetwork","Transcription","Classification","NetworkInference"],"description":"A R implementation of HVDM (Genome Biol 2006, V7(3) R25)","downloads_month":142,"downloads_total":42934,"page":"http://bioconductor.org/packages/release/bioc/html/rHVDM.html"},{"authors":"Alexandra Popa","name":"RiboProfiling","license":"GPL-3","tags":["RiboSeq","Sequencing","Coverage","Alignment","QualityControl","Software","PrincipalComponent"],"description":"Starting with a BAM file, this package provides the\nnecessary functions for quality assessment, read start position\nrecalibration, the counting of reads on CDS, 3'UTR, and 5'UTR,\nplotting of count data: pairs, log fold-change, codon frequency\nand coverage assessment, principal component analysis on codon\ncoverage.","downloads_month":191,"downloads_total":14718,"page":"http://bioconductor.org/packages/release/bioc/html/RiboProfiling.html"},{"authors":"Thomas J. Hardcastle","name":"riboSeqR","license":"GPL-3","tags":["Sequencing","Genetics","Visualization","RiboSeq"],"description":"Plotting functions, frameshift detection and parsing of\nsequencing data from ribosome profiling experiments.","downloads_month":150,"downloads_total":18620,"page":"http://bioconductor.org/packages/release/bioc/html/riboSeqR.html"},{"authors":"Ravi Shankar <rshankar@stanford.edu>","name":"RImmPort","license":"GPL-3","tags":["BiomedicalInformatics","DataImport","DataRepresentation"],"description":"The RImmPort package simplifies access to ImmPort data for\nanalysis in the R environment. It provides a standards-based\ninterface to the ImmPort study data that is in a proprietary\nformat.","downloads_month":128,"downloads_total":8290,"page":"http://bioconductor.org/packages/release/bioc/html/RImmPort.html"},{"authors":"Joern Toedling, Oleg Sklyar, Tammo Krueger, Matt Ritchie and Wolfgang Huber","name":"Ringo","license":"Artistic-2.0","tags":["Microarray","TwoChannel","DataImport","QualityControl","Preprocessing"],"description":"The package Ringo facilitates the primary analysis of\nChIP-chip data. The main functionalities of the package are\ndata read-in, quality assessment, data visualisation and\nidentification of genomic regions showing enrichment in\nChIP-chip. The package has functions to deal with two-color\noligonucleotide microarrays from NimbleGen used in ChIP-chip\nprojects, but also contains more general functions for\nChIP-chip data analysis, given that the data is supplied as\nRGList (raw) or ExpressionSet (pre- processed). The package\nemploys functions from various other packages of the\nBioconductor project and provides additional ChIP-chip-specific\nand NimbleGen-specific functionalities.","downloads_month":470,"downloads_total":109004,"page":"http://bioconductor.org/packages/release/bioc/html/Ringo.html"},{"authors":"Yue Li","name":"RIPSeeker","license":"GPL-2","tags":["Sequencing","RIPSeq"],"description":"Infer and discriminate RIP peaks from RIP-seq alignments\nusing two-state HMM with negative binomial emission\nprobability. While RIPSeeker is specifically tailored for\nRIP-seq data analysis, it also provides a suite of\nbioinformatics tools integrated within this self-contained\nsoftware package comprehensively addressing issues ranging from\npost-alignments processing to visualization and annotation.","downloads_month":195,"downloads_total":39828,"page":"http://bioconductor.org/packages/release/bioc/html/RIPSeeker.html"},{"authors":"Alejandra Gonzalez-Beltran, Audrey Kauffmann, Steffen Neumann, Gabriella Rustici and ISA Team","name":"Risa","license":"LGPL","tags":["Annotation","DataImport","MassSpectrometry"],"description":"The Investigation / Study / Assay (ISA) tab-delimited\nformat is a general purpose framework with which to collect and\ncommunicate complex metadata (i.e. sample characteristics,\ntechnologies used, type of measurements made) from experiments\nemploying a combination of technologies, spanning from\ntraditional approaches to high-throughput techniques. Risa\nallows to access metadata/data in ISA-Tab format and build\nBioconductor data structures. Currently, data generated from\nmicroarray, flow cytometry and metabolomics-based (i.e. mass\nspectrometry) assays are supported. The package is extendable\nand efforts are undergoing to support metadata associated to\nproteomics assays.","downloads_month":160,"downloads_total":30258,"page":"http://bioconductor.org/packages/release/bioc/html/Risa.html"},{"authors":"Michael Zimmermann","name":"RITAN","license":"file LICENSE","tags":["QualityControl","Network","NetworkEnrichment","NetworkInference","GeneSetEnrichment","FunctionalGenomics"],"description":"Tools for comprehensive gene set enrichment and extraction\nof multi-resource high confidence subnetworks. RITAN\nfacilitates bioinformatic tasks for enabling network biology\nresearch.","downloads_month":128,"downloads_total":4722,"page":"http://bioconductor.org/packages/release/bioc/html/RITAN.html"},{"authors":"Yungil Kim [aut, cre] and Alexis Battle [aut]","name":"RIVER","license":"GPL (>= 2)","tags":["GeneExpression","GeneticVariability","SNP","Transcription","FunctionalPrediction","GeneRegulation","GenomicVariation","BiomedicalInformatics","FunctionalGenomics","Genetics","SystemsBiology","Transcriptomics","Bayesian","Clustering","TranscriptomeVariant","Regression"],"description":"An implementation of a probabilistic modeling framework\nthat jointly analyzes personal genome and transcriptome data to\nestimate the probability that a variant has regulatory impact\nin that individual. It is based on a generative model that\nassumes that genomic annotations, such as the location of a\nvariant with respect to regulatory elements, determine the\nprior probability that variant is a functional regulatory\nvariant, which is an unobserved variable. The functional\nregulatory variant status then influences whether nearby genes\nare likely to display outlier levels of gene expression in that\nperson. See the RIVER website for more information,\ndocumentation and examples.","downloads_month":145,"downloads_total":4996,"page":"http://bioconductor.org/packages/release/bioc/html/RIVER.html"},{"authors":"Pascal Belleau [aut], Rawane Samb [aut], Astrid Deschênes [cre, aut], Khader Khadraoui [aut], Lajmi Lakhal-Chaieb [aut] and Arnaud\nDroit [aut]","name":"RJMCMCNucleosomes","license":"Artistic-2.0","tags":["BiologicalQuestion","ChIPSeq","NucleosomePositioning","Software","StatisticalMethod","Bayesian","Sequencing","Coverage"],"description":"This package does nucleosome positioning using informative\nMultinomial-Dirichlet prior in a t-mixture with reversible jump\nestimation of nucleosome positions for genome-wide profiling.","downloads_month":138,"downloads_total":5390,"page":"http://bioconductor.org/packages/release/bioc/html/RJMCMCNucleosomes.html"},{"authors":"Nusrat Rabbee <nrabbee@post.harvard.edu> and Gary Wong\n<wongg62@berkeley.edu>","name":"RLMM","license":"LGPL (>= 2)","tags":["Microarray","OneChannel","SNP","GeneticVariability"],"description":"A classification algorithm, based on a multi-chip,\nmulti-SNP approach for Affymetrix SNP arrays. Using a large\ntraining sample where the genotype labels are known, this\naglorithm will obtain more accurate classification results on\nnew data. RLMM is based on a robust, linear model and uses the\nMahalanobis distance for classification. The chip-to-chip\nnon-biological variation is removed through normalization. This\nmodel-based algorithm captures the similarities across genotype\ngroups and probes, as well as thousands other SNPs for accurate\nclassification. NOTE: 100K-Xba only at for now.","downloads_month":123,"downloads_total":41730,"page":"http://bioconductor.org/packages/release/bioc/html/RLMM.html"},{"authors":"Camille Maumet <Rmagpie@gmail.com> and with contributions from C.\nAmbroise J. Zhu","name":"Rmagpie","license":"GPL (>= 3)","tags":["Microarray","Classification"],"description":"Microarray Classification is designed for both biologists\nand statisticians. It offers the ability to train a classifier\non a labelled microarray dataset and to then use that\nclassifier to predict the class of new observations. A range of\nmodern classifiers are available, including support vector\nmachines (SVMs), nearest shrunken centroids (NSCs)... Advanced\nmethods are provided to estimate the predictive error rate and\nto report the subset of genes which appear essential in\ndiscriminating between classes.","downloads_month":137,"downloads_total":39858,"page":"http://bioconductor.org/packages/release/bioc/html/Rmagpie.html"},{"authors":"Michael Stravs, Emma Schymanski, Steffen Neumann, Erik Mueller and with contributions from Tobias Schulze","name":"RMassBank","license":"Artistic-2.0","tags":["ImmunoOncology","Bioinformatics","MassSpectrometry","Metabolomics","Software"],"description":"Workflow to process tandem MS files and build MassBank\nrecords. Functions include automated extraction of tandem MS\nspectra, formula assignment to tandem MS fragments,\nrecalibration of tandem MS spectra with assigned fragments,\nspectrum cleanup, automated retrieval of compound information\nfrom Internet databases, and export to MassBank records.","downloads_month":230,"downloads_total":34910,"page":"http://bioconductor.org/packages/release/bioc/html/RMassBank.html"},{"authors":"Charles Cheung and Arnaud Droit and Raphael Gottardo","name":"rMAT","license":"Artistic-2.0","tags":["Microarray","Preprocessing"],"description":"This package is an R version of the package MAT and\ncontains functions to parse and merge Affymetrix BPMAP and CEL\ntiling array files (using C++ based Fusion SDK and Bioconductor\npackage affxparser), normalize tiling arrays using sequence\nspecific models, detect enriched regions from ChIP-chip\nexperiments. Note: users should have GSL and GenomeGraphs\ninstalled. Windows users: 'consult the README file available in\nthe inst directory of the source distribution for necessary\nconfiguration instructions'. Snow Leopard users can take\nadvantage of increase speed with Grand Central Dispatch!","downloads_month":166,"downloads_total":38262,"page":"http://bioconductor.org/packages/release/bioc/html/rMAT.html"},{"authors":"J. Aravind [aut, cre]\n(<https://orcid.org/0000-0002-4791-442X>), G. K. Krishna [aut], Bob Rudis [ctb] (melting5jars), Nicolas Le Novère [ctb]\n(MELTING 5 Java Library), Marine Dumousseau [ctb] (MELTING 5\nJava Library) and William John Gowers [ctb] (MELTING 5 Java\nLibrary)","name":"rmelting","license":"GPL-2 | GPL-3","tags":["BiomedicalInformatics","Cheminformatics",""],"description":"R interface to the MELTING 5 program\n(<https://www.ebi.ac.uk/biomodels/tools/melting/>) to compute\nmelting temperatures of nucleic acid duplexes along with other\nthermodynamic parameters.","downloads_month":0,"downloads_total":26,"page":"http://bioconductor.org/packages/release/bioc/html/rmelting.html"},{"authors":"Francesco Favero","name":"RmiR","license":"Artistic-2.0","tags":["Software","GeneExpression","Microarray","TimeCourse","Visualization"],"description":"Useful functions to merge microRNA and respective targets\nusing differents databases","downloads_month":133,"downloads_total":47266,"page":"http://bioconductor.org/packages/release/bioc/html/RmiR.html"},{"authors":"Zytnicki Matthias [aut and cre]","name":"Rmmquant","license":"GPL-3","tags":["GeneExpression","Transcription"],"description":"RNA-Seq is currently used routinely, and it provides\naccurate information on gene transcription. However, the method\ncannot accurately estimate duplicated genes expression. Several\nstrategies have been previously used, but all of them provide\nbiased results. With Rmmquant, if a read maps at different\npositions, the tool detects that the corresponding genes are\nduplicated; it merges the genes and creates a merged gene. The\ncounts of ambiguous reads is then based on the input genes and\nthe merged genes. Rmmquant is a drop-in replacement of the\nwidely used tools findOverlaps and featureCounts that handles\nmulti-mapping reads in an unabiased way.","downloads_month":102,"downloads_total":1108,"page":"http://bioconductor.org/packages/release/bioc/html/Rmmquant.html"},{"authors":"Reed Sorenson [aut, cre], Katrina Johnson [aut], Frederick\nAdler [aut] and Leslie Sieburth [aut]","name":"RNAdecay","license":"GPL-2","tags":["ImmunoOncology","Software","GeneExpression","GeneRegulation","DifferentialExpression","Transcription","Transcriptomics","TimeCourse","Regression","RNASeq","Normalization","WorkflowStep"],"description":"RNA degradation is monitored through measurement of RNA\nabundance after inhibiting RNA synthesis. This package has\nfunctions and example scripts to facilitate (1) data\nnormalization, (2) data modeling using constant decay rate or\ntime-dependent decay rate models, (3) the evaluation of\ntreatment or genotype effects, and (4) plotting of the data and\nmodels. Data Normalization: functions and scripts make easy the\nnormalization to the initial (T0) RNA abundance, as well as a\nmethod to correct for artificial inflation of Reads per Million\n(RPM) abundance in global assessments as the total size of the\nRNA pool decreases. Modeling: Normalized data is then modeled\nusing maximum likelihood to fit parameters. For making\ntreatment or genotype comparisons (up to four), the modeling\nstep models all possible treatment effects on each gene by\nrepeating the modeling with constraints on the model parameters\n(i.e., the decay rate of treatments A and B are modeled once\nwith them being equal and again allowing them to both vary\nindependently). Model Selection: The AICc value is calculated\nfor each model, and the model with the lowest AICc is chosen.\nModeling results of selected models are then compiled into a\nsingle data frame. Graphical Plotting: a function is provided\nto easily visualize the data and the selected model using\nggplot2 package functions.","downloads_month":109,"downloads_total":1744,"page":"http://bioconductor.org/packages/release/bioc/html/RNAdecay.html"},{"authors":"Bernd Fischer <b.fischer@dkfz.de>","name":"RNAinteract","license":"Artistic-2.0","tags":["ImmunoOncology","CellBasedAssays","QualityControl","Preprocessing","Visualization"],"description":"RNAinteract estimates genetic interactions from\nmulti-dimensional read-outs like features extracted from\nimages. The screen is assumed to be performed in multi-well\nplates or similar designs. Starting from a list of features\n(e.g. cell number, area, fluorescence intensity) per well,\ngenetic interactions are estimated. The packages provides\nfunctions for reporting interacting gene pairs, plotting\nheatmaps and double RNAi plots. An HTML report can be written\nfor quality control and analysis.","downloads_month":191,"downloads_total":32996,"page":"http://bioconductor.org/packages/release/bioc/html/RNAinteract.html"},{"authors":"Nora Rieber, Lars Kaderali, University of Heidelberg, Viroquant Research Group Modeling, Im Neuenheimer Feld 267, 69120 Heidelberg and Germany","name":"RNAither","license":"Artistic-2.0","tags":["CellBasedAssays","QualityControl","Preprocessing","Visualization","Annotation","GO","ImmunoOncology"],"description":"RNAither analyzes cell-based RNAi screens, and includes\nquality assessment, customizable normalization and statistical\ntests, leading to lists of significant genes and biological\nprocesses.","downloads_month":201,"downloads_total":46544,"page":"http://bioconductor.org/packages/release/bioc/html/RNAither.html"},{"authors":"Lukasz Jan Kielpinski [aut], Nikos Sidiropoulos [cre, aut] and Jeppe Vinther [aut]","name":"RNAprobR","license":"GPL (>=2)","tags":["Coverage","Normalization","Sequencing","GenomeAnnotation"],"description":"This package facilitates analysis of Next Generation\nSequencing data for which positional information with a single\nnucleotide resolution is a key. It allows for applying\ndifferent types of relevant normalizations, data visualization\nand export in a table or UCSC compatible bedgraph file.","downloads_month":143,"downloads_total":12650,"page":"http://bioconductor.org/packages/release/bioc/html/RNAprobR.html"},{"authors":"Mingxiang Teng and Rafael A. Irizarry","name":"rnaseqcomp","license":"GPL-3","tags":["RNASeq","Visualization","QualityControl"],"description":"Several quantitative and visualized benchmarks for RNA-seq\nquantification pipelines. Two-condition quantifications for\ngenes, transcripts, junctions or exons by each pipeline with\nnecessary meta information should be organized into numeric\nmatrices in order to proceed the evaluation.","downloads_month":142,"downloads_total":10822,"page":"http://bioconductor.org/packages/release/bioc/html/rnaseqcomp.html"},{"authors":"Anna Lesniewska <alesniewska@cs.put.poznan.pl>; Michal\nOkoniewski <michal@fgcz.ethz.ch>","name":"rnaSeqMap","license":"GPL-2","tags":["ImmunoOncology","Annotation","ReportWriting","Transcription","GeneExpression","DifferentialExpression","Sequencing","RNASeq","SAGE","Visualization"],"description":"The rnaSeqMap library provides classes and functions to\nanalyze the RNA-sequencing data using the coverage profiles in\nmultiple samples at a time","downloads_month":193,"downloads_total":34404,"page":"http://bioconductor.org/packages/release/bioc/html/rnaSeqMap.html"},{"authors":"Terry M Therneau [aut, cre] and Hart Stephen [ctb]","name":"RNASeqPower","license":"LGPL (>=2)","tags":["ImmunoOncology","RNASeq"],"description":"RNA-seq, sample size","downloads_month":268,"downloads_total":36336,"page":"http://bioconductor.org/packages/release/bioc/html/RNASeqPower.html"},{"authors":"Kuan-Hao Chao","name":"RNASeqR","license":"Artistic-2.0","tags":["Genetics","Infrastructure","DataImport","Sequencing","RNASeq","GeneExpression","GeneSetEnrichment","Alignment","QualityControl","DifferentialExpression","FunctionalPrediction","ExperimentalDesign","GO","KEGG","Visualization","Normalization","Pathways","Clustering","ImmunoOncology"],"description":"This R package is designed for case-control RNA-Seq\nanalysis (two-group). There are six steps: \"RNASeqRParam S4\nObject Creation\", \"Environment Setup\", \"Quality Assessment\",\n\"Reads Alignment & Quantification\", \"Gene-level Differential\nAnalyses\" and \"Functional Analyses\". Each step corresponds to a\nfunction in this package. After running functions in order, a\nbasic RNASeq analysis would be done easily.","downloads_month":87,"downloads_total":634,"page":"http://bioconductor.org/packages/release/bioc/html/RNASeqR.html"},{"authors":"Shilin Zhao, Chung-I Li, Yan Guo, Quanhu Sheng and Yu Shyr","name":"RnaSeqSampleSize","license":"GPL (>= 2)","tags":["ImmunoOncology","ExperimentalDesign","Sequencing","RNASeq","GeneExpression","DifferentialExpression"],"description":"RnaSeqSampleSize package provides a sample size\ncalculation method based on negative binomial model and the\nexact test for assessing differential expression analysis of\nRNA-seq data","downloads_month":246,"downloads_total":16712,"page":"http://bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html"},{"authors":"Yassen Assenov [aut], Pavlo Lutsik [aut], Michael Scherer\n[aut], Fabian Mueller [aut and cre]","name":"RnBeads","license":"GPL-3","tags":["DNAMethylation","MethylationArray","MethylSeq","Epigenetics","QualityControl","Preprocessing","BatchEffect","DifferentialMethylation","Sequencing","CpGIsland","TwoChannel","DataImport"],"description":"RnBeads facilitates comprehensive analysis of various\ntypes of DNA methylation data at the genome scale.","downloads_month":676,"downloads_total":47418,"page":"http://bioconductor.org/packages/release/bioc/html/RnBeads.html"},{"authors":"Dipen P. Sangurdekar <dipen.sangurdekar@gmail.com>","name":"Rnits","license":"GPL-3","tags":["GeneExpression","Microarray","TimeCourse","DifferentialExpression","Normalization"],"description":"R/Bioconductor package for normalization, curve\nregistration and inference in time course gene expression data","downloads_month":152,"downloads_total":15926,"page":"http://bioconductor.org/packages/release/bioc/html/Rnits.html"},{"authors":"Elena Grassi","name":"roar","license":"GPL-3","tags":["Sequencing","HighThroughputSequencing","RNAseq","Transcription"],"description":"Identify preferential usage of APA sites, comparing two\nbiological conditions, starting from known alternative sites\nand alignments obtained from standard RNA-seq experiments.","downloads_month":167,"downloads_total":20518,"page":"http://bioconductor.org/packages/release/bioc/html/roar.html"},{"authors":"Vince Carey <stvjc@channing.harvard.edu> and Henning Redestig for\nC++ language enhancements","name":"ROC","license":"Artistic-2.0","tags":["DifferentialExpression"],"description":"utilities for ROC, with uarray focus","downloads_month":1383,"downloads_total":290920,"page":"http://bioconductor.org/packages/release/bioc/html/ROC.html"},{"authors":"Yue Li","name":"Roleswitch","license":"GPL-2","tags":["miRNA"],"description":"Infer Probabilities of MiRNA-mRNA Interaction Signature\n(ProMISe) using paired expression data from a single sample.\nRoleswitch operates in two phases by inferring the probability\nof mRNA (miRNA) being the targets (\"targets\") of miRNA (mRNA),\ntaking into account the expression of all of the mRNAs (miRNAs)\ndue to their potential competition for the same miRNA (mRNA).\nDue to dynamic miRNA repression in the cell, Roleswitch assumes\nthat the total transcribed mRNA levels are higher than the\nobserved (equilibrium) mRNA levels and iteratively updates the\ntotal transcription of each mRNA targets based on the above\ninference. NB: in the paper, we used ProMISe as both the model\nname and inferred score name.","downloads_month":160,"downloads_total":23852,"page":"http://bioconductor.org/packages/release/bioc/html/Roleswitch.html"},{"authors":"Laurent Gatto <lg390@cam.ac.uk> and with contributions from Tiago\nChedraoui Silva.","name":"rols","license":"GPL-2","tags":["ImmunoOncology","Software","Annotation","MassSpectrometry","GO"],"description":"The rols package is an interface to the Ontology Lookup\nService (OLS) to access and query hundred of ontolgies directly\nfrom R.","downloads_month":527,"downloads_total":47666,"page":"http://bioconductor.org/packages/release/bioc/html/rols.html"},{"authors":"Calin Voichita <calin@wayne.edu> and Sahar Ansari\n<saharansari@wayne.edu> and Sorin Draghici <sorin@wayne.edu>","name":"ROntoTools","license":"CC BY-NC-ND 4.0 + file LICENSE","tags":["NetworkAnalysis","Microarray","GraphsAndNetworks"],"description":"Suite of tools for functional analysis.","downloads_month":186,"downloads_total":30168,"page":"http://bioconductor.org/packages/release/bioc/html/ROntoTools.html"},{"authors":"Etienne A. Thevenot <etienne.thevenot@cea.fr>","name":"ropls","license":"CeCILL","tags":["Regression","Classification","PrincipalComponent","Transcriptomics","Proteomics","Metabolomics","Lipidomics","MassSpectrometry","ImmunoOncology"],"description":"Latent variable modeling with Principal Component\nAnalysis(PCA) and Partial Least Squares (PLS) are powerful\nmethods for visualization, regression, classification, and\nfeature selection of omics data where the number of variables\nexceeds the number of samples and with multicollinearity among\nvariables. Orthogonal Partial Least Squares (OPLS) enables to\nseparately model the variation correlated (predictive) to the\nfactor of interest and the uncorrelated (orthogonal) variation.\nWhile performing similarly to PLS, OPLS facilitates\ninterpretation. Successful applications of these chemometrics\ntechniques include spectroscopic data such as Raman\nspectroscopy, nuclear magnetic resonance (NMR), mass\nspectrometry (MS) in metabolomics and proteomics, but also\ntranscriptomics data. In addition to scores, loadings and\nweights plots, the package provides metrics and graphics to\ndetermine the optimal number of components (e.g. with the R2\nand Q2 coefficients), check the validity of the model by\npermutation testing, detect outliers, and perform feature\nselection (e.g. with Variable Importance in Projection or\nregression coefficients). The package can be accessed via a\nuser interface on the Workflow4Metabolomics.org online resource\nfor computational metabolomics (built upon the Galaxy\nenvironment).","downloads_month":836,"downloads_total":39016,"page":"http://bioconductor.org/packages/release/bioc/html/ropls.html"},{"authors":"Fatemeh Seyednasrollah, Tomi Suomi and Laura L. Elo","name":"ROTS","license":"GPL (>= 2)","tags":["Software","GeneExpression","DifferentialExpression","Microarray","RNASeq","Proteomics","ImmunoOncology"],"description":"Calculates the Reproducibility-Optimized Test Statistic\n(ROTS) for differential testing in omics data.","downloads_month":326,"downloads_total":13974,"page":"http://bioconductor.org/packages/release/bioc/html/ROTS.html"},{"authors":"Leo Lahti [aut and cre]","name":"RPA","license":"BSD_2_clause + file LICENSE","tags":["GeneExpression","Microarray","Preprocessing","QualityControl"],"description":"Probabilistic analysis of probe reliability and\ndifferential gene expression on short oligonucleotide arrays.","downloads_month":212,"downloads_total":45608,"page":"http://bioconductor.org/packages/release/bioc/html/RPA.html"},{"authors":"Mike Jiang","name":"RProtoBufLib","license":"BSD_3_clause","tags":["Infrastructure"],"description":"This package provides the headers and static library of\nProtocol buffers 2.6.0 for other R packages to compile and link\nagainst.","downloads_month":818,"downloads_total":26316,"page":"http://bioconductor.org/packages/release/bioc/html/RProtoBufLib.html"},{"authors":"Jitao David Zhang, Stefan Wiemann, Marc Carlson and with\ncontributions from Tony Chiang","name":"RpsiXML","license":"LGPL-3","tags":["Infrastructure","Proteomics"],"description":"Queries, data structure and interface to visualization of\ninteraction datasets. This package inplements the PSI-MI 2.5\nstandard and supports up to now 8 databases. Further databases\nsupporting PSI-MI 2.5 standard will be added continuously.","downloads_month":209,"downloads_total":55186,"page":"http://bioconductor.org/packages/release/bioc/html/RpsiXML.html"},{"authors":"Laurent Gatto","name":"rpx","license":"GPL-2","tags":["ImmunoOncology","Proteomics","MassSpectrometry","DataImport","ThirdPartyClient"],"description":"The rpx package implements an interface to proteomics data\nsubmitted to the ProteomeXchange consortium.","downloads_month":562,"downloads_total":43508,"page":"http://bioconductor.org/packages/release/bioc/html/rpx.html"},{"authors":"Welliton Souza and Benilton Carvalho <beniltoncarvalho@gmail.com>","name":"Rqc","license":"GPL (>= 2)","tags":["Sequencing","QualityControl","DataImport"],"description":"Rqc is an optimised tool designed for quality control and\nassessment of high-throughput sequencing data. It performs\nparallel processing of entire files and produces a report which\ncontains a set of high-resolution graphics.","downloads_month":223,"downloads_total":22326,"page":"http://bioconductor.org/packages/release/bioc/html/Rqc.html"},{"authors":"I. Y. Zhbannikov, K. G. Arbeev and A. I. Yashin.","name":"rqt","license":"GPL","tags":["GenomeWideAssociation","Regression","Survival","PrincipalComponent","StatisticalMethod","Sequencing"],"description":"Despite the recent advances of modern GWAS methods, it\nstill remains an important problem of addressing calculation an\neffect size and corresponding p-value for the whole gene rather\nthan for single variant. The R- package rqt offers gene-level\nGWAS meta-analysis. For more information, see: \"Gene-set\nassociation tests for next-generation sequencing data\" by Lee\net al (2016), Bioinformatics, 32(17), i611-i619,\n<doi:10.1093/bioinformatics/btw429>.","downloads_month":139,"downloads_total":5020,"page":"http://bioconductor.org/packages/release/bioc/html/rqt.html"},{"authors":"Jitao David Zhang, with inputs from Laura Badi and Martin\nEbeling","name":"rqubic","license":"GPL-2","tags":["Microarray","Clustering"],"description":"This package implements the QUBIC algorithm introduced by\nLi et al. for the qualitative biclustering with gene expression\ndata.","downloads_month":216,"downloads_total":36946,"page":"http://bioconductor.org/packages/release/bioc/html/rqubic.html"},{"authors":"Michael Hahsler and Anurag Nagar","name":"rRDP","license":"GPL-2 | file LICENSE","tags":["Genetics","Sequencing","Infrastructure","Classification","Microbiome"],"description":"Seamlessly interfaces RDP classifier (version 2.9).","downloads_month":187,"downloads_total":15562,"page":"http://bioconductor.org/packages/release/bioc/html/rRDP.html"},{"authors":"Jonathan Rosenblatt and Jason Stein","name":"RRHO","license":"GPL-2","tags":["Genetics","SequenceMatching","Microarray","Transcription"],"description":"The package is aimed at inference on the amount of\nagreement in two sorted lists using the Rank-Rank\nHypergeometric Overlap test.","downloads_month":158,"downloads_total":24786,"page":"http://bioconductor.org/packages/release/bioc/html/RRHO.html"},{"authors":"Martin Morgan, Herv\\'e Pag\\`es, Valerie Obenchain and Nathaniel\nHayden","name":"Rsamtools","license":"Artistic-2.0 | file LICENSE","tags":["DataImport","Sequencing","Coverage","Alignment","QualityControl"],"description":"This package provides an interface to the 'samtools',\n'bcftools', and 'tabix' utilities for manipulating SAM\n(Sequence Alignment / Map), FASTA, binary variant call (BCF)\nand compressed indexed tab-delimited (tabix) files.","downloads_month":23936,"downloads_total":2331062,"page":"http://bioconductor.org/packages/release/bioc/html/Rsamtools.html"},{"authors":"Michael Lawrence <michafla@gene.com>","name":"rsbml","license":"Artistic-2.0","tags":["GraphAndNetwork","Pathways","Network"],"description":"Links R to libsbml for SBML parsing, validating output,\nprovides an S4 SBML DOM, converts SBML to R graph objects.\nOptionally links to the SBML ODE Solver Library (SOSLib) for\nsimulating models.","downloads_month":254,"downloads_total":74726,"page":"http://bioconductor.org/packages/release/bioc/html/rsbml.html"},{"authors":"Matteo Ciciani [aut, cre], Thomas Cantore [aut], Enrica\nColasurdo [ctb] and Mario Lauria [ctb]","name":"rScudo","license":"GPL-3","tags":["GeneExpression","DifferentialExpression","BiomedicalInformatics","Classification","Clustering","GraphAndNetwork","Network","Proteomics","Transcriptomics","SystemsBiology","FeatureExtraction"],"description":"SCUDO (Signature-based Clustering for Diagnostic Purposes)\nis a rank-based method for the analysis of gene expression\nprofiles for diagnostic and classification purposes. It is\nbased on the identification of sample-specific gene signatures\ncomposed of the most up- and down-regulated genes for that\nsample. Starting from gene expression data, functions in this\npackage identify sample-specific gene signatures and use them\nto build a graph of samples. In this graph samples are joined\nby edges if they have a similar expression profile, according\nto a pre-computed similarity matrix. The similarity between the\nexpression profiles of two samples is computed using a method\nsimilar to GSEA. The graph of samples can then be used to\nperform community clustering or to perform supervised\nclassification of samples in a testing set.","downloads_month":41,"downloads_total":200,"page":"http://bioconductor.org/packages/release/bioc/html/rScudo.html"},{"authors":"August Guang [aut and cre]","name":"RSeqAn","license":"BSD_3_clause + file LICENSE","tags":["Infrastructure","Software"],"description":"Headers and some wrapper functions from the SeqAn C++\nlibrary for ease of usage in R.","downloads_month":118,"downloads_total":1896,"page":"http://bioconductor.org/packages/release/bioc/html/RSeqAn.html"},{"authors":"Matt Settles <mattsettles@gmail.com>, Sam Hunter, Brice Sarver, Ilia Zhbannikov and Kyu-Chul Cho","name":"rSFFreader","license":"Artistic-2.0","tags":["DataImport","Sequencing"],"description":"rSFFreader reads sequence, qualities and clip point values\nfrom sff files generated by Roche 454 and Life Sciences Ion\nTorrent sequencers into similar classes as are present for\nfastq files.","downloads_month":128,"downloads_total":15678,"page":"http://bioconductor.org/packages/release/bioc/html/rSFFreader.html"},{"authors":"Wei Shi, Yang Liao and Gordon K Smyth with contributions from\nJenny Dai","name":"Rsubread","license":"GPL-3","tags":["Sequencing","Alignment","SequenceMatching","RNASeq","ChIPSeq","GeneExpression","GeneRegulation","Genetics","ImmunoOncology","SNP","GeneticVariability","Preprocessing","QualityControl","GenomeAnnotation","GeneFusionDetection","IndelDetection","VariantAnnotation","VariantDetection","MultipleSequenceAlignment"],"description":"Alignment, quantification and analysis of second and third\ngeneration sequencing data. Includes functionality for read\nmapping, read counting, SNP calling, structural variant\ndetection and gene fusion discovery. Can be applied to all\nmajor sequencing techologies and to both short and long\nsequence reads.","downloads_month":2098,"downloads_total":250052,"page":"http://bioconductor.org/packages/release/bioc/html/Rsubread.html"},{"authors":"Christoph Bartenhagen","name":"RSVSim","license":"LGPL-3","tags":["Sequencing"],"description":"RSVSim is a package for the simulation of deletions,\ninsertions, inversion, tandem-duplications and translocations\nof various sizes in any genome available as FASTA-file or\nBSgenome data package. SV breakpoints can be placed uniformly\naccross the whole genome, with a bias towards repeat regions\nand regions of high homology (for hg19) or at user-supplied\ncoordinates.","downloads_month":165,"downloads_total":28510,"page":"http://bioconductor.org/packages/release/bioc/html/RSVSim.html"},{"authors":"Frederic Fournier <frederic.fournier@crchuq.ulaval.ca>, Charles\nJoly Beauparlant <charles.joly-beauparlant@crchul.ulaval.ca>, Rene Paradis <rene.paradis@genome.ulaval.ca> and Arnaud Droit\n<arnaud.droit@crchuq.ulaval.ca>","name":"rTANDEM","license":"Artistic-1.0 | file LICENSE","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"This package interfaces the tandem protein identification\nalgorithm in R. Identification can be launched in the X!Tandem\nstyle, by using as sole parameter the path to a parameter file.\nBut rTANDEM aslo provides extended syntax and functions to\nstreamline launching analyses, as well as function to convert\nresults, parameters and taxonomy to/from R. A related package,\nshinyTANDEM, provides visualization interface for result\nobjects.","downloads_month":421,"downloads_total":43448,"page":"http://bioconductor.org/packages/release/bioc/html/rTANDEM.html"},{"authors":"Jitao David Zhang","name":"RTCA","license":"LGPL-3","tags":["ImmunoOncology","CellBasedAssays","Infrastructure","Visualization","TimeCourse"],"description":"Import, analyze and visualize data from Roche(R)\nxCELLigence RTCA systems. The package imports real-time cell\nelectrical impedance data into R. As an alternative to\ncommercial software shipped along the system, the Bioconductor\npackage RTCA provides several unique transformation\n(normalization) strategies and various visualization tools.","downloads_month":181,"downloads_total":39566,"page":"http://bioconductor.org/packages/release/bioc/html/RTCA.html"},{"authors":"Marcin Kosinski <m.p.kosinski@gmail.com> and Przemyslaw Biecek\n<przemyslaw.biecek@gmail.com>","name":"RTCGA","license":"GPL-2","tags":["ImmunoOncology","Software","DataImport","DataRepresentation","Preprocessing","RNASeq"],"description":"The Cancer Genome Atlas (TCGA) Data Portal provides a\nplatform for researchers to search, download, and analyze data\nsets generated by TCGA. It contains clinical information,\ngenomic characterization data, and high level sequence analysis\nof the tumor genomes. The key is to understand genomics to\nimprove cancer care. RTCGA package offers download and\nintegration of the variety and volume of TCGA data using\npatient barcode key, what enables easier data possession. This\nmay have an benefcial infuence on impact on development of\nscience and improvement of patients' treatment. Furthermore,\nRTCGA package transforms TCGA data to tidy form which is\nconvenient to use.","downloads_month":1077,"downloads_total":43384,"page":"http://bioconductor.org/packages/release/bioc/html/RTCGA.html"},{"authors":"Mehmet Samur [aut], Marcel Ramos [aut, cre] and Ludwig Geistlinger\n[ctb]","name":"RTCGAToolbox","license":"file LICENSE","tags":["DifferentialExpression","GeneExpression","Sequencing"],"description":"Managing data from large scale projects such as The Cancer\nGenome Atlas (TCGA) for further analysis is an important and\ntime consuming step for research projects. Several efforts,\nsuch as Firehose project, make TCGA pre-processed data publicly\navailable via web services and data portals but it requires\nmanaging, downloading and preparing the data for following\nsteps. We developed an open source and extensible R based data\nclient for Firehose pre-processed data and demonstrated its use\nwith sample case studies. Results showed that RTCGAToolbox\ncould improve data management for researchers who are\ninterested with TCGA data. In addition, it can be integrated\nwith other analysis pipelines for following data analysis.","downloads_month":731,"downloads_total":43786,"page":"http://bioconductor.org/packages/release/bioc/html/RTCGAToolbox.html"},{"authors":"Mauro Castro, Xin Wang, Michael Fletcher and Florian Markowetz and\nKerstin Meyer","name":"RTN","license":"Artistic-2.0","tags":["NetworkInference","NetworkEnrichment","GeneRegulation","GeneExpression","GraphAndNetwork","GeneSetEnrichment","GeneticVariability","SNP"],"description":"This package provides classes and methods for\ntranscriptional network inference and analysis. Modulators of\ntranscription factor activity are assessed by conditional\nmutual information, and master regulators are mapped to\nphenotypes using different strategies, e.g., gene set\nenrichment, shadow and synergy analyses. Additionally, master\nregulators can be linked to genetic markers using eQTL/VSE\nanalysis, taking advantage of the haplotype block structure\nmapped to the human genome in order to explore risk-associated\nSNPs identified in GWAS studies.","downloads_month":213,"downloads_total":30470,"page":"http://bioconductor.org/packages/release/bioc/html/RTN.html"},{"authors":"Vinicius S. Chagas, Clarice S. Groeneveld, Gordon Robertson, Kerstin B. Meyer and Mauro A. A. Castro","name":"RTNduals","license":"Artistic-2.0","tags":["GeneRegulation","GeneExpression","NetworkEnrichment","NetworkInference","GraphAndNetwork"],"description":"RTNduals is a tool that searches for possible\nco-regulatory loops between regulon pairs generated by the RTN\npackage. It compares the shared targets in order to infer 'dual\nregulons', a new concept that tests whether regulators can\nco-operate or compete in influencing targets.","downloads_month":149,"downloads_total":5258,"page":"http://bioconductor.org/packages/release/bioc/html/RTNduals.html"},{"authors":"Clarice S. Groeneveld, Vinicius S. Chagas and Mauro A. A. Castro","name":"RTNsurvival","license":"Artistic-2.0","tags":["NetworkEnrichment","Survival","GeneRegulation","GeneSetEnrichment","NetworkInference","GraphAndNetwork"],"description":"RTNsurvival is a tool for integrating regulons generated\nby the RTN package with survival information. For a given\nregulon, the 2-tailed GSEA approach computes a differential\nEnrichment Score (dES) for each individual sample, and the dES\ndistribution of all samples is then used to assess the survival\nstatistics for the cohort. There are two main survival analysis\nworkflows: a Cox Proportional Hazards approach used to model\nregulons as predictors of survival time, and a Kaplan-Meier\nanalysis assessing the stratification of a cohort based on the\nregulon activity. All plots can be fine-tuned to the user's\nspecifications.","downloads_month":135,"downloads_total":3700,"page":"http://bioconductor.org/packages/release/bioc/html/RTNsurvival.html"},{"authors":"Luigi Marchionni <marchion@jhu.edu> and Svitlana Tyekucheva\n<svitlana@jimmy.harvard.edu>","name":"RTopper","license":"GPL (>= 3)","tags":["Microarray"],"description":"the RTopper package is designed to perform and integrate\ngene set enrichment results across multiple genomic platforms.","downloads_month":138,"downloads_total":32494,"page":"http://bioconductor.org/packages/release/bioc/html/RTopper.html"},{"authors":"Michael Lawrence, Vince Carey and Robert Gentleman","name":"rtracklayer","license":"Artistic-2.0 + file LICENSE","tags":["Annotation","Visualization","DataImport"],"description":"Extensible framework for interacting with multiple genome\nbrowsers (currently UCSC built-in) and manipulating annotation\ntracks in various formats (currently GFF, BED, bedGraph, BED15,\nWIG, BigWig and 2bit built-in). The user may export/import\ntracks to/from the supported browsers, as well as query and\nmodify the browser state, such as the current viewport.","downloads_month":23393,"downloads_total":2812036,"page":"http://bioconductor.org/packages/release/bioc/html/rtracklayer.html"},{"authors":"Jasmina Bogojeska","name":"Rtreemix","license":"LGPL","tags":["StatisticalMethod"],"description":"Rtreemix is a package that offers an environment for\nestimating the mutagenetic trees mixture models from\ncross-sectional data and using them for various predictions. It\nincludes functions for fitting the trees mixture models,\nlikelihood computations, model comparisons, waiting time\nestimations, stability analysis, etc.","downloads_month":154,"downloads_total":43366,"page":"http://bioconductor.org/packages/release/bioc/html/Rtreemix.html"},{"authors":"Diego Diez","name":"rTRM","license":"GPL-3","tags":["Transcription","Network","GeneRegulation","GraphAndNetwork"],"description":"rTRM identifies transcriptional regulatory modules (TRMs)\nfrom protein-protein interaction networks.","downloads_month":163,"downloads_total":23506,"page":"http://bioconductor.org/packages/release/bioc/html/rTRM.html"},{"authors":"Diego Diez","name":"rTRMui","license":"GPL-3","tags":["Transcription","Network","GeneRegulation","GraphAndNetwork","GUI"],"description":"This package provides a web interface to compute\ntranscriptional regulatory modules with rTRM.","downloads_month":130,"downloads_total":21314,"page":"http://bioconductor.org/packages/release/bioc/html/rTRMui.html"},{"authors":"Patryk Orzechowski and Artur Pańszczyk","name":"runibic","license":"MIT + file LICENSE","tags":["Microarray","Clustering","GeneExpression","Sequencing","Coverage"],"description":"This package implements UbiBic algorithm in R. This\nbiclustering algorithm for analysis of gene expression data was\nintroduced by Zhenjia Wang et al. in 2016. It is currently\nconsidered the most promising biclustering method for\nidentification of meaningful structures in complex and noisy\ndata.","downloads_month":168,"downloads_total":4228,"page":"http://bioconductor.org/packages/release/bioc/html/runibic.html"},{"authors":"Saskia Freytag","name":"RUVcorr","license":"GPL-2","tags":["GeneExpression","Normalization"],"description":"RUVcorr allows to apply global removal of unwanted\nvariation (ridged version of RUV) to real and simulated gene\nexpression data.","downloads_month":356,"downloads_total":30950,"page":"http://bioconductor.org/packages/release/bioc/html/RUVcorr.html"},{"authors":"Laurent Jacob","name":"RUVnormalize","license":"GPL-3","tags":["StatisticalMethod","Normalization"],"description":"RUVnormalize is meant to remove unwanted variation from\ngene expression data when the factor of interest is not\ndefined, e.g., to clean up a dataset for general use or to do\nany kind of unsupervised analysis.","downloads_month":150,"downloads_total":17834,"page":"http://bioconductor.org/packages/release/bioc/html/RUVnormalize.html"},{"authors":"Davide Risso [aut, cre, cph], Sandrine Dudoit [aut], Lorena\nPantano [ctb] and Kamil Slowikowski [ctb]","name":"RUVSeq","license":"Artistic-2.0","tags":["ImmunoOncology","DifferentialExpression","Preprocessing","RNASeq","Software"],"description":"This package implements the remove unwanted variation\n(RUV) methods of Risso et al. (2014) for the normalization of\nRNA-Seq read counts between samples.","downloads_month":1850,"downloads_total":95130,"page":"http://bioconductor.org/packages/release/bioc/html/RUVSeq.html"},{"authors":"Alexandre Bureau, Ingo Ruczinski, Samuel Younkin and Thomas\nSherman","name":"RVS","license":"GPL-2","tags":["ImmunoOncology","Genetics","GenomeWideAssociation","VariantDetection","ExomeSeq","WholeGenome"],"description":"Rare Variant Sharing (RVS) implements tests of association\nand linkage between rare genetic variant genotypes and a\ndichotomous phenotype, e.g. a disease status, in family\nsamples. The tests are based on probabilities of rare variant\nsharing by relatives under the null hypothesis of absence of\nlinkage and association between the rare variants and the\nphenotype and apply to single variants or multiple variants in\na region (e.g. gene-based test).","downloads_month":180,"downloads_total":3696,"page":"http://bioconductor.org/packages/release/bioc/html/RVS.html"},{"authors":"Egon Willighagen [aut, cre]\n(<https://orcid.org/0000-0001-7542-0286>) and Alex Pico [aut]\n(<https://orcid.org/0000-0001-5706-2163>)","name":"rWikiPathways","license":"MIT + file LICENSE","tags":["Visualization","GraphAndNetwork","ThirdPartyClient","Network","Metabolomics"],"description":"Use this package to interface with the WikiPathways API.","downloads_month":161,"downloads_total":3124,"page":"http://bioconductor.org/packages/release/bioc/html/rWikiPathways.html"},{"authors":"H. Pagès, M. Lawrence and P. Aboyoun","name":"S4Vectors","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"The S4Vectors package defines the Vector and List virtual\nclasses and a set of generic functions that extend the semantic\nof ordinary vectors and lists in R. Package developers can\neasily implement vector-like or list-like objects as concrete\nsubclasses of Vector or List. In addition, a few low-level\nconcrete subclasses of general interest (e.g. DataFrame, Rle,\nand Hits) are implemented in the S4Vectors package itself (many\nmore are implemented in the IRanges package and in other\nBioconductor infrastructure packages).","downloads_month":46662,"downloads_total":3430996,"page":"http://bioconductor.org/packages/release/bioc/html/S4Vectors.html"},{"authors":"William T. Barry","name":"safe","license":"GPL (>= 2)","tags":["DifferentialExpression","Pathways","GeneSetEnrichment","StatisticalMethod","Software"],"description":"SAFE is a resampling-based method for testing functional\ncategories in gene expression experiments. SAFE can be applied\nto 2-sample and multi-class comparisons, or simple linear\nregressions. Other experimental designs can also be\naccommodated through user-defined functions.","downloads_month":694,"downloads_total":80384,"page":"http://bioconductor.org/packages/release/bioc/html/safe.html"},{"authors":"Tim Beissbarth <tim.beissbarth@ams.med.uni-goettingen.de>, with\ncontributions from Gordon Smyth <smyth@wehi.edu.au> and Lavinia\nHyde <hyde@wehi.edu.au>.","name":"sagenhaft","license":"GPL (>= 2)","tags":["SAGE"],"description":"This package implements several functions useful for\nanalysis of gene expression data by sequencing tags as done in\nSAGE (Serial Analysis of Gene Expressen) data, i.e. extraction\nof a SAGE library from sequence files, sequence error\ncorrection, library comparison. Sequencing error correction is\nimplementing using an Expectation Maximization Algorithm based\non a Mixture Model of tag counts.","downloads_month":122,"downloads_total":42308,"page":"http://bioconductor.org/packages/release/bioc/html/sagenhaft.html"},{"authors":"Per Broberg","name":"SAGx","license":"GPL-3","tags":["Microarray","OneChannel","Preprocessing","DataImport","DifferentialExpression","Clustering","MultipleComparison","GeneExpression","GeneSetEnrichment","Pathways","Regression","KEGG"],"description":"A package for retrieval, preparation and analysis of data\nfrom the Affymetrix GeneChip. In particular the issue of\nidentifying differentially expressed genes is addressed.","downloads_month":284,"downloads_total":72546,"page":"http://bioconductor.org/packages/release/bioc/html/SAGx.html"},{"authors":"Alexey Stupnikov, Shailesh Tripathi and Frank Emmert-Streib","name":"samExploreR","license":"GPL-3","tags":["ImmunoOncology","Sequencing","SequenceMatching","RNASeq","ChIPSeq","DNASeq","WholeGenome","GeneTarget","Alignment","GeneExpression","GeneticVariability","GeneRegulation","Preprocessing","GenomeAnnotation","Software"],"description":"This R package is designed for subsampling procedure to\nsimulate sequencing experiments with reduced sequencing depth.\nThis package can be used to anlayze data generated from all\nmajor sequencing platforms such as Illumina GA, HiSeq, MiSeq,\nRoche GS-FLX, ABI SOLiD and LifeTech Ion PGM Proton sequencers.\nIt supports multiple operating systems incluidng Linux, Mac OS\nX, FreeBSD and Solaris. Was developed with usage of Rsubread.","downloads_month":159,"downloads_total":4640,"page":"http://bioconductor.org/packages/release/bioc/html/samExploreR.html"},{"authors":"Khadija El-Amrani","name":"sampleClassifier","license":"Artistic-2.0","tags":["ImmunoOncology","Classification","Microarray","RNASeq","GeneExpression"],"description":"The package is designed to classify gene expression\nprofiles.","downloads_month":185,"downloads_total":5272,"page":"http://bioconductor.org/packages/release/bioc/html/sampleClassifier.html"},{"authors":"Habil Zare and Parisa Shooshtari","name":"SamSPECTRAL","license":"GPL (>= 2)","tags":["FlowCytometry","CellBiology","Clustering","Cancer","FlowCytometry","StemCells","HIV","ImmunoOncology"],"description":"Samples large data such that spectral clustering is\npossible while preserving density information in edge weights.\nMore specifically, given a matrix of coordinates as input,\nSamSPECTRAL first builds the communities to sample the data\npoints. Then, it builds a graph and after weighting the edges\nby conductance computation, the graph is passed to a classic\nspectral clustering algorithm to find the spectral clusters.\nThe last stage of SamSPECTRAL is to combine the spectral\nclusters. The resulting \"connected components\" estimate\nbiological cell populations in the data. See the vignette for\nmore details on how to use this package, some illustrations,\nand simple examples.","downloads_month":276,"downloads_total":42878,"page":"http://bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html"},{"authors":"Jonathon T. Hill and Bradley Demarest","name":"sangerseqR","license":"GPL-2","tags":["Sequencing","SNP","Visualization"],"description":"This package contains several tools for analyzing Sanger\nSequencing data files in R, including reading .scf and .ab1\nfiles, making basecalls and plotting chromatograms.","downloads_month":462,"downloads_total":51046,"page":"http://bioconductor.org/packages/release/bioc/html/sangerseqR.html"},{"authors":"Alex J. Cornish and Florian Markowetz","name":"SANTA","license":"GPL (>= 2)","tags":["Network","NetworkEnrichment","Clustering"],"description":"This package provides methods for measuring the strength\nof association between a network and a phenotype. It does this\nby measuring clustering of the phenotype across the network\n(Knet). Vertices can also be individually ranked by their\nstrength of association with high-weight vertices (Knode).","downloads_month":219,"downloads_total":26348,"page":"http://bioconductor.org/packages/release/bioc/html/SANTA.html"},{"authors":"Shaohang Xu and Bo Wen","name":"sapFinder","license":"GPL-2","tags":["ImmunoOncology","MassSpectrometry","Proteomics","SNP","RNASeq","Visualization","ReportWriting"],"description":"sapFinder is developed to automate (1)\nvariation-associated database construction, (2) database\nsearching, (3) post-processing, (4) HTML-based report\ngeneration in shotgun proteomics.","downloads_month":148,"downloads_total":19376,"page":"http://bioconductor.org/packages/release/bioc/html/sapFinder.html"},{"authors":"R. Brent Calder","name":"savR","license":"AGPL-3","tags":["Sequencing"],"description":"Parse Illumina Sequence Analysis Viewer (SAV) files,\naccess data, and generate QC plots.","downloads_month":219,"downloads_total":21958,"page":"http://bioconductor.org/packages/release/bioc/html/savR.html"},{"authors":"Tomas Radivoyevitch and Vishak Venkateswaran","name":"SBMLR","license":"GPL-2","tags":["GraphAndNetwork","Pathways","Network"],"description":"This package contains a systems biology markup language\n(SBML) interface to R.","downloads_month":134,"downloads_total":46148,"page":"http://bioconductor.org/packages/release/bioc/html/SBMLR.html"},{"authors":"Vladimir Kiselev","name":"SC3","license":"GPL-3","tags":["ImmunoOncology","SingleCell","Software","Classification","Clustering","DimensionReduction","SupportVectorMachine","RNASeq","Visualization","Transcriptomics","DataRepresentation","GUI","DifferentialExpression","Transcription"],"description":"A tool for unsupervised clustering and analysis of single\ncell RNA-Seq data.","downloads_month":1054,"downloads_total":42866,"page":"http://bioconductor.org/packages/release/bioc/html/SC3.html"},{"authors":"Carolin Walter","name":"Scale4C","license":"LGPL-3","tags":["Visualization","QualityControl","DataImport","Sequencing","Coverage"],"description":"Scale4C is an R/Bioconductor package for scale-space\ntransformation and visualization of 4C-seq data. The\nscale-space transformation is a multi-scale visualization\ntechnique to transform a 2D signal (e.g. 4C-seq reads on a\ngenomic interval of choice) into a tesselation in the scale\nspace (2D, genomic position x scale factor) by applying\ndifferent smoothing kernels (Gauss, with increasing sigma).\nThis transformation allows for explorative analysis and\ncomparisons of the data's structure with other samples.","downloads_month":134,"downloads_total":3168,"page":"http://bioconductor.org/packages/release/bioc/html/Scale4C.html"},{"authors":"Stephen R. Piccolo and Andrea H. Bild and W. Evan Johnson","name":"SCAN.UPC","license":"MIT","tags":["ImmunoOncology","Software","Microarray","Preprocessing","RNASeq","TwoChannel","OneChannel"],"description":"SCAN is a microarray normalization method to facilitate\npersonalized-medicine workflows. Rather than processing\nmicroarray samples as groups, which can introduce biases and\npresent logistical challenges, SCAN normalizes each sample\nindividually by modeling and removing probe- and array-specific\nbackground noise using only data from within each array. SCAN\ncan be applied to one-channel (e.g., Affymetrix) or two-channel\n(e.g., Agilent) microarrays. The Universal exPression Codes\n(UPC) method is an extension of SCAN that estimates whether a\ngiven gene/transcript is active above background levels in a\ngiven sample. The UPC method can be applied to one-channel or\ntwo-channel microarrays as well as to RNA-Seq read counts.\nBecause UPC values are represented on the same scale and have\nan identical interpretation for each platform, they can be used\nfor cross-platform data integration.","downloads_month":328,"downloads_total":45602,"page":"http://bioconductor.org/packages/release/bioc/html/SCAN.UPC.html"},{"authors":"Davis McCarthy [aut, cre], Kieran Campbell [aut], Aaron Lun\n[aut, ctb], Quin Wills [aut] and Vladimir Kiselev [ctb]","name":"scater","license":"GPL-3","tags":["ImmunoOncology","SingleCell","RNASeq","QualityControl","Preprocessing","Normalization","Visualization","DimensionReduction","Transcriptomics","GeneExpression","Sequencing","Software","DataImport","DataRepresentation","Infrastructure","Coverage"],"description":"A collection of tools for doing various analyses of\nsingle-cell RNA-seq gene expression data, with a focus on\nquality control and visualization.","downloads_month":3983,"downloads_total":152904,"page":"http://bioconductor.org/packages/release/bioc/html/scater.html"},{"authors":"Yan Zhou","name":"SCBN","license":"GPL-2","tags":["DifferentialExpression","GeneExpression","Normalization"],"description":"This package provides a scale based normalization (SCBN)\nmethod to identify genes with differential expression between\ndifferent species. It takes into account the available\nknowledge of conserved orthologous genes and the hypothesis\ntesting framework to detect differentially expressed\northologous genes. The method on this package are described in\nthe article 'A statistical normalization method and\ndifferential expression analysis for RNA-seq data between\ndifferent species' by Yan Zhou, Jiadi Zhu, Tiejun Tong, Junhui\nWang, Bingqing Lin, Jun Zhang (2018, pending publication).","downloads_month":84,"downloads_total":630,"page":"http://bioconductor.org/packages/release/bioc/html/SCBN.html"},{"authors":"Keegan Korthauer","name":"scDD","license":"GPL-2","tags":["ImmunoOncology","Bayesian","Clustering","RNASeq","SingleCell","MultipleComparison","Visualization","DifferentialExpression"],"description":"This package implements a method to analyze single-cell\nRNA- seq Data utilizing flexible Dirichlet Process mixture\nmodels. Genes with differential distributions of expression are\nclassified into several interesting patterns of differences\nbetween two conditions. The package also includes functions for\nsimulating data with these patterns from negative binomial\ndistributions.","downloads_month":355,"downloads_total":9936,"page":"http://bioconductor.org/packages/release/bioc/html/scDD.html"},{"authors":"Peter Kharchenko [aut, cre] and Jean Fan [aut]","name":"scde","license":"GPL-2","tags":["ImmunoOncology","RNASeq","StatisticalMethod","DifferentialExpression","Bayesian","Transcription","Software"],"description":"The scde package implements a set of statistical methods\nfor analyzing single-cell RNA-seq data. scde fits individual\nerror models for single-cell RNA-seq measurements. These models\ncan then be used for assessment of differential expression\nbetween groups of cells, as well as other types of analysis.\nThe scde package also contains the pagoda framework which\napplies pathway and gene set overdispersion analysis to\nidentify and characterize putative cell subpopulations based on\ntranscriptional signatures. The overall approach to the\ndifferential expression analysis is detailed in the following\npublication: \"Bayesian approach to single-cell differential\nexpression analysis\" (Kharchenko PV, Silberstein L, Scadden DT,\nNature Methods, doi: 10.1038/nmeth.2967). The overall approach\nto subpopulation identification and characterization is\ndetailed in the following pre-print: \"Characterizing\ntranscriptional heterogeneity through pathway and gene set\noverdispersion analysis\" (Fan J, Salathia N, Liu R, Kaeser G,\nYung Y, Herman J, Kaper F, Fan JB, Zhang K, Chun J, and\nKharchenko PV, Nature Methods, doi:10.1038/nmeth.3734).","downloads_month":838,"downloads_total":39306,"page":"http://bioconductor.org/packages/release/bioc/html/scde.html"},{"authors":"Angeles Arzalluz-Luque [aut], Guillaume Devailly [aut, cre] and Anagha Joshi [aut]","name":"scFeatureFilter","license":"MIT + file LICENSE","tags":["ImmunoOncology","SingleCell","RNASeq","Preprocessing","GeneExpression"],"description":"An R implementation of the correlation-based method\ndeveloped in the Joshi laboratory to analyse and filter\nprocessed single-cell RNAseq data. It returns a filtered\nversion of the data containing only genes expression values\nunaffected by systematic noise.","downloads_month":119,"downloads_total":1878,"page":"http://bioconductor.org/packages/release/bioc/html/scFeatureFilter.html"},{"authors":"Vladimir Kiselev","name":"scfind","license":"GPL-3","tags":["ImmunoOncology","SingleCell","Software","RNASeq","Transcriptomics","DataRepresentation","Transcription","Sequencing","GeneExpression"],"description":"Recently a very large collection of single-cell RNA-seq\n(scRNA-seq) datasets has been generated and publicly released.\nFor the collection to be useful, the information must be\norganized in a way that supports queries that are relevant to\nresearchers. `scfind` builds an index from scRNA-seq datasets\nwhich organizes the information in a suitable and compact\nmanner so that the datasets can be very efficiently searched\nfor either cells or cell types in which a given list of genes\nis expressed.","downloads_month":220,"downloads_total":4994,"page":"http://bioconductor.org/packages/release/bioc/html/scfind.html"},{"authors":"Tony Chiang <tchiang@fhcrc.org>","name":"ScISI","license":"LGPL","tags":["GraphAndNetwork","Proteomics","NetworkInference","DecisionTree"],"description":"Package to create In Silico Interactomes","downloads_month":188,"downloads_total":53438,"page":"http://bioconductor.org/packages/release/bioc/html/ScISI.html"},{"authors":"Vladimir Kiselev","name":"scmap","license":"GPL-3","tags":["ImmunoOncology","SingleCell","Software","Classification","SupportVectorMachine","RNASeq","Visualization","Transcriptomics","DataRepresentation","Transcription","Sequencing","Preprocessing","GeneExpression","DataImport"],"description":"Single-cell RNA-seq (scRNA-seq) is widely used to\ninvestigate the composition of complex tissues since the\ntechnology allows researchers to define cell-types using\nunsupervised clustering of the transcriptome. However, due to\ndifferences in experimental methods and computational analyses,\nit is often challenging to directly compare the cells\nidentified in two different experiments. scmap is a method for\nprojecting cells from a scRNA-seq experiment on to the\ncell-types or individual cells identified in a different\nexperiment.","downloads_month":419,"downloads_total":7898,"page":"http://bioconductor.org/packages/release/bioc/html/scmap.html"},{"authors":"Divy Kangeyan <divyswar01@g.harvard.edu>","name":"scmeth","license":"GPL-2","tags":["DNAMethylation","QualityControl","Preprocessing","SingleCell","ImmunoOncology"],"description":"Functions to analyze methylation data can be found here.\nSome functions are relevant for single cell methylation data\nbut most other functions can be used for any methylation data.\nHighlight of this workflow is the comprehensive quality control\nreport.","downloads_month":165,"downloads_total":2352,"page":"http://bioconductor.org/packages/release/bioc/html/scmeth.html"},{"authors":"Rhonda Bacher","name":"SCnorm","license":"GPL (>= 2)","tags":["Normalization","RNASeq","SingleCell","ImmunoOncology"],"description":"This package implements SCnorm — a method to normalize\nsingle-cell RNA-seq data.","downloads_month":429,"downloads_total":8726,"page":"http://bioconductor.org/packages/release/bioc/html/SCnorm.html"},{"authors":"Michael Cole [aut, cre, cph], Davide Risso [aut and cph]","name":"scone","license":"Artistic-2.0","tags":["ImmunoOncology","Normalization","Preprocessing","QualityControl","GeneExpression","RNASeq","Software","Transcriptomics","Sequencing","SingleCell","Coverage"],"description":"SCONE is an R package for comparing and ranking the\nperformance of different normalization schemes for single-cell\nRNA-seq and other high-throughput analyses.","downloads_month":436,"downloads_total":11888,"page":"http://bioconductor.org/packages/release/bioc/html/scone.html"},{"authors":"Tyler J Burns","name":"Sconify","license":"Artistic-2.0","tags":["ImmunoOncology","SingleCell","FlowCytometry","Software","MultipleComparison","Visualization"],"description":"This package does k-nearest neighbor based statistics and\nvisualizations with flow and mass cytometery data. This gives\ntSNE maps\"fold change\" functionality and provides a data\nquality metric by assessing manifold overlap between fcs files\nexpected to be the same. Other applications using this package\ninclude imputation, marker redundancy, and testing the relative\ninformation loss of lower dimension embeddings compared to the\noriginal manifold.","downloads_month":183,"downloads_total":2508,"page":"http://bioconductor.org/packages/release/bioc/html/Sconify.html"},{"authors":"Carlos Ruiz [aut, cre] and Juan R. Gonzalez [aut]","name":"scoreInvHap","license":"file LICENSE","tags":["SNP","Genetics","GenomicVariation"],"description":"scoreInvHap can get the samples' inversion status of known\ninversions. scoreInvHap uses SNP data as input and requires the\nfollowing information about the inversion: genotype frequencies\nin the different haplotypes, R2 between the region SNPs and\ninversion status and heterozygote genotypes in the reference.\nThe package include this data for two well known inversions\n(8p23 and 17q21.31) and for two additional regions.","downloads_month":130,"downloads_total":3324,"page":"http://bioconductor.org/packages/release/bioc/html/scoreInvHap.html"},{"authors":"Luyi Tian","name":"scPipe","license":"GPL (>= 2)","tags":["ImmunoOncology","Software","Sequencing","RNASeq","GeneExpression","SingleCell","Visualization","SequenceMatching","Preprocessing","QualityControl","GenomeAnnotation"],"description":"A preprocessing pipeline for single cell RNA-seq data that\nstarts from the fastq files and produces a gene count matrix\nwith associated quality control information. It can process\nfastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium\n10x and SMART-seq protocols.","downloads_month":268,"downloads_total":6314,"page":"http://bioconductor.org/packages/release/bioc/html/scPipe.html"},{"authors":"Aaron Lun [aut, cre], Karsten Bach [aut], Jong Kyoung Kim\n[ctb], Antonio Scialdone [ctb] and Laleh Haghverdi [ctb]","name":"scran","license":"GPL-3","tags":["ImmunoOncology","Normalization","Sequencing","RNASeq","Software","GeneExpression","Transcriptomics","SingleCell","Visualization","BatchEffect","Clustering"],"description":"Implements functions for low-level analyses of single-cell\nRNA-seq data. Methods are provided for normalization of\ncell-specific biases, assignment of cell cycle phase, detection\nof highly variable and significantly correlated genes,\ncorrection of batch effects, identification of marker genes,\nand other common tasks in single-cell analysis workflows.","downloads_month":2744,"downloads_total":94846,"page":"http://bioconductor.org/packages/release/bioc/html/scran.html"},{"authors":"Zhe Wang [aut, cre], Junming Hu [aut] and Joshua Campbell [aut]","name":"scruff","license":"MIT + file LICENSE","tags":["Software","Technology","Sequencing","Alignment","RNASeq","SingleCell","WorkflowStep","Preprocessing","QualityControl","Visualization","ImmunoOncology"],"description":"A pipeline which processes single cell RNA-seq (scRNA-seq)\nreads from CEL-seq and CEL-seq2 protocols. Demultiplex\nscRNA-seq FASTQ files, align reads to reference genome using\nRsubread, and generate UMI filtered count matrix. Also provide\nvisualizations of read alignments and pre- and post-alignment\nQC metrics.","downloads_month":83,"downloads_total":716,"page":"http://bioconductor.org/packages/release/bioc/html/scruff.html"},{"authors":"Andrea Franceschini","name":"scsR","license":"GPL-2","tags":["Preprocessing"],"description":"Corrects genome-wide siRNA screens for seed mediated\noff-target effect. Suitable functions to identify the effective\nseeds/miRNAs and to visualize their effect are also provided in\nthe package.","downloads_month":148,"downloads_total":17750,"page":"http://bioconductor.org/packages/release/bioc/html/scsR.html"},{"authors":"Koki Tsuyuzaki [aut and cre]","name":"scTensor","license":"Artistic-2.0","tags":["DimensionReduction","SingleCell"],"description":"The algorithm is based on the non-negative tucker\ndecomposition (NTD) of nnTensor.","downloads_month":40,"downloads_total":208,"page":"http://bioconductor.org/packages/release/bioc/html/scTensor.html"},{"authors":"Yuntong Li <yuntong.li@uky.edu>, Chi Wang <chi.wang@uky.edu> and Li Chen <lichenuky@uky.edu>","name":"SDAMS","license":"GPL","tags":["ImmunoOncology","DifferentialExpression","Metabolomics","Proteomics","MassSpectrometry"],"description":"This Package utilizes a Semi-parametric Differential\nAbundance analysis (SDA) method for metabolomics and proteomics\ndata from mass spectrometry. SDA is able to robustly handle\nnon-normally distributed data and provides a clear\nquantification of the effect size.","downloads_month":169,"downloads_total":2310,"page":"http://bioconductor.org/packages/release/bioc/html/SDAMS.html"},{"authors":"Thomas J. Hardcastle","name":"segmentSeq","license":"GPL-3","tags":["MultipleComparison","Sequencing","Alignment","DifferentialExpression","QualityControl","DataImport"],"description":"High-throughput sequencing technologies allow the\nproduction of large volumes of short sequences, which can be\naligned to the genome to create a set of matches to the genome.\nBy looking for regions of the genome which to which there are\nhigh densities of matches, we can infer a segmentation of the\ngenome into regions of biological significance. The methods in\nthis package allow the simultaneous segmentation of data from\nmultiple samples, taking into account replicate data, in order\nto create a consensus segmentation. This has obvious\napplications in a number of classes of sequencing experiments,\nparticularly in the discovery of small RNA loci and novel mRNA\ntranscriptome discovery.","downloads_month":222,"downloads_total":49162,"page":"http://bioconductor.org/packages/release/bioc/html/segmentSeq.html"},{"authors":"Chaitanya Rastogi, Dahong Liu, and Harmen Bussemaker","name":"SELEX","license":"GPL (>=2)","tags":["Software","MotifDiscovery","MotifAnnotation","GeneRegulation","Transcription"],"description":"Tools for quantifying DNA binding specificities based on\nSELEX-seq data","downloads_month":133,"downloads_total":12482,"page":"http://bioconductor.org/packages/release/bioc/html/SELEX.html"},{"authors":"Ian Gonzalez and Wyatt Clark","name":"SemDist","license":"GPL (>= 2)","tags":["Classification","Annotation","GO","Software"],"description":"This package implements methods to calculate information\naccretion for a given version of the gene ontology and uses\nthis data to calculate remaining uncertainty, misinformation,\nand semantic similarity for given sets of predicted annotations\nand true annotations from a protein function predictor.","downloads_month":143,"downloads_total":14370,"page":"http://bioconductor.org/packages/release/bioc/html/SemDist.html"},{"authors":"Armin Rauschenberger [aut and cre]","name":"semisup","license":"GPL-3","tags":["SNP","GenomicVariation","SomaticMutation","Genetics","Classification","Clustering","DNASeq","Microarray","MultipleComparison"],"description":"Useful for detecting SNPs with interactive effects on a\nquantitative trait. This R packages moves away from testing\ninteraction terms, and moves towards testing whether an\nindividual SNP is involved in any interaction. This reduces the\nmultiple testing burden to one test per SNP, and allows for\ninteractions with unobserved factors. Analysing one SNP at a\ntime, it splits the individuals into two groups, based on the\nnumber of minor alleles. If the quantitative trait differs in\nmean between the two groups, the SNP has a main effect. If the\nquantitative trait differs in distribution between some\nindividuals in one group and all other individuals, it possibly\nhas an interactive effect. Implicitly, the membership\nprobabilities may suggest potential interacting variables.","downloads_month":133,"downloads_total":4788,"page":"http://bioconductor.org/packages/release/bioc/html/semisup.html"},{"authors":"Zhicheng Ji and Hongkai Ji","name":"SEPA","license":"GPL(>=2)","tags":["GeneExpression","Visualization","GUI","GO"],"description":"Given single-cell RNA-seq data and true experiment time of\ncells or pseudo-time cell ordering, SEPA provides convenient\nfunctions for users to assign genes into different gene\nexpression patterns such as constant, monotone increasing and\nincreasing then decreasing. SEPA then performs GO enrichment\nanalysis to analysis the functional roles of genes with same or\nsimilar patterns.","downloads_month":137,"downloads_total":10644,"page":"http://bioconductor.org/packages/release/bioc/html/SEPA.html"},{"authors":"Yuting Chen [aut, cre] and Andrew Teschendorff [aut]","name":"SEPIRA","license":"GPL-3","tags":["GeneExpression","Transcription","GeneRegulation","GeneTarget","NetworkInference","Network","Software"],"description":"SEPIRA (Systems EPigenomics Inference of Regulatory\nActivity) is an algorithm that infers sample-specific\ntranscription factor activity from the genome-wide expression\nor DNA methylation profile of the sample.","downloads_month":119,"downloads_total":1882,"page":"http://bioconductor.org/packages/release/bioc/html/SEPIRA.html"},{"authors":"Xinan Yang <xyang2@uchicago.edu>; Bin Wang <binw@uchicago.edu>","name":"seq2pathway","license":"GPL-2","tags":["Software"],"description":"Seq2pathway is a novel tool for functional gene-set (or\ntermed as pathway) analysis of next-generation sequencing data,\nconsisting of \"seq2gene\" and \"gene2path\" components. The\nseq2gene links sequence-level measurements of genomic regions\n(including SNPs or point mutation coordinates) to gene-level\nscores, and the gene2pathway summarizes gene scores to\npathway-scores for each sample. The seq2gene has the\nfeasibility to assign both coding and non-exon regions to a\nbroader range of neighboring genes than only the nearest one,\nthus facilitating the study of functional non-coding regions.\nThe gene2pathway takes into account the quantity of\nsignificance for gene members within a pathway compared those\noutside a pathway. The output of seq2pathway is a general\nstructure of quantitative pathway-level scores, thus allowing\none to functional interpret such datasets as RNA-seq, ChIP-seq,\nGWAS, and derived from other next generational sequencing\nexperiments.","downloads_month":152,"downloads_total":14888,"page":"http://bioconductor.org/packages/release/bioc/html/seq2pathway.html"},{"authors":"Xiuwen Zheng [aut, cre], Stephanie Gogarten [aut], David Levine\n[ctb] and Cathy Laurie [ctb]","name":"SeqArray","license":"GPL-3","tags":["Infrastructure","DataRepresentation","Sequencing","Genetics"],"description":"Data management of large-scale whole-genome sequencing\nvariant calls with thousands of individuals: genotypic data\n(e.g., SNVs, indels and structural variation calls) and\nannotations in SeqArray files are stored in an array-oriented\nand compressed manner, with efficient data access using the R\nprogramming language.","downloads_month":739,"downloads_total":44996,"page":"http://bioconductor.org/packages/release/bioc/html/SeqArray.html"},{"authors":"Daniel Jones <dcjones@cs.washington.edu>","name":"seqbias","license":"LGPL-3","tags":["Sequencing"],"description":"This package implements a model of per-position sequencing\nbias in high-throughput sequencing data using a simple Bayesian\nnetwork, the structure and parameters of which are trained on a\nset of aligned reads and a reference genome sequence.","downloads_month":179,"downloads_total":37452,"page":"http://bioconductor.org/packages/release/bioc/html/seqbias.html"},{"authors":"Erik Fasterius [aut and cre]","name":"seqCAT","license":"MIT + file LICENCE","tags":["Coverage","GenomicVariation","Sequencing","VariantAnnotation"],"description":"The seqCAT package uses variant calling data (in the form\nof VCF files) from high throughput sequencing technologies to\nauthenticate and validate the source, function and\ncharacteristics of biological samples used in scientific\nendeavours.","downloads_month":182,"downloads_total":3606,"page":"http://bioconductor.org/packages/release/bioc/html/seqCAT.html"},{"authors":"David Mosen-Ansorena","name":"seqCNA","license":"GPL-3","tags":["CopyNumberVariation","Genetics","Sequencing"],"description":"Copy number analysis of high-throughput sequencing cancer\ndata with fast summarization, extensive filtering and improved\nnormalization","downloads_month":158,"downloads_total":23682,"page":"http://bioconductor.org/packages/release/bioc/html/seqCNA.html"},{"authors":"Guangchuang Yu [aut and cre]","name":"seqcombo","license":"Artistic-2.0","tags":["Alignment","Software","Visualization"],"description":"Provides useful functions for visualizing sequence\nrecombination and virus reassortment events.","downloads_month":174,"downloads_total":3736,"page":"http://bioconductor.org/packages/release/bioc/html/seqcombo.html"},{"authors":"Xi Wang <Xi.Wang@newcastle.edu.au>","name":"SeqGSEA","license":"GPL (>= 3)","tags":["Sequencing","RNASeq","GeneSetEnrichment","GeneExpression","DifferentialExpression","DifferentialSplicing","ImmunoOncology"],"description":"The package generally provides methods for gene set\nenrichment analysis of high-throughput RNA-Seq data by\nintegrating differential expression and splicing. It uses\nnegative binomial distribution to model read count data, which\naccounts for sequencing biases and biological variation. Based\non permutation tests, statistical significance can also be\nachieved regarding each gene's differential expression and\nsplicing, respectively.","downloads_month":387,"downloads_total":52794,"page":"http://bioconductor.org/packages/release/bioc/html/SeqGSEA.html"},{"authors":"Oliver Bembom","name":"seqLogo","license":"LGPL (>= 2)","tags":["SequenceMatching"],"description":"seqLogo takes the position weight matrix of a DNA sequence\nmotif and plots the corresponding sequence logo as introduced\nby Schneider and Stephens (1990).","downloads_month":1772,"downloads_total":274426,"page":"http://bioconductor.org/packages/release/bioc/html/seqLogo.html"},{"authors":"Vanja Haberle <vanja.haberle@gmail.com>","name":"seqPattern","license":"GPL-3","tags":["Visualization","SequenceMatching"],"description":"Visualising oligonucleotide patterns and sequence motifs\noccurrences across a large set of sequences centred at a common\nreference point and sorted by a user defined feature.","downloads_month":424,"downloads_total":31284,"page":"http://bioconductor.org/packages/release/bioc/html/seqPattern.html"},{"authors":"Przemyslaw Stempor [aut, cph and cre]","name":"seqplots","license":"GPL-3","tags":["ImmunoOncology","ChIPSeq","RNASeq","Sequencing","Software","Visualization"],"description":"SeqPlots is a tool for plotting next generation sequencing\n(NGS) based experiments' signal tracks, e.g. reads coverage\nfrom ChIP-seq, RNA-seq and DNA accessibility assays like\nDNase-seq and MNase-seq, over user specified genomic features,\ne.g. promoters, gene bodies, etc. It can also calculate\nsequence motif density profiles from reference genome. The data\nare visualized as average signal profile plot, with error\nestimates (standard error and 95% confidence interval) shown as\nfields, or as series of heatmaps that can be sorted and\nclustered using hierarchical clustering, k-means algorithm and\nself organising maps. Plots can be prepared using R programming\nlanguage or web browser based graphical user interface (GUI)\nimplemented using Shiny framework. The dual-purpose\nimplementation allows running the software locally on desktop\nor deploying it on server. SeqPlots is useful for both for\nexploratory data analyses and preparing replicable, publication\nquality plots. Other features of the software include\ncollaboration and data sharing capabilities, as well as ability\nto store pre-calculated result matrixes, that combine many\nsequencing experiments and in-silico generated tracks with\nmultiple different features. These binaries can be further used\nto generate new combination plots on fly, run automated batch\noperations or share with colleagues, who can adjust their\nplotting parameters without loading actual tracks and\nrecalculating numeric values. SeqPlots relays on Bioconductor\npackages, mainly on rtracklayer for data input and BSgenome\npackages for reference genome sequence and annotations.","downloads_month":349,"downloads_total":26208,"page":"http://bioconductor.org/packages/release/bioc/html/seqplots.html"},{"authors":"Joseph R Boyd [aut and cre]","name":"seqsetvis","license":"MIT + file LICENSE","tags":["Software","ChIPSeq","Sequencing","Visualization"],"description":"seqsetvis enables the visualization and analysis of\nmultiple genomic datasets. Although seqsetvis was designed for\nthe comparison of mulitple ChIP-seq datasets, this package is\ndomain-agnostic and allows the processing of multiple genomic\ncoordinate files (bed-like files) and signal files (bigwig\nfiles or bam pileups).","downloads_month":131,"downloads_total":2286,"page":"http://bioconductor.org/packages/release/bioc/html/seqsetvis.html"},{"authors":"Qian Liu [aut and cre]","name":"SeqSQC","license":"GPL-3","tags":["Experiment Data","Homo_sapiens_Data","Sequencing Data","Project1000genomes","Genome"],"description":"The SeqSQC is designed to identify problematic samples in\nNGS data, including samples with gender mismatch,\ncontamination, cryptic relatedness, and population outlier.","downloads_month":144,"downloads_total":3296,"page":"http://bioconductor.org/packages/release/bioc/html/SeqSQC.html"},{"authors":"Wolfgang Kaisers","name":"seqTools","license":"Artistic-2.0","tags":["QualityControl","Sequencing"],"description":"Analyze read length, phred scores and alphabet frequency\nand DNA k-mers on uncompressed and compressed fastq files.","downloads_month":191,"downloads_total":19130,"page":"http://bioconductor.org/packages/release/bioc/html/seqTools.html"},{"authors":"Stephanie M. Gogarten, Xiuwen Zheng and Adrienne Stilp","name":"SeqVarTools","license":"GPL-3","tags":["SNP","GeneticVariability","Sequencing","Genetics"],"description":"An interface to the fast-access storage format for VCF\ndata provided in SeqArray, with tools for common operations and\nanalysis.","downloads_month":581,"downloads_total":34728,"page":"http://bioconductor.org/packages/release/bioc/html/SeqVarTools.html"},{"authors":"Wanding Zhou [aut, cre], Hui Shen [aut] and Timothy Triche [ctb]","name":"sesame","license":"Artistic-2.0","tags":["DNAMethylation","MethylationArray","Preprocessing","QualityControl"],"description":"Tools For analyzing Illumina Infinium DNA methylation\narrays.","downloads_month":117,"downloads_total":4952,"page":"http://bioconductor.org/packages/release/bioc/html/sesame.html"},{"authors":"Nan Xiao [aut, cre], Tengfei Yin [aut], Emile Young [ctb], Dusan Randjelovic [ctb], Seven Bridges Genomics [cph and fnd]","name":"sevenbridges","license":"Apache License 2.0 | file LICENSE","tags":["Software","DataImport","ThirdPartyClient"],"description":"R client and utilities for Seven Bridges platform API,\nfrom Cancer Genomics Cloud to other Seven Bridges supported\nplatforms.","downloads_month":148,"downloads_total":11880,"page":"http://bioconductor.org/packages/release/bioc/html/sevenbridges.html"},{"authors":"Jonas Ibn-Salem [aut and cre]","name":"sevenC","license":"GPL-3","tags":["DNA3DStructure","ChIPchip","Coverage","DataImport","Epigenetics","FunctionalGenomics","Classification","Regression","ChIPSeq","HiC","Annotation"],"description":"Chromatin looping is an essential feature of eukaryotic\ngenomes and can bring regulatory sequences, such as enhancers\nor transcription factor binding sites, in the close physical\nproximity of regulated target genes. Here, we provide sevenC,\nan R package that uses protein binding signals from ChIP-seq\nand sequence motif information to predict chromatin looping\nevents. Cross-linking of proteins that bind close to loop\nanchors result in ChIP-seq signals at both anchor loci. These\nsignals are used at CTCF motif pairs together with their\ndistance and orientation to each other to predict whether they\ninteract or not. The resulting chromatin loops might be used to\nassociate enhancers or transcription factor binding sites\n(e.g., ChIP-seq peaks) to regulated target genes.","downloads_month":130,"downloads_total":2018,"page":"http://bioconductor.org/packages/release/bioc/html/sevenC.html"},{"authors":"Leonard Goldstein","name":"SGSeq","license":"Artistic-2.0","tags":["AlternativeSplicing","ImmunoOncology","RNASeq","Transcription"],"description":"SGSeq is a software package for analyzing splice events\nfrom RNA-seq data. Input data are RNA-seq reads mapped to a\nreference genome in BAM format. Genes are represented as a\nsplice graph, which can be obtained from existing annotation or\npredicted from the mapped sequence reads. Splice events are\nidentified from the graph and are quantified locally using\nstructurally compatible reads at the start or end of each\nsplice variant. The software includes functions for splice\nevent prediction, quantification, visualization and\ninterpretation.","downloads_month":445,"downloads_total":25344,"page":"http://bioconductor.org/packages/release/bioc/html/SGSeq.html"},{"authors":"Jean-Philippe Fortin [cre, aut] and Kasper Daniel Hansen [aut]","name":"shinyMethyl","license":"Artistic-2.0","tags":["DNAMethylation","Microarray","TwoChannel","Preprocessing","QualityControl"],"description":"Interactive tool for visualizing Illumina methylation\narray data. Both the 450k and EPIC array are supported.","downloads_month":254,"downloads_total":25756,"page":"http://bioconductor.org/packages/release/bioc/html/shinyMethyl.html"},{"authors":"Frederic Fournier <frederic.fournier@crchuq.ulaval.ca> and Arnaud\nDroit <arnaud.droit@crchuq.ulaval.ca>","name":"shinyTANDEM","license":"GPL-3","tags":["ImmunoOncology","MassSpectrometry","Proteomics"],"description":"This package provides a GUI interface for rTANDEM. The GUI\nis primarily designed to visualize rTANDEM result object or\nresult xml files. But it will also provides an interface for\ncreating parameter objects, launching searches or performing\nconversions between R objects and xml files.","downloads_month":165,"downloads_total":22640,"page":"http://bioconductor.org/packages/release/bioc/html/shinyTANDEM.html"},{"authors":"Martin Morgan, Michael Lawrence and Simon Anders","name":"ShortRead","license":"Artistic-2.0","tags":["DataImport","Sequencing","QualityControl"],"description":"This package implements sampling, iteration, and input of\nFASTQ files. The package includes functions for filtering and\ntrimming reads, and for generating a quality assessment report.\nData are represented as DNAStringSet-derived objects, and\neasily manipulated for a diversity of purposes.  The package\nalso contains legacy support for early single-end, ungapped\nalignment formats.","downloads_month":8964,"downloads_total":1087168,"page":"http://bioconductor.org/packages/release/bioc/html/ShortRead.html"},{"authors":"Konrad Zych [aut, cre]\n(<https://orcid.org/0000-0001-7426-0516>), Jakob Wirbel [aut]\n(<https://orcid.org/0000-0002-4073-3562>), Georg Zeller [aut]\n(<https://orcid.org/0000-0003-1429-7485>), Morgan Essex [ctb], Nicolai Karcher [ctb] and Kersten Breuer [ctb]","name":"SIAMCAT","license":"GPL-3","tags":["ImmunoOncology","Metagenomics","Classification","Microbiome","Sequencing","Preprocessing","Clustering","FeatureExtraction","GeneticVariability","MultipleComparison","Regression"],"description":"Pipeline for Statistical Inference of Associations between\nMicrobial Communities And host phenoTypes (SIAMCAT). A primary\ngoal of analyzing microbiome data is to determine changes in\ncommunity composition that are associated with environmental\nfactors. In particular, linking human microbiome composition to\nhost phenotypes such as diseases has become an area of intense\nresearch. For this, robust statistical modeling and biomarker\nextraction toolkits are crucially needed. SIAMCAT provides a\nfull pipeline supporting data preprocessing, statistical\nassociation testing, statistical modeling (LASSO logistic\nregression) including tools for evaluation and interpretation\nof these models (such as cross validation, parameter selection,\nROC analysis and diagnostic model plots).","downloads_month":184,"downloads_total":2456,"page":"http://bioconductor.org/packages/release/bioc/html/SIAMCAT.html"},{"authors":"Xiaobin Xing and Wu Wei","name":"SICtools","license":"GPL (>=2)","tags":["Alignment","Sequencing","Coverage","SequenceMatching","QualityControl","DataImport","Software","SNP","VariantDetection"],"description":"This package is to find SNV/Indel differences between two\nbam files with near relationship in a way of pairwise\ncomparison thourgh each base position across the genome region\nof interest. The difference is inferred by fisher test and\neuclidean distance, the input of which is the base count\n(A,T,G,C) in a given position and read counts for indels that\nspan no less than 2bp on both sides of indel region.","downloads_month":134,"downloads_total":6790,"page":"http://bioconductor.org/packages/release/bioc/html/SICtools.html"},{"authors":"Wessel N. van Wieringen <w.vanwieringen@vumc.nl>","name":"sigaR","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","aCGH","GeneExpression","Pathways"],"description":"Facilitates the joint analysis of high-throughput data\nfrom multiple molecular levels. Contains functions for\nmanipulation of objects, various analysis types, and some\nvisualization.","downloads_month":157,"downloads_total":31802,"page":"http://bioconductor.org/packages/release/bioc/html/sigaR.html"},{"authors":"Rory Stark <rory.stark@cruk.cam.ac.uk> and Justin Norden","name":"SigCheck","license":"Artistic-2.0","tags":["GeneExpression","Classification","GeneSetEnrichment"],"description":"While gene signatures are frequently used to predict\nphenotypes (e.g. predict prognosis of cancer patients), it it\nnot always clear how optimal or meaningful they are (cf David\nVenet, Jacques E. Dumont, and Vincent Detours' paper \"Most\nRandom Gene Expression Signatures Are Significantly Associated\nwith Breast Cancer Outcome\"). Based on suggestions in that\npaper, SigCheck accepts a data set (as an ExpressionSet) and a\ngene signature, and compares its performance on survival and/or\nclassification tasks against a) random gene signatures of the\nsame length; b) known, related and unrelated gene signatures;\nand c) permuted data and/or metadata.","downloads_month":158,"downloads_total":15580,"page":"http://bioconductor.org/packages/release/bioc/html/SigCheck.html"},{"authors":"Pijush Das Developer [aut, cre], Dr. Susanta Roychudhury User\n[ctb] and Dr. Sucheta Tripathy User [ctb]","name":"sigFeature","license":"GPL","tags":["FeatureExtraction","GeneExpression","Microarray","Transcription","mRNAMicroarray","GenePrediction","Normalization","Classification","SupportVectorMachine"],"description":"This package provides a novel feature selection algorithm\nfor binary classification using support vector machine\nrecursive feature elimination SVM-RFE and t-statistic. In this\nfeature selection process, the selected features are\ndifferentially significant between the two classes and also\nthey are good classifier with higher degree of classification\naccuracy.","downloads_month":118,"downloads_total":1070,"page":"http://bioconductor.org/packages/release/bioc/html/sigFeature.html"},{"authors":"Patrick Kimes and Christopher Cabanski","name":"SigFuge","license":"GPL-3","tags":["Clustering","Visualization","RNASeq"],"description":"Algorithm for testing significance of clustering in\nRNA-seq data.","downloads_month":141,"downloads_total":21444,"page":"http://bioconductor.org/packages/release/bioc/html/SigFuge.html"},{"authors":"Holger Schwender","name":"siggenes","license":"LGPL (>= 2)","tags":["MultipleComparison","Microarray","GeneExpression","SNP","ExonArray","DifferentialExpression"],"description":"Identification of differentially expressed genes and\nestimation of the False Discovery Rate (FDR) using both the\nSignificance Analysis of Microarrays (SAM) and the Empirical\nBayes Analyses of Microarrays (EBAM).","downloads_month":2939,"downloads_total":436056,"page":"http://bioconductor.org/packages/release/bioc/html/siggenes.html"},{"authors":"Elika Garg [aut, cre], Carl Murie [aut], Heydar Ensha [ctb] and Robert Nadon [aut]","name":"sights","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","CellBasedAssays","MicrotitrePlateAssay","Normalization","MultipleComparison","Preprocessing","QualityControl","BatchEffect","Visualization"],"description":"SIGHTS is a suite of normalization methods, statistical\ntests, and diagnostic graphical tools for high throughput\nscreening (HTS) assays. HTS assays use microtitre plates to\nscreen large libraries of compounds for their biological,\nchemical, or biochemical activity.","downloads_month":168,"downloads_total":6982,"page":"http://bioconductor.org/packages/release/bioc/html/sights.html"},{"authors":"Rafael Rosales, Rodrigo Drummond, Renan Valieris and Israel Tojal\nda Silva","name":"signeR","license":"GPL-3","tags":["GenomicVariation","SomaticMutation","StatisticalMethod","Visualization"],"description":"The signeR package provides an empirical Bayesian approach\nto mutational signature discovery. It is designed to analyze\nsingle nucleotide variaton (SNV) counts in cancer genomes, but\ncan also be applied to other features as well. Functionalities\nto characterize signatures or genome samples according to\nexposure patterns are also provided.","downloads_month":260,"downloads_total":10190,"page":"http://bioconductor.org/packages/release/bioc/html/signeR.html"},{"authors":"Alexandre Gouy [aut and cre]","name":"signet","license":"GPL-2","tags":["Software","Pathways","DifferentialExpression","GeneExpression","NetworkEnrichment","GraphAndNetwork","KEGG"],"description":"An R package to detect selection in biological pathways.\nUsing gene selection scores and biological pathways data, one\ncan search for high-scoring subnetworks of genes within\npathways and test their significance.","downloads_month":134,"downloads_total":2272,"page":"http://bioconductor.org/packages/release/bioc/html/signet.html"},{"authors":"Weil Lai (optimized R, C code) and Lu Tian and Peter Park\n(algorithm development and initial R code)","name":"sigPathway","license":"GPL-2","tags":["DifferentialExpression","MultipleComparison"],"description":"Conducts pathway analysis by calculating the NT_k and NE_k\nstatistics as described in Tian et al. (2005)","downloads_month":247,"downloads_total":68290,"page":"http://bioconductor.org/packages/release/bioc/html/sigPathway.html"},{"authors":"Renee X. de Menezes and Judith M. Boer","name":"SIM","license":"GPL (>= 2)","tags":["Microarray","Visualization"],"description":"Finds associations between two human genomic datasets.","downloads_month":143,"downloads_total":45400,"page":"http://bioconductor.org/packages/release/bioc/html/SIM.html"},{"authors":"M. R. Nezami Ranjbar <nranjbar@vt.edu>","name":"SIMAT","license":"GPL-2","tags":["ImmunoOncology","Software","Metabolomics","MassSpectrometry"],"description":"This package provides a pipeline for analysis of GC-MS\ndata acquired in selected ion monitoring (SIM) mode. The tool\nalso provides a guidance in choosing appropriate fragments for\nthe targets of interest by using an optimization algorithm.\nThis is done by considering overlapping peaks from a provided\nlibrary by the user.","downloads_month":209,"downloads_total":13260,"page":"http://bioconductor.org/packages/release/bioc/html/SIMAT.html"},{"authors":"Bettina Fischer, Enrico Ferrero, Robert Stojnic and Steve Russell","name":"SimBindProfiles","license":"GPL-3","tags":["Microarray","Software"],"description":"SimBindProfiles identifies common and unique binding\nregions in genome tiling array data. This package does not rely\non peak calling, but directly compares binding profiles\nprocessed on the same array platform. It implements a simple\nthreshold approach, thus allowing retrieval of commonly and\ndifferentially bound regions between datasets as well as events\nof compensation and increased binding.","downloads_month":144,"downloads_total":21086,"page":"http://bioconductor.org/packages/release/bioc/html/SimBindProfiles.html"},{"authors":"Yan Zhou","name":"SIMD","license":"GPL-3","tags":["ImmunoOncology","DifferentialMethylation","SingleCell","DifferentialExpression"],"description":"This package provides a inferential analysis method for\ndetecting differentially expressed CpG sites in MeDIP-seq data.\nIt uses statistical framework and EM algorithm, to identify\ndifferentially expressed CpG sites. The methods on this package\nare described in the article 'Methylation-level Inferences and\nDetection of Differential Methylation with Medip-seq Data' by\nYan Zhou, Jiadi Zhu, Mingtao Zhao, Baoxue Zhang, Chunfu Jiang\nand Xiyan Yang (2018, pending publication).","downloads_month":83,"downloads_total":716,"page":"http://bioconductor.org/packages/release/bioc/html/SIMD.html"},{"authors":"Astrid Deschenes [cre, aut], Elsa Bernatchez [aut], Charles\nJoly Beauparlant [aut], Fabien Claude Lamaze [aut], Rawane Samb\n[aut], Pascal Belleau [aut] and Arnaud Droit [aut]","name":"similaRpeak","license":"Artistic-2.0","tags":["BiologicalQuestion","ChIPSeq","Genetics","MultipleComparison","DifferentialExpression"],"description":"This package calculates metrics which assign a level of\nsimilarity between ChIP-Seq profiles.","downloads_month":134,"downloads_total":15176,"page":"http://bioconductor.org/packages/release/bioc/html/similaRpeak.html"},{"authors":"Daniele Ramazzotti [aut, cre], Bo Wang [aut], Luca De Sano\n[aut] and Serafim Batzoglou [ctb]","name":"SIMLR","license":"file LICENSE","tags":["ImmunoOncology","Clustering","GeneExpression","Sequencing","SingleCell"],"description":"Single-cell RNA-seq technologies enable high throughput\ngene expression measurement of individual cells, and allow the\ndiscovery of heterogeneity within cell populations. Measurement\nof cell-to-cell gene expression similarity is critical for the\nidentification, visualization and analysis of cell populations.\nHowever, single-cell data introduce challenges to conventional\nmeasures of gene expression similarity because of the high\nlevel of noise, outliers and dropouts. We develop a novel\nsimilarity-learning framework, SIMLR (Single-cell\nInterpretation via Multi-kernel LeaRning), which learns an\nappropriate distance metric from the data for dimension\nreduction, clustering and visualization.","downloads_month":274,"downloads_total":14410,"page":"http://bioconductor.org/packages/release/bioc/html/SIMLR.html"},{"authors":"Crispin J Miller","name":"simpleaffy","license":"GPL (>= 2)","tags":["Microarray","OneChannel","QualityControl","Preprocessing","Transcription","DataImport","DifferentialExpression","Annotation","ReportWriting","Visualization"],"description":"Provides high level functions for reading Affy .CEL files,\nphenotypic data, and then computing simple things with it, such\nas t-tests, fold changes and the like. Makes heavy use of the\naffy library. Also has some basic scatter plot functions and\nmechanisms for generating high resolution journal figures...","downloads_month":1096,"downloads_total":631706,"page":"http://bioconductor.org/packages/release/bioc/html/simpleaffy.html"},{"authors":"Yuqing Zhang, Christoph Bernau and Levi Waldron","name":"simulatorZ","license":"Artistic-2.0","tags":["Survival"],"description":"simulatorZ is a package intended primarily to simulate\ncollections of independent genomic data sets, as well as\nperforming training and validation with predicting algorithms.\nIt supports ExpressionSet and RangedSummarizedExperiment\nobjects.","downloads_month":148,"downloads_total":15048,"page":"http://bioconductor.org/packages/release/bioc/html/simulatorZ.html"},{"authors":"Miguel Julia <migueljuliamolina@gmail.com>, Amalio Telenti\n<atelenti@jcvi.org> and Antonio Rausell\n<antonio.rausell@isb-sib.ch>","name":"sincell","license":"GPL (>= 2)","tags":["ImmunoOncology","Sequencing","RNASeq","Clustering","GraphAndNetwork","Visualization","GeneExpression","GeneSetEnrichment","BiomedicalInformatics","CellBiology","FunctionalGenomics","SystemsBiology"],"description":"Cell differentiation processes are achieved through a\ncontinuum of hierarchical intermediate cell-states that might\nbe captured by single-cell RNA seq. Existing computational\napproaches for the assessment of cell-state hierarchies from\nsingle-cell data might be formalized under a general workflow\ncomposed of i) a metric to assess cell-to-cell similarities\n(combined or not with a dimensionality reduction step), and ii)\na graph-building algorithm (optionally making use of a\ncells-clustering step). Sincell R package implements a\nmethodological toolbox allowing flexible workflows under such\nframework. Furthermore, Sincell contributes new algorithms to\nprovide cell-state hierarchies with statistical support while\naccounting for stochastic factors in single-cell RNA seq.\nGraphical representations and functional association tests are\nprovided to interpret hierarchies.","downloads_month":178,"downloads_total":15994,"page":"http://bioconductor.org/packages/release/bioc/html/sincell.html"},{"authors":"Aaron Lun [aut, cph], Davide Risso [aut, cre, cph] and Keegan\nKorthauer [ctb]","name":"SingleCellExperiment","license":"GPL-3","tags":["ImmunoOncology","DataRepresentation","DataImport","Infrastructure","SingleCell"],"description":"Defines a S4 class for storing data from single-cell\nexperiments. This includes specialized methods to store and\nretrieve spike-in information, dimensionality reduction\ncoordinates and size factors for each cell, along with the\nusual metadata for genes and libraries.","downloads_month":5507,"downloads_total":104820,"page":"http://bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html"},{"authors":"David Jenkins","name":"singleCellTK","license":"MIT + file LICENSE","tags":["SingleCell","GeneExpression","DifferentialExpression","Alignment","Clustering","ImmunoOncology"],"description":"Run common single cell analysis directly through your\nbrowser including differential expression, downsampling\nanalysis, and clustering.","downloads_month":198,"downloads_total":3134,"page":"http://bioconductor.org/packages/release/bioc/html/singleCellTK.html"},{"authors":"Momeneh Foroutan [aut, ctb], Dharmesh Bhuva [aut, ctb], Ruqian\nLyu [aut and cre]","name":"singscore","license":"GPL-3","tags":["Software","GeneExpression","GeneSetEnrichment"],"description":"A simple single-sample gene signature scoring method that\nuses rank-based statistics to analyze the sample's gene\nexpression profile. It scores the expression activities of gene\nsets at a single-sample level.","downloads_month":123,"downloads_total":2330,"page":"http://bioconductor.org/packages/release/bioc/html/singscore.html"},{"authors":"Bhakti Dwivedi and Jeanne Kowalski","name":"SISPA","license":"GPL-2","tags":["GeneSetEnrichment","GenomeWideAssociation"],"description":"Sample Integrated Set Profile Analysis (SISPA) is a method\ndesigned to define sample groups with similar gene set\nenrichment profiles.","downloads_month":140,"downloads_total":9682,"page":"http://bioconductor.org/packages/release/bioc/html/SISPA.html"},{"authors":"Weiliang Qiu <weiliang.qiu@gmail.com> and Mei-Ling Ting Lee\n<meilinglee@sph.osu.edu> and George Alex Whitmore\n<george.whitmore@mcgill.ca>","name":"sizepower","license":"LGPL","tags":["Microarray"],"description":"This package has been prepared to assist users in\ncomputing either a sample size or power value for a microarray\nexperimental study. The user is referred to the cited\nreferences for technical background on the methodology\nunderpinning these calculations. This package provides support\nfor five types of sample size and power calculations. These\nfive types can be adapted in various ways to encompass many of\nthe standard designs encountered in practice.","downloads_month":134,"downloads_total":56812,"page":"http://bioconductor.org/packages/release/bioc/html/sizepower.html"},{"authors":"Ryan Putney [cre, aut], Steven Eschrich [aut] and Anders Berglund\n[aut]","name":"skewr","license":"GPL-2","tags":["DNAMethylation","TwoChannel","Preprocessing","QualityControl"],"description":"The skewr package is a tool for visualizing the output of\nthe Illumina Human Methylation 450k BeadChip to aid in quality\ncontrol. It creates a panel of nine plots. Six of the plots\nrepresent the density of either the methylated intensity or the\nunmethylated intensity given by one of three subsets of the\n485,577 total probes. These subsets include Type I-red, Type\nI-green, and Type II.The remaining three distributions give the\ndensity of the Beta-values for these same three subsets. Each\nof the nine plots optionally displays the distributions of the\n\"rs\" SNP probes and the probes associated with imprinted genes\nas series of 'tick' marks located above the x-axis.","downloads_month":128,"downloads_total":11480,"page":"http://bioconductor.org/packages/release/bioc/html/skewr.html"},{"authors":"Florian Buettner [aut], Naruemon Pratanwanich [aut], Davis\nMcCarthy [aut, cre], John Marioni [aut] and Oliver Stegle [aut]","name":"slalom","license":"GPL-2","tags":["ImmunoOncology","SingleCell","RNASeq","Normalization","Visualization","DimensionReduction","Transcriptomics","GeneExpression","Sequencing","Software","Reactome"],"description":"slalom is a scalable modelling framework for single-cell\nRNA-seq data that uses gene set annotations to dissect\nsingle-cell transcriptome heterogeneity, thereby allowing to\nidentify biological drivers of cell-to-cell variability and\nmodel confounding factors.","downloads_month":195,"downloads_total":4174,"page":"http://bioconductor.org/packages/release/bioc/html/slalom.html"},{"authors":"Nolwenn LeMeur, Zhen Jiang, Ting-Yuan Liu, Jess Mar and Robert\nGentleman","name":"SLGI","license":"Artistic-2.0","tags":["GraphAndNetwork","Proteomics","Genetics","Network"],"description":"A variety of data files and functions for the analysis of\ngenetic interactions","downloads_month":163,"downloads_total":47246,"page":"http://bioconductor.org/packages/release/bioc/html/SLGI.html"},{"authors":"Kelly Street [aut, cre, cph], Davide Risso [aut], Diya Das\n[aut], Sandrine Dudoit [ths], Koen Van den Berge [ctb] and Robrecht Cannoodt [ctb]","name":"slingshot","license":"Artistic-2.0","tags":["Clustering","DifferentialExpression","GeneExpression","RNASeq","Sequencing","Software","Sequencing","SingleCell","Transcriptomics","Visualization"],"description":"Provides functions for inferring continuous, branching\nlineage structures in low-dimensional data. Slingshot was\ndesigned to model developmental trajectories in single-cell RNA\nsequencing data and serve as a component in an analysis\npipeline after dimensionality reduction and clustering. It is\nflexible enough to handle arbitrarily many branching events and\nallows for the incorporation of prior knowledge through\nsupervised graph construction.","downloads_month":274,"downloads_total":3252,"page":"http://bioconductor.org/packages/release/bioc/html/slingshot.html"},{"authors":"Eric J. Kort","name":"slinky","license":"MIT + file LICENSE","tags":["DataImport","ThirdPartyClient","GeneExpression","DifferentialExpression","GeneSetEnrichment","PatternLogic"],"description":"Wrappers to query the L1000 metadata available via the\nclue.io REST API as well as helpers for dealing with LINCS gctx\nfiles, extracting data sets of interest, converting to\nSummarizedExperiment objects, and some facilities for\nperforming streamlined differential expression analysis of\nthese data sets.","downloads_month":97,"downloads_total":1008,"page":"http://bioconductor.org/packages/release/bioc/html/slinky.html"},{"authors":"Matthias Kohl","name":"SLqPCR","license":"GPL (>= 2)","tags":["MicrotitrePlateAssay","qPCR"],"description":"Functions for analysis of real-time quantitative PCR data\nat SIRS-Lab GmbH","downloads_month":168,"downloads_total":49856,"page":"http://bioconductor.org/packages/release/bioc/html/SLqPCR.html"},{"authors":"Qingzhou Zhang [aut and cre]","name":"SMAD","license":"MIT + file LICENSE","tags":["MassSpectrometry","Proteomics","Software"],"description":"Assigning probability scores to prey proteins captured in\naffinity purification mass spectrometry (AP-MS) expriments to\ninfer protein- protein interactions. The output would\nfacilitate non-specific background removal as contaminants are\ncommonly found in AP-MS data.","downloads_month":0,"downloads_total":58,"page":"http://bioconductor.org/packages/release/bioc/html/SMAD.html"},{"authors":"Robin Andersson <robin.andersson@lcb.uu.se>","name":"SMAP","license":"GPL-2","tags":["Microarray","TwoChannel","CopyNumberVariation"],"description":"Functions and classes for DNA copy number profiling of\narray-CGH data","downloads_month":123,"downloads_total":41870,"page":"http://bioconductor.org/packages/release/bioc/html/SMAP.html"},{"authors":"Neil Ari Wijetunga, Andrew Damon Johnston and John Murray Greally","name":"SMITE","license":"GPL (>=2)","tags":["ImmunoOncology","DifferentialMethylation","DifferentialExpression","SystemsBiology","NetworkEnrichment","GenomeAnnotation","Network","Sequencing","RNASeq","Coverage"],"description":"This package builds on the Epimods framework which\nfacilitates finding weighted subnetworks (\"modules\") on\nIllumina Infinium 27k arrays using the SpinGlass algorithm, as\nimplemented in the iGraph package. We have created a class of\ngene centric annotations associated with p-values and effect\nsizes and scores from any researchers prior statistical results\nto find functional modules.","downloads_month":195,"downloads_total":9972,"page":"http://bioconductor.org/packages/release/bioc/html/SMITE.html"},{"authors":"David Venet <davenet@ulb.ac.be>","name":"SNAGEE","license":"Artistic-2.0","tags":["Microarray","OneChannel","TwoChannel","QualityControl"],"description":"Signal-to-Noise applied to Gene Expression Experiments.\nSignal-to-noise ratios can be used as a proxy for quality of\ngene expression studies and samples. The SNRs can be calculated\non any gene expression data set as long as gene IDs are\navailable, no access to the raw data files is necessary. This\nallows to flag problematic studies and samples in any public\ndata set.","downloads_month":120,"downloads_total":24824,"page":"http://bioconductor.org/packages/release/bioc/html/SNAGEE.html"},{"authors":"Mike L. Smith, John C. Marioni, Steven McKinney, Thomas\nHardcastle and Natalie P. Thorne","name":"snapCGH","license":"GPL","tags":["Microarray","CopyNumberVariation","TwoChannel","Preprocessing"],"description":"Methods for segmenting, normalising and processing aCGH\ndata; including plotting functions for visualising raw and\nsegmented data for individual and multiple arrays.","downloads_month":185,"downloads_total":66442,"page":"http://bioconductor.org/packages/release/bioc/html/snapCGH.html"},{"authors":"Brig Mecham and John D. Storey <jstorey@princeton.edu>","name":"snm","license":"LGPL","tags":["Microarray","OneChannel","TwoChannel","MultiChannel","DifferentialExpression","ExonArray","GeneExpression","Transcription","MultipleComparison","Preprocessing","QualityControl"],"description":"SNM is a modeling strategy especially designed for\nnormalizing high-throughput genomic data. The underlying\npremise of our approach is that your data is a function of what\nwe refer to as study-specific variables. These variables are\neither biological variables that represent the target of the\nstatistical analysis, or adjustment variables that represent\nfactors arising from the experimental or biological setting the\ndata is drawn from. The SNM approach aims to simultaneously\nmodel all study-specific variables in order to more accurately\ncharacterize the biological or clinical variables of interest.","downloads_month":222,"downloads_total":44178,"page":"http://bioconductor.org/packages/release/bioc/html/snm.html"},{"authors":"Robert Scharpf <rscharpf@jhu.edu> and Ingo Ruczinski","name":"SNPchip","license":"LGPL (>= 2)","tags":["CopyNumberVariation","SNP","GeneticVariability","Visualization"],"description":"Functions for plotting SNP array data; maintained for\nhistorical reasons","downloads_month":392,"downloads_total":101252,"page":"http://bioconductor.org/packages/release/bioc/html/SNPchip.html"},{"authors":"David Montaner [aut and cre]","name":"SNPediaR","license":"GPL-2","tags":["SNP","VariantAnnotation"],"description":"SNPediaR provides some tools for downloading and parsing\ndata from the SNPedia web site <http://www.snpedia.com>. The\nimplemented functions allow users to import the wiki text\navailable in SNPedia pages and to extract the most relevant\ninformation out of them. If some information in the downloaded\npages is not automatically processed by the library functions,\nusers can easily implement their own parsers to access it in an\nefficient way.","downloads_month":138,"downloads_total":7812,"page":"http://bioconductor.org/packages/release/bioc/html/SNPediaR.html"},{"authors":"Christian Arnold [aut, cre], Pooja Bhat [aut] and Judith Zaugg\n[aut]","name":"SNPhood","license":"LGPL (>= 3)","tags":["Software"],"description":"To date, thousands of single nucleotide polymorphisms\n(SNPs) have been found to be associated with complex traits and\ndiseases. However, the vast majority of these\ndisease-associated SNPs lie in the non-coding part of the\ngenome, and are likely to affect regulatory elements, such as\nenhancers and promoters, rather than function of a protein.\nThus, to understand the molecular mechanisms underlying genetic\ntraits and diseases, it becomes increasingly important to study\nthe effect of a SNP on nearby molecular traits such as\nchromatin environment or transcription factor (TF) binding.\nTowards this aim, we developed SNPhood, a user-friendly\n*Bioconductor* R package to investigate and visualize the local\nneighborhood of a set of SNPs of interest for NGS data such as\nchromatin marks or transcription factor binding sites from\nChIP-Seq or RNA- Seq experiments. SNPhood comprises a set of\neasy-to-use functions to extract, normalize and summarize reads\nfor a genomic region, perform various data quality checks,\nnormalize read counts using additional input files, and to\ncluster and visualize the regions according to the binding\npattern. The regions around each SNP can be binned in a\nuser-defined fashion to allow for analysis of very broad\npatterns as well as a detailed investigation of specific\nbinding shapes. Furthermore, SNPhood supports the integration\nwith genotype information to investigate and visualize\ngenotype-specific binding patterns. Finally, SNPhood can be\nemployed for determining, investigating, and visualizing\nallele-specific binding patterns around the SNPs of interest.","downloads_month":144,"downloads_total":11206,"page":"http://bioconductor.org/packages/release/bioc/html/SNPhood.html"},{"authors":"Xiuwen Zheng [aut, cre, cph], Stephanie Gogarten [ctb], Cathy\nLaurie [ctb], Bruce Weir [ctb and ths]","name":"SNPRelate","license":"GPL-3","tags":["Infrastructure","Genetics","StatisticalMethod","PrincipalComponent"],"description":"Genome-wide association studies (GWAS) are widely used to\ninvestigate the genetic basis of diseases and traits, but they\npose many computational challenges. We developed an R package\nSNPRelate to provide a binary format for single-nucleotide\npolymorphism (SNP) data in GWAS utilizing CoreArray Genomic\nData Structure (GDS) data files. The GDS format offers the\nefficient operations specifically designed for integers with\ntwo bits, since a SNP could occupy only two bits. SNPRelate is\nalso designed to accelerate two key computations on SNP data\nusing parallel computing for multi-core symmetric\nmultiprocessing computer architectures: Principal Component\nAnalysis (PCA) and relatedness analysis using\nIdentity-By-Descent measures. The SNP GDS format is also used\nby the GWASTools package with the support of S4 classes and\ngeneric functions. The extended GDS format is implemented in\nthe SeqArray package to support the storage of single\nnucleotide variations (SNVs), insertion/deletion polymorphism\n(indel) and structural variation calls.","downloads_month":1819,"downloads_total":106646,"page":"http://bioconductor.org/packages/release/bioc/html/SNPRelate.html"},{"authors":"David Clayton <dc208@cam.ac.uk>","name":"snpStats","license":"GPL-3","tags":["Microarray","SNP","GeneticVariability"],"description":"Classes and statistical methods for large SNP association\nstudies. This extends the earlier snpMatrix package, allowing\nfor uncertainty in genotypes.","downloads_month":2195,"downloads_total":456736,"page":"http://bioconductor.org/packages/release/bioc/html/snpStats.html"},{"authors":"Gopuraja Dharmalingam and Tom Carroll","name":"soGGi","license":"GPL (>= 3)","tags":["Sequencing","ChIPSeq","Coverage"],"description":"The soGGi package provides a toolset to create genomic\ninterval aggregate/summary plots of signal or motif occurence\nfrom BAM and bigWig files as well as PWM, rlelist, GRanges and\nGAlignments Bioconductor objects. soGGi allows for\nnormalisation, transformation and arithmetic operation on and\nbetween summary plot objects as well as grouping and subsetting\nof plots by GRanges objects and user supplied metadata. Plots\nare created using the GGplot2 libary to allow user defined\nmanipulation of the returned plot object. Coupled together,\nsoGGi features a broad set of methods to visualise genomics\ndata in the context of groups of genomic intervals such as\ngenes, superenhancers and transcription factor binding events.","downloads_month":200,"downloads_total":14132,"page":"http://bioconductor.org/packages/release/bioc/html/soGGi.html"},{"authors":"Julian Gehring","name":"SomaticSignatures","license":"MIT + file LICENSE","tags":["Sequencing","SomaticMutation","Visualization","Clustering","GenomicVariation","StatisticalMethod"],"description":"The SomaticSignatures package identifies mutational\nsignatures of single nucleotide variants (SNVs).  It provides a\ninfrastructure related to the methodology described in\nNik-Zainal (2012, Cell), with flexibility in the matrix\ndecomposition algorithms.","downloads_month":465,"downloads_total":49496,"page":"http://bioconductor.org/packages/release/bioc/html/SomaticSignatures.html"},{"authors":"Gregory Ryslik and Hongyu Zhao","name":"SpacePAC","license":"GPL-2","tags":["Clustering","Proteomics"],"description":"Identifies clustering of somatic mutations in proteins via\na simulation approach while considering the protein's tertiary\nstructure.","downloads_month":157,"downloads_total":22356,"page":"http://bioconductor.org/packages/release/bioc/html/SpacePAC.html"},{"authors":"Boyu Ren<bor158@mail.harvard.edu>, Emma\nSchwager<emma.schwager@gmail.com>, Timothy\nTickle<ttickle@hsph.harvard.edu> and Curtis Huttenhower\n<chuttenh@hsph.harvard.edu>","name":"sparseDOSSA","license":"MIT + file LICENSE","tags":["ImmunoOncology","Bayesian","Microbiome","Metagenomics","Software"],"description":"The package is to provide a model based Bayesian method to\ncharacterize and simulate microbiome data. sparseDOSSA's model\ncaptures the marginal distribution of each microbial feature as\na truncated, zero-inflated log-normal distribution, with\nparameters distributed as a parent log-normal distribution. The\nmodel can be effectively fit to reference microbial datasets in\norder to parameterize their microbes and communities, or to\nsimulate synthetic datasets of similar population structure.\nMost importantly, it allows users to include both known\nfeature-feature and feature-metadata correlation structures and\nthus provides a gold standard to enable benchmarking of\nstatistical methods for metagenomic data analysis.","downloads_month":145,"downloads_total":4828,"page":"http://bioconductor.org/packages/release/bioc/html/sparseDOSSA.html"},{"authors":"Irene Zeng [aut, cre] and Thomas Lumley [ctb]","name":"sparsenetgls","license":"GPL-3","tags":["ImmunoOncology","GraphAndNetwork","Regression","Metabolomics","CopyNumberVariation","MassSpectrometry","Proteomics","Software","Visualization"],"description":"The package provides methods of combining the graph\nstructure learning and generalized least squares regression to\nimprove the regression estimation. The main function\nsparsenetgls() provides solutions for multivariate regression\nwith Gaussian distributed dependant variables and explanatory\nvariables utlizing multiple well-known graph structure learning\napproaches to estimating the precision matrix, and uses a\npenalized variance covariance matrix with a distance tuning\nparameter of the graph structure in deriving the sandwich\nestimators in generalized least squares (gls) regression. This\npackage also provides functions for assessing a Gaussian\ngraphical model which uses the penalized approach. It uses\nReceiver Operative Characteristics curve as a visualization\ntool in the assessment.","downloads_month":105,"downloads_total":786,"page":"http://bioconductor.org/packages/release/bioc/html/sparsenetgls.html"},{"authors":"Daniele Ramazzotti [cre, aut], Avantika Lal [aut], Keli Liu\n[ctb], Luca De Sano [ctb], Robert Tibshirani [ctb] and Arend Sidow\n[aut]","name":"SparseSignatures","license":"file LICENSE","tags":["BiomedicalInformatics","SomaticMutation"],"description":"Point mutations occurring in a genome can be divided into\n96 categories based on the base being mutated, the base it is\nmutated into and its two flanking bases. Therefore, for any\npatient, it is possible to represent all the point mutations\noccurring in that patient’s tumor as a vector of length 96,\nwhere each element represents the count of mutations for a\ngiven category in the patient. A mutational signature\nrepresents the pattern of mutations produced by a mutagen or\nmutagenic process inside the cell. Each signature can also be\nrepresented by a vector of length 96, where each element\nrepresents the probability that this particular mutagenic\nprocess generates a mutation of the 96 above mentioned\ncategories. In this R package, we provide a set of functions to\nextract and visualize the mutational signatures that best\nexplain the mutation counts of a large number of patients.","downloads_month":127,"downloads_total":2314,"page":"http://bioconductor.org/packages/release/bioc/html/SparseSignatures.html"},{"authors":"Joseph Bergenstraahle [aut and cre]","name":"SpatialCPie","license":"MIT + file LICENSE","tags":["Transcriptomics","Clustering","RNASeq","Software"],"description":"SpatialCPie is an R package designed to facilitate cluster\nevaluation for spatial transcriptomics data by providing\nintuitive visualizations that display the relationships between\nclusters in order to guide the user during cluster\nidentification and other downstream applications. The package\nis built around a shiny \"gadget\" to allow the exploration of\nthe data with multiple plots in parallel and an interactive UI.\nThe user can easily toggle between different cluster\nresolutions in order to choose the most appropriate visual\ncues.","downloads_month":11,"downloads_total":38,"page":"http://bioconductor.org/packages/release/bioc/html/SpatialCPie.html"},{"authors":"Christian Panse [aut, cre]\n(<https://orcid.org/0000-0003-1975-3064>), Jonas Grossmann\n[aut] (<https://orcid.org/0000-0002-6899-9020>), Christian\nTrachsel [aut] and Witold E. Wolski [ctb]","name":"specL","license":"GPL-3","tags":["MassSpectrometry","Proteomics"],"description":"provides a function for generating spectra libraries which\ncan be used for MRM SRM MS workflows in proteomics. The package\nprovides a BiblioSpec reader, a function which can add the\nprotein information using a FASTA formatted amino acid file,\nand an export method for using the created library in the\nSpectronaut software.","downloads_month":169,"downloads_total":16958,"page":"http://bioconductor.org/packages/release/bioc/html/specL.html"},{"authors":"Florence Cavalli","name":"SpeCond","license":"LGPL (>=2)","tags":["Microarray","DifferentialExpression","MultipleComparison","Clustering","ReportWriting"],"description":"This package performs a gene expression data analysis to\ndetect condition-specific genes. Such genes are significantly\nup- or down-regulated in a small number of conditions. It does\nso by fitting a mixture of normal distributions to the\nexpression values. Conditions can be environmental conditions,\ndifferent tissues, organs or any other sources that you wish to\ncompare in terms of gene expression.","downloads_month":218,"downloads_total":45742,"page":"http://bioconductor.org/packages/release/bioc/html/SpeCond.html"},{"authors":"Kellen Cresswell <cresswellkg@vcu.edu>, John Stansfield\n<stansfieldjc@vcu.edu> and Mikhail Dozmorov\n<mikhail.dozmorov@vcuhealth.org>","name":"SpectralTAD","license":"MIT + file LICENSE","tags":["Software","HiC","Sequencing","FeatureExtraction","Clustering"],"description":"SpectralTAD is an R package designed to identify\nTopologically Associated Domains (TADs) from Hi-C contact\nmatrices. It uses a modified version of spectral clustering\nthat uses a sliding window to quickly detect TADs. The function\nworks on a range of different formats of contact matrices and\nreturns a bed file of TAD coordinates. The method does not\nrequire users to adjust any parameters to work and gives them\ncontrol over the number of hierarchical levels to be returned.","downloads_month":0,"downloads_total":2,"page":"http://bioconductor.org/packages/release/bioc/html/SpectralTAD.html"},{"authors":"Xinyi YANG Developer, Jennifer E. DENT Developer and Christine\nNARDINI Supervisor","name":"SPEM","license":"GPL-2","tags":["Network","NetworkInference","Software"],"description":"This package can optimize the parameter in S-system models\ngiven time series data","downloads_month":146,"downloads_total":24012,"page":"http://bioconductor.org/packages/release/bioc/html/SPEM.html"},{"authors":"Adi Laurentiu Tarca <atarca@med.wayne.edu>, Purvesh Kathri\n<purvesh@cs.wayne.edu> and Sorin Draghici <sorin@wayne.edu>","name":"SPIA","license":"file LICENSE","tags":["Microarray","GraphAndNetwork"],"description":"This package implements the Signaling Pathway Impact\nAnalysis (SPIA) which uses the information form a list of\ndifferentially expressed genes and their log fold changes\ntogether with signaling pathways topology, in order to identify\nthe pathways most relevant to the condition under the study.","downloads_month":791,"downloads_total":110018,"page":"http://bioconductor.org/packages/release/bioc/html/SPIA.html"},{"authors":"Claudia Cava, Antonio Colaprico, Alex Graudenzi, Gloria\nBertoli, Tiago C. Silva, Catharina Olsen, Houtan Noushmehr, Gianluca Bontempi, Giancarlo Mauri and Isabella Castiglioni","name":"SpidermiR","license":"GPL (>= 3)","tags":["GeneRegulation","miRNA","Network"],"description":"The aims of SpidermiR are : i) facilitate the network\nopen-access data retrieval from GeneMania data, ii) prepare the\ndata using the appropriate gene nomenclature, iii) integration\nof miRNA data in a specific network, iv) provide different\nstandard analyses and v) allow the user to visualize the\nresults. In more detail, the package provides multiple methods\nfor query, prepare and download network data (GeneMania), and\nthe integration with validated and predicted miRNA data\n(mirWalk, miRTarBase, miRandola,Pharmaco-miR,DIANA, Miranda,\nPicTar and TargetScan) and the use of standard analysis\n(igraph) and visualization methods (networkD3).","downloads_month":214,"downloads_total":13118,"page":"http://bioconductor.org/packages/release/bioc/html/SpidermiR.html"},{"authors":"Delphine Baillon, Paul Leclercq <paulleclercq@hotmail.com>, Sarah Ternisien, Thomas Heim and Enrico Carlon\n<enrico.carlon@fys.kuleuven.be>","name":"spikeLI","license":"GPL-2","tags":["Microarray","QualityControl"],"description":"SpikeLI is a package that performs the analysis of the\nAffymetrix spike-in data using the Langmuir Isotherm. The aim\nof this package is to show the advantages of a\nphysical-chemistry based analysis of the Affymetrix microarray\ndata compared to the traditional methods. The spike-in (or\nLatin square) data for the HGU95 and HGU133 chipsets have been\ndownloaded from the Affymetrix web site. The model used in the\nspikeLI package is described in details in E. Carlon and T.\nHeim, Physica A 362, 433 (2006).","downloads_month":123,"downloads_total":40656,"page":"http://bioconductor.org/packages/release/bioc/html/spikeLI.html"},{"authors":"Matthew N McCall <mccallm@gmail.com> and Rafael A Irizarry\n<rafa@jhu.edu>","name":"spkTools","license":"GPL (>= 2)","tags":["Software","Technology","Microarray"],"description":"The package contains functions that can be used to compare\nexpression measures on different array platforms.","downloads_month":139,"downloads_total":39566,"page":"http://bioconductor.org/packages/release/bioc/html/spkTools.html"},{"authors":"Luke Zappia","name":"splatter","license":"GPL-3 + file LICENSE","tags":["SingleCell","RNASeq","Transcriptomics","GeneExpression","Sequencing","Software","ImmunoOncology"],"description":"Splatter is a package for the simulation of single-cell\nRNA sequencing count data. It provides a simple interface for\ncreating complex simulations that are reproducible and\nwell-documented. Parameters can be estimated from real data and\nfunctions are provided for comparing real and simulated\ndatasets.","downloads_month":470,"downloads_total":14766,"page":"http://bioconductor.org/packages/release/bioc/html/splatter.html"},{"authors":"Laurent Gautier <laurent@cbs.dtu.dk>","name":"splicegear","license":"LGPL","tags":["Infrastructure","Transcription"],"description":"A set of tools to work with alternative splicing","downloads_month":141,"downloads_total":42770,"page":"http://bioconductor.org/packages/release/bioc/html/splicegear.html"},{"authors":"Wolfgang Kaisers","name":"spliceSites","license":"GPL-2","tags":["RNAseq","GeneExpression","DifferentialExpression","Proteomics"],"description":"Performs splice centered analysis on RNA-seq data.","downloads_month":147,"downloads_total":22312,"page":"http://bioconductor.org/packages/release/bioc/html/spliceSites.html"},{"authors":"D. Bindreither, M. Carlson, M. Morgan and H. Pagès","name":"SplicingGraphs","license":"Artistic-2.0","tags":["Genetics","Annotation","DataRepresentation","Visualization","Sequencing","RNASeq","GeneExpression","AlternativeSplicing","Transcription","ImmunoOncology"],"description":"This package allows the user to create, manipulate, and\nvisualize splicing graphs and their bubbles based on a gene\nmodel for a given organism. Additionally it allows the user to\nassign RNA-seq reads to the edges of a set of splicing graphs,\nand to summarize them in different ways.","downloads_month":233,"downloads_total":34452,"page":"http://bioconductor.org/packages/release/bioc/html/SplicingGraphs.html"},{"authors":"Agata Michna","name":"splineTimeR","license":"GPL-3","tags":["GeneExpression","DifferentialExpression","TimeCourse","Regression","GeneSetEnrichment","NetworkEnrichment","NetworkInference","GraphAndNetwork"],"description":"This package provides functions for differential gene\nexpression analysis of gene expression time-course data.\nNatural cubic spline regression models are used. Identified\ngenes may further be used for pathway enrichment analysis\nand/or the reconstruction of time dependent gene regulatory\nassociation networks.","downloads_month":145,"downloads_total":8334,"page":"http://bioconductor.org/packages/release/bioc/html/splineTimeR.html"},{"authors":"Diana Low","name":"SPLINTER","license":"GPL-2","tags":["ImmunoOncology","GeneExpression","RNASeq","Visualization","AlternativeSplicing"],"description":"SPLINTER provides tools to analyze alternative splicing\nsites, interpret outcomes based on sequence information, select\nand design primers for site validiation and give visual\nrepresentation of the event to guide downstream experiments.","downloads_month":181,"downloads_total":7532,"page":"http://bioconductor.org/packages/release/bioc/html/SPLINTER.html"},{"authors":"Wolfgang Huber and Oleg Sklyar","name":"splots","license":"LGPL","tags":["Visualization","Sequencing","MicrotitrePlateAssay"],"description":"The splots package provides the plotScreen function for\nvisualising data in microtitre plate or slide format.","downloads_month":297,"downloads_total":75752,"page":"http://bioconductor.org/packages/release/bioc/html/splots.html"},{"authors":"Markus List, Azim Dehghani Amirabad, Dennis Kostka and Marcel H.\nSchulz","name":"SPONGE","license":"GPL (>=3)","tags":["GeneExpression","Transcription","GeneRegulation","NetworkInference","Transcriptomics","SystemsBiology","Regression"],"description":"This package provides methods to efficiently detect\ncompetitive endogeneous RNA interactions between two genes.\nSuch interactions are mediated by one or several miRNAs such\nthat both gene and miRNA expression data for a larger number of\nsamples is needed as input.","downloads_month":137,"downloads_total":3192,"page":"http://bioconductor.org/packages/release/bioc/html/SPONGE.html"},{"authors":"Qunhua Li, Chris Fraley, Adrian Raftery Department of\nStatistics and University of Washington","name":"spotSegmentation","license":"GPL (>= 2)","tags":["Microarray","TwoChannel","QualityControl","Preprocessing"],"description":"Spot segmentation via model-based clustering and gridding\nfor blocks within microarray slides, as described in Li et al,\nRobust Model-Based Segmentation of Microarray Images, Technical\nReport no. 473, Department of Statistics, University of\nWashington.","downloads_month":120,"downloads_total":44068,"page":"http://bioconductor.org/packages/release/bioc/html/spotSegmentation.html"},{"authors":"Martial Sankar, supervised by Christian Hardtke and Ioannis\nXenarios","name":"SQUADD","license":"GPL (>=2)","tags":["GraphAndNetwork","Network","Visualization"],"description":"This package SQUADD is a SQUAD add-on. It permits to\ngenerate SQUAD simulation matrix, prediction Heat-Map and\nCorrelation Circle from PCA analysis.","downloads_month":122,"downloads_total":31740,"page":"http://bioconductor.org/packages/release/bioc/html/SQUADD.html"},{"authors":"Jack Zhu and Sean Davis","name":"SRAdb","license":"Artistic-2.0","tags":["Infrastructure","Sequencing","DataImport"],"description":"The Sequence Read Archive (SRA) is the largest public\nrepository of sequencing data from the next generation of\nsequencing platforms including Roche 454 GS System, Illumina\nGenome Analyzer, Applied Biosystems SOLiD System, Helicos\nHeliscope, and others. However, finding data of interest can be\nchallenging using current tools. SRAdb is an attempt to make\naccess to the metadata associated with submission, study,\nsample, experiment and run much more feasible. This is\naccomplished by parsing all the NCBI SRA metadata into a SQLite\ndatabase that can be stored and queried locally. Fulltext\nsearch in the package make querying metadata very flexible and\npowerful.  fastq and sra files can be downloaded for doing\nalignment locally. Beside ftp protocol, the SRAdb has funcitons\nsupporting fastp protocol (ascp from Aspera Connect) for faster\ndownloading large data files over long distance. The SQLite\ndatabase is updated regularly as new data is added to SRA and\ncan be downloaded at will for the most up-to-date metadata.","downloads_month":772,"downloads_total":149150,"page":"http://bioconductor.org/packages/release/bioc/html/SRAdb.html"},{"authors":"Charles Warden","name":"sRAP","license":"GPL-3","tags":["GeneExpression","RNAseq","Microarray","Preprocessing","QualityControl","Statistics","DifferentialExpression","Visualization","GeneSetEnrichment","GO"],"description":"This package provides a pipeline for gene expression\nanalysis (primarily for RNA-Seq data).  The normalization\nfunction is specific for RNA-Seq analysis, but all other\nfunctions (Quality Control Figures, Differential Expression and\nVisualization, and Functional Enrichment via BD-Func) will work\nwith any type of gene expression data.","downloads_month":134,"downloads_total":24230,"page":"http://bioconductor.org/packages/release/bioc/html/sRAP.html"},{"authors":"Isar Nassiri [aut, cre], Matthew McCall [aut and cre]","name":"SRGnet","license":"GPL-2","tags":["Software","StatisticalMethod","Regression"],"description":"We developed SRGnet to analyze synergistic regulatory\nmechanisms in transcriptome profiles that act to enhance the\noverall cell response to combination of mutations, drugs or\nenvironmental exposure. This package can be used to identify\nregulatory modules downstream of synergistic response genes,\nprioritize synergistic regulatory genes that may be potential\nintervention targets, and contextualize gene perturbation\nexperiments.","downloads_month":132,"downloads_total":6044,"page":"http://bioconductor.org/packages/release/bioc/html/SRGnet.html"},{"authors":"Zytnicki Matthias [aut and cre]","name":"srnadiff","license":"GPL-3","tags":["ImmunoOncology","GeneExpression","Coverage","SmallRNA","Epigenetics","StatisticalMethod","Preprocessing","DifferentialExpression"],"description":"Differential expression of small RNA-seq when reference\nannotation is not given.","downloads_month":161,"downloads_total":2382,"page":"http://bioconductor.org/packages/release/bioc/html/srnadiff.html"},{"authors":"Richard Kennedy <rkennedy@ms.soph.uab.edu>, based on C++ code\nfrom Li Zhang <zhangli@odin.mdacc.tmc.edu> and Borland Delphi\ncode from Robnet Kerns <rtkerns@vcu.edu>.","name":"sscore","license":"GPL (>= 2)","tags":["DifferentialExpression"],"description":"This package contains an implementation of the S-Score\nalgorithm as described by Zhang et al (2002).","downloads_month":136,"downloads_total":43164,"page":"http://bioconductor.org/packages/release/bioc/html/sscore.html"},{"authors":"Yu Sun","name":"sscu","license":"GPL (>= 2)","tags":["Genetics","GeneExpression","WholeGenome"],"description":"The package calculates the indexes for selective stength\nin codon usage in bacteria species. (1) The package can\ncalculate the strength of selected codon usage bias (sscu, also\nnamed as s_index) based on Paul Sharp's method. The method take\ninto account of background mutation rate, and focus only on\nfour pairs of codons with universal translational advantages in\nall bacterial species. Thus the sscu index is comparable among\ndifferent species. (2) The package can detect the strength of\ntranslational accuracy selection by Akashi's test. The test\ntabulating all codons into four categories with the feature as\nconserved/variable amino acids and optimal/non-optimal codons.\n(3) Optimal codon lists (selected codons) can be calculated by\neither op_highly function (by using the highly expressed genes\ncompared with all genes to identify optimal codons), or\nop_corre_CodonW/op_corre_NCprime function (by correlative\nmethod developed by Hershberg & Petrov). Users will have a list\nof optimal codons for further analysis, such as input to the\nAkashi's test. (4) The detailed codon usage information, such\nas RSCU value, number of optimal codons in the highly/all gene\nset, as well as the genomic gc3 value, can be calculate by the\noptimal_codon_statistics and genomic_gc3 function. (5)\nFurthermore, we added one test function low_frequency_op in the\npackage. The function try to find the low frequency optimal\ncodons, among all the optimal codons identified by the\nop_highly function.","downloads_month":129,"downloads_total":7724,"page":"http://bioconductor.org/packages/release/bioc/html/sscu.html"},{"authors":"Danni Yu <dyu@purdue.edu> and Wolfgang Huber <whuber@embl.de> and\nOlga Vitek <ovitek@purdue.edu>","name":"sSeq","license":"GPL (>= 3)","tags":["ImmunoOncology","RNASeq"],"description":"The purpose of this package is to discover the genes that\nare differentially expressed between two conditions in RNA-seq\nexperiments. Gene expression is measured in counts of\ntranscripts and modeled with the Negative Binomial (NB)\ndistribution using a shrinkage approach for dispersion\nestimation. The method of moment (MM) estimates for dispersion\nare shrunk towards an estimated target, which minimizes the\naverage squared difference between the shrinkage estimates and\nthe initial estimates. The exact per-gene probability under the\nNB model is calculated, and used to test the hypothesis that\nthe expected expression of a gene in two conditions identically\nfollow a NB distribution.","downloads_month":375,"downloads_total":26830,"page":"http://bioconductor.org/packages/release/bioc/html/sSeq.html"},{"authors":"Gregory R. Warnes, Peng Liu, and Fasheng Li","name":"ssize","license":"LGPL","tags":["Microarray","DifferentialExpression"],"description":"Functions for computing and displaying sample size\ninformation for gene expression arrays.","downloads_month":160,"downloads_total":62620,"page":"http://bioconductor.org/packages/release/bioc/html/ssize.html"},{"authors":"Maarten van Iterson","name":"SSPA","license":"GPL (>= 2)","tags":["ImmunoOncology","GeneExpression","RNASeq","Microarray","StatisticalMethod"],"description":"General Sample size and power analysis for microarray and\nnext-generation sequencing data.","downloads_month":612,"downloads_total":60216,"page":"http://bioconductor.org/packages/release/bioc/html/SSPA.html"},{"authors":"Diana Low","name":"ssviz","license":"GPL-2","tags":["ImmunoOncology","Sequencing","RNASeq","Visualization","MultipleComparison","Genetics"],"description":"Small RNA sequencing viewer","downloads_month":184,"downloads_total":16040,"page":"http://bioconductor.org/packages/release/bioc/html/ssviz.html"},{"authors":"Koen Van den Berge and Lieven Clement","name":"stageR","license":"GNU General Public License version 3","tags":["Software","StatisticalMethod"],"description":"The stageR package allows automated stage-wise analysis of\nhigh-throughput gene expression data. The method is published\nin Genome Biology at\nhttps://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1277-0","downloads_month":171,"downloads_total":4748,"page":"http://bioconductor.org/packages/release/bioc/html/stageR.html"},{"authors":"Benedikt Zacher, Julia Ertl, Julien Gagneur and Achim Tresch","name":"STAN","license":"GPL (>= 2)","tags":["HiddenMarkovModel","GenomeAnnotation","Microarray","Sequencing","ChIPSeq","RNASeq","ChipOnChip","Transcription"],"description":"Genome segmentation with hidden Markov models has become a\nuseful tool to annotate genomic elements, such as promoters and\nenhancers. STAN (genomic STate ANnotation) implements\n(bidirectional) hidden Markov models (HMMs) using a variety of\ndifferent probability distributions, which can model a wide\nrange of current genomic data (e.g. continuous, discrete,\nbinary). STAN de novo learns and annotates the genome into a\ngiven number of 'genomic states'. The 'genomic states' may for\ninstance reflect distinct genome-associated protein complexes\n(e.g. 'transcription states') or describe recurring patterns of\nchromatin features (referred to as 'chromatin states'). Unlike\nother tools, STAN also allows for the integration of\nstrand-specific (e.g. RNA) and non-strand-specific data (e.g.\nChIP).","downloads_month":213,"downloads_total":18370,"page":"http://bioconductor.org/packages/release/bioc/html/STAN.html"},{"authors":"Juliane Siebourg and Niko Beerenwinkel","name":"staRank","license":"GPL","tags":["ImmunoOncology","MultipleComparison","CellBiology","CellBasedAssays","MicrotitrePlateAssay"],"description":"Detecting all relevant variables from a data set is\nchallenging, especially when only few samples are available and\ndata is noisy. Stability ranking provides improved variable\nrankings of increased robustness using resampling or\nsubsampling.","downloads_month":180,"downloads_total":26668,"page":"http://bioconductor.org/packages/release/bioc/html/staRank.html"},{"authors":"Claudia Cava and Isabella Castiglioni","name":"StarBioTrek","license":"GPL (>= 3)","tags":["GeneRegulation","Network","Pathways","KEGG"],"description":"This tool StarBioTrek presents some methodologies to\nmeasure pathway activity and cross-talk among pathways\nintegrating also the information of network data.","downloads_month":136,"downloads_total":7364,"page":"http://bioconductor.org/packages/release/bioc/html/StarBioTrek.html"},{"authors":"Benedikt Zacher, Johannes Soeding, Pei Fen Kuan, Matthias\nSiebert and Achim Tresch","name":"Starr","license":"Artistic-2.0","tags":["Microarray","OneChannel","DataImport","QualityControl","Preprocessing","ChIPchip"],"description":"Starr facilitates the analysis of ChIP-chip data, in\nparticular that of Affymetrix tiling arrays. The package\nprovides functions for data import, quality assessment, data\nvisualization and exploration. Furthermore, it includes\nhigh-level analysis features like association of ChIP signals\nwith annotated features, correlation analysis of ChIP signals\nand other genomic data (e.g. gene expression), peak-finding\nwith the CMARRT algorithm and comparative display of multiple\nclusters of ChIP-profiles. It uses the basic Bioconductor\nclasses ExpressionSet and probeAnno for maximum compatibility\nwith other software on Bioconductor. All functions from Starr\ncan be used to investigate preprocessed data from the Ringo\npackage, and vice versa. An important novel tool is the the\nautomated generation of correct, up-to-date microarray probe\nannotation (bpmap) files, which relies on an efficient mapping\nof short sequences (e.g. the probe sequences on a microarray)\nto an arbitrary genome.","downloads_month":154,"downloads_total":46246,"page":"http://bioconductor.org/packages/release/bioc/html/Starr.html"},{"authors":"STATegra Consortia","name":"STATegRa","license":"GPL-2","tags":["Software","StatisticalMethod","Clustering","DimensionReduction","PrincipalComponent"],"description":"Classes and tools for multi-omics data integration.","downloads_month":159,"downloads_total":16996,"page":"http://bioconductor.org/packages/release/bioc/html/STATegRa.html"},{"authors":"Hemi Luan","name":"statTarget","license":"LGPL (>= 3)","tags":["ImmunoOncology","Metabolomics","Proteomics","Machine Learning","Lipidomics","MassSpectrometry","QualityControl","Normalization","QC-RFSC","QC-RLSC","ComBat","DifferentialExpression","BatchEffect","Visualization","MultipleComparison","Preprocessing","GUI","Software"],"description":"A streamlined tool provides a graphical user interface for\nquality control based signal drift correction (QC-RFSC),\nintegration of data from multi-batch MS-based experiments, and\nthe comprehensive statistical analysis in metabolomics and\nproteomics.","downloads_month":280,"downloads_total":21456,"page":"http://bioconductor.org/packages/release/bioc/html/statTarget.html"},{"authors":"Yuanyuan Xiao <yxiao@itsa.ucsf.edu> and Yee Hwa (Jean) Yang\n<jean@biostat.ucsf.edu>","name":"stepNorm","license":"LGPL","tags":["Microarray","TwoChannel","Preprocessing"],"description":"Stepwise normalization functions for cDNA microarray data.","downloads_month":126,"downloads_total":43398,"page":"http://bioconductor.org/packages/release/bioc/html/stepNorm.html"},{"authors":"Thu-Hien To [aut, cre] and Steve Pederson [aut]","name":"strandCheckR","license":"GPL (>= 2)","tags":["RNASeq","Alignment","QualityControl","Coverage","ImmunoOncology"],"description":"This package aims to quantify and remove putative double\nstrand DNA from a strand-specific RNA sample. There are also\noptions and methods to plot the positive/negative proportions\nof all sliding windows, which allow users to have an idea of\nhow much the sample was contaminated and the appropriate\nthreshold to be used for filtering.","downloads_month":85,"downloads_total":738,"page":"http://bioconductor.org/packages/release/bioc/html/strandCheckR.html"},{"authors":"Martin Morgan and Nishant Gopalakrishnan","name":"Streamer","license":"Artistic-2.0","tags":["Infrastructure","DataImport"],"description":"Large data files can be difficult to work with in R, where\ndata generally resides in memory. This package encourages a\nstyle of programming where data is 'streamed' from disk into R\nvia a `producer' and through a series of `consumers' that,\ntypically reduce the original data to a manageable size. The\npackage provides useful Producer and Consumer stream components\nfor operations such as data input, sampling, indexing, and\ntransformation; see package?Streamer for details.","downloads_month":152,"downloads_total":32816,"page":"http://bioconductor.org/packages/release/bioc/html/Streamer.html"},{"authors":"Andrea Franceschini <andrea.franceschini@isb-sib.ch>","name":"STRINGdb","license":"GPL-2","tags":["Network"],"description":"The STRINGdb package provides a R interface to the STRING\nprotein-protein interactions database\n(http://www.string-db.org).","downloads_month":659,"downloads_total":60634,"page":"http://bioconductor.org/packages/release/bioc/html/STRINGdb.html"},{"authors":"Sadiq Saleh [aut, cre] and Michael Hallett [aut]","name":"STROMA4","license":"GPL-3","tags":["ImmunoOncology","GeneExpression","BiomedicalInformatics","Classification","Microarray","RNASeq","Software"],"description":"This package estimates four stromal properties identified\nin TNBC patients in each patient of a gene expression datasets.\nThese stromal property assignments can be combined to subtype\npatients. These four stromal properties were identified in\nTriple negative breast cancer (TNBC) patients and represent the\npresence of different cells in the stroma: T-cells (T), B-cells\n(B), stromal infiltrating epithelial cells (E), and desmoplasia\n(D). Additionally this package can also be used to estimate\ngenerative properties for the Lehmann subtypes, an alternative\nTNBC subtyping scheme (PMID: 21633166).","downloads_month":172,"downloads_total":5008,"page":"http://bioconductor.org/packages/release/bioc/html/STROMA4.html"},{"authors":"David Robinson, John D. Storey and with contributions from Andrew\nJ. Bass","name":"subSeq","license":"MIT + file LICENSE","tags":["ImmunoOncology","Sequencing","Transcription","RNASeq","GeneExpression","DifferentialExpression"],"description":"Subsampling of high throughput sequencing count data for\nuse in experiment design and analysis.","downloads_month":221,"downloads_total":12200,"page":"http://bioconductor.org/packages/release/bioc/html/subSeq.html"},{"authors":"Alejandro Reyes <alejandro.reyes.ds@gmail.com> and Patrick Kimes\n<patrick.kimes@gmail.com>","name":"SummarizedBenchmark","license":"GPL (>= 3)","tags":["Software","Infrastructure"],"description":"This package defines the BenchDesign and\nSummarizedBenchmark classes for building, executing, and\nevaluating benchmark experiments of computational methods. The\nSummarizedBenchmark class extends the\nRangedSummarizedExperiment object, and is designed to provide\ninfrastructure to store and compare the results of applying\ndifferent methods to a shared data set. This class provides an\nintegrated interface to store metadata such as method\nparameters and software versions as well as ground truths (when\nthese are available) and evaluation metrics.","downloads_month":181,"downloads_total":2516,"page":"http://bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html"},{"authors":"Martin Morgan, Valerie Obenchain, Jim Hester and Hervé Pagès","name":"SummarizedExperiment","license":"Artistic-2.0","tags":["Genetics","Infrastructure","Sequencing","Annotation","Coverage","GenomeAnnotation"],"description":"The SummarizedExperiment container contains one or more\nassays, each represented by a matrix-like object of numeric or\nother mode. The rows typically represent genomic ranges of\ninterest and the columns represent samples.","downloads_month":31803,"downloads_total":1705436,"page":"http://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html"},{"authors":"Hai Fang and Julian Gough","name":"supraHex","license":"GPL-2","tags":["Software","Clustering","Visualization","GeneExpression"],"description":"A supra-hexagonal map is a giant hexagon on a\n2-dimensional grid seamlessly consisting of smaller hexagons.\nIt is supposed to train, analyse and visualise a\nhigh-dimensional omics input data. The supraHex is able to\ncarry out gene clustering/meta-clustering and sample\ncorrelation, plus intuitive visualisations to facilitate\nexploratory analysis. More importantly, it allows for\noverlaying additional data onto the trained map to explore\nrelations between input and additional data. So with supraHex,\nit is also possible to carry out multilayer omics data\ncomparisons. Newly added utilities are advanced heatmap\nvisualisation and tree-based analysis of sample relationships.\nUniquely to this package, users can ultrafastly understand any\ntabular omics data, both scientifically and artistically,\nespecially in a sample-specific fashion but without loss of\ninformation on large genes.","downloads_month":415,"downloads_total":71674,"page":"http://bioconductor.org/packages/release/bioc/html/supraHex.html"},{"authors":"Benjamin Haibe-Kains, Markus Schroeder, Catharina Olsen, Christos Sotiriou, Gianluca Bontempi, John Quackenbush, Samuel\nBranders and Zhaleh Safikhani","name":"survcomp","license":"Artistic-2.0","tags":["GeneExpression","DifferentialExpression","Visualization"],"description":"Assessment and Comparison for Performance of Risk\nPrediction (Survival) Models.","downloads_month":1067,"downloads_total":115088,"page":"http://bioconductor.org/packages/release/bioc/html/survcomp.html"},{"authors":"Douglas H Phanstiel <dphansti@stanford.edu>","name":"Sushi","license":"GPL (>= 2)","tags":["DataRepresentation","Visualization","Genetics","Sequencing","Infrastructure","HiC"],"description":"Flexible, quantitative, and integrative genomic\nvisualizations for publication-quality multi-panel figures","downloads_month":492,"downloads_total":42054,"page":"http://bioconductor.org/packages/release/bioc/html/Sushi.html"},{"authors":"Jeffrey T. Leek <jtleek@gmail.com>, W. Evan Johnson\n<wej@bu.edu>, Hilary S. Parker <hiparker@jhsph.edu>, Elana J.\nFertig <ejfertig@jhmi.edu>, Andrew E. Jaffe <ajaffe@jhsph.edu>, John D. Storey <jstorey@princeton.edu>, Yuqing Zhang\n<zhangyuqing.pkusms@gmail.com> and Leonardo Collado Torres\n<lcollado@jhu.edu>","name":"sva","license":"Artistic-2.0","tags":["ImmunoOncology","Microarray","StatisticalMethod","Preprocessing","MultipleComparison","Sequencing","RNASeq","BatchEffect","Normalization"],"description":"The sva package contains functions for removing batch\neffects and other unwanted variation in high-throughput\nexperiment. Specifically, the sva package contains functions\nfor the identifying and building surrogate variables for\nhigh-dimensional data sets. Surrogate variables are covariates\nconstructed directly from high-dimensional data (like gene\nexpression/RNA sequencing/methylation/brain imaging data) that\ncan be used in subsequent analyses to adjust for unknown,\nunmodeled, or latent sources of noise. The sva package can be\nused to remove artifacts in three ways: (1) identifying and\nestimating surrogate variables for unknown sources of variation\nin high-throughput experiments (Leek and Storey 2007 PLoS\nGenetics,2008 PNAS), (2) directly removing known batch effects\nusing ComBat (Johnson et al. 2007 Biostatistics) and (3)\nremoving batch effects with known control probes (Leek 2014\nbiorXiv). Removing batch effects and using surrogate variables\nin differential expression analysis have been shown to reduce\ndependence, stabilize error rate estimates, and improve\nreproducibility, see (Leek and Storey 2007 PLoS Genetics, 2008\nPNAS or Leek et al. 2011 Nat. Reviews Genetics).","downloads_month":9010,"downloads_total":478434,"page":"http://bioconductor.org/packages/release/bioc/html/sva.html"},{"authors":"Sutirtha Chakraborty","name":"SVAPLSseq","license":"GPL-3","tags":["ImmunoOncology","GeneExpression","RNASeq","Normalization","BatchEffect"],"description":"The package contains functions that are intended for\nextracting the signatures of latent variation in RNAseq data\nand using them to perform an improved differential expression\nanalysis for a set of features (genes, transcripts) between two\nspecified biological groups.","downloads_month":179,"downloads_total":6874,"page":"http://bioconductor.org/packages/release/bioc/html/SVAPLSseq.html"},{"authors":"Guidantonio Malagoli Tagliazucchi <malagoli@ingm.org>","name":"SVM2CRM","license":"GPL-3","tags":["ChIPSeq","SupportVectorMachine","Software","Preprocessing","ChipOnChip"],"description":"Detection of cis-regulatory elements using svm implemented\nin LiblineaR.","downloads_month":125,"downloads_total":11914,"page":"http://bioconductor.org/packages/release/bioc/html/SVM2CRM.html"},{"authors":"Peter Blattmann, Moritz Heusel and Ruedi Aebersold","name":"SWATH2stats","license":"GPL-3","tags":["Proteomics","Annotation","ExperimentalDesign","Preprocessing","MassSpectrometry","ImmunoOncology"],"description":"This package is intended to transform SWATH data from the\nOpenSWATH software into a format readable by other statistics\npackages while performing filtering, annotation and FDR\nestimation.","downloads_month":181,"downloads_total":13044,"page":"http://bioconductor.org/packages/release/bioc/html/SWATH2stats.html"},{"authors":"J WU and D Pascovici","name":"SwathXtend","license":"GPL-2","tags":["Software"],"description":"Contains utility functions for integrating spectral\nlibraries for SWATH and statistical data analysis for SWATH\ngenerated data.","downloads_month":125,"downloads_total":7810,"page":"http://bioconductor.org/packages/release/bioc/html/SwathXtend.html"},{"authors":"Jeffrey T. Leek, Leah Jager and Simina M. Boca","name":"swfdr","license":"GPL (>= 3)","tags":["MultipleComparison","StatisticalMethod","Software"],"description":"This package allows users to estimate the science-wise\nfalse discovery rate from Jager and Leek, \"Empirical estimates\nsuggest most published medical research is true,\" 2013,\nBiostatistics, using an EM approach due to the presence of\nrounding and censoring. It also allows users to estimate the\nproportion of true null hypotheses in the presence of\ncovariates, using a regression framework, as per Boca and Leek,\n\"A regression framework for the proportion of true null\nhypotheses,\" 2015, bioRxiv preprint.","downloads_month":136,"downloads_total":4724,"page":"http://bioconductor.org/packages/release/bioc/html/swfdr.html"},{"authors":"Jing Wang <jing.wang.2@vanderbilt.edu>, Andrew Hardaway\n<hardawayja@gmail.com> and Bing Zhang\n<bing.zhang@vanderbilt.edu>","name":"SwimR","license":"LGPL-2","tags":["Visualization"],"description":"SwimR is an R-based suite that calculates, analyses, and\nplots the frequency of C. elegans swimming behavior over time.\nIt places a particular emphasis on identifying paralysis and\nquantifying the kinetic elements of paralysis during swimming.\nData is input to SwipR from a custom built program that fits a\n5 point morphometric spine to videos of single worms swimming\nin a buffer called Worm Tracker.","downloads_month":132,"downloads_total":19866,"page":"http://bioconductor.org/packages/release/bioc/html/SwimR.html"},{"authors":"Bahman Afsari <bahman@jhu.edu>, Luigi Marchionni\n<marchion@jhu.edu> and Wikum Dinalankara <wdinala1@jhmi.edu>","name":"switchBox","license":"GPL-2","tags":["Software","StatisticalMethod","Classification"],"description":"The package offer different classifiers based on\ncomparisons of pair of features (TSP), using various decision\nrules (e.g., majority wins principle).","downloads_month":134,"downloads_total":16818,"page":"http://bioconductor.org/packages/release/bioc/html/switchBox.html"},{"authors":"Kieran Campbell [aut and cre]","name":"switchde","license":"GPL (>= 2)","tags":["ImmunoOncology","Software","Transcriptomics","GeneExpression","RNASeq","Regression","DifferentialExpression","SingleCell"],"description":"Inference and detection of switch-like differential\nexpression across single-cell RNA-seq trajectories.","downloads_month":219,"downloads_total":7602,"page":"http://bioconductor.org/packages/release/bioc/html/switchde.html"},{"authors":"Laurent Gatto, Nick J. Bond, Pavel V. Shliaha and Sebastian\nGibb.","name":"synapter","license":"GPL-2","tags":["ImmunoOncology","MassSpectrometry","Proteomics","QualityControl"],"description":"The synapter package provides functionality to reanalyse\nlabel-free proteomics data acquired on a Synapt G2 mass\nspectrometer. One or several runs, possibly processed with\nadditional ion mobility separation to increase identification\naccuracy can be combined to other quantitation files to\nmaximise identification and quantitation accuracy.","downloads_month":361,"downloads_total":39684,"page":"http://bioconductor.org/packages/release/bioc/html/synapter.html"},{"authors":"Liye He <liye.he@helsinki.fi> and Jing Tang\n<jing.tang@helsinki.fi>","name":"synergyfinder","license":"Mozilla Public License 2.0 + file LICENSE","tags":["Software","Statistical Method"],"description":"Efficient implementations for all the popular synergy\nscoring models for drug combinations, including HSA, Loewe,\nBliss and ZIP and visualization of the synergy scores as either\na two-dimensional or a three-dimensional interaction surface\nover the dose matrix.","downloads_month":239,"downloads_total":10824,"page":"http://bioconductor.org/packages/release/bioc/html/synergyfinder.html"},{"authors":"Chunxuan Shao <c.shao@dkfz.de>","name":"synlet","license":"GPL-3","tags":["ImmunoOncology","CellBasedAssays","QualityControl","Preprocessing","Visualization","FeatureExtraction"],"description":"Select hits from synthetic lethal RNAi screen data. For\nexample, there are two identical celllines except one gene is\nknocked-down in one cellline. The interest is to find genes\nthat lead to stronger lethal effect when they are knocked-down\nfurther by siRNA. Quality control and various visualisation\ntools are implemented. Four different algorithms could be used\nto pick up the interesting hits. This package is designed based\non 384 wells plates, but may apply to other platforms with\nproper configuration.","downloads_month":176,"downloads_total":10038,"page":"http://bioconductor.org/packages/release/bioc/html/synlet.html"},{"authors":"Thomas Girke","name":"systemPipeR","license":"Artistic-2.0","tags":["Genetics","Infrastructure","DataImport","Sequencing","RNASeq","RiboSeq","ChIPSeq","MethylSeq","SNP","GeneExpression","Coverage","GeneSetEnrichment","Alignment","QualityControl","ImmunoOncology"],"description":"R package for building and running automated end-to-end\nanalysis workflows for a wide range of next generation sequence\n(NGS) applications such as RNA-Seq, ChIP-Seq, VAR-Seq and\nRibo-Seq. Important features include a uniform workflow\ninterface across different NGS applications, automated report\ngeneration, and support for running both R and command-line\nsoftware, such as NGS aligners or peak/variant callers, on\nlocal computers or compute clusters. Efficient handling of\ncomplex sample sets and experimental designs is facilitated by\na consistently implemented sample annotation infrastructure.\nInstructions for using systemPipeR are given in the Overview\nVignette (HTML). The remaining Vignettes, linked below, are\nworkflow templates for common NGS use cases.","downloads_month":1833,"downloads_total":110642,"page":"http://bioconductor.org/packages/release/bioc/html/systemPipeR.html"},{"authors":"Yue Li","name":"TargetScore","license":"GPL-2","tags":["miRNA"],"description":"Infer the posterior distributions of microRNA targets by\nprobabilistically modelling the likelihood\nmicroRNA-overexpression fold-changes and sequence-based scores.\nVariaitonal Bayesian Gaussian mixture model (VB-GMM) is applied\nto log fold-changes and sequence scores to obtain the\nposteriors of latent variable being the miRNA targets. The\nfinal targetScore is computed as the sigmoid-transformed\nfold-change weighted by the averaged posteriors of target\ncomponents over all of the features.","downloads_month":176,"downloads_total":23360,"page":"http://bioconductor.org/packages/release/bioc/html/TargetScore.html"},{"authors":"Alvaro Cuadros-Inostroza <acuadros+bioc@gmail.com>, Jan Lisec, Henning Redestig and Matt Hannah","name":"TargetSearch","license":"GPL (>= 2)","tags":["MassSpectrometry","Preprocessing","DecisionTree","ImmunoOncology"],"description":"This packages provides a targeted pre-processing method\nfor GC-MS data.","downloads_month":176,"downloads_total":46482,"page":"http://bioconductor.org/packages/release/bioc/html/TargetSearch.html"},{"authors":"Gabriela A. Merino, Cristobal Fresno, Yanina Murua, Andrea S.\nLlera and Elmer A. Fernandez","name":"TarSeqQC","license":"GPL (>=2)","tags":["Software","Sequencing","TargetedResequencing","QualityControl","Visualization","Coverage","Alignment","DataImport"],"description":"The package allows the representation of targeted\nexperiment in R. This is based on current packages and\nincorporates functions to do a quality control over this kind\nof experiments and a fast exploration of the sequenced regions.\nAn xlsx file is generated as output.","downloads_month":150,"downloads_total":10860,"page":"http://bioconductor.org/packages/release/bioc/html/TarSeqQC.html"},{"authors":"Jianqiang Sun, Tomoaki Nishiyama, Kentaro Shimizu, and Koji\nKadota","name":"TCC","license":"GPL-2","tags":["ImmunoOncology","Sequencing","DifferentialExpression","RNASeq"],"description":"This package provides a series of functions for performing\ndifferential expression analysis from RNA-seq count data using\nrobust normalization strategy (called DEGES). The basic idea of\nDEGES is that potential differentially expressed genes or\ntranscripts (DEGs) among compared samples should be removed\nbefore data normalization to obtain a well-ranked gene list\nwhere true DEGs are top-ranked and non-DEGs are bottom ranked.\nThis can be done by performing a multi-step normalization\nstrategy (called DEGES for DEG elimination strategy). A major\ncharacteristic of TCC is to provide the robust normalization\nmethods for several kinds of count data (two-group with or\nwithout replicates, multi-group/multi-factor, and so on) by\nvirtue of the use of combinations of functions in depended\npackages.","downloads_month":437,"downloads_total":48636,"page":"http://bioconductor.org/packages/release/bioc/html/TCC.html"},{"authors":"Antonio Colaprico, Tiago Chedraoui Silva, Catharina Olsen, Luciano Garofano, Davide Garolini, Claudia Cava, Thais Sabedot, Tathiane Malta, Stefano M. Pagnotta, Isabella Castiglioni, Michele Ceccarelli, Gianluca Bontempi and Houtan Noushmehr","name":"TCGAbiolinks","license":"GPL (>= 3)","tags":["DNAMethylation","DifferentialMethylation","GeneRegulation","GeneExpression","MethylationArray","DifferentialExpression","Pathways","Network","Sequencing","Survival"],"description":"The aim of TCGAbiolinks is : i) facilitate the GDC\nopen-access data retrieval, ii) prepare the data using the\nappropriate pre-processing strategies, iii) provide the means\nto carry out different standard analyses and iv) to easily\nreproduce earlier research results. In more detail, the package\nprovides multiple methods for analysis (e.g., differential\nexpression analysis, identifying differentially methylated\nregions) and methods for visualization (e.g., survival plots,\nvolcano plots, starburst plots) in order to easily develop\ncomplete analysis pipelines.","downloads_month":2940,"downloads_total":154744,"page":"http://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html"},{"authors":"Tiago Chedraoui Silva <tiagochst@gmail.com>, Antonio Colaprico\n<antonio.colaprico@ulb.ac.be>, Catharina Olsen\n<colsen@ulb.ac.be>, Michele Ceccarelli, Gianluca Bontempi\n<gbonte@ulb.ac.be>, Benjamin P. Berman\n<Benjamin.Berman@cshs.org> and Houtan Noushmehr\n<houtana@gmail.com>","name":"TCGAbiolinksGUI","license":"GPL (>= 3)","tags":["Genetics","GUI","DNAMethylation","StatisticalMethod","DifferentialMethylation","GeneRegulation","GeneExpression","MethylationArray","DifferentialExpression","Sequencing","Pathways","Network","DNASeq"],"description":"\"TCGAbiolinksGUI: A Graphical User Interface to analyze\ncancer molecular and clinical data. A demo version of GUI is\nfound in https://tcgabiolinksgui.shinyapps.io/tcgabiolinks/\"","downloads_month":312,"downloads_total":13676,"page":"http://bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html"},{"authors":"Marcel Ramos [aut, cre], Lucas Schiffer [aut], Sean Davis\n[ctb] and Levi Waldron [aut]","name":"TCGAutils","license":"Artistic-2.0","tags":["Software","WorkflowStep","Preprocessing"],"description":"A suite of helper functions for checking and manipulating\nTCGA data including data obtained from the curatedTCGAData\nexperiment package. These functions aim to simplify and make\nworking with TCGA data more manageable.","downloads_month":574,"downloads_total":7458,"page":"http://bioconductor.org/packages/release/bioc/html/TCGAutils.html"},{"authors":"Mengjun Wu <minervajunjun@gmail.com> and Lei Gu\n<leigu@broadinstitute.org>","name":"TCseq","license":"GPL (>= 2)","tags":["ImmunoOncology","Epigenetics","TimeCourse","Sequencing","ChIPSeq","RNASeq","DifferentialExpression","Clustering","Visualization"],"description":"Quantitative and differential analysis of epigenomic and\ntranscriptomic time course sequencing data, clustering analysis\nand visualization of temporal patterns of time course data.","downloads_month":232,"downloads_total":7268,"page":"http://bioconductor.org/packages/release/bioc/html/TCseq.html"},{"authors":"Zoppoli P., Morganella S. and Ceccarelli M.","name":"TDARACNE","license":"GPL-2","tags":["Microarray","TimeCourse"],"description":"To infer gene networks from time-series measurements is a\ncurrent challenge into bioinformatics research area. In order\nto detect dependencies between genes at different time delays,\nwe propose an approach to infer gene regulatory networks from\ntime-series measurements starting from a well known algorithm\nbased on information theory. The proposed algorithm is expected\nto be useful in reconstruction of small biological directed\nnetworks from time course data.","downloads_month":149,"downloads_total":35064,"page":"http://bioconductor.org/packages/release/bioc/html/TDARACNE.html"},{"authors":"Vince Carey","name":"tenXplore","license":"Artistic-2.0","tags":["ImmunoOncology","DimensionReduction","PrincipalComponent","Transcriptomics","SingleCell"],"description":"Perform ontological exploration of scRNA-seq of 1.3\nmillion mouse neurons from 10x genomics.","downloads_month":159,"downloads_total":3916,"page":"http://bioconductor.org/packages/release/bioc/html/tenXplore.html"},{"authors":"M. Hummel, S. Bonnin, E. Lowy and G. Roma","name":"TEQC","license":"GPL (>= 2)","tags":["QualityControl","Microarray","Sequencing","Genetics"],"description":"Target capture experiments combine hybridization-based (in\nsolution or on microarrays) capture and enrichment of genomic\nregions of interest (e.g. the exome) with high throughput\nsequencing of the captured DNA fragments. This package provides\nfunctionalities for assessing and visualizing the quality of\nthe target enrichment process, like specificity and sensitivity\nof the capture, per-target read coverage and so on.","downloads_month":186,"downloads_total":40730,"page":"http://bioconductor.org/packages/release/bioc/html/TEQC.html"},{"authors":"Matthew N. McCall <mccallm@gmail.com>, Anthony Almudevar\n<Anthony_Almudevar@urmc.rochester.edu>, David Burton\n<David_Burton@urmc.rochester.edu> and Harry Stern\n<harry.stern@rochester.edu>","name":"ternarynet","license":"GPL (>= 2)","tags":["Software","CellBiology","GraphAndNetwork"],"description":"A computational Bayesian approach to ternary gene\nregulatory network estimation from gene perturbation\nexperiments.","downloads_month":123,"downloads_total":27434,"page":"http://bioconductor.org/packages/release/bioc/html/ternarynet.html"},{"authors":"Liuba Nausicaa Martino, Alice Parodi, Gaia Ceddia, Piercesare\nSecchi, Stefano Campaner and Marco Masseroli","name":"TFARM","license":"Artistic-2.0","tags":["BiologicalQuestion","Infrastructure","StatisticalMethod","Transcription"],"description":"It searches for relevant associations of transcription\nfactors with a transcription factor target, in specific genomic\nregions. It also allows to evaluate the Importance Index\ndistribution of transcription factors (and combinations of\ntranscription factors) in association rules.","downloads_month":129,"downloads_total":3190,"page":"http://bioconductor.org/packages/release/bioc/html/TFARM.html"},{"authors":"Ge Tan [aut and cre]","name":"TFBSTools","license":"GPL-2","tags":["MotifAnnotation","GeneRegulation","MotifDiscovery","Transcription","Alignment"],"description":"TFBSTools is a package for the analysis and manipulation\nof transcription factor binding sites. It includes matrices\nconversion between Position Frequency Matirx (PFM), Position\nWeight Matirx (PWM) and Information Content Matrix (ICM). It\ncan also scan putative TFBS from sequence/alignment, query\nJASPAR database and provides a wrapper of de novo motif\ndiscovery software.","downloads_month":2312,"downloads_total":223226,"page":"http://bioconductor.org/packages/release/bioc/html/TFBSTools.html"},{"authors":"Laura Puente Santamaría and Luis del Peso","name":"TFEA.ChIP","license":"Artistic-2.0","tags":["Transcription","GeneRegulation","GeneSetEnrichment","Transcriptomics","Sequencing","ChIPSeq","RNASeq","ImmunoOncology"],"description":"Package to analize transcription factor enrichment in a\ngene set using data from ChIP-Seq experiments.","downloads_month":176,"downloads_total":2470,"page":"http://bioconductor.org/packages/release/bioc/html/TFEA.ChIP.html"},{"authors":"Alberto Marchesi and Marco Masseroli","name":"TFHAZ","license":"Artistic-2.0","tags":["Software","BiologicalQuestion","Transcription","ChIPSeq","Coverage"],"description":"It finds trascription factor (TF) high accumulation DNA\nzones, i.e., regions along the genome where there is a high\npresence of different transcription factors. Starting from a\ndataset containing the genomic positions of TF binding regions,\nfor each base of the selected chromosome the accumulation of\nTFs is computed. Three different types of accumulation (TF,\nregion and base accumulation) are available, together with the\npossibility of considering, in the single base accumulation\ncomputing, the TFs present not only in that single base, but\nalso in its neighborhood, within a window of a given width. Two\ndifferent methods for the search of TF high accumulation DNA\nzones, called \"binding regions\" and \"overlaps\", are available.\nIn addition, some functions are provided in order to analyze,\nvisualize and compare results obtained with different input\nparameters.","downloads_month":126,"downloads_total":3090,"page":"http://bioconductor.org/packages/release/bioc/html/TFHAZ.html"},{"authors":"Vincent Carey [aut], Shweta Gopaulakrishnan [cre and aut]","name":"TFutils","license":"Artistic-2.0","tags":["Transcriptomics"],"description":"Package to work with TF data.","downloads_month":126,"downloads_total":2380,"page":"http://bioconductor.org/packages/release/bioc/html/TFutils.html"},{"authors":"Antti Honkela, Pei Gao, Jonatan Ropponen, Miika-Petteri\nMatikainen, Magnus Rattray and Neil D. Lawrence","name":"tigre","license":"AGPL-3","tags":["Microarray","TimeCourse","GeneExpression","Transcription","GeneRegulation","NetworkInference","Bayesian"],"description":"The tigre package implements our methodology of Gaussian\nprocess differential equation models for analysis of gene\nexpression time series from single input motif networks. The\npackage can be used for inferring unobserved transcription\nfactor (TF) protein concentrations from expression measurements\nof known target genes, or for ranking candidate targets of a\nTF.","downloads_month":170,"downloads_total":39296,"page":"http://bioconductor.org/packages/release/bioc/html/tigre.html"},{"authors":"Wolfgang Huber, Zhenyu Xu and Joern Toedling with contributions\nfrom Matt Ritchie","name":"tilingArray","license":"Artistic-2.0","tags":["Microarray","OneChannel","Preprocessing","Visualization"],"description":"The package provides functionality that can be useful for\nthe analysis of high-density tiling microarray data (such as\nfrom Affymetrix genechips) for measuring transcript abundance\nand architecture. The main functionalities of the package are:\n1. the class 'segmentation' for representing partitionings of a\nlinear series of data; 2. the function 'segment' for fitting\npiecewise constant models using a dynamic programming algorithm\nthat is both fast and exact; 3. the function 'confint' for\ncalculating confidence intervals using the strucchange package;\n4. the function 'plotAlongChrom' for generating pretty plots;\n5. the function 'normalizeByReference' for probe-sequence\ndependent response adjustment from a (set of) reference\nhybridizations.","downloads_month":225,"downloads_total":74304,"page":"http://bioconductor.org/packages/release/bioc/html/tilingArray.html"},{"authors":"Yu Chuan Tai","name":"timecourse","license":"LGPL","tags":["Microarray","TimeCourse","DifferentialExpression"],"description":"Functions for data analysis and graphical displays for\ndevelopmental microarray time course data.","downloads_month":167,"downloads_total":76042,"page":"http://bioconductor.org/packages/release/bioc/html/timecourse.html"},{"authors":"Maia Smith [aut and cre]","name":"timescape","license":"GPL-3","tags":["Visualization","BiomedicalInformatics"],"description":"TimeScape is an automated tool for navigating temporal\nclonal evolution data. The key attributes of this\nimplementation involve the enumeration of clones, their\nevolutionary relationships and their shifting dynamics over\ntime. TimeScape requires two inputs: (i) the clonal phylogeny\nand (ii) the clonal prevalences. Optionally, TimeScape accepts\na data table of targeted mutations observed in each clone and\ntheir allele prevalences over time. The output is the TimeScape\nplot showing clonal prevalence vertically, time horizontally,\nand the plot height optionally encoding tumour volume during\ntumour-shrinking events. At each sampling time point (denoted\nby a faint white line), the height of each clone accurately\nreflects its proportionate prevalence. These prevalences form\nthe anchors for bezier curves that visually represent the\ndynamic transitions between time points.","downloads_month":140,"downloads_total":5758,"page":"http://bioconductor.org/packages/release/bioc/html/timescape.html"},{"authors":"Lan Huong Nguyen","name":"TimeSeriesExperiment","license":"LGPL (>= 3)","tags":["TimeCourse","Sequencing","RNASeq","Microbiome","GeneExpression","ImmunoOncology","Transcription","Normalization","DifferentialExpression","PrincipalComponent","Clustering","Visualization","Pathways"],"description":"Visualization and analysis toolbox for short time course\ndata which includes dimensionality reduction, clustering,\ntwo-sample differential expression testing and gene ranking\ntechniques. The package also provides methods for retrieving\nenriched pathways.","downloads_month":122,"downloads_total":1070,"page":"http://bioconductor.org/packages/release/bioc/html/TimeSeriesExperiment.html"},{"authors":"Bjarne Johannessen, Anita Sveen and Rolf I. Skotheim","name":"TIN","license":"Artistic-2.0","tags":["ExonArray","Microarray","GeneExpression","AlternativeSplicing","Genetics","DifferentialSplicing"],"description":"The TIN package implements a set of tools for\ntranscriptome instability analysis based on exon expression\nprofiles. Deviating exon usage is studied in the context of\nsplicing factors to analyse to what degree transcriptome\ninstability is correlated to splicing factor expression. In the\ntranscriptome instability correlation analysis, the data is\ncompared to both random permutations of alternative splicing\nscores and expression of random gene sets.","downloads_month":126,"downloads_total":12520,"page":"http://bioconductor.org/packages/release/bioc/html/TIN.html"},{"authors":"Ashish Jain [aut, cre] and Geetu Tuteja [aut]","name":"TissueEnrich","license":"MIT + file LICENSE","tags":["GeneSetEnrichment","GeneExpression","Sequencing"],"description":"The TissueEnrich package is used to calculate enrichment\nof tissue-specific genes in a set of input genes. For example,\nthe user can input the most highly expressed genes from RNA-Seq\ndata, or gene co-expression modules to determine which\ntissue-specific genes are enriched in those datasets.\nTissue-specific genes were defined by processing RNA-Seq data\nfrom the Human Protein Atlas (HPA) (Uhlén et al. 2015), GTEx\n(Ardlie et al. 2015), and mouse ENCODE (Shen et al. 2012) using\nthe algorithm from the HPA (Uhlén et al. 2015).The\nhypergeometric test is being used to determine if the\ntissue-specific genes are enriched among the input genes. Along\nwith tissue-specific gene enrichment, the TissueEnrich package\ncan also be used to define tissue-specific genes from\nexpression datasets provided by the user, which can then be\nused to calculate tissue-specific gene enrichments.","downloads_month":164,"downloads_total":2718,"page":"http://bioconductor.org/packages/release/bioc/html/TissueEnrich.html"},{"authors":"Gavin Ha","name":"TitanCNA","license":"GPL-3","tags":["Sequencing","WholeGenome","DNASeq","ExomeSeq","StatisticalMethod","CopyNumberVariation","HiddenMarkovModel","Genetics","GenomicVariation","ImmunoOncology"],"description":"Hidden Markov model to segment and predict regions of\nsubclonal copy number alterations (CNA) and loss of\nheterozygosity (LOH), and estimate cellular prevalence of\nclonal clusters in tumour whole genome sequencing data.","downloads_month":274,"downloads_total":24584,"page":"http://bioconductor.org/packages/release/bioc/html/TitanCNA.html"},{"authors":"J. Zhang <jzhang@jimmy.harvard.edu>","name":"tkWidgets","license":"Artistic-2.0","tags":["Infrastructure"],"description":"Widgets to provide user interfaces. tcltk should have been\ninstalled for the widgets to run.","downloads_month":888,"downloads_total":251390,"page":"http://bioconductor.org/packages/release/bioc/html/tkWidgets.html"},{"authors":"Monica Golumbeanu <golumbeanu.monica@gmail.com>","name":"TMixClust","license":"GPL (>=2)","tags":["Software","StatisticalMethod","Clustering","TimeCourse","GeneExpression"],"description":"Implementation of a clustering method for time series gene\nexpression data based on mixed-effects models with Gaussian\nvariables and non-parametric cubic splines estimation. The\nmethod can robustly account for the high levels of noise\npresent in typical gene expression time series datasets.","downloads_month":155,"downloads_total":3964,"page":"http://bioconductor.org/packages/release/bioc/html/TMixClust.html"},{"authors":"Jialin Ma [cre, aut], Miguel Pignatelli [aut] and Toby Hocking\n[aut]","name":"TnT","license":"AGPL-3","tags":["Infrastructure","Visualization"],"description":"A R interface to the TnT javascript library\n(https://github.com/ tntvis) to provide interactive and\nflexible visualization of track-based genomic data.","downloads_month":139,"downloads_total":3500,"page":"http://bioconductor.org/packages/release/bioc/html/TnT.html"},{"authors":"Lorenz Gerber and Viet Mai Hoang","name":"tofsims","license":"GPL-3","tags":["ImmunoOncology","Infrastructure","DataImport","MassSpectrometry","ImagingMassSpectrometry","Proteomics","Metabolomics"],"description":"This packages offers a pipeline for import, processing and\nanalysis of ToF-SIMS 2D image data. Import of Iontof and\nUlvac-Phi raw or preprocessed data is supported. For rawdata,\nmass calibration, peak picking and peak integration exist.\nGeneral funcionality includes data binning, scaling, image\nsubsetting and visualization. A range of multivariate tools\ncommon in the ToF-SIMS community are implemented (PCA, MCR,\nMAF, MNF). An interface to the bioconductor image processing\npackage EBImage offers image segmentation functionality.","downloads_month":196,"downloads_total":9298,"page":"http://bioconductor.org/packages/release/bioc/html/tofsims.html"},{"authors":"Ivana Ihnatova, Eva Budinska and Ludwig Geistlinger","name":"ToPASeq","license":"AGPL-3","tags":["ImmunoOncology","GeneExpression","RNASeq","DifferentialExpression","GraphAndNetwork","Pathways","NetworkEnrichment","Visualization"],"description":"Implementation of methods for topology-based pathway\nanalysis of RNA-seq data. This includes Topological Analysis of\nPathway Phenotype Association (TAPPA; Gao and Wang, 2007),\nPathWay Enrichment Analysis (PWEA; Hung et al., 2010), and the\nPathway Regulation Score (PRS; Ibrahim et al., 2012).","downloads_month":206,"downloads_total":17498,"page":"http://bioconductor.org/packages/release/bioc/html/ToPASeq.html"},{"authors":"Paul Harrison [aut and cre]\n(<https://orcid.org/0000-0002-3980-268X>)","name":"topconfects","license":"LGPL-2.1 | file LICENSE","tags":["GeneExpression","DifferentialExpression","Transcriptomics","RNASeq","mRNAMicroarray","Regression","MultipleComparison"],"description":"Find the largest confident effect sizes from a large\ncollection, while maintaining False Discovery Rate and False\nCoverage-statement Rate control. The main application is\ndifferential gene expression analysis, providing genes ranked\nin order of confident log2 fold change.","downloads_month":0,"downloads_total":40,"page":"http://bioconductor.org/packages/release/bioc/html/topconfects.html"},{"authors":"Sebastian Gibb [aut, cre]\n(<https://orcid.org/0000-0001-7406-4443>), Pavel Shliaha [aut]\n(<https://orcid.org/0000-0003-3092-0724>) and Ole Nørregaard\nJensen [aut] (<https://orcid.org/0000-0003-1862-8528>)","name":"topdownr","license":"GPL (>= 3)","tags":["ImmunoOncology","Infrastructure","Proteomics","MassSpectrometry","Coverage"],"description":"The topdownr package allows automatic and systemic\ninvestigation of fragment conditions. It creates Thermo\nOrbitrap Fusion Lumos method files to test hundreds of\nfragmentation conditions. Additionally it provides functions to\nanalyse and process the generated MS data and determine the\nbest conditions to maximise overall fragment coverage.","downloads_month":161,"downloads_total":3368,"page":"http://bioconductor.org/packages/release/bioc/html/topdownr.html"},{"authors":"Adrian Alexa and Jorg Rahnenfuhrer","name":"topGO","license":"LGPL","tags":["Microarray","Visualization"],"description":"topGO package provides tools for testing GO terms while\naccounting for the topology of the GO graph. Different test\nstatistics and different methods for eliminating local\nsimilarities and dependencies between GO terms can be\nimplemented and applied.","downloads_month":4025,"downloads_total":425584,"page":"http://bioconductor.org/packages/release/bioc/html/topGO.html"},{"authors":"Dorothee Childs, Nils Kurzawa, Holger Franken, Carola Doce, Mikhail Savitski and Wolfgang Huber","name":"TPP","license":"Artistic-2.0","tags":["ImmunoOncology","Proteomics","MassSpectrometry"],"description":"Analyze thermal proteome profiling (TPP) experiments with\nvarying temperatures (TR) or compound concentrations (CCR).","downloads_month":251,"downloads_total":18186,"page":"http://bioconductor.org/packages/release/bioc/html/TPP.html"},{"authors":"Tom Carroll, Sanjay Khadayate, Anne Pajon and Ziwei Liang","name":"tracktables","license":"GPL (>= 3)","tags":["Sequencing","ReportWriting"],"description":"Methods to create complex IGV genome browser sessions and\ndynamic IGV reports in HTML pages.","downloads_month":170,"downloads_total":16072,"page":"http://bioconductor.org/packages/release/bioc/html/tracktables.html"},{"authors":"person(\"Jianhong\", \"Ou\", email=\"jianhong.ou@duke.edu\", role=c(\"aut\", \"cre\"), comment=c(ORCID=\"0000-0002-8652-2488\"))\nand person(\"Julie\", \"Zhu\" and \"Lihua\")","name":"trackViewer","license":"GPL (>= 2)","tags":["Visualization"],"description":"Visualize mapped reads along with annotation as track\nlayers for NGS dataset such as ChIP-seq, RNA-seq, miRNA-seq,\nDNA-seq, SNPs and methylation data.","downloads_month":356,"downloads_total":40384,"page":"http://bioconductor.org/packages/release/bioc/html/trackViewer.html"},{"authors":"Diego Morais [aut, cre] and Rodrigo Dalmolin [aut]","name":"transcriptogramer","license":"GPL (>= 2)","tags":["Software","Network","Visualization","SystemsBiology","GeneExpression","GeneSetEnrichment","GraphAndNetwork","Clustering","DifferentialExpression","Microarray","RNASeq","Transcription","ImmunoOncology"],"description":"R package for transcriptional analysis based on\ntranscriptograms, a method to analyze transcriptomes that\nprojects expression values on a set of ordered proteins,\narranged such that the probability that gene products\nparticipate in the same metabolic pathway exponentially\ndecreases with the increase of the distance between two\nproteins of the ordering. Transcriptograms are, hence, genome\nwide gene expression profiles that provide a global view for\nthe cellular metabolism, while indicating gene sets whose\nexpression are altered.","downloads_month":150,"downloads_total":3616,"page":"http://bioconductor.org/packages/release/bioc/html/transcriptogramer.html"},{"authors":"Armen R. Karapetyan <armen.karapetyan87@gmail.com>","name":"transcriptR","license":"GPL-3","tags":["ImmunoOncology","Transcription","Software","Sequencing","RNASeq","Coverage"],"description":"The differences in the RNA types being sequenced have an\nimpact on the resulting sequencing profiles. mRNA-seq data is\nenriched with reads derived from exons, while GRO-, nucRNA- and\nchrRNA-seq demonstrate a substantial broader coverage of both\nexonic and intronic regions. The presence of intronic reads in\nGRO-seq type of data makes it possible to use it to\ncomputationally identify and quantify all de novo continuous\nregions of transcription distributed across the genome. This\ntype of data, however, is more challenging to interpret and\nless common practice compared to mRNA-seq. One of the\nchallenges for primary transcript detection concerns the\nsimultaneous transcription of closely spaced genes, which needs\nto be properly divided into individually transcribed units. The\nR package transcriptR combines RNA-seq data with ChIP-seq data\nof histone modifications that mark active Transcription Start\nSites (TSSs), such as, H3K4me3 or H3K9/14Ac to overcome this\nchallenge. The advantage of this approach over the use of, for\nexample, gene annotations is that this approach is data driven\nand therefore able to deal also with novel and case specific\nevents. Furthermore, the integration of ChIP- and RNA-seq data\nallows the identification all known and novel active\ntranscription start sites within a given sample.","downloads_month":208,"downloads_total":8946,"page":"http://bioconductor.org/packages/release/bioc/html/transcriptR.html"},{"authors":"Konstantin Krismer [aut, cre, cph]\n(<https://orcid.org/0000-0001-8994-3416>), Anna Gattinger [aut]\n(<https://orcid.org/0000-0001-7094-9279>), Michael Yaffe [ths, cph] (<https://orcid.org/0000-0002-9547-3251>) and Ian Cannell\n[ths] (<https://orcid.org/0000-0001-5832-9210>)","name":"transite","license":"MIT + file LICENSE","tags":["GeneExpression","Transcription","DifferentialExpression","Microarray","mRNAMicroarray","Genetics","GeneSetEnrichment"],"description":"transite is a computational method that allows\ncomprehensive analysis of the regulatory role of RNA-binding\nproteins in various cellular processes by leveraging\npreexisting gene expression data and current knowledge of\nbinding preferences of RNA-binding proteins.","downloads_month":86,"downloads_total":770,"page":"http://bioconductor.org/packages/release/bioc/html/transite.html"},{"authors":"Toma Tebaldi, Erik Dassi and Galena Kostoska","name":"tRanslatome","license":"GPL-3","tags":["CellBiology","GeneRegulation","Regulation","GeneExpression","DifferentialExpression","Microarray","HighThroughputSequencing","QualityControl","GO","MultipleComparisons","Bioinformatics"],"description":"Detection of differentially expressed genes (DEGs) from\nthe comparison of two biological conditions (treated vs.\nuntreated, diseased vs. normal, mutant vs. wild-type) among\ndifferent levels of gene expression (transcriptome\n,translatome, proteome), using several statistical methods:\nRank Product, Translational Efficiency, t-test, Limma, ANOTA,\nDESeq, edgeR. Possibility to plot the results with\nscatterplots, histograms, MA plots, standard deviation (SD)\nplots, coefficient of variation (CV) plots. Detection of\nsignificantly enriched post-transcriptional regulatory factors\n(RBPs, miRNAs, etc) and Gene Ontology terms in the lists of\nDEGs previously identified for the two expression levels.\nComparison of GO terms enriched only in one of the levels or in\nboth. Calculation of the semantic similarity score between the\nlists of enriched GO terms coming from the two expression\nlevels. Visual examination and comparison of the enriched terms\nwith heatmaps, radar plots and barplots.","downloads_month":156,"downloads_total":24934,"page":"http://bioconductor.org/packages/release/bioc/html/tRanslatome.html"},{"authors":"Julius Muller","name":"TransView","license":"GPL-3","tags":["ImmunoOncology","DNAMethylation","GeneExpression","Transcription","Microarray","Sequencing","Sequencing","ChIPSeq","RNASeq","MethylSeq","DataImport","Visualization","Clustering","MultipleComparison"],"description":"This package provides efficient tools to generate, access\nand display read densities of sequencing based data sets such\nas from RNA-Seq and ChIP-Seq.","downloads_month":198,"downloads_total":29506,"page":"http://bioconductor.org/packages/release/bioc/html/TransView.html"},{"authors":"Li Chen and Zhaohui S.Qin","name":"traseR","license":"GPL","tags":["Genetics","Sequencing","Coverage","Alignment","QualityControl","DataImport"],"description":"traseR performs GWAS trait-associated SNP enrichment\nanalyses in genomic intervals using different hypothesis\ntesting approaches, also provides various functionalities to\nexplore and visualize the results.","downloads_month":127,"downloads_total":10818,"page":"http://bioconductor.org/packages/release/bioc/html/traseR.html"},{"authors":"Guangchuang Yu [aut, cre]\n(<https://orcid.org/0000-0002-6485-8781>), Tommy Tsan-Yuk Lam\n[ctb, ths], Casey Dunn [ctb], Bradley Jones [ctb] and Tyler\nBradley [ctb]","name":"treeio","license":"Artistic-2.0","tags":["Software","Annotation","Clustering","DataImport","DataRepresentation","Alignment","MultipleSequenceAlignment"],"description":"Base classes and functions for parsing and exporting\nphylogenetic trees. 'treeio' supports parsing analysis findings\nfrom commonly used software packages, allows linking external\ndata to phylogeny and merging tree data obtained from different\nsources. It also supports exporting phylogenetic tree with\nheterogeneous associated data to a single tree file.","downloads_month":2355,"downloads_total":67054,"page":"http://bioconductor.org/packages/release/bioc/html/treeio.html"},{"authors":"Seth Ament <seth.ament@systemsbiology.org>, Paul Shannon\n<pshannon@systemsbioloyg.org> and Matthew Richards\n<mrichard@systemsbiology.org>","name":"trena","license":"GPL-3","tags":["Transcription","GeneRegulation","NetworkInference","FeatureExtraction","Regression","SystemsBiology","GeneExpression"],"description":"Methods for reconstructing transcriptional regulatory\nnetworks, especially in species for which genome-wide TF\nbinding site information is available.","downloads_month":130,"downloads_total":3052,"page":"http://bioconductor.org/packages/release/bioc/html/trena.html"},{"authors":"Rhonda Bacher and Ning Leng","name":"Trendy","license":"GPL-3","tags":["TimeCourse","RNASeq","Regression","ImmunoOncology"],"description":"Trendy implements segmented (or breakpoint) regression\nmodels to estimate breakpoints which represent changes in\nexpression for each feature/gene in high throughput data with\nordered conditions.","downloads_month":150,"downloads_total":2516,"page":"http://bioconductor.org/packages/release/bioc/html/Trendy.html"},{"authors":"Karl Kornacker Developer [aut], Tony Handstad Developer [aut and cre]","name":"triform","license":"GPL-2","tags":["Sequencing","ChIPSeq"],"description":"The Triform algorithm uses model-free statistics to\nidentify peak-like distributions of TF ChIP sequencing reads,\ntaking advantage of an improved peak definition in combination\nwith known profile characteristics.","downloads_month":131,"downloads_total":26428,"page":"http://bioconductor.org/packages/release/bioc/html/triform.html"},{"authors":"Lin S. Chen <lchen@health.bsd.uchicago.edu>, Dipen P.\nSangurdekar <dps@genomics.princeton.edu> and John D. Storey\n<jstorey@princeton.edu>","name":"trigger","license":"GPL-3","tags":["GeneExpression","SNP","GeneticVariability","Microarray","Genetics"],"description":"This R package provides tools for the statistical analysis\nof integrative genomic data that involve some combination of:\ngenotypes, high-dimensional intermediate traits (e.g., gene\nexpression, protein abundance), and higher-order traits\n(phenotypes). The package includes functions to: (1) construct\nglobal linkage maps between genetic markers and gene\nexpression; (2) analyze multiple-locus linkage (epistasis) for\ngene expression; (3) quantify the proportion of genome-wide\nvariation explained by each locus and identify eQTL hotspots;\n(4) estimate pair-wise causal gene regulatory probabilities and\nconstruct gene regulatory networks; and (5) identify causal\ngenes for a quantitative trait of interest.","downloads_month":140,"downloads_total":31722,"page":"http://bioconductor.org/packages/release/bioc/html/trigger.html"},{"authors":"Holger Schwender, Qing Li, Philipp Berger, Christoph Neumann, Margaret Taub and Ingo Ruczinski","name":"trio","license":"LGPL-2","tags":["SNP","GeneticVariability","Microarray","Genetics"],"description":"Testing SNPs and SNP interactions with a genotypic TDT.\nThis package furthermore contains functions for computing\npairwise values of LD measures and for identifying LD blocks,\nas well as functions for setting up matched case pseudo-control\ngenotype data for case-parent trios in order to run trio logic\nregression, for imputing missing genotypes in trios, for\nsimulating case-parent trios with disease risk dependent on SNP\ninteraction, and for power and sample size calculation in trio\ndata.","downloads_month":189,"downloads_total":28210,"page":"http://bioconductor.org/packages/release/bioc/html/trio.html"},{"authors":"Jiri Hon, Matej Lexa, Tomas Martinek and Kamil Rajdl with\ncontributions from Daniel Kopecek","name":"triplex","license":"BSD_2_clause + file LICENSE","tags":["SequenceMatching","GeneRegulation"],"description":"This package provides functions for identification and\nvisualization of potential intramolecular triplex patterns in\nDNA sequence. The main functionality is to detect the positions\nof subsequences capable of folding into an intramolecular\ntriplex (H-DNA) in a much larger sequence. The potential H-DNA\n(triplexes) should be made of as many cannonical nucleotide\ntriplets as possible. The package includes visualization\nshowing the exact base-pairing in 1D, 2D or 3D.","downloads_month":148,"downloads_total":26334,"page":"http://bioconductor.org/packages/release/bioc/html/triplex.html"},{"authors":"Felix GM Ernst [aut and cre]","name":"tRNA","license":"GPL-3 + file LICENSE","tags":["Software","Visualization"],"description":"The tRNA package allows tRNA sequences and structures to\nbe accessed and used for subsetting. In addition, it provides\nvisualization tools to compare feature parameters of multiple\ntRNA sets and correlate them to additional data. The tRNA\npackage uses GRanges objects as inputs requiring only few\nadditional column data sets.","downloads_month":106,"downloads_total":814,"page":"http://bioconductor.org/packages/release/bioc/html/tRNA.html"},{"authors":"Felix GM Ernst [aut and cre]","name":"tRNAdbImport","license":"GPL-3 + file LICENSE","tags":["Software","Visualization","DataImport"],"description":"tRNAdbImport imports the entries of the tRNAdb and mtRNAdb\n(http://trna.bioinf.uni-leipzig.de) as GRanges object.","downloads_month":79,"downloads_total":658,"page":"http://bioconductor.org/packages/release/bioc/html/tRNAdbImport.html"},{"authors":"Felix GM Ernst [aut and cre]","name":"tRNAscanImport","license":"GPL-3 + file LICENSE","tags":["Software","DataImport","WorkflowStep","Preprocessing","Visualization"],"description":"The package imports the result of tRNAscan-SE as a GRanges\nobject.","downloads_month":126,"downloads_total":1958,"page":"http://bioconductor.org/packages/release/bioc/html/tRNAscanImport.html"},{"authors":"Marco Antoniotti [ctb], Giulio Caravagna [aut, cre], Luca De\nSano [aut], Alex Graudenzi [aut], Giancarlo Mauri [ctb], Bud\nMishra [ctb] and Daniele Ramazzotti [aut]","name":"TRONCO","license":"file LICENSE","tags":["BiomedicalInformatics","Bayesian","GraphAndNetwork","SomaticMutation","NetworkInference","Network","Clustering","DataImport","SingleCell","ImmunoOncology"],"description":"The TRONCO (TRanslational ONCOlogy) R package collects\nalgorithms to infer progression models via the approach of\nSuppes-Bayes Causal Network, both from an ensemble of tumors\n(cross-sectional samples) and within an individual patient\n(multi-region or single-cell samples). The package provides\nparallel implementation of algorithms that process binary\nmatrices where each row represents a tumor sample and each\ncolumn a single-nucleotide or a structural variant driving the\nprogression; a 0/1 value models the absence/presence of that\nalteration in the sample. The tool can import data from plain,\nMAF or GISTIC format files, and can fetch it from the\ncBioPortal for cancer genomics. Functions for data manipulation\nand visualization are provided, as well as functions to\nimport/export such data to other bioinformatics tools for, e.g,\nclustering or detection of mutually exclusive alterations.\nInferred models can be visualized and tested for their\nconfidence via bootstrap and cross-validation. TRONCO is used\nfor the implementation of the Pipeline for Cancer Inference\n(PICNIC).","downloads_month":188,"downloads_total":16004,"page":"http://bioconductor.org/packages/release/bioc/html/TRONCO.html"},{"authors":"Zhicheng Ji and Hongkai Ji","name":"TSCAN","license":"GPL(>=2)","tags":["GeneExpression","Visualization","GUI"],"description":"TSCAN enables users to easily construct and tune\npseudotemporal cell ordering as well as analyzing\ndifferentially expressed genes. TSCAN comes with a\nuser-friendly GUI written in shiny. More features will come in\nthe future.","downloads_month":299,"downloads_total":23270,"page":"http://bioconductor.org/packages/release/bioc/html/TSCAN.html"},{"authors":"Jeffrey T. Leek <jtleek@jhu.edu>","name":"tspair","license":"GPL-2","tags":["Microarray"],"description":"These functions calculate the pair of genes that show the\nmaximum difference in ranking between two user specified\ngroups. This \"top scoring pair\" maximizes the average of\nsensitivity and specificity over all rank based classifiers\nusing a pair of genes in the data set. The advantage of\nclassifying samples based on only the relative rank of a pair\nof genes is (a) the classifiers are much simpler and often more\ninterpretable than more complicated classification schemes and\n(b) if arrays can be classified using only a pair of genes, PCR\nbased tests could be used for classification of samples. See\nthe references for the tspcalc() function for references\nregarding TSP classifiers.","downloads_month":148,"downloads_total":46678,"page":"http://bioconductor.org/packages/release/bioc/html/tspair.html"},{"authors":"R. Taylor Raborn [aut, cre, cph] Volker P. Brendel [aut and cph]\nKrishnakumar Sridharan [ctb]","name":"TSRchitect","license":"GPL-3","tags":["Clustering","FunctionalGenomics","GeneExpression","GeneRegulation","GenomeAnnotation","Sequencing","Transcription"],"description":"In recent years, large-scale transcriptional sequence data\nhas yielded considerable insights into the nature of gene\nexpression and regulation in eukaryotes. Techniques that\nidentify the 5' end of mRNAs, most notably CAGE, have mapped\nthe promoter landscape across a number of model organisms. Due\nto the variability of TSS distributions and the transcriptional\nnoise present in datasets, precisely identifying the active\npromoter(s) for genes from these datasets is not\nstraightforward. TSRchitect allows the user to efficiently\nidentify the putative promoter (the transcription start region,\nor TSR) from a variety of TSS profiling data types, including\nboth single-end (e.g. CAGE) as well as paired-end (RAMPAGE,\nPEAT). In addition, (new with version 1.2.0) TSRchitect\nprovides the ability to import aligned EST and cDNA data. Along\nwith the coordiantes of identified TSRs, TSRchitect also\ncalculates the width, abundance and Shape Index, and handles\nbiological replicates for expression profiling. Finally,\nTSRchitect imports annotation files, allowing the user to\nassociate identified promoters with genes and other genomic\nfeatures. Three detailed examples of TSRchitect's utility are\nprovided in the User's Guide, included with this package.","downloads_month":137,"downloads_total":5484,"page":"http://bioconductor.org/packages/release/bioc/html/TSRchitect.html"},{"authors":"Julian Gehring and Clemens Kreutz","name":"TSSi","license":"GPL-3","tags":["Sequencing","RNASeq","Genetics","Preprocessing"],"description":"Identify and normalize transcription start sites in\nhigh-throughput sequencing data.","downloads_month":148,"downloads_total":32686,"page":"http://bioconductor.org/packages/release/bioc/html/TSSi.html"},{"authors":"Rachel Jeitziner","name":"TTMap","license":"GPL-2","tags":["Software","Microarray","DifferentialExpression","MultipleComparison","Clustering","Classification"],"description":"TTMap is a clustering method that groups together samples\nwith the same deviation in comparison to a control group. It is\nspecially useful when the data is small. It is parameter free.","downloads_month":128,"downloads_total":2052,"page":"http://bioconductor.org/packages/release/bioc/html/TTMap.html"},{"authors":"Maarten van Iterson and Chantal van Leeuwen","name":"TurboNorm","license":"LGPL","tags":["Microarray","OneChannel","TwoChannel","Preprocessing","DNAMethylation","CpGIsland","MethylationArray","Normalization"],"description":"A fast scatterplot smoother based on B-splines with\nsecond-order difference penalty. Functions for microarray\nnormalization of single-colour data i.e. Affymetrix/Illumina\nand two-colour data supplied as marray MarrayRaw-objects or\nlimma RGList-objects are available.","downloads_month":141,"downloads_total":32882,"page":"http://bioconductor.org/packages/release/bioc/html/TurboNorm.html"},{"authors":"Kevin Rue-Albrecht [aut and cre]","name":"TVTB","license":"Artistic-2.0","tags":["Software","Genetics","GeneticVariability","GenomicVariation","DataRepresentation","GUI","Genetics","DNASeq","WholeGenome","Visualization","MultipleComparison","DataImport","VariantAnnotation","Sequencing","Coverage","Alignment","SequenceMatching"],"description":"The package provides S4 classes and methods to filter,\nsummarise and visualise genetic variation data stored in VCF\nfiles. In particular, the package extends the FilterRules class\n(S4Vectors package) to define news classes of filter rules\napplicable to the various slots of VCF objects. Functionalities\nare integrated and demonstrated in a Shiny web-application, the\nShiny Variant Explorer (tSVE).","downloads_month":133,"downloads_total":6484,"page":"http://bioconductor.org/packages/release/bioc/html/TVTB.html"},{"authors":"Juan R Gonzalez <jrgonzalez@creal.cat> and Mikel Esnaola\n<mesnaola@creal.cat> (with contributions from Robert Castelo\n<robert.castelo@upf.edu>)","name":"tweeDEseq","license":"GPL (>= 2)","tags":["ImmunoOncology","StatisticalMethod","DifferentialExpression","Sequencing","RNASeq"],"description":"Differential expression analysis of RNA-seq using the\nPoisson-Tweedie family of distributions.","downloads_month":329,"downloads_total":41014,"page":"http://bioconductor.org/packages/release/bioc/html/tweeDEseq.html"},{"authors":"Stefanie Scheid <stefanie.scheid@gmx.de>","name":"twilight","license":"GPL (>= 2)","tags":["Microarray","DifferentialExpression","MultipleComparison"],"description":"In a typical microarray setting with gene expression data\nobserved under two conditions, the local false discovery rate\ndescribes the probability that a gene is not differentially\nexpressed between the two conditions given its corrresponding\nobserved score or p-value level. The resulting curve of\np-values versus local false discovery rate offers an insight\ninto the twilight zone between clear differential and clear\nnon-differential gene expression. Package 'twilight' contains\ntwo main functions: Function twilight.pval performs a\ntwo-condition test on differences in means for a given input\nmatrix or expression set and computes permutation based\np-values. Function twilight performs a stochastic downhill\nsearch to estimate local false discovery rates and effect size\ndistributions. The package further provides means to filter for\npermutations that describe the null distribution correctly.\nUsing filtered permutations, the influence of hidden\nconfounders could be diminished.","downloads_month":268,"downloads_total":68988,"page":"http://bioconductor.org/packages/release/bioc/html/twilight.html"},{"authors":"Anthony Chiu [aut and cre]","name":"twoddpcr","license":"GPL-3","tags":["ddPCR","Software","Classification"],"description":"The twoddpcr package takes Droplet Digital PCR (ddPCR)\ndroplet amplitude data from Bio-Rad's QuantaSoft and can\nclassify the droplets. A summary of the positive/negative\ndroplet counts can be generated, which can then be used to\nestimate the number of molecules using the Poisson\ndistribution. This is the first open source package that\nfacilitates the automatic classification of general two channel\nddPCR data. Previous work includes 'definetherain' (Jones et\nal., 2014) and 'ddpcRquant' (Trypsteen et al., 2015) which both\nhandle one channel ddPCR experiments only. The 'ddpcr' package\navailable on CRAN (Attali et al., 2016) supports automatic\ngating of a specific class of two channel ddPCR experiments\nonly.","downloads_month":133,"downloads_total":5148,"page":"http://bioconductor.org/packages/release/bioc/html/twoddpcr.html"},{"authors":"Michael Love [aut, cre], Rob Patro [aut, ctb], Peter Hickey\n[aut, ctb], Charlotte Soneson [aut and ctb]","name":"tximeta","license":"GPL-2","tags":["Annotation","DataImport","Preprocessing","RNASeq","Transcriptomics","Transcription","GeneExpression","ImmunoOncology"],"description":"Transcript quantification import from Salmon with\nautomatic population of metadata and transcript ranges.\nFiltered, combined, or de novo transcriptomes can be linked to\nthe appropriate sources with linkedTxomes and shared for\nreproducible analyses.","downloads_month":231,"downloads_total":1960,"page":"http://bioconductor.org/packages/release/bioc/html/tximeta.html"},{"authors":"Michael Love [cre, aut], Charlotte Soneson [aut], Mark Robinson\n[aut], Rob Patro [ctb], Andrew Parker Morgan [ctb], Ryan C.\nThompson [ctb], Matt Shirley [ctb] and Avi Srivastava [ctb]","name":"tximport","license":"GPL (>=2)","tags":["DataImport","Preprocessing","RNASeq","Transcriptomics","Transcription","GeneExpression","ImmunoOncology"],"description":"Imports transcript-level abundance, estimated counts and\ntranscript lengths, and summarizes into matrices for use with\ndownstream gene-level analysis packages. Average transcript\nlength, weighted by sample-specific transcript abundance\nestimates, is provided as a matrix which can be used as an\noffset for different expression of gene-level counts.","downloads_month":4847,"downloads_total":185028,"page":"http://bioconductor.org/packages/release/bioc/html/tximport.html"},{"authors":"Vince Carey","name":"TxRegInfra","license":"Artistic-2.0","tags":["Network"],"description":"This package provides interfaces to genomic metadata\nemployed in regulatory network creation, with a focus on noSQL\nsolutions.  Currently quantitative representations of eQTLs,\nDnaseI hypersensitivity sites and digital genomic footprints\nare assembled using an out-of-memory extension of the\nRaggedExperiment API.","downloads_month":118,"downloads_total":1894,"page":"http://bioconductor.org/packages/release/bioc/html/TxRegInfra.html"},{"authors":"Duncan Temple Lang Robert Gentleman (<rgentlem@fhcrc.org>)","name":"TypeInfo","license":"BSD","tags":["Infrastructure"],"description":"A prototype for a mechanism for specifying the types of\nparameters and the return value for an R function. This is\nmeta-information that can be used to generate stubs for servers\nand various interfaces to these functions. Additionally, the\narguments in a call to a typed function can be validated using\nthe type specifications. We allow types to be specified as\neither i) by class name using either inheritance - is(x,\nclassName), or strict instance of - class(x) %in% className, or\nii) a dynamic test given as an R expression which is evaluated\nat run-time. More precise information and interesting tests can\nbe done via ii), but it is harder to use this information as\nmeta-data as it requires more effort to interpret it and it is\nof course run-time information. It is typically more\nmeaningful.","downloads_month":135,"downloads_total":45708,"page":"http://bioconductor.org/packages/release/bioc/html/TypeInfo.html"},{"authors":"David Humphreys [aut and cre]","name":"Ularcirc","license":"file LICENSE","tags":["DataRepresentation","Visualization","Genetics","Sequencing","Annotation","Coverage","AlternativeSplicing","DifferentialSplicing"],"description":"Ularcirc reads in STAR aligned splice junction files and\nprovides visualisation and analysis tools for splicing\nanalysis. Users can assess backsplice junctions and forward\ncanonical junctions.","downloads_month":85,"downloads_total":756,"page":"http://bioconductor.org/packages/release/bioc/html/Ularcirc.html"},{"authors":"Niya Wang <wangny@vt.edu>","name":"UNDO","license":"GPL-2","tags":["Software"],"description":"UNDO is an R package for unsupervised deconvolution of\ntumor and stromal mixed expression data. It detects marker\ngenes and deconvolutes the mixing expression data without any\nprior knowledge.","downloads_month":152,"downloads_total":18204,"page":"http://bioconductor.org/packages/release/bioc/html/UNDO.html"},{"authors":"Jan R. De Neve & Joris Meys","name":"unifiedWMWqPCR","license":"GPL (>=2)","tags":["DifferentialExpression","GeneExpression","MicrotitrePlateAssay","MultipleComparison","QualityControl","Software","Visualization","qPCR"],"description":"This packages implements the unified Wilcoxon-Mann-Whitney\nTest for qPCR data. This modified test allows for testing\ndifferential expression in qPCR data.","downloads_month":136,"downloads_total":17784,"page":"http://bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html"},{"authors":"Marc Carlson [aut, cre] and Csaba Ortutay [ctb]","name":"UniProt.ws","license":"Artistic License 2.0","tags":["Annotation","Infrastructure","GO","KEGG","BioCarta"],"description":"A collection of functions for retrieving, processing and\nrepackaging the UniProt web services.","downloads_month":403,"downloads_total":49974,"page":"http://bioconductor.org/packages/release/bioc/html/UniProt.ws.html"},{"authors":"Raik Otto","name":"Uniquorn","license":"Artistic-2.0","tags":["ImmunoOncology","StatisticalMethod","WholeGenome","ExomeSeq"],"description":"This packages enables users to identify cancer cell lines.\nCancer cell line misidentification and cross-contamination\nreprents a significant challenge for cancer researchers. The\nidentification is vital and in the frame of this package based\non the locations/ loci of somatic and germline mutations/\nvariations. The input format is vcf/ vcf.gz and the files have\nto contain a single cancer cell line sample (i.e. a single\nmember/genotype/gt column in the vcf file). The implemented\nmethod is optimized for the Next-generation whole exome and\nwhole genome DNA-sequencing technology. RNA-seq data is very\nlikely to work as well but hasn't been rigiously tested yet.\nPanel-seq will require manual adjustment of thresholds","downloads_month":181,"downloads_total":8120,"page":"http://bioconductor.org/packages/release/bioc/html/Uniquorn.html"},{"authors":"Benjamin Jean-Marie Tremblay [aut and cre]","name":"universalmotif","license":"GPL-3","tags":["MotifAnnotation","MotifDiscovery","DataImport","GeneRegulation"],"description":"Allows for importing most common motif types into R for\nuse by functions provided by other Bioconductor motif-related\npackages. Motifs can be exported into most major motif formats\nfrom various classes as defined by other Bioconductor packages.\nA suite of motif and sequence manipulation and analysis\nfunctions are included, including enrichment, comparison,\nP-value calculation, shuffling, trimming, higher-order motifs,\nand others.","downloads_month":152,"downloads_total":1248,"page":"http://bioconductor.org/packages/release/bioc/html/universalmotif.html"},{"authors":"Mai Chan Lau, Hao Chen and Jinmiao Chen","name":"uSORT","license":"Artistic-2.0","tags":["ImmunoOncology","RNASeq","GUI","CellBiology","DNASeq"],"description":"This package is designed to uncover the intrinsic cell\nprogression path from single-cell RNA-seq data. It incorporates\ndata pre-processing, preliminary PCA gene selection,\npreliminary cell ordering, feature selection, refined cell\nordering, and post-analysis interpretation and visualization.","downloads_month":203,"downloads_total":7048,"page":"http://bioconductor.org/packages/release/bioc/html/uSORT.html"},{"authors":"Robert Scharpf <rscharpf@jhu.edu>, Kevin Scharpf, and Ingo\nRuczinski <ingo@jhsph.edu>","name":"VanillaICE","license":"LGPL-2","tags":["CopyNumberVariation"],"description":"Hidden Markov Models for characterizing chromosomal\nalterations in high throughput SNP arrays.","downloads_month":187,"downloads_total":59862,"page":"http://bioconductor.org/packages/release/bioc/html/VanillaICE.html"},{"authors":"Gabriel E. Hoffman","name":"variancePartition","license":"GPL (>= 2)","tags":["RNASeq","GeneExpression","GeneSetEnrichment","DifferentialExpression","BatchEffect","QualityControl","Regression","Epigenetics","FunctionalGenomics","Transcriptomics","Normalization","Preprocessing","Microarray","ImmunoOncology","Software"],"description":"Quantify and interpret multiple sources of biological and\ntechnical variation in gene expression experiments. Uses a\nlinear mixed model to quantify variation in gene expression\nattributable to individual, tissue, time point, or technical\nvariables.  Includes dream differential expression analysis for\nrepeated measures.","downloads_month":346,"downloads_total":17982,"page":"http://bioconductor.org/packages/release/bioc/html/variancePartition.html"},{"authors":"Valerie Obenchain [aut, cre], Martin Morgan [aut], Michael\nLawrence [aut] and Stephanie Gogarten [ctb]","name":"VariantAnnotation","license":"Artistic-2.0","tags":["DataImport","Sequencing","SNP","Annotation","Genetics","VariantAnnotation"],"description":"Annotate variants, compute amino acid coding changes,\npredict coding outcomes.","downloads_month":15530,"downloads_total":1370950,"page":"http://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html"},{"authors":"Robert Castelo [aut, cre], Dei Martinez Elurbe [ctb], Pau\nPuigdevall [ctb] and Joan Fernandez [ctb]","name":"VariantFiltering","license":"Artistic-2.0","tags":["Genetics","Homo_sapiens","Annotation","SNP","Sequencing","HighThroughputSequencing"],"description":"Filter genetic variants using different criteria such as\ninheritance model, amino acid change consequence, minor allele\nfrequencies across human populations, splice site strength,\nconservation, etc.","downloads_month":169,"downloads_total":30892,"page":"http://bioconductor.org/packages/release/bioc/html/VariantFiltering.html"},{"authors":"Michael Lawrence, Jeremiah Degenhardt and Robert Gentleman","name":"VariantTools","license":"Artistic-2.0","tags":["Genetics","GeneticVariability","Sequencing"],"description":"Explore, diagnose, and compare variant calls using\nfilters.","downloads_month":213,"downloads_total":28854,"page":"http://bioconductor.org/packages/release/bioc/html/VariantTools.html"},{"authors":"Nicola Lama <nicola.lama@unina2.it> and Mark Girolami\n<girolami@dcs.gla.ac.uk>","name":"vbmp","license":"GPL (>= 2)","tags":["Classification"],"description":"Variational Bayesian Multinomial Probit Regression with\nGaussian Process Priors. It estimates class membership\nposterior probability employing variational and sparse\napproximation to the full posterior. This software also\nincorporates feature weighting by means of Automatic Relevance\nDetermination.","downloads_month":186,"downloads_total":50160,"page":"http://bioconductor.org/packages/release/bioc/html/vbmp.html"},{"authors":"Qian Liu [aut, cre] and Martin Morgan [aut]","name":"VCFArray","license":"GPL-3","tags":["Infrastructure","DataRepresentation","Sequencing","VariantAnnotation"],"description":"VCFArray extends the DelayedArray to represent VCF data\nentries as array-like objects with on-disk / remote VCF file as\nbackend. Data entries from VCF files, including info fields,\nFORMAT fields, and the fixed columns (REF, ALT, QUAL, FILTER)\ncould be converted into VCFArray instances with different\ndimensions.","downloads_month":37,"downloads_total":172,"page":"http://bioconductor.org/packages/release/bioc/html/VCFArray.html"},{"authors":"Sandro Morganella","name":"Vega","license":"GPL-2","tags":["aCGH","CopyNumberVariation"],"description":"Vega (Variational Estimator for Genomic Aberrations) is an\nalgorithm that adapts a very popular variational model (Mumford\nand Shah) used in image segmentation so that chromosomal\naberrant regions can be efficiently detected.","downloads_month":128,"downloads_total":31254,"page":"http://bioconductor.org/packages/release/bioc/html/Vega.html"},{"authors":"S. Morganella and M. Ceccarelli","name":"VegaMC","license":"GPL-2","tags":["aCGH","CopyNumberVariation"],"description":"This package enables the detection of driver chromosomal\nimbalances including loss of heterozygosity (LOH) from array\ncomparative genomic hybridization (aCGH) data. VegaMC performs\na joint segmentation of a dataset and uses a statistical\nframework to distinguish between driver and passenger mutation.\nVegaMC has been implemented so that it can be immediately\nintegrated with the output produced by PennCNV tool. In\naddition, VegaMC produces in output two web pages that allows a\nrapid navigation between both the detected regions and the\naltered genes. In the web page that summarizes the altered\ngenes, the link to the respective Ensembl gene web page is\nreported.","downloads_month":140,"downloads_total":29020,"page":"http://bioconductor.org/packages/release/bioc/html/VegaMC.html"},{"authors":"Brandon Monier [aut, cre], Adam McDermaid [aut], Jing Zhao\n[aut], Qin Ma [aut and fnd]","name":"vidger","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","Visualization","RNASeq","DifferentialExpression","GeneExpression"],"description":"The aim of vidger is to rapidly generate information-rich\nvisualizations for the interpretation of differential gene\nexpression results from three widely-used tools: Cuffdiff,\nDESeq2, and edgeR.","downloads_month":238,"downloads_total":3292,"page":"http://bioconductor.org/packages/release/bioc/html/vidger.html"},{"authors":"Mariano J Alvarez <reef103@gmail.com>","name":"viper","license":"file LICENSE","tags":["SystemsBiology","NetworkEnrichment","GeneExpression","FunctionalPrediction","GeneRegulation"],"description":"Inference of protein activity from gene expression data,\nincluding the VIPER and msVIPER algorithms","downloads_month":402,"downloads_total":28994,"page":"http://bioconductor.org/packages/release/bioc/html/viper.html"},{"authors":"Wolfgang Huber, with contributions from Anja von Heydebreck.\nMany comments, suggestions by users are acknowledged, among\nthem Dennis Kostka, David Kreil, Hans-Ulrich Klein, Robert\nGentleman and Deepayan Sarkar and Gordon Smyth","name":"vsn","license":"Artistic-2.0","tags":["Microarray","OneChannel","TwoChannel","Preprocessing"],"description":"The package implements a method for normalising microarray\nintensities, and works for single- and multiple-color arrays.\nIt can also be used for data from other technologies, as long\nas they have similar format. The method uses a robust variant\nof the maximum-likelihood estimator for an\nadditive-multiplicative error model and affine calibration. The\nmodel incorporates data calibration step (a.k.a.\nnormalization), a model for the dependence of the variance on\nthe mean intensity and a variance stabilizing data\ntransformation. Differences between transformed intensities are\nanalogous to \"normalized log-ratios\". However, in contrast to\nthe latter, their variance is independent of the mean, and they\nare usually more sensitive and specific in detecting\ndifferential transcription.","downloads_month":5210,"downloads_total":991308,"page":"http://bioconductor.org/packages/release/bioc/html/vsn.html"},{"authors":"VJ Carey <stvjc@channing.harvard.edu>","name":"vtpnet","license":"Artistic-2.0","tags":["Network"],"description":"variant-transcription factor-phenotype networks, inspired\nby Maurano et al., Science (2012), PMID 22955828","downloads_month":137,"downloads_total":21796,"page":"http://bioconductor.org/packages/release/bioc/html/vtpnet.html"},{"authors":"Federico M. Giorgi, Andrew N. Holding and Florian Markowetz","name":"vulcan","license":"LGPL-3","tags":["SystemsBiology","NetworkEnrichment","GeneExpression","ChIPSeq"],"description":"Vulcan (VirtUaL ChIP-Seq Analysis through Networks) is a\npackage that interrogates gene regulatory networks to infer\ncofactors significantly enriched in a differential binding\nsignature coming from ChIP-Seq data. In order to do so, our\npackage combines strategies from different BioConductor\npackages: DESeq for data normalization, ChIPpeakAnno and\nDiffBind for annotation and definition of ChIP-Seq genomic\npeaks, csaw to define optimal peak width and viper for applying\na regulatory network over a differential binding signature.","downloads_month":133,"downloads_total":3800,"page":"http://bioconductor.org/packages/release/bioc/html/vulcan.html"},{"authors":"Leonard C Schalkwyk, Ruth Pidsley, Chloe CY Wong, with\nfunctions contributed by Nizar Touleimat, Matthieu Defrance, Andrew Teschendorff, Jovana Maksimovic, Tyler Gorrie-Stone and Louis El Khoury","name":"wateRmelon","license":"GPL-3","tags":["DNAMethylation","Microarray","TwoChannel","Preprocessing","QualityControl"],"description":"15 flavours of betas and three performance metrics, with\nmethods for objects produced by methylumi and minfi packages.","downloads_month":1073,"downloads_total":110796,"page":"http://bioconductor.org/packages/release/bioc/html/wateRmelon.html"},{"authors":"Federico Comoglio and Cem Sievers","name":"wavClusteR","license":"GPL-2","tags":["ImmunoOncology","Sequencing","Technology","RIPSeq","RNASeq","Bayesian"],"description":"The package provides an integrated pipeline for the\nanalysis of PAR-CLIP data. PAR-CLIP-induced transitions are\nfirst discriminated from sequencing errors, SNPs and additional\nnon-experimental sources by a non- parametric mixture model.\nThe protein binding sites (clusters) are then resolved at high\nresolution and cluster statistics are estimated using a\nrigorous Bayesian framework. Post-processing of the results,\ndata export for UCSC genome browser visualization and motif\nsearch analysis are provided. In addition, the package allows\nto integrate RNA-Seq data to estimate the False Discovery Rate\nof cluster detection. Key functions support parallel multicore\ncomputing. Note: while wavClusteR was designed for PAR-CLIP\ndata analysis, it can be applied to the analysis of other NGS\ndata obtained from experimental procedures that induce\nnucleotide substitutions (e.g. BisSeq).","downloads_month":188,"downloads_total":17298,"page":"http://bioconductor.org/packages/release/bioc/html/wavClusteR.html"},{"authors":"Kristof De Beuf <kristof.debeuf@UGent.be>, Peter Pipelers\n<peter.pipelers@ugent.be> and Lieven Clement\n<lieven.clement@gmail.com>","name":"waveTiling","license":"GPL (>=2)","tags":["Microarray","DifferentialExpression","TimeCourse","GeneExpression"],"description":"This package is designed to conduct transcriptome analysis\nfor tiling arrays based on fast wavelet-based functional\nmodels.","downloads_month":143,"downloads_total":27062,"page":"http://bioconductor.org/packages/release/bioc/html/waveTiling.html"},{"authors":"Seth Falcon","name":"weaver","license":"GPL-2","tags":["Infrastructure"],"description":"This package provides enhancements on the Sweave()\nfunction in the base package.  In particular a facility for\ncaching code chunk results is included.","downloads_month":196,"downloads_total":60932,"page":"http://bioconductor.org/packages/release/bioc/html/weaver.html"},{"authors":"Colin A. Smith <colin@colinsmith.org>","name":"webbioc","license":"GPL (>= 2)","tags":["Infrastructure","Microarray","OneChannel","DifferentialExpression"],"description":"An integrated web interface for doing microarray analysis\nusing several of the Bioconductor packages. It is intended to\nbe deployed as a centralized bioinformatics resource for use by\nmany users. (Currently only Affymetrix oligonucleotide analysis\nis supported.)","downloads_month":148,"downloads_total":46876,"page":"http://bioconductor.org/packages/release/bioc/html/webbioc.html"},{"authors":"Jianhua Zhang","name":"widgetTools","license":"LGPL","tags":["Infrastructure"],"description":"This packages contains tools to support the construction\nof tcltk widgets","downloads_month":904,"downloads_total":256078,"page":"http://bioconductor.org/packages/release/bioc/html/widgetTools.html"},{"authors":"Kaur Alasoo [aut and cre]","name":"wiggleplotr","license":"Apache License 2.0","tags":["ImmunoOncology","Coverage","RNASeq","ChIPSeq","Sequencing","Visualization","GeneExpression","Transcription","AlternativeSplicing"],"description":"Tools to visualise read coverage from sequencing\nexperiments together with genomic annotations (genes,\ntranscripts, peaks). Introns of long transcripts can be\nrescaled to a fixed length for better visualisation of exonic\nread coverage.","downloads_month":203,"downloads_total":5826,"page":"http://bioconductor.org/packages/release/bioc/html/wiggleplotr.html"},{"authors":"Senthil Kumar Muthiah [aut], Hector Corrada Bravo [aut and cre]","name":"Wrench","license":"Artistic-2.0","tags":["Normalization","Sequencing","Software"],"description":"Wrench is a package for normalization sparse genomic count\ndata, like that arising from 16s metagenomic surveys.","downloads_month":93,"downloads_total":720,"page":"http://bioconductor.org/packages/release/bioc/html/Wrench.html"},{"authors":"Yuanhang Liu","name":"XBSeq","license":"GPL (>=3)","tags":["ImmunoOncology","RNASeq","DifferentialExpression","Sequencing","Software","ExperimentalDesign"],"description":"We developed a novel algorithm, XBSeq, where a statistical\nmodel was established based on the assumption that observed\nsignals are the convolution of true expression signals and\nsequencing noises. The mapped reads in non-exonic regions are\nconsidered as sequencing noises, which follows a Poisson\ndistribution. Given measureable observed and noise signals from\nRNA-seq data, true expression signals, assuming governed by the\nnegative binomial distribution, can be delineated and thus the\naccurate detection of differential expressed genes.","downloads_month":207,"downloads_total":12066,"page":"http://bioconductor.org/packages/release/bioc/html/XBSeq.html"},{"authors":"Colin A. Smith <csmith@scripps.edu>, Ralf Tautenhahn\n<rtautenh@gmail.com>, Steffen Neumann <sneumann@ipb-halle.de>, Paul Benton <hpbenton@scripps.edu>, Christopher Conley\n<cjconley@ucdavis.edu> and Johannes Rainer\n<Johannes.Rainer@eurac.edu>","name":"xcms","license":"GPL (>= 2) + file LICENSE","tags":["ImmunoOncology","MassSpectrometry","Metabolomics"],"description":"Framework for processing and visualization of\nchromatographically separated and single-spectra mass spectral\ndata. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML\nfiles. Preprocesses data for high-throughput, untargeted\nanalyte profiling.","downloads_month":2710,"downloads_total":1393118,"page":"http://bioconductor.org/packages/release/bioc/html/xcms.html"},{"authors":"R.B. Scharpf, G. Parmigiani, A.B. Nobel, and H. Tjelmeland","name":"XDE","license":"LGPL-2","tags":["Microarray","DifferentialExpression"],"description":"Multi-level model for cross-study detection of\ndifferential gene expression.","downloads_month":151,"downloads_total":43894,"page":"http://bioconductor.org/packages/release/bioc/html/XDE.html"},{"authors":"Lee, Lang Ho, Singh and Sasha A.","name":"XINA","license":"GPL-3","tags":["ImmunoOncology","SystemsBiology","Proteomics","RNASeq","Network"],"description":"An intuitive R package simplifies network analyses output\nfrom multiplexed high-dimensional proteomics/trascriptomics\nkinetics data.","downloads_month":121,"downloads_total":1048,"page":"http://bioconductor.org/packages/release/bioc/html/XINA.html"},{"authors":"Tim Yates <Tim.Yates@cruk.manchester.ac.uk> and Crispin J\nMiller <Crispin.Miller@cruk.manchester.ac.uk>","name":"xmapbridge","license":"LGPL-3","tags":["Annotation","ReportWriting","Visualization"],"description":"xmapBridge can plot graphs in the X:Map genome browser.\nThis package exports plotting files in a suitable format.","downloads_month":124,"downloads_total":39558,"page":"http://bioconductor.org/packages/release/bioc/html/xmapbridge.html"},{"authors":"Christian Stratowa, Vienna and Austria","name":"xps","license":"GPL (>= 2.0)","tags":["ExonArray","GeneExpression","Microarray","OneChannel","DataImport","Preprocessing","Transcription","DifferentialExpression"],"description":"The package handles pre-processing, normalization,\nfiltering and analysis of Affymetrix GeneChip expression\narrays, including exon arrays (Exon 1.0 ST: core, extended,\nfull probesets), gene arrays (Gene 1.0 ST) and plate arrays on\ncomputers with 1 GB RAM only. It imports Affymetrix .CDF, .CLF,\n.PGF and .CEL as well as annotation files, and computes e.g.\nRMA, MAS5, FARMS, DFW, FIRMA, tRMA, MAS5-calls, DABG-calls,\nI/NI-calls. It is an R wrapper to XPS (eXpression Profiling\nSystem), which is based on ROOT, an object-oriented framework\ndeveloped at CERN. Thus, the prior installation of ROOT is a\nprerequisite for the usage of this package, however, no\nknowledge of ROOT is required. ROOT is licensed under LGPL and\ncan be downloaded from http://root.cern.ch.","downloads_month":187,"downloads_total":77346,"page":"http://bioconductor.org/packages/release/bioc/html/xps.html"},{"authors":"Hervé Pagès and Patrick Aboyoun","name":"XVector","license":"Artistic-2.0","tags":["Infrastructure","DataRepresentation"],"description":"Memory efficient S4 classes for storing sequences\n\"externally\" (behind an R external pointer, or on disk).","downloads_month":35132,"downloads_total":2482974,"page":"http://bioconductor.org/packages/release/bioc/html/XVector.html"},{"authors":"Leslie Myint [cre, aut] and Kasper Daniel Hansen [aut]","name":"yamss","license":"Artistic-2.0","tags":["MassSpectrometry","Metabolomics","ImmunoOncology","Software"],"description":"Tools to analyze and visualize high-throughput\nmetabolomics data aquired using chromatography-mass\nspectrometry. These tools preprocess data in a way that enables\nreliable and powerful differential analysis.","downloads_month":177,"downloads_total":7998,"page":"http://bioconductor.org/packages/release/bioc/html/yamss.html"},{"authors":"Daniel Huebschmann, Zuguang Gu and Matthias Schlesner","name":"YAPSA","license":"GPL-3","tags":["Sequencing","DNASeq","SomaticMutation","Visualization","Clustering","GenomicVariation","StatisticalMethod","BiologicalQuestion"],"description":"This package provides functions and routines useful in the\nanalysis of somatic signatures (cf. L. Alexandrov et al.,\nNature 2013). In particular, functions to perform a signature\nanalysis with known signatures (LCD = linear combination\ndecomposition) and a signature analysis on stratified\nmutational catalogue (SMC = stratify mutational catalogue) are\nprovided.","downloads_month":188,"downloads_total":8194,"page":"http://bioconductor.org/packages/release/bioc/html/YAPSA.html"},{"authors":"Laurent Gatto","name":"yaqcaffy","license":"Artistic-2.0","tags":["Microarray","OneChannel","QualityControl","ReportWriting"],"description":"Quality control of Affymetrix GeneChip expression data and\nreproducibility analysis of human whole genome chips with the\nMAQC reference datasets.","downloads_month":166,"downloads_total":51236,"page":"http://bioconductor.org/packages/release/bioc/html/yaqcaffy.html"},{"authors":"Joseph N Paulson [aut, cre], Cho-Yi Chen [aut], Camila\nLopes-Ramos [aut], Marieke Kuijjer [aut], John Platig [aut], Abhijeet Sonawane [aut], Maud Fagny [aut], Kimberly Glass\n[aut] and John Quackenbush [aut]","name":"yarn","license":"Artistic-2.0","tags":["Software","QualityControl","GeneExpression","Sequencing","Preprocessing","Normalization","Annotation","Visualization","Clustering"],"description":"Expedite large RNA-Seq analyses using a combination of\npreviously developed tools. YARN is meant to make it easier for\nthe user in performing basic mis-annotation quality control,\nfiltering, and condition-aware normalization. YARN leverages\nmany Bioconductor tools and statistical techniques to account\nfor the large heterogeneity and sparsity found in very large\nRNA-seq experiments.","downloads_month":166,"downloads_total":8010,"page":"http://bioconductor.org/packages/release/bioc/html/yarn.html"},{"authors":"Ron Ammar [aut, cre] and John Thompson [aut]","name":"zFPKM","license":"GPL-3 | file LICENSE","tags":["ImmunoOncology","RNASeq","FeatureExtraction","Software","GeneExpression"],"description":"Perform the zFPKM transform on RNA-seq FPKM data. This\nalgorithm is based on the publication by Hart et al., 2013\n(Pubmed ID 24215113). Reference recommends using zFPKM > -3 to\nselect expressed genes. Validated with encode open/closed\nchromosome data. Works well for gene level data using FPKM or\nTPM. Does not appear to calibrate well for transcript level\ndata.","downloads_month":246,"downloads_total":4580,"page":"http://bioconductor.org/packages/release/bioc/html/zFPKM.html"},{"authors":"Davide Risso [aut, cre, cph], Svetlana Gribkova [aut], Fanny\nPerraudeau [aut] and Jean-Philippe Vert [aut]","name":"zinbwave","license":"Artistic-2.0","tags":["ImmunoOncology","DimensionReduction","GeneExpression","RNASeq","Software","Transcriptomics","Sequencing","SingleCell"],"description":"Implements a general and flexible zero-inflated negative\nbinomial model that can be used to provide a low-dimensional\nrepresentations of single-cell RNA-seq data. The model accounts\nfor zero inflation (dropouts), over-dispersion, and the count\nnature of the data. The model also accounts for the difference\nin library sizes and optionally for batch effects and/or other\ncovariates, avoiding the need for pre-normalize the data.","downloads_month":1139,"downloads_total":20356,"page":"http://bioconductor.org/packages/release/bioc/html/zinbwave.html"},{"authors":"Martin Morgan","name":"zlibbioc","license":"Artistic-2.0 + file LICENSE","tags":["Infrastructure"],"description":"This package uses the source code of zlib-1.2.5 to create\nlibraries for systems that do not have these available via\nother means (most Linux and Mac users should have system-level\naccess to zlib, and no direct need for this package). See the\nvignette for instructions on use.","downloads_month":38627,"downloads_total":3041846,"page":"http://bioconductor.org/packages/release/bioc/html/zlibbioc.html"},{"authors":"Ernest Turro, Graeme Ambler and Anne-Mette K Hein","name":"bgx","license":"GPL-2","tags":["Microarray","DifferentialExpression"],"description":"Bayesian integrated analysis of Affymetrix GeneChips","downloads_month":189,"downloads_total":53168,"page":"http://bioconductor.org/packages/release/bioc/html/bgx.html"},{"authors":"Holger Froehlich and contributions by Benedikt Zacher","name":"birte","license":"GPL (>= 2)","tags":["Microarray","Sequencing","GeneExpression","Transcription","Network","Bayesian","Regression","NetworkInference"],"description":"Expression levels of mRNA molecules are regulated by\ndifferent processes, comprising inhibition or activation by\ntranscription factors and post-transcriptional degradation by\nmicroRNAs. biRte uses regulatory networks of TFs, miRNAs and\npossibly other factors, together with mRNA, miRNA and other\navailable expression data to predict the relative influence of\na regulator on the expression of its target genes. Inference is\ndone in a Bayesian modeling framework using\nMarkov-Chain-Monte-Carlo. A special feature is the possibility\nfor follow-up network reverse engineering between active\nregulators.","downloads_month":171,"downloads_total":14992,"page":"http://bioconductor.org/packages/release/bioc/html/birte.html"},{"authors":"Min Li <limin@mail.csu.edu.cn>, Peng Ni <nipeng@csu.edu.cn>\nwith contributions from Zhihui Fei and Ping Huang.","name":"dSimer","license":"GPL (>= 2)","tags":["Software","Visualization","Network"],"description":"dSimer is an R package which provides computation of nine\nmethods for measuring disease-disease similarity, including a\nstandard cosine similarity measure and eight function-based\nmethods. The disease similarity matrix obtained from these nine\nmethods can be visualized through heatmap and network.\nBiological data widely used in disease-disease associations\nstudy are also provided by dSimer.","downloads_month":164,"downloads_total":8616,"page":"http://bioconductor.org/packages/release/bioc/html/dSimer.html"},{"authors":"Elsa Bernard, Laurent Jacob, Julien Mairal and Jean-Philippe\nVert","name":"flipflop","license":"GPL-3","tags":["RNASeq","RNASeqData","AlternativeSplicing","Regression"],"description":"Flipflop discovers which isoforms of a gene are expressed\nin a given sample together with their abundances, based on\nRNA-Seq read data. It takes an alignment file in SAM format as\ninput. It can also discover transcripts from several samples\nsimultaneously, increasing statistical power.","downloads_month":185,"downloads_total":27178,"page":"http://bioconductor.org/packages/release/bioc/html/flipflop.html"},{"authors":"R. Gentleman, F. Hahne, J. Kettman, N. Le Meur and N.\nGopalakrishnan","name":"flowQ","license":"Artistic-2.0","tags":["Infrastructure","FlowCytometry","CellBasedAssays"],"description":"Provides quality control and quality assessment tools for\nflow cytometry data.","downloads_month":50,"downloads_total":49868,"page":"http://bioconductor.org/packages/release/bioc/html/flowQ.html"},{"authors":"David L Gibbs","name":"ProCoNA","license":"GPL (>= 2)","tags":["GraphAndNetwork","Software","Proteomics"],"description":"Protein co-expression network construction using peptide\nlevel data, with statisical analysis. (Journal of Clinical\nBioinformatics 2013, 3:11 doi:10.1186/2043-9113-3-11)","downloads_month":35,"downloads_total":22042,"page":"http://bioconductor.org/packages/release/bioc/html/ProCoNA.html"}],"top":500},"settings":[]},"evals":[],"jsHooks":[]}</script>
</div>
<div id="dependency-graphs" class="section level1">
<h1 class="hasAnchor">
<a href="#dependency-graphs" class="anchor"></a>Dependency graphs</h1>
<p>The Bioconductor ecosystem is built around the concept of interoperability and dependencies. These interdependencies are available as part of the <code><a href="../reference/biocPkgList.html">biocPkgList()</a></code> output. The <code>BiocPkgTools</code> provides some convenience functions to convert package dependencies to R graphs. A modular approach leads to the following workflow.</p>
<ol style="list-style-type: decimal">
<li>Create a <code>data.frame</code> of dependencies using <code>buildPkgDependencyDataFrame</code>.</li>
<li>Create an <code>igraph</code> object from the dependency data frame using <code>buildPkgDependencyIgraph</code>
</li>
<li>Use native <code>igraph</code> functionality to perform arbitrary network operations. Convenience functions, <code>inducedSubgraphByPkgs</code> and <code>subgraphByDegree</code> are available.</li>
<li>Visualize with packages such as <em><a href="https://CRAN.R-project.org/package=visNetwork">visNetwork</a></em>.</li>
</ol>
<div id="working-with-dependency-graphs" class="section level2">
<h2 class="hasAnchor">
<a href="#working-with-dependency-graphs" class="anchor"></a>Working with dependency graphs</h2>
<p>A dependency graph for all of Bioconductor is a starting place.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(BiocPkgTools)</a>
<a class="sourceLine" id="cb13-2" data-line-number="2">dep_df =<span class="st"> </span><span class="kw"><a href="../reference/buildPkgDependencyDataFrame.html">buildPkgDependencyDataFrame</a></span>()</a>
<a class="sourceLine" id="cb13-3" data-line-number="3">g =<span class="st"> </span><span class="kw"><a href="../reference/buildPkgDependencyIgraph.html">buildPkgDependencyIgraph</a></span>(dep_df)</a>
<a class="sourceLine" id="cb13-4" data-line-number="4">g</a></code></pre></div>
<pre><code>## IGRAPH 0696809 DN-- 3011 24560 -- 
## + attr: name (v/c), edgetype (e/c)
## + edges from 0696809 (vertex names):
##  [1] a4       -&gt;a4Base        a4       -&gt;a4Preproc    
##  [3] a4       -&gt;a4Classif     a4       -&gt;a4Core       
##  [5] a4       -&gt;a4Reporting   a4Base   -&gt;methods      
##  [7] a4Base   -&gt;graphics      a4Base   -&gt;grid         
##  [9] a4Base   -&gt;Biobase       a4Base   -&gt;AnnotationDbi
## [11] a4Base   -&gt;annaffy       a4Base   -&gt;mpm          
## [13] a4Base   -&gt;genefilter    a4Base   -&gt;limma        
## [15] a4Base   -&gt;multtest      a4Base   -&gt;glmnet       
## + ... omitted several edges</code></pre>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(igraph)</a>
<a class="sourceLine" id="cb15-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/igraph/topics/V">V</a></span>(g))</a></code></pre></div>
<pre><code>## + 6/3011 vertices, named, from 0696809:
## [1] a4          a4Base      a4Classif   a4Core      a4Preproc   a4Reporting</code></pre>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/igraph/topics/E">E</a></span>(g))</a></code></pre></div>
<pre><code>## + 6/24560 edges from 0696809 (vertex names):
## [1] a4    -&gt;a4Base      a4    -&gt;a4Preproc   a4    -&gt;a4Classif  
## [4] a4    -&gt;a4Core      a4    -&gt;a4Reporting a4Base-&gt;methods</code></pre>
<p>See <code>inducedSubgraphByPkgs</code> and <code>subgraphByDegree</code> to produce subgraphs based on a subset of packages.</p>
<p>See the igraph documentation for more detail on graph analytics, setting vertex and edge attributes, and advanced subsetting.</p>
</div>
<div id="graph-visualization" class="section level2">
<h2 class="hasAnchor">
<a href="#graph-visualization" class="anchor"></a>Graph visualization</h2>
<p>The visNetwork package is a nice interactive visualization tool that implements graph plotting in a browser. It can be integrated into shiny applications. Interactive graphs can also be included in Rmarkdown documents (see vignette)</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" data-line-number="1">igraph_network =<span class="st"> </span><span class="kw"><a href="../reference/buildPkgDependencyIgraph.html">buildPkgDependencyIgraph</a></span>(<span class="kw"><a href="../reference/buildPkgDependencyDataFrame.html">buildPkgDependencyDataFrame</a></span>())</a></code></pre></div>
<p>The full dependency graph is really not that informative to look at, though doing so is possible. A common use case is to visualize the graph of dependencies “centered” on a package of interest. In this case, I will focus on the <em><a href="https://bioconductor.org/packages/3.9/GEOquery">GEOquery</a></em> package.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb20-1" data-line-number="1">igraph_geoquery_network =<span class="st"> </span><span class="kw"><a href="../reference/subgraphByDegree.html">subgraphByDegree</a></span>(igraph_network, <span class="st">"GEOquery"</span>)</a></code></pre></div>
<p>The <code><a href="../reference/subgraphByDegree.html">subgraphByDegree()</a></code> function returns all nodes and connections within <code>degree</code> of the named package; the default <code>degree</code> is <code>1</code>.</p>
<p>The visNework package can plot <code>igraph</code> objects directly, but more flexibility is offered by first converting the graph to visNetwork form.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb21-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(visNetwork)</a>
<a class="sourceLine" id="cb21-2" data-line-number="2">data &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork-igraph">toVisNetworkData</a></span>(igraph_geoquery_network)</a></code></pre></div>
<p>The next few code chunks highlight just a few examples of the visNetwork capabilities, starting with a basic plot.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork">visNetwork</a></span>(<span class="dt">nodes =</span> data<span class="op">$</span>nodes, <span class="dt">edges =</span> data<span class="op">$</span>edges, <span class="dt">height =</span> <span class="st">"500px"</span>)</a></code></pre></div>
<div id="htmlwidget-6ba51b1b39faf9896831" style="width:700px;height:500px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-6ba51b1b39faf9896831">{"x":{"nodes":{"id":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"],"label":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"]},"edges":{"from":["bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","Biobase","Biobase","BiocGenerics","BiocGenerics","BiocGenerics","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","ChIPXpress","ChIPXpress","ChIPXpress","coexnet","coexnet","coexnet","coexnet","coexnet","coexnet","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","diffcoexp","diffcoexp","DrugVsDisease","DrugVsDisease","DrugVsDisease","dyebias","dyebias","dyebias","dyebias","EGAD","EGAD","EGAD","EGAD","EGAD","EGAD","ELBOW","ELBOW","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","fgsea","fgsea","fgsea","fgsea","fgsea","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GeneExpressionSignature","GeneExpressionSignature","GeneExpressionSignature","GEOmetadb","GEOmetadb","GEOmetadb","GEOmetadb","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","limma","limma","MACPET","MACPET","MACPET","MACPET","minfi","minfi","minfi","minfi","minfi","minfi","minfi","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","omicsPrint","omicsPrint","omicsPrint","omicsPrint","omicsPrint","pathprint","PCAtools","PCAtools","PCAtools","PCAtools","PGSEA","PGSEA","PGSEA","PGSEA","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","recount","recount","recount","recount","recount","RGSEA","RGSEA","RGSEA","Rnits","Rnits","Rnits","Rnits","Rnits","runibic","runibic","runibic","runibic","runibic","SCAN.UPC","SCAN.UPC","SCAN.UPC","skewr","skewr","skewr","SRAdb","TargetScore","TargetScore","zFPKM","zFPKM","zFPKM","zFPKM","zFPKM","AUCell","AUCell","AUCell","AUCell","AUCell","AUCell","multiClust","multiClust","multiClust","multiClust"],"to":["Biobase","BiocGenerics","BiocGenerics","GEOquery","minfi","methods","BiocGenerics","methods","Biobase","methods","methods","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","rmarkdown","knitr","dplyr","testthat","BiocGenerics","GEOquery","BiocGenerics","GEOquery","limma","Biobase","GEOquery","limma","methods","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","Biobase","GEOquery","rmarkdown","knitr","dplyr","testthat","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","rmarkdown","knitr","magrittr","testthat","Biobase","GEOquery","PGSEA","GEOquery","rmarkdown","knitr","dplyr","Biobase","BiocGenerics","limma","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr","Biobase","GEOquery","methods","rmarkdown","knitr","Biobase","methods","GEOquery","methods","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","methods","rmarkdown","knitr","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","limma","minfi","methods","rmarkdown","knitr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","GEOquery","Biobase","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","Biobase","fgsea","GEOquery","limma","methods","rmarkdown","knitr","httr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","knitr","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","GEOquery","minfi","knitr","GEOquery","Biobase","GEOquery","GEOquery","limma","knitr","tidyr","dplyr","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","knitr"],"edgetype":["Depends","Suggests","Imports","Imports","Depends","Depends","Depends","Imports","Suggests","Imports","Depends","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Depends","Depends","Depends","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Suggests","Depends","Suggests","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Imports","Suggests","Imports","Imports","Suggests","Depends","Imports","Imports","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Depends","Imports","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Depends","Suggests","Suggests","Depends","Imports","Imports","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Imports","Suggests","Suggests","Imports","Depends","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false}},"groups":null,"width":null,"height":"500px","idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)"},"evals":[],"jsHooks":[]}</script><p>For fun, we can watch the graph stabilize during drawing, best viewed interactively.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb23-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork">visNetwork</a></span>(<span class="dt">nodes =</span> data<span class="op">$</span>nodes, <span class="dt">edges =</span> data<span class="op">$</span>edges, <span class="dt">height =</span> <span class="st">"500px"</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-2" data-line-number="2"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visPhysics">visPhysics</a></span>(<span class="dt">stabilization=</span><span class="ot">FALSE</span>)</a></code></pre></div>
<div id="htmlwidget-f952eb042b255a8414b0" style="width:700px;height:500px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-f952eb042b255a8414b0">{"x":{"nodes":{"id":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"],"label":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"]},"edges":{"from":["bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","Biobase","Biobase","BiocGenerics","BiocGenerics","BiocGenerics","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","ChIPXpress","ChIPXpress","ChIPXpress","coexnet","coexnet","coexnet","coexnet","coexnet","coexnet","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","diffcoexp","diffcoexp","DrugVsDisease","DrugVsDisease","DrugVsDisease","dyebias","dyebias","dyebias","dyebias","EGAD","EGAD","EGAD","EGAD","EGAD","EGAD","ELBOW","ELBOW","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","fgsea","fgsea","fgsea","fgsea","fgsea","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GeneExpressionSignature","GeneExpressionSignature","GeneExpressionSignature","GEOmetadb","GEOmetadb","GEOmetadb","GEOmetadb","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","limma","limma","MACPET","MACPET","MACPET","MACPET","minfi","minfi","minfi","minfi","minfi","minfi","minfi","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","omicsPrint","omicsPrint","omicsPrint","omicsPrint","omicsPrint","pathprint","PCAtools","PCAtools","PCAtools","PCAtools","PGSEA","PGSEA","PGSEA","PGSEA","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","recount","recount","recount","recount","recount","RGSEA","RGSEA","RGSEA","Rnits","Rnits","Rnits","Rnits","Rnits","runibic","runibic","runibic","runibic","runibic","SCAN.UPC","SCAN.UPC","SCAN.UPC","skewr","skewr","skewr","SRAdb","TargetScore","TargetScore","zFPKM","zFPKM","zFPKM","zFPKM","zFPKM","AUCell","AUCell","AUCell","AUCell","AUCell","AUCell","multiClust","multiClust","multiClust","multiClust"],"to":["Biobase","BiocGenerics","BiocGenerics","GEOquery","minfi","methods","BiocGenerics","methods","Biobase","methods","methods","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","rmarkdown","knitr","dplyr","testthat","BiocGenerics","GEOquery","BiocGenerics","GEOquery","limma","Biobase","GEOquery","limma","methods","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","Biobase","GEOquery","rmarkdown","knitr","dplyr","testthat","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","rmarkdown","knitr","magrittr","testthat","Biobase","GEOquery","PGSEA","GEOquery","rmarkdown","knitr","dplyr","Biobase","BiocGenerics","limma","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr","Biobase","GEOquery","methods","rmarkdown","knitr","Biobase","methods","GEOquery","methods","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","methods","rmarkdown","knitr","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","limma","minfi","methods","rmarkdown","knitr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","GEOquery","Biobase","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","Biobase","fgsea","GEOquery","limma","methods","rmarkdown","knitr","httr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","knitr","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","GEOquery","minfi","knitr","GEOquery","Biobase","GEOquery","GEOquery","limma","knitr","tidyr","dplyr","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","knitr"],"edgetype":["Depends","Suggests","Imports","Imports","Depends","Depends","Depends","Imports","Suggests","Imports","Depends","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Depends","Depends","Depends","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Suggests","Depends","Suggests","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Imports","Suggests","Imports","Imports","Suggests","Depends","Imports","Imports","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Depends","Imports","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Depends","Suggests","Suggests","Depends","Imports","Imports","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Imports","Suggests","Suggests","Imports","Depends","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false},"physics":{"stabilization":false}},"groups":null,"width":null,"height":"500px","idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)"},"evals":[],"jsHooks":[]}</script><p>Add arrows and colors to better capture dependencies.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb24-1" data-line-number="1">data<span class="op">$</span>edges<span class="op">$</span>color=<span class="st">'lightblue'</span></a>
<a class="sourceLine" id="cb24-2" data-line-number="2">data<span class="op">$</span>edges[data<span class="op">$</span>edges<span class="op">$</span>edgetype<span class="op">==</span><span class="st">'Imports'</span>,<span class="st">'color'</span>]=<span class="st"> 'red'</span></a>
<a class="sourceLine" id="cb24-3" data-line-number="3">data<span class="op">$</span>edges[data<span class="op">$</span>edges<span class="op">$</span>edgetype<span class="op">==</span><span class="st">'Depends'</span>,<span class="st">'color'</span>]=<span class="st"> 'green'</span></a>
<a class="sourceLine" id="cb24-4" data-line-number="4"></a>
<a class="sourceLine" id="cb24-5" data-line-number="5"><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork">visNetwork</a></span>(<span class="dt">nodes =</span> data<span class="op">$</span>nodes, <span class="dt">edges =</span> data<span class="op">$</span>edges, <span class="dt">height =</span> <span class="st">"500px"</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb24-6" data-line-number="6"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visEdges">visEdges</a></span>(<span class="dt">arrows=</span><span class="st">'from'</span>) </a></code></pre></div>
<div id="htmlwidget-263922d197f8a7212c95" style="width:700px;height:500px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-263922d197f8a7212c95">{"x":{"nodes":{"id":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"],"label":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"]},"edges":{"from":["bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","Biobase","Biobase","BiocGenerics","BiocGenerics","BiocGenerics","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","ChIPXpress","ChIPXpress","ChIPXpress","coexnet","coexnet","coexnet","coexnet","coexnet","coexnet","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","diffcoexp","diffcoexp","DrugVsDisease","DrugVsDisease","DrugVsDisease","dyebias","dyebias","dyebias","dyebias","EGAD","EGAD","EGAD","EGAD","EGAD","EGAD","ELBOW","ELBOW","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","fgsea","fgsea","fgsea","fgsea","fgsea","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GeneExpressionSignature","GeneExpressionSignature","GeneExpressionSignature","GEOmetadb","GEOmetadb","GEOmetadb","GEOmetadb","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","limma","limma","MACPET","MACPET","MACPET","MACPET","minfi","minfi","minfi","minfi","minfi","minfi","minfi","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","omicsPrint","omicsPrint","omicsPrint","omicsPrint","omicsPrint","pathprint","PCAtools","PCAtools","PCAtools","PCAtools","PGSEA","PGSEA","PGSEA","PGSEA","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","recount","recount","recount","recount","recount","RGSEA","RGSEA","RGSEA","Rnits","Rnits","Rnits","Rnits","Rnits","runibic","runibic","runibic","runibic","runibic","SCAN.UPC","SCAN.UPC","SCAN.UPC","skewr","skewr","skewr","SRAdb","TargetScore","TargetScore","zFPKM","zFPKM","zFPKM","zFPKM","zFPKM","AUCell","AUCell","AUCell","AUCell","AUCell","AUCell","multiClust","multiClust","multiClust","multiClust"],"to":["Biobase","BiocGenerics","BiocGenerics","GEOquery","minfi","methods","BiocGenerics","methods","Biobase","methods","methods","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","rmarkdown","knitr","dplyr","testthat","BiocGenerics","GEOquery","BiocGenerics","GEOquery","limma","Biobase","GEOquery","limma","methods","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","Biobase","GEOquery","rmarkdown","knitr","dplyr","testthat","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","rmarkdown","knitr","magrittr","testthat","Biobase","GEOquery","PGSEA","GEOquery","rmarkdown","knitr","dplyr","Biobase","BiocGenerics","limma","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr","Biobase","GEOquery","methods","rmarkdown","knitr","Biobase","methods","GEOquery","methods","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","methods","rmarkdown","knitr","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","limma","minfi","methods","rmarkdown","knitr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","GEOquery","Biobase","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","Biobase","fgsea","GEOquery","limma","methods","rmarkdown","knitr","httr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","knitr","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","GEOquery","minfi","knitr","GEOquery","Biobase","GEOquery","GEOquery","limma","knitr","tidyr","dplyr","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","knitr"],"edgetype":["Depends","Suggests","Imports","Imports","Depends","Depends","Depends","Imports","Suggests","Imports","Depends","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Depends","Depends","Depends","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Suggests","Depends","Suggests","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Imports","Suggests","Imports","Imports","Suggests","Depends","Imports","Imports","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Depends","Imports","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Depends","Suggests","Suggests","Depends","Imports","Imports","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Imports","Suggests","Suggests","Imports","Depends","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests"],"color":["green","lightblue","red","red","green","green","green","red","lightblue","red","green","red","lightblue","red","lightblue","lightblue","lightblue","red","lightblue","red","red","lightblue","red","red","red","lightblue","red","red","red","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","red","green","green","green","lightblue","lightblue","lightblue","red","red","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","red","red","red","lightblue","lightblue","red","lightblue","green","lightblue","green","green","lightblue","lightblue","lightblue","green","lightblue","red","green","lightblue","lightblue","red","red","red","red","lightblue","red","red","lightblue","green","red","red","lightblue","lightblue","lightblue","red","red","red","lightblue","red","red","green","red","red","green","lightblue","lightblue","red","red","red","lightblue","lightblue","lightblue","green","red","lightblue","red","lightblue","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","green","lightblue","lightblue","green","red","red","red","red","red","lightblue","lightblue","red","lightblue","red","red","lightblue","lightblue","lightblue","red","lightblue","lightblue","green","lightblue","green","green","lightblue","lightblue","red","lightblue","lightblue","red","green","green","red","lightblue","red","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","red","red","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false},"edges":{"arrows":"from"}},"groups":null,"width":null,"height":"500px","idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)"},"evals":[],"jsHooks":[]}</script><p>Add a legend.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb25-1" data-line-number="1">ledges &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/data.frame">data.frame</a></span>(<span class="dt">color =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"green"</span>, <span class="st">"lightblue"</span>, <span class="st">"red"</span>),</a>
<a class="sourceLine" id="cb25-2" data-line-number="2">  <span class="dt">label =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"Depends"</span>, <span class="st">"Suggests"</span>, <span class="st">"Imports"</span>), <span class="dt">arrows =</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"from"</span>, <span class="st">"from"</span>, <span class="st">"from"</span>))</a>
<a class="sourceLine" id="cb25-3" data-line-number="3"><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork">visNetwork</a></span>(<span class="dt">nodes =</span> data<span class="op">$</span>nodes, <span class="dt">edges =</span> data<span class="op">$</span>edges, <span class="dt">height =</span> <span class="st">"500px"</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb25-4" data-line-number="4"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visEdges">visEdges</a></span>(<span class="dt">arrows=</span><span class="st">'from'</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb25-5" data-line-number="5"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visLegend">visLegend</a></span>(<span class="dt">addEdges=</span>ledges)</a></code></pre></div>
<div id="htmlwidget-792b6bd06647a7255c74" style="width:700px;height:500px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-792b6bd06647a7255c74">{"x":{"nodes":{"id":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"],"label":["bigmelon","Biobase","BiocGenerics","CAMTHC","ChIPXpress","coexnet","crossmeta","ctsGE","diffcoexp","DrugVsDisease","dyebias","EGAD","ELBOW","EpiDISH","fgsea","GAPGOM","GeneExpressionSignature","GEOmetadb","GEOquery","GSEABenchmarkeR","limma","MACPET","minfi","MoonlightR","MultiDataSet","omicsPrint","pathprint","PCAtools","PGSEA","phantasus","recount","RGSEA","Rnits","runibic","SCAN.UPC","skewr","SRAdb","TargetScore","zFPKM","AUCell","multiClust","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr"]},"edges":{"from":["bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","bigmelon","Biobase","Biobase","BiocGenerics","BiocGenerics","BiocGenerics","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","CAMTHC","ChIPXpress","ChIPXpress","ChIPXpress","coexnet","coexnet","coexnet","coexnet","coexnet","coexnet","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","crossmeta","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","ctsGE","diffcoexp","diffcoexp","DrugVsDisease","DrugVsDisease","DrugVsDisease","dyebias","dyebias","dyebias","dyebias","EGAD","EGAD","EGAD","EGAD","EGAD","EGAD","ELBOW","ELBOW","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","EpiDISH","fgsea","fgsea","fgsea","fgsea","fgsea","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GAPGOM","GeneExpressionSignature","GeneExpressionSignature","GeneExpressionSignature","GEOmetadb","GEOmetadb","GEOmetadb","GEOmetadb","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GEOquery","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","GSEABenchmarkeR","limma","limma","MACPET","MACPET","MACPET","MACPET","minfi","minfi","minfi","minfi","minfi","minfi","minfi","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MoonlightR","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","MultiDataSet","omicsPrint","omicsPrint","omicsPrint","omicsPrint","omicsPrint","pathprint","PCAtools","PCAtools","PCAtools","PCAtools","PGSEA","PGSEA","PGSEA","PGSEA","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","phantasus","recount","recount","recount","recount","recount","RGSEA","RGSEA","RGSEA","Rnits","Rnits","Rnits","Rnits","Rnits","runibic","runibic","runibic","runibic","runibic","SCAN.UPC","SCAN.UPC","SCAN.UPC","skewr","skewr","skewr","SRAdb","TargetScore","TargetScore","zFPKM","zFPKM","zFPKM","zFPKM","zFPKM","AUCell","AUCell","AUCell","AUCell","AUCell","AUCell","multiClust","multiClust","multiClust","multiClust"],"to":["Biobase","BiocGenerics","BiocGenerics","GEOquery","minfi","methods","BiocGenerics","methods","Biobase","methods","methods","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","rmarkdown","knitr","dplyr","testthat","BiocGenerics","GEOquery","BiocGenerics","GEOquery","limma","Biobase","GEOquery","limma","methods","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","GEOquery","limma","Biobase","GEOquery","rmarkdown","knitr","dplyr","testthat","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","rmarkdown","knitr","magrittr","testthat","Biobase","GEOquery","PGSEA","GEOquery","rmarkdown","knitr","dplyr","Biobase","BiocGenerics","limma","methods","rmarkdown","knitr","tidyr","dplyr","magrittr","httr","testthat","xml2","readr","covr","Biobase","GEOquery","methods","rmarkdown","knitr","Biobase","methods","GEOquery","methods","rmarkdown","knitr","Biobase","BiocGenerics","GEOquery","limma","methods","rmarkdown","knitr","Biobase","GEOquery","limma","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","limma","minfi","methods","rmarkdown","knitr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","GEOquery","Biobase","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","Biobase","fgsea","GEOquery","limma","methods","rmarkdown","knitr","httr","testthat","GEOquery","methods","rmarkdown","knitr","testthat","BiocGenerics","GEOquery","knitr","Biobase","GEOquery","limma","methods","knitr","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","GEOquery","methods","GEOquery","minfi","knitr","GEOquery","Biobase","GEOquery","GEOquery","limma","knitr","tidyr","dplyr","Biobase","GEOquery","methods","rmarkdown","knitr","testthat","Biobase","BiocGenerics","GEOquery","knitr"],"edgetype":["Depends","Suggests","Imports","Imports","Depends","Depends","Depends","Imports","Suggests","Imports","Depends","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Imports","Depends","Depends","Depends","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Suggests","Depends","Suggests","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Imports","Suggests","Imports","Imports","Suggests","Depends","Imports","Imports","Suggests","Suggests","Suggests","Imports","Imports","Imports","Suggests","Imports","Imports","Depends","Imports","Imports","Depends","Suggests","Suggests","Imports","Imports","Imports","Suggests","Suggests","Suggests","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Depends","Suggests","Suggests","Depends","Imports","Imports","Imports","Imports","Imports","Suggests","Suggests","Imports","Suggests","Imports","Imports","Suggests","Suggests","Suggests","Imports","Suggests","Suggests","Depends","Suggests","Depends","Depends","Suggests","Suggests","Imports","Suggests","Suggests","Imports","Depends","Depends","Imports","Suggests","Imports","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Imports","Imports","Suggests","Suggests","Imports","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests","Suggests"],"color":["green","lightblue","red","red","green","green","green","red","lightblue","red","green","red","lightblue","red","lightblue","lightblue","lightblue","red","lightblue","red","red","lightblue","red","red","red","lightblue","red","red","red","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","red","green","green","green","lightblue","lightblue","lightblue","red","red","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","red","red","red","lightblue","lightblue","red","lightblue","green","lightblue","green","green","lightblue","lightblue","lightblue","green","lightblue","red","green","lightblue","lightblue","red","red","red","red","lightblue","red","red","lightblue","green","red","red","lightblue","lightblue","lightblue","red","red","red","lightblue","red","red","green","red","red","green","lightblue","lightblue","red","red","red","lightblue","lightblue","lightblue","green","red","lightblue","red","lightblue","red","lightblue","lightblue","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","green","lightblue","lightblue","green","red","red","red","red","red","lightblue","lightblue","red","lightblue","red","red","lightblue","lightblue","lightblue","red","lightblue","lightblue","green","lightblue","green","green","lightblue","lightblue","red","lightblue","lightblue","red","green","green","red","lightblue","red","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","red","red","lightblue","lightblue","red","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false},"edges":{"arrows":"from"}},"groups":null,"width":null,"height":"500px","idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","legend":{"width":0.2,"useGroups":true,"position":"left","ncol":1,"stepX":100,"stepY":100,"zoom":true,"edges":{"color":["green","lightblue","red"],"label":["Depends","Suggests","Imports"],"arrows":["from","from","from"]},"edgesToDataframe":true}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="integration-with-biocviews" class="section level2">
<h2 class="hasAnchor">
<a href="#integration-with-biocviews" class="anchor"></a>Integration with <em><a href="https://bioconductor.org/packages/3.9/BiocViews">BiocViews</a></em>
</h2>
<p>[Work in progress]</p>
<p>The <em><a href="https://bioconductor.org/packages/3.9/biocViews">biocViews</a></em> package is a small ontology of terms describing Bioconductor packages. This is a work-in-progress section, but here is a small example of plotting the biocViews graph.</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb26-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(biocViews)</a>
<a class="sourceLine" id="cb26-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/data">data</a></span>(biocViewsVocab)</a>
<a class="sourceLine" id="cb26-3" data-line-number="3">biocViewsVocab</a></code></pre></div>
<pre><code>## A graphNEL graph with directed edges
## Number of Nodes = 476 
## Number of Edges = 475</code></pre>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb28-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(igraph)</a>
<a class="sourceLine" id="cb28-2" data-line-number="2">g =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/igraph/topics/graph_from_graphnel">igraph.from.graphNEL</a></span>(biocViewsVocab)</a>
<a class="sourceLine" id="cb28-3" data-line-number="3"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(visNetwork)</a>
<a class="sourceLine" id="cb28-4" data-line-number="4">gv =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork-igraph">toVisNetworkData</a></span>(g)</a>
<a class="sourceLine" id="cb28-5" data-line-number="5"><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNetwork">visNetwork</a></span>(gv<span class="op">$</span>nodes, gv<span class="op">$</span>edges, <span class="dt">width=</span><span class="st">"100%"</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb28-6" data-line-number="6"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visIgraphLayout">visIgraphLayout</a></span>(<span class="dt">layout =</span> <span class="st">"layout_as_tree"</span>, <span class="dt">circular=</span><span class="ot">TRUE</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb28-7" data-line-number="7"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visNodes">visNodes</a></span>(<span class="dt">size=</span><span class="dv">20</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb28-8" data-line-number="8"><span class="st">    </span><span class="kw"><a href="https://www.rdocumentation.org/packages/visNetwork/topics/visPhysics">visPhysics</a></span>(<span class="dt">stabilization=</span><span class="ot">FALSE</span>)</a></code></pre></div>
<div id="htmlwidget-0ed3c69476811f6cde3f" style="width:100%;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-0ed3c69476811f6cde3f">{"x":{"nodes":{"id":["BiocViews","Software","AnnotationData","ExperimentData","Workflow","AssayDomain","Technology","ResearchField","BiologicalQuestion","WorkflowStep","StatisticalMethod","Infrastructure","aCGH","CellBasedAssays","ChIPchip","CopyNumberVariation","CpGIsland","DNAMethylation","ExonArray","GeneExpression","GeneticVariability","SNP","Transcription","Sequencing","Microarray","FlowCytometry","MassSpectrometry","qPCR","MicrotitrePlateAssay","SAGE","CRISPR","SingleCell","ddPCR","ATACSeq","DNASeq","DNaseSeq","RiboSeq","RNASeq","ChIPSeq","RIPSeq","MethylSeq","ExomeSeq","miRNA","SangerSeq","SmallRNA","Microbiome","WholeGenome","DenovoGenome","TargetedResequencing","DenovoTranscriptome","MicrobialStrain","HiC","PooledScreens","MNaseSeq","MultiChannel","OneChannel","TwoChannel","MethylationArray","GenotypingArray","MicroRNAArray","mRNAMicroarray","ChipOnChip","ReversePhaseProteinArray","TissueMicroarray","ProprietaryPlatforms","ImagingMassSpectrometry","CellBiology","Genetics","Metabolomics","Metagenomics","Proteomics","Lipidomics","Epigenetics","Pharmacogenomics","Pharmacogenetics","Cheminformatics","StructuralGenomics","StructuralPrediction","Biophysics","MathematicalBiology","BiomedicalInformatics","ComparativeGenomics","FunctionalGenomics","SystemsBiology","ComputationalChemistry","Agroinformatics","Transcriptomics","ImmunoOncology","AlternativeSplicing","Coverage","DemethylateRegionDetection","DenovoAssembler","DifferentialExpression","DifferentialMethylation","DifferentialPeakCalling","DifferentialSplicing","DNA3DStructure","DriverMutation","FunctionalPrediction","GeneFusionDetection","GenePrediction","GeneRegulation","GeneSetEnrichment","GeneSignaling","GeneTarget","GenomeAssembly","GenomeWideAssociation","GenomicVariation","GenomeAnnotation","GermlineMutation","HistoneModification","IndelDetection","LinkageDisequilibrium","MetagenomeAssembly","MicrosatelliteDetection","MotifAnnotation","MotifDiscovery","NetworkEnrichment","NetworkInference","NucleosomePositioning","PeakDetection","QuantitativeTrailLocus","Scaffolding","SequenceMatching","SomaticMutation","SplicedAlignment","StructuralVariation","TranscriptomeVariant","VariantAnnotation","VariantDetection","ExperimentalDesign","Alignment","Annotation","BatchEffect","MultipleComparison","Normalization","Pathways","Preprocessing","QualityControl","ReportWriting","Visualization","GenomeBrowsers","MultipleSequenceAlignment","GO","KEGG","Reactome","BioCarta","NCINatureCurated","Network","Bayesian","Classification","Clustering","DecisionTree","DimensionReduction","FeatureExtraction","GraphAndNetwork","HiddenMarkovModel","MultidimensionalScaling","NeuralNetwork","PatternLogic","PrincipalComponent","Regression","StructuralEquationModels","SupportVectorMachine","Survival","TimeCourse","ThirdPartyClient","DataImport","DataRepresentation","GUI","Organism","ChipManufacturer","CustomCDF","CustomArray","CustomDBSchema","FunctionalAnnotation","SequenceAnnotation","ChipName","PackageType","Anopheles_gambiae","Apis_mellifera","Arabidopsis_lyrata","Arabidopsis_thaliana","Asparagus_officinalis","Bacillus_subtilis","Bos_taurus","Caenorhabditis_elegans","Canis_familiaris","Cicer_arietinum","Ciona_intestinalis","Danio_rerio","Drosophila_melanogaster","Eremothecium_gossypii","Escherichia_coli","Gallus_gallus","Gasterosteus_aculeatus","Glycine_max","Homo_sapiens","Hordeum_vulgare","Kluyveromyces_lactis","Macaca_fascicularis","Macaca_mulatta","Magnaporthe_grisea","Medicago_truncatula","Mus_musculus","Neurospora_crassa","Oncorhynchus_mykiss","Oryza_sativa","Pan_troglodytes","Plasmodium_falciparum","Pseudomonas_aeruginosa","Rattus_norvegicus","Saccharomyces_cerevisiae","Saccharum_officinarum","Schizosaccharomyces_pombe","Staphylococcus_aureus","Sus_scrofa","Taeniopygia_guttata","Toxoplasma_gondii","Triticum_aestivum","Vitis_vinifera","Xenopus_laevis","Xenopus_tropicalis","Zea_mays","AffymetrixChip","AgilentChip","ClonetechChip","GEChip","INDACChip","IlluminaChip","QiagenChip","RNG_MRCChip","RocheChip","UniversityHealthNetwork","CodelinkChip","GACustomCDF","MBNICustomCDF","GeneCardsCustomSchema","GenomicSequence","adme16cod","ag","ath1121501","celegans","drosgenome1","drosophila2","h10kcod","h20kcod","hcg110","hgfocus","hgu133a2","hgu133a","hgu133b","hgu133plus2","hgu95a","hgu95av2","hgu95b","hgu95c","hgu95d","hgu95e","hguatlas13k","hgug4100a","hgug4101a","hgug4110b","hgug4111a","hgug4112a","hguqiagenv3","hi16cod","hs25kresogen","hu35ksuba","hu35ksubb","hu35ksubc","hu35ksubd","hu6800","HuO22","hwgcod","indac","illuminaHumanv1","illuminaHumanv2","illuminaMousev1","illuminaMousev1p1","illuminaRatv1","JazaerimetaData","lumiHumanV1","lumiMouseV1","lumiHumanV2","lumiRatV1","m10kcod","m20kcod","mi16cod","mm24kresogen","mgu74a","mgu74av2","mgu74b","mgu74bv2","mgu74c","mgu74cv2","mguatlas5k","mgug4121a","mgug4122a","moe430a","moe430b","mouse4302","mouse430a2","mpedbarray","mu11ksuba","mu11ksubb","mu19ksuba","mu19ksubb","mu19ksubc","Mu15v1","Mu22v3","mwgcod","Norway981","OperonHumanV3","pedbarrayv9","pedbarrayv10","PartheenMetaData","r10kcod","rae230a","rae230b","rat2302","rgu34a","rgu34b","rgu34c","rgug4130a","ri16cod","rnu34","Roberts2005Annotation","rtu34","rwgcod","SHDZ","u133x3p","xenopuslaevis","yeast2","ygs98","zebrafish","hcgi12k","hcgi8k","BSgenome","cdf","ChipDb","db0","InparanoidDb","OrganismDb","OrgDb","PolyPhen","probe","SIFT","SNPlocs","XtraSNPlocs","TxDb","MeSHDb","FRMA","AnnotationHub","EuPathDB","ReproducibleResearch","SpecimenSource","OrganismData","DiseaseModel","TechnologyData","AssayDomainData","RepositoryData","PackageTypeData","Tissue","Proteome","Genome","StemCell","CellCulture","Germline","Somatic","Anopheles_gambiae_Data","Apis_mellifera_Data","Arabidopsis_lyrata_Data","Arabidopsis_thaliana_Data","Bacillus_subtilis_Data","Bos_taurus_Data","Caenorhabditis_elegans_Data","Canis_familiaris_Data","Ciona_intestinalis_Data","Danio_rerio_Data","Drosophila_melanogaster_Data","Eremothecium_gossypii_Data","Escherichia_coli_Data","Gallus_gallus_Data","Gasterosteus_aculeatus_Data","Glycine_max_Data","Homo_sapiens_Data","Hordeum_vulgare_Data","Kluyveromyces_lactis_Data","Macaca_mulatta_Data","Magnaporthe_grisea_Data","Medicago_truncatul_Dataa","Mus_musculus_Data","Neurospora_crassa_Data","Oncorhynchus_mykiss_Data","Oryza_sativa_Data","Pan_troglodytes_Data","Plasmodium_falciparum_Data","Pseudomonas_aeruginosa_Data","Rattus_norvegicus_Data","Saccharomyces_cerevisiae_Data","Saccharum_officinarum_Data","Schizosaccharomyces_pombe_Data","Staphylococcus_aureus_Data","Sus_scrofa_Data","Taeniopygia_guttata_Data","Triticum_aestivum_Data","Vitis_vinifera_Data","Xenopus_laevis_Data","Xenopus_tropicalis_Data","Zea_mays_Data","CancerData","HIVData","COPDData","BreastCancerData","ColonCancerData","KidneyCancerData","LeukemiaCancerData","LungCancerData","OvarianCancerData","ProstateCancerData","FlowCytometryData","HighThroughputImagingData","MassSpectrometryData","qPCRData","MicrotitrePlateAssayData","SAGEData","CGHData","SequencingData","MicroarrayData","ImagingMassSpectrometryData","DNASeqData","RNASeqData","ChIPSeqData","RIPSeqData","MethylSeqData","ExomeSeqData","miRNAData","SangerSeqData","SmallRNAData","MicrobiomeData","SingleCellData","MultiChannelData","OneChannelData","TwoChannelData","MethylationArrayData","GenotypingArrayData","MicroRNAArrayData","mRNAArrayData","ChipOnChipData","ReversePhaseProteinArrayData","TissueMicroarrayData","ProprietaryPlatformsData","CopyNumberVariationData","CpGIslandData","SNPData","ExpressionData","HapMap","GEO","ArrayExpress","NCI","PathwayInteractionDatabase","Project1000genomes","ENCODE","ExperimentHub","ImmunoOncologyData","BasicWorkflow","AnnotationWorkflow","GeneExpressionWorkflow","SingleCellWorkflow","GenomicVariantsWorkflow","EpigeneticsWorkflow","ProteomicsWorkflow","ResourceQueryingWorkflow","DifferentialSplicingWorkflow","ImmunoOncologyWorkflow"],"label":["BiocViews","Software","AnnotationData","ExperimentData","Workflow","AssayDomain","Technology","ResearchField","BiologicalQuestion","WorkflowStep","StatisticalMethod","Infrastructure","aCGH","CellBasedAssays","ChIPchip","CopyNumberVariation","CpGIsland","DNAMethylation","ExonArray","GeneExpression","GeneticVariability","SNP","Transcription","Sequencing","Microarray","FlowCytometry","MassSpectrometry","qPCR","MicrotitrePlateAssay","SAGE","CRISPR","SingleCell","ddPCR","ATACSeq","DNASeq","DNaseSeq","RiboSeq","RNASeq","ChIPSeq","RIPSeq","MethylSeq","ExomeSeq","miRNA","SangerSeq","SmallRNA","Microbiome","WholeGenome","DenovoGenome","TargetedResequencing","DenovoTranscriptome","MicrobialStrain","HiC","PooledScreens","MNaseSeq","MultiChannel","OneChannel","TwoChannel","MethylationArray","GenotypingArray","MicroRNAArray","mRNAMicroarray","ChipOnChip","ReversePhaseProteinArray","TissueMicroarray","ProprietaryPlatforms","ImagingMassSpectrometry","CellBiology","Genetics","Metabolomics","Metagenomics","Proteomics","Lipidomics","Epigenetics","Pharmacogenomics","Pharmacogenetics","Cheminformatics","StructuralGenomics","StructuralPrediction","Biophysics","MathematicalBiology","BiomedicalInformatics","ComparativeGenomics","FunctionalGenomics","SystemsBiology","ComputationalChemistry","Agroinformatics","Transcriptomics","ImmunoOncology","AlternativeSplicing","Coverage","DemethylateRegionDetection","DenovoAssembler","DifferentialExpression","DifferentialMethylation","DifferentialPeakCalling","DifferentialSplicing","DNA3DStructure","DriverMutation","FunctionalPrediction","GeneFusionDetection","GenePrediction","GeneRegulation","GeneSetEnrichment","GeneSignaling","GeneTarget","GenomeAssembly","GenomeWideAssociation","GenomicVariation","GenomeAnnotation","GermlineMutation","HistoneModification","IndelDetection","LinkageDisequilibrium","MetagenomeAssembly","MicrosatelliteDetection","MotifAnnotation","MotifDiscovery","NetworkEnrichment","NetworkInference","NucleosomePositioning","PeakDetection","QuantitativeTrailLocus","Scaffolding","SequenceMatching","SomaticMutation","SplicedAlignment","StructuralVariation","TranscriptomeVariant","VariantAnnotation","VariantDetection","ExperimentalDesign","Alignment","Annotation","BatchEffect","MultipleComparison","Normalization","Pathways","Preprocessing","QualityControl","ReportWriting","Visualization","GenomeBrowsers","MultipleSequenceAlignment","GO","KEGG","Reactome","BioCarta","NCINatureCurated","Network","Bayesian","Classification","Clustering","DecisionTree","DimensionReduction","FeatureExtraction","GraphAndNetwork","HiddenMarkovModel","MultidimensionalScaling","NeuralNetwork","PatternLogic","PrincipalComponent","Regression","StructuralEquationModels","SupportVectorMachine","Survival","TimeCourse","ThirdPartyClient","DataImport","DataRepresentation","GUI","Organism","ChipManufacturer","CustomCDF","CustomArray","CustomDBSchema","FunctionalAnnotation","SequenceAnnotation","ChipName","PackageType","Anopheles_gambiae","Apis_mellifera","Arabidopsis_lyrata","Arabidopsis_thaliana","Asparagus_officinalis","Bacillus_subtilis","Bos_taurus","Caenorhabditis_elegans","Canis_familiaris","Cicer_arietinum","Ciona_intestinalis","Danio_rerio","Drosophila_melanogaster","Eremothecium_gossypii","Escherichia_coli","Gallus_gallus","Gasterosteus_aculeatus","Glycine_max","Homo_sapiens","Hordeum_vulgare","Kluyveromyces_lactis","Macaca_fascicularis","Macaca_mulatta","Magnaporthe_grisea","Medicago_truncatula","Mus_musculus","Neurospora_crassa","Oncorhynchus_mykiss","Oryza_sativa","Pan_troglodytes","Plasmodium_falciparum","Pseudomonas_aeruginosa","Rattus_norvegicus","Saccharomyces_cerevisiae","Saccharum_officinarum","Schizosaccharomyces_pombe","Staphylococcus_aureus","Sus_scrofa","Taeniopygia_guttata","Toxoplasma_gondii","Triticum_aestivum","Vitis_vinifera","Xenopus_laevis","Xenopus_tropicalis","Zea_mays","AffymetrixChip","AgilentChip","ClonetechChip","GEChip","INDACChip","IlluminaChip","QiagenChip","RNG_MRCChip","RocheChip","UniversityHealthNetwork","CodelinkChip","GACustomCDF","MBNICustomCDF","GeneCardsCustomSchema","GenomicSequence","adme16cod","ag","ath1121501","celegans","drosgenome1","drosophila2","h10kcod","h20kcod","hcg110","hgfocus","hgu133a2","hgu133a","hgu133b","hgu133plus2","hgu95a","hgu95av2","hgu95b","hgu95c","hgu95d","hgu95e","hguatlas13k","hgug4100a","hgug4101a","hgug4110b","hgug4111a","hgug4112a","hguqiagenv3","hi16cod","hs25kresogen","hu35ksuba","hu35ksubb","hu35ksubc","hu35ksubd","hu6800","HuO22","hwgcod","indac","illuminaHumanv1","illuminaHumanv2","illuminaMousev1","illuminaMousev1p1","illuminaRatv1","JazaerimetaData","lumiHumanV1","lumiMouseV1","lumiHumanV2","lumiRatV1","m10kcod","m20kcod","mi16cod","mm24kresogen","mgu74a","mgu74av2","mgu74b","mgu74bv2","mgu74c","mgu74cv2","mguatlas5k","mgug4121a","mgug4122a","moe430a","moe430b","mouse4302","mouse430a2","mpedbarray","mu11ksuba","mu11ksubb","mu19ksuba","mu19ksubb","mu19ksubc","Mu15v1","Mu22v3","mwgcod","Norway981","OperonHumanV3","pedbarrayv9","pedbarrayv10","PartheenMetaData","r10kcod","rae230a","rae230b","rat2302","rgu34a","rgu34b","rgu34c","rgug4130a","ri16cod","rnu34","Roberts2005Annotation","rtu34","rwgcod","SHDZ","u133x3p","xenopuslaevis","yeast2","ygs98","zebrafish","hcgi12k","hcgi8k","BSgenome","cdf","ChipDb","db0","InparanoidDb","OrganismDb","OrgDb","PolyPhen","probe","SIFT","SNPlocs","XtraSNPlocs","TxDb","MeSHDb","FRMA","AnnotationHub","EuPathDB","ReproducibleResearch","SpecimenSource","OrganismData","DiseaseModel","TechnologyData","AssayDomainData","RepositoryData","PackageTypeData","Tissue","Proteome","Genome","StemCell","CellCulture","Germline","Somatic","Anopheles_gambiae_Data","Apis_mellifera_Data","Arabidopsis_lyrata_Data","Arabidopsis_thaliana_Data","Bacillus_subtilis_Data","Bos_taurus_Data","Caenorhabditis_elegans_Data","Canis_familiaris_Data","Ciona_intestinalis_Data","Danio_rerio_Data","Drosophila_melanogaster_Data","Eremothecium_gossypii_Data","Escherichia_coli_Data","Gallus_gallus_Data","Gasterosteus_aculeatus_Data","Glycine_max_Data","Homo_sapiens_Data","Hordeum_vulgare_Data","Kluyveromyces_lactis_Data","Macaca_mulatta_Data","Magnaporthe_grisea_Data","Medicago_truncatul_Dataa","Mus_musculus_Data","Neurospora_crassa_Data","Oncorhynchus_mykiss_Data","Oryza_sativa_Data","Pan_troglodytes_Data","Plasmodium_falciparum_Data","Pseudomonas_aeruginosa_Data","Rattus_norvegicus_Data","Saccharomyces_cerevisiae_Data","Saccharum_officinarum_Data","Schizosaccharomyces_pombe_Data","Staphylococcus_aureus_Data","Sus_scrofa_Data","Taeniopygia_guttata_Data","Triticum_aestivum_Data","Vitis_vinifera_Data","Xenopus_laevis_Data","Xenopus_tropicalis_Data","Zea_mays_Data","CancerData","HIVData","COPDData","BreastCancerData","ColonCancerData","KidneyCancerData","LeukemiaCancerData","LungCancerData","OvarianCancerData","ProstateCancerData","FlowCytometryData","HighThroughputImagingData","MassSpectrometryData","qPCRData","MicrotitrePlateAssayData","SAGEData","CGHData","SequencingData","MicroarrayData","ImagingMassSpectrometryData","DNASeqData","RNASeqData","ChIPSeqData","RIPSeqData","MethylSeqData","ExomeSeqData","miRNAData","SangerSeqData","SmallRNAData","MicrobiomeData","SingleCellData","MultiChannelData","OneChannelData","TwoChannelData","MethylationArrayData","GenotypingArrayData","MicroRNAArrayData","mRNAArrayData","ChipOnChipData","ReversePhaseProteinArrayData","TissueMicroarrayData","ProprietaryPlatformsData","CopyNumberVariationData","CpGIslandData","SNPData","ExpressionData","HapMap","GEO","ArrayExpress","NCI","PathwayInteractionDatabase","Project1000genomes","ENCODE","ExperimentHub","ImmunoOncologyData","BasicWorkflow","AnnotationWorkflow","GeneExpressionWorkflow","SingleCellWorkflow","GenomicVariantsWorkflow","EpigeneticsWorkflow","ProteomicsWorkflow","ResourceQueryingWorkflow","DifferentialSplicingWorkflow","ImmunoOncologyWorkflow"],"x":[-0.142809687504831,-0.0349101568808167,-0.427837689859176,0.0710530989184579,0.141954226620336,0.42696849319394,0.361448794461975,0.264272526135511,0.0259919659482768,-0.209003953648044,-0.333102590537546,-0.41740285081883,0.714399294846885,0.714297578123792,0.713992452093999,0.713483989170231,0.712772310021272,0.711857583543325,0.710740026819929,0.709419905070441,0.707897531587095,0.706173267660651,0.704247522494651,0.702120753108309,0.641309292318024,0.635880668795484,0.605992855703297,0.599476284656294,0.592783553773731,0.585916251379268,0.578876007225998,0.571664492109678,0.564283417472212,1,0.999593165293609,0.998915214728585,0.99796630919664,0.99674667389271,0.995256598261516,0.993496435928865,0.991466604617737,0.989167586049145,0.986599925827811,0.98376423331269,0.980661181472346,0.977291506725255,0.973656008765026,0.969755550370624,0.965591057201614,0.96116351757848,0.956473982248079,0.951523564134275,0.946313438073821,0.940844840537543,0.935119069336905,0.929137483316004,0.922901502029096,0.916412605403701,0.909672333389387,0.902682285592309,0.895444120895589,0.887959557065627,0.880230370344441,0.87225839502813,0.864045523031552,0.855593703439339,0.55673453499549,0.549019636185676,0.541140551948051,0.533099152152498,0.524897345189744,0.516537077518457,0.508020333203316,0.499349133444146,0.490525536096251,0.481551635182038,0.472429560394061,0.463161476589606,0.453749583276924,0.444196114093237,0.434503336274656,0.424673550118116,0.414709088435468,0.404612315999856,0.394385628984504,0.384031454394059,0.373552249488609,0.362950501200524,0.352228725544259,0.341389467019249,0.330435298006047,0.319368818155843,0.308192653773517,0.296909457194362,0.285521906154623,0.274032703156024,0.262444574824403,0.250760271262629,0.238982565397944,0.227114252323893,0.215158148636985,0.203117091768265,0.190993939309921,0.178791568337132,0.166512874725265,0.154160772462634,0.141738192958944,0.129248084349611,0.116693410796106,0.104077151782496,0.0914023014083567,0.0786718676782057,0.0658888717876418,0.0530563474063543,0.040177339958168,0.0272549058983032,0.0142921119880135,0.0012920345667824,-0.0117422411777584,-0.0248076219420063,-0.0379010070404929,-0.0510192891417425,-0.0641593550057035,-0.077318086222587,-0.090492359952929,-0.103679049668706,-0.116875025895322,-0.130077156954302,-0.1432823097065,-0.15648735029566,-0.169689144892146,-0.182884560436662,-0.196070465383792,-0.209243730445185,-0.222401229332193,-0.235539839497811,-0.248656442877716,-0.261747926630254,-0.274811183875175,-0.287843114430969,-0.300840625550593,-0.313800632655452,-0.196242851413938,-0.248931743274647,-0.266449890162137,-0.283938694668678,-0.301394006338728,-0.31881168266529,-0.35351760489918,-0.32672006006743,-0.339595841738814,-0.352424921979925,-0.365204256184305,-0.37793081155126,-0.390601567805605,-0.403213517914443,-0.415763668800791,-0.428249042053904,-0.440666674636108,-0.453013619585996,-0.465286946717799,-0.477483743316782,-0.489601114830488,-0.50163618555568,-0.513586099320802,-0.52544802016381,-0.537219133005201,-0.548896644316092,-0.560477782781182,-0.571959799956442,-0.582776625707438,-0.694959108530496,-0.706923448795799,-0.708264733405493,-0.709471823820983,-0.710544433574712,-0.71148230811417,-0.590303271142183,-0.146266295646213,-0.58333997092138,-0.594615594925717,-0.605783996030334,-0.616842523742327,-0.627788553644024,-0.638619488015811,-0.649332756452632,-0.65992581647399,-0.670396154127337,-0.680741284584684,-0.690958752732304,-0.701046133753379,-0.711001033703461,-0.720821090078603,-0.730503972376033,-0.740047382647229,-0.749449056043269,-0.758706761352325,-0.767818301529181,-0.776781514216632,-0.785594272258662,-0.794254484205254,-0.802760094808741,-0.811109085511558,-0.819299474925285,-0.827329319300874,-0.835196712989938,-0.842899788897003,-0.850436718922611,-0.857805714397158,-0.865005026505392,-0.872032946701435,-0.878887807114261,-0.885567980943515,-0.892071882845587,-0.898397969309844,-0.904544739024943,-0.910510733235116,-0.916294536086368,-0.921894774962488,-0.927310120810798,-0.932539288457563,-0.937581036912994,-0.942434169665757,-0.94709753496693,-0.951570026103338,-0.955850581660201,-0.959938185773025,-0.963831868368691,-0.967530705395671,-0.971033819043329,-0.974340377950239,-0.977449597401485,-0.980360739514895,-0.983073113416147,-0.985586075402736,-0.987899029096734,-0.990011425586322,-0.992802891979059,-0.995818618418839,-0.997022993541039,-0.998024646153089,-0.998823338542079,-0.999418881161761,-0.999811132677538,-1,-0.99998543830702,-0.999767451054389,-0.999346089974997,-0.998721455066555,-0.997893694567865,-0.996863004923639,-0.995629630737876,-0.994193864715816,-0.992556047594475,-0.990716568061778,-0.988675862664318,-0.986434415703752,-0.983992759121871,-0.981351472374354,-0.978511182293251,-0.975472562938228,-0.972236335436595,-0.968803267812166,-0.965174174802996,-0.961349917668022,-0.957331403982671,-0.953119587423473,-0.948715467541732,-0.944120089526317,-0.93933454395561,-0.934359966538694,-0.929197537845825,-0.923848483028257,-0.918314071527489,-0.912595616773999,-0.906694475875543,-0.900612049295081,-0.89434978051842,-0.887909155711648,-0.88129170336843,-0.874498993947268,-0.867532639498801,-0.860394293283228,-0.853085649377956,-0.845608442275562,-0.837964446472161,-0.83015547604628,-0.822183384228344,-0.814050062960859,-0.805757442449421,-0.797307490704636,-0.788702213075069,-0.779943651771338,-0.771033885381448,-0.761975028377503,-0.7527692306139,-0.74341867681712,-0.733925586067251,-0.724292211271348,-0.714520838628781,-0.704613787088662,-0.69457340779952,-0.684402083551318,-0.67410222820997,-0.663676286144483,-0.65312673164685,-0.64245606834486,-0.631666828607927,-0.620761572946109,-0.609742889402446,-0.598613392938764,-0.587375724815091,-0.576032551962829,-0.564586566351837,-0.553040484351573,-0.541397046086439,-0.529659014785498,-0.517829176126697,-0.505910337575771,-0.493905327719975,-0.481816995596796,-0.469648210017825,-0.45740185888792,-0.445080848519853,-0.43268810294458,-0.420226563217307,-0.407699186719517,-0.395108946457122,-0.382458830354908,-0.369751840547438,-0.356990992666581,-0.344179315125847,-0.331319848401672,-0.318415644311858,-0.305469765291311,-0.292485283665259,-0.279465280920129,-0.266412846972245,-0.253331079434527,-0.240223082881364,-0.227091968111835,-0.213940851411446,-0.200772853812581,-0.187591100353806,-0.174398719338242,-0.161198841591146,-0.147994599716904,-0.134789127355605,-0.121585558439351,-0.108387026448524,-0.0951966636681345,-0.0820176004444702,-0.0688529644422033,-0.0557058799021246,-0.0425794668997,-0.0498401490694974,-0.0411650588734809,0.155182422809914,0.300351525047478,0.338202392562222,0.399416914937813,0.412745022576831,0.420688455819464,-0.0294768406046128,-0.0164011105414655,-0.00335537985183065,0.00965725544219431,0.0226337071727347,0.0355708957590226,0.0484657509382502,0.061315212494202,0.0741162309835106,0.0868657684593521,0.0995607991924112,0.112198310388955,0.124775302905823,0.137288791962197,0.149735807847942,0.162113396628381,0.174418620845331,0.186648560214218,0.198800312317119,0.210870993291572,0.222857738514971,0.234757703284403,0.246568063491754,0.258286016293928,0.269908780778021,0.281433598621287,0.292857734745749,0.304178477967286,0.315393141639053,0.326499064289081,0.337493610251896,0.348374170294022,0.359138162233199,0.369783031551195,0.380306252000043,0.39070532620157,0.400977786240081,0.411121194248044,0.421133142984646,0.431011256407083,0.440753190234442,0.45035663250405,0.459819304120147,0.469138959394765,0.478313386580679,0.487340408396295,0.496217882542366,0.504943702210405,0.513515796582675,0.521932131323633,0.530190709062726,0.697390440363337,0.709227072558098,0.720861498782151,0.732290957945177,0.743512737599788,0.754524174585246,0.765322655659481,0.538289569868403,0.546226791713248,0.554000490930123,0.56160882265919,0.569049981285747,0.576322200868729,0.583423755559804,0.59710816978445,0.659503582896238,0.78627055040844,0.796414992713864,0.806336537549274,0.816032830326582,0.825501569914682,0.834740509185552,0.843747455547543,0.852520271465732,0.861056874969194,0.869355240145108,0.87741339761954,0.88522943502482,0.892801497453383,0.900127787897982,0.90720656767815,0.914036156852829,0.920614934619052,0.926941339696594,0.933013870698496,0.938831086487377,0.944391606517442,0.949694111162117,0.954737342027224,0.664057981147665,0.668420892989325,0.672591283011873,0.67656816149538,0.68035058464421,0.683937654811009,0.68732852070973,0.690522377617662,0.693518467566411,0.696316079521774,0.698914549552492,0.701313260987804,0.703511644563806,0.422087136502313,0.423351755468574,0.424482012597976,0.425477639657091,0.426338400362996,0.427064090439348,0.427654537664861,0.428109601914179,0.428429175191129,0.428613181654355],"y":[0.14233022730836,-0.122397527307756,0.162448285603347,0.33192950843818,0.165896706422095,0.0983305622407877,-0.126713973213234,-0.258972588718393,-0.403946692139775,-0.425611321788586,-0.396828840288872,-0.359126554578174,0.14233022730836,0.129117788571708,0.115908485420427,0.102705452695748,0.0895118237508001,0.0763307297070015,0.0631652987109776,0.0500186551921891,0.0368939191214381,0.0237942052704365,0.0107226224726049,-0.00231772711471756,-0.204221452497652,-0.216266209487455,-0.275157010593538,-0.286649006707704,-0.298039197164539,-0.309324878835933,-0.320503373395872,-0.331572027956068,-0.342528215695537,0.124713642326157,0.107101238124449,0.0894971944915439,0.0719056892316139,0.0543308971732157,0.0367769891785172,0.0192481311534654,0.00174848305913078,-0.015717802075538,-0.0331465791393132,-0.05053371192241,-0.0678750740980929,-0.0851665502019403,-0.102404036608528,-0.119583442505307,-0.136700690863434,-0.153751719405337,-0.17073248156878,-0.187638947467193,-0.204467104846048,-0.221212960035051,-0.237872538895922,-0.254441887765542,-0.270917074394237,-0.287294188878981,-0.303569344591302,-0.319738679099655,-0.33579835508606,-0.351744561256773,-0.36757351324679,-0.38328145451795,-0.398864657250446,-0.414319423227504,-0.353369336484001,-0.364092817498955,-0.374696113836247,-0.385176709114038,-0.395532116069993,-0.405759877151558,-0.41585756509919,-0.425822783522393,-0.435653167468433,-0.445346383983592,-0.454900132666822,-0.464312146215683,-0.473580190964416,-0.482702067414039,-0.491675610754339,-0.500498691377618,-0.509169215384099,-0.51768512507885,-0.526044399460117,-0.534245054698951,-0.542285144610011,-0.550162761113431,-0.557876034687654,-0.565423134813101,-0.572802270406596,-0.580011690246424,-0.587049683387934,-0.593914579569582,-0.600604749609318,-0.607118605791221,-0.613454602242304,-0.619611235299377,-0.625587043865899,-0.631380609758724,-0.636990558044668,-0.642415557366809,-0.647654320260444,-0.652705603458632,-0.657568208187248,-0.662240980449473,-0.666722811299667,-0.67101263710654,-0.675109439805578,-0.679012247140644,-0.682720132894725,-0.68623221710973,-0.689547666295334,-0.692665693626773,-0.695585559131583,-0.6983065698652,-0.700828080075421,-0.703149491355646,-0.705270252786896,-0.70718986106856,-0.708907860637831,-0.710423843777829,-0.711737450714356,-0.712848369701277,-0.713756337094508,-0.71446113741458,-0.714962603397779,-0.71526061603584,-0.715355104604193,-0.715246046678744,-0.714933468141196,-0.71441744317291,-0.7136980942373,-0.712775592050768,-0.711650155542189,-0.710322051800957,-0.708791596013598,-0.707059151388969,-0.705125129072063,-0.702989988046432,-0.700654235025266,-0.698118424331136,-1,-0.996310283159039,-0.99453947817073,-0.992498870454251,-0.990188944288079,-0.987610247865537,-0.981649055803142,-0.695383157764441,-0.692449084460591,-0.689316900735951,-0.685987349922595,-0.682461222191892,-0.678739354366987,-0.674822629724203,-0.670711977783421,-0.666408374087487,-0.661912839970696,-0.657226442316406,-0.652350293303846,-0.647285550144171,-0.642033414805837,-0.636595133729341,-0.630971997531423,-0.625165340698768,-0.619176541271313,-0.613007020515203,-0.606658242585496,-0.600131714178693,-0.22257797634884,-0.00510164213915321,0.0508646473975021,0.0595703682243536,0.0682957296799036,0.0770386610551572,0.085797087471414,0.497957797429668,0.714110075769453,-0.593428984175164,-0.586551643271567,-0.579501323603342,-0.572279698357375,-0.564888481374912,-0.557329426744832,-0.549604328387363,-0.541715019628347,-0.53366337276416,-0.525451298617369,-0.517080746083259,-0.508553701667319,-0.499872189013803,-0.491038268425475,-0.482054036374663,-0.472921625005714,-0.463643201628999,-0.454220968206562,-0.444657160829548,-0.434954049187535,-0.425113936029889,-0.415139156619273,-0.40503207817744,-0.394795099323444,-0.384430649504397,-0.373941188418913,-0.363329205433365,-0.352597218991115,-0.341747776014827,-0.330783451302035,-0.31970684691409,-0.308520591558633,-0.297227339965752,-0.285829772257957,-0.274330593314135,-0.262732532127622,-0.251038341158559,-0.239250795680674,-0.227372693122656,-0.215406852404263,-0.203356113267337,-0.191223335601872,-0.179011398767304,-0.166723200909179,-0.154361658271366,-0.141929704503967,-0.129430289967103,-0.116866381030735,-0.104240959370679,-0.0915570212609955,-0.0788175768629098,-0.0660256495104389,-0.0531842749928928,-0.0402965008344175,-0.027365385570755,-0.0143939980233885,-0.00138541657124913,0.0116572715798486,0.0312784808656754,0.0575305175529408,0.0706882819048382,0.0838630483742413,0.0970516903161889,0.110251077792782,0.123458078315985,0.136669557591027,0.149882380260238,0.163093410647132,0.176299513500569,0.189497554738813,0.202684402193321,0.215856926352061,0.229012001102219,0.242146504472086,0.255257319371969,0.268341334333938,0.281395444250244,0.294416551110225,0.307401564735527,0.320347403513469,0.333250995128372,0.346109277290685,0.358919198463727,0.371677718587882,0.384381809802068,0.39702845716231,0.409614659357251,0.42213742942042,0.434593795439109,0.446980801259663,0.459295507189037,0.471534990692452,0.483696347086964,0.495776690230814,0.507773153208362,0.519682889010472,0.53150307121016,0.543230894633369,0.554863576024692,0.566398354707897,0.57783249324109,0.589163278066372,0.600388020153819,0.611504055639645,0.622508746458392,0.633399480968996,0.644173674574587,0.654828770335866,0.665362239577918,0.675771582490322,0.686054328720403,0.696208037959502,0.706230300522113,0.716118737917746,0.725871003415395,0.735484782600468,0.744957793924041,0.75428778924432,0.763472554360172,0.772509909536597,0.78139771002203,0.790133846557329,0.798716245876349,0.807142871197972,0.815411722709476,0.82352083804113,0.831468292731909,0.839252200686206,0.846870714621441,0.854322026506461,0.861604367990624,0.868716010823462,0.875655267264831,0.882420490485453,0.889010074957734,0.895422456836794,0.9016561143316,0.907709568066113,0.913581381430384,0.919270160921483,0.924774556474208,0.930093261781489,0.935225014604393,0.940168597071685,0.944922835968855,0.949486603016539,0.953858815138293,0.958038434717622,0.962024469844231,0.965815974549425,0.969412049030607,0.972811839864819,0.976014540211281,0.979019390002863,0.981825676126475,0.984432732592295,0.986839940691825,0.989046729144725,0.991052574234388,0.992856999932228,0.994459578010654,0.995859928144694,0.997057718002258,0.998052663323004,0.998844527985799,0.999433124064758,0.99981831187384,1,0.999978145324883,0.999752753035055,0.999323876620775,0.998691617863299,0.997856126810727,0.996817601742389,0.995576289121794,0.994132483538138,0.706503259261507,0.705003344783836,0.630229922986316,0.503348103000295,0.45106028215293,0.322905769706268,0.276344811643855,0.237518999698674,0.99248652763639,0.990638812035977,0.988589775238081,0.986339903521573,0.983889730827613,0.98123983863293,0.978390855811824,0.97534345848693,0.97209836986875,0.968656360084027,0.965018245992974,0.961184890995417,0.957157204825896,0.95293614333776,0.948522708276328,0.943917947041151,0.939122952437446,0.934138862416748,0.928966859806848,0.92360817203109,0.918064070817074,0.912335871894848,0.906424934684659,0.900332661974337,0.894060499586379,0.88760993603483,0.880982502172028,0.874179770825295,0.86720335642368,0.860054914614817,0.852736141872007,0.845248775091611,0.837594591180843,0.829775406636083,0.821793077111775,0.813649496980045,0.805346598881131,0.796886353264723,0.788270767922336,0.779501887510819,0.770581793067116,0.761512601514395,0.752296465159652,0.742935571182931,0.733432141118259,0.723788430326424,0.714006727459743,0.704089353918903,0.694038663302058,0.683857040846263,0.67354690286141,0.917607831332446,0.904565560843537,0.891342396122417,0.877941475299958,0.864365978692231,0.85061912804576,0.836704185772932,0.663110696156789,0.652550897460406,0.64187001283121,0.631070577064347,0.620155153089605,0.609126331363175,0.597986729252885,0.57538578417703,0.444326850426801,0.80838327467216,0.79398402698301,0.77943012835002,0.764725032714781,0.74987222990106,0.734875244786599,0.719737636466587,0.70446299740901,0.689054952602089,0.673517158693991,0.65785330312503,0.642067103252568,0.626162305468799,0.610142684311662,0.594012041569058,0.577774205376611,0.561433029309172,0.544992391466281,0.528456193551821,0.511828359948058,0.495112836784304,0.478313591000422,0.461434609405386,0.431924701990956,0.419453826788133,0.406917184415318,0.394317750077407,0.381658513881129,0.368942480125435,0.356172666588512,0.343352103811602,0.330483834379796,0.317570912199959,0.304616401775979,0.291623377481496,0.278594922830289,0.228822325794907,0.22010512552473,0.211369467660318,0.202617425354215,0.193851075647338,0.185072498976039,0.176283778678388,0.167487000499744,0.15868425209777,0.149877622546987]},"edges":{"from":["BiocViews","BiocViews","BiocViews","BiocViews","Software","Software","Software","Software","Software","Software","Software","AnnotationData","AnnotationData","AnnotationData","AnnotationData","AnnotationData","AnnotationData","AnnotationData","AnnotationData","AnnotationData","ExperimentData","ExperimentData","ExperimentData","ExperimentData","ExperimentData","ExperimentData","ExperimentData","ExperimentData","Workflow","Workflow","Workflow","Workflow","Workflow","Workflow","Workflow","Workflow","Workflow","Workflow","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","AssayDomain","Technology","Technology","Technology","Technology","Technology","Technology","Technology","Technology","Technology","Technology","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","ResearchField","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","BiologicalQuestion","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","WorkflowStep","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","StatisticalMethod","Infrastructure","Infrastructure","Infrastructure","Infrastructure","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Sequencing","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","Microarray","MassSpectrometry","Alignment","Pathways","Pathways","Pathways","Pathways","Pathways","Visualization","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","Organism","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","ChipManufacturer","CustomCDF","CustomCDF","CustomDBSchema","SequenceAnnotation","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","ChipName","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","PackageType","SpecimenSource","SpecimenSource","SpecimenSource","SpecimenSource","SpecimenSource","SpecimenSource","SpecimenSource","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","OrganismData","DiseaseModel","DiseaseModel","DiseaseModel","TechnologyData","TechnologyData","TechnologyData","TechnologyData","TechnologyData","TechnologyData","TechnologyData","TechnologyData","TechnologyData","AssayDomainData","AssayDomainData","AssayDomainData","AssayDomainData","RepositoryData","RepositoryData","RepositoryData","RepositoryData","RepositoryData","RepositoryData","RepositoryData","PackageTypeData","PackageTypeData","CancerData","CancerData","CancerData","CancerData","CancerData","CancerData","CancerData","MassSpectrometryData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","SequencingData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData","MicroarrayData"],"to":["Software","AnnotationData","ExperimentData","Workflow","AssayDomain","Technology","ResearchField","BiologicalQuestion","WorkflowStep","StatisticalMethod","Infrastructure","Organism","ChipManufacturer","CustomCDF","CustomArray","CustomDBSchema","FunctionalAnnotation","SequenceAnnotation","ChipName","PackageType","ReproducibleResearch","SpecimenSource","OrganismData","DiseaseModel","TechnologyData","AssayDomainData","RepositoryData","PackageTypeData","BasicWorkflow","AnnotationWorkflow","GeneExpressionWorkflow","SingleCellWorkflow","GenomicVariantsWorkflow","EpigeneticsWorkflow","ProteomicsWorkflow","ResourceQueryingWorkflow","DifferentialSplicingWorkflow","ImmunoOncologyWorkflow","aCGH","CellBasedAssays","ChIPchip","CopyNumberVariation","CpGIsland","DNAMethylation","ExonArray","GeneExpression","GeneticVariability","SNP","Transcription","Sequencing","Microarray","FlowCytometry","MassSpectrometry","qPCR","MicrotitrePlateAssay","SAGE","CRISPR","SingleCell","ddPCR","CellBiology","Genetics","Metabolomics","Metagenomics","Proteomics","Lipidomics","Epigenetics","Pharmacogenomics","Pharmacogenetics","Cheminformatics","StructuralGenomics","StructuralPrediction","Biophysics","MathematicalBiology","BiomedicalInformatics","ComparativeGenomics","FunctionalGenomics","SystemsBiology","ComputationalChemistry","Agroinformatics","Transcriptomics","ImmunoOncology","GenePrediction","GeneRegulation","GeneSetEnrichment","GeneSignaling","GeneTarget","GenomeAssembly","GenomeWideAssociation","GenomicVariation","GenomeAnnotation","GermlineMutation","HistoneModification","IndelDetection","LinkageDisequilibrium","MetagenomeAssembly","MicrosatelliteDetection","MotifAnnotation","MotifDiscovery","NetworkEnrichment","NetworkInference","NucleosomePositioning","PeakDetection","QuantitativeTrailLocus","Scaffolding","SequenceMatching","SomaticMutation","SplicedAlignment","StructuralVariation","TranscriptomeVariant","VariantAnnotation","VariantDetection","AlternativeSplicing","Coverage","DemethylateRegionDetection","DenovoAssembler","DifferentialExpression","DifferentialMethylation","DifferentialPeakCalling","DifferentialSplicing","DNA3DStructure","DriverMutation","FunctionalPrediction","GeneFusionDetection","ExperimentalDesign","Alignment","Annotation","BatchEffect","MultipleComparison","Normalization","Pathways","Preprocessing","QualityControl","ReportWriting","Visualization","GenomeBrowsers","Bayesian","Classification","Clustering","DecisionTree","DimensionReduction","FeatureExtraction","GraphAndNetwork","HiddenMarkovModel","MultidimensionalScaling","NeuralNetwork","PatternLogic","PrincipalComponent","Regression","StructuralEquationModels","SupportVectorMachine","Survival","TimeCourse","ThirdPartyClient","DataImport","DataRepresentation","GUI","ATACSeq","DNASeq","DNaseSeq","RiboSeq","RNASeq","ChIPSeq","RIPSeq","MethylSeq","ExomeSeq","miRNA","SangerSeq","SmallRNA","Microbiome","WholeGenome","DenovoGenome","TargetedResequencing","DenovoTranscriptome","MicrobialStrain","HiC","PooledScreens","MNaseSeq","MultiChannel","OneChannel","TwoChannel","MethylationArray","GenotypingArray","MicroRNAArray","mRNAMicroarray","ChipOnChip","ReversePhaseProteinArray","TissueMicroarray","ProprietaryPlatforms","ImagingMassSpectrometry","MultipleSequenceAlignment","GO","KEGG","Reactome","BioCarta","NCINatureCurated","Network","Anopheles_gambiae","Apis_mellifera","Arabidopsis_lyrata","Arabidopsis_thaliana","Asparagus_officinalis","Bacillus_subtilis","Bos_taurus","Caenorhabditis_elegans","Canis_familiaris","Cicer_arietinum","Ciona_intestinalis","Danio_rerio","Drosophila_melanogaster","Eremothecium_gossypii","Escherichia_coli","Gallus_gallus","Gasterosteus_aculeatus","Glycine_max","Homo_sapiens","Hordeum_vulgare","Kluyveromyces_lactis","Macaca_fascicularis","Macaca_mulatta","Magnaporthe_grisea","Medicago_truncatula","Mus_musculus","Neurospora_crassa","Oncorhynchus_mykiss","Oryza_sativa","Pan_troglodytes","Plasmodium_falciparum","Pseudomonas_aeruginosa","Rattus_norvegicus","Saccharomyces_cerevisiae","Saccharum_officinarum","Schizosaccharomyces_pombe","Staphylococcus_aureus","Sus_scrofa","Taeniopygia_guttata","Toxoplasma_gondii","Triticum_aestivum","Vitis_vinifera","Xenopus_laevis","Xenopus_tropicalis","Zea_mays","AffymetrixChip","AgilentChip","ClonetechChip","GEChip","INDACChip","IlluminaChip","QiagenChip","RNG_MRCChip","RocheChip","UniversityHealthNetwork","CodelinkChip","GACustomCDF","MBNICustomCDF","GeneCardsCustomSchema","GenomicSequence","moe430b","mouse4302","mouse430a2","mpedbarray","mu11ksuba","mu11ksubb","mu19ksuba","mu19ksubb","mu19ksubc","Mu15v1","Mu22v3","mwgcod","Norway981","OperonHumanV3","pedbarrayv9","pedbarrayv10","PartheenMetaData","r10kcod","rae230a","rae230b","rat2302","rgu34a","rgu34b","rgu34c","rgug4130a","ri16cod","rnu34","Roberts2005Annotation","rtu34","rwgcod","SHDZ","u133x3p","xenopuslaevis","yeast2","ygs98","zebrafish","hcgi12k","hcgi8k","adme16cod","ag","ath1121501","celegans","drosgenome1","drosophila2","h10kcod","h20kcod","hcg110","hgfocus","hgu133a2","hgu133a","hgu133b","hgu133plus2","hgu95a","hgu95av2","hgu95b","hgu95c","hgu95d","hgu95e","hguatlas13k","hgug4100a","hgug4101a","hgug4110b","hgug4111a","hgug4112a","hguqiagenv3","hi16cod","hs25kresogen","hu35ksuba","hu35ksubb","hu35ksubc","hu35ksubd","hu6800","HuO22","hwgcod","indac","illuminaHumanv1","illuminaHumanv2","illuminaMousev1","illuminaMousev1p1","illuminaRatv1","JazaerimetaData","lumiHumanV1","lumiMouseV1","lumiHumanV2","lumiRatV1","m10kcod","m20kcod","mi16cod","mm24kresogen","mgu74a","mgu74av2","mgu74b","mgu74bv2","mgu74c","mgu74cv2","mguatlas5k","mgug4121a","mgug4122a","moe430a","BSgenome","cdf","ChipDb","db0","InparanoidDb","OrganismDb","OrgDb","PolyPhen","probe","SIFT","SNPlocs","XtraSNPlocs","TxDb","MeSHDb","FRMA","AnnotationHub","EuPathDB","Tissue","Proteome","Genome","StemCell","CellCulture","Germline","Somatic","Anopheles_gambiae_Data","Apis_mellifera_Data","Arabidopsis_lyrata_Data","Arabidopsis_thaliana_Data","Bacillus_subtilis_Data","Bos_taurus_Data","Caenorhabditis_elegans_Data","Canis_familiaris_Data","Ciona_intestinalis_Data","Danio_rerio_Data","Drosophila_melanogaster_Data","Eremothecium_gossypii_Data","Escherichia_coli_Data","Gallus_gallus_Data","Gasterosteus_aculeatus_Data","Glycine_max_Data","Homo_sapiens_Data","Hordeum_vulgare_Data","Kluyveromyces_lactis_Data","Macaca_mulatta_Data","Magnaporthe_grisea_Data","Medicago_truncatul_Dataa","Mus_musculus_Data","Neurospora_crassa_Data","Oncorhynchus_mykiss_Data","Oryza_sativa_Data","Pan_troglodytes_Data","Plasmodium_falciparum_Data","Pseudomonas_aeruginosa_Data","Rattus_norvegicus_Data","Saccharomyces_cerevisiae_Data","Saccharum_officinarum_Data","Schizosaccharomyces_pombe_Data","Staphylococcus_aureus_Data","Sus_scrofa_Data","Taeniopygia_guttata_Data","Triticum_aestivum_Data","Vitis_vinifera_Data","Xenopus_laevis_Data","Xenopus_tropicalis_Data","Zea_mays_Data","CancerData","HIVData","COPDData","FlowCytometryData","HighThroughputImagingData","MassSpectrometryData","qPCRData","MicrotitrePlateAssayData","SAGEData","CGHData","SequencingData","MicroarrayData","CopyNumberVariationData","CpGIslandData","SNPData","ExpressionData","HapMap","GEO","ArrayExpress","NCI","PathwayInteractionDatabase","Project1000genomes","ENCODE","ExperimentHub","ImmunoOncologyData","BreastCancerData","ColonCancerData","KidneyCancerData","LeukemiaCancerData","LungCancerData","OvarianCancerData","ProstateCancerData","ImagingMassSpectrometryData","DNASeqData","RNASeqData","ChIPSeqData","RIPSeqData","MethylSeqData","ExomeSeqData","miRNAData","SangerSeqData","SmallRNAData","MicrobiomeData","SingleCellData","MultiChannelData","OneChannelData","TwoChannelData","MethylationArrayData","GenotypingArrayData","MicroRNAArrayData","mRNAArrayData","ChipOnChipData","ReversePhaseProteinArrayData","TissueMicroarrayData","ProprietaryPlatformsData"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false,"size":20},"manipulation":{"enabled":false},"edges":{"smooth":false},"physics":{"stabilization":false}},"groups":null,"width":"100%","height":null,"idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","igraphlayout":{"type":"square"}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="provenance" class="section level1">
<h1 class="hasAnchor">
<a href="#provenance" class="anchor"></a>Provenance</h1>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb29-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/sessionInfo">sessionInfo</a></span>()</a></code></pre></div>
<pre><code>## R Under development (unstable) (2019-01-14 r75992)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS Mojave 10.14.2
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] biocViews_1.51.14  visNetwork_2.0.5   igraph_1.2.2      
## [4] bindrcpp_0.2.2     dplyr_0.7.8        BiocPkgTools_1.1.5
## [7] htmlwidgets_1.3    knitr_1.21         BiocStyle_2.11.0  
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_0.2.5    xfun_0.4            purrr_0.2.5        
##  [4] stats4_3.6.0        htmltools_0.3.6     yaml_2.2.0         
##  [7] XML_3.98-1.16       RBGL_1.59.1         utf8_1.1.4         
## [10] rlang_0.3.1         pkgdown_1.3.0       pillar_1.3.1       
## [13] glue_1.3.0          BiocGenerics_0.29.1 bindr_0.1.1        
## [16] stringr_1.3.1       commonmark_1.7      rvest_0.3.2        
## [19] memoise_1.1.0       evaluate_0.12       Biobase_2.43.1     
## [22] rex_1.1.2           RUnit_0.4.32        curl_3.3           
## [25] parallel_3.6.0      fansi_0.4.0         Rcpp_1.0.0         
## [28] readr_1.3.1         backports_1.1.3     DT_0.5             
## [31] BiocManager_1.30.4  desc_1.2.0          graph_1.61.0       
## [34] jsonlite_1.6        fs_1.2.6            hms_0.4.2          
## [37] digest_0.6.18       stringi_1.2.4       bookdown_0.9       
## [40] rprojroot_1.3-2     bitops_1.0-6        cli_1.0.1          
## [43] tools_3.6.0         magrittr_1.5        RCurl_1.95-4.11    
## [46] lazyeval_0.2.1      tibble_2.0.1        crayon_1.3.4       
## [49] tidyr_0.8.2         pkgconfig_2.0.2     MASS_7.3-51.1      
## [52] xml2_1.2.0          assertthat_0.2.0    rmarkdown_1.11     
## [55] roxygen2_6.1.1      httr_1.4.0          rstudioapi_0.9.0   
## [58] R6_2.4.0            compiler_3.6.0</code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li>
<a href="#build-reports">Build reports</a><ul class="nav nav-pills nav-stacked">
<li><a href="#personal-build-report">Personal build report</a></li>
      </ul>
</li>
      <li><a href="#download-statistics">Download statistics</a></li>
      <li><a href="#package-details">Package details</a></li>
      <li><a href="#package-explorer">Package Explorer</a></li>
      <li>
<a href="#dependency-graphs">Dependency graphs</a><ul class="nav nav-pills nav-stacked">
<li><a href="#working-with-dependency-graphs">Working with dependency graphs</a></li>
      <li><a href="#graph-visualization">Graph visualization</a></li>
      <li><a href="#integration-with-biocviews">Integration with <em><a href="https://bioconductor-org/packages/3-9/BiocViews">BiocViews</a></em></a></li>
      </ul>
</li>
      <li><a href="#provenance">Provenance</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Shian Su, Sean Davis.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
